---
layout: group
author: esmb
day: "Monday (Tuesday)"
subgroup: "CDEV"
title: "CDEV Subgroup Contributed Talks"
code: "CT01"
permalink: /CDEV/CDEV-CT01.html
author1: "Denis Patterson"
inst1: "Princeton University"
title1: "A Mathematical Model of Neuronal Identity with Ectopic Domains"
abstract1: "Recent experiments studying the development of cortical structures in mice have identified COUP-TF1 as a crucial determinant of both the position and sharpness of the boundary between the neo and entorhinal cortices. When COUP-TF1 is under expressed, neocortex invades into territory occupied by the entorhinal cortex in wild-type mice, but the sharp boundary between cortical regions is maintained. However, if COUP-TF1 is over-expressed, the boundary fractures and entorhinal cortex invades the neocortical domain, resulting in mice with ectopic regions of misplaced cortex.We introduce a novel PDE model based on a Keller-Segel-type chemotaxis mechanism to account for both the sharp cortical boundaries of wild-type mice and the ectopic regions observed in mutant mice. Competition between entorhinal and neocortical progenitor cells is mediated by a gradient of COUP-TF1 across the spatial domain and chemotaxis operators model each cell's affinity for cells of their own type. We verify the well-posedness of the system and establish necessary conditions for pattern forming Turing bifurcations; we also numerically study the structure of the Turing space and its dependence on model parameters. Numerical simulations show excellent agreement with experimental observations and we present experimental data verifying the differential adhesion hypothesis underpinning the model's phenomenology."
author2: "Yoshito Hirata"
inst2: "University of Tsukuba"
title2: "Reconstructing 3D chromosome structures from single diploid cell Hi-C data via  recurrence plots"
abstract2: "Previously, we have proposed a method for reconstructing 3D chromosome structures from single haploid cell Hi-C data by regarding a contact map as a recurrence plot and applying a method for converting a recurrence plot back to its original time series (Hirata, Oda, Ohta, and Aihara, Sci. Rep. 2016). Here, we extend our previous method to single diploid cell Hi-C data. We discuss that the reconstructed 3D chromosome structures are consistent mathematically as well as biologically. We will start our presentation with a small intuitive quiz for understanding what kind of question we have to solve. The research of Y.H. was partially supported by AMED under Grant Number JP21gm1310004."
author3: "Dan Tudor"
inst3: "University of Edinburgh"
title3: "Inferring chemoattractant properties from cell tracking data using mathematical modelling and Bayesian inference"
abstract3: "The rapid recruitment of immune cells during the inflammatory response is vital to dealing with injury or infection. Immune cells are guided by chemoattractants produced at the wound site. Visualising the underlying chemoattractant gradient can be experimentally complex. In comparison, the cells response to the chemoattractant gradient can be captured more easily via their trajectories. Thus, we are faced with the inverse problem of inferring the chemoattractant gradient from the observed cell movements, which are also subject to noise.  We use an established mathematical framework to model cell migration as a biased persistent random walk, and chemoattractant production and diffusion using a reaction-diffusion equation. By applying Bayesian inference, we can infer the underlying chemoattractant properties. We apply this framework to analyse different wound conditions, to answer if immune cell recruitment can be explained by a single chemoattractant model. We also use Bayesian model comparison to compare different chemoattractant production and release dynamics. Furthermore, we extend the model to infer subpopulations of immune cells with different migratory behaviour without labelling."
author4: "Philipp Thomas"
inst4: "Imperial College London"
title4: "Exact solutions for stochastic gene expression in growing cell populations"
abstract4: "The chemical master equation and the stochastic simulation algorithm are widely used to model reaction kinetics inside living cells. It is sometimes assumed that cell growth and division can be modelled through a  chemical master equation with effective dilution reactions and extrinsic noise sources. We here re-examine this paradigm by developing an analytical agent-based framework of growing and dividing cells. Apart from the common intrinsic noise contribution the theory predicts extrinsic noise without the need to introduce fluctuating rate constants. Instead, extrinsic fluctuations arise from the population structure of a growing cell population that includes cell cycle fluctuations, differences in cell age and cell size variability. We show that, surprisingly, the solution of the chemical master equation - including effective dilution reactions and static extrinsic noise - exactly agrees with the agent-based formulation when the network under study exhibits stochastic concentration homeostasis, a novel condition that generalises concentration homeostasis in deterministic systems to higher order moments and distributions. We illustrate that this result allows us to exactly solve agent-based models for a range of common gene expression networks inside growing cells."
---
