<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Posters-02 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Posters-02 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Posters-02" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/PS02/" />
<meta property="og:url" content="http://localhost:4000/PS02/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/PS02/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Posters-02","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Posters-02">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Posters-02" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Posters-02" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Posters-02</h1>
        <h4>Tuesday, June 15 at 03:15pm (PDT)</br>Tuesday, June 15 at 11:15pm (BST)</br>Wednesday, June 16 07:15am (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="PS02">Posters-02</h1>
		                
		                
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-11 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-11.html">
											    	</br><strong>Georgia Pope</strong>
									    	 
									    		<em>Harvey Mudd College</em>
									    	
									    	
									    		
									    			</br>"Stability Analysis of a Mathematical Model of Hormonal Contraception"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Combination oral contraceptives (COCs), containing a combination of synthetic progestin and estrogen, have become a leading form of contraception in the United States. The pill is taken cyclically, meaning it is taken regularly during a certain “on” period, followed by a shorter “off” period during which menstruation occurs. In order for the pill to be effective, it must be taken daily during the “on” period and it is recommended that it be taken around the same time every day. This requirement poses a challenge to many users and can result in unwanted pregnancies. We explore the stability of the contraceptive state achieved by hormonal birth control using a mechanistic mathematical model of the menstrual cycle. Specifically, we build off a model by Wright and colleagues, in which the authors model concentrations of exogenous progestin and estrogen as a constant. We include the dynamics of the on/off dosing of COC's by introducing a time-dependent function for exogenous estrogen and progestin and investigate the stability of the model in response to changes in exogenous hormone dosing. Accurately modeling COC dosing could provide insight into when a contraceptive state has been lost due to inconsistency or changes in hormonal birth control use.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-12 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-12.html">
											    	</br><strong>Frederick Laud Amoah-Darko</strong>
									    	 
									    		<em>Clarkson University</em>
									    	
									    	
									    		
									    			</br>"Continuous Model for Microtubule Dynamic instability with Pausing"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Microtubules (MTs) are protein polymers found in all eukaryotic cells. They are crucial for normal cell development, providing structural support for the cell and aiding in the transportation of proteins and organelles. In order to perform these functions, MTs go through random periods of relatively slow polymerization (growth) and very fast depolymerization (shrinkage), a unique type of dynamics called dynamic instability. The onset of a MT shortening event is called a catastrophe, while the event at which a MT starts to grow again is called a rescue. Although MT dynamic instability has traditionally been described solely in terms of growth and shortening, MTs have also been shown to pause for extended periods of time. Here, we present a novel mathematical model to describe the population dynamics of MTs. The goal is to use this model to determine MT catastrophe rates, in addition to time spent growing, shortening and pausing. These are quantities that can be used to compare our model results with experimental findings.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-13 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-13.html">
											    	</br><strong>Victor Matveev</strong>
									    	 
									    		<em>New Jersey Institute of Technology, Dept of Math Sciences</em>
									    	
									    	
									    		
									    			</br>"Approximations of stationary calcium nanodomains in the presence of buffers with two binding sites"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Calcium ion (Ca2+) elevations near open Ca2+ channels, termed Ca2+ nanodomains, trigger secretory vesicle fusion, myocyte contraction, and other fundamental physiological processes. Ca2+ nanodomains are shaped by the interplay between Ca2+ influx, diffusion, and binding to Ca2+ buffers and sensors, which absorb most of the Ca2+ entering the cell. The dependence of Ca2+ concentration on the distance from the Ca2+ channel can be modeled with reasonable accuracy using closed-form approximations of quasi-stationary solutions of the corresponding reaction-diffusion equations. Such stationary approximations help to reveal the qualitative characteristics of Ca2+ nanodomains without resorting to computationally expensive numerical simulations. Although a variety of nanodomain approximations are known when Ca2+ is diffusing in the presence of Ca2+ buffers with a single Ca2+ binding site, most biological buffers have more complex Ca2+-binding stoichiometry. We present several closed-form approximations of Ca2+ nanodomains in the presence of buffers with two binding sites, extending prior work on stationary Ca2+ nanodomains. Our approximants interpolate between the short-range and long-range distance-dependence of Ca2+ concentration using a combination of rational and exponential functions. We shows that this method achieves significant accuracy for a very wide range of Ca2+ buffering parameter values. Supported by NSF DMS-1517085 (V.M)
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-14 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-14.html">
											    	</br><strong>Jia Lu</strong>
									    	 
									    		<em>Duke University</em>
									    	
									    	
									    		
									    			</br>"Distributed information encoding and decoding using self-organized spatial patterns"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Biology can generate distinct self-organized patterns according to different initial conditions, and one could infer the corresponding condition given a pattern. Moreover, under the same or similar conditions, these patterns share global similarity but vary in detail due to random noise. Here, we leverage the above properties of bacterial colony patterns and combine with machine learning (ML) to achieve distributed information encoding and decoding with guaranteed security. Specifically, to encode, a message is converted into cell seeding configuration followed by colony growth, during which a colony pattern develops; to decode, we input the pattern into a trained CNN to convert it back to the original message. By modulating the patterning dynamic and encoding setup, we could tune the trade-off among encoding capacity, decoding accuracy and security, characterized by ML decoding performance. We also implemented ensemble techniques for enhancing decoding reliability and making full use of the expensive-to-obtain patterning data, and combined the framework with established cryptography techniques (e.g., encryption and hashing) to further enhance the security. Our method is applicable for a wide variety of pattern-formation systems and demonstrates a novel way of utilizing biological noise, as well as quantifying the extent of convergence for dynamical systems by using ML.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-15 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-15.html">
											    	</br><strong>J. Cody Herron</strong>
									    	 
									    		<em>UNC Chapel Hill, Bioinformatics and Computational Biology</em>
									    	
									    	
									    		
									    			</br>"Quantification and modeling of podosomes during frustrated phagocytosis"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Podosomes are complex, actin-rich cellular adhesion structures important for migration, motility, tumor invasion, and more. One system in which podosomes play a crucial role is in phagocytosis, the recognition and engulfment of small particles by cells. To observe podosome dynamics, we use the experimental system of “frustrated” phagocytosis, in which cells attempt to engulf fixed, micropatterned discs of antibody. This process is frustrated because cells can recognize the antibody and engage in signaling yet are unable to fully engulf and internalize the fixed discs of antibody. This system is advantageous for studying both the physical structure of podosomes and the highly dynamic spatiotemporal signaling that occurs during phagocytosis. Strikingly, we observe as podosomes form rosettes (puncta in a ring) around the discs. We use computational approaches, including persistent homology (a type of topological data analysis), to automatically identify podosomes and quantify their nanoarchitecture from 3D super-resolution microscopy data. Furthermore, we use reaction-diffusion models to investigate the molecular mechanisms that generate the rosette patterns formed during frustrated phagocytosis.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-16 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-16.html">
											    	</br><strong>Alireza Ramezani</strong>
									    	 
									    		<em>UCR, Physics department</em>
									    	
									    	
									    		
									    			</br>"Subcellular Mechanochemical Model to Study Growth Regulation"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Growth regulation is an important question in developmental biology and remains unclear for many living systems. Abnormal development and fatal diseases, such as cancer, can be result of uncontrolled tissue growth. The Drosophila wing disc, an epithelial primordial organ that later forms the adult fruit fly wing, is appropriate to study growth regulation because of its relatively simple geometry, limited number of cells, rapid growth, and a well understood molecular signaling network. Nevertheless, the mechanism of growth regulation in Drosophila wing disc tissue remains a subject of intense debate. Multiple mechanisms for growth regulation have been proposed, following the substantial evidence that suggests morphogens regulate growth. However, most existing models focus on either the biochemical signaling pathway or mechanical properties. Very few attempt to incorporate both factors in a mechanistic way. Here we developed a coupled mechanochemical model at sub-cellular level to study growth regulation controlled by the morphogen gradient with cell mechanics taken into account to achieve spatial homogeneous growth and the asymmetric shape of the tissue. The model suggested the shape of the morphogen gradient affected the tissue growth rate and final shape. Moreover, the feedback regulation on the morphogen facilitated the tissue growth through shaping the gradient.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-17 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-17.html">
											    	</br><strong>Daniel Cruz</strong>
									    	 
									    		<em>Georgia Institute of Technology</em>
									    	
									    	
									    		
									    			</br>"Agent-based Modeling of Emergent Patterns Within Stem Cell Colonies"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The differentiation of stem cell colonies into specified tissue types is possible through local and long-distance intercellular communication; however, it is unclear which mechanisms take priority in context-specific situations. Here we consider human induced pluripotent stem cells (hiPSCs) whose therapeutic potential arises from their ability to differentiate into all germ lineages. Prior work in the literature suggests that both cell-autonomous and non-autonomous  (e.g. positional) mechanisms determine cell fate during the differentiation of hiSPCs, producing patterns and other system-level features in the process. Informed by experimental data, we develop a collection of agent-based models (ABMs) whose agents (i.e. cells) are each equipped with local rules that govern how the agents interact with their environment and with each other; the purpose of these ABMs is to simulate the early differentiation of hiPSCs according to different sets of biological assumptions. We also extend an existing mathematical framework which formalizes ABMs to estimate long-term model behavior with respect to time. Our estimates aim to establish connections between local interactions and certain system-level observations. Thus, we study both the emergent behaviors of our ABMs and the dynamics of the local rules governing each agent to ascertain which modes of intercellular communication determine cell fate.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-18 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-18.html">
											    	</br><strong>Mary Ellen Rosen</strong>
									    	 
									    		<em>Brigham Young University</em>
									    	
									    	
									    		
									    			</br>"A Mathematical Analysis of Focal Adhesion Lifetimes and Their Effect on Cell Motility"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Active cell motion is a fundamental process for most living organisms.  It is crucial for embryogenesis, pathologies such as fighting infections or the spread of cancer, and single cell organisms in finding favorable environments. In this research we analyze data regarding a subprocess of amoeboid cell motion - the lifetime of focal adhesions (FAs).  Cells attach to and gain traction from the extracellular matrix via FAs. We collect and analyze existing FA lifetime data and find that it is gamma distributed.  We also find that cell speed decreases as the mean FA lifetime increases. Mathematical modeling suggests that the detach-rate is both force and time dependent.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-19 (Session: PS02)</small>
										    	<a href="/CDEV/CDEV-PS02-CDEV-19.html">
											    	</br><strong>Josué Manik Nava-Sedeño</strong>
									    	 
									    		<em>National Autonomous University of Mexico</em>
									    	
									    	
									    		
									    			</br>"Collective migrantion and pattern formation with zero-range interactions"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We investigate the collective migration and pattern formation potential of zero-range-interacting agents, that is, agents interacting exclusively with others at exactly the same position. To this end, we use a lattice-gas cellular automaton model with no exclusion principles and polar/nematic velocity alignment interactions. We find that, in the case of polar alignment, the model shows a transition towards a highly ordered and condensed state with moving point clusters. In the nematic case, we observe the formation of high density, nematically-ordered bands. This suggests that migration is enough to relay information among individuals and to generate collective effects at the population level, even in the absence of spatially extended sensing.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-EDUC">EDUC</h5> </strong>EDUC-2 (Session: PS02)</small>
										    	<a href="/EDUC/EDUC-PS02-EDUC-2.html">
											    	</br><strong>Randy Heiland</strong>
									    	 
									    		<em>Indiana University</em>
									    	
									    	
									    		
									    			</br>"Creating a graphical user interface to define model parameters for PhysiCell"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Randy Heiland, Adam Morrow, Grant Waldrow, Drew Willis, Kim Crevecoeur, Paul MacklinPhysiCell is an open source, hybrid continuum-discrete mathematical modeling system that combines off-lattice discrete agents with a reaction-diffusion framework and has been applied to a broad variety of problems in mathematical biology. Early versions of PhysiCell generally required both writing C++ code and hand-editing XML configuration files to define a model. As the framework has evolved, most of the C++ model definition code has moved into the XML, making it easier to create, run, and share models. Our poster describes the design and development of a graphical user interface to generate the entire XML model. Our goal is to help create a future where powerful agent based models can be defined and simulated with relative ease. This project, which involves several undergraduate (REU) students, provides a nice complement to a previously published undergraduate-related project: xml2jupyter.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-16 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-16.html">
											    	</br><strong>Katherine Royce</strong>
									    	 
									    		<em>Harvard University</em>
									    	
									    	
									    		
									    			</br>"Application of a novel mathematical model to identify intermediate hosts of SARS-CoV-2"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Intermediate host species provide a crucial link in the emergence of zoonotic infectiousdiseases, serving as a population where an emerging pathogen can mutate to becomehuman-transmissible. Identifying such species is thus a key component of predictingand possibly mitigating future epidemics. Despite this importance, intermediate hostspecies have not been investigated in much detail, and have generally only beenidentified by testing for the presence of pathogens in multiple candidate species. In thispaper, we present a mathematical model able to identify likely intermediate hostspecies for emerging zoonoses based on ecological data for the candidates andepidemiological data for the pathogen. Since coronaviruses frequently emerge throughintermediate host species and, at the time of writing, pose an urgent pandemic threat,we apply the model to the three emerging coronaviruses of the twenty-first century,accurately predicting palm civets as intermediate hosts for SARS-CoV-1 anddromedary camels as intermediate hosts for MERS. Further, we suggest mink,pangolins, and ferrets as intermediate host species for SARS-CoV-2. With the capacityto evaluate intermediate host likelihood among different species, researchers canfocus testing for possible infection sources and interventions more effectively.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-17 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-17.html">
											    	</br><strong>Juan Pablo Restrepo</strong>
									    	 
									    		<em>Department of Mathematical Sciences, Universidad EAFIT</em>
									    	
									    	
									    		
									    			</br>"Non-Homogeneous Poisson Process & Functional Data: A procedure for infectious diseases count data modeling"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			In some epidemiological studies it is of interest to observe the behavior of the number of cases of a disease in a population, such as Dengue, Zika, Covid-19, among others; in order to predict the evolution of future cases. In this study, we propose to combine Non-Homogeneous Poisson Processes (NHPP) and Functional Data Analysis (FDA) methodologies for count-data prediction. We consider cumulative cases, subjected to time evolution and influence of explanatory variables. The proposed procedure allows to estimate the most representative cumulative-cases trajectory included its non-parametric confidence bands, as well as detect possible outlier trajectories, and predict future cumulative counting. An application with real infectious diseases data is also presented.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-18 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-18.html">
											    	</br><strong>Joanna Sooknanan</strong>
									    	 
									    		<em>University of the West Indies Open Campus</em>
									    	
									    	
									    		
									    			</br>"Harnessing social media in the modelling of pandemics – challenges and opportunities"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As COVID-19 spreads throughout the world without a straightforward treatment or widespread vaccine coverage in the near future, mathematical models of disease spread and of the potential impact of mitigation measures have been thrust into the limelight. With their popularity and ability to disseminate information relatively freely and rapidly, information from social media platforms offers a user-generated, spontaneous insight into users' minds that may capture beliefs, opinions, attitudes, intentions and behaviour towards outbreaks of infectious disease not obtainable elsewhere.  The interactive, immersive nature of social media may reveal emergent behaviour that does not occur in engagement with traditional mass media or conventional surveys. In recognition of the dramatic shift to life online during the COVID-19 pandemic to mitigate disease spread and the increasing threat of further pandemics, we examine the challenges and opportunities inherent in the use of social media data in infectious disease modelling with particular focus on their inclusion in compartmental models.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-19 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-19.html">
											    	</br><strong>Nao Yamamoto</strong>
									    	 
									    		<em>Arizona state university</em>
									    	
									    	
									    		
									    			</br>"Quantifying Compliance with COVID-19 Mitigation Policies in the US"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The outbreak of COVID-19 disrupts the life of many people in the world. In response to this global pandemic, governments in the United States had implemented COVID-19 mitigation policies. However, those policies, which were designed to slow the spread of COVID-19, and its compliance, have varied across the states, which led to spatial and temporal heterogeneity in COVID-19 spread. This study aims to proposea spatio-temporal model for quantifying compliance with the US COVID-19 mitigation policies at a regional level. To achieve this goal, a partial differential equation is developed and validated with the COVID-19 data. The proposed model describes the combined effects of transboundary spread among state clusters and human mobilities on the transmission of COVID-19. The model may inform policymakers as they decide how to react to future outbreaks.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-20 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-20.html">
											    	</br><strong>Orhun Davarci</strong>
									    	 
									    		<em>The University of Texas at Austin</em>
									    	
									    	
									    		
									    			</br>"Integrating epidemiological data and mathematical models to forecast COVID-19 spread in the United States"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The rapid global outbreak of COVID-19 has raised interest in the computational forecast of the spread of infectious diseases, but the early projections in the current pandemic were limited in their ability to describe longer-term outcomes. This issue was partially due to the limited knowledge of the mechanisms of disease spread and development. Our study aims to integrate epidemiological time-series data into a mathematical model that can describe the fundamental mechanisms of COVID-19 spread, with the ultimate goal of utilizing model forecasts to determine early indicators of large outbreaks as well as assessing public health interventions to control their severity. We used publicly available data from the 10 most heavily impacted states in the US to calibrate a SEIRD-type model and obtain state-specific sets of epidemiological parameters. Our model was able to recapitulate the early observations of cumulative infections and deaths (CCC > 0.9, R2 > 0.9). We further explore the use of model parameters and forecasts as early indicators of subsequent large outbreaks. Finally, we argue that mechanistic models that describe infectious disease spread can help mitigate the human cost of pandemics by anticipating effective public health interventions and enabling the optimized allocation of key medical resources.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-21 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-21.html">
											    	</br><strong>Tarun Mohan</strong>
									    	 
									    		<em>Texas Christian University</em>
									    	
									    	
									    		
									    			</br>"Modeling the effect of multiple vaccines on the spread of SARS-CoV-2"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Several different vaccines have been introduced to combat the spread of SARS-CoV-2 infections. As the virus is capable of mutating to escape the protection given by the vaccine, using multiple vaccines is believed to help prevent the virus from mutating to escape all vaccines, helping to combat spread of the virus. We simulate the effect of using multiple vaccines on the virus using a mathematical model. With the model, we can better understand the effect of multiple types of vaccines in helping to control pandemics.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-22 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-22.html">
											    	</br><strong>Julia Mautone</strong>
									    	 
									    		<em>Universidade de São Paulo</em>
									    	
									    	
									    		
									    			</br>"Mathematical modeling of Influenza H1N1 over vaccine influence based on real data"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We build a mathematical model applied to influenza H1N1. The model structure consists of splitting the human population according to susceptible, infected by the disease, and recovered which includes the vaccinated population. We develop stability analysis and calculate equilibrium point and basic reproduction number. We analyze model parameters and their role over the representation of São Paulo's real data, provided by SINAN (a serious notifications system), which helps to estimate the disease transmission rate, as well as population mortality and birth rates, through the least-squares method. We take into account the numerical method accuracy related to the infected curve fitting and the real data from 2013. In an attempt to study the vaccination influence over the number of cases, and to identify risks and forecasting outbreaks, we carry out numerical simulations by varying the vaccination rate parameter. Spite of vaccination reaches a small group of the population each year (around 20% based on 2010-2018 data), we conclude it is a key parameter that plays a role over the possibility of reducing cases through the curve flattening. We encourage public policies as an effective measure, to provide significant stimulus and adherence to vaccination programs, and a decrease of infected cases number.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-23 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-23.html">
											    	</br><strong>Gabriel McCarthy</strong>
									    	 
									    		<em>Texas Christian University</em>
									    	
									    	
									    		
									    			</br>"Quantifying the effectiveness of quarantine measures"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Using a SEIR model we evaluate the COVID-19 pandemic in various states. We assume a changing infection rate to reflect the restrictions put into place to combat spread of the infection. Doing this allows us to mathematically represent the changes in behavior and restrictions in actions after the outbreak of COVID-19 and how they affected the spread of COVID-19. We test different formulations for the changing infection rate, from abrupt change to gradual decay of the infection rate to determine how best to model changes due to various mandates. This analysis helps us understand the effectiveness of different preventative measures found across the U.S so that the pandemic is stopped and dealt with effectively.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-24 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-24.html">
											    	</br><strong>Quiyana Murphy</strong>
									    	 
									    		<em>Virginia Tech</em>
									    	
									    	
									    		
									    			</br>"Modeling testing strategies to reduce SARS-COV-2 transmission"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As vaccines against SARS-CoV-2 are not yet available for everyone, it is important to implement non-pharmaceutical interventions to reduce SARS-CoV-2 transmission. Testing is a necessary factor in quantifying the number of infected individuals and reducing their interaction with the population (isolation). Additionally, identifying positive cases allows public health officials to track transmission via contact tracing and prevent additional infections with quarantine. To better inform testing strategies, we develop a deterministic ordinary differential equation mathematical model for given available resources in a community. Specifically, our model includes various characteristics to be attributed to the variability in testing strategies, including the sensitivity of testing, availability of testing, delay in testing results, and priority of testing. Three different tests with varying sensitivity, availability, and return time are incorporated: antibody tests, RT-PCR tests, and antigen tests. Three scenarios are considered to investigate the effects of priority testing on disease transmission: test only symptomatic individuals, equally spread available tests across all testable populations (surveillance), and prioritize tests for symptomatic individuals but use the remaining testing for surveillance. Our model can determine which allocation of testing type and strategy will most significantly decrease the infectious population (peak and duration) given locally available testing information.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-25 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-25.html">
											    	</br><strong>Mohammad Mihrab Uddin Chowdhury</strong>
									    	 
									    		<em>Texas Tech University</em>
									    	
									    	
									    		
									    			</br>"The Influence of Annual Birth Pulses on Disease Transmissibility in Amphibian Populations."</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The dynamics of infectious disease in amphibians with multiple routes of transmissibility is a complex interconnected system. Depending on the population level and age stages (larvae, juveniles, and adults), infection spreads in a variety of ways. Due to seasonal reproductive behaviors, the population density of larvae rises at a certain time of year. We developed compartmental models using ordinary differential equations and difference equations to observe the effects of annual birth pulses on transmission dynamics of a fungal pathogen (Batrachochytrium salamandrivorans, Bsal) on a North American salamander population. Model simulations and analyses offer insights into control strategies aimed at reducing transmission and preventing epidemic outbreaks.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-27 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-27.html">
											    	</br><strong>Erica Rutter</strong>
									    	 
									    		<em>University of California, Merced</em>
									    	
									    	
									    		
									    			</br>"A Different COVID-19 Model: Characterizing the Spread of Misinformation of COVID-19 on Twitter"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Since the World Health Organization (WHO) declared COVID-19 a pandemic, mathematicians have mobilized to create models to predict the rise of COVID-19 through communities. In parallel to the spread of the virus, there has been an equally insidious spread of misinformation across various social media platforms. In this presentation, we will analyze the similarities and differences in transmission of various types of misinformation spread over Twitter in the past year. We build and analyze network graphs for the tweets (and retweets) of multiple types of misinformation (e.g, benign, conspiracy theory) and determine the characteristics that distinguish them. Can we predict the type of misinformation based on the way it spreads over twitter?
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-28 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-28.html">
											    	</br><strong>Harvir Singh Bhattal</strong>
									    	 
									    		<em>University of British Columbia</em>
									    	
									    	
									    		
									    			</br>"Underreporting of SARS-CoV-2 Infections in British Columbia"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Our understanding on the epidemiology of COVID-19 is limited by the ability of health systems to ascertain SARS-CoV-2 infections. Due to asymptomatic infection and testing hesitancy in symptomatic individuals (which may vary throughout time and with age, disease severity, socioeconomic status, etc.), reported cases represent only a fraction of total infections. The gold standard to estimate the burden of disease in a population is seroprevalence surveys. However, such surveys are operationally expensive and provide only a snapshot on the ascertainment of cumulative infections. In the current poster, we introduce a method to estimate instantaneous infection ascertainment from the the instantaneous case-hospitalization fraction and the infection-hospitalization fraction (which itself is constrained from seroprevalence data). We applied these methods in an age-structured manner to estimate the instantaneous ascertainment of SARS-CoV-2 infections in the B.C. population, from the outset of the epidemic to date. This allowed us to back-calculate the true epidemic, from which we could more acutely identify epidemiological trends.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-29 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-29.html">
											    	</br><strong>Caroline Franco</strong>
									    	 
									    		<em>Sao Paulo State University</em>
									    	
									    	
									    		
									    			</br>"Modelling COVID-19 in Brazil: better fit to data obtained when including the percolation effect approximation"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The SARS-CoV-2 pandemic has had an unprecedented impact on multiple levels of society. Not only has the pandemic completely overwhelmed some health systems but it has also changed how scientific evidence is shared and increased the pace at which such evidence is published and consumed, by scientists, policymakers and the wider public. Through the COVID-19 Modelling (CoMo) Consortium and the Observatório COVID-19 BR, we created modelling frameworks that could help simulate the effect of different non-pharmaceutical interventions on mitigating the epidemic in numerous locations. Here, we describe how this framework was adapted to the Brazilian context and, more specifically, fitted to data from the city of Sao Paulo. We propose an approximation for the percolation effect observed in social networks connectivity, due to the adoption of social distancing measures, and we show that this leads to better fitting to data, indicating the importance of this effect in such a system.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-30 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-30.html">
											    	</br><strong>Miller Ceron</strong>
									    	 
									    		<em>Universidad de Nariño</em>
									    	
									    	
									    		
									    			</br>"A SEI model with nonlinear incidence rate: global stability analysis"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We propose a SEI epidemic model where the infected and the exposed are the spreaders of the disease. Besides, a general nonlinear incidence rate and death rate functions are considered to study the stability of the model. We prove that the endemic equilibrium is globally asymptotically stable when the basic reproduction number R0 is greater than unity and the disease free equilibrium is globally asymptotically stable when R0 is lower than unity.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-31 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-31.html">
											    	</br><strong>Alexandra Catano-Lopez</strong>
									    	 
									    		<em>Departamento de Ciencias Matemáticas, Escuela de Ciencias, Universidad EAFIT</em>
									    	
									    	
									    		
									    			</br>"A platform to simulate COVID-19 that allows inclusion of mathematical modeling into decision-making management"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The COVID-19 pandemic affected the entire world, forcing several institutions to designate a part of their resources to implement control strategies to reduce its incidence. Thus, mathematical modeling becomes an important tool to study the effects of control policies, as it communicates academic information to the public and decision-makers. Following these ideas, the group of mathematical epidemiology of EAFIT University in Colombia developed an online tool that shows the effect of modifying control strategies over different localities in Colombia. We developed this tool based on a novel discrete-time model with variations on parameters related to quarantine, identification, social distancing, migrations, vaccination, among others; besides visual tools that allow communication to the public. At the moment, we included in the platform over forty Colombian localities, in which we individualize the mathematical model estimating the corresponding parameters with real data provided from the Colombian national health institution. Every time we update the model with new data, the user can simulate and project different control scenarios over the affected population. In this work, we will present the structure of a platform that allows the non-expert users to simulate different control strategies; also, we could use it for monitoring other infectious diseases.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-49 (Session: PS02)</small>
										    	<a href="/MEPI/MEPI-PS02-MEPI-49.html">
											    	</br><strong>Samantha Bardwell</strong>
									    	 
									    		<em>University of British Columbia</em>
									    	
									    	
									    		
									    			</br>"A Mathematical Model for Overdose in a Population of People Who Inject Drugs"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The presence of synthetic opioids (fentanyl and carfentanil) in heroin and other drugs has greatly increased the risk of fatal overdose among people who inject drugs (PWID) across Canada and elsewhere in the world. We sought to represent the dynamics of the population of PWID and various public health interventions using a mathematical framework that hybridizes an individual-based model with a compartmental analysis model. The goal of this study was to accurately formulate a simulated population of users whose risk is uniquely and dynamically determined. The model construction involved a significant literature review, and synthesis and analysis of data from collaborators implementing drug policy. We calibrated the model to represent the PWID population in downtown Toronto, but it can be adapted to examine effects of similar interventions in any location. The model results suggest that recruitment to the at-risk population is currently over-reported, and the present values should be re-evaluated. The model results also suggest that various factors, including age, previous overdoses, and history of incarceration, have a significant effect on the individual risk of fatal overdose. The information we obtain can be used to strategically target intervention strategies, and guide future research on the PWID population.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-1 (Session: PS02)</small>
										    	<a href="/MFBM/MFBM-PS02-MFBM-1.html">
											    	</br><strong>Paulina Wodarz</strong>
									    	 
									    		<em>University High School, Irvine</em>
									    	
									    	
									    		
									    			</br>"Analyzing Text Corpora to Determine the Emotional State of Humans"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			People's feelings are often reflected in the way they write. For this project, texts were used to characterize the emotional state of people both throughout the decades and in different parts of society in the present time. For temporal analysis, an online Corpus of Historical American English was used (400 million words, 1810-2000). For the present-day analysis, a collection of bloggers' posts from Kaggle (grouped by gender, age, and occupation) was put through a sentiment analysis tool. It was found that in the course of 200 years, energetic words decreased in frequency, and less energetic words increased. Negative and positive words decreased, and neutral words increased, indicating that there may have been a rise of apathy in society. Further, it was found that in present day's common usage, females, younger people, and those with a background in the arts exhibit more negative emotions than the other groups. These findings indicate that mathematical and computational analysis can be used to detect not only long-term societal trends, but also to study the emotional characteristics of different groups of people. In particular, methods of data science could be a valid tool to identify vulnerable populations that can be targeted for depression evaluation.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-2 (Session: PS02)</small>
										    	<a href="/MFBM/MFBM-PS02-MFBM-2.html">
											    	</br><strong>Samarth Kadaba</strong>
									    	 
									    		<em>University of California, Santa Barbara</em>
									    	
									    	
									    		
									    			</br>"Discovering Sequence-Activity Relationships using Machine Learning: Convolutional Neural Networks (CNNs) and Gaussian Process Regressions (GPRs)."</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We show how recent machine learning methods can be utilized to learn representations for classes of proteins and other macromolecules that relate sequence information to predicted activity on specific substrates. In the synthetic evolution of enzymes, predicting activity is a crucial step towards finding functional unscreened variants. However, identifying sequence-activity relationships for organic polymers is complicated by latent features associated with the 3D structure of the folded molecule. Here we propose methods that use Convolutional Neural Networks (CNN's) to extract from 3D structural information, such as crystallographic data and Molecular Dynamics (MD) simulation data, relationships between sequence and activity. In particular, we develop CNN feature extractors for kernels within Gaussian Process Regressions (GPRs) to make predictions on sequence space. As a demonstration, we use data of amino acid rotamers to show how CNN's can predict amino acid torsion angles from a sequence of Chi angle conformation types. Applying these networks to develop kernel functions for GPRs we predict conformational state information and predicted activities. We perform studies also on toy models to show how using deep learning approximations of macromolecular structure can yield representations of sequence-activity relationships potentially useful for synthetic evolution.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-3 (Session: PS02)</small>
										    	<a href="/MFBM/MFBM-PS02-MFBM-3.html">
											    	</br><strong>Emilee Carson</strong>
									    	 
									    		<em>University of Waterloo</em>
									    	
									    	
									    		
									    			</br>"A machine learning approach for analyzing bistable systems in biology"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Bistable systems arise frequently in the modelling of biological systems, particularly in systems biology. A famous example is the Collins toggle switch, a gene regulatory network with two genes that repress the expression of each other. Typically, the qualitative behaviour of these systems is examined using traditional techniques such as phase portraits and stability analysis. These approaches rely on the accuracy of the proposed model equations. Recently, machine learning has been used increasingly in the analysis of models stemming from applications in physics, but these methods have not yet been used widely for biological models. We develop a machine learning approach to analyze the behaviour of bistable systems in biology, particularly in cases where there may be information missing in the model equations and demonstrate its effectiveness in the case of the Collins toggle switch.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-4 (Session: PS02)</small>
										    	<a href="/MFBM/MFBM-PS02-MFBM-4.html">
											    	</br><strong>Alex John Quijano</strong>
									    	 
									    		<em>University of California Merced</em>
									    	
									    	
									    		
									    			</br>"Evolving Contextual Semantics"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Similar to biological systems, natural languages are evolving systems with words as their measurable units. Words have certain functions within a body of text to convey ideas and thought. This poster presentation introduces the method of Latent Semantic Analysis (LSA) and the Skip-Gram with Negative Sampling (SGNS) approaches to extract contextual semantics within a body of text taken from a social media platform called Twitter. Contextual semantics refers to a semantic space that is expressed as a linear combination of words from a matrix subspace. Due to the natural volatility of some words and languages as a whole, the semantic space is evolving in time. The objective is to study the emergence of online social movements particularly the use of hashtags. We explore the evolving contextual semantics of the social movement hashtags “#blacklivesmatter” and “#metoo” as examples. From our results and observations, we hypothesize that these hashtags exhibit selective language transmission (i.e. horizontal transmission), the process of passing words and phrases between people that lead to changes in meanings due to selective cultural pressures.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-NEUR">NEUR</h5> </strong>NEUR-1 (Session: PS02)</small>
										    	<a href="/NEUR/NEUR-PS02-NEUR-1.html">
											    	</br><strong>Leonid Rubchinsky</strong>
									    	 
									    		<em>Indiana University Purdue University Indianapolis and Indiana University School of Medicine</em>
									    	
									    	
									    		
									    			</br>"Modeling intermittent synchronization of gamma-band neural oscillations"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Synchronization in neural system plays important role in many brain functions such as perception and memory. Abnormal synchronization can be observed in neurological disorders such as Parkinson's disease, schizophrenia, autism, and addiction. Neural synchronization is frequently intermittent even in a short time scale. That is, neural systems exhibit intervals of synchronization followed by intervals of desynchronization. Thus, neural circuits dynamics may show different distributions of duration of desynchronization even if the synchronization strength is similar. In general, some partially synchronized systems can exhibit a few but long desynchronized intervals while other systems can yield many but short desynchronized intervals. Experimental data thus far has shown that neural synchronization follows the latter trend in either healthy or diseased brains. In this study, we use a conductance-based PING network to study neural synchronization specifically in the low gamma band. This study explores the cellular and synaptic effects on the temporal patterning of the partially synchronized model gamma rhythms and considers potential functional implications of different temporal patterns.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-12 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-12.html">
											    	</br><strong>Maria Eliza Antunes</strong>
									    	 
									    		<em>Graduate Program in Biometrics - São Paulo State University</em>
									    	
									    	
									    		
									    			</br>"Numerical simulations for a metastatic papillary thyroid cancer model using RAI 131I treatment"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			In this work, we numerically simulated a mathematical model to study A metastatic papillary thyroid cancer (PTC) response to different periodic radioiodine 131I (RAI) treatment.  Within the simulated scenarios we consider different values for the RAI efficiency ratio.  Besides that, periodic treatment protocols with the same dose were considered and also with decreasing amount of doses, with a higher dose first, followed by smaller ones. Some protocols failed to decrease the number of tumor cells, where can understand as a resistance towards RAI treatment conditions by the lack of response. These failures may mean a poorly structured treatment protocol regarding the type of therapy, doses, application intervals, or any of their combinations. Notably, RAI treatment scenarios with alternating dosages presented a successful treatment response, i.e., tumor elimination. Therefore, mathematical models are essential tools in the study of cancer biology and could assist in determining the most suitable treatment protocols, including for the metastatic PTC.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-13 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-13.html">
											    	</br><strong>Hannah Anderson</strong>
									    	 
									    		<em>University of Florida</em>
									    	
									    	
									    		
									    			</br>"Team Approach to Integrating Mathematical and Biological Models to Target Myeloid-Derived Immune Cells in Glioblastoma"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Objective: Integrate mathematical models of immunosuppressive glioblastoma (GBM) infiltrating myeloid cells with experimental data to predict therapeutic responses to combined chemokine receptor and immune checkpoint blockade.Methods: Orthotopic murine KR158-luc gliomas were established in mice. Subsequently, an anti-CD31 injection was administered to label the vasculature. Fluorescent imaging and quantification of anti-CD3 stained sections were performed on a range of tumor sizes to acquire vasculature, tumor, T cell, and myeloid cell densities. In parallel, a system of ordinary differential equations was formulated based on biological assumptions to evaluate the change over time of tumor cells, T cells, and infiltrating myeloid cells. The model was then refined and validated by experimental results.Results: Fluorescent imaging and quantification revealed a correlation between tumor size and abundance of myeloid cell populations in the tumor microenvironment. The density of these cell populations and vasculature remained constant as the tumors increased in size. Computer simulations of the mathematical model will predict tumor, myeloid, and T cell dynamics. These simulations will be particularly useful to understand myeloid cell dynamics, such as cell entry time into the tumor microenvironment. Parameter sensitivity analysis of the model will inform us of the biological processes driving these tumor-immune cell dynamics.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-14 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-14.html">
											    	</br><strong>Jessica Kingsley</strong>
									    	 
									    		<em>University of South Florida</em>
									    	
									    	
									    		
									    			</br>"Bridging cell-scale simulations and radiologic images to explain short-time intratumoral oxygen fluctuations"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Radiologic images provide a longitudinal way to monitor tumor responses, but operate on a macroscopic scale and are not able to capture microscopic scale phenomena. We provide a link between the average data recorded in radiological image voxels and the tissue architecture that fills these voxels. Our in silico model includes individual tumor and stromal cells, tumor vasculature, and tumor metabolic landscape. This architecture was based on tissue characteristics acquired from electron paramagnetic resonance (EPR) images.  We used this model to optimize vascular influx and cellular uptake schedules to reproduce oxygen fluctuations recorded experimentally. By comparing simulation results within the schedules, we showed that sole alterations in vascular influx were able to reproduce experimental data well. On the other hand, in order to fit experimental data with metabolic changes in tumor cells, the cells would need to increase their oxygen absorption by 50-fold over a period of 3 minutes, which may not be biologically feasible. Additionally, we developed a procedure to identify plausible tissue morphologies for a given temporal series of average data from radiology images. In the future our approach can be used to simulate hypoxia-sensitive anti-cancer treatments on a cell-scale based on clinically-collected images.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-15 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-15.html">
											    	</br><strong>Rebecca Bekker</strong>
									    	 
									    		<em>H. Lee Moffitt Cancer Center & Research Institute</em>
									    	
									    	
									    		
									    			</br>"Investigating inter-replicate differences in cancer wound healing assays using an agent-based model"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Cellular migration, and thus motility, are important factors in a tumors ability to metastasize. Wound healing assays are a way of quantifying these properties in vitro, while mathematical modeling can be used to do so in silico. In silico models can be calibrated and validated on the collected migration data, and used to predict the effects of therapeutics on the migration of cancer cells. We focus on the murine cell lines TC-1 and mEER, both transformed using the oncogenes HPV16 E6, HPV16 E7 and hRAS. Wound healing assays were performed on these cell lines after irradiation with 0Gy, 2Gy, 8Gy and 10Gy to quantify the dose dependent effects of radiation on the motility of the cell lines. Herein we report on a calibrated agent-based model used to investigate how assumptions about the underlying distributions of migration speed data affects in silico experiments. Additionally, we discuss whether combining the replicates per experiment lends itself to more accurate predictions than using each data set individually.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-16 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-16.html">
											    	</br><strong>Javier Urcuyo</strong>
									    	 
									    		<em>Mayo Clinic</em>
									    	
									    	
									    		
									    			</br>"Exploring the Glioblastoma-Immune Dynamics with Mathematical Modeling and Transcriptome Sequencing"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Glioblastoma (GBM) is a deadly, heterogeneous disease with poor overall survival. Adding to the complexity of the disease, the tumor-immune environment is also heterogeneous. Glioma-associated macrophages and microglia (GAMMs) can exhibit either a tumor-suppressive or tumor-permissive response, resulting in a variety of different GBM growth patterns. However, the mechanism by which GAMMs affect GBM growth remains unclear. To explore the potential dynamics of these tumor-GAMM interactions, we created four candidate mathematical models, each capturing a different biological mechanism for the conversion between GAMM phenotypes. To better understand the parameters influential on tumor growth, we performed a sensitivity analysis. Initial analyses indicate that, beyond the growth kinetics of the tumor, the initial population of tumor-suppressive GAMMs is influential on tumor velocity. This preliminary finding is somewhat surprising, as it suggests that changes to the relative abundance of immune populations over time would not significantly impact the tumor growth. In future work, we plan to utilize deconvolution techniques on RNAseq from image-localized biopsies to identify relative cellular-subtype compositions and investigate if tumor growth kinetics are dependent on current GAMM composition. By developing a better understanding of the tumor-immune interface, we can aid in identifying potential immunotherapy strategies and in assessing their effectiveness.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-17 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-17.html">
											    	</br><strong>Chandler Gatenbee</strong>
									    	 
									    		<em>Moffitt Cancer Center</em>
									    	
									    	
									    		
									    			</br>"Immune escape at the onset of human colorectal cancer"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The evolutionary dynamics of tumor initiation remain undetermined, and the interplay between neoplastic cells and the immune system is hypothesized to be critical in transformation. Colorectal cancer (CRC) presents a unique opportunity to study the transition to malignancy as pre-cancers (adenomas) and early stage cancers are frequently detected and surgically removed. Here, we examine the role of the immune response in tumor initiation by studying tumor-immune eco-evolutionary dynamics from pre-cancer to carcinoma using a computational model, ecological analysis of digital pathology, and multi-region neoantigen prediction. Observed changes in antigenic intra-tumor heterogeneity (aITH), the tumor ecology, and spatial patterns of both cell associations and gene expression are consistent with simulations where immunogenic adenomas do not progress to CRC because they are under immune control. Conversely, adenomas that progress initially avoid detection through low immunogenicity, but gradually construct an immunosuppressive niche isolated from CD8+ cytotoxic T cells, thereby evading immune elimination and allowing for an increase in neoantigen burden. Both modeling and data indicate that immune blockade (e.g. PD-L1 expression) plays a secondary role to immune suppression in tumor initiation or progression. These results suggest that re-engineering the immunosuppressive niche may prove to be a most effective immunotherapy in CRC.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-18 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-18.html">
											    	</br><strong>Lee Curtin</strong>
									    	 
									    		<em>Mayo Clinic</em>
									    	
									    	
									    		
									    			</br>"Discerning Glioblastoma Subpopulation Interactions through In Vitro Experiments and Mathematical Modeling"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Glioblastoma is the most aggressive primary brain tumor with dismal median survival. Glioblastoma is known to be heterogeneous with multiple molecularly-distinct subpopulations, however, both the baseline change in growth with a given mutation and the compounded change due to dynamic interactions between subpopulations remain unknown. It is important to characterize these interactions for their potential impact on treatment resistance. Using both in vitro data and mathematical modeling, we aim to determine the interaction between two more-common glioblastoma subpopulations: amplification of epidermal growth factor receptor (EGFR) and platelet-derived growth factor receptor alpha (PDGFRA). As a preliminary study, two glioblastoma cell lines, LN229 and GBM22, were identified with similar genetic profiles. Then, two variants of each of these lines were developed to over-express EGFR and PDGFRA, resulting in 6 molecularly-distinct cell lines in total. Variants of each cell line were allowed to grow both independently, and separately in co-culture with their sister variant. Preliminary calibration of an exponential model with the independently-grown cell culture data demonstrated that the variant influence on the growth rate was different between the cell lines. Future analysis will involve calibrating a system of reaction-diffusion equations to identify the effects of co-culturing these subpopulations on the growth kinetics.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-19 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-19.html">
											    	</br><strong>Brydon Eastman</strong>
									    	 
									    		<em>University of Waterloo</em>
									    	
									    	
									    		
									    			</br>"Reinforcement learning derived chemotherapeutic schedules for robust patient-specific therapy given unknown patient response parameters"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			When developing a chemotherapeutic dosing schedule for treating cancer in-silico one relies upon a parameterization of a particular tumour growth model to describe the dynamics of the cancer in response to the dose of the drug. It is often prohibitively difficult, in practice, to ensure the validity of patient-specific parameterizations of these models for any particular patient. As a result, sensitivities to these particular parameters can result in therapeutic dosing schedules that are optimal in principle not performing well on particular patients. In this study, we demonstrate that chemotherapeutic dosing strategies learned via reinforcement learning methods are more robust to perturbations in patient-specific parameter values than those learned via classical optimal control methods. By training a reinforcement learning agent on mean-value parameters and allowing the agent access to a more easily measurable metric, relative bone marrow density, we are able to develop drug dosing schedules that outperform schedules learned via classical optimal control methods, even when such methods are allowed to leverage the same bone marrow measurements.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-20 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-20.html">
											    	</br><strong>Janielly Matos Vieira</strong>
									    	 
									    		<em>Graduate Program in Biometrics -São Paulo State University</em>
									    	
									    	
									    		
									    			</br>" Mathematical model of mestastasis involving immunotherapy with CAR T cells"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The essence of cancer is characterized by the disordered growth of cells having the ability to invade tissues and injured adjacent organs. Classified as a world health problem, cancer lies among the four leading death causes worldwide, being metastasis responsible for over 90% of all cancer-related deaths. In order to eradicate the disease, several therapies are under development, being the immunotherapy on prominence for not causing severe damages to the normal cells, reinforcing the patient's immune system so it can better fight the cancer cells.  Using differential equations, in this work, we proposed a mathematical model of mestastasis involving immunotherapy with CAR T cells. In the model, metastasis is modeled as a migratory phenomenon, where two populations of cancer cells coexist and develop in two different locations.Through numerical simulations, we studied the tumor dynamics in different scenarios, where these were obtained by varying the initial condition of the tumor cells and in the amount of CAR T cells used.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-21 (Session: PS02)</small>
										    	<a href="/ONCO/ONCO-PS02-ONCO-21.html">
											    	</br><strong>Thomas Veith</strong>
									    	 
									    		<em>Moffitt Cancer Center</em>
									    	
									    	
									    		
									    			</br>"Spatial Constraints On In-Vitro Cancer Cell Line Growth"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Population density puts a constraint on cell growth through the mechanism of contact inhibition. Dysregulation of this process is one of the hallmarks of cancer. Contact inhibition is correlated with expression of pathways associated with E-cadherin signaling such as Hippo and Wnt, or inhibition of mTOR signaling. We observe heterogeneous expression of these pathways across and within nine metastatic gastric cancer cell lines via single cell RNA sequencing data. Our working hypothesis is that high or low population densities will benefit certain cells over others, selecting for subclonal populations in an evolutionary process known as density dependent selection. Here, we present a method which integrates cell culturing experiments with single cell sequencing data to investigate the effects spatial constraints have on subclonal growth of cancer cells in-vitro. Our results evaluate the usefulness of population density as an informer of subclonal growth dynamics in a data driven, mechanistic model.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-11 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-11.html">
											    	</br><strong>Amanda Laubmeier</strong>
									    	 
									    		<em>Texas Tech University</em>
									    	
									    	
									    		
									    			</br>"Interplay between pesticide use and natural predator behaviors"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We are interested in the combination of natural predators and conventional pesticides which contribute to the control of aphids, an agricultural pest. Although aphids are prey to many insects, the unique landscape for large-scale farming can reduce migration to and mobility within agricultural fields. In contrast, some small-scale and natural practices can foster an efficient natural predator community. Alongside these landscape choices, insecticide use can cause predator disorientation and sluggishness, further impacting mobility. To investigate how these different effects come together to determine pest control, we develop a partial differential equation model for predator-prey interactions within an agricultural field for a single season. We describe realistic use of pesticide sprays, which occur in pulses after pests pass a threshold abundance. The model also describes predator prey-taxis, or movement towards food sources, and how this behavior is impacted by pesticides. We consider these effects for a variety of migration and hunting behaviors and discuss the implications of our results for different agricultural practices.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-12 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-12.html">
											    	</br><strong>Nusrat Tabassum</strong>
									    	 
									    		<em>Texas Tech University</em>
									    	
									    	
									    		
									    			</br>"The effects of temperature change on prey suppression by natural predators"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The sustainability of an ecosystem is determined by the relationship between predators and prey. The factors that play an important role in this context are temperature, body mass, foraging area, intraspecific competition and intraguild predation, all of which impact a predator's functional response. In the context of global warming, changing temperature could play a key role in changing prey suppression. Depending on the temperature, prey and predator can become active or inactive and temperature can affect other behaviors such as eating habit, foraging area, body growth etc. We use a dynamic model to describe prey suppression. We illustrate how predator behaviors would change with temperature at different times in a day or when average temperature increases.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-13 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-13.html">
											    	</br><strong>Jorge Arroyo-Esquivel</strong>
									    	 
									    		<em>Department of Mathematics, UC Davis</em>
									    	
									    	
									    		
									    			</br>"Long transients appear in predator-prey systems with group defense and nonreproductive stages"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			During recent years, the study of long transients has been expanded in ecological theory to account for shifts in long-term behavior of ecological systems. These long transients consist of long periods of time where a system is apparently in equilibrium; after which the system undergoes an abrupt change into qualitatively different dynamics. In this work, we analyze the potential for long transients in a model for a predator-prey system in which the prey present group defense, and their nonreproductive stages do not contribute to predator growth. This model has been previously used to analyze kelp-urchin dynamics, but it can be used in other systems such as colonial spider-wasp or honeybee-hornet systems. We have identified this system presents crawl-by transients near the extinction and carrying capacity states of prey. In addition, we identify a transcritical bifurcation in our system, under which a ghost limit cycle appears. We are able to estimate the escape time of our system from these transients using perturbation theory. This work advances an understanding of how systems shift between alternate stable states and their duration of staying in a given regime.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-14 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-14.html">
											    	</br><strong>Russell Milne</strong>
									    	 
									    		<em>University of Waterloo</em>
									    	
									    	
									    		
									    			</br>"Effects of overfishing on coral reefs over local and regional scales"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Coral reefs are highly connected habitats, with dynamics that take place over very large spatial scales. However, performing field work over these large scales is challenging, and most mathematical modelling of coral reefs has focused on local dynamics. Here, we use a mechanistic, spatially explicit coral reef model to simulate the regional and local effects of three coral reef stressors (overfishing, nutrient loading and crown-of-thorns starfish invasions). We find three different local regimes (coral-dominant, macroalgae-dominant, macroalgae-only with no coral or fish), with sharp boundaries that depend on the interaction between fishing rate and nutrient loading rate. We also find that overfishing within a single patch can decrease coral cover by significant amounts in non-overfished patches. Additionally, increasing the proportion of patches that are overfished causes nonlinear declines in coral cover in non-overfished patches; this decline is strongly dependent on the configuration of which patches are overfished. The combination of crown-of-thorns starfish presence and high nutrient loading increases the variability of coral populations, and limits the space covered by both coral and macroalgae. These effects are present systemwide even when nutrient loading is restricted to one patch. Our findings have implications for both future field work and implementing conservation objectives.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-15 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-15.html">
											    	</br><strong>Clara Woodie</strong>
									    	 
									    		<em>University of California, Riverside</em>
									    	
									    	
									    		
									    			</br>"The stabilizing and destabilizing effects of cannibalism in an intraguild predation system"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Intraguild predation (IGP), an interaction in which the intraguild (IG) predator competes with its intraguild (IG) prey for a shared resource, is ubiquitous in nature despite original theory predicting limited coexistence. A proposed stabilizing mechanism is cannibalism in the IG predator through its regulation of the predator population, which decreases predation pressure on the IG prey. We add cannibalism to an IG predator and include a cannibalism preference parameter to explore how the predator's preference for IG prey vs. conspecifics affects dynamics. We perform linear stability analyses. Our results show that strong cannibalism preference in the IG predator can 1) stabilize unstable IGP systems or 2) destabilize already-stable IGP systems depending on prey competitive ability. When the prey is a superior competitor, keeping with the assumption of original IGP theory, strong cannibalism preference drives the predator extinct. When the predator is a similar competitor for the resource as the prey, a common occurrence in natural IGP systems, preference for conspecifics over heterospecifics stabilizes this otherwise unstable system where the prey goes extinct. These results suggest that cannibalism preference, by altering the relative strengths of competition vs. predation between the predator and prey, determines the long-term stability of an IGP system.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-16 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-16.html">
											    	</br><strong>Thaddeus Seher</strong>
									    	 
									    		<em>University of California, Merced</em>
									    	
									    	
									    		
									    			</br>"AddTag, a two-step approach that overcomes targeting limitations of precision genome editing"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			CRISPR/Cas-induced genome editing is a powerful tool for genetic engineering, however targeting constraints limit which loci are editable with this method. Since the length of a DNA sequence impacts the likelihood it overlaps a unique target site, precision editing of small genomic features with CRISPR/Cas remains an obstacle. We introduce a novel genome editing strategy that virtually eliminates CRISPR/Cas targeting constraints and facilitates precision genome editing of elements as short as a single base-pair at virtually any locus in any organism that supports CRISPR/Cas-induced genome editing. Our two-step approach first replaces the locus of interest with an “AddTag” sequence, which is subsequently replaced with any engineered sequence, and thus circumvents the need for direct overlap with a unique CRISPR/Cas target site. In this study, we demonstrate the feasibility of our approach by editing transcription factor binding sites within Candida albicans that could not be targeted directly using the traditional gene editing approach. We also demonstrate the utility of the AddTag approach for combinatorial genome editing and gene complementation analysis, and we present a software package that automates the design of AddTag editing.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-17 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-17.html">
											    	</br><strong>Benjamin Garcia de Figueiredo</strong>
									    	 
									    		<em>Instituto de Física Teórica - Unesp</em>
									    	
									    	
									    		
									    			</br>"Investigating first-crossing statistics in movement models with home-ranging behavior"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Ecological populations are, in general, not well mixed, and their non-homogeneous use of space modulates their local interactions. Although this range-residency is known to affect important observables such as encounter rates between individuals, many models in population dynamics implicitly or explicitly assume that populations make homogenous use of space. The Ornstein-Uhlenbeck (OU) process is a stochastic process in space that displays the basic characteristics of movement bounded by and centered around a home-range. Within the framework of OU movement models, the crossing statistics of two simultaneous processes serve as a proxy for the encounter statistics of two individuals. While this mathematical problem has been investigated, especially in one dimension, fewer studies have addressed the more ecologically relevant two-dimensional case (2D). In this work, we conduct a numerical and semi-analytical study of the first-crossing statistics of a pair of 2D OU models. We believe this can help build the foundations of more mechanistic and realistic models of population dynamics, based on the scaling properties of individual interactions. Further analytical investigation of this problem may elucidate its general properties.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-18 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-18.html">
											    	</br><strong>Rafael Menezes</strong>
									    	 
									    		<em>University of São Paulo</em>
									    	
									    	
									    		
									    			</br>"Feasibility and Resilience in Randomly Assembled Communities"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As our world faces ever-increasing pressure upon many natural environments, it is essential to understand the stability of ecological communities. One of the crucial aspects of stability in rich  communities is resilience, which entails information on how quickly the community can recover from small fluctuations in the densities of the populations. Equally relevant is their feasibility, which is indicative of how likely all the populations in the community can coexist, on the assumption that relative growth rates are variable. Despite substantial advancements in the investigation of these measures of stability, their interplay remains largely unexplored. In this work, we performed a comprehensive ecologically-informed exploration of the parameter space of the generalized Lotka-Volterra model integrating variability in type, intensity, and distribution of interspecific ecological interactions to study the broad patterns linking these two aspects of stability. We found a positive correlation between resilience and feasibility, suggesting that more resilient communities are more likely to be feasible. Additionally, we also found that communities with lower densities and intensities of interactions and more competition/exploitation are more resilient, and communities with equal proportions of positive and negative interactions are more feasible. Our study highlights the importance of investigations integrating different aspects of ecological stability.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-19 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-19.html">
											    	</br><strong>Joany Mariño</strong>
									    	 
									    		<em>Memorial Univesity of Newfoundland</em>
									    	
									    	
									    		
									    			</br>"Resource seasonality explains latitudinal size and clutch size patterns in a Dynamic Energy Budget model "</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Animals show a vast array of geographical variation in phenotypic traits. The most common patterns are the tendency of size and clutch size to increase with latitude among related species. Nevertheless, the precise mechanisms behind these patterns remain controversial. Here, we show how resource seasonality can drive latitudinal trait variation. We conducted numerical simulations of a dynamic energy budget model, quantifying individual biomass and reproductive output, both under constant and seasonal resource conditions. We evaluated 48 different genetically-determined physiological characters (equivalent to different species and represented by the model parameters for assimilation, mobilization, and energy allocation).  In both scenarios, we found that resource availability determines interspecific trait differences in the DEB model. Our findings show that individuals can reach greater biomass and reproductive output in a seasonal environment than in a constant environment of equal average resource due to the peaks of food surplus. Our results agree with the classical patterns of interspecific trait variation and provide a mechanistic understanding supporting recent explanatory hypotheses: the resource and the eNPP (net primary production during the growing season) rules. The current alterations to ecosystems and communities make disentangling trait variation increasingly important to understand and predict biodiversity dynamics under environmental change.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-20 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-20.html">
											    	</br><strong>Anuraag Bukkuri</strong>
									    	 
									    		<em>Moffitt Cancer Center</em>
									    	
									    	
									    		
									    			</br>"Tortoise and the Hare: On the Contribution of Evolvability to Eco-Evolutionary Dynamics of Competing Species"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Evolvability, the capacity for a population to generate heritable variation and respond to natural selection, is a fundamental concept influencing the adaptations and fitness of individual organisms. For many species, evolvability may be a trait that is subject to natural selection. Evolvability plays a critical role in eco-evolutionary dynamics and may help us understand how species respond to changes in their environment and how species coexistence can arise and be maintained. We create a model of competing species, each with a different evolvability. We then analyze the population and strategy dynamics of the two populations under the conditions of clade initiation, evolutionary tracking, adaptive radiation, and evolutionary rescue. We find that more stable environments favor slower evolving species, while unstable environments favor faster evolving ones. When several niches are available for species to occupy, slower evolving species outcompete faster evolving ones due to the cost of evolvability. Finally, we promote coexistence by disrupting the environment at intermediate frequencies, allowing for cyclical population dynamics of species with differential evolvabilities. Though we frame our discussion in the context of ecology and cancer, our model and analyses are agnostic of any specific application and thus broadly apply to any system capable of evolving.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-21 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-21.html">
											    	</br><strong>Evan Haskell</strong>
									    	 
									    		<em>Nova Southeastern University</em>
									    	
									    	
									    		
									    			</br>"Attraction-Repulsion Taxis Mechanisms in a Predator-Prey Model"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We consider a predator-prey model where the predator population favors the prey through biased diffusion toward the prey density, while the prey population employs a chemical repulsive mechanism. This leads to a quasilinear parabolic system. We first establish the global existence of positive solutions. Thereafter we show the existence of nontrivial steady state solutions via bifurcation theory, then we discuss the stability of these branch solutions. Through numerical simulation we analyze the nature of patterns formed and interpret results in terms of the survival and distribution of the two populations.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-22 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-22.html">
											    	</br><strong>Rebecca Everett</strong>
									    	 
									    		<em>Haverford College</em>
									    	
									    	
									    		
									    			</br>"Stoichiometric regulation of immune responses in primary producers"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			All organisms require carbon and nutrients such as nitrogen for their growth and reproduction. In the presence of pathogens, host defense has been shown to increase with enhanced nutrient availability. Thus, availability of nitrogen may stimulate a host by enhancing its growth as well as immunity response. However, at the same time, nutrient availability may promote infection as higher host growth trades-off with reduced resistance as well as through enhanced pathogen performance. We explore the role of nitrogen availability on infection dynamics of a primary producer host and its pathogen using a stoichiometry-based disease model. Specifically, we test how changes in nitrogen investments in host immune response will alter host biomass build-up and pathogen infection rates.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-23 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-23.html">
											    	</br><strong>Daniel Cooney</strong>
									    	 
									    		<em>University of Pennsylvania</em>
									    	
									    	
									    		
									    			</br>"Persistence vs Extinction of Cooperation via Multilevel Selection: The Dynamical Shadow of Lower-Level Selection"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Natural selection often acts simultaneously upon multilevel levels of biological organization, inducing a tension between traits favoring selfish individuals and traits providing collective benefit for the group. Examples of such conflicts arise in settings including the evolution of the early cell, the evolution of virulence, and the sustainable management of common-pool resources. In this talk, we consider a PDE model for the evolution of a cooperative trait in which competition takes place both within groups through individual-level reproduction and between-groups through a group-level birth-death process. Generalizing previous work from evolutionary game theory, we show that there exists a threshold intensity of between-group competition separating regimes in which cooperation goes extinct or persists in the population. We additional provide bounds on the long-time average payoff of the population, showing that the population cannot outperform the payoff of a full-cooperator group in the long run and allowing us to determine when measure-valued solutions to the multilevel dynamics converge to a steady-state density or forever oscillate. When intermediate levels of cooperation are most favorable to the group, this means that multilevel selection will always promote suboptimal collective outcomes, and no level of between-group competition can erase the shadow of lower-level selection.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-24 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-24.html">
											    	</br><strong>Vahini Reddy Nareddy</strong>
									    	 
									    		<em>University of Massachusetts Amherst</em>
									    	
									    	
									    		
									    			</br>"Transition states in two-cycle ecological oscillators: dynamics and forecasting"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Many spatially-extended systems of ecological oscillators exhibit spatial synchrony with periodic oscillations in time. If the individual oscillators have two-cycle behavior, the transition to synchrony as a function of noise and coupling strength is in the Ising universality class, ensuring that the stationary properties of the ecological systems can be replicated by the simple Ising model [1]. In the Ising representation, the two phases of oscillations (high at odd times or high at even times) of an individual oscillator are represented by spin-up and spin-down. However, the behavior of an individual ecological oscillator suggests the existence of a transition state along with the two phases of oscillations. The oscillations at this transition state have amplitude very close to zero. To study such systems, we use Blume-Capel representation where the spin can take three values S={+1,-1,0} with S=0 as the transition state and S={-1,+1} as the two phases of oscillations. We model the spatially-extended ecological systems with coupled lattice maps in two-cycle regime and represent them with three state model. We also discuss maximum likelihood methods to infer the Blume-Capel representation. [1] V.Nareddy,et.al,J R Soc Interface(2020)
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-25 (Session: PS02)</small>
										    	<a href="/POPD/POPD-PS02-POPD-25.html">
											    	</br><strong>Silas Poloni Lyra</strong>
									    	 
									    		<em>Institute for Theoretical Physics - UNESP</em>
									    	
									    	
									    		
									    			</br>"Intraguild Predation in Periodic Habitats"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Fragmentation of natural landscapes is an ongoing process, mainly led by human activities, such as urban growth, roadway construction and farming. This phenomena may lead to many changes in the dynamics of populations that live in such landscapes, posing new challenges to our understanding of population persistence and diversity therein. In this work we consider an Intraguild Predation (IGP) model, a community module composed of two consumers of a shared resource, with a predation relation between such consumers, usually referred as IG-Prey and IG-Predator.  Using Cobbold and Yurk's homogenization technique, we formulate and investigate the problem in a periodic habitat, composed of two types of patches where IGP relations are present, but allowed to have different parameters, such as less resource consumption, enhanced mortality or reduced resource productivity in one of the patches. Our results show that coexistence  between IG-Prey and IG-Predator in heterogeneous landscapes is facilitated or hardened depending on the resource's habitat preference, allowing for coexistence in parameter regions which, in homogeneous landscapes, would be impossible, for example.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  

		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Posters-02
										</br>
										Tuesday, June 15 at 03:15pm (PDT)</br>Tuesday, June 15 at 11:15pm (BST)</br>Wednesday, June 16 07:15am (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>














    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
