<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-19 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-19 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-19" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS19/" />
<meta property="og:url" content="http://localhost:4000/MS19/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS19/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-19","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-19">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-19" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-19" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-19</h1>
        <h4>Thursday, June 17 at 09:30am (PDT)</br>Thursday, June 17 at 05:30pm (BST)</br>Friday, June 18 01:30am (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS19">Minisymposia-19</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/CBBS-MS19.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS19-CBBS: </small></br>Stochastic methods for biochemical reaction networks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Wasiur KhudaBukhsh (The Ohio State University, United States), Hye-Won Kang (University of Maryland at Baltimore County, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-CBBS</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/CBBS-MS19.html#author1"><strong class="card-body-subgroup">David Anderson</strong></a>
		    	 
		    		<em> (University of Wisconsin Madison, USA)</em>
		    	
		    	
		    		
		    			</br>"Time-dependent product-form Poisson distributions for reaction networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is well known that stochastically modeled reaction networks that are complex balanced admit a stationary distribution that is a product of Poisson distributions. In this talk, I will discuss the following related question:  under what conditions will the time-dependent distribution of a reaction network be a product of Poissons for all time? I will provide a necessary and sufficient condition for such a product-form distribution to hold for all time. Interestingly, the condition is a dynamical “complex-balancing” for only those complexes that have multiplicity greater than or equal to two (i.e. the higher order complexes that yield non-linear terms to the dynamics). This is joint work with Chaojie Yuan (Indiana) and David Schnoerr (Imperial College London).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CBBS-MS19.html#author2"><strong class="card-body-subgroup">Lea Popovic</strong></a>
		    	 
		    		<em> (Concordia University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Stochastic reduction of spatially heterogeneous biochemical reaction networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We start from a measure valued process which models the full particle behaviour of chemical reaction networks in spatially heterogeneous systems. Scaling of such a process with a high abundance of some species types and large reaction rates of some reactions leads to a reaction-diffusion pde deterministic limit, or to a mixture of discrete-Markov-and-continuous-deterministic limit process. In this talk we consider a reduced stochastic description of the original measure-valued process by approximating its fluctuations around the limiting process.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CBBS-MS19.html#author3"><strong class="card-body-subgroup">Hye-Won Kang</strong></a>
		    	 
		    		<em> (University of Maryland at Baltimore County, United States)</em>
		    	
		    	
		    		
		    			</br>"Stochastic modeling of metabolic enzyme complexes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Enzymes in purine biosynthesis and glucose metabolism have been shown to spatially organize into different types of multienzyme complexes. These multienzyme complexes regulate metabolic flux in living human cells. Metabolic pathways for purine biosynthesis and glucose metabolism are associated with each other, but their metabolic enzyme complexes are spatially independent in human cells. We hypothesize that these metabolic enzyme complexes communicate with each other when they are in close location.  This talk will introduce a stochastic model for metabolic enzyme complexes using the Langevin dynamics to investigate their spatial communication.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CBBS-MS19.html#author4"><strong class="card-body-subgroup">Felipe Campos</strong></a>
		    	 
		    		<em> (University of California, San Diego, USA)</em>
		    	
		    	
		    		
		    			</br>"Error bounds for the one-dimensional constrained Langevin approximation for density-dependent Markov chains"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Constrained Langevin Approximation (CLA) is a reflected diffusion approximation for stochastic chemical reaction networks proposed by Leite & Williams. In this work, we extend this approximation to (nearly) density dependent Markov chains, when the diffusion state space is one-dimensional. Then, we provide a bound for the error of the CLA in a strong approximation. Finally, we discuss some applications for chemical reaction networks and epidemic models, illustrating these with numerical examples. Joint work with Ruth Williams.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/CDEV-MS19.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS19-CDEV: </small></br>Dynamics and networks in single-cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/CDEV-MS19.html#author1"><strong class="card-body-subgroup">Stephanie Hicks</strong></a>
		    	 
		    		<em> (Johns Hopkins University, USA)</em>
		    	
		    	
		    		
		    			</br>"Scalable statistical methods and software for single-cell data science"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Single-cell RNA-Seq (scRNA-seq) is the most widely used high-throughput technology to measure genome-wide gene expression at the single-cell level. However, single-cell data present unique challenges that have required the development of specialized methods and software infrastructure to successfully derive biological insights. Compared to bulk RNA-seq, there is an increased scale of the number of observations (or cells) that are measured and there is increased sparsity of the data, or fraction of observed zeros. Furthermore, as single-cell technologies mature, the increasing complexity and volume of data require fundamental changes in data access, management, and infrastructure alongside specialized statistical methods to facilitate scalable analyses. I will discuss some challenges in the analysis of scRNA-seq data and present some solutions that we have made towards addressing these challenges.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CDEV-MS19.html#author2"><strong class="card-body-subgroup">Geoffrey Schiebinger</strong></a>
		    	 
		    		<em> (University of British Colombia, Canada)</em>
		    	
		    	
		    		
		    			</br>"Towards a mathematical theory of trajectory inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk develops a rigorous mathematical framework for trajectory inference. We examine the problem of recovering temporal couplings of stochastic processes, motivated by applications in developmental biology and cellular reprogramming. We develop methodology based on optimal transport and test it on data from stem cell reprogramming, sea urchin embryonic development, arabidopsis root growth, and hematopoeisis. We then perform a theoretical analysis and establish rigorous guarantees. Our approach provides a rigorous, general framework for investigating cellular differentiation, and poses some interesting questions in probability, statistics and optimization.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CDEV-MS19.html#author3"><strong class="card-body-subgroup">Gioele La Manno</strong></a>
		    	 
		    		<em> (Swiss Federal Institute of Technology in Lausanne, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Revealing the brain’s molecular anatomy with single-cell and tomography-based spatial transcriptomics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			I will present our comprehensive single-cell transcriptome atlas of mouse brain development spanning from gastrulation to birth. In this atlasing effort, we identified almost a thousand distinct cellular states, including the initial emergence of the neuroepithelium, different glioblasts, and a rich set of region-specific secondary organizers that we localize spatially. In this context, I will provide an example of how the  spatially-resolved transcriptomic data can be particularly useful to interpret the complexity of such complex atlases.
Continuing in this direction, I will show the approach that we recently proposed as a general way to spatially resolve different types of next-generation sequencing data.  We designed an imaging-free framework to localize high throughput readouts within a tissue by combining compressive sampling and image reconstruction. Our first implementation of this framework transformed a low-input RNA sequencing protocol into an imaging-free spatial transcriptomics technique (STRP-seq).
Finally, I will showcase the technique with the profiling of the brain of the Australian bearded dragon Pogona vitticeps. With this analysis, we revealed the molecular anatomy of the telencephalon of this lizard and provided evidence for a marked regionalization of the reptilian pallium and subpallium.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/CDEV-MS19.html#author4"><strong class="card-body-subgroup">Kenji Kamimoto</strong></a>
		    	 
		    		<em> (Washington University in St. Louis, USA)</em>
		    	
		    	
		    		
		    			</br>"CellOracle: Dissecting cell identity via network inference and in silico gene perturbation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent technological advances in single-cell sequencing enable the acquisition of multi-dimensional data in a high-throughput manner. These technologies reveal the existence of heterogeneity and the diversity of cell states and identities. To reveal the regulatory mechanism underlying such phenomena, many computational Gene regulatory Network (GRN) inference methods have been proposed. However, understanding biological events from a GRN perspective remains difficult. Even if a computational algorithm can infer GRN, the biological network is so complex that it is challenging to understand how it systematically dictates cell identities. There is significant demand for new methodologies that bridge the gap between cellular phenotypes and the underlying GRN. Thus, we have developed a new method, CellOracle, a new computational approach for the inference and analysis of GRN. By utilizing machine learning algorithms and genetic information, CellOracle infers sample-specific GRN configurations from single-cell RNA-seq and ATAC-seq data. Our GRN models are designed to be used for the simulation of cell identity changes in response to gene perturbation. This simulation enables network configurations to be interrogated in relation to cell-fate regulation, facilitating their interpretation. To validate CellOracle’s GRN inference method, we present benchmarking on various tissues and cell-types. We also validate the efficacy of CellOracle to recapitulate known outcomes of well-characterized gene perturbations in developmental processes, including mouse hematopoiesis and zebrafish embryogenesis. Our benchmarking and validation results demonstrate the efficacy of CellOracle to infer and interpret the dynamics of GRN configurations, promoting new mechanistic insights into the regulation of cell identity.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/DDMB-MS19.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS19-DDMB: </small></br>Data-Driven Modeling and Analysis in Mathematical Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Tomas Carino-Bazan (University of California, Santa Barbara, United States), Daniel Wilson (Boston University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/DDMB-MS19.html#author1"><strong class="card-body-subgroup">Julie Hussin</strong></a>
		    	 
		    		<em> (Université de Montréal, Montreal Heart Institute, Canada)</em>
		    	
		    	
		    		
		    			</br>"Evolutionary approaches to detect epistasis in large-scale genomic data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Whether gene-gene interactions, or epistasis, plays a major or minor role for any given human trait in any population remains an open question, and analytical methods to detect epistasis have become very popular in the last decade. However, there are important computational and statistical challenges for identifying novel epistatic interactions in human genetics. To help solve the paucity of uncovered epistasis in humans, we propose new approaches to characterize gene-by-gene interactions, in studying signatures of co-evolution. The underlying model is that interacting genes will undergo compensatory genetic mutations to maintain their interaction, which will result in correlation of allelic frequencies between physically unlinked loci. In this talk, I will present data-driven projects on two distinct systems, interactions among Cytochrome P450 genes and co-evolution involving the cholesterol metabolism gene CETP, and their implications for precision medicine. Our studies also demonstrate how data from diverse human populations in genetic studies can be leveraged to uncover biological mechanisms of importance for world-wide population health.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/DDMB-MS19.html#author2"><strong class="card-body-subgroup">Elana Fertig</strong></a>
		    	 
		    		<em> (Johns Hopkins University, United States)</em>
		    	
		    	
		    		
		    			</br>"Identifying therapeutic resistance mechanisms in cancer with single-cell data and transfer learning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumors employ complex, multi-scale cellular and molecular interactions that evolve over the course of therapeutic response. The changes in these pathways enables tumors to overcome therapeutic regimens, and ultimately acquire resistance. New molecular profiling technologies, including notably single cell technologies, provide an unprecedented opportunity to characterize these molecular relationships. However, interpreting the specific cellular and molecular pathways in therapeutic response requires complementary computational analysis methods. We developed an unsupervised learning method, CoGAPS, that employs Bayesian non-negative matrix factorization to disentangle distinct biological processes from high-throughput molecular data. Notably, this algorithm discovers dynamic compensatory signaling in acquired therapeutic resistance from time course bulk RNA-seq data and novel NK cell activation in anti-CTLA4 response from post-treatment scRNA-seq data. To further demonstrate that the inferred pathways are biological rather than computational artifacts, we developed a complementary transfer learning method to relate learned patterns between datasets. We demonstrate that this approach identifies robust molecular processes between model systems and human tumors and enables multi-platform data integration to delineate the drivers of therapeutic response and resistance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/DDMB-MS19.html#author3"><strong class="card-body-subgroup">Tomas Carino-Bazan</strong></a>
		    	 
		    		<em> (University of California, Santa Barbara, United States)</em>
		    	
		    	
		    		
		    			</br>"Machine learning methods for fluid mechanics for learning low dimensional representations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many empirical studies and experiments in applications ranging from biophysics to engineering design yield partial information of the flow fields and related hydrodynamic responses. We develop data-driven methods for learning representations of hydrodynamic responses for inference tasks.  From an analytic perspective the field of fluid mechanics traditionally has used transformations such as vorticity to represent localized flow structures and for numerical simulations.  For example for inviscid flows this often yields a sparse representation.  We seek to develop related machine learning methods that learn more general non-linear transformations that can featurize hydrodynamic flow data for making inferences about flow structure and dynamics.  We discuss our progress toward studying hydrodynamic flows using auto-encoders with associated regularizations to learn smooth low dimensional representations of flow structures.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/DDMB-MS19.html#author4"><strong class="card-body-subgroup">Lorin Crawford</strong></a>
		    	 
		    		<em> (Microsoft Research New England, United States)</em>
		    	
		    	
		    		
		    			</br>"Statistical Frameworks for Discovering Biophysical Signatures in 3D Shapes and Images"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The recent curation of large-scale databases with 3D surface scans of shapes has motivated the development of tools that better detect global patterns in morphological variation. Studies which focus on identifying differences between shapes have been limited to simple pairwise comparisons and rely on pre-specified landmarks (that are often known). In this talk, we present SINATRA: a statistical pipeline for analyzing collections of shapes without requiring any correspondences. Our method takes in two classes of shapes and highlights the physical features that best describe the variation between them. We develop a rigorous simulation framework to assess our approach, which themselves are a novel contribution to 3D image and shape analyses. Lastly, as case studies, we use SINATRA to (1) analyze mandibular molars from four different suborders of primates and (2) facilitate the visual identification of structural signatures differentiating between two protein ensembles.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/ECOP-MS19.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>MS19-ECOP: </small></br>Population Dynamics Across Interacting Networks or Scales</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Necibe Tuncer (Florida Atlantic University, USA), Hayriye Gulbudak (	 University of Louisiana at Lafayette, USA), 	 Cameron Browne (University of Louisiana at Lafayette, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-ECOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/ECOP-MS19.html#author1"><strong class="card-body-subgroup">Glenn Webb</strong></a>
		    	 
		    		<em> (Vanderbilt University, USA)</em>
		    	
		    	
		    		
		    			</br>"A COVID-19 epidemic model predicting the effectiveness of vaccination in the US"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A model of a COVID-19 epidemic is used to predict the effectiveness of vaccination in the US. The model incorporates key features of COVID-19 epidemics: asymptomatic and symptomatic infectiousness, reported and unreported cases data, and social measures implemented to decrease infection transmission.  The model analyzes the effectiveness of vaccination in terms of vaccination efficiency, vaccination scheduling, and relaxation of social measures that decrease disease transmission. The model demonstrates that the subsiding of the epidemic as vaccination is implemented depends strongly on the scale of relaxation of social measures that reduce disease transmission.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ECOP-MS19.html#author2"><strong class="card-body-subgroup">Cameron Browne</strong></a>
		    	 
		    		<em> (University of Louisiana at Lafayette, USA)</em>
		    	
		    	
		    		
		    			</br>"Connecting predator prey dynamics and population genetics in an evolving virus immune network"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Integrating population evolution and dynamics offers a promising avenue for understanding rapidly evolving pathogens.  For example, during HIV infection, the virus can escape several immune response populations via resistance mutations at distinct epitopes (proteins coded in viral genome), precipitating a dynamic network of interacting virus and immune variants.  Understanding the main factors shaping viral resistance pathways and immune dynamics is crucial for designing effective vaccines and immunotherapies.  While the virus-immune interactions may be quite complex, I will talk about my recent work to link pathogen population genetics with dynamics theoretically and through data to characterize their evolution.  We start with a general differential equation ecosystem model of multiple virus and immune populations, and then prove that different stable and persistent patterns emerge in the virus-immune network dependent on the virus fitness landscape.   Next, I will present a collaborative project where the 'eco-evolutionary' modeling framework is connected to genomic and population data.  We describe the interaction between several immune cell populations and viral 'quasi-species' sampled from experiments of the simian immunodeficiency virus (SIV)-infected macaque model of HIV infection.  The mathematical models can recapitulate the data and shed light on pathogen evolution, along with motivating ongoing work on jointly deciphering the population genetics and dynamics of pathogens and their complex ecosystems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ECOP-MS19.html#author3"><strong class="card-body-subgroup">Andrea Pugliese</strong></a>
		    	 
		    		<em> (University of Trento, Italy)</em>
		    	
		    	
		    		
		    			</br>"mmune memory build-up in models of repeated infections; how does this affect epidemic dynamics?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is well known that memory cells can help to build a quick immune response in case of a new infection with the same (or similar) pathogen. This is indeed the principle at the basis of vaccination.
It is also known that for certain pathogens a single vaccine dose can be insufficient to achieve a complete control of an infection, and that a second dose may be necessary.
On the other hand, in several models of virus-immune interactions, the lower is the immune level before an infection, the higher it will be afterwards. This property is an important feature of the immuno-epidemiological models developed recently by Diekmann and co-workers.
Recently, Zarnitsyna et al. have proposed a realistic model for immune response to infection by influenza virus that results in a progressive build-up of immune memory.
In the talk, I will discuss several simplifications of the model in order to assess which components of the model are essential for its qualitative behaviour. Furthermore I will show how these features can be incorporated in a consistent multi-scale epidemic models, where the susceptible population is stratified through the number of times it has been infected. Strain coexistence is then common, and potential evolutionary consequences are explored.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ECOP-MS19.html#author4"><strong class="card-body-subgroup">Lauren M Childs</strong></a>
		    	 
		    		<em> (Virginia Tech, USA)</em>
		    	
		    	
		    		
		    			</br>"Trade-offs in Malaria Population Dynamics Across Scales"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Malaria is a disease endemic in areas encompassing over half the world’s population and remains detrimental to the health and livelihood of millions of individuals. Plasmodium parasites, the causative agents of malaria, have a complex life cycle requiring two hosts – a vertebrate, such as a human, and the Anopheles mosquito. During the time in each of these hosts, the population dynamics of the parasite are quite variable in density and stage. In previous work using a stochastic model of malaria population dynamics, we showed how density of parasite stages alter the timing and probability of onward transmission at the mosquito to human interface. Here, we bridge within-host modeling of parasite dynamics in the mosquito and the human to investigate maintenance of parasite diversity at the population level.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/EVOP-MS19.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>MS19-EVOP: </small></br>Evolutionary Theory of Disease</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jesse Kreger (University of California, Irvine, United States), Natalia Komarova (University of California, Irvine, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-EVOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/EVOP-MS19.html#author1"><strong class="card-body-subgroup">Joceline Lega</strong></a>
		    	 
		    		<em> (University of Arizona, United States)</em>
		    	
		    	
		    		
		    			</br>"A novel take on outbreak dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During an outbreak, public health data typically consist of time series for the daily or weekly incidence (reported new cases) of the disease. This information is location-specific (e.g. at the level of a county, a state, or a country) and noisy. For each such time series, plotting incidence as a function of cumulative cases instead of time leads to a remarkable simplification: the data appear to fluctuate about a mean curve of universal shape. In this talk, I will illustrate the previous statement through examples of Influenza A and COVID-19 outbreaks, and describe recent work aimed at elucidating this behavior [1, 2]. In particular, exact results will be provided for the deterministic and stochastic SIR models. In addition, I will explain how this property can be combined with data assimilation to provide short-term forecasts of COVID-19 cases and deaths in the US [3]. This is joint work with Hannh Biegel, Bill Fries, Faryad Sahneh, and Joe Watkins.
[1] J. Lega, Parameter estimation from ICC curves, Journal of Biological Dynamics 15, 195-212 (2021). [2] F.D. Sahneh, W. Fries, J.C. Watkins, J. Lega, The COVID-19 Pandemic from the Eye of the Virus (2021); arxiv.org/abs/2103.12848 [3] H. Biegel & J. Lega, EpiCovDA: a mechanistic COVID-19 forecasting model with data assimilation (2021).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/EVOP-MS19.html#author2"><strong class="card-body-subgroup">Dylan H. Morris</strong></a>
		    	 
		    		<em> (University of California, Los Angeles, United States)</em>
		    	
		    	
		    		
		    			</br>"Evolving fast and slow: how asynchrony between virus diversity and antibody selection limits influenza virus evolution"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Seasonal influenza viruses create a persistent global disease burden by evolving to escape immunity induced by prior infections and vaccinations. New antigenic variants have a substantial selective advantage at the population level, but these variants are rarely selected within-host, even in previously immune individuals. Using a mathematical model, we show that the temporal asynchrony between within-host virus exponential growth and antibody-mediated selection could limit within-host antigenic evolution. If selection for new antigenic variants acts principally at the point of initial virus inoculation, where small virus populations encounter well-matched mucosal antibodies in previously-infected individuals, there can exist protection against reinfection that does not regularly produce observable new antigenic variants within individual infected hosts. Our results provide a theoretical explanation for how virus antigenic evolution can be highly selective at the global level but nearly neutral within-host, and providing a clear example of how evolution and ecology only make sense in light of one another.
Relevant reading: https://elifesciences.org/articles/62105
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/EVOP-MS19.html#author3"><strong class="card-body-subgroup">Jesse Kreger</strong></a>
		    	 
		    		<em> (University of California, Irvine, United States)</em>
		    	
		    	
		    		
		    			</br>"The role of migration in mutant evolution in fragmented populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Complex population structures are an important determinant of the evolutionary dynamics of mutants. In fragmented populations, this has been studied using metapopulation models, which have been of great interest for questions related to ecology and population conservation. However, such models also have high relevance in a biomedical context – such as deme population structures that apply to evolution in hematopoietic systems. In this talk, we investigate the effects of population fragmentation on mutant cell dynamics using stochastic metapopulation modeling in conjunction with in vitro laboratory experiments. In the case of neutral mutations, we find that migration makes the demes look homogeneous to each other, resulting in a one-humped (unimodal) distribution, which matches well with experimental simulations. For disadvantageous mutations, we find that migration not only similarly impacts the distribution of mutant cells, but it can also change the expected frequency of mutants at stationary state compared to the selection-mutation balance. This could play an important role in disease progression.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/EVOP-MS19.html#author4"><strong class="card-body-subgroup">Ali Mahdipour-Shirayeh</strong></a>
		    	 
		    		<em> (University of Toronto, Canada)</em>
		    	
		    	
		    		
		    			</br>"Clonal evolution and Intra-tumoral heterogeneity in cancer: A single-cell viewpoint"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Despite intense therapeutic advances, therapy failures in diverse cancers may suggest the existence of intra-tumoral diversity and presumably rare subclones of minimal residual disease that are persistent to current therapies. Although there is no comprehensive technique to determine such subclones and to identify evolution of the disease, single-cell data can shed light on intra-cellular heterogeneity and clonal evolution of individual cells in alternative cell contexts. Utilizing single-cell data, the potential genetic pathways can be detected across diverse pheno/geno-types within a heterogeneous population of cells. Moreover, in many cancers, particularly in Multiple Myeloma, the most reliable clonal features are copy number variations (CNVs) which can be best inferred from single-cell DNA/RNA study. To address all these challenges, we developed an extensive pipeline, referred to as sciCNV, which covers a range of analysis from a novel normalization to inferring CNV from single-cell data. During this talk, we first introduce some fundamental tools which are commonly used in studying single cells and then will introduce the sciCNV pipeline to be implemented to segregate tumor cells from normal individuals and to understand the genetic background of the disease. This technique may offer an efficient way to clone distinct CNV-compartments and to construct a phylogeny of subclonal structure and pathogenesis of the disease. Such analysis can reflect evolutionary dynamics and clonal dependencies of cancer in time/space frame. Our approach is general and can be applied to any transcription data and may tend to a better understanding of histological/pathogenesis of diverse cancers and their associated therapeutic strategies.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/IMMU-MS19.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>MS19-IMMU: </small></br>The pressing need for within-host models of the pulmonary immune response</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Luis Sordo Vieira (Division of Pulmonary, Critical Care, and Sleep Medicine, University of Florida, United States), Marissa Renardy (University of Michigan/Applied BioMath, United States), Tracy Stepien (Department of Mathematics, University of Florida, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-IMMU</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/IMMU-MS19.html#author1"><strong class="card-body-subgroup">Borna Mehrad</strong></a>
		    	 
		    		<em> (Division of Pulmonary, Critical Care, and Sleep Medicine, University of Florida, United States)</em>
		    	
		    	
		    		
		    			</br>"Big Problems in Pulmonary Medicine: A Research Agenda"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			According to the World Health Organization, 3 of the 10 leading causes of death worldwide are lung diseases. In order, these are pneumonia (in which category I include COVID-19 and tuberculosis), chronic obstructive pulmonary disease, and lung cancer — these illnesses are a good place to start a discussion about a research agenda about the big problems in pulmonary medicine. In this talk, I will give an overview of each illness from a clinical and biological perspective, discuss some recent discoveries in each field, and end with key unresolved questions for each category.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/IMMU-MS19.html#author2"><strong class="card-body-subgroup">Josh Mattila</strong></a>
		    	 
		    		<em> (University of Pittsburgh, United States)</em>
		    	
		    	
		    		
		    			</br>"Converting pathology into data points and back again: using systems immunology to investigate cause-effect relationships in tuberculosis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tuberculosis is caused by Mycobacterium tuberculosis (Mtb), a bacterium that infects nearly a third of the world’s population. The human immune system is very effective at combatting Mtb and most infected people never experience symptomatic TB but there are still more than 10 million new TB cases and almost 2 million deaths from TB per year. Granulomas are the hallmark of TB and these multicellular lesions form in Mtb-infected tissues. Under optimal conditions, granulomas prevent bacterial dissemination and can generate sterilizing immunity but under suboptimal conditions, granulomas are sites of bacterial persistence and replication. Unfortunately, it is difficult to identify correlates of immunity in TB because granulomas occur in tissues that cannot be sampled and most of our information on immunity in TB comes from peripheral blood or murine TB models, neither of which replicate fully immunity in granulomas. Granulomas from experimentally-infected nonhuman primates (NHP) offer a human-like alternative but inter-granuloma heterogeneity and difficulties assessing the temporal trajectory of granuloma maturation and function make it difficult to interpret data from NHP granulomas. Computational models of granulomas, powered by biologic data obtained from ex vivo wet-lab studies on NHP granulomas, can model aspects of granuloma biology that correlate with protective or detrimental immunity. Here, I review how we have used biologic data from NHP granulomas to calibrate and validate GranSim, a computational granuloma model developed by the Kirschner Lab at the University of Michigan.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/IMMU-MS19.html#author3"><strong class="card-body-subgroup">Maral Budak</strong></a>
		    	 
		    		<em> (University of Michigan Medical School, United States)</em>
		    	
		    	
		    		
		    			</br>"Optimization of multidrug therapies for tuberculosis using a multi-scale computational model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tuberculosis (TB) is caused by the inhalation of Mycobacterium tuberculosis (Mtb), leading to ~1.5 million deaths every year. Mtb mainly infects lungs and triggers the formation of dense cellular structures composed of immune cells, bacteria, and dead tissue, called granulomas. The complex structure of granulomas prevents the effective penetration of antibiotics used to treat TB. Moreover, the heterogeneity of granulomas gives rise to various microenvironments for Mtb, where bacteria acquire different metabolic states that determine the potency of antibiotics either singly or in combination. Due to these reasons, TB treatment requires treatment with multiple antibiotics over long periods (6-9 months), causing prolonged side effects and compliance issues. Optimizing multidrug therapies and regimens for TB is essential to treat TB more effectively. In this study, we combined in vitro drug interaction predictions within GranSim, our computational model of granuloma formation and drug activity that simulates spatio-temporal granuloma drug dynamics. By systematically testing drug candidate regimens and considering drug interactions, we predict optimal drug regimens to be tested in vivo. This study will potentially lead to the discovery of more effective drug regimens that shorten the treatment window and have fewer side effects.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/IMMU-MS19.html#author4"><strong class="card-body-subgroup">Henrique de Assis Lopes Ribeiro</strong></a>
		    	 
		    		<em> (Division of Pulmonary, Critical Care, and Sleep Medicine, University of Florida, United States)</em>
		    	
		    	
		    		
		    			</br>"Computational Modeling Reveals the Role of Macrophages in Respiratory A. fumigatus Infection in Immunocompromised Hosts"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Fungal infections of the respiratory system are a life-threatening complication for immunocompromised patients. Invasive pulmonary aspergillosis, caused by the airborne mold Aspergillus fumigatus, has a mortality rate of up to 50% in this patient population. The lack of neutrophils, a common immunodeficiency caused by, e.g.,chemotherapy, disables a mechanism of sequestering iron from the pathogen, an important virulence factor. This paper shows that a key reason why macrophages are unable to control the infection in the absence of neutrophils is the onset of hemorrhaging, as the fungus punctures the alveolar wall. The result is that the fungus gains access to heme-bound iron. At the same time, the macrophage response to the fungus is impaired. We show that these two phenomena together enable the infection to be successful. A key technology used in this work is a novel dynamic computational model used as a virtual laboratory to guide the discovery process. The paper shows how it can be used further to explore potential therapeutics to strengthen the macrophage response.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/MEPI-MS19.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS19-MEPI: </small></br>Women in Mathematical Epidemiology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Katharine Gurski (Howard University, United States), Kathleen Hoffman (University of Maryland, Baltimore County, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-MEPI</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/MEPI-MS19.html#author1"><strong class="card-body-subgroup">Christina Edholm</strong></a>
		    	 
		    		<em> (Scripps College, United States)</em>
		    	
		    	
		    		
		    			</br>"Stochastic Models and Superspreaders: Effects of Environmental Variability"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The importance of host transmissibility in disease emergence has been demonstrated in historical and recent pandemics that involve infectious individuals, known as superspreaders, who are capable of transmitting the infection to a large number of susceptible individuals. To investigate the impact of superspreaders on epidemic dynamics, we formulate deterministic and stochastic models that incorporate differences in superspreaders versus nonsuperspreaders. In particular, continuous-time Markov chain models are used to investigate epidemic features associated with the presence of superspreaders in a population. We parameterize the models for two case studies, Middle East respiratory syndrome (MERS) and Ebola.  In this talk, we will explore how superspreaders and environmental variability impact important epidemiological measures via mathematical analysis and numerical simulations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MEPI-MS19.html#author2"><strong class="card-body-subgroup">Angela Peace</strong></a>
		    	 
		    		<em> (Texas Tech University, United States)</em>
		    	
		    	
		    		
		    			</br>"Spatial influences on Ebola and MERS epidemic dynamics: an agent-based modeling approach"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			For many communicable diseases, superspreaders are defined as specific infected individuals that transmit disproportionately to more susceptible individuals than other infected individuals, which may result from increased contact with susceptible individuals, higher pathogen shedding or increased strain virulence.  Epidemiological studies show that epidemics such as EBOV and MERS were largely driven and sustained by superspreaders that are ubiquitous throughout the outbreak. Hence understanding the dynamics of superspreaders can facilitate devising individually-targeted control measures. Studies have identified host heterogeneity (e.g.,~behavioral and immunological differences), population density and urbanization as underlying factors in disease outbreaks, therefore to capture disease transmission dynamics, we need a spatial modeling approach which can incorporate social phenomenons associated with human interactions. We developed an agent-based model (ABM) for simulating the actions and interactions of autonomous agents (both individual or collective entities such as organizations or groups) during an epidemic. We show that ABMs of infectious disease dynamics can provide additional insights by incorporating individual heterogeneity and spatial information.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MEPI-MS19.html#author3"><strong class="card-body-subgroup">Carrie Manore</strong></a>
		    	 
		    		<em> (Los Alamos National Laboratory, United States)</em>
		    	
		    	
		    		
		    			</br>"COVID-19 modeling and forecasting to inform decision makers"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We will present mathematical and statistical models for COVID-19 spread and impacts including hospital capacity, case counts, and deaths. Different methods are needed for supporting decision making depending on if we need accurate forecasting or exploration of 'what-if' scenarios. We will show how detailed agent based models, differential equation, and high level statistical models can be used together to support modeling of an ongoing pandemic.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MEPI-MS19.html#author4"><strong class="card-body-subgroup">Sylvia Gutowska</strong></a>
		    	 
		    		<em> (University of Maryland, Baltimore County, United States)</em>
		    	
		    	
		    		
		    			</br>"Effects of PrEP on the spread of HIV in the MSM population"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			talk will describe the convergence-divergence model and discuss some
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/MFBM-MS19.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS19-MFBM: </small></br>Algebra, Combinatorics, and Topology in Modern Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Daniel Cruz (Georgia Institute of Technology, U.S.), Margherita Maria Ferrari (University of South Florida, U.S.)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/MFBM-MS19.html#author1"><strong class="card-body-subgroup">Margherita Maria Ferrari</strong></a>
		    	 
		    		<em> (University of South Florida, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Formal grammar modeling three-stranded DNA:RNA braids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A formal grammar is a system to generate words; it consists of a set of symbols, partitioned into terminals and non-terminals, and a set of production rules. The production rules specify how to rewrite non-terminal symbols, so that successive applications of those rules yield words formed by only terminals. Adding probabilities to the production rules defines stochastic grammars, which can be used for biological sequence analysis. In this talk, we focus on a 'braid grammar' to model R-loops, that are three-stranded structures formed by a DNA:RNA hybrid plus a single strand of DNA, often appearing during transcription. R-loops are described as strings of terminal symbols representing the braiding of the strands in the structure, where each symbol corresponds to a different state of the braided structure. We discuss approaches to develop a stochastic grammar and a probabilistic model for R-loop prediction, as well as refinements of the model by incorporating the effect of DNA topology on R-loop formation
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MFBM-MS19.html#author2"><strong class="card-body-subgroup">Svetlana Poznanovic</strong></a>
		    	 
		    		<em> (Clemson University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Using Polytopes to Improve RNA Branching Predictions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Minimum free energy prediction of RNA secondary structures is based on the Nearest Neighbor Thermodynamics Model. While such predictions are typically good, the accuracy can vary widely even for short sequences, and the branching thermodynamics are an important factor in this variance. Recently, the simplest model for multiloop energetics - a linear function of the number of branches and unpaired nucleotides - was found to be the best. We develop a branch-and-bound algorithm that finds the set of optimal parameters with the highest average accuracy for a given set of sequences. The search uses the  branching polytopes for RNA sequences. Our analysis shows that previous ad hoc parameters are nearly optimal for tRNA and 5S rRNA sequences on both training and testing sets. Moreover, cross-family improvement is possible but more difficult because competing parameter regions favor different families. The results also indicate that restricting the unpaired nucleotide penalty to small values is warranted. This reduction makes analyzing longer sequences using the present techniques more feasible.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MFBM-MS19.html#author3"><strong class="card-body-subgroup">Chad Giusti</strong></a>
		    	 
		    		<em> (University of Delaware, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Comparing Topological Feature Coding Across Neural Populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A common feature of the types of information neural populations in the brain encode is cyclicity, meaning that the data is well-represented by one or more independent circular coordinate systems. Persistent homology, a common tool from topological data analysis can be applied to detect and study representations of cyclic features in neural population activity, even without reference to a behavioral correlate. Recent advances in experimental techniques have led to simultaneous recording of activity from populations of neurons across several brain regions, providing an opportunity to study how these representations propagate and change as they move through the brain. Classical topological wisdom tells us that we should apply functoriality to compare topological features across locations. However, in this setting  the goal is, in effect, to impute the map we would need in order to do so. Here, we present a novel method for comparing topological features detected in different brain regions leveraging dissimilarity matrices obtained from observations of activity. No background in topology and neuroscience on behalf of the audience will be assumed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MFBM-MS19.html#author4"><strong class="card-body-subgroup">Abdulmelik Mohammed</strong></a>
		    	 
		    		<em> (University of South Florida, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Topological Eulerian Circuits for the Design of DNA Nanostructures"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Graph theory has recently emerged as a powerful framework for the automated design of biomolecular nanostructures. A prime example of this is in the design of wireframe DNA origami nanostructures, where the routing of a circular viral DNA, called a scaffold strand, is modeled as an Eulerian circuit of a reconditioned triangulated mesh. In this setting, the knot type of the scaffold strand dictates the feasibility of an Eulerian circuit to be used as the scaffold route in the design. We investigate the knottedness of Eulerian circuits on surface-embedded graphs to characterize the class of such graphs that are constructible from unknotted and knotted scaffold strands. We show that certain graph embeddings, called checkerboard colorable, always admit unknotted Eulerian circuits. On the other hand, we prove that if a graph admits an embedding in a torus such that the embedding is not checkerboard colorable, then the graph can be re-embedded so that all its non-intersecting Eulerian circuits are knotted. For surfaces of genus greater than one, we present an infinite family of checkerboard-colorable graph embeddings where there exist knotted Eulerian circuits.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/MMPB-MS19.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>MS19-MMPB: </small></br>Mathematics of Microswimming</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Qixuan Wang (UC Riverside, United States), Bhargav Rallabandi (UC Riverside, United States), Mykhailo Potomkin (UC Riverside, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-MMPB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/MMPB-MS19.html#author1"><strong class="card-body-subgroup">Rishabh V. More</strong></a>
		    	 
		    		<em> (Mechanical Engineering, Purdue University, United States)</em>
		    	
		    	
		    		
		    			</br>"Micro-swimmer dynamics in stratified fluids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the motion of microorganisms in aquatic bodies like lakes and oceans has been an active area of research for decades with wide ecological and environmental impacts. Especially, the upper layer of oceans which sustains an intense biological activity, observes a vertical variation in the density (stratification) which can either be due to variations in water temperature or salinity, or both. From our fully resolved numerical simulations, we show that fluid stratification affects the locomotion of an individual, interactions between a pair, and the dynamics of suspensions of marine micro-swimmers in interesting and non-intuitive ways. At low Re, the vertical migration of small organisms is hydrodynamically affected due to the rapid velocity decay as well as higher energy expenditure in stratified fluids. At a finite Re, stratification even leads to striking differences in the swimming speeds and stability of swimmers as compared to their motion in a homogeneous fluid. The reduced flow signature of a swimming organism due to stratification can save them from getting detected by predators. Stratification increases the contact time of two colliding swimmers, thus, increasing the probability of successful reproduction. These results can explain the commonly observed accumulation of phytoplankton in oceans. Finally, collective motion microorganisms alter the temperature microstructure and lead to higher mixing with increasing stratification. Insights obtained from the investigations for an individual swimmer's motion and interactions between a pair of swimmers in a stratified fluid explain these observations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MMPB-MS19.html#author2"><strong class="card-body-subgroup">Jeffrey L. Moran</strong></a>
		    	 
		    		<em> (Department of Mechanical Engineering, George Mason University, United States)</em>
		    	
		    	
		    		
		    			</br>"Chemokinesis-driven Accumulation of Artificial Microswimmers in Low-Motility Regions of Fuel Gradients"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Motile cells often detect and respond to changes in their local chemical environment by changing their speed or direction, which allows them to carry out important functions including finding nutrients, immune response, or predator evasion. Two common examples are chemotaxis (motion up or down a chemical concentration gradient) and chemokinesis (dependence of speed on chemical concentration). Chemokinesis is distinct from chemotaxis in that no directional sensing or reorientation capabilities are required. Over the past 15+ years, researchers have developed 'artificial microswimmers' or 'microrobots' that move at speeds that usually depend on the concentration of a chemical 'fuel' (chemokinesis). However, the behavior of artificial microswimmers in fuel gradients has not been thoroughly characterized and the extent to which they exhibit chemotaxis is not fully known. Here, we study the behavior of half-platinum half-gold self-propelled rods in steady state, antiparallel gradients of hydrogen peroxide fuel and potassium chloride salt, which tend to increase and decrease the rods' speed, respectively. Brownian Dynamics simulations, a Fokker-Planck theoretical model, and experiments demonstrate that at steady state, the chemokinetic self-propelled rods accumulate in low-speed (salt-rich, peroxide poor) regions not because of chemotaxis, but because of chemokinesis. The agreement between simulations, model, and experiments bolsters the role of chemokinesis in this system and validates previous theoretical findings [Popescu et al., Nano Lett. 18, 9 (2018)] that chemokinesis alone cannot lead to chemotaxis. This work suggests a novel strategy of exploiting chemokinesis to effect the accumulation of artificial microswimmers in desired areas, which could find application in environmental remediation, wound healing, and drug delivery for cancer treatment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MMPB-MS19.html#author3"><strong class="card-body-subgroup">Eva Kanso</strong></a>
		    	 
		    		<em> (University of Southern California, United States)</em>
		    	
		    	
		    		
		    			</br>"Emergent Waves in Ciliary Carpets"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Motile cilia often line internal epithelial surfaces with thousands of multiciliated cells, each containing hundreds of cilia. Their coordinated motion drives flows with important biological functions in the respiratory, cerebrospinal, and reproductive systems in humans. Cilia coordination has been studied extensively at the level of pairs of cilia, and even in collections of cilia with metachronal waves. However, a general theory for investigating the hydrodynamics of cilia coordination in large systems remains lacking. Here, starting from discrete arrays of cilia, wherein each cilium is represented by a well-known oscillator model, we devise a fast numerical algorithm for investigating the dynamics of thousands of hydrodynamically-coupled cilia. We then develop a continuum theory in the limit of infinitely many independently beating cilia by combining  tools from active matter with classical Stokes flow methods. We analyze  the stability of isotropic and synchronized states and show that they are unstable. Surprisingly, traveling wave patterns emerge in both the discrete and continuum theory regardless of initial conditions, indicating that these waves are global attractors.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/MMPB-MS19.html#author4"><strong class="card-body-subgroup">David Saintillan</strong></a>
		    	 
		    		<em> (Mechanical and Aerospace Engineering, University of California San Diego, United States)</em>
		    	
		    	
		    		
		    			</br>"An Integrated Chemomechanical Model of Sperm Locomotion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mammalian sperm cells achieve locomotion by the spontaneous periodic oscillation of their flagellum. Dynein motors inside the flagellum consume energy from ATP to exert active sliding forces between microtubule doublets, thus creating bending waves along the flagellum and enabling the sperm cell to swim in a viscous medium. Using a sliding-control model of the axoneme that accounts for the coupling of motor kinetics with elastic deformations, we develop a chemomechanical model of a freely swimming sperm cell that accounts for the effect of non-local hydrodynamic interactions between the sperm head and flagellum. The model is shown to produce realistic beating patterns and swimming trajectories, which we analyze as a function of sperm number and motor activity. Remarkably, we find that the swimming velocity does not vary monotonically with motor activity, but instead displays two local maxima corresponding to distinct modes of swimming.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/NEUR-MS19.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS19-NEUR: </small></br>Biological Rhythms and Motor Control</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yangyang Wang (University of Iowa, USA), Peter Thomas (Case Western Reserve University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-NEUR</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/NEUR-MS19.html#author1"><strong class="card-body-subgroup">Yangyang Wang</strong></a>
		    	 
		    		<em> (University of Iowa, USA)</em>
		    	
		    	
		    		
		    			</br>"Shape and timing: using variational analysis to dissect motor robustness"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			To survive and reproduce, an animal must adjust to changes in its internal state and the external environment. We refer to the ability of a motor system to maintain performance despite perturbations as “robustness”. Although it is well known that sensory feedback supports robust adaptive motor behaviors, specific mechanisms of robustness are not well understood either experimentally or theoretically. In this work, we explore how sensory feedback could alter a neuromechanical trajectory to enhance robustness for motor control. As a concrete example, we focus on a piecewise smooth neuromechanical model of triphasic motor patterns in the feeding apparatus of the marine mollusk, <em>Aplysia californica</em>. We investigate the mechanisms by which sensory feedback generates robust adaptive behavior, quantify the robustness of the <em>Aplysia</em> model to the applied perturbation (increased mechanical load), and compare them to experimental observations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/NEUR-MS19.html#author2"><strong class="card-body-subgroup">Zhuojun Yu</strong></a>
		    	 
		    		<em> (Case Western Reserve University, USA)</em>
		    	
		    	
		    		
		    			</br>"A homeostasis criterion for Limit cycle systems based on infinitesimal shape response curves"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Homeostasis occurs in a control system when a quantity remains approximately constant as a parameter, representing an external perturbation, varies over some range. Golubitsky and Stewart (J.~Math.~Biol., 2017) developed a notion of infinitesimal homeostasis for equilibrium systems using singularity theory. Rhythmic physiological systems (breathing, locomotion, feeding) maintain homeostasis through control of large-amplitude limit cycles rather than equilibrium points. Here we take an initial step to study (infinitesimal) homeostasis for limit-cycle systems in terms of the emph{average} of a quantity taken around the limit cycle. We apply the infinitesimal shape response curve (iSRC) introduced by Wang et al.~(SIAM J.~Appl.~Dyn.~Sys, to appear) to study infinitesimal homeostasis for limit-cycle systems in terms of the emph{mean} value of a quantity of interest, averaged around the limit cycle. Using the iSRC, which captures the linearized emph{shape} displacement of an oscillator upon a static perturbation, we provide a formula for the derivative of the averaged quantity with respect to the control parameter. Our expression allows one to identify homeostasis points for limit cycle systems in the averaging sense. We demonstrate in the Hodgkin-Huxley model and in a metabolic regulatory network model that the iSRC-based method provides an accurate representation of the sensitivity of averaged quantities.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/NEUR-MS19.html#author3"><strong class="card-body-subgroup">Silvia Daun</strong></a>
		    	 
		    		<em> (University of Cologne, Germany)</em>
		    	
		    	
		    		
		    			</br>"Stimulus transformation into motor action: Dynamic graph analysis on neural oscillations reveals aging effects on brain network communication"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cognitive performance slows down with increasing age. This includes cognitive processes that are essential for the performance of a motor act, such as the slowing down in response to an external stimulus. The objective of this study was to identify aging-associated functional changes in the brain networks that are involved in the transformation of external stimuli into motor action. To investigate this topic, we employed dynamic graphs based on phase-locking of Electroencephalography signals recorded from healthy younger and older subjects while performing a simple visually-cued finger-tapping task. The network analysis yielded specific age-related network structures varying in time in the low frequencies (2-7 Hz), which are closely connected to stimulus processing, movement initiation and execution in both age groups. The networks in older subjects, however, contained several additional, particularly interhemispheric, connections and showed an overall increased coupling density. Cluster analyses revealed reduced variability of the subnetworks in older subjects, particularly during movement preparation. In younger subjects, occipital, parietal, sensorimotor and central regions were-temporally arranged in this order-heavily involved in hub nodes. Whereas in older subjects, a hub in frontal regions preceded the noticeably delayed occurrence of sensorimotor hubs, indicating different neural information processing in older subjects. All observed changes in brain network organization, which are based on neural synchronization in the low frequencies, provide a possible neural mechanism underlying previous fMRI data, which report an overactivation, especially in the prefrontal and pre-motor areas, associated with a loss of hemispheric lateralization in older subjects.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/NEUR-MS19.html#author4"><strong class="card-body-subgroup">Ansgar Bueschges</strong></a>
		    	 
		    		<em> (University of Cologne, Germany)</em>
		    	
		    	
		    		
		    			</br>"Task-specificity in the control of insect walking"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			When terrestrial animals locomote through their environment they need to control the rhythmic stepping movements of each leg as well as the coordination between all stepping legs, being it two, four, six or eight legs to continuously assure stability as well as to optimally serve the actual behavioral task. The presentation will report recent advances in unravelling the neural organization and operation of the walking system in six legged insects by focusing on walking direction and speed in the fruit fly. Individual descending interneurons from the brain were identified, which are in charge of controlling walking direction. Fruit flies generate a continuum of interleg coordination patterns spanning from wave gait to tetrapod to tripod coordination with increasing walking speed from less than one bodylength/s to more than 15 bodylengths/s assuring optimal stability. Removal of single legs indicates that the leg muscle control system of the fruit fly is organized in a modular fashion with segmental rhythm generating networks.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS19/ONCO-MS19.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS19-ONCO: </small></br>Measuring and modeling the cell-state transitions in cancer progression and treatment</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Mohit Kumar Jolly ( Assistant Professor, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India), Kishore Hari (PhD Student, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS19/ONCO-MS19.html#author1"><strong class="card-body-subgroup">Sabrina L Spencer</strong></a>
		    	 
		    		<em> (Assistant Professor, Department of Biochemistry, University of Colerado-Boulder, United States of America)</em>
		    	
		    	
		    		
		    			</br>"  Real-time visualization of rapid escape from BRAF inhibition in single melanoma cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Despite the increasing number of effective anti-cancer therapies, successful treatment is limited by the development of drug resistance. While the contribution of genetic factors to drug resistance is undeniable, little is known about how drug-sensitive cells first evade drug action to proliferate in drug. Here we track the responses of thousands of single melanoma cells to BRAF inhibitors and show that a subset of cells escapes drug via non-genetic mechanisms within the first three days of treatment. Cells that escape drug rely on ATF4 stress signalling to cycle periodically in drug, experience DNA replication defects leading to DNA damage, and yet out-proliferate other cells over extended treatment. Together, our work reveals just how rapidly melanoma cells can adapt to drug treatment, generating a mutagenesis-prone subpopulation that expands over time.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ONCO-MS19.html#author2"><strong class="card-body-subgroup">Yogesh Goyal</strong></a>
		    	 
		    		<em> (Postdoctoral researcher, University of Pennsylvania, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Cellular plasticity and fate choices in single cancer cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			While cellular processes are often reproducible and precise, cells may also alter their molecular states and adopt new fates in response to stimuli, a phenomena referred to as “plasticity”. I am interested in understanding the control principles governing cellular plasticity and fate decisions in response to mutational and pharmacologic stresses in tissue development and single cancer cells. My postdoctoral work is motivated by recent studies revealing how rare and transient non-genetic fluctuations in individual cancer cells enable them to survive pharmacologic stress, such as molecularly targeted therapies. Unlike the binary nature of Darwinian selection whereby mutations are either present or not, non-genetic fluctuations can exist on one, or even multiple continuums of variation. How this non-genetic variability maps to the eventual resistant fates upon drug exposure is an emerging paradigm of cellular plasticity. Integrating novel theoretical and experimental frameworks, I will present my findings on 1. Identifying the origins and nature of the unique transcriptional molecular states underlying this plasticity; and 2. Connecting these molecular states to their eventual drug-resistant fates by tracking thousands of uniquely barcoded cell lineages. Moving forward, my own group will adapt these quantitative approaches and concepts to measure, model, and engineer plasticity and its roles in tissue development and disease.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ONCO-MS19.html#author3"><strong class="card-body-subgroup">Qing Nie</strong></a>
		    	 
		    		<em> ( Professor of Mathematics and Developmental & Cell Biology, University of California, Irvine ; Director of The NSF-Simons Center for Multiscale Cell Fate Research, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Inference and Multiscale Model of Epithelial-to-Mesenchymal Transition via Single-cell Transcriptomic Data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Epithelial to mesenchymal transition (EMT) plays an important role in many biological processes during development and cancer. The advent of single-cell transcriptome sequencing techniques allows the dissection of dynamical details underlying EMT with unprecedented resolution. We develop an integrative tool that combines unsupervised learning of single-cell transcriptomic data and multiscale mathematical modeling to analyze transitions during cell fate decision. Our approach allows identification of individual cells making transition between all cell states and inference of genes that drive transitions. Multiscale extractions of single-cell scale outputs naturally reveal intermediate cell states (ICS) and ICS-regulated transition trajectories, producing emergent population-scale models to be explored for design principles. Testing on the single-cell gene regulatory network model and applying to published single-cell EMT datasets in cancer and embryogenesis, we uncover the roles of ICS on adaptation, noise attenuation, and transition efficiency in EMT, and reveal their trade-off relations. Meanwhile, network topology analysis and multilayer gene-gene regulation networks suggest that the ICS during EMT serve as the signaling hub in the TGF-β signaling communication.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS19/ONCO-MS19.html#author4"><strong class="card-body-subgroup">Einar Gunnarsson</strong></a>
		    	 
		    		<em> (Graduate student, University of Minnesota, Twin Cities, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Modeling the role of phenotypic switching in cancer drug resistance"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The emergence of acquired drug resistance in cancer represents a major barrier to treatment success. In this talk, we describe a simple mathematical model for studying how phenotypic switching at the single-cell level affects resistance evolution in cancer. We discuss how even short-term epigenetic modifications and stochastic fluctuations in gene expression can drive long-term drug resistance in the absence of any bona fide resistance mechanisms. We also show that an epigenetic drug that slightly perturbs the average retention of the resistant phenotype can turn guaranteed treatment failure into guaranteed success. We finally examine how the mode and time scale of resistance acquisition depends on the underlying switching dynamics and discuss potential implications for treatment.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-19
										</br>
										Thursday, June 17 at 09:30am (PDT)</br>Thursday, June 17 at 05:30pm (BST)</br>Friday, June 18 01:30am (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>










    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
