<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Contributed Talk Session - CT08 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Contributed Talk Session - CT08 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Contributed Talk Session - CT08" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/CT08/" />
<meta property="og:url" content="http://localhost:4000/CT08/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/CT08/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Contributed Talk Session - CT08","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Contributed Talk Session - CT08">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Contributed Talk Session - CT08" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Contributed Talk Session - CT08" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Contributed Talk Session - CT08</h1>
        <h4>Wednesday, June 16 at 10:30pm (PDT)</br>Thursday, June 17 at 06:30am (BST)</br>Thursday, June 17 02:30pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="CT08">Contributed Talk Session - CT08</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT08.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT08-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT08.html#author1"><strong class="card-body-subgroup">Flavia Feliciangeli</strong></a>
		    	 
		    		<em>University of Leeds, Bayer AG</em>
		    	
		    	
		    		
		    			</br>"How is a population of final cells maintained? A compartmental branching approach for cell differentiation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell differentiation is a process through which a generic cell evolves into a given type of cell, usually into a more specialized type. Cells of the human body have nearly identical genome but exhibit very different phenotypes that allow them to carry out specific functions and react to changes in the surrounding environment. We can model cells sharing the same surface attributes (same phenotype) as belonging to the same mathematical state (or compartment). Cells can either die, divide or change phenotype (entering another compartment). We derive a cell-compartmental model for cell differentiation; by defining a family of random variables we can model the progeny of a founder cell as a stochastic process. We can describe the evolution of mean quantities by a set of ordinary differential equations and we analyse a number of summary statistics to bring insight to the understanding of cellular dynamics. We show, with two case studies from Cellular Immunology, how our mathematical techniques can shed light on the dynamics of cell differentiation in different systems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author2"><strong class="card-body-subgroup">Sakurako Tanida</strong></a>
		    	 
		    		<em>The University of Tokyo</em>
		    	
		    	
		    		
		    			</br>"Organoid morphogenesis at various luminal fluid pressure and proliferation time in a multicellular phase-field model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Organoids are self-organizing cells that are grown from stem cells in vitro and are widely used to model organ development and disease. In organoids, while cell growth and hence proliferation are mechanically constrained due to the geometrical requirements to keep maintaining the cell cluster, various morphologies of organoids are achieved. However, it remains elusive how such mechanical constraint can affect organoid growth and the final morphology. In this study, we investigate the influences of mechanical constraint on organoid morphogenesis by numerical simulations with a multicellular phase-field model. In this mathematical model, we can isolate out mechanical interaction from other biological processes. More specifically, we examine the pattern formations of organoids emerging when changing luminal fluid pressure and proliferation time. Even if most organoids seem to be the same in the initial phase, they have distinctive features in the later phase in this numerical model. The patterns in the later phase include spheroid-like shape, star-like shape, and so on. Although all cells have identical natures, in the star-like organoid, cells that can divide are spatially fixed and show behavior like spontaneous differentiation. Classifying the patterns of organoids by several indexes, we discuss the mechanisms which generate the different pattern.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author3"><strong class="card-body-subgroup">Kana Fuji</strong></a>
		    	 
		    		<em>The University of Tokyo</em>
		    	
		    	
		    		
		    			</br>"Lumenogenesis simulations of organoids using a multicellular phase-field model with molecules of apical components"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Organoids are three-dimensional cultured organ models grown from stem cells. Epithelial organoids which have apico-basal polarity in each cell form lumens on the apical side and the lumens grow during the cell self-organization process. In addition to osmotic pressure in luminal fluid and cortical tension, the presence of non-adhesive apical membranes is involved in luminal area expansion recently. However, it remains unclear how the lumenogenesis processes were affected localized adhesive property changes in the membrane. In this study, to investigate the luminal pattern formed by cells with localized non-adhesive membranes, we extended the multicellular phase-field model with luminal fluid to a multicellular model with polarity by introducing the molecules of the apical component. The results of simulations with this model showed that lumens were formed even at pressures lower than the pressure required for the lumen growth without the introduction of the cell polarity. This model reproduces not only the round lumen but also the squeezed lumen that was not available in the previous models. In this talk, we will discuss the quantitative analysis of the lumen structure and its comparison with experiments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author4"><strong class="card-body-subgroup">Mete Demircigil</strong></a>
		    	 
		    		<em>Institut Camille Jordan, Lyon</em>
		    	
		    	
		    		
		    			</br>"Aerotactic Waves in Dictyostelium discoideum : When Self-Generated Gradients engage with Expansion by Cell Division."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Using a self-generated hypoxic assay, it is shown that Dictyostelium discoideum displays a remarkable collective aerotactic behavior: when a cell colony is covered, cells quickly consume the available oxygen and form a dense ring moving outwards at constant speed and density. We propose a simple, yet original PDE model, that enables an analytical qualitative and quantitative study of the phenomenon and reveals that the collective migration can be explained by the interplay between cell division and the modulation of aerotaxis. The modeling and its conclusions supplement and are confirmed by an experimental investigation of the cell population behavior. This approach also gives rise to an explicit and novel formula of the collective migration speed of cells that encapsulates a surprising combination of expansion by cell divison, such as described by the Fisher/KPP equation, and aerotaxis. The conclusions of this model appear to extend to more complex models.This is joint work with Christophe ANJARD Vincent CALVEZ, Jean-Paul RIEU, Olivier COCHET-ESCARTIN and and is a subpart of the work presented in the preprint bioRxiv 2020.08.17.246082; doi: https://doi.org/10.1101/2020.08.17.246082.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ECOP/ECOP-CT08.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>CT08-ECOP: </small></br>ECOP Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ECOP/ECOP-CT08.html#author1"><strong class="card-body-subgroup">Prince Harvim</strong></a>
		    	 
		    		<em>University of Ottawa</em>
		    	
		    	
		    		
		    			</br>"Transmission Dynamics and Control Mechanisms of Vector-Borne Diseases with Active and Passive Movements Between Urban and Satellite Cities"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We formulate a metapopulation model to investigate the role of active and passive mobility on the spread of an epidemic between an urban center connected to a satellite city. The epidemic disease considered is transmitted via both sexual and vector mode (eg Zika virus). The basic reproduction number of the disease is explicitly determined as a combination of sexual and vector-borne transmission parameters. The sensitivity analysis reveals that the disease is primarily transmitted via the vector-borne mode, rather than via sexual transmission, and that sexual transmission by itself may not initiate or sustain an outbreak. Furthermore, increasing the mobility of the population from urban center to the satellite city leads to an increase in the basic reproduction number of the satellite city but a decrease in the basic reproduction number in the urban center. We explore the potential effects of optimal control strategies relying upon several distinct restrictions on population movement. We find that although travel restrictions from the urban center to the satellite city may reduce the prevalence of the disease in the satellite city, significant control measures targeting the densely populated cities are required in order to eradicate the disease in the entire region.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ECOP/ECOP-CT08.html#author2"><strong class="card-body-subgroup">Qianying Lin</strong></a>
		    	 
		    		<em>University of Michigan, Ann Arbor</em>
		    	
		    	
		    		
		    			</br>"Viral Phylodynamics and A Class of Markov Genealogy Processes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Phylodynamic studies aim to extract information on the population process of pathogens from genome sequences. In this talk, I focus on the relationship between genealogies or phylogenies reconstructed from sampled virus genomes and the population processes that generate them. I show how the problem is naturally formulated in terms of a class of interrelated Markov processes that are built on the stochastic dynamics of births and deaths in the population. For interesting transmission models, the exact likelihood is intractable, but I show how to construct an efficient sequential Monte Carlo algorithm to estimate it with high accuracy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ECOP/ECOP-CT08.html#author3"><strong class="card-body-subgroup">Sipkaduwa Arachchige Sashika Sureni Wickramasooriya</strong></a>
		    	 
		    		<em>Clarkson University</em>
		    	
		    	
		    		
		    			</br>"Biological Control via Alternative Food to Predator"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Biological control is a means by which pest/invasive populations are kept in check by the use ofnatural enemies of the pest, or perhaps even parasites, pathogens or a combination thereof. The classic work ofSrinivasu et.  al. demonstrates how such a process can be facilitated, by providing additional food to an introducedpredator to control a target pest. A critical assumption in the literature is that the additional food is constant.Theoretical studies carried out previously in this direction indicate that incorporating mutual interference betweenpredators can stabilize the system. In this work, Beddington–DeAngelis type functional response has been used tomodel the mutual interference between predators. The conditions for eradication of pest is derived and the mainconcern is to determine whether the model exhibit different bifurcation. Various biological implications of ourmathematical results are drawn in conclusion.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-CT08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/EVOP/EVOP-CT08.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>CT08-EVOP: </small></br>EVOP Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/EVOP/EVOP-CT08.html#author1"><strong class="card-body-subgroup">Laurence Ketchemen Tchouaga</strong></a>
		    	 
		    		<em>University of Ottawa</em>
		    	
		    	
		    		
		    			</br>"Population density in fragmented landscapes under monostable and bistable dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A model for a single species population which propagates in a heterogeneous landscape in a one dimensional space is presented. The landscape is composed of two kind of patches with different diffusivities. The dynamics of the population is studied through a reaction diffusion model on which the net growth function can be a monostable or bistable function. In addition, we consider that at the interface between patch types, individuals may show preference for more favorable regions. We study the different nonlinear steady state models. We prove existence of monotone solution in each model and classify their qualitative shape. An analysis is done to study the effect of the diffusivity coefficient. A stability analysis is also done for each model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT08.html#author2"><strong class="card-body-subgroup">Luigi Esercito</strong></a>
		    	 
		    		<em>Bielefeld University</em>
		    	
		    	
		    		
		    			</br>"Lines of descent in a Moran model with frequency-dependent selection and mutation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Dealing  with  the  interplay  of  mutation  and  selection  is  one  of  the  important challenges in population genetics. We consider two variants of the two-type Moran model with mutation and frequency-dependent selection, namely a scheme with nonlinear dominance and another with what we name fittest-type-wins scheme. We show the equivalence of the two variants and pursue the latter for further analysis.In particular, we trace the genealogy of a sample of individuals backward in time, via an appropriate version of the so-called ancestral selection graph (ASG), originally introduced by Krone and Neuhauser. We use the information contained in mutation events to reduce the ASG to the parts informative with respect to the type distribution of the present population and their ancestors, respectively. This leads to the killed ASG and the pruned lookdown ASG in this setting, which we use to derive representations for the (factorial) moments of the type distribution and the ancestral type distribution by connecting forward and backward graphical models via duality relationships.Finally, we show how the results carry over to the diffusion limit.[1]  Baake, Ellen, Luigi Esercito, and Sebastian Hummel. 'Lines of descent in a Moran model with frequency-dependent selection and mutation.' textit{arXiv preprint arXiv:2011.08888} (2020).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT08.html#author3"><strong class="card-body-subgroup">Léonard Dekens</strong></a>
		    	 
		    		<em>Institut Camille Jordan, Université Claude Bernard Lyon 1</em>
		    	
		    	
		    		
		    			</br>"Quantitative Trait in a Patchy Environment: Beneath the Gaussian Approximation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Assuming Gaussian trait distributions is central in quantitative genetic models in order to describe complex evolutionary dynamics, like source-sink scenarii in heterogeneous environments. However, the mechanisms of why and when this is a reasonable approximation remain unclear. Here, we investigate the underlying role of sexual reproduction by introducing a new framework that directly involves the dynamics of the trait distributions. We opt for an infinitesimal model operator to model the transmission of a complex trait under sexual reproduction. We apply this approach to revisit a classical study in a patchy environment (following Ronce and Kirkpatrick 2001). We first justify the Gaussian assumption in a small variance regime with perturbative techniques. We next perform a rigorous separation of ecological and evolutionary time scales to complete the analytical description of source-sink type equilibria, numerically described in Ronce and Kirkpatrick 2001. Our analysis highlights the relative influence of the blending effects of migration and sexual reproduction on local adaptation patterns.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-CT08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MEPI/MEPI-CT08.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>CT08-MEPI: </small></br>MEPI Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MEPI/MEPI-CT08.html#author1"><strong class="card-body-subgroup">Wongyeong Choi</strong></a>
		    	 
		    		<em> Department of Mathematics, Soongsil University</em>
		    	
		    	
		    		
		    			</br>" Mathematical modeling and optimal intervention strategies to control COVID-19"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The ongoing outbreak of the novel coronavirus disease (COVID-19) has considerably affected public health and the economy worldwide.Optimizing control measures is urgent given the substantial societal and economic impacts associated with infection and interventions. We established mathematical models to determine the optimal strategies. We used game theory to identify the individually optimal strategy, and optimal control theory  to find optimal strategies that minimize the costs associated with infection and intervention. When social distancing and testing with contact tracing are considered as intervention strategies, the results demonstrate that testing should be maintained at a maximum level in the early phases and after the peak of the epidemic, whereas social distancing should be intensified when the prevalence of the disease is greater than 15%. After the peak of the pandemic, it would be optimal to gradually relax social distancing and switch back to testing. Additionally, we identified the individually optimal strategy based on the Nash strategy when social distancing and vaccination are available as control strategies. We determined the relative costs of control strategies at which individuals preferentially adopt vaccination over social distancing (or vice versa).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT08.html#author2"><strong class="card-body-subgroup">David Wu</strong></a>
		    	 
		    		<em>University of Auckland</em>
		    	
		    	
		    		
		    			</br>"Likelihood-based estimation and prediction for misspecified epidemic models: an application to measles in Samoa"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prediction of the progression of an infectious disease outbreak in a population is an important task. Differential equations are often used to model an epidemic outbreak's behaviour but are challenging to parametrise.  Furthermore, these models can suffer from misspecification, which biases the estimates. In this talk we present our recent work on an explicitly likelihood-based variation of the generalised profiling method as a tool for prediction and inference under model misspecification. Our approach allows us to carry out identifiability analysis and uncertainty quantification using profile likelihood-based methods without the need for marginalisation. Rather than introduce an explicit stochastic process model, generalised profiling uses a deterministic model as an approximately enforced `smoothing penalty' term and uses maximisation rather than integration to handle nuisance parameters. We provide additional justification for this approach by introducing a novel interpretation of the model approximation component as a stochastic constraint, and preserves the rationale for using profiling rather than integration to remove nuisance parameters while still providing a link back to explicitly stochastic models. We present results applying our approach to data from an outbreak of measles in Samoa and show that fast, accurate prediction is possible.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT08.html#author3"><strong class="card-body-subgroup">Jiyeon Suh</strong></a>
		    	 
		    		<em>School of Mathematics and Computing (Computational Science and Engineering), Yonsei University, Seoul, Republic of Korea</em>
		    	
		    	
		    		
		    			</br>"Cost-benefit analysis of tafenoquine for the relapse prevention of Plasmodium vivax malaria in South Korea"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Plasmodium vivax malaria has not been eradicated in South Korea since 1993 and the government is aiming to grant certification for malaria elimination from the WHO in 2024. P. vivax malaria has a dormant liver-stage, and this can cause relapse. Tafenoquine has been proven to effectively prevent relapse as an alternative to primaquine. In this study, we developed a model for P. vivax malaria using delay differential equations to estimate the impact of tafenoquine introduction on malaria burden. We also conducted a cost-benefit analysis of tafenoquine from the payer's perspective based on the cost and benefit extracted from the national health insurance data and performed probabilistic sensitivity analysis. The results showed that the introduction of tafenoquine could prevent 77.78% of relapse and 12.27% of total malaria cases over 10 years compared to primaquine. And the cost-benefit analysis provided an incremental cost of $13,115 and an incremental benefit of $165,520 resulting in an incremental benefit-cost ratio of 12.26. Furthermore, the sensitivity analysis showed a consistent result with a probability of 98.3%. Hence, the introduction of tafenoquine can reduce the malaria burden and is beneficial over primaquine. These findings support the introduction of tafenoquine to step toward malaria elimination in South Korea.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT08.html#author4"><strong class="card-body-subgroup">Andrew Nugent</strong></a>
		    	 
		    		<em>University of Warwick</em>
		    	
		    	
		    		
		    			</br>"Analysing early warning signals of disease elimination by approximating the potential surface"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The theory of critical slowing down states that a system displays increasing relaxation times as it approaches a critical transition. Such changes in relaxation times can be seen in statistics generated from data timeseries, which can be used as early warning signals of a transition. While analytic equations have been derived for various early warning signals in a variety of epidemiological models, there is frequent disagreement with the general theory of critical slowing down, with some indicators performing well when used in prevalence data but not when applied to incidence data. We investigate this effect in an SIS model by reconstructing the potential surface for different types of data. By modelling prevalence, incidence and the rate of infection as stochastic differential equations, then using an equation-free method to approximate their drift functions from simulated timeseries, we reconstruct the potential surface for each data type. Slowly varying parameters provides insight into how the shape of the potential surface changes. Analytic equations for the drift functions are also derived for comparison with simulated results, showing that the potential surface for all data types becomes shallower upon the approach to a critical transition from either direction, as predicted by critical slowing down.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-CT08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Contributed Talk Session - CT08
										</br>
										Wednesday, June 16 at 10:30pm (PDT)</br>Thursday, June 17 at 06:30am (BST)</br>Thursday, June 17 02:30pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
