<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-20 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-20 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-20" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS20/" />
<meta property="og:url" content="http://localhost:4000/MS20/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS20/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-20","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-20">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-20" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-20" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-20</h1>
        <h4>Thursday, June 17 at 11:30am (PDT)</br>Thursday, June 17 at 07:30pm (BST)</br>Friday, June 18 03:30am (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS20">Minisymposia-20</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/CBBS-MS20.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS20-CBBS: </small></br>WiMB: Mathematical modeling and its application</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Qimin Huang (Case Western Reserve University, USA), Katie Storey (University of Michigan, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/CBBS-MS20.html#author1"><strong class="card-body-subgroup">Atanaska Dobreva</strong></a>
		    	 
		    		<em> (Arizona State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Investigating pathological mechanisms in cone photoreceptor vitality and the timing of rescue strategies via bifurcation analysis and time-varying sensitivity analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Photoreceptors are the sensory cells of the eye, which perform the most essential role in vision. There are two types of photoreceptors: rods for night and peripheral vision and cones for color vision. Glucose is the main fuel for photoreceptors, and they break it down to form lactate, lipids and other metabolites needed to create energy and to renew the light-absorbing outer segments, which are periodically shed. Thus, properly functioning metabolic processes ensure the structural integrity and vitality of photoreceptors. The progression of degenerative retinal diseases such as age-related macular degeneration (AMD) and retinitis pigmentosa (RP) has been linked to nutrient deprivation. We analyzed a mathematical model for the metabolic dynamics of a cone photoreceptor via bifurcation analysis and time-varying global sensitivity analysis (GSA) in order to identify factors that increase the risk of cone degeneration in AMD and RP when glucose supply to photoreceptors is low. Our results indicate that the factors of greatest importance include glucose availability and transport, utilization of lipids for photoreceptor outer segment renewal and -oxidation of fatty acids to provide auxiliary metabolites for energy production. In addition, the GSA helped to uncover insights into timing of intervention strategies to rescue the cone cell.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CBBS-MS20.html#author2"><strong class="card-body-subgroup">Katie Storey</strong></a>
		    	 
		    		<em> (University of Michigan, USA)</em>
		    	
		    	
		    		
		    			</br>"A Framework for Performing Data-Driven Modeling of Tumor Growth with Radiotherapy Treatment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent technological advances make it possible to collect detailed information about tumors, and yet clinical assessments about treatment responses are typically based on sparse datasets. In this work, we propose a workflow for choosing an appropriate model, verifying parameter identifiability, and assessing the amount of data necessary to accurately calibrate model parameters. We compare tumor growth models of varying complexity in an effort to determine the level of complexity needed to accurately predict tumor growth dynamics and response to radiotherapy. We consider a simple, one-compartment ordinary differential equation model which tracks tumor volume and a two-compartment model that accounts for tumor volume and the fraction of necrotic cells within the tumor. We investigate the structural and practical identifiability of these models, and the impact of noise on identifiability. We also generate synthetic data from a complex, spatially- resolved, cellular automaton model (CA), investigating the fit of the ODE models to tumor volume data generated by the CA, using sequential model calibration. Our results suggest that if tumor volume data alone is provided then a tumor with a large necrotic volume is the most challenging case to fit. However, supplementing data on total tumor volume with additional necrotic information enables the two-compartment ODE model to perform significantly better than the one-compartment model, in terms of parameter convergence and predictive power.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CBBS-MS20.html#author3"><strong class="card-body-subgroup">Qimin Huang</strong></a>
		    	 
		    		<em> (Case Western Reserve University, USA)</em>
		    	
		    	
		    		
		    			</br>"Investigating the impact of combination phage and antibiotic therapy: A modeling study"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Antimicrobial resistance (AMR) is a serious threat to global health today. The spread of AMR, along with the lack of new drug classes in the antibiotic pipeline, has resulted in a renewed interest in phage therapy, which is the use of bacteriophages to treat pathogenic bacterial infections. This therapy, which was successfully used to treat a variety of infections in the early twentieth century, had been largely dismissed due to the discovery of easy-to-use antibiotics. However, the continuing emergence of antibiotic resistance has motivated new interest in the use of phage therapy to treat bacterial infections. We have modeled an ODE system to investigate the effect of immune system on combination treatment of the phage and antibiotic. Our result shows the frequency and concentration of dose as well as the timing of phage administration are important factors of the combination phage therapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CBBS-MS20.html#author4"><strong class="card-body-subgroup">Hwayeon Ryu</strong></a>
		    	 
		    		<em> (Elon University, USA)</em>
		    	
		    	
		    		
		    			</br>"Bifurcation and sensitivity analysis reveal key drivers of multistability in a model of macrophage polarization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We analyze a mathematical model for polarization of a single macrophage which, despite its simplicity, exhibits complex dynamics in terms of multistability. In particular, we demonstrate that an asymmetry in the regulatory mechanisms and parameter values is important for observing multiple phenotypes. Bifurcation and sensitivity analyses show that external signaling cues are necessary for macrophage commitment and emergence to a phenotype, but that the intrinsic macrophage pathways are equally important. Based on our numerical results, we formulate hypotheses that could be further investigated by laboratory experiments to deepen our understanding of macrophage polarization.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/CDEV-MS20.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS20-CDEV: </small></br>Dynamics and networks in single-cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/CDEV-MS20.html#author1"><strong class="card-body-subgroup">Amy Brock</strong></a>
		    	 
		    		<em> (University of Texas at Austin, USA)</em>
		    	
		    	
		    		
		    			</br>"Clonally-resolved mapping of cancer cell trajectories under therapeutic pressure"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Heterogeneity across individual cancer cells and clonal populations impacts growth rate, tumor composition, and response to therapy. To improve treatment, new tools are required to measure and control the contributions of diverse cell subpopulations. Our lab has developed a high-complexity expressed barcode system, ClonMapper, that integrates expressed cell barcoding with single-cell RNA-sequencing and clonal isolation to characterize and track subpopulation trajectories. Using this approach, we uncovered subsets of cells from breast cancer cell models with distinct transcriptional signatures and chemotherapy survivorship trajectories. To gain a deeper understanding of the process of clonal diversification, we profiled clones and retrieved sub-clones over the course of expansion and treatment. Supervised learning indicated that clonal subpopulations have characteristic transcriptomic signatures that are well-conserved under a variety of therapeutic perturbations. By providing the capability for systematic dissection of complex clonal dynamics, ClonMapper enables the delineation of an underlying engine of clonal diversification in cancer cell populations and refines our understanding of clonal identity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CDEV-MS20.html#author2"><strong class="card-body-subgroup">Meghan Ferrall-Fairbanks</strong></a>
		    	 
		    		<em> (University of Florida, USA)</em>
		    	
		    	
		    		
		    			</br>"Single-cell eco-evolutionary dynamics of intratumor heterogeneity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Researchers have recognized that a one-size-fits-all approach is not effective at treating cancer and that tumor heterogeneity plays an important role in response. Current dogma stipulates that this heterogeneity results from compounding genetic and epigenetic changes and instability, ultimately driving unfavorable outcomes for these patients. Nonetheless, some cancers, including many pediatric cancers and some leukemias, have limited genomic diversity. As a result, we have a limited ability to stratify patients into high versus low-risk groups. To address this, we explore intratumor heterogeneity at the single-cell transcriptomic level to quantify and identify driving phenotypes in tumor evolution. We leverage the generalized diversity index (GDI) from ecology, which allows us to tailor the scale of cellular diversity in a given context. We show that the order of diversity parameter in GDI allows us to either emphasize clonal richness at low values while high values shift the analysis toward the abundance of potential drivers of the tumor evolution. We have explored GDI changes in both in vitro sequential single-cell RNA sequencing samples across many cancer tissue types including breast, lung, and ovarian cell lines as well as treatment naïve and treated patient samples in chronic myelomonocytic leukemia. Our analyses show how quantifying intratumor heterogeneity with GDI is a powerful tool to understand eco-evolutionary dynamics of a patient’s tumor.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CDEV-MS20.html#author3"><strong class="card-body-subgroup">Stephen Williams</strong></a>
		    	 
		    		<em> (10X Genomics, USA)</em>
		    	
		    	
		    		
		    			</br>"Analyzing spatial and high-resolution single cell multi-omic data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chromium Single Cell and Visium Spatial Solutions from 10x Genomics provide the ability to build a more comprehensive multidimensional understanding of complex biological systems. Analyzing datasets that measure different molecular modes in thousands of single cells or in a spatial context can be challenging. To enable interpretation of these complex datasets, 10x Genomics provides analysis and visualization software for evaluating genomic, epigenomic, transcriptomic, and proteomic data. In this seminar, we will discuss software, methods, and guidance that can be used to analyze your 10x Genomics datasets.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/CDEV-MS20.html#author4"><strong class="card-body-subgroup">Sihem Cheloufi</strong></a>
		    	 
		    		<em> (University of California, Riverside, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical modeling of chromatin accessibility to predict stem cell plasticity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Stem and progenitor cells become progressively more restricted in their differentiation potential. This process of cell fate determination is driven by lineage-specific transcription factors and is accompanied by dynamic changes in chromatin structure. The chromatin assembly factor complex CAF-1 plays a central role in assembling nucleosomes during DNA replication and has been implicated in regulating cellular plasticity in various lineages in different organisms. However, whether CAF-1 sustains lineage identity during normal homeostasis by influencing chromatin accessibility is unknown. To address this question, we investigated the role of CAF-1 in a myeloid lineage differentiation paradigm. CAF-1 suppression in myeloid progenitors triggered their rapid commitment but incomplete differentiation toward granulocyte, megakaryocyte, and erythrocyte lineages, resulting in a mixed cellular state. Through comparison with a canonical program of directed terminal myeloid differentiation, we define changes in chromatin accessibility that underlie a unique single cell transcriptome of the matured CAF-1 deficient cells. Using mathematical modeling we further predict the cell fate trajectories of the mixed cellular state caused by CAF-1 inhibition. We use nonlinear dimensionality reduction algorithms to produce RNA velocity maps, pseudotime alignment and corresponding differentiation trajectories. We infer an equation of motion from the RNA velocity vector field, which allows a probability density function to be derived and, ultimately, to compute state transition probabilities. Importantly, the resulting mathematical model of cell fate will be used to identify, interrogate, and quantify states of genome organization and predict corresponding changes to cell identities. Furthermore, we can use the model to predict how interfering with lineage specific transcription factors or factors involved in chromatin organization will modify the differentiation trajectories. Future work with this system will identify new targets to restore normal hematopoiesis in disease and generate clinically relevant cell types.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/DDMB-MS20.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS20-DDMB: </small></br>Data-Driven Modeling and Analysis in Mathematical Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Tomas Carino-Bazan (University of California, Santa Barbara, United States), Daniel Wilson (Boston University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/DDMB-MS20.html#author1"><strong class="card-body-subgroup">Daniel Wilson</strong></a>
		    	 
		    		<em> (Boston University, United States)</em>
		    	
		    	
		    		
		    			</br>"Inferring the molecular reach of antibodies from antigen binding data using an agent-based spatial model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Surface Plasmon Resonance (SPR) is a widely-used biophysical technique used to produce high-resolution temporal signals of molecular binding interactions. In SPR, one molecule is immobilised on a 3D matrix whilst another, known as the analyte, is injected over the surface. The instrument provides a highly sensitive measure of binding in the matrix. When the analyte is monovalent, the binding data can be fit by a well-mixed 1:1 binding model to determine the kinetic rate constants. However, there are many situations where the analyte is bivalent. A prominent example is the study of antibodies that have two binding sites for their immobilised antigen. This produces complex SPR binding data that is not well fit by the 1:1 binding model. In this talk, we present a computational method to infer the binding parameters from bivalent analytes. Using a stochastic spatial model of bivalent binding we train a surrogate model that allows for highly efficient parametrisation of antibody SPR data. In addition to inferring binding parameters, our new method allows us to estimate the ‘molecular reach’ of antibodies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/DDMB-MS20.html#author2"><strong class="card-body-subgroup">Paul Atzberger</strong></a>
		    	 
		    		<em> (University of California, Santa Barbara, United States)</em>
		    	
		    	
		    		
		    			</br>"Variational Autoencoders with Manifold Latent Spaces for Learning Nonlinear Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We develop data-driven methods for learning parsimonious representations of nonlinear dynamical systems by incorporating physical information and other priors.  Our approach is based on Variational Autoencoders (VAEs) for learning nonlinear state space models from observation data.  VAE use noise-based regularizations and priors to help ensure continuity in latent encoding and in disentangling latent features.  To obtain low dimensional parsimonious representations, we introduce ways to incorporate geometric and topological priors through general manifold latent spaces.  We demonstrate our methods for learning non-linear dynamics in non-linear fluid mechanics and reaction-diffusion systems. Co-authors: Ryan Lopez, Paul J. Atzberger, University of California Santa Barbara.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/DDMB-MS20.html#author3"><strong class="card-body-subgroup">Guy Wolf</strong></a>
		    	 
		    		<em>(Université de Montréal; Mila - Quebec AI Institute, Canada)</em>
		    	
		    	
		    		
		    			</br>"Multiscale exploration of single cell data with geometric harmonic analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			High-throughput data collection technologies are becoming increasingly common in many fields, especially in biomedical applications involving single cell data genomics and transcriptomics. These introduce a rising need for exploratory analysis to reveal and understand hidden structure in the collected (high-dimensional) Big Data. A crucial aspect in such analysis is the separation of intrinsic data geometry from data distribution, as (a) the latter is typically biased by collection artifacts and data availability, and (b) rare subpopulations and sparse transitions between meta-stable states are often of great interest in biomedical data analysis. In this talk, I will show several tools that leverage manifold learning, graph signal processing, and harmonic analysis for biomedical (in particular, genomic/proteomic) data exploration, with emphasis on visualization, and nonlinear feature extraction, and multiresolution analysis. A common thread in the presented tools is the construction of a data-driven diffusion geometry that both captures intrinsic structure in data and provides a generalization of Fourier harmonics on it. These, in turn, are used to process data features along the data geometry for multiple purposes, including preprocessing of single cell data and enabling batch-level geometric exploration, e.g., over and between medical conditions, health states, and drug reactions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/DDMB-MS20.html#author4"><strong class="card-body-subgroup">John Lagergren</strong></a>
		    	 
		    		<em> (Oak Ridge National Laboratory, United States)</em>
		    	
		    	
		    		
		    			</br>"Data-driven network analysis detects longitudinal environmental changes with impacts on food, energy, and pandemics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			To address the needs of a growing human population, which includes the significant expansion of sustainable food and bio-energy production capacities in the context of a changing climate, we develop novel climatype identification methods to predict longitudinal processes relevant to these challenges. In this work, we leverage the DUO algorithm to compute two-way and three-way environmental comparisons at unprecedented scale and accuracy to find high-order relationships between geospatial coordinates with high resolution at global scale. Novel network analysis methods are applied to the series of emergent climatype networks to identify climate zones that share similar environmental relationships and to track how these relationships are changing over time. The methods discussed herein are also applicable to correlation analyses in other diverse fields such as systems biology, ecology, materials science, carbon cycles, biogeochemistry, additive manufacturing, and zoonosis research.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/ECOP-MS20.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>MS20-ECOP: </small></br>Population Dynamics Across Interacting Networks or Scales</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Necibe Tuncer (Florida Atlantic University, USA), Hayriye Gulbudak (	 University of Louisiana at Lafayette, USA), 	 Cameron Browne (University of Louisiana at Lafayette, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-ECOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/ECOP-MS20.html#author1"><strong class="card-body-subgroup">Maia Martcheva</strong></a>
		    	 
		    		<em> (University of Florida, USA)</em>
		    	
		    	
		    		
		    			</br>"A Network Immuno-epidemiological Model of HIV and Opioid Epidemics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We introduce a network immuno-epidemiological model of HIV and opoid epidemics where the jointly affected class is structured by the within-host dynamics. We fit the within-host model to data, collected in monkeys. We compute the reproduction numbers of the HIV and opiod epidemics. We show that the disease-free equilibrium is locally stable if both reproduction numbers are below one, and unstable if at least one of the reproduction numbers is above one. The HIV-only equilibrium exists if the reproduction number of HIV is larger than one. The opioid-use only equilibrium exists if the reproduction number of opioid use is larger than one. The HIV-only equilibrium is locally asymptotically stable if the invasion number of the opioid epidemic is below one and unstable if the invasion number of opoioid epidemic is above one. The opoioid-only equilibrium is locally asymptotically stable if the invasion number of the HIV epidemic is below one and unstable if the invasion number of HIV epidemic is above one. Simulation suggest that larger networks lead to higher reproduction numbers.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ECOP-MS20.html#author2"><strong class="card-body-subgroup">Stanca M. Ciupe</strong></a>
		    	 
		    		<em> (Virginia Tech, USA)</em>
		    	
		    	
		    		
		    			</br>"Neutrophil dynamics and their role in disease: a multi-scale investigation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The highly controlled migration of neutrophils toward the site of an infection can be altered when they are challenged with competing external signals, leading to their dysregulation and oscillatory movement. In this talk, I will use mathematical models to evaluate the mechanistic interactions responsible for neutrophil migratory decision-making and to determine molecular and cellular contributions to disease pathogenesis. The results are applicable to  sepsis and SARS-CoV-2 infections.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ECOP-MS20.html#author3"><strong class="card-body-subgroup">Michael Cortez</strong></a>
		    	 
		    		<em> (Florida State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Using sensitivity analysis to explore the context dependent relationships between host species richness and disease prevalence"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In multi-host communities, the dilution effect is the phenomenon wherein focal host infection prevalence (i.e., the fraction of infected individuals in a focal host species) decreases with increases in host species richness.  The opposite phenomenon is called an amplification effect.  Empirical and theoretical studies show that relationships between host species richness and prevalence are likely to be context dependent, depending on the identity of the host species present in and added to a given community. However, current theory is limited in its ability to identify the context-dependent rules governing host species richness-prevalence relationships.  This is due, in part, to modeling studies making different assumptions about the pathogen transmission mechanism, the presence/absence of interspecific interactions between host species, and the characteristics of the host species (e.g., competence and competitive ability).
In this talk, I show how sensitivity analysis applied multihost-pathogen models can yield insight into how host characteristics, host density, and the pathogen transmission mechanism affect infection prevalence in a focal host.  Specifically, I present an n-host model of an environmentally transmitted pathogen and show that it can unify common epidemiological ODE models for direct and environmental transmission under a single framework via fast-slow dynamical systems theory. I then use local sensitivity analysis applied to endemic equilibrium of the model to analytically derive the relationships between focal host infection prevalence and host densities and model parameters.  This identifies how host competence, density, and the pathogen transmission mechanism jointly shape host richness-disease relationships. For example, the strength of interspecific host competition determines whether responses in focal host infection prevalence to increased density of a non-focal host are driven by the characteristics of the non-focal host or other host species in the community.  I interpret these results in terms of factors promoting amplification and dilution of disease.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ECOP-MS20.html#author4"><strong class="card-body-subgroup">Juan B. Gutiérrez</strong></a>
		    	 
		    		<em> (University of Texas at San Antonio, USA)</em>
		    	
		    	
		    		
		    			</br>"Data, reality, and cognitive dissonance. On modeling what we don’t see with data we don’t have."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During the ongoing COVID-19 pandemic, the discrepancy between daily reports of cases and the trajectory of the disease has posed a substantial challenge to modeling efforts. In this talk, we will present the contrast between patient data and daily counts for the City of San Antonio, TX. We will demonstrate that a non-autonomous adjustment to data deficiencies can substantially improve forecasts. We present the extension of this method to multi-strain outbreaks. An exact data correction is possible with detailed patient data and genomic sequencing of the pathogen, which might not be available in all localities. To alleviate this problem, we propose a framework that incorporates information at multiple spatial and temporal scales to estimate the non-autonomous data correction. A derivation of classic quantities (R_o, R_e) is presented for a SEYAR model (Susceptible, Exposed, Symptomatic, Asymptomatic, Recovered) under this framework.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/EVOP-MS20.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>MS20-EVOP: </small></br>Evolutionary Theory of Disease</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jesse Kreger (University of California, Irvine, United States), Natalia Komarova (University of California, Irvine, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-EVOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/EVOP-MS20.html#author1"><strong class="card-body-subgroup">Chadi M. Saad-Roy</strong></a>
		    	 
		    		<em> (Princeton University, United States)</em>
		    	
		    	
		    		
		    			</br>"The evolution of an asymptomatic infectious stage: analysis of a simple evolutionary-epidemiological model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Pathogens exhibit numerous life-history strategies. An important pathogen characteristic is the degree of symptoms exhibited by hosts at the onset of infectiousness. Additionally, mediated by host immunity, a pathogen may elicit reduced (or no) symptoms in the first stage leading to simultaneously slower progression and lower transmission. In this talk, we examine the evolutionary implications of these trade-offs. We couple a simple epidemiological model with evolutionary analyses, and we find that numerous evolutionary outcomes are possible. For simple trade-off formulations, these include a fully symptomatic or asymptomatic first infectious stage, a subsymptomatic first stage, or bistability between a fully symptomatic and asymptomatic first stage. Then, we discuss the ensuing implications for disease mitigation measures.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/EVOP-MS20.html#author2"><strong class="card-body-subgroup">Jasmine Foo</strong></a>
		    	 
		    		<em> (University of Minnesota, United States)</em>
		    	
		    	
		    		
		    			</br>"Power law transitions in site frequency spectra of neutrally evolving tumors"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The site frequency spectrum (SFS) is a popular genomic summary statistic that tracks the frequencies of mutations in a population sample.  In the context of cancer, the site frequency spectrum of tumor samples are commonly used to gain insights into tumor evolutionary processes.  However, recent analyses of the SFS in tumor population models have generally dealt with special or limiting cases, such as considering only cells with an infinite line of descent, assuming deterministic tumor growth, or taking large time/population limits.  I will discuss recent work in which we derive exact expressions for the mean SFS in a neutrally evolving tumor. We find that the rates of cell birth and death change the shape of the SFS at the small frequency end, inducing a transition between power laws as cell viability decreases. We demonstrate how, in principle, this insight may be used to estimate the ratio between cell birth and cell death rates, as well as the mutation rate, using SFS data alone.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/EVOP-MS20.html#author3"><strong class="card-body-subgroup">Mohammad Kohandel</strong></a>
		    	 
		    		<em> (University of Waterloo, Canada)</em>
		    	
		    	
		    		
		    			</br>"Predicting mutability of the genomic segments of a pathogen"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			There is an essential need to better understand how a pathogenesis, like SARS-CoV-2, is affected by mutations and to determine the conserved segments in the genome that can serve as stable targets for novel therapeutics. We introduce a text-mining method to estimate the mutability of genomic segments directly from a reference (ancestral) whole genome sequence. The method relies on calculating the importance of genomic segments based on their spatial distribution and frequency over the whole genome. To validate our approach, we perform a large-scale analysis of the viral mutations in nearly 80,000 publicly-available SARS-CoV-2 predecessor whole genome sequences and show that these results are highly correlated with the segments predicted by the text-mining method. Importantly, these correlations are found to hold at the codon and gene levels, as well as for gene coding regions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/EVOP-MS20.html#author4"><strong class="card-body-subgroup">Alison Hill</strong></a>
		    	 
		    		<em> (Johns Hopkins University, Institute for Computational Medicine, United States)</em>
		    	
		    	
		    		
		    			</br>"Selection for SARS-CoV-2 variants at the within-host and population scale"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The evolution of novel variants has become an increasing concern as the COVID-19 pandemic has progressed into 2021. In this talk we will discuss modeling work to understand the factors driving SARS-CoV-2 evolution within individual hosts and across populations. We examine and compare mutations that increase transmission and those that evade adaptive immunity. We build models to include and analyze the roll of within-host clinical course, heterogeneities in transmission, population structure, and the nature of acquired and vaccine-induced immunity on the fate of variants emerging at different times and places throughout the pandemic.  We find that several unique features of COVID-19, including the timing of peak infectiousness vs the onset of adaptive immune responses in infected individuals, and the asynchronous spatiotemporal nature of epidemics around the world, contribute to patterns observed in the evolution of new variants.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/IMMU-MS20.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>MS20-IMMU: </small></br>The pressing need for within-host models of the pulmonary immune response</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Luis Sordo Vieira (Division of Pulmonary, Critical Care, and Sleep Medicine, University of Florida, United States), Marissa Renardy (University of Michigan/Applied BioMath, United States), Tracy Stepien (Department of Mathematics, University of Florida, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-IMMU</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/IMMU-MS20.html#author1"><strong class="card-body-subgroup">Julie Leonard-Duke</strong></a>
		    	 
		    		<em> (University of Virginia/Robert M. Berne Cardiovascular Research Center, United States)</em>
		    	
		    	
		    		
		    			</br>"Computational Modeling of Fibroblast Subpopulations in Idiopathic Pulmonary Fibrosis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Each year in this country, 40,000 patients are diagnosed with idiopathic pulmonary fibrosis (IPF), a progressive and terminal disease caused by excessive extracellular matrix production by fibroblasts in distributed lesions, or “fibrotic foci”, throughout the lung.  Fibroblasts are the primary pathologic cell population in fibrosis and their presence has been shown to be essential for fibrotic foci formation. Their actions, such as proliferating, secreting collagen, or differentiation into myofibroblasts, is driven by a combination of mechanical and chemical cues that eventually lead to a pathologic phenotype in IPF. Recent literature suggests that there are sub-populations of fibroblasts in the lung that exhibit different phenotypes depending on chemical and mechanical signals present in their local environment. Understanding how fibroblast phenotypic heterogeneity contributes to fibrotic foci formation in the dynamic lung environment of progressive IPF is an overarching goal of our research team and has important implications in the design of new therapies for IPF. Our group has recently performed single-cell RNAseq analysis on human lung fibroblasts exposed to a combination of pro-inflammatory cytokines to recapitulate the IPF lung environment. This analysis has led to the identification of fibroblast sub-populations that may behave differently from one another in response to their local and changing environment. To better understand the consequences of these phenotypic differences on lung tissue remodeling, our team is combining data-driven analyses with multi-scale agent-based modeling that simulates intracellular signaling and multi-cell interactions to predict cell-specific behaviors that are crucial to the formation of fibrotic foci in IPF.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/IMMU-MS20.html#author2"><strong class="card-body-subgroup">Amber M. Smith</strong></a>
		    	 
		    		<em> (University of Tennessee Health Science Center, USA)</em>
		    	
		    	
		    		
		    			</br>"Bacterial coinfections: from influenza to SARS-CoV-2"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Influenza virus infected individuals often become coinfected with a bacterial pathogen, which significantly enhances morbidity and mortality. These bacterial coinfections have contributed to 45-95% of mortality during influenza pandemics, and numerous host and pathogen mechanisms have been identified through various experimental and mathematical modeling approaches. Given the history of influenza-bacterial coinfections, this was an obvious fear for the ongoing SARS-CoV-2 pandemic. Thus far, there is some evidence that SARS-CoV-2 also increases susceptibility to bacterial infections but does so to a lesser extent compared to influenza. To better understand the potential for SARS-bacteria coinfection, we infected mice with SARS-CoV-2 followed by pneumococcus. Our data support clinical observations and highlight specific host responses that may play a role in the increased pathogenicity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/IMMU-MS20.html#author3"><strong class="card-body-subgroup">Elsje Pienaar</strong></a>
		    	 
		    		<em> (Purdue University, United States)</em>
		    	
		    	
		    		
		    			</br>"Mycobacterium avium infection in the lungs: and agent-based model exploring early infection events"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			INTRODUCTION: Mycobacterium avium complex (MAC), members of the nontuberculous mycobacteria family, are environmental microbes, capable of colonizing and infecting humans following inhalation of the bacteria. MAC-pulmonary disease is notoriously difficult to treat and prone to recurrence, and both incidence and prevalence have been increasing [1]. There are two types of MAC lung infection – fibrocavitary and nodular, with fibrocavitary much harder to treat, and with much lower cure rates, as low as 76% even with optimal treatment [2]. MAC are well known to form biofilms and diverse colonies in the environment. These biofilms have been shown to aid in epithelial cell invasion [3], cause premature apoptosis in macrophages [4], and inhibit antibiotic efficacy [5]. We hypothesize that both phenotypic diversity and biofilm formation are key to establishing and prolonging infections in the lung. To address these hypotheses, we developed a model that shows the interactions between bacteria, biofilm and immune cells as an agent-based model (ABM). This model allows us to explore both the intracellular scale (bacterial phenotypes and macrophage killing), and tissue scale (biofilm formation and epithelial invasion). METHODS: We used Repast Simphony to develop a three-dimensional ABM of in vivo MAC colonization to infection within the first 14 days post-deposition. The grid represents a length of lung airway with a layer of mucus/epithelial lining fluid (ELF). Bacteria agents are divided into either sessile (slow-growing, within biofilm and less susceptible to antibiotics), or planktonic (more quickly growing but not protected by biofilm) phenotypes. Biofilm is represented by continuous variables in each grid compartment, with values corresponding to the amount of extracellular matrix produced by bacteria in that grid compartment. To represent the protective properties of biofilm, the amount of biofilm is inversely related to the likelihood of a macrophage phagocytosing bacteria from that biofilm. All bacterial agents also release a chemoattractant that is represented by continuous variables in each grid compartment, and that diffuses throughout the grid. Macrophages probabilistically follow this chemoattractant gradient. Macrophages can phagocytose bacteria, prioritizing planktonic bacteria (not within biofilms), which infect the macrophage. Infected macrophages then have a probabilistic chance of killing internal bacteria. Macrophages also accumulate “apoptotic signal” through exposure to biofilm and internal bacteria. RESULTS: The model was parameterized through a literature search, test cases based on in vitro experiments and Latin Hypercube Sampling for unknown parameter values. We found that parameters affecting macrophage chemotaxis and recruitment have significant impact on the number of macrophages, but not on the number or distribution of bacteria. Initial parameters – the initial bacteria count, initial macrophage count, and ratio of planktonic to sessile bacteria - have lasting impacts throughout the simulation. Parameters that pertain to only one bacterial subpopulation (e.g. extracellular growth rates) are not significantly correlated with outcomes overall, because the composition of the bacterial populations varies so much between simulations. Finally, we have found that biofilm increases the number of bacterial cells that invade the epithelium, but in the absence of biofilm bacteria are able to persist in the airways. Higher biofilm levels also increase macrophage chemo-attractant production, death and recruitment. The most significant biofilm parameter is the amount that is deposited with bacteria in the lung upon initial exposure. Our simulations indicate that, based on in vitro data, once bacteria are deposited in the lung they cannot generate biofilm quickly enough to have a significant an impact. CONCLUSIONS: We have developed a multiscale agent-based model that allows us to study the initial colonization and infection in MAC-pulmonary disease on both the cellular- and tissue level. Early results show that initial parameters have lasting effects on the outcome of the deposition. Further, we have found that biofilms are not necessary to establish fibrocavitary type of MAC infection. Future directions of this work include organization of the infection into nodules, adding drug pharmacokinetics and pharmacodynamics to better understand the role of bi¬¬ofilm in treatment efficacy.
REFERENCES: 1. Lee, et al. Antimicrob Agents Chemother, 59(6): 2972-2977, 2015. 2. Hwang, et al. Eur Respir J, 49(3): 2017. 3. Yamazaki, et al. Cell Microbiol, 8(5): 806-814, 2006. 4. Rose and Bermudez. Infect Immun, 82(1): 405-412, 2014. 5. Falkinham. J Med Microbiol, 56(Pt 2): 250-254, 2007.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/IMMU-MS20.html#author4"><strong class="card-body-subgroup">Angela Reynolds</strong></a>
		    	 
		    		<em> (Virginia Commonwealth University, United States)</em>
		    	
		    	
		    		
		    			</br>" Mathematical modeling of lung inflammation from insult to recovery"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Lung inflammation can be triggered by many insults including viral and bacterial infections, structural damage, or inhalation of dangerous particles. The associated lung injury can resolve quickly, be treated effectively through various interventions, become a chronic problem, or lead to death. Given the variety of possible responses, often seen from the same insult, and the necessity for the lungs to function effectively mathematical modeling has become a necessary tool for improving lung health. Researchers have used mathematical modeling to understand immune system dynamics during a number of pulmonary infections and injuries, identify key mechanisms, and provide important insights into new treatments and to help identify who needs an intervention. In this talk we will review and explore recent research in mathematical modeling of inflammation in the lung and look into how mathematical modeling and computational methods can be used to guide interventions.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/MEPI-MS20.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS20-MEPI: </small></br>Models of COVID-19 Vaccination, Non-Pharmaceutical Interventions, and Relaxation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jane Heffernan (York University, Canada), Miranda Teboh Ewungkem (Lehigh University, USA), Zhilan Feng (Purdue University, USA), John Glasser (Centres for Disease Control, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-MEPI</strong>.
					
					
						The third session is <strong>MS16-MEPI</strong>.
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/MEPI-MS20.html#author1"><strong class="card-body-subgroup">Melanie Prague</strong></a>
		    	 
		    		<em> (University of Bordeaux, France)</em>
		    	
		    	
		    		
		    			</br>"Multi-level modeling of COVID-19 epidemic dynamics in French regions, estimating the combined effects of multiple non-pharmaceutical interventions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We developed a multi-level model of the French COVID-19 epidemic at the regional level. We rely on a global extended Susceptible-Exposed-Infectious-Recovered (SEIR) mechanistic model as a simplified representation of the average epidemic process, with the addition of region specific random effects. We estimate region-specific key parameters of epidemics dynamics such as the transmission rate conditionally on the mechanistic model through Stochastic Approximation Expectation Maximization (SAEM) optimization using Monolix software. Using French dataset of hospitalisation in France over a course of one year, we estimate the effect of non-pharmaceutical interventions adjusting for weather, vaccination and apparition of more transmissible variants. The proposed novel methodology, consisting in using population approach to compartmental epidemic models, allows to compare with satisfactory efficiency the different effects of intervention and derive informative epidemics parameters such as region-specific effective reproductive numbers and attack rates.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MEPI-MS20.html#author2"><strong class="card-body-subgroup">Gerardo Chowell</strong></a>
		    	 
		    		<em> (Georgia State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Ensemble modeling approaches for forecasting infectious disease spread"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The ongoing COVID-19 pandemic presents with an unprecedented opportunity to evaluate the performance of mathematical modeling frameworks for forecasting the trajectory of the pandemic at different spatial and temporal scales. I will present new ensemble modeling approaches that can outperform individual models in short-term forecasts without substantially increasing model complexity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MEPI-MS20.html#author3"><strong class="card-body-subgroup">Iain Moyles</strong></a>
		    	 
		    		<em>(York University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Cost and Social Distancing Dynamics in a Mathematical Model of COVID-19"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present an SEIAR mathematical model of COVID-19 which includes social distancing and relaxation. Our model has a dynamic behavioural influence where the decision for susceptible people to isolate is a function of the total and active cases, but the decision to stop isolating is a function of the perceived cost of isolation. Along with this social distancing cost, we define an overburden healthcare cost due to the strain placed on the healthcare system with a high caseload. We demonstrate that, non-intuitively, increasing either isolation activity or incentive to isolate do not always lead to optimal health outcomes. We further demonstrate that an increase in the frequency of isolation events, each of shorter duration, can lead to improved outcomes compared to sustained isolation activity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MEPI-MS20.html#author4"><strong class="card-body-subgroup">Seyed Moghadas</strong></a>
		    	 
		    		<em> (York University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Effectiveness of COVID-19 Vaccines in the Context of Emerging Variants"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Newly emerged SARS-CoV-2 variants represent a challenge for current vaccines as preliminary results suggest increased transmissibility as well as variable levels of cross-reaction depending on the viral strain. These selection advantages along with constraints in vaccine supply and distribution may drive certain immune escape variants to dominance in the near future, hampering the ability of vaccination to control the pandemic. In our research, we utilize dynamic transmission models to evaluate COVID-19 vaccination strategies including evaluating the effects of a delayed second dose and projected the shifting dynamics of viral circulation in the presence of emerging variants.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/MFBM-MS20.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS20-MFBM: </small></br>Algebra, Combinatorics, and Topology in Modern Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Daniel Cruz (Georgia Institute of Technology, U.S.), Margherita Maria Ferrari (University of South Florida, U.S.)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/MFBM-MS20.html#author1"><strong class="card-body-subgroup">Mustafa Hajij</strong></a>
		    	 
		    		<em> (Santa Clara University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"TDA-Net: Fusion of Persistent Homology and Deep Learning Features for COVID-19 Detection in Chest X-Ray Images"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Topological Data Analysis (TDA) has emerged recently as a robust tool to extract and compare the structure of datasets. TDA identifies features in data such as connected components and holes and assigns a quantitative measure to these features. Several studies reported that topological features extracted by TDA tools provide unique information about the data, discover new insights, and determine which feature is more related to the outcome. On the other hand, the overwhelming success of deep neural networks in learning patterns and relationships has been proven on a vast array of data applications, images in particular. To capture the characteristics of both powerful tools, we propose TDA-Net, a novel ensemble network that fuses topological and deep features for the purpose of enhancing model generalizability and accuracy. We apply the proposed TDA-Net to a critical application, which is the automated detection of COVID-19 from CXR images. The experimental results showed that the proposed network achieved excellent performance and suggests the applicability of our method in practice.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MFBM-MS20.html#author2"><strong class="card-body-subgroup">Hector Banos</strong></a>
		    	 
		    		<em> (Dalhousie University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Identifiability of Species Network Topologies from Genomic Sequences"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Hybridization plays an important role during the evolutionary process of some species. In such cases, phylogenetic trees are sometimes insufficient to describe species-level relationships. We show that most topological features of a level-1 species network (a network with no interlocking cycles) are identifiable under the network multi-species coalescent model (NMSC) using the log-det distance between aligned DNA sequences of concatenated genes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MFBM-MS20.html#author3"><strong class="card-body-subgroup">Nida Obatake</strong></a>
		    	 
		    		<em> (Texas A&M University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Mixed Volume of Chemical Reaction Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chemical reaction networks model the interactions of chemical substances. An important invariant of a chemical reaction network is its maximum number of positive steady states. This number, however, is in general difficult to compute. We introduce an upper bound on this number - namely, a network's mixed volume - that is easy to compute. We show that, for certain biological signaling networks, the mixed volume does not greatly exceed the maximum number of positive steady states. Furthermore, we investigate this overcount and also compute the mixed volumes of small networks (those with only a few species or reactions).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MFBM-MS20.html#author4"><strong class="card-body-subgroup">Raina Robeva</strong></a>
		    	 
		    		<em> (Randolph-Macon College, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Algebraic Biology in the Curriculum"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Unlike difference and differential equations models, algebraic models in biology have remained largely invisible at the undergraduate level despite their increasing popularity in solving a wide range of biological problems. This discrepancy is puzzling as, in many cases, an introduction to algebraic modeling relies on mathematics covered in traditional mathematics courses. In addition, Boolean and finite dynamical systems, could be used as an alternative to modeling system dynamics by way of difference and differential equations. The talk will discuss the benefits of increasing the profile of algebraic models in the undergraduate curriculum and share insights from an honors class in systems biology for students with minimal mathematics and biology backgrounds.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/MMPB-MS20.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>MS20-MMPB: </small></br>Deterministic and stochastic models for complex cardiovascular phenomena</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Martina Bukac (University of Notre Dame, United States), Daniele Schiavazzi (University of Notre Dame, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-MMPB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/MMPB-MS20.html#author1"><strong class="card-body-subgroup">Mitchel Colebank</strong></a>
		    	 
		    		<em> (North Carolina State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling and simulation of fluid dynamics in chronic thromboembolic pulmonary hypertension"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A compromised pulmonary vasculature can lead to pulmonary hypertension (PH), defined by a mean pulmonary arterial blood pressure (mPAP) exceeding 20 mmHg. Though there have been advances in PH treatments, only chronic thromboembolic pulmonary hypertension (CTEPH) is considered curable. CTEPH is characterized by multiple recurrent or unresolved pulmonary emboli that impede flow to the alveoli. The disease causes perfusion defects, causing small vessel disease in both obstructed and unobstructed territories. Those with lesions in the smaller arteries are treated by balloon pulmonary angioplasty (BPA), though treatment planning is clinic dependent. To address this, we propose a multiscale model of CTEPH hemodynamics that couples a one-dimensional computational fluid dynamics model (1D CFD) of the large arteries to a linearized CFD model of the small arteries and arterioles. The former is conducted in an image based geometry, while the latter fluid dynamics are simulated in a fractal, structured tree. We also integrate two pressure-loss models, mimicking typical CTEPH lesions. Our results show that the model framework predicts common phenotypes of CTEPH, including perfusion deficits, small vessel flow imbalances, and elevated mPAP. Lastly, we use the 1D model to predict hemodynamic improvements after virtual BPA, laying the foundation for an in-clinic treatment planning tool.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MMPB-MS20.html#author2"><strong class="card-body-subgroup">Charles Puelz</strong></a>
		    	 
		    		<em> (Baylor College of Medicine and Texas Children's Hospital, United States)</em>
		    	
		    	
		    		
		    			</br>"A fluid/structure interaction model of the human heart"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk will focus on our efforts towards building a computational model of the entire human heart, including the blood, valves, heart chambers, great vessels, and peripheral circulations. The heart tissues are assumed to be anisotropic hyperelastic materials immersed in blood, and blood itself is modeled as a viscous incompressible Newtonian fluid. The equations of motion are solved using the immersed finite element method. In this numerical approach, tissue displacements and forces are approximated on finite element meshes and blood velocities and pressures are approximated on a fixed and possibly locally refined Cartesian grid.  Tissue geometries are generally imaged based, and constitutive laws for the tissues depend on fiber directions calculated using Poisson interpolation.  Peripheral circulations in the form of 3-element Windkessel models provide boundary conditions for the heart model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MMPB-MS20.html#author3"><strong class="card-body-subgroup">Jae Lee</strong></a>
		    	 
		    		<em> (Johns Hopkins University, United States)</em>
		    	
		    	
		    		
		    			</br>"Fluid-structure interaction models of bioprosthetic heart valves to study leaflet kinematics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bioprosthetic heart valves (BHVs) are commonly used in surgical and percutaneous valve replacement. The durability of percutaneous valve replacement is unknown, but surgical valves have been shown to require reintervention after 10--15 years. Further, smaller-diameter surgical BHVs generally experience higher rates of prosthesis-patient mismatch (PPM), which leads to higher rates of failure. Bioprosthetic aortic valves can flutter in systole, and fluttering is associated with fatigue and failure in flexible structures. The determinants of flutter in BHVs have not been well characterized, however, despite their potential to impact durability. We use an experimental pulse duplicator and a computational fluid-structure interaction model of this system to study the role of device geometry on BHV dynamics. The experimental system mimics physiological conditions, and the computational model enables precise control of leaflet biomechanics and flow conditions to isolate the effects of variations in BHV geometry on leaflet dynamics. We systematically characterize the impact of BHV diameter and leaflet thickness on fluttering dynamics. Ultimately, understanding the effects of device geometry on leaflet kinematics may lead to more durable valve replacements.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/MMPB-MS20.html#author4"><strong class="card-body-subgroup">Zachary Sexton</strong></a>
		    	 
		    		<em> (Stanford University, United States)</em>
		    	
		    	
		    		
		    			</br>"Multiscale Hemodynamics of Autogenerated Cardiovascular Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recapitulating the complex topologies and flow physics of meso/microvascular circulation precedes the manufacturing of functional, biofabricated tissues. In this work we leverage stochastic constrained constructive optimization (CCO) methods to automatically vascularize proposed cardiac tissue perfusion volumes. This approach seeks to optimize vascular topologies with respect to costs functions derived from total hydraulic resistance and blood volume constrained to geometric assumptions imposed by Murray’s law. We introduce techniques to partially bind intermediate network solutions to accelerate the optimization process while improving algorithmic precision compared to recent literature. To assess hemodynamics within these networks, we utilize multiscale 0D-3D models for computational fluid dynamics simulations with prescribed pulsatile inflows. We compare time-averaged pressures and volumetric flow rates across CCO models constructed with varying power law constraints and cost function formulations. Furthermore, we predict hemodynamic metrics crucial in wall homeostasis and adaptation including time-averaged wall shear stress, oscillatory shear index, and regions of low shear to better identify viable network topologies for biofabrication. Our pipeline will serve as an end-to-end, open-source solution for autogenerating vascular networks and verifying local flow behavior in future engineered tissues.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/NEUR-MS20.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS20-NEUR: </small></br>Biological Rhythms and Motor Control</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yangyang Wang (University of Iowa, USA), Peter Thomas (Case Western Reserve University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-NEUR</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/NEUR-MS20.html#author1"><strong class="card-body-subgroup">Jon Rubin</strong></a>
		    	 
		    		<em> (University of Pittsburgh, USA)</em>
		    	
		    	
		    		
		    			</br>"Combining rhythm generation and pattern formation in a core respiratory neural circuit"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Although respiration seems simple on the surface (breathe in, breathe out, repeat!), looks can be deceiving.  In this talk, I will (briefly) comment on two of the topics under active debate in the theory of the neural generation of respiratory rhythms.  First, I will consider the issue of how rhythmic activity in the inspiratory core (in the famous pre-Botzinger complex of the mammalian brain stem) can succeed or fail to recruit widespread neural activation and motor output. This work, with Ryan Phillips, is done in the setting of Hodgkin-Huxley type neural models with synaptic coupling that also takes into account dynamics of certain relevant ion concentrations.  Second, I will consider what happens when this rhythmic activity is embedded in the full neural circuit for respiration.  This part of the talk will be based on work with Jeff Smith done in the simpler setting of coupled relaxation oscillators.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/NEUR-MS20.html#author2"><strong class="card-body-subgroup">Casey Diekman</strong></a>
		    	 
		    		<em> (New Jersey Institute of Technology, USA)</em>
		    	
		    	
		    		
		    			</br>"Oxygen handling and parameter space interrogation in a minimalist closed-loop model of the respiratory oscillator"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Silent Hypoxemia, or happy hypoxemia is a puzzling phenomenon in which patients who have contracted COVID-19 exhibit very low oxygen saturations (SaO2 < 80%) yet experience no discomfort in breathing, or dyspnea. The mechanism by which this blunted response to hypoxia occurs is unknown. Our group has previously shown that a computational model (Diekman et al, 2017, J. Neurophys.) of the respiratory neural network can be used to test hypotheses focused on changes in chemosensory inputs to the central pattern generator (CPG). We hypothesize that altered chemosensory function at the level of the carotid bodies and/or the nucleus tractus solitarii are responsible for this blunted response to hypoxia. In this talk, we will use our model to explore this hypothesis by altering the properties of the gain function representing oxygen sensing inputs to the CPG. (Joint work with Christopher G. Wilson, Loma Linda University, and Peter J. Thomas, Case Western Reserve University.)
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/NEUR-MS20.html#author3"><strong class="card-body-subgroup">Todd Young</strong></a>
		    	 
		    		<em> (Ohio University, USA)</em>
		    	
		    	
		    		
		    			</br>"An Altered Van der Pol Oscillator and Stomatogastric Ganglion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In the Stomatogastric Ganglion or Pyloric Network of Lobsters the LP neuron bursts 1:1 with a pacemaker group (PD) in the intact network. However, isolated LP neurons cycle much more slowly than the pacemaker group. How is the LP neuron able to adjust its firing rate to match the fast pacemaker? We propose that an alteration of a slow conductance is sufficient to explain this phenomenon and we illustrate the principal in an altered van der Pol system.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/NEUR-MS20.html#author4"><strong class="card-body-subgroup">Yaroslav Molkov</strong></a>
		    	 
		    		<em> (Georgia State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Control of steering in quadrupedal locomotion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Traditionally, the studies of locomotor control in mammalian quadrupeds focus on spinal neural circuit organization that underlies varying patterns of limb movements (gaits) depending on the locomotor speed and other conditions. This intricate circuit includes neural rhythm generators that provide alternating flexion and extension phases for each limb, network interconnections between the generators providing proper interlimb coordination, descending control signals as well as proprioceptive feedback from the limbs. In the related experiments, the animal movements are usually restricted to walking or running along a straight path on a treadmill or over ground. Besides, to isolate particular functional components, the animals are often suspended or partly fixed. These restrictions make the balance control mechanisms irrelevant. However, during such complex maneuvers as turning, the timing of limb lifting and landing as well as limb positioning have to be tightly coordinated with the position of the center of mass to prevent the animal from falling. In addition, during turning movements quadrupedal mammals actively involve head/shoulder turning and body bending which further adds to the complexity of the control system. In this talk, we will use the experimental data on freely moving mice to develop  a simple mathematical model of quadrupedal locomotion that includes a balance control system interacting with the locomotor pattern generating circuits. We show that the balance control is involved not only in complex maneuvers but also operates during straight-line locomotion. We argue that body bending is a mechanism involved in the appropriate limb positioning which is an integral part of the balance control system and as such is necessary for efficient turning.  (Joint work with Ilya Rybak, Drexel University.)
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS20/ONCO-MS20.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS20-ONCO: </small></br>Integrating quantitative imaging and mechanistic modeling to characterize tumor growth and therapeutic response</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Guillermo Lorenzo (University of Pavia, Italy), David Hormuth (The University of Texas at Austin, US), Angela Jarrett (The University of Texas at Austin, US), Thomas Yankeelov (The University of Texas at Austin, US)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS20/ONCO-MS20.html#author1"><strong class="card-body-subgroup">Darren Tyson</strong></a>
		    	 
		    		<em> (Vanderbilt University, US)</em>
		    	
		    	
		    		
		    			</br>"The many dimensions of anticancer drug response—quantifying cell population dynamics at single-cell resolution using automated live-cell microscopy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A tumor in a human patient is an evolving system of interacting components, including different cell types containing various genetic alterations, adjacent stromal cells, and many different types of cell–cell and cell–matrix interactions. In addition, many parameters affect how therapeutic drugs can (ideally) kill all the tumor cells while sparing normal adjacent cells, including pharmacodynamic/pharmacokinetic properties of the drugs, the specificity with which they target tumor cells, specific genetic alterations that affect how a cell responds to the drug, etc. I will demonstrate how human cancer cell can be analyzed for their responses to anticancer drugs in high throughput using automated fluorescence microscopy to enable the direct visualization of many features of individual cells over time and how we have modeled the dynamics of cell population-level changes by simultaneously estimating the rates of cell division, death and entry into a non-dividing state from the single-cell measurements. This model facilitates the interpretation of how single-cell fate decisions affect the overall cell population dynamics in a drug concentration- and time-dependent manner that removes biases inherent in more traditional end-point measurements. I will describe how we have used this basic model as a framework to develop more detailed models to interrogate different aspects of tumor cell biology, including: 1) transitions into more drug-tolerant cell states; 2) potential synergistic action of combinations of drugs at different concentration; and 3) effects of matrix stiffness on cellular responses to drugs.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ONCO-MS20.html#author2"><strong class="card-body-subgroup">Victor Perez-Garcia</strong></a>
		    	 
		    		<em> (University of Castilla-La Mancha, Spain)</em>
		    	
		    	
		    		
		    			</br>"From metabolic imaging to biomarkers through mathematical models in cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumor initiation and progression are evolutionary processes driven by the accumulation of genetic alterations, some of which confer selective fitness advantages to cancer cells. Selective pressures induced by microenvironmental conditions, treatments, the immune system and other effects have a role in the complex evolutionary dynamics in tumors. However, although the situation is changing fast, it is still very difficult to obtain longitudinal biological data of evolutionary dynamics of tumors in individual patients. Metabolic imaging provides a global perspective of the tumor metabolism and proliferation status and can be performed sequentially to assess tumor dynamics and response to treatments. In this talk I will present a extension of the Fisher-Kolmogorov classical model displaying evolutionary dynamics. The analysis of the model predicts a displacement of the location of  metabolic hotspots from the tumor core to its periphery during its natural history. This fact allows to define a novel metabolic imaging biomarker based on the distance from the metabolic hotspot to the tumor centroid, that is found to correlate with tumor aggressiveness and patient survival for different tumor histologies.
Moreover, further analysis of the model shows that the maximum metabolic activity (SUVmax) grows with tumor size following a scaling law with power 1/4. A fact that was confirmed in different metabolic imaging datasets. Deviations from this scaling law allow to define another biomarker related to the relation between observed peak activity and the value expected from the scaling law. That provides another biomarker with a strong prognostic factor in breast cancer, lung cancer, head and neck cancer and glioblastoma. The metric found outperformed classical metabolic prognostic variables used in nuclear medicine.
In conclusion, mathematical models with evolutionary dynamics suggests how to construct different metabolic imaging biomarkers with a strong prognostic value and thus clinical utility for different tumor histologies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ONCO-MS20.html#author3"><strong class="card-body-subgroup">Jana Lipkova</strong></a>
		    	 
		    		<em> (Brigham and Women’s Hospital, Harvard Medical School, US)</em>
		    	
		    	
		    		
		    			</br>"Personalized Radiotherapy Design for Glioblastoma:Integrating Mathematical Tumor Models,Multimodal Scans and Bayesian Inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma is a highly invasive brain tumor, whose cells infiltrate surrounding normal brain tissue beyond the lesion outlines visible in the current medical scans. These infiltrative cells are treated mainly by radiotherapy. Existing radiotherapy plans for brain tumors derive from population studies and scarcely account for patient-specific conditions. Here we provide a Bayesian machine learning framework for the rational design of improved, personalized radiotherapy plans using mathematical modeling and patient multimodal medical scans. Our method, for the first time, integrates complementary information from high resolution MRI scans and highly specific FET-PET metabolic maps to infer patient-specific tumor cell density, which in turn allow design of personalized radiotherapy plans. Initial clinical study showed that the proposed treatment plans spare more healthy tissue, this reducing radiation toxicity while yielding comparable accuracy with standard radiotherapy protocols. Moreover, the inferred regions of high tumor cell densities coincide with the tumor radioresistant areas, providing guidance for personalized dose-escalation. The proposed integration of multimodal scans and mathematical modeling provides a robust, non-invasive tool to assist personalized radiotherapy design.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS20/ONCO-MS20.html#author4"><strong class="card-body-subgroup">Guillermo Lorenzo</strong></a>
		    	 
		    		<em> (University of Pavia, Italy)</em>
		    	
		    	
		    		
		    			</br>"Personalized image-based modeling of organ-confined prostate cancer: exploring the mechanical interactions between tumor growth and coexisting benign prostatic hyperplasia"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prostate cancer (PCa) is a public health burden and a major concern among ageing men worldwide, with high rates of incidence and mortality. Thanks to regular screening and risk-group triaging most patients are currently diagnosed and successfully treated when the tumor is in early stage and confined within the prostate. Benign prostatic hyperplasia (BPH) is another common pathology in ageing men that causes the prostate to gradually enlarge over time, which may produce bothersome lower urinary tract symptoms. PCa originating in men with larger prostates tend to present more favorable pathological features, but the fundamental mechanisms that explain this interaction between BPH and prostate cancer are largely unknown. Here, we propose a mechanical explanation for this phenomenon: the mechanical stress fields that originate as tumors grow are known to slow down their dynamics, and BPH contributes to these mechanical stress fields, hence further restraining PCa growth. To explore this hypothesis, we run a qualitative simulation study using a mechanically-coupled mathematical model of PCa growth. We run our study leveraging a patient-specific geometric model of the prostate and tumor extracted from magnetic resonance imaging data. Our simulations show that the mechanical stress fields accumulated in the prostate by BPH over time impede prostatic tumor growth and limit its invasiveness. We further explore the effect on tumor growth of a type of BPH drugs that are being investigated for the chemoprevention of PCa: 5-alpha reductase inhibitors (e.g., finasteride, dutasteride), which reduce the size of the prostate (thereby treating BPH symptoms) and might promote apoptosis in the tumor. Depending on the intensity of these two mechanisms, our simulations show different tumor growth dynamics ranging from long-term inhibition of PCa growth to rapidly growing large tumors, which may evolve towards advanced disease. The latter case may provide a mechanistic explanation for the controversial advanced PCa cases found in chemoprevention trials of these drugs. In the future, we think that our computational technology can contribute to further investigate the biophysical mechanisms underlying PCa and BPH, and ultimately assist physicians in the clinical management of these diseases by forecasting pathological and therapeutic outcomes on an organ-scale, patient-specific basis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>

<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-20
										</br>
										Thursday, June 17 at 11:30am (PDT)</br>Thursday, June 17 at 07:30pm (BST)</br>Friday, June 18 03:30am (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>













    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
