<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Combining modeling and inference in cell biology | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Combining modeling and inference in cell biology | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Combining modeling and inference in cell biology" />
<meta name="author" content="esmb" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="In many cell and developmental processes, both modeling and data analytic approaches are necessary in order to generate useful modeling predictions to guide the design of further experiments for both validating and improving biological insight. There is an increased understanding that the application of machine learning methods can also be used to enhance common data-driven modeling techniques, including parameter and equation inference, classification, and sensitivity analysis. The speakers in this session will discuss how continuous differential equation models, individual-based stochastic models, and methods from machine learning can be used to address questions related to mitosis, intracellular transport, cell migration, and tissue development. The speakers will highlight current research progress and challenges associated with combining modeling and inference approaches in cell and developmental biology." />
<meta property="og:description" content="In many cell and developmental processes, both modeling and data analytic approaches are necessary in order to generate useful modeling predictions to guide the design of further experiments for both validating and improving biological insight. There is an increased understanding that the application of machine learning methods can also be used to enhance common data-driven modeling techniques, including parameter and equation inference, classification, and sensitivity analysis. The speakers in this session will discuss how continuous differential equation models, individual-based stochastic models, and methods from machine learning can be used to address questions related to mitosis, intracellular transport, cell migration, and tissue development. The speakers will highlight current research progress and challenges associated with combining modeling and inference approaches in cell and developmental biology." />
<link rel="canonical" href="http://localhost:4000/CDEV/CDEV-MS13.html" />
<meta property="og:url" content="http://localhost:4000/CDEV/CDEV-MS13.html" />
<meta property="og:site_name" content="SMB 2021" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2021-06-17T15:54:54-04:00" />
<script type="application/ld+json">
{"description":"In many cell and developmental processes, both modeling and data analytic approaches are necessary in order to generate useful modeling predictions to guide the design of further experiments for both validating and improving biological insight. There is an increased understanding that the application of machine learning methods can also be used to enhance common data-driven modeling techniques, including parameter and equation inference, classification, and sensitivity analysis. The speakers in this session will discuss how continuous differential equation models, individual-based stochastic models, and methods from machine learning can be used to address questions related to mitosis, intracellular transport, cell migration, and tissue development. The speakers will highlight current research progress and challenges associated with combining modeling and inference approaches in cell and developmental biology.","author":{"@type":"Person","name":"esmb"},"@type":"BlogPosting","url":"http://localhost:4000/CDEV/CDEV-MS13.html","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"},"name":"esmb"},"headline":"Combining modeling and inference in cell biology","dateModified":"2021-06-17T15:54:54-04:00","datePublished":"2021-06-17T15:54:54-04:00","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/CDEV/CDEV-MS13.html"},"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Combining modeling and inference in cell biology">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Combining modeling and inference in cell biology" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Combining modeling and inference in cell biology" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        
<div class="container">
<div class="jumbotron jumbotron-fluid mb-3 pl-0 pt-0 pb-0 bg-white position-relative">
		<div class="h-100 tofront">
			<div class="row  justify-content-center ">
				<div class=" col-md-8  pr-0 pr-md-4 pt-4 pb-4 align-self-center">
					<p class="text-uppercase font-weight-bold">
              <span class="catlist">
              	<span class="text-CDEV">CDEV-MS13</span>						
              </span>
					</p>
					<h1 class="display-4 mb-4 article-headline">Combining modeling and inference in cell biology</h1>
					<h4>Wednesday, June 16 at 09:30am (PDT)</br>Wednesday, June 16 at 05:30pm (BST)</br>Thursday, June 17 01:30am (KST)</h4>
					<div class="d-flex align-items-center">
                        
                        <img class="rounded-circle" src="../assets/images/logo.jpg" alt="SMB2021" width="70"/>
                        
						<small class="ml-3"> SMB2021 <span><a target="_blank" href="https://twitter.com/smb2021" class="btn btn-outline-success btn-sm btn-round ml-1">Follow</a></span>
              <span class="text-muted d-block mt-1">Wednesday (Thursday) during the "MS13" time block.</span>
						</small>
						
					</div>
				</div>
				
					<small class="ml-3">Note: this minisymposia has multiple sessions. The second session is <strong>MS14-CDEV</strong> <a href="../CDEV/CDEV-MS14.html">(click here)</a>. 
						
						
						
					</small></br></br>
				
        
			</div>
		</div>
	</div>
</div>





<div class="container-lg pt-4 pb-4">
	<div class="row justify-content-center">
        
        <!-- Share -->
				<div class="col-lg-2 pr-4 mb-4 col-md-12">
					<div class="sticky-top sticky-top-offset text-center">
						<div class="text-muted">
							Share this
						</div>
						<div class="share d-inline-block">
							<!-- AddToAny BEGIN -->
							<div class="a2a_kit a2a_kit_size_32 a2a_default_style">
								<a class="a2a_dd" href="https://www.addtoany.com/share"></a>
								<a class="a2a_button_facebook"></a>
								<a class="a2a_button_twitter"></a>
							</div>
							<script async src="https://static.addtoany.com/menu/page.js"></script>
							<!-- AddToAny END -->
						</div>
					</div>
				</div>
        
        
		<div class="col-md-12 col-lg-10">
			
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem CDEV position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <p><strong>Organizers:</strong></p>
		                <h4 class="mb-3">Maria-Veronica Ciocanel (Duke University, United States), John Nardini (North Carolina State University, United States)</h4>
		                
		                
		                	<p><strong>Description</strong>:</p> 
		                	<p>In many cell and developmental processes, both modeling and data analytic approaches are necessary in order to generate useful modeling predictions to guide the design of further experiments for both validating and improving biological insight. There is an increased understanding that the application of machine learning methods can also be used to enhance common data-driven modeling techniques, including parameter and equation inference, classification, and sensitivity analysis. The speakers in this session will discuss how continuous differential equation models, individual-based stochastic models, and methods from machine learning can be used to address questions related to mitosis, intracellular transport, cell migration, and tissue development. The speakers will highlight current research progress and challenges associated with combining modeling and inference approaches in cell and developmental biology.</p>
			              
			              
			              
		            </div>             
		        </div>
		    </div>
			</div>
			</br>
			</br>
			
			
			
			
			
						
			
			
			
			
			
	    	<h4 id="author1">Alexandria Volkening</h4>
	    	 
	    		<em> (Northwestern University, United States)</em>
	    	
	    	
	    	
	    		
	    			<h6>"Topological methods for quantitatively describing cell-based patterns"</h6>
	    		
	    	
	    	
	    	
	    	
	    		<article class="article-post text-justify" style="font-size:12px;">
		    		
		    			Self-organization is present at many scales in biology, and here I will focus specifically on elucidating how brightly colored cells interact to form skin patterns in zebrafish. Wild-type zebrafish are named for their dark and light stripes, but mutant zebrafish feature variable skin patterns, including spots and labyrinth curves. All of these patterns form as the fish grow due to the interactions of tens of thousands of pigment cells, making agent-based modeling a natural approach for describing pattern formation. By identifying cell interactions that may change to create mutant patterns, the longterm motivation for my work is to help link genes, cell behavior, and visible animal characteristics in fish. However, agent-based models are stochastic and have many parameters, so comparing simulated patterns and fish images is often a qualitative process. Developing analytically tractable continuum models from agent-based systems is one means of addressing these challenges and better understanding the roles of different parameters in pattern formation. Alternatively, methods from topological data analysis can be applied to cell-based systems directly. In this talk, I will overview our models and present quantitative comparisons of in silico and in vivo cell-based patterns using our topological methods.
		    		
	    		</article>
	    	
				
	    
	    
	    </br>
			</br>
	    
	    
	    	<h4 id="author2">Fiona Macfarlane</h4>
	    	 
	    		<em> (University of Saint Andrews, United Kingdom)</em>
	    	
	    	
	    		
	    			<h6>"A hybrid discrete-continuum approach to model Turing pattern formation"</h6>
	    		
	    	
	    	
	    	
	    		<article class="article-post text-justify" style="font-size:12px;">
		    		
		    			We have developed a hybrid discrete-continuum modelling framework to investigate the formation of cellular patterns through the Turing mechanism. In this framework, a stochastic individual-based model of cell migration and proliferation is combined with a reaction-diffusion system for the concentrations of some interacting chemical species. As an illustrative example, we consider a model in which the dynamics of the morphogens are governed by an activator-inhibitor system that gives rise to Turing pre-patterns. The cells then interact with the morphogens in their local area through either of two forms of chemically-dependent cell action: Chemotaxis or chemically-controlled proliferation. We consider both the case of static spatial domains and additionally investigate the case of growing domains. In all cases we are able to derive the corresponding deterministic continuum limits, inferring an appropriate system of PDEs to model the dynamics of the hybrid model. We investigate parameter situations in which the numerical simulations of the PDE models give an accurate description of the hybrid models, and cases where they do not qualitatively match the hybrid models. This framework is intended to present a proof of concept for the ideas underlying the models, with the aim to then apply the related methods to the study of specific patterning and morphogenetic processes in the future.
		    		
	    		</article>
	    	
	    
	    
	    </br>
			</br>
	    
	    
	    	<h4 id="author3">Suzanne Sindi</h4>
	    	 
	    		<em> (University of California Merced, United States)</em>
	    	
	    	
	    		
	    			<h6>"Multi-Scale Modeling and Parameter Inference in Yeast Protein Aggregation"</h6>
	    		
	    	
	    	
	    		<article class="article-post text-justify" style="font-size:12px;">
		    		
		    			Unlike a disease caused by a virus or a bacteria, in prion diseases the infectious agent is created by the host organism itself. Prion proteins are responsible for a variety of neurodegenerative diseases in mammals such as Creutzfeldt-Jakob disease in humans and “mad-cow disease” (Bovine Spongiform Encephalopathy or BSE) in cattle. While these diseases are fatal to mammals, prions are harmful to yeast, making yeast an ideal model organism for prion diseases. Most mathematical approaches to modeling prion dynamics have focused on either the protein dynamics in isolation, absent from a changing cellular environment, or modeling prion dynamics in a population of cells by considering the “average” behavior. However, such models have been unable to recapitulate in vivo properties of yeast prion strains. My group develops physiologically relevant mathematical models by considering both the prion aggregates (which evolve inside individual yeast cells) and the yeast cells (which grow and divide). In this talk, I will present a stochastic biochemical reaction system for protein aggregation and demonstrate that the standard computational assumption - fixed protein monomer mass - leads to incorrect biological conclusions. We relax the mass conservation restriction through the use of an additional “slack” species and discover new regimes of biologically relevant behavior. These regimes necessarily correspond to the biologically feasible regions of parameter space for prion aggregation.
		    		
	    		</article>
	    	
	    
	    
			</br>
			</br>
	    
	    
	    	<h4 id="author4">Adam MacLean</h4>
	    	 
	    		<em> (University of Southern California, United States)</em>
	    	
	    	
	    		
	    			<h6>"Bayesian inference of Calcium signaling dynamic provides a map from single-cell gene expression to cellular phenotypes"</h6>
	    		
	    	
	    	
	    		<article class="article-post text-justify" style="font-size:12px;">
		    		
		    			Since single-cell RNA sequencing technologies have become widespread, great efforts have been made to develop appropriate computational methods to learn biological features from high dimensional datasets. Much less effort has gone into the important yet challenging task of learning about dynamic processes from genomic data. Here we employ spatial transcriptomic data (MERFISH) linked to dynamic Ca2+ responses in single cells for parameter inference. We quantify cell-cell similarity -- learnt via nonnegative matrix factorization of transcriptomic signatures -- and use it to define informative cell-specific priors. We show that these informative priors dramatically speed up Bayesian parameter inference for an ODE model of Ca2+ dynamics. Analysis of posterior parameter distributions across hundreds of single cells allows us to identify genes driving phenotypic changes and link these genes to specific Calcium pathway parameters that are sensitive to outputs. Finally, we test our ability to predict Ca2+ responses using only the cell-cell similarity. This allows us to quantify the amount of information on a dynamic cell phenotype that is contained in the gene expression data alone.
		    		
	    		</article>
	    	
	    
	    
	    </br>
			</br>
	    
	    
	    
	    </br>
			</br>
	    
	    
	    
	      
			<!-- Author Box -->
      				
			<div class="row mt-5">
				<div class="col-md-2 align-self-center">
                       
                      <img class="rounded-circle" src="../assets/images/logo.jpg" alt="SMB2021" width="90"/>
                       
				</div>
				<div class="col-md-10">		
                      <h5 class="font-weight-bold">Hosted by SMB2021 <span><a target="_blank" href="https://twitter.com/smb2021" class="btn btn-outline-success btn-sm btn-round ml-2">Follow</a></span></h5>
					Virtual conference of the Society for Mathematical Biology, 2021.					
				</div>
			</div>				
      
      
      <!-- Comments -->
      
          <!--  Don't edit anything here. Set your disqus id in _config.yml -->

<div id="comments" class="mt-5">
    <div id="disqus_thread">
    </div>
    <script type="text/javascript">
        var disqus_shortname = ''; 
        var disqus_developer = 0;
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = window.location.protocol + '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>
    Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a>
    </noscript>
</div>
      
            
		</div>
        
        
	</div>
</div>



    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
