<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Cell and Developmental Biology Subgroup (CDEV) | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Cell and Developmental Biology Subgroup (CDEV) | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Cell and Developmental Biology Subgroup (CDEV)" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/CDEV/" />
<meta property="og:url" content="http://localhost:4000/CDEV/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/CDEV/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Cell and Developmental Biology Subgroup (CDEV)","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Cell and Developmental Biology Subgroup (CDEV)">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Cell and Developmental Biology Subgroup (CDEV)" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Cell and Developmental Biology Subgroup (CDEV)" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
	    <div class="row">
	        <div class="col-md-12  text-center">
<!-- 		        <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
		        <h1 class="mb-3">Cell and Developmental Biology Subgroup (CDEV)</h1>
		        
<!-- 		        <div class="cleanslate w24tz-current-time w24tz-small" style="display: inline-block !important; visibility: hidden !important; min-width:200px !important; min-height:100px !important;"><p><a href="//24timezones.com/Tampa/time" style="text-decoration: none" class="clock24" id="tz24-1596739689-c1410-eyJob3VydHlwZSI6MTIsInNob3dkYXRlIjoiMSIsInNob3dzZWNvbmRzIjoiMCIsImNvbnRhaW5lcl9pZCI6ImNsb2NrX2Jsb2NrX2NiNWYyYzUwNjllNmUzMSIsInR5cGUiOiJkYiIsImxhbmciOiJlbiJ9" title="Tampa local time" target="_blank">US Eastern Time Zone (EDT)</a></p><div id="clock_block_cb5f2c5069e6e31"></div></div>
<script type="text/javascript" src="//w.24timezones.com/l.js" async></script>
	        	</br> -->
		        
		        
	        </div>
	    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			<h6>Jump to:</h6>
<ul class="pagination text-left">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#"><span class="text-CDEV">Minisymposia</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#CT"><span class="text-CDEV">Contributed Talks</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#Posters"><span class="text-CDEV">Posters</span></a></li>
</ul>


			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS1">Sub-group minisymposia</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS01.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS01-CDEV: </small></br>Data-driven modeling across scales - from cytoskeleton to bacterial swarms to multicellular motility to angiogenesis</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Alex Mogilner (NYU, United States), Angelika Manhart (UCL, UK)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS01.html#author1"><strong class="card-body-subgroup">Angelika Manhart</strong></a>
		    	 
		    		<em> (UCL, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Explaining the dynamic steady state of actin networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many motile cells use dense, branched actin networks for movement. This requires “macroscopic treadmilling”, where assembly at the front balances disassembly at the rear. We combine the use of a biomimetic motility system with data-driven mathematical modeling to investigate how cofilin, a known disassembly agent, creates dynamic networks of fixed lengths. To capture the observed macroscopic fragmentation of the network, we combine PDE-based modeling of the cofilin binding dynamics with a discrete network disassembly model. This allows to derive a simple formula predicting the equilibrium network length across various control parameters.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS01.html#author2"><strong class="card-body-subgroup">Hannah Jeckel</strong></a>
		    	 
		    		<em> (University of Marburg, Germany)</em>
		    	
		    	
		    		
		    			</br>"Learning the space-time phase diagram of bacterial swarm expansion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Coordinated dynamics of individual components in active matter are an essential aspect of life on all scales. Establishing a comprehensive, causal connection between intracellular, intercellular, and macroscopic behaviors has remained a major challenge due to limitations in data acquisition and analysis techniques suitable for multiscale dynamics. Here, we combine a high-through-put adaptive microscopy approach with machine learning, to identify key biological and physical mechanisms that determine distinct microscopic and macroscopic collective behavior phases which develop as Bacillus subtilis swarms expand over five orders of magnitude in space. Our experiments, continuum modeling, and particle-based simulations reveal that macroscopic swarm expansion is primarily driven by cellular growth kinetics, whereas the microscopic swarming motility phases are dominated by physical cell–cell interactions. These results provide a unified understanding of bacterial multiscale behavioral complexity in swarms.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS01.html#author3"><strong class="card-body-subgroup">Dhananjay Bhaskar</strong></a>
		    	 
		    		<em> (Yale and Brown Universities, USA)</em>
		    	
		    	
		    		
		    			</br>"Discrete Agent Modeling and Topological Data Analysis of Self-Organized Multicellular Architectures"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Animal tissues are spatially patterned into complex topologies via directed motility and cell-cell interactions during development, repair, and disease. Segregation and cell sorting, driven by differential adhesion and interfacial tension, generate complex yet stable configurations that underlie tissue morphogenesis. Moreover, alterations of cell-cell adhesion and polarity in epithelial-mesenchymal transition (EMT) are associated with tumor dissemination and metastasis. In this talk, I will describe the use of topological data analysis for the automated classification of multicellular structures associated with EMT and cell sorting.
First, individual and collective phases of epithelial migration are characterized by varying adhesion and random propulsion parameters in an agent-based model derived from experimental observations. Next, persistent homology is computed using cell positions as input, followed by unsupervised classification of the topological features (connected components and loops). Finally, classification results are mapped onto the adhesion-propulsion phase diagram for automatic delineation of phase boundaries. A similar methodology, applied to co-culture simulations with varying adhesion parameters, reveals phase transitions between various patterns of self-assembly in cell sorting. I envision this computational approach will enable new quantitative insights into the emergence of complex tissue topologies via spatiotemporal interactions between one or more cell types.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS01.html#author4"><strong class="card-body-subgroup">John Nardini</strong></a>
		    	 
		    		<em> (NC State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Topology discriminates parameter regimes in a model of angiogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Angiogenesis is the process by which blood vessels form from pre-existing vessels. It plays a key role in many biological processes, including embryonic development and wound healing, and contributes to many diseases including cancer and rheumatoid arthritis. The structure of the resulting vessel networks determines their ability to deliver nutrients and remove waste products from biological tissues. Here we simulate the Anderson-Chaplain model of angiogenesis at different parameter values and quantify the vessel architectures of the resulting synthetic data. Specifically, we propose a topological data analysis (TDA) pipeline for systematic analysis of the model. TDA is a vibrant and relatively new field of computational mathematics for studying the shape of data. We compute topological and standard descriptors of model simulations generated by different parameter values. We show that TDA of model simulation data stratifies parameter space into regions with similar vessel morphology. The methodologies proposed here are widely applicable to other synthetic and experimental data including wound healing, development, and plant biology.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS02.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS02-CDEV: </small></br>Multiscale modeling in tissue growth and morphogenesis to understand biological data</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Weitao Chen (University of California, Riverside, United States), Qixuan Wang (University of California, Riverside, United States)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS02.html#author1"><strong class="card-body-subgroup">Dagmar Iber</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"From Networks to Function – Computational Models of Morphogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			One of the major challenges in biology concerns the integration of data across length and time scales into a consistent framework: how do macroscopic properties and functionalities arise from the molecular regulatory networks and how do they evolve? Morphogenesis provides an excellent model system to study how simple molecular networks robustly control complex pattern forming processes and how mechanical constraints shape organs. In my talk, I will focus on self-organizing principles in organogenesis, with a particular focus on lung and kidney development, as well as on epithelial organisation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS02.html#author2"><strong class="card-body-subgroup">Zhan Chen</strong></a>
		    	 
		    		<em> (Georgia Southern University, United States)</em>
		    	
		    	
		    		
		    			</br>"Anterior-Posterior patterning and scaling of Drosophila wing disc: Mathematical modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Wing imaginal disc of Drosophila is one of the commonly used model systems for the studies of patterning, growth, and scaling. The development of the wing disc involves many interacting components as well as a variety of compound processes whose underlying mechanisms are still under investigation. For instance, it remains unclear about how to form compound experimentally-measured patterns of Decapentaplegic (Dpp) type-I receptor Thickveins (Tkv), as well as phosphorylated Mothers Against Dpp (pMad) which is the indicator of Dpp signaling activities. In this work, we proposed mathematical models that integrate established experimental data to investigate the formation of pMad and Tkv gradients. Our model is validated by the accurate reproduction of complex asymmetric profiles of Tkv and pMad in both anterior and posterior compartments of the wing disc.  Moreover, it provides a comprehensive view of the formation of Tkv gradients in wing discs. We found that engrailed (En), Hedgehog (Hh) signaling and Dpp signaling cooperate to establish the asymmetric gradients of  Tkv and pMad in the wing disc. Finally, our model suggests a Brinker-mediated mechanism of Dpp-dependent repression of Tkv.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS02.html#author3"><strong class="card-body-subgroup">John Dallon</strong></a>
		    	 
		    		<em> (Brigham Young University, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling collagen tissue: How structure affects mechanical properties"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The fibrous protein collagen is the main protein in mammalian connective tissue.  Although the properties of a collagen filament are well understood, how they come together to form tissues with vastly different properties is not.  In this talk, with the aid of a mathematical model, we will explore properties of a fibrous tissue and determine their impact on the elasticity of the tissue.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS03.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS03-CDEV: </small></br>Diverse quantitative approaches integrating data and modelling in development and medicine</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adriana Dawes (Ohio State University, USA), Sungrim Seirin-Lee (Hiroshima University, Japan)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS04-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS03.html#author1"><strong class="card-body-subgroup">Sungrim Seirin-Lee</strong></a>
		    	 
		    		<em> (Hiroshima University, Japan)</em>
		    	
		    	
		    		
		    			</br>"A one-line mathematical model that solved the mystery of urticaria"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Urticaria is a common skin disease characterized by the rapid appearance and disappearance of local skin edema and flares with itching. It affects about one in 5 people at some point in their lives and globally about 56/100000 population suffer from urticaria daily. It is characterized by various macroscopic skin eruptions unique to patients with respect to shape, size, and/or duration of eruptions. Nevertheless, the mechanism underlying multifarious eruptions in urticaria is largely unknown in medicine. The eruptions are believed to be evoked by histamine release from mast cells in the skin. However, the majority of visible characteristics of urticaria cannot be explained by a simple injection of histamine to the skin. In this study, we succeeded in developing a mathematical model that can explain various geometrical shapes of eruptions typically observed in patients. Our mathematical model suggests that simultaneous self-regulation of positive and negative feedback of histamine through mast cells plays a critical role in generating the wide-spread wheal patterns. The study findings increase the understanding of the pathogenesis of urticaria and may aid decision-making for appropriate treatments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS03.html#author2"><strong class="card-body-subgroup">Yoichiro Mori</strong></a>
		    	 
		    		<em> (University of Pennsylvania, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical Justification of Slender Body Theory"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Systems in which thin filaments interact with the surrounding fluid abound in science and engineering. The computational and analytical difficulties associated with treating thin filaments as 3D objects has led to the development of slender body theory, in which filaments are approximated as 1D curves in a 3D fluid. In the 70-80s, Keller, Rubinow, Johnson and others derived an expression for the Stokesian flow field around a thin filament given a one-dimensional force density along the center-line curve. Through the work of Shelley, Tornberg and others, this slender body approximation has become firmly established as an important computational tool for the study of filament dynamics in Stokes flow. An issue with slender body approximation has been that it is unclear what it is an approximation to. As is well-known, it is not possible to specify some value along a 1D curve to solve the 3D exterior Stokes problem. What is the PDE problem that slender body approximation is approximating? Here, we answer this question by formulating a physically natural PDE problem with non-conventional boundary conditions on the filament surface, which incorporates the idea that the filament must maintain its integrity (velocity along filament cross sections must be constant). We prove that this PDE problem is well-posed, and show furthermore that the slender body approximation does indeed provide an approximation to this PDE problem by proving error estimates. This is joint work with Laurel Ohm, Will Mitchell and Dan Spirn.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS03.html#author3"><strong class="card-body-subgroup">Benjamin Walker</strong></a>
		    	 
		    		<em> (University of Oxford, UK)</em>
		    	
		    	
		    		
		    			</br>"Hypothesis generation and hypothesis testing in spermatozoa"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Spermatozoa are perhaps the canonical microscopic swimmer, propelled along the path to fertilisation via the wavelike motion of a long slender flagellum. Owing not least to their key role in fertility, they have long been the subject of significant study, driving both experimental and theoretical developments. In this talk, I hope to survey a number of recent advances in the way in which we are able to study and investigate the microscale world of sperm, with applications beyond these cellular swimmers. These new methodologies promise to enable the next generation of quantitative analysis of flagellated swimmers, with the potential to both enhance clinical diagnostics in the future and investigate fundamental and widely conserved cellular biology. In particular, I will begin by recounting recent step changes in data acquisition, with fully automated schemes now replacing tiresome by-hand analysis. Further, I will then highlight how these developments can be coupled to population-level statistical analyses that incorporate the fine details of the flagellar beat, which have classically been absent from quantitative study. Finally, I will touch upon another exciting area of rapid development with broad applicability, that of flagellar simulation, which is enabling sophisticated data-driven modelling and hypothesis generation in spermatozoa, in addition to newly realising exploratory in silico study of these complex microscale organisms.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS03.html#author4"><strong class="card-body-subgroup">Kang-Ling Liao</strong></a>
		    	 
		    		<em> (University of Manitoba, Canada)</em>
		    	
		    	
		    		
		    			</br>"The role of CD200-CD200R in cancer immunotherapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			CD200 is a cell membrane protein that interacts with CD200 receptor (CD200R) of myeloid lineage cells. CD200-positive tumor cells can interact with M1 and M2 macrophages through CD200–CD200R-compex and downregulate IL-10 and IL-12 productions secreted primarily by M2 and M1 macrophages, respectively. In this talk, I will introduce a PDEs model to determine the combined effect of CD200–CD200R interaction on tumor proliferation. We demonstrate that blocking CD200 on tumor cells may have opposite effects on tumor proliferation depending on the “affinity” of the macrophages to form the CD200–CD200R-complex with tumor cells. We also extend these results to an ODEs model to study how the populations of M1 and M2 macrophages affect the tumor growth.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS04.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS04-CDEV: </small></br>Diverse quantitative approaches integrating data and modelling in development and medicine</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adriana Dawes (Ohio State University, USA), Sungrim Seirin-Lee (Hiroshima University, Japan)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS03-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS04.html#author1"><strong class="card-body-subgroup">Adriana Dawes</strong></a>
		    	 
		    		<em> (Ohio State University, USA)</em>
		    	
		    	
		    		
		    			</br>"The causes and consequences of centrosome asymmetry during development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Asymmetric cell division, where daughter cells inherit unequal amounts of specific factors, is critical for development and cell fate specification, and is implicated in disease processes such as tumour growth. In polarized cells, where specific factors are segregated to opposite ends of the cell as seen in early embryos of the nematode worm C. elegans, asymmetric cell division occurs as a result of dynein-mediated centrosome positioning along the polarity axis. Using a combination of stochastic and continuum models with experimental validation, we show that centrosome asymmetry is critical for centrosome positioning in the early C. elegans embryo, and that this asymmetry arises from differential recruitment of proteins to centrosomes during their maturation process.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS04.html#author2"><strong class="card-body-subgroup">Susanne Rafelski</strong></a>
		    	 
		    		<em> (Allen Institute for Cell Science, USA)</em>
		    	
		    	
		    		
		    			</br>"Decoding the variance in intracellular organization of human stem cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Allen Institute for Cell Science aims to understand the principles by which human induced pluripotent stem cells (hiPSCs) establish and maintain robust dynamic localization of subcellular structures. Initial steps aim to determine the full range of natural variation in intracellular organization in hiPSCs under normal, unperturbed conditions. We used 25 of the endogenous fluorescently tagged hiPSC lines in the Allen Cell Collection (www.allencell.org), each expressing a monoallelic EGFP-tagged protein labeling a particular organelle or structure. We imaged thousands of cells at high resolution in 3D for each structure and developed segmentation algorithms and workflows for quantitative analyses. To measure variation in cell and nuclear shapes, we fit 3D segmented masks using spherical harmonic functions, and then performed a principal component analysis of the spherical harmonic coefficients. We found that the largest axes of shape variation corresponds to 1) cell height, due to differences in cell packing colonies, and 2) cell volume, representing cell growth through the cell cycle. We performed a survey analysis of size scaling and found that structures differ in the strength of their scaling with cell size and nuclear size. To explore variation in intracellular organization, we parameterized the cytoplasm and nucleoplasm via spherical harmonics to generate maps for each structure in each cell in 3D. Analysis of these maps allowed us to quantify and rank how stereotyped the locations for each of these structures are and further to cluster structures via correlations in their location relative to each other over different spatial scales. This systematic approach has enabled us to quantify how subcellular organelle organization varies with changes in cell shape in an integrated fashion across 25 EGFP-tagged subcellular structures.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS04.html#author3"><strong class="card-body-subgroup">Naoki Honda</strong></a>
		    	 
		    		<em> (honda.naoki.4v@kyoto-u.ac.jp, Japan)</em>
		    	
		    	
		    		
		    			</br>"Data-driven hierarchical modeling of collective cell migration"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Collective cell migration is a fundamental process of development. It has been known that cell migration within an epithelial sheet is oriented by traveling waves of ERK activation. However, its mechanism has remained elusive. To extract control roles in the epithelial sheet dynamics, we first developed mathematical models at the different hierarchical levels of individual cells and continuum, which can be seamlessly linked. Based on this hierarchical modeling, we mathematically predicted that migration velocity is directed by several mechano-chemical signals: cellular density, ERK activity, velocity field and their temporal and/or spatial derivatives. To test this model prediction, we live-imaged ERK activity during the collective cell migration with FRET-based biosensor. From the live imaging data, we quantified the time-series data corresponding to variables in the model. We then analyzed the time-series data with help of machine learning and then obtained a reverse-engineered model, which describes how the cells intracellularly process these mechano-chemical signals. We also confirmed that this model has an ability to forecast cell migration, hence showing validity of the model. By interpreting the reverse-engineered continuum model at the individual cellular level, we elucidated intercellular mechanical interaction is up-regulated by temporal derivative of ERK signal. Therefore, our data-driven hierarchical modeling approach is powerful to understand multicellular dynamics.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS04.html#author4"><strong class="card-body-subgroup">Hiroshi Suito</strong></a>
		    	 
		    		<em> (Tohoku University, Japan)</em>
		    	
		    	
		    		
		    			</br>"Patient-specific approaches to cardiovascular diseases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In blood vessels with congenital heart diseases, characteristic flow structures are formed, in which pulsating flows affect strongly on wall shear stresses and energy dissipation patterns. In this talk, we present computational analyses for blood flows in patient-specific cases, through which we aim at understanding the relationships between differences in geometries and in energy dissipations. Our present targets include an aortic coarctation case and a Norwood surgery for hypoplastic left heart syndrome. These analyses yield deeper understandings in clinical medicine.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS04" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS06.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS06-CDEV: </small></br>Computational models of extracellular matrix effects on cell migration and tissue formation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Magdalena Stolarska (University of St. Thomas, United States), Lisanne Rens (Delft University of Technology, Netherlands)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS06.html#author1"><strong class="card-body-subgroup">Leonie van Steijn</strong></a>
		    	 
		    		<em> (Leiden University, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Obstacle-induced contact-inihibition of locomotion explains topotactic cell navigation in dense microenvironments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During biological development, cancer metastasis and in the immune system, cells navigate through dense environments filled with obstacles such as other cells and the extracellular matrix. Recently, the term `topotaxis' has been introduced for the navigation of cells along topographic cues such as density gradients of obstacles. As a model of amoeboid cell motility through pores in the ECM, we study the motility of Dictyostelium discoideum cells on a substrate covered with microscopic pillars. The pillars are spaced widely enough to let the cells through and there is a gradient from densely packed pillars to more widely spaced pillars. The D. discoideum cells perform a random walk with a bias towards the more widely spaced area. A previous model based on active Brownian particles (ABP) has shown that ABPs perform topotaxis in a persistence-driven manner. However, the predicted drift is lower than measured experimentally. Here, we use a Cellular Potts model to how cell persistence mode affects topotaxis using the actin-derived persistence of the Act-model [1] and an active Brownian particle-based persistence [2]. Both persistence modes predict topotaxis, but the actin-based persistent cells show a more efficient drift.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS06.html#author2"><strong class="card-body-subgroup">Lisanne Rens</strong></a>
		    	 
		    		<em> (Delft University of Technology, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Computational models for feedback between cell shape, cell signaling and extracellular matrix"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell shape changes and cell migration in mammalian cells are regulated by many sig- naling proteins within the cell. Cells also interact with a meshwork of protein fibers, called the extracellular matrix (ECM), that affects signaling proteins that regulate cell motility, Rac and Rho. The feedback between Rac-Rho-ECM affects the invasiveness of melanoma cancer cells. In our models, we expand on a previous 2-compartment model (coupled ODEs in [3] and [1]) that describes Rac-Rho mutual inhibition, self-activation, the effect of each protein on the amount of contact with the ECM, and ECM activation of Rho [4]. We explore the effects of slip and catch-bond dynamics [2] for the assembly of cell-ECM adhesion. We study the full spatial dynamics in 1D and in static 2D domains, demon- strating oscillations and static/dynamic waves. These results give insight into how distinct types of cell migration emerge. By simulating the set of PDEs in a fully deformable 2D cell using a Cellular Potts model, we predict how spatially distributed signaling is coupled to cell motility. Predicted cell shapes and behavior resemble experimental observations. This full 2D model reveals how ECM anisotropy, cell stiffness, and other cell parameters affect cell migration, leading to experimentally testable predictions. Our computational models suggests insights into how the invasiveness of melanoma cells is regulated.
References [1] William R Holmes, JinSeok Park, Andre Levchenko, and Leah Edelstein-Keshet. A mathematical model coupling polarity signaling to cell adhesion explains diverse cell migration patterns. PLoS computational biology, 13(5):e1005524, 2017.
[2] Elizaveta A Novikova and Cornelis Storm. Contractile fibers and catch-bond clusters: A biological force sensor? Biophys. J., 105(6):1336–1345, 2013.
[3] JinSeok Park, William R Holmes, Sung Hoon Lee, Hong-Nam Kim, Deok-Ho Kim, Moon Kyu Kwak, Chiaochun Joanne Wang, Leah Edelstein-Keshet, and Andre Levchenko. Mechanochemical feedback underlies coexistence of qualitatively distinct cell polarity patterns within diverse cell populations. Proceedings of the National Academy of Sciences, 114(28):E5750–E5759, 2017.
[4] Elisabeth G. Rens and Leah Edelstein-Keshet. Cellular tango: How extracellular matrix adhesion choreographs rac-rho signaling and cell movement, 2021.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS06.html#author3"><strong class="card-body-subgroup">Magda Stolarska</strong></a>
		    	 
		    		<em> (University of St. Thomas, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling the effects of membrane mechanics on cell-substrate interaction during spreading"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It has been well established that the mechanical stiffness of the substrate with which cells interact affects various intracellular processes, including cell spread areas, speeds at which motile cells translocate, and the number and strength of cell-substrate adhesions.   This mechanosensitivity is modulated through conformational changes in cell-substrate adhesion proteins that in turn regulate downstream processes, including those associated with the cell membrane (Kalappurakkal et al., Cell, 2019).  Membrane dynamics, including unfolding and exocytosis from intracellular reservoirs to the lipid bilayer, is necessary for large changes in cell shape, which occur during cell spreading and motility (Figard & Sokac, BioArchitecture, 2014) and for the release of membrane tension that occurs during these shape changes (Pontes et al., J Cell Bio, 2017).  The aim of this work is to understand how membrane dynamics affects the mechanics of cell spreading. To do this, we model the cell as viscous material surrounded by a viscoelastic, actively deforming membrane.  The model also incorporates stress-dependent focal adhesion dynamics and their effect on actin polymerization and myosin contractility.  By using the finite element method to simulate cell spreading in an axisymmetric geometry, we show that the membrane plays a critical role in controlling focal adhesions and in balancing protrusive activity and actin retrograde flow. This balance of protrusive activity not only recapitulate the three phases of cell spreading dynamics described in Gianonne et al. (Cell, 2004), but also plays a critical role in modulating the dependence of total amounts of adhesion proteins and cell spread areas on substrate stiffness.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS06.html#author4"><strong class="card-body-subgroup">Wanda Strychalski</strong></a>
		    	 
		    		<em> (Case Western Reserve University, United States)</em>
		    	
		    	
		    		
		    			</br>"Computational estimates of mechanical constraints on cell migration through the extracellular matrix"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell migration through a three-dimensional (3D) extracellular matrix (ECM) underlies important physiological phenomena and is based on a variety of mechanical strategies depending on the cell type and the properties of the ECM. Using computational simulations, we investigate two such migration mechanisms: 'push-pull' (forming a finger-like protrusion, adhering to an ECM node, and pulling the cell body forward) and 'rear-squeezing' (pushing the cell body through the ECM by contracting the cell cortex and ECM at the cell rear). We present a computational model that accounts for both elastic deformation and forces of the ECM, an active cell cortex and nucleus, and for hydrodynamic forces and flow of the extracellular fluid, cytoplasm, and nucleoplasm. The model is formulated using the method of regularized Stokeslets to simulate fluid-structure interactions. We find that relations between three mechanical parameters, the cortex's contractile force, nuclear elasticity, and ECM rigidity, determine the effectiveness of cell migration through the dense ECM. The cell can migrate persistently even if its cortical contraction cannot deform a near-rigid ECM, but then the contraction of the cortex has to be able to sufficiently deform the nucleus. The cell can also migrate even if it fails to deform a stiff nucleus, but then it has to be able to sufficiently deform the ECM. Simulations show the rear-squeezing mechanism of motility results in more robust migration with larger cell displacements than those with the push-pull mechanism over a range of parameter values. Additionally, results show that the rear-squeezing mechanism is aided by hydrodynamics through a pressure gradient.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS07.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS07-CDEV: </small></br>Computational models of extracellular matrix effects on cell migration and tissue formation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Magdalena Stolarska (University of St. Thomas, United States), Lisanne Rens (Delft University of Technology, Netherlands)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS06-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS07.html#author1"><strong class="card-body-subgroup">Qiyao Peng</strong></a>
		    	 
		    		<em> (Delft University of Technology, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"A cell shape evolution model for wound contraction and cancer cell metastasis using morphoelasticity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cells may attain various shapes and sizes. It has been widely documented that cell geometry influences cellular activities like cell growth and death, cell mobility and adhesion to the direct environment. The shape of a motile cell is determined by its boundaries, which dynamically vary with a local balance between retraction and protrusion. During wound healing, epidermal cells alter their shape for re-epithelialization and fibroblasts (spindle shape) differentiate into myofibroblasts (dendric shape) to regenerate collagen bundles in the extracellular matrix, while they exert traction forces causing skin contraction. For cancer cell metastasis, which is the main reason of death of cancer patients, cancer cells need to deform in order to migrate through and around obstacles during invasion and they are observed to apply traction forces on their immediate environment.
We developed a phenomenological model to simulate cell shape evolution during cell migration, based on the work in [1] and [2], where the traction forces exerted by the cells were not yet considered. Plastic deformations of the direct environment of the cells are modeled by morphoelasticity theory and point forces, which result into partial differential equations describing the momentum balance with Dirac Delta distributions for point forces over the boundary elements of the cells. The partial differential equations are solved by finite-element methods.  In our model, the cell membrane is split into line segments by nodal points, and each point is connected to the cell center by an elastic spring to maintain the cell cytoskeleton (see Figure 1). Together with chemotaxis/mechanotaxis, passive convection and random walk, the displacement of the nodal point is determined. Hence, the cell shape evolves over time during cell migration.
To validate the model, we managed to reproduce the most important trends observed in the experiment in [3]. The model can be applied to mimic various (microscopic) biological observations with several equilibrium shapes of cell, for instance, circular, elliptic and hypercloid-shaped. These equilibrium shapes are characteristic for the phenotype of the cell. Furthermore, the current model provides a basis that can be expanded to describe more experimentally observed phenomena in cell geometry. For more details about this part of work, we refer to [4].

References: [1] Chen J, Weihs D, Dijk MV, Vermolen FJ (2018) A phenomenological model for cell and nucleus deformation during cancer metastasis. Biomechanics and Modeling in Mechanobiology 17(5):1429–1450, DOI 10.1007/s10237-018-1036-5, URL https://doi.org/10.1007/s10237-018-1036-5
[2] Vermolen FJ, Gefen A (2012) A phenomenological model for chemico-mechanically induced cell shape changes during migration and cell–cell contacts. Biomechanics and Modeling in Mechanobiology 12(2):301–323, DOI 10.1007/s10237-012-0400-0, URL https://doi.org/10.1007/s10237-012-0400-0
[3]  Mak M, Reinhart-King CA, Erickson D (2013) Elucidating mechanical transition effects of invading cancer cells with a subnucleus-scaled microfluidic serial dimensional modulation device. Lab Chip 13(3):340–348, DOI 10.1039/c2lc41117b, URL https://doi.org/10.1039/c2lc41117b
[4] Peng Q, Vermolen FJ and Weihs D (2021) A Formalism for Modelling Traction forces and Cell Shape Evolution during Cell Migration in Various Biomedical Processes. Journal Biomechanics and Modeling in Mechanobiology. Online from April 2021.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS07.html#author2"><strong class="card-body-subgroup">Haryana Thomas</strong></a>
		    	 
		    		<em> (University at Buffalo, The State University of New York, United States)</em>
		    	
		    	
		    		
		    			</br>"Excess Collagen Deposition in Diabetic Kidney Disease Enhances Cellular Communication: A Mathematical Model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Diabetic kidney disease is a significant health burden in the US and worldwide. During diabetic kidney disease collagen deposition occurs in the mesangium, a tissue located at the center of the filtration unit of the kidney. The collagen deposition that occurs in the mesangium changes the transport property of the matrix, and, therefore, the ability of signaling molecules to traverse through that medium. The extent to which collagen deposition impacts the ability of glomerular cells to communicate has not been previously investigated. Using established models, we investigated whether collagen deposition impacts glomerular cell communication. We hypothesize that the pathological deposition of collagen decreases the ability of glomerular cells to communicate.
Our model predicted that collagen deposition enhances the signaling range of the mesangial cell. This enhancement can disrupt the controlled, localized inter-cellular signaling that occurs in health and thus contribute to the exacerbation of diabetic kidney damage. Previously, many models have been developed to study the parameters that impact the signaling range of cells, however, the mathematical interrogation of inter-cellular signaling in the context of diabetic kidney damage has not been previously done. The novel insight gained from this mathematical study enhances our understanding of how pathological tissue damage induced by diabetes contributes to the disruption of cellular function.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS07.html#author3"><strong class="card-body-subgroup">Robyn Shuttleworth</strong></a>
		    	 
		    		<em> (University of Saskatchewan, Canada)</em>
		    	
		    	
		    		
		    			</br>"Cell-scale degradation of peritumoural extracellular matrix fibre network and its role within tissue-scale cancer invasion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Local cancer invasion of tissue is a complex, multiscale process which plays an essential role in tumour progression. During the interaction between cancer cell population and the extracellular matrix (ECM), of key importance is the role played by both bulk two-scale dynamics of ECM fibres within collective movement of the tumour cells and the multiscale leading-edge dynamics driven by proteolytic activity of the matrix-degrading enzymes (MDEs) that are secreted by the cancer cells. We focus on understanding the cell-scale cross talk between the micro-scale parts of these two multiscale subsystems which get to interact directly in the peritumoural region, with immediate consequences both for MDE micro-dynamics occurring at the leading edge of the tumour and for the cell-scale rearrangement of the naturally oriented ECM fibres in the peritumoural region, ultimately influencing the way a tumour progresses in the surrounding tissue.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS07.html#author4"><strong class="card-body-subgroup">Katarzyna Rejniak</strong></a>
		    	 
		    		<em> (H. Lee Moffitt Cancer Center & Research Institute, United States)</em>
		    	
		    	
		    		
		    			</br>"ECM mechanical and metabolic architecture during early ductal invasions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Progression from a ductal carcinoma in situ (DCIS) to an invasive tumor is a major step initiating a devastating and often lethal metastatic cascade. One sentinel event that initiate this process is the development of ductal microinvasions, i.e., small cohorts of tumor cells that breach the basement membrane surrounding the duct and migrate through the extracellular matrix (ECM) leading to irreversible changes in tumor and stromal architecture. We used a combination of advanced image analysis techniques applied to patients’ histology data to extract features which identify specific properties of individual tumor cells inside the duct and on the invasive front. By integrating these histology-based data with a hybrid agent-based mathematical model, we investigated the biomechanical interactions between the cells and the ECM fiber architecture, and microenvironmental physical and metabolic features that define tumor niche prone to microinvasions.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS08.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS08-CDEV: </small></br>Control theory for microbiology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Robert Planqué (Vrije Universiteit Amsterdam, Netherlands), Diego Oyarzún (University of Edinburgh, United Kingdom), Mustafa Khammash (ETH Zurich, Switzerland)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS08.html#author1"><strong class="card-body-subgroup">Elisa Franco</strong></a>
		    	 
		    		<em> (University of California Los Angeles, United States)</em>
		    	
		    	
		    		
		    			</br>"Ultrasensitive feedback controllers for quasi-integral feedback"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Biological organisms regulate many of their properties so they fall in a prescribed range, for example temperature, osmotic pressure, and glucose levels. The capacity to preserve a desired condition is enabled by feedback loops that adjust gene expression or metabolism in response to changes or perturbations in the environment. Theory developed in automation engineering indicates that the best way to reject perturbations in a feedback system is to include components that integrate (maintain memory) of past effects of the disturbances, and are known as integrators. While models of biological networks such as osmoregulation and chemotaxis are known to include integral feedback, a different question is how to build molecular integral control systems from the bottom up. With mathematical modeling I will describe how ultrasensitive components can be helpful within feedback loops to maintain a desired gene expression level. I will also discuss a particular ultrasensitive reaction network that combines molecular sequestration and an activation/deactivation cycle, and could be used not only for maintaining a steady state but also for setting a tunable reference. I will finally provide an overview of ongoing projects in our group focused on the role of ultrasensitivity in the context of molecular computation and non-equilibrium kinetics.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS08.html#author2"><strong class="card-body-subgroup">Jorge I. Poveda</strong></a>
		    	 
		    		<em> (University of Colorado, Boulder, United States)</em>
		    	
		    	
		    		
		    			</br>"High-Performance Online Optimization of Bioreactor Systems via Non-Smooth and Hybrid Extremizing Feedback Controllers"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is well-known that bioreactor systems are highly nonlinear and difficult to model in a precise way by using first principles. Yet, substantial economic benefits can be achieved when the system operates at optimal points, which are difficult to calculate offline. Instead, to achieve this optimal operation in real time, different types of feedback controllers with online adaptation have been developed during the last decades. However, a persistent challenge in most existing approaches is the emergence of prohibitively slow rates of convergence and potentially small basins of attraction, which difficult the tuning of the controller in practical settings. To address these problems, in this talk we will explore a new class of extremizing control algorithms, grounded on ideas from non-smooth control and hybrid control theory, which can overcome some of the limitations of existing approaches based on smooth feedback control laws. We will illustrate our theoretical results via numerical examples in two different types of models of bioreactors.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS08.html#author3"><strong class="card-body-subgroup">Mustafa Khammash</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Beyond Perfect Adaptation: Biological Antithetic Controllers for Enhanced Transient Performance"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Proportional-Integral-Derivative (PID) feedback controllers have been the most widely used controllers in the industry for almost a century due to their simplicity and intuitive operation. Motivated by their success in various engineering disciplines, PID controllers are being explored for use in molecular biology. In this talk, we consider the mathematical realization of PID controllers using biomolecular interactions based on the antithetic controller motif. We propose a simple PID architecture based on a combination of feedback and incoherent feedforward and demonstrate its capability of enhancing the transient dynamics and reducing cell-to-cell variability.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS08.html#author4"><strong class="card-body-subgroup">Diego Oyarzún</strong></a>
		    	 
		    		<em> (University of Edinburgh, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Multiobjective optmization of metabolic control systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Progress in genetic engineering now allows the construction of molecular circuits inside living cells. In this talk I will present our approach to design such systems using multiobjective optimization. We focus on feedback control circuits designed to steer cellular metabolism toward the production of high-value chemicals. Starting from two-timescale ODE model, we pose and solve cost-benefit optimisation problems for control systems built in the literature so far. The results reveal previously unknown trade-offs between optimality, performance and robustness of metabolic control systems. Our results lay the groundwork for the automated design of control circuits in synthetic biology, with applications in the food, energy and pharma sectors.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS12.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS12-CDEV: </small></br>Synergy between experiments and modelling in understanding morphogenetic processes</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Alessandra Bonfanti (Sainsbury Laboratory University of Cambridge, United Kingdom), Alexandre Kabla (University of Cambridge, United Kingdom)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS12.html#author1"><strong class="card-body-subgroup">Shiladitya Banerjee</strong></a>
		    	 
		    		<em> (Carnegie Mellon University, USA)</em>
		    	
		    	
		    		
		    			</br>"Cell-scale modeling of epithelial morphogenesis using quantitative theory and optogenetics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During development, epithelial tissues form complex structures like organs through precise spatiotemporal coordination of cell shape changes. In vivo, many morphogenetic events are driven by pulsatile cellular contractions, which are rectified to produce irreversible tissue deformations. The functional significance of these pulsed contractions and their underlying mechanochemical circuits remain unknown. Here we develop quantitative cell-resolution models of epithelial tissues using live-cell imaging and optogenetic control of cytoskeletal force generation. We demonstrate that pulsed contraction acts as a mechanical ratchet to guide directed morphogenesis in epithelia and uncover the underlying feedback designs between cellular force generation and cell-cell adhesion. Our data and mathematical modeling provide new insights into how the localized production of cytoskeletal forces encode a fine-tuned instruction for cellular deformations that mediate epithelial morphogenesis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS12.html#author2"><strong class="card-body-subgroup">Jean-François Rupprecht</strong></a>
		    	 
		    		<em> (CNRS & Turing Centre for Living Systems Group Leader, Aix-Marseille University., France)</em>
		    	
		    	
		    		
		    			</br>"Epithelial tissues flows over hills, valleys and around potholes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    				Epithelial tissues constantly flow and renew while acting as a barrier against environmental stress and abrasion. Flows within epithelial tissues are known to be associated with cell shape changes - e.g. with shear flows contributing to cell stretching – yet, by exerting forces on their neighbours, elongated cells could in turn contribute to flows. Hydrodynamic theories incorporating such cell shape/tissue flow mechanical feedback have been proposed to explain the specific flow patterns observed within in vitro confluent epithelial tissues [1,2].
In this talk, I will present our recent results on the role of flows and cell-shape driven stresses in processes related to: (i) the loss of epithelial integrity [3]. Motivated by recent experiments revealing the spontaneous formation of holes within MDCK cell monolayers cultured on soft hydrogels, we implemented a cell-based computational framework (called vertex model) whereby cell-cell junctions can rupture. We also introduced cell-based nematic stresses which we show triggers global spontaneous flows. In both experiments and simulations, we observe the onset of specific patterns in cell shapes called topological defects. While cells at the tip of comet-like +1/2 defects were shown to be compressed and highly prone to extrusion [1], here, our simulations explain the experimental observation that holes are created in high tension regions located either at the tail of comet-like +1/2 defects or near trefoil-like -1/2 defects.  In addition, our work indicate that the progressive deformation of cells at the border of the hole further drives the hole opening process itself, hence suggesting an unexpected role of active stresses in regulating tissue integrity [3]. (ii) tissue flows and renewal within curved environment [4]. Several recent experimental work have shown that epithelial cells spread over curved substrates with a preferential orientation along specific curvature directions. In a recent preprint [4], we work out a set of hydrodynamic equations governing the cell shape and long-time flows of confluent tissues on non-deformable curved substrate. We derive analytical expressions for the threshold value of the local curvature and active stress strength above which a spontaneous global tissue flow arise at steady state. In particular, we predict the stability of a double-shear flow pattern which I will argue shares some similarities with the one observed during the Drosophila embryogenesis process of germ band extension.
1. Saw, T. B. et al. Topological defects in epithelia govern cell death and extrusion, Nature, (2017). 2. Duclos, G. et al. Spontaneous shear flow in confined cellular nematics, Nature Physics (2018). 3. S. Sonam, L. Balasubramaniam, S-Z. Lin, Y. M. Yow Ivan, C. Jebane, Y. Toyama, Philippe Marcq, J. Prost, R.-M. Mège,  J-F. R., B. Ladoux, Mechanical stress driven by rigidity sensing governs epithelial stability (2021). 4. Shear transitions of an active nematic in curved geometries, S. Bell, S.-Z. Lin, J-F. R., and Jacques Prost (2021).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS12.html#author3"><strong class="card-body-subgroup">Alan Lowe</strong></a>
		    	 
		    		<em> (University College London, UK)</em>
		    	
		    	
		    		
		    			</br>"Learning the rules of cell competition"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell competition is a quality control mechanism through which tissues eliminate unfit cells. In biochemical and mechanical competition, individual cell fate is determined by the local cellular neighbourhood. Despite this, cell competition remains poorly understood -- we do not know the interaction 'rules' that determine each cell's fate. This is largely because most studies only quantify whole population shifts for very few time points and for few cells. One major obstacle to understanding how population shifts occur as a result of single cell behaviours is that it requires thousands of cells to be tracked over long periods of time. To address this challenge, we recently built the first deep learning and automated single-cell microscopy system to analyse cell competition. We used this to analyse the cell cycle state of millions of single cells in mechanical competition, including cell division and death. These data suggest that tissue-scale population shifts are strongly affected by cellular-scale tissue organization. We find that local density has a dramatic effect on the rate of division and apoptosis under competitive conditions. Strikingly, our analysis reveals that proliferation of the winner cells is up-regulated in neighbourhoods mostly populated by loser cells. Finally, I present our current progress on developing a machine learning approach to learn interpretable “rules” of cell competition, by predicting the fate of cells in an evolving tissue.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS12.html#author4"><strong class="card-body-subgroup">Pasquale Ciarletta</strong></a>
		    	 
		    		<em> (Politecnico di Milano, Italy)</em>
		    	
		    	
		    		
		    			</br>"Pattern formation and self-organization during cancer cell budding in-vitro"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tissue self-organization into defined and well-controlled three-dimensional structures is essential during development for the generation of organs. A similar, but highly deranged process might also occur during the aberrant growth of cancers, which frequently display a loss of the orderly structures of the tissue of origin, but retain a multicellular organization in the form of spheroids, strands, and buds. The latter structures are often seen when tumor masses switch to an invasive behavior into surrounding tissues. However, the general physical principles governing the self-organized architectures of tumor cell populations remain by and large unclear. In this work, we perform in-vitro experiments to characterize the growth properties of glioblastoma budding emerging from monolayers. We further propose a theoretical model and its finite element implementation to characterize such a topological transition, that is modelled as a self-organised, non-equilibrium phenomenon driven by the trade–off of mechanical forces and physical interactions exerted at cell-cell and cell–substrate adhesions. Notably, the unstable disorder states of uncontrolled cellular proliferation macroscopically emerge as complex spatio–temporal patterns that evolve statistically correlated by a universal law.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS13.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS13-CDEV: </small></br>Combining modeling and inference in cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Maria-Veronica Ciocanel (Duke University, United States), John Nardini (North Carolina State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS13.html#author1"><strong class="card-body-subgroup">Alexandria Volkening</strong></a>
		    	 
		    		<em> (Northwestern University, United States)</em>
		    	
		    	
		    		
		    			</br>"Topological methods for quantitatively describing cell-based patterns"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Self-organization is present at many scales in biology, and here I will focus specifically on elucidating how brightly colored cells interact to form skin patterns in zebrafish. Wild-type zebrafish are named for their dark and light stripes, but mutant zebrafish feature variable skin patterns, including spots and labyrinth curves. All of these patterns form as the fish grow due to the interactions of tens of thousands of pigment cells, making agent-based modeling a natural approach for describing pattern formation. By identifying cell interactions that may change to create mutant patterns, the longterm motivation for my work is to help link genes, cell behavior, and visible animal characteristics in fish. However, agent-based models are stochastic and have many parameters, so comparing simulated patterns and fish images is often a qualitative process. Developing analytically tractable continuum models from agent-based systems is one means of addressing these challenges and better understanding the roles of different parameters in pattern formation. Alternatively, methods from topological data analysis can be applied to cell-based systems directly. In this talk, I will overview our models and present quantitative comparisons of in silico and in vivo cell-based patterns using our topological methods.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS13.html#author2"><strong class="card-body-subgroup">Fiona Macfarlane</strong></a>
		    	 
		    		<em> (University of Saint Andrews, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"A hybrid discrete-continuum approach to model Turing pattern formation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We have developed a hybrid discrete-continuum modelling framework to investigate the formation of cellular patterns through the Turing mechanism. In this framework, a stochastic individual-based model of cell migration and proliferation is combined with a reaction-diffusion system for the concentrations of some interacting chemical species. As an illustrative example, we consider a model in which the dynamics of the morphogens are governed by an activator-inhibitor system that gives rise to Turing pre-patterns. The cells then interact with the morphogens in their local area through either of two forms of chemically-dependent cell action: Chemotaxis or chemically-controlled proliferation. We consider both the case of static spatial domains and additionally investigate the case of growing domains. In all cases we are able to derive the corresponding deterministic continuum limits, inferring an appropriate system of PDEs to model the dynamics of the hybrid model. We investigate parameter situations in which the numerical simulations of the PDE models give an accurate description of the hybrid models, and cases where they do not qualitatively match the hybrid models. This framework is intended to present a proof of concept for the ideas underlying the models, with the aim to then apply the related methods to the study of specific patterning and morphogenetic processes in the future.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS13.html#author3"><strong class="card-body-subgroup">Suzanne Sindi</strong></a>
		    	 
		    		<em> (University of California Merced, United States)</em>
		    	
		    	
		    		
		    			</br>"Multi-Scale Modeling and Parameter Inference in Yeast Protein Aggregation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Unlike a disease caused by a virus or a bacteria, in prion diseases the infectious agent is created by the host organism itself. Prion proteins are responsible for a variety of neurodegenerative diseases in mammals such as Creutzfeldt-Jakob disease in humans and “mad-cow disease” (Bovine Spongiform Encephalopathy or BSE) in cattle. While these diseases are fatal to mammals, prions are harmful to yeast, making yeast an ideal model organism for prion diseases. Most mathematical approaches to modeling prion dynamics have focused on either the protein dynamics in isolation, absent from a changing cellular environment, or modeling prion dynamics in a population of cells by considering the “average” behavior. However, such models have been unable to recapitulate in vivo properties of yeast prion strains. My group develops physiologically relevant mathematical models by considering both the prion aggregates (which evolve inside individual yeast cells) and the yeast cells (which grow and divide). In this talk, I will present a stochastic biochemical reaction system for protein aggregation and demonstrate that the standard computational assumption - fixed protein monomer mass - leads to incorrect biological conclusions. We relax the mass conservation restriction through the use of an additional “slack” species and discover new regimes of biologically relevant behavior. These regimes necessarily correspond to the biologically feasible regions of parameter space for prion aggregation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS13.html#author4"><strong class="card-body-subgroup">Adam MacLean</strong></a>
		    	 
		    		<em> (University of Southern California, United States)</em>
		    	
		    	
		    		
		    			</br>"Bayesian inference of Calcium signaling dynamic provides a map from single-cell gene expression to cellular phenotypes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Since single-cell RNA sequencing technologies have become widespread, great efforts have been made to develop appropriate computational methods to learn biological features from high dimensional datasets. Much less effort has gone into the important yet challenging task of learning about dynamic processes from genomic data. Here we employ spatial transcriptomic data (MERFISH) linked to dynamic Ca2+ responses in single cells for parameter inference. We quantify cell-cell similarity -- learnt via nonnegative matrix factorization of transcriptomic signatures -- and use it to define informative cell-specific priors. We show that these informative priors dramatically speed up Bayesian parameter inference for an ODE model of Ca2+ dynamics. Analysis of posterior parameter distributions across hundreds of single cells allows us to identify genes driving phenotypic changes and link these genes to specific Calcium pathway parameters that are sensitive to outputs. Finally, we test our ability to predict Ca2+ responses using only the cell-cell similarity. This allows us to quantify the amount of information on a dynamic cell phenotype that is contained in the gene expression data alone.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS13" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS14.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS14-CDEV: </small></br>Combining modeling and inference in cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Maria-Veronica Ciocanel (Duke University, United States), John Nardini (North Carolina State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS14.html#author1"><strong class="card-body-subgroup">Keisha Cook</strong></a>
		    	 
		    		<em> (Tulane University, United States)</em>
		    	
		    	
		    		
		    			</br>"Single Particle Tracking with applications to lysosome transport"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Live cell imaging and single particle tracking techniques have become increasingly popular amongst the mathematical biology community. We study endocytosis, the cellular internalization and transport of bioparticles. This transport is carried out in membrane-bound vesicles through the use of motor proteins. Lysosomes, known for endocytosis, phagocytic destruction, and autophagy, move about the cell along microtubules. Single particle tracking methods utilize stochastic models to simulate intracellular transport and give rise to rigorous analysis of the resulting properties, specifically related to transitioning between inactive to active states. This confidence in the stochastic modeling of particle tracking is useful not only for particle-containing lysosomes, but also broad questions of cellular transport studied with single particle tracking.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS14.html#author2"><strong class="card-body-subgroup">Christopher Miles</strong></a>
		    	 
		    		<em> (New York University, United States)</em>
		    	
		    	
		    		
		    			</br>"Stochastic organization of the mitotic spindle from spatiotemporal trajectories"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			For cells to divide, they must undergo mitosis: the process of spatially organizing their copied DNA (chromosomes) to precise locations in the cell. Stochastically driven, this task is accomplished with mysterious speed and accuracy. Our collaborators in the New York State Dept of Health have recently obtained 3D spatial trajectories of every chromosome in a cell during mitosis. Can these trajectories tell us anything about the mechanisms driving them? Fundamental goals of data science (e.g., classification, inference) are challenging here due to the structure and context of this cutting-edge data. I will discuss progress on developing analysis for this data and efforts to model the emerging phenomena.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS14.html#author3"><strong class="card-body-subgroup">Ruth Baker</strong></a>
		    	 
		    		<em> (University of Oxford, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Quantifying the impact of electric fields on single-cell motility"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Electrotaxis is attracting much interest and development as a technique to control cell migration due to the precision of electric fields as actuation signals. However, precise control of electrotactic migration relies on an accurate model of how cell motility changes in response to applied electric fields. We present and calibrate a parametrised stochastic model that accurately replicates experimental single-cell data and enables the prediction of input–output behaviour while quantifying uncertainty and stochasticity. The model allows us to elucidate and quantify how electric fields perturb the motile behaviour of the cell, and to make predictions about cellular motility under different electric fields.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS14.html#author4"><strong class="card-body-subgroup">Carter Jameson</strong></a>
		    	 
		    		<em> (North Carolina State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Parameterizing agent based models of collective cell migration using topological information"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Agent-based models (ABMs) are valuable tools for investigating how rules that govern individual cell behavior affect collective population level migration. ABMs have been used to determine many key features of cell interactions during collective cell migration experiments, including how cells migrate and proliferate and the effects of pushing and pulling between cells. However, to the best of our knowledge, there do not currently exist ABMs of mesenchymal cell migration that have been parameterized with data using rigorous statistical methodology. A primary main reason for the lack of validated models is that current approaches to ABM parameter inference are computationally burdensome or may lead to inaccurate estimates. We developed a novel framework for parameter estimation of ABMs using topological data analysis (TDA). To validate this new approach, we simulated point-cloud datasets using a stochastic variant of the agent-based D’Orsogna model of interactive particle motion. We compared this framework, which relies on least-squares inference and Nelder-Mead direct search optimization on summaries of the topology, to least-squares inference on the particle density. We found that it was feasible to recover model parameters from either deterministic and stochastic variants of the D’Orsogna model.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS14" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS15.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS15-CDEV: </small></br>Modeling of energy-utilizing biopolymers</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Holly Goodson (University of Notre dame, USA), Shant Mahserejian (Pacific Northwest National Laboratory, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS15.html#author1"><strong class="card-body-subgroup">Jared Scripture</strong></a>
		    	 
		    		<em> (University of Notre Dame, USA)</em>
		    	
		    	
		    		
		    			</br>"Quantification of Microtubule Stutters: Dynamic Instability Behaviors that are Strongly Associated with Catastrophe"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microtubules (MTs) are cytoskeletal fibers that undergo dynamic instability (DI), a remarkable process involving phases of growth and shortening separated by stochastic transitions called catastrophe and rescue. Dissecting dynamic instability mechanism(s) requires first characterizing and quantifying these dynamics, a subjective process that often ignores complexity in MT behavior. We present a Statistical Tool for Automated Dynamic Instability Analysis (STADIA), which identifies and quantifies not only growth and shortening, but also a category of intermediate behaviors that we term ‘stutters.’ During stutters, the rate of MT length change tends to be smaller in magnitude than during typical growth or shortening phases. Quantifying stutters and other behaviors with STADIA demonstrates that stutters precede most catastrophes in our dimer-scale MT simulations and in vitro experiments, suggesting that stutters are mechanistically involved in catastrophes. Related to this idea, we show that the anti-catastrophe factor CLASP2γ works by promoting the return of stuttering MTs to growth. STADIA enables more comprehensive and data-driven analysis of MT dynamics compared to previous methods. The treatment of stutters as distinct and quantifiable DI behaviors provides new opportunities for analyzing mechanisms of MT dynamics and their regulation by binding proteins.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS15.html#author2"><strong class="card-body-subgroup">Diana White</strong></a>
		    	 
		    		<em> (Clarkson University, USA)</em>
		    	
		    	
		    		
		    			</br>"Modelling microtubule dynamic instability: microtubule growth, shortening and pausing"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microtubules (MTs) are protein polymers which help form the cytoskeleton of all eukaryotic cells. They are crucial for normal cell development, providing structural support for cells, aiding in cell polarization, as well as aiding in cell motility and division. In order to perform these functions, MTs take on different organizations, in addition to being very dynamic. In particular, MTs go through random periods of relatively slow polymerization (growth) followed by very fast depolymerization (shrinkage), a unique type of dynamics called dynamic instability. The onset of a MT shrinking event is called a catastrophe, while the event at which a MT starts to grow again is called a rescue. Although MT dynamic instability has traditionally been described solely in terms of growth and shortening, MTs have also been shown to pause for extended periods of time. Here, we present a novel mathematical model to describe dynamic instability of MTs in terms of growth, shortening and pausing. Our model is a coupled PDE model, that describes length variations in polymerized tubulin (those growing, shrinking, and pausing), with an ODE model to describe the temporal dynamics of free tubulin. Here, we explore how MT dynamics, and in particular MT catastrophe frequency, is altered in the presence of a pausing/quiescent phase, and compare these results with experimental findings.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS15.html#author3"><strong class="card-body-subgroup">Kimberly Weirich</strong></a>
		    	 
		    		<em> (Clemson University, USA)</em>
		    	
		    	
		    		
		    			</br>"Self-organization and shape change in active biopolymer droplets"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Complex mixtures of macromolecules self-organize to form the soft and active biological materials that structure the cellular cytoplasm. Ordered assemblies of cytoskeletal filaments, such as stress fibers and mitotic spindles, orchestrate the complex mechanical behavior of cells. Key to understanding these exquisite mechanics is elucidating the physical principles of self-organization in these systems. We recently reported dense condensates of cytoskeletal filaments that form liquid crystal condensed phases, where structure arises from the anisotropy of the filaments. Here, we discuss emergent self-organization and shape changes that result from forming composites of these liquid crystals with biological polymers of different rigidities and activity. Our results highlight the role of anisotropy in the self-organization of biological materials and suggest physical mechanisms of controlling shape change in bio-inspired, soft materials.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS15.html#author4"><strong class="card-body-subgroup">Sidney Shaw</strong></a>
		    	 
		    		<em> (Indiana University, USA)</em>
		    	
		    	
		    		
		    			</br>"Extracting local polymer dynamics for global cellular models."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Eukaryotic cells create dynamic polymer systems that affect a wide variety of critical cellular functions. The microtubule polymers in plant cells, for example, form patterns at the cell cortex that template the deposition of cellulose into the nascent primary wall with subsequent effects on the wall material properties that govern cell expansion. A key factor in creating and maintaining the patterned microtubule array is the persistent addition of tubulin subunits to the microtubule ‘plus’ end, with concurrent loss of subunits from the ‘minus’ end, affecting a form of polymer treadmilling that is critical to microtubule array patterning in this system. Prior simulations of this microtubule system coming to a steady-state with a fixed subunit number and cell volume indicated that the frequency of stochastic switching between states of plus-end growth and shortening, termed catastrophe, should critically impact all facets of the steady state polymer system. To further investigate the nature and conditions under which catastrophe occurs, we developed high temporal/spatial in vivo microscopy methods for examining the dynamic properties of cortical microtubules in super-resolved detail. Using model- based tracking algorithms, we observe that polymer growth shows a spectrum of intermediate growth states with transitions from growth to shortening being preceded by a bona fide pause state. Using high temporal resolution data, we find that the decision to resume growth or to catastrophe into depolymerization is temporally consistent with the conversion of GTP-tubulin at the microtubule plus end to GDP-tubulin through stochastic hydrolysis. In cells lacking expression of a known microtubule binding protein, we find evidence that the rate of GTP hydrolysis for tubulin subunits binding to the microtubule plus end differs significantly from wild type. Using computational modeling approaches to compare these systems, we provide evidence that plant cells modulate the tubulin GTPase rate constant in order to control the persistence of plus end growth and the frequency of microtubule catastrophe in this treadmilling system. These data are now being used to revise our cellular scale models for understanding how these microtubule achieve and maintain a steady-state microtubule array.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS15" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS16.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS16-CDEV: </small></br>Mechanisms Underlying Cell Polarization and Its Role in Cell Development</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Weitao Chen (University of California, Riverside, United States), Michael Trogdon (Salk Institute for Biological Studies, United States), Roya Zandi (University of California, Riverside, United States)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS16.html#author1"><strong class="card-body-subgroup">Cole Zmurchok</strong></a>
		    	 
		    		<em> (Vanderbilt University, United States)</em>
		    	
		    	
		    		
		    			</br>"Mechanosensing can enhance adaptation to maintain polarity of migrating cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Migratory cells are known to adapt to environments that contain wide-ranging levels of chemoattractant. While biochemical models of adaptation have been previously proposed, here we discuss a different mechanism based on mechanosensing, where the interaction between biochemical signaling and cell tension facilitates adaptation. In this talk, we develop and analyze a model of mechanochemical-based adaptation consisting of a mechanics-based physical model coupled with the wave-pinning reaction-diffusion model for Rac GTPase activity. We use Local Perturbation Analysis to predict how cells adapt signaling parameters via feedback from mechanics to maintain polarity in response to chemoattractant levels. To confirm this prediction, we simulate the mechanochemical model in moving cells, demonstrating how mechanosensing results in persistent cell polarity when cells are stimulated with wide-ranging levels of chemoattractant in silico. These results demonstrate how mechanosensing may help cells adapt to maintain polarity in variable environments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS16.html#author2"><strong class="card-body-subgroup">Nan Hao</strong></a>
		    	 
		    		<em> (University of California, San Diego, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling the landscape of divergent aging in yeast"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cellular aging is a complex process that involves many interwoven molecular processes. Studies in model organisms have identified many individual genes and factors that have profound effects on lifespan. However, how these genes and factors interact and function collectively to drive the aging process remains unclear. We investigated single-cell aging dynamics throughout the replicative lifespans of S. cerevisiae, and found that isogenic cells diverge towards two aging paths, with distinct phenotypic changes and death forms. We further identified specific molecular pathways driving each aging fate and revealed that these pathways interact and operate dynamically to enable an early-life switch that governs the aging fate decision and the progression towards death. Based on the identified molecular circuit, we developed a computational model that can simulate the landscape of divergent aging trajectories under various conditions. Guided by the model, we genetically engineered a new mode of aging with a dramatically extended lifespan. Our work uncovers the interconnected molecular pathways that drives the aging process and opens up the possibility of designing interventions that simultaneously target multiple network nodes, instead of single genes, to more effectively extend the healthspan.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS16.html#author3"><strong class="card-body-subgroup">Kevin Tsai</strong></a>
		    	 
		    		<em> (University of California, Riverside, United States)</em>
		    	
		    	
		    		
		    			</br>"Yeast budding: linking shape generation with biochemical-mechanical feedback"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			How cells regulate behaviors such as expansion, division, and dynamical shape change is a fundamental question in biological science. The budding of Saccharomyces cerevisiae (yeast) is a prime example of asymmetrical cell growth where reproduction takes the form of a local surface protrusion. During this reproduction event, biochemically the budding process requires the recruitment of cell surface materials and the polarization of growth-associated proteins such as Cdc42. On the other hand, the mechanical properties of the cell wall potentially play a crucial role in bud formation. In this work we developed a novel 3D coarse-grained cell model incorporating probabilistic remeshing and cell growth via surface expansion. We computationally investigated different cell surface weakening and growth patterns and observed both proper and improper bud generation arising from different prescribed conditions. Furthermore, we coupled our mechanical model with a biochemical signaling model to probe the influence of the interplay between the mechanical properties and the biochemical properties on bud development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS16.html#author4"><strong class="card-body-subgroup">Wing Cheong Lo</strong></a>
		    	 
		    		<em> (City University of Hong Kong, P. R. China)</em>
		    	
		    	
		    		
		    			</br>"Deterministic and stochastic analysis for the spontaneous emergence of cell polarity in budding yeast"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Spontaneous emergence of cell polarity intrinsically lies at the localization of signaling molecules on a particular region of cell membrane. Such a process necessarily contains a positive feedback loop to amplify the localized cluster. To describe the polarizing process and explore different feedback functions involved, deterministic and stochastic models with various regulations will be discussed in this talk.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS16" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS17.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS17-CDEV: </small></br>Mathematical approaches to vascular biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jessica Crawshaw (The University of Melbourne, Australia), James Osborne (The University of Melbourne, Australia), Lowell Edgar (The University of Edinburgh, Scotland)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS17.html#author1"><strong class="card-body-subgroup">Alys Clark</strong></a>
		    	 
		    		<em> (The University of Auckland, New Zealand)</em>
		    	
		    	
		    		
		    			</br>"What drives vascular remodelling in the uterus in pregnancy? Vascular adaptions to elevated blood flow."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During pregnancy, the placenta transfers nutrients between the mother and the developing fetus. To do this it must establish a supply of nutrients from the mother’s circulation in the uterus, and so it adapts the maternal blood vessels of the uterus to carry increasing volumes of blood to its surface. If this process does not occur as it should, it can lead to pregnancy complications such as fetal growth restriction. Uterine vascular adaption occurs due to changes in mechanical forces acting on the blood vessel walls (with  increases in blood flow), changes in the structure of the vascular walls (termed outward remodelling) and changes in the hormonal environment of the uterus. This occurs in a multi-scale manner, with adaption at each level in the circulatory network potentially impacting up and downstream function. Here we present data-driven mathematical models of uterine vascular adaption that aim to tease apart the impact of individual contributors to function in a healthy pregnancy. We show that small radial arteries that are potential rate limiters for the volume of blood that can be delivered through the uterus in pregnancy, adapt to be more compliant in rodent pregnancies, and that arteries from rodent pregnancies are more robust to increases in flow without vasoconstriction than outside of pregnancy. Finally, we demonstrate how quantitative descriptions of vascular anatomy and numerical simulations can help to translate data from rodent models to human pregnancies at the organ scale.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS17.html#author2"><strong class="card-body-subgroup">Richard Clarke</strong></a>
		    	 
		    		<em> (The University of Auckland, New Zealand)</em>
		    	
		    	
		    		
		    			</br>"Understanding the mechanical impact of the endothelial glycocalyx’s microstructure"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Endothelial Glycocalyx Layer (EGL) is a thin, brush-like layer that coats the inside of blood vessels. It is believed to serve as a protective barrier against excessive ﬂuid shear, as well as perform a number of other biological functions, such as mechanotransduction. The fragile nature of the EGL, however, makes it very difficult to examine experimentally, and so theoretical models can provide interesting and useful insights. In the past the EGL has been modelled as an isotropic, homogeneous porous layer. However, there is an increasing volume of evidence to suggest that the EGL has a microstructural organisation that brings in to question this assumption. In this talk I will explain some of our recent work using Homogenisation Theory to explore the connections between the EGL’s microstructure, and its bulk macroscopic properties.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS17.html#author3"><strong class="card-body-subgroup">Michael Watson</strong></a>
		    	 
		    		<em> (The University of Sydney, Australia)</em>
		    	
		    	
		    		
		    			</br>"A Multiphase Model of Cap Formation in the Atherosclerotic Plaque"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Atherosclerosis is characterised by the growth of fat-filled plaques in the artery wall. In advanced disease, vascular smooth muscle cells (SMCs) enter the plaque and deposit a cap of fibrous tissue over the fatty plaque core. The fibrous cap isolates the thrombogenic plaque material from the bloodstream and prevents the formation of blood clots that cause heart attacks or strokes. Despite the protective role of the cap, the mechanisms that regulate cap formation and maintenance remain poorly understood. In this talk, I will discuss recent work on modelling the dynamics of cap formation. We use multiphase PDEs with non-standard boundary conditions to simulate plaque SMC migration and tissue remodelling in response to endothelium-derived growth factors. The model results reproduce several observations from experiments in atherosclerosis-prone mice and provide novel insight into the relationship between fibrous cap stability and cap region SMC numbers.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS17.html#author4"><strong class="card-body-subgroup">Fabian Spill</strong></a>
		    	 
		    		<em> (The University of Birmingham, England)</em>
		    	
		    	
		    		
		    			</br>"Organisation and dynamics of the microvasculature"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The microvasculature is a highly dynamic organ. Naturally, during its formation, blood vessel cells move, divide and form networks. Interestingly, the cells maintain dynamic features after the formation of stable networks, where they move around, exert forces on neighbouring cells and extracellular matrix, and form gaps in between the cells. These gaps are critical for the passage of fluid or transmigrating cells. The latter is a critical feature for the immune system, where immune cells need to cross the vasculature into surrounding tissues to reach sites of infection. It is also a deadly process, where cancer cells cross the vasculature during metastasis.
I will discuss some ongoing work on characterising 3D microvascular networks through image analysis and extracting relevant features such as transport capabilities. The analysis shows how network formation depends on conditions such as extracellular matrix. Next, I will discuss a model of blood vessel cell dynamics that can predict how gaps in between the cells form, in dependence on forces and adhesion properties. Experiments validated the model predictions and indicate that these gaps can be exploited by metastasising cancer cells that cross the vasculature to invade surrounding tissues.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS17" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS18.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS18-CDEV: </small></br>Mathematical approaches to vascular biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jessica Crawshaw (The University of Melbourne, Australia), James Osborne (The University of Melbourne, Australia), Lowell Edgar (The University of Edinburgh, Scotland)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS17-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS18.html#author1"><strong class="card-body-subgroup">Jessica Crawshaw</strong></a>
		    	 
		    		<em> (The University of Melbourne, Australia)</em>
		    	
		    	
		    		
		    			</br>"To collapse or not to collapse: how do mechanical forces drive vascular regression?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Vascular regression is a critical process concluding the maturation of developing capillary networks, in which redundant blood vessels are removed. Recent research suggests that forces from the local blood flow (haemodynamic forces) trigger polarized endothelial cell migration against the flow, resulting in capillary collapse and regression. However, vascular regression is also driven by several additional pathways including local adhesion forces and cellular signalling factors. Due to the delicate nature of these microvessels, it is difficult to experimentally untangle the roles of each pathway during vascular development. As such, the development of computational models to analyse the relationship between the local haemodynamic forces and the surrounding vasculature during regression are invaluable.
In this talk, we will present a novel computational framework to mathematically study and isolate the role of haemodynamic in vessel deformation and collapse during vascular regression. To model regression, we describe the capillary wall as a discretised hyperelastic membrane, coupled with a lattice-Boltzmann model of blood flow in an iterative manner. This discrete approach provides a natural framework to consider the relationship between the capillary wall and the local blood flow, and allows for the easy inclusion of structural heterogeneities across the capillary wall. Using this model we are able to examine the role of the haemodynamic forces during vascular regression, as well as the network level ramifications of local regression.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS18.html#author2"><strong class="card-body-subgroup">Daria Stepanova</strong></a>
		    	 
		    		<em> (CRM Centre for Mathematical Research, Spain)</em>
		    	
		    	
		    		
		    			</br>"Multiscale approach to understanding cell rearrangements in early angiogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Angiogenesis is the process whereby endothelial cells (ECs) migrate from a pre-existing vascular bed guided by local environmental cues and interacting with each other to eventually create a new vascular network. We introduce a multiscale model of migration-driven angiogenic sprouting which accounts for the individual phenotype selection of ECs, cell-cell and cell-extracellular matrix interactions. The model, calibrated and validated against various experimental data, captures the characteristic behavior of ECs: branching, cell mixing and, chemotactic sensitivity. These properties, rather than being hard-wired into the model, emerge naturally due to accounting for heterogeneous behavior of ECs depending on their gene expression pattern. This allows us to use the model to investigate the role of cell rearrangements during angiogenic sprouting on the vascular network structure. In particular, we show how cells with impaired gene expression of a specific receptor are characterised by reduced levels of cell rearrangement which influences the branching pattern of vascular networks. Overall, our results support the hypothesis that cell rearrangements play a central role in angiogenesis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS18.html#author3"><strong class="card-body-subgroup">Katie Bentley</strong></a>
		    	 
		    		<em> (The Crick Institute, England)</em>
		    	
		    	
		    		
		    			</br>"Filopodia speed up Notch selection of endothelial tip cells: in silico predictions confirmed in vivo"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			I will describe our recent proof of concept in silica/in vivo study demonstrating that filopodia (actin-rich, dynamic, finger-like cell membrane protrusions) play an unexpected role in speeding up collective endothelial decisions during the time-constrained process of 'tip cell' selection during blood vessel formation (angiogenesis). We first validate simulation predictions in vivo with live imaging of zebrafish intersegmental vessel growth. Further simulation studies then indicate the effect is due to the coupled positive feedback between movement and sensing on filopodia conferring a bistable switch-like property to Notch lateral inhibition, ensuring tip selection is a rapid and robust process. We then employ measures from computational neuroscience to assess whether filopodia function as a primitive (basal) form of active perception due to the sensorimotor coordination apparent in filopodia and find evidence in support. By viewing cell behaviour through the 'basal cognitive lens' we acquire a fresh perspective on the tip cell selection process, revealing a hidden, yet vital time-keeping role for filopodia. Finally, I’ll discuss a myriad of new and exciting research directions stemming from our conceptual approach to interpreting cell behaviour.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS18.html#author4"><strong class="card-body-subgroup">Lowell Edgar</strong></a>
		    	 
		    		<em> (The University of Edinburgh, U.K)</em>
		    	
		    	
		    		
		    			</br>"Force transmission between migrating endothelial agents regulates functional shunting during angiogenic remodelling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During angiogenic remodelling endothelial cells (ECs) composing blood vessels polarise and migrate against the direction of flow. The cellular mechanisms which prevent functional shunting during this process remain poorly understood despite being relevant to arteriovenous malformations and dysfunctional microcirculation and local hypoxia in cancer. We hypothesise that force transmission between migrating ECs plays a crucial role in shunt formation and have designed a model based on force-transmitting agents to investigate. EC agents consisted of nested ellipses polarised against flow. Force transmission between neighbouring agents, based on overlap, consists of extrusive (pushing) forces which maintain spacing and cohesive (pulling) forces which maintain the collective. We simulated migration within an idealised capillary plexus in which agents either split apart or combined at bifurcations based on the convergence/divergence of flow. Extrusion forces stabilise the vasculature and allow cells to intercalate to reduce stress. Excessive amounts of cohesion disrupted this intercalation, creating tension and prolonged flow reversals. Flow reversals switch convergence/divergence at bifurcations, which aggregates cells and leads to shunting and perfusion loss. Our results implicate dysfunctional junctional remodelling and/or force transmission as a possible mechanism vascular malformation and implicate new targets for investigation in future experimental studies.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS18" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS19.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS19-CDEV: </small></br>Dynamics and networks in single-cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS19.html#author1"><strong class="card-body-subgroup">Stephanie Hicks</strong></a>
		    	 
		    		<em> (Johns Hopkins University, USA)</em>
		    	
		    	
		    		
		    			</br>"Scalable statistical methods and software for single-cell data science"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Single-cell RNA-Seq (scRNA-seq) is the most widely used high-throughput technology to measure genome-wide gene expression at the single-cell level. However, single-cell data present unique challenges that have required the development of specialized methods and software infrastructure to successfully derive biological insights. Compared to bulk RNA-seq, there is an increased scale of the number of observations (or cells) that are measured and there is increased sparsity of the data, or fraction of observed zeros. Furthermore, as single-cell technologies mature, the increasing complexity and volume of data require fundamental changes in data access, management, and infrastructure alongside specialized statistical methods to facilitate scalable analyses. I will discuss some challenges in the analysis of scRNA-seq data and present some solutions that we have made towards addressing these challenges.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS19.html#author2"><strong class="card-body-subgroup">Geoffrey Schiebinger</strong></a>
		    	 
		    		<em> (University of British Colombia, Canada)</em>
		    	
		    	
		    		
		    			</br>"Towards a mathematical theory of trajectory inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk develops a rigorous mathematical framework for trajectory inference. We examine the problem of recovering temporal couplings of stochastic processes, motivated by applications in developmental biology and cellular reprogramming. We develop methodology based on optimal transport and test it on data from stem cell reprogramming, sea urchin embryonic development, arabidopsis root growth, and hematopoeisis. We then perform a theoretical analysis and establish rigorous guarantees. Our approach provides a rigorous, general framework for investigating cellular differentiation, and poses some interesting questions in probability, statistics and optimization.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS19.html#author3"><strong class="card-body-subgroup">Gioele La Manno</strong></a>
		    	 
		    		<em> (Swiss Federal Institute of Technology in Lausanne, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Revealing the brain’s molecular anatomy with single-cell and tomography-based spatial transcriptomics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			I will present our comprehensive single-cell transcriptome atlas of mouse brain development spanning from gastrulation to birth. In this atlasing effort, we identified almost a thousand distinct cellular states, including the initial emergence of the neuroepithelium, different glioblasts, and a rich set of region-specific secondary organizers that we localize spatially. In this context, I will provide an example of how the  spatially-resolved transcriptomic data can be particularly useful to interpret the complexity of such complex atlases.
Continuing in this direction, I will show the approach that we recently proposed as a general way to spatially resolve different types of next-generation sequencing data.  We designed an imaging-free framework to localize high throughput readouts within a tissue by combining compressive sampling and image reconstruction. Our first implementation of this framework transformed a low-input RNA sequencing protocol into an imaging-free spatial transcriptomics technique (STRP-seq).
Finally, I will showcase the technique with the profiling of the brain of the Australian bearded dragon Pogona vitticeps. With this analysis, we revealed the molecular anatomy of the telencephalon of this lizard and provided evidence for a marked regionalization of the reptilian pallium and subpallium.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS19.html#author4"><strong class="card-body-subgroup">Kenji Kamimoto</strong></a>
		    	 
		    		<em> (Washington University in St. Louis, USA)</em>
		    	
		    	
		    		
		    			</br>"CellOracle: Dissecting cell identity via network inference and in silico gene perturbation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent technological advances in single-cell sequencing enable the acquisition of multi-dimensional data in a high-throughput manner. These technologies reveal the existence of heterogeneity and the diversity of cell states and identities. To reveal the regulatory mechanism underlying such phenomena, many computational Gene regulatory Network (GRN) inference methods have been proposed. However, understanding biological events from a GRN perspective remains difficult. Even if a computational algorithm can infer GRN, the biological network is so complex that it is challenging to understand how it systematically dictates cell identities. There is significant demand for new methodologies that bridge the gap between cellular phenotypes and the underlying GRN. Thus, we have developed a new method, CellOracle, a new computational approach for the inference and analysis of GRN. By utilizing machine learning algorithms and genetic information, CellOracle infers sample-specific GRN configurations from single-cell RNA-seq and ATAC-seq data. Our GRN models are designed to be used for the simulation of cell identity changes in response to gene perturbation. This simulation enables network configurations to be interrogated in relation to cell-fate regulation, facilitating their interpretation. To validate CellOracle’s GRN inference method, we present benchmarking on various tissues and cell-types. We also validate the efficacy of CellOracle to recapitulate known outcomes of well-characterized gene perturbations in developmental processes, including mouse hematopoiesis and zebrafish embryogenesis. Our benchmarking and validation results demonstrate the efficacy of CellOracle to infer and interpret the dynamics of GRN configurations, promoting new mechanistic insights into the regulation of cell identity.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-MS20.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS20-CDEV: </small></br>Dynamics and networks in single-cell biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-MS20.html#author1"><strong class="card-body-subgroup">Amy Brock</strong></a>
		    	 
		    		<em> (University of Texas at Austin, USA)</em>
		    	
		    	
		    		
		    			</br>"Clonally-resolved mapping of cancer cell trajectories under therapeutic pressure"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Heterogeneity across individual cancer cells and clonal populations impacts growth rate, tumor composition, and response to therapy. To improve treatment, new tools are required to measure and control the contributions of diverse cell subpopulations. Our lab has developed a high-complexity expressed barcode system, ClonMapper, that integrates expressed cell barcoding with single-cell RNA-sequencing and clonal isolation to characterize and track subpopulation trajectories. Using this approach, we uncovered subsets of cells from breast cancer cell models with distinct transcriptional signatures and chemotherapy survivorship trajectories. To gain a deeper understanding of the process of clonal diversification, we profiled clones and retrieved sub-clones over the course of expansion and treatment. Supervised learning indicated that clonal subpopulations have characteristic transcriptomic signatures that are well-conserved under a variety of therapeutic perturbations. By providing the capability for systematic dissection of complex clonal dynamics, ClonMapper enables the delineation of an underlying engine of clonal diversification in cancer cell populations and refines our understanding of clonal identity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS20.html#author2"><strong class="card-body-subgroup">Meghan Ferrall-Fairbanks</strong></a>
		    	 
		    		<em> (University of Florida, USA)</em>
		    	
		    	
		    		
		    			</br>"Single-cell eco-evolutionary dynamics of intratumor heterogeneity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Researchers have recognized that a one-size-fits-all approach is not effective at treating cancer and that tumor heterogeneity plays an important role in response. Current dogma stipulates that this heterogeneity results from compounding genetic and epigenetic changes and instability, ultimately driving unfavorable outcomes for these patients. Nonetheless, some cancers, including many pediatric cancers and some leukemias, have limited genomic diversity. As a result, we have a limited ability to stratify patients into high versus low-risk groups. To address this, we explore intratumor heterogeneity at the single-cell transcriptomic level to quantify and identify driving phenotypes in tumor evolution. We leverage the generalized diversity index (GDI) from ecology, which allows us to tailor the scale of cellular diversity in a given context. We show that the order of diversity parameter in GDI allows us to either emphasize clonal richness at low values while high values shift the analysis toward the abundance of potential drivers of the tumor evolution. We have explored GDI changes in both in vitro sequential single-cell RNA sequencing samples across many cancer tissue types including breast, lung, and ovarian cell lines as well as treatment naïve and treated patient samples in chronic myelomonocytic leukemia. Our analyses show how quantifying intratumor heterogeneity with GDI is a powerful tool to understand eco-evolutionary dynamics of a patient’s tumor.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS20.html#author3"><strong class="card-body-subgroup">Stephen Williams</strong></a>
		    	 
		    		<em> (10X Genomics, USA)</em>
		    	
		    	
		    		
		    			</br>"Analyzing spatial and high-resolution single cell multi-omic data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chromium Single Cell and Visium Spatial Solutions from 10x Genomics provide the ability to build a more comprehensive multidimensional understanding of complex biological systems. Analyzing datasets that measure different molecular modes in thousands of single cells or in a spatial context can be challenging. To enable interpretation of these complex datasets, 10x Genomics provides analysis and visualization software for evaluating genomic, epigenomic, transcriptomic, and proteomic data. In this seminar, we will discuss software, methods, and guidance that can be used to analyze your 10x Genomics datasets.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-MS20.html#author4"><strong class="card-body-subgroup">Sihem Cheloufi</strong></a>
		    	 
		    		<em> (University of California, Riverside, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical modeling of chromatin accessibility to predict stem cell plasticity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Stem and progenitor cells become progressively more restricted in their differentiation potential. This process of cell fate determination is driven by lineage-specific transcription factors and is accompanied by dynamic changes in chromatin structure. The chromatin assembly factor complex CAF-1 plays a central role in assembling nucleosomes during DNA replication and has been implicated in regulating cellular plasticity in various lineages in different organisms. However, whether CAF-1 sustains lineage identity during normal homeostasis by influencing chromatin accessibility is unknown. To address this question, we investigated the role of CAF-1 in a myeloid lineage differentiation paradigm. CAF-1 suppression in myeloid progenitors triggered their rapid commitment but incomplete differentiation toward granulocyte, megakaryocyte, and erythrocyte lineages, resulting in a mixed cellular state. Through comparison with a canonical program of directed terminal myeloid differentiation, we define changes in chromatin accessibility that underlie a unique single cell transcriptome of the matured CAF-1 deficient cells. Using mathematical modeling we further predict the cell fate trajectories of the mixed cellular state caused by CAF-1 inhibition. We use nonlinear dimensionality reduction algorithms to produce RNA velocity maps, pseudotime alignment and corresponding differentiation trajectories. We infer an equation of motion from the RNA velocity vector field, which allows a probability density function to be derived and, ultimately, to compute state transition probabilities. Importantly, the resulting mathematical model of cell fate will be used to identify, interrogate, and quantify states of genome organization and predict corresponding changes to cell identities. Furthermore, we can use the model to predict how interfering with lineage specific transcription factors or factors involved in chromatin organization will modify the differentiation trajectories. Future work with this system will identify new targets to restore normal hematopoiesis in disease and generate clinically relevant cell types.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <a id="CT"><h1 class="mb-3" id="MS1">Sub-group contributed talks</h1></a>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT01.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT01-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT01.html#author1"><strong class="card-body-subgroup">Denis Patterson</strong></a>
		    	 
		    		<em>Princeton University</em>
		    	
		    	
		    		
		    			</br>"A Mathematical Model of Neuronal Identity with Ectopic Domains"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent experiments studying the development of cortical structures in mice have identified COUP-TF1 as a crucial determinant of both the position and sharpness of the boundary between the neo and entorhinal cortices. When COUP-TF1 is under expressed, neocortex invades into territory occupied by the entorhinal cortex in wild-type mice, but the sharp boundary between cortical regions is maintained. However, if COUP-TF1 is over-expressed, the boundary fractures and entorhinal cortex invades the neocortical domain, resulting in mice with ectopic regions of misplaced cortex.We introduce a novel PDE model based on a Keller-Segel-type chemotaxis mechanism to account for both the sharp cortical boundaries of wild-type mice and the ectopic regions observed in mutant mice. Competition between entorhinal and neocortical progenitor cells is mediated by a gradient of COUP-TF1 across the spatial domain and chemotaxis operators model each cell's affinity for cells of their own type. We verify the well-posedness of the system and establish necessary conditions for pattern forming Turing bifurcations; we also numerically study the structure of the Turing space and its dependence on model parameters. Numerical simulations show excellent agreement with experimental observations and we present experimental data verifying the differential adhesion hypothesis underpinning the model's phenomenology.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author2"><strong class="card-body-subgroup">Yoshito Hirata</strong></a>
		    	 
		    		<em>University of Tsukuba</em>
		    	
		    	
		    		
		    			</br>"Reconstructing 3D chromosome structures from single diploid cell Hi-C data via  recurrence plots"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Previously, we have proposed a method for reconstructing 3D chromosome structures from single haploid cell Hi-C data by regarding a contact map as a recurrence plot and applying a method for converting a recurrence plot back to its original time series (Hirata, Oda, Ohta, and Aihara, Sci. Rep. 2016). Here, we extend our previous method to single diploid cell Hi-C data. We discuss that the reconstructed 3D chromosome structures are consistent mathematically as well as biologically. We will start our presentation with a small intuitive quiz for understanding what kind of question we have to solve. The research of Y.H. was partially supported by AMED under Grant Number JP21gm1310004.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author3"><strong class="card-body-subgroup">Dan Tudor</strong></a>
		    	 
		    		<em>University of Edinburgh</em>
		    	
		    	
		    		
		    			</br>"Inferring chemoattractant properties from cell tracking data using mathematical modelling and Bayesian inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The rapid recruitment of immune cells during the inflammatory response is vital to dealing with injury or infection. Immune cells are guided by chemoattractants produced at the wound site. Visualising the underlying chemoattractant gradient can be experimentally complex. In comparison, the cells response to the chemoattractant gradient can be captured more easily via their trajectories. Thus, we are faced with the inverse problem of inferring the chemoattractant gradient from the observed cell movements, which are also subject to noise.  We use an established mathematical framework to model cell migration as a biased persistent random walk, and chemoattractant production and diffusion using a reaction-diffusion equation. By applying Bayesian inference, we can infer the underlying chemoattractant properties. We apply this framework to analyse different wound conditions, to answer if immune cell recruitment can be explained by a single chemoattractant model. We also use Bayesian model comparison to compare different chemoattractant production and release dynamics. Furthermore, we extend the model to infer subpopulations of immune cells with different migratory behaviour without labelling.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author4"><strong class="card-body-subgroup">Philipp Thomas</strong></a>
		    	 
		    		<em>Imperial College London</em>
		    	
		    	
		    		
		    			</br>"Exact solutions for stochastic gene expression in growing cell populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The chemical master equation and the stochastic simulation algorithm are widely used to model reaction kinetics inside living cells. It is sometimes assumed that cell growth and division can be modelled through a  chemical master equation with effective dilution reactions and extrinsic noise sources. We here re-examine this paradigm by developing an analytical agent-based framework of growing and dividing cells. Apart from the common intrinsic noise contribution the theory predicts extrinsic noise without the need to introduce fluctuating rate constants. Instead, extrinsic fluctuations arise from the population structure of a growing cell population that includes cell cycle fluctuations, differences in cell age and cell size variability. We show that, surprisingly, the solution of the chemical master equation - including effective dilution reactions and static extrinsic noise - exactly agrees with the agent-based formulation when the network under study exhibits stochastic concentration homeostasis, a novel condition that generalises concentration homeostasis in deterministic systems to higher order moments and distributions. We illustrate that this result allows us to exactly solve agent-based models for a range of common gene expression networks inside growing cells.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT02.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT02-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT02.html#author1"><strong class="card-body-subgroup">Stephen Zhang</strong></a>
		    	 
		    		<em>University of British Columbia</em>
		    	
		    	
		    		
		    			</br>"Learning cell state dynamics from noisy time-series data using optimal transport"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We devise a theoretical framework and a numerical method to infer trajectories of a stochastic process from snapshots of its temporal marginals. This problem arises in the analysis of single cell RNA-sequencing data, which provide high dimensional measurements of cell states but cannot track the trajectories of the cells over time. We prove that for a class of stochastic processes it is possible to recover the ground truth trajectories from limited samples of the temporal marginals at each time-point, and provide an efficient algorithm to do so in practice. The method we develop, Global Waddington-OT (gWOT), boils down to a smooth convex optimization problem posed globally over all time-points involving entropy-regularized optimal transport. We demonstrate that this problem can be solved efficiently in practice and yields good reconstructions, as we show on several synthetic and real datasets.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT02.html#author2"><strong class="card-body-subgroup">Joshua Forrest</strong></a>
		    	 
		    		<em>University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"Energy based modelling of bacterial signalling systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A key challenge in systems biology is creating mathematical models that can be easily and accurately combined with other models. Such models will need to share a consistent modelling framework and be easily reusable by systems biologists.One solution to this challenge is to use a physics-based approach to modelling. Bond graphs are an energy-based modelling framework that describe the rate of energy flow (power) moving through system components. By construction, bond graphs models enforce physical and thermodynamic constraints, making model components physically consistent with one another. Bond graphs also provide a graphical representation of the model and allow for easy hierarchical modelling.To demonstrate bond graph modelling applied to biological systems, we have applied this framework to Two Component Systems (TCS). TCS are a signalling mechanism found in many common bacteria such as E. coli and B. subtilis. By modelling the explicit energy dependence of TCS using bond graphs, we find new insights into the behaviour of the system in different energy contexts. A modular framework also means we can combine models together to investigate coupling dynamics of TCS. In future, we argue that such an approach could lead towards the development of a systems-wide, physically plausible whole-cell model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT02.html#author3"><strong class="card-body-subgroup">Nishtha Pandey</strong></a>
		    	 
		    		<em>Center for Computational Natural Sciences and Bioinformatics, International Institute of Information Technology - Hyderabad, India</em>
		    	
		    	
		    		
		    			</br>"Mathematical modelling of neuronal cell cycle re-entry in Alzheimer's disease"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Neurodegenerative disease (ND) is an umbrella term used to classify medical conditions associated with neuronal atrophy and gradual loss of cognitive abilities. The most common ND is Alzheimer's disease (AD). However, the approved drugs mostly treat the symptoms of AD. Therapeutic approaches targeting Amyloid beta (Aβ) aggregation fail to reverse or inhibit disease progression. These observations point towards gaps in the understanding of disease mechanisms. During development, the progenitor cells mature into neurons and they switch to a post mitotic, resting state. However, cell cycle reentry often precedes neuronal apoptosis hinting at a close interaction between the two processes. In this study we develop mathematical models of  multiple pathways leading to cell cycle re-entry in neurons. These models incorporate the cross talk between cell cycle, neuronal and apoptotic signaling mechanisms. Our study shows that different self-sustaining feedback loops operate in post mitotic neurons that can make the cell cycle re-entry and transition to an apoptotic state irreversible. Important cell cycle regulators that function as hub nodes were identified. Further, we propose a combinatorial therapy targeting Aβ proteolysis as well as blocking the cell cycle feedback loop may alleviate the severity of the disease.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT02.html#author4"><strong class="card-body-subgroup">Domenic Germano</strong></a>
		    	 
		    		<em>The University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"Towards a realistic 3D deformable model of dynamic tissues"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Colorectal Cancer is one of the most prevalent forms of cancer within western society. It is known to develop within the epithelia of the colon, localised to distinct invaginations within the intestinal wall, known as the crypts of Lieberkürn. While much is known about these crypts, the biomechanical process responsible for their structural maintenance remains unknown. One such process believed to be responsible for the crypts structural stability is believed to be a result of the surrounding stromal tissue.Here, we will present a 3D, multilayer, cell-centre model of tissue deformation, where cell movement is governed by the minimisation of a bending potential across the epithelium, cell-cell adhesion, and viscous effects. Using this model, we will show how the tissue is capable of maintaining a consistent structure while undergoing self renewal. We will also show how the model extends natural to describe general tissue deformations, and we hope to further extend it to describe crypt dynamic homeostasis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT03.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT03-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT03.html#author1"><strong class="card-body-subgroup">Bahti Zakirov</strong></a>
		    	 
		    		<em>Francis Crick Institute</em>
		    	
		    	
		    		
		    			</br>"Active Perception during Angiogenesis: Filopodia speed up Notch selection of Tip Cells in silico and in vivo"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			How do cells make efficient collective decisions during tissue morphogenesis? Humans and other organisms use feedback between movement and sensing known as 'sensorimotor coordination' or 'active perception' to inform behaviour. Here we provide the first proof of concept in silico/in vivo study demonstrating that filopodia (actin-rich, dynamic, finger-like cell membrane protrusions) play an unexpected role in speeding up collective endothelial decisions during the time-constrained process of 'tip cell' selection during blood vessel formation (angiogenesis). We first validate simulation predictions in vivo with live imaging of zebrafish intersegmental vessel growth. Further simulation studies then indicate the effect is due to the feedback between movement and sensing on filopodia conferring a bistable switch-like property to Notch lateral inhibition, ensuring tip selection is a rapid and robust process. We then employ measures from computational neuroscience to assess whether filopodia function as a primitive (basal) form of active perception and find evidence in support. By viewing cell behaviour through the 'basal cognitive lens' we acquire a fresh perspective on the tip cell selection process, revealing a hidden, yet vital time-keeping role for filopodia. Finally, we discuss a myriad of new and exciting research directions stemming from our conceptual approach to interpreting cell behaviour.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT03.html#author2"><strong class="card-body-subgroup">Eva Deinum</strong></a>
		    	 
		    		<em>Wageningen University</em>
		    	
		    	
		    		
		    			</br>"Zebrastripes for life!"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The plant cell wall is a versatile material that can meet a wide range of mechanical requirements. The banded patterns in protoxylem form a striking example, enabling these vessels withstand substantial negative pressure and allow for extension at the same time. The required anisotropic material properties largely derive from the location and orientation of the constituting cellulose microfibrils. These, in turn are deposited along the cortical microtubule cytoskeleton. So, using the case of protoxylem as a model system for complex cell wall patterns, the question becomes how cortical microtubules can self-organize into banded patterns. This happens in interaction with another well-known patterning system, the ROP proteins.         Studying their interaction provides interesting methodological challenges, as cortical microtubules are most often studied in ``particle based'' stochastic simulations, whereas ROPs, or their animal/yeast counterparts, are typically described in terms of partial differential equations. We, therefore, started by addressing both parts of the interaction in isolation: how can dynamic microtubules collectively adjust to a predefined ROP pattern and how can an –implicitly microtubule derived– field of diffusion anisotropy orient and change ROP patterns? Despite the very different modelling frameworks, our ROP work provided critical insights into a problem in the stochastic microtubule simulations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT03.html#author3"><strong class="card-body-subgroup">Franziska Krämer</strong></a>
		    	 
		    		<em>Buchmann Institute for Molecular Life Sciences</em>
		    	
		    	
		    		
		    			</br>"Investigating Mechanical Force Dynamics of Extra-Embryonic Membranes in Tribolium castaneum (part 1)"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Efficient energy use and storage is crucial in living organisms. In the context of evolution, energy management is continuously optimized to ensure an individual's ability to successfully compete. This is especially true for oviparous species, as all required energy has to be provided at the moment of oviposition in order to give rise to a fully functional organism. Based on our preliminary imaging data in the emerging insect model Tribolium castaneum, we formulate the hypothesis that extra-embryonic serosa cells utilize shape change during gastrulation to allocate and store energy that is later on required for their extensive movement during dorsal closure. To investigate this possible functional connection, we want to gain further insights into the multi- scale effects of force propagation from cellular to tissue level. Spatial and temporal dynamics of forces are calculated using non-invasive Force Inference (FI). FI utilizes a biomechanical model, a mathematical inverse method and a Bayesian framework to estimate cell and tissue stress from segmented image data and for the whole system simultaneously. Here we highlight our workflow from obtaining 3D time-lapse light sheet-based fluorescent microscopy images of live Tribolium embryos to multi-scale estimation of tensions and pressures acting in the serosa membrane.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT03.html#author4"><strong class="card-body-subgroup">Zoë Lange</strong></a>
		    	 
		    		<em>Frankfurt Institute for Advanced Studies</em>
		    	
		    	
		    		
		    			</br>"Investigating Mechanical Force Dynamics of Extra-Embryonic Membranes in Tribolium castaneum (part 2)"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Efficient energy use and storage is crucial in living organisms. In the context of evolution, energy management is continuously optimized to ensure an individual's ability to successfully compete. This is especially true for oviparous species, as all required energy has to be provided at the moment of oviposition in order to give rise to a fully functional organism. Based on our preliminary imaging data in the emerging insect model Tribolium castaneum, we formulate the hypothesis that extra-embryonic serosa cells utilize shape change during gastrulation to allocate and store energy that is later on required for their extensive movement during dorsal closure. To investigate this possible functional connection, we want to gain further insights into the multi- scale effects of force propagation from cellular to tissue level. Spatial and temporal dynamics of forces are calculated using non-invasive Force Inference (FI). FI utilizes a biomechanical model, a mathematical inverse method and a Bayesian framework to estimate cell and tissue stress from segmented image data and for the whole system simultaneously. Here we highlight our workflow from obtaining 3D time-lapse light sheet-based fluorescent microscopy images of live Tribolium embryos to multi-scale estimation of tensions and pressures acting in the serosa membrane.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT04.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT04-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT04.html#author1"><strong class="card-body-subgroup">Tricity Andrew</strong></a>
		    	 
		    		<em>North Carolina State University</em>
		    	
		    	
		    		
		    			</br>"Computational modeling to gain insight into developmental processes shaping stomach curvature"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many organs develop left-right (LR) asymmetrical shapes and positions internally. Failure to properly establish LR asymmetry causes common, severe birth defects. The leftward curvature of the stomach is one of the most recognized LR asymmetries in the body. During its development the left side of the stomach undergo a specialized type of cell rearrangement (radial intercalation) that expands the left layers of the tube, thus forcing it to curve. Experiments with frog embryos (X. Laevis), however, show that inhibiting these cell rearrangements alone is not sufficient to prevent curvature, indicating the presence of additional left- and/or right-specific mechanisms. To explore what these mechanisms may be, I am integrating 2D and 3D agent-based computational modeling and animal experimentation. Starting with stomach shapes derived from nano-CT tomography, I am exploring which combinations of LR mechanisms, such as radial intercalation, cell shape changes, and differential adhesion, can reproduce patterns of stomach curvature under different conditions. The knowledge gained will help prioritize and contextualize the study of candidate genes from our ongoing genomic sequencing of human LR birth defects patients. On a broader level, we will gain novel insights into the physical forces that shapes 3D tissues, revealing general “rules” of tubular morphogenesis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT04.html#author2"><strong class="card-body-subgroup">Amjad Khan</strong></a>
		    	 
		    		<em>Dalhousie University</em>
		    	
		    	
		    		
		    			</br>"Modeling the transmission and loss of an important class of mobile genetic elements"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Horizontal gene transfer (HGT) allows the transmission of genetic information between microorganisms. Integrative Conjugative Elements (ICE) are segments of DNA that contain genes for insertion and excision from the genome, and transfer between microorganisms. ICE frequently contain antimicrobial resistance (AMR) genes and are significant contributors to the global increase in AMR incidence. Despite being widely distributed and contributing to the spread of AMR genes, their role, transfer rate, and impact on the fitness of the host are largely unexplored. In this study, we have developed a Partial Differential Equation model that considers the distribution of ICE across a population of bacteria. We assume that after a time step, this distribution may change due to four processes, each of which has a corresponding parameter in the model: excision, HGT, mutational degradation, and conference of a selective advantage. We fit this PDE to the data obtained from more than a thousand genomes from the genus Enterococcus, an opportunistic pathogen that is a frequent cause of hospital-derived infections. This will result in the estimation of transfer rate, conjugation rate, degradation rate, and selective advantage, which can be further utilized to study the genetic repertoire of ICEs.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT04.html#author3"><strong class="card-body-subgroup">Julio M Belmonte</strong></a>
		    	 
		    		<em>North Carolina State University</em>
		    	
		    	
		    		
		    			</br>"Non-Linear Mechanical Response Transforms a Graded Molecular Distribution into a Step-Wise Output in Cell Behavior"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The intrinsic genetic programme of a cell is not always sufficient to explain the cell's activities. External mechanical stimuli are increasingly being recognized as determinants of cell behavior. In the epithelial folding event that constitutes the beginning of gastrulation in Drosophila, the genetic programme of the future mesoderm leads to the establishment of a contractile actomyosin network that triggers apical constriction of cells, and thereby, furrow formation. However, some cells do not constrict but instead stretch, even though they share the same genetic programme as their constricting neighbors. We show here that tissue-wide interactions override the intrinsic programme of a subset of cells, forcing them to expand even when an otherwise sufficient amount and concentration of apical, active actomyosin has been accumulated. Models based on contractile forces and linear stress-strain responses are not sufficient to reproduce experimental observations, but simulations in which cells behave as materials with non-linear mechanical properties do. Our models also show that this behavior is an emergent property of supracellular actomyosin networks, in accordance with our experimental observations of actin reorganization within stretching cells, with this event being stochastic and rare in cells with high myosin levels, but reproducible in cells with lower concentrations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT04.html#author4"><strong class="card-body-subgroup">Sarafa Iyaniwura</strong></a>
		    	 
		    		<em>The University of British Columbia, Vancouver</em>
		    	
		    	
		    		
		    			</br>"Oscillatory instabilities for a 2-D coupled ODE-PDE Model of Diffusion-Mediated Communication Between Small Signaling Compartments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We analyze a class of cell-bulk coupled ODE-PDE models that characterize communication between localized spatially segregated dynamically active signaling compartments/cells. In this model, the cells are disks of a common radius coupled through a passive extracellular bulk diffusion field in a bounded 2-D domain. Each cell secretes a signaling chemical into the bulk region at a constant rate and receives bulk chemical feedback from the entire collection of cells. This global feedback, which activates signaling pathways within the cells, modifies their intracellular dynamics according to the external environment. In the limit of finite diffusion, the method of matched asymptotic expansions is used to construct steady-state solutions of the ODE-PDE model and to derive a globally coupled nonlinear matrix eigenvalue problem (GCEP) that characterizes the linear stability properties of the steady-states. We also used matched asymptotic analysis to derive a nonlinear ODE system from the coupled ODE-PDE model in the limit of asymptotically large bulk diffusivity. For Sel'kov reaction kinetics, we investigated oscillatory instabilities in the dynamics of the cells triggered by global coupling. We also studied quorum sensing and how coupling defective cells to a group of identical cells change the intracellular dynamics of the cells.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT04" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT05.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT05-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT05.html#author1"><strong class="card-body-subgroup">David Holloway</strong></a>
		    	 
		    		<em>British Columbia Institute of Technology</em>
		    	
		    	
		    		
		    			</br>"Controlling the number of cotyledons in conifer embryos"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Conifers, unlike flowering plants, generate variable numbers of cotyledons (embryonic 'seed leaves'). Conifer cotyledons do not form all over the dome-shaped embryo, but form in a single ring at a particular distance from the tip. A 3-fold increase in this ring radius from 55 to 180 µm corresponds to the experimentally observed range of 2 to 10 regularly-spaced cotyledons. In the flowering plant Arabidopsis, leaves are also initiated at a particular distance from the growing tip. Molecularly, this is at a 'trough' between two expression domains, of REV (an HD-ZIP III protein) above the leaves and KAN below the leaves. REV and KAN are mutually inhibitory via miRNAs (tasiARF from REV, miR166 from KAN). This is at least partly shared by conifers: overexpression of miR166 in larch decreases HD-ZIP III expression and affects cotyledon formation. We have developed a model for HD-ZIP III (H), KAN (K) regulation to investigate how their interface position is controlled - in particular, what allows for the 3-fold natural variability in conifer cotyledon ring radius. Simulating Arabidopsis H/K experimental perturbations contributes to a general mechanism for radial positioning, as well as quantitatively predicting radial shifts in new conifer experiments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author2"><strong class="card-body-subgroup">Tamsin Spelman</strong></a>
		    	 
		    		<em>Sainsbury Laboratory, University of Cambridge, UK</em>
		    	
		    	
		    		
		    			</br>"Links between microtubules and nucleus shape in a plant root hair cell"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Root hair cells develop out of Trichoblast cells in the plant root epidermis and are characterised by a long thin protrusion which in Arabidopsis is ≈10μm wide and can grow to ≈1mm in length [1]. For growth of this protrusion, nucleus migration up the root hair is necessary, with the nucleus positioned ≈80μm back from the growing tip [2]. Our aim is to understand nucleus shape and position in the root hair, particularly focusing on how it is affected by the cytoskeleton (microtubules and actin). By segmenting experimental data, we analyse 3D nucleus shape and motion before and after treatment with drugs effecting osmolarity and microtubule organisation: Mannitol and Oryzalin. To further understand the relationship between the nucleus and microtubules, we then analyse the microtubule distribution in the nucleus-tip region of the root hair. Using 3D microtubule simulations and simple nuclear dynamics models, we analyse the microtubule density distributions and organisation for a range of conditions in the root hair cell and compare these to the distributions obtained experimentally. [1] Grierson C et. al. (2014) Root hairs. Arabidopsis Book. 12:e0172. [2]T. Ketelaar et. al. (2002) Positioning of Nuclei in Arabidopsis Root Hairs. The Plant Cell, 14(11):2941-2955;
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author3"><strong class="card-body-subgroup">Daniel Koch</strong></a>
		    	 
		    		<em>King's College London</em>
		    	
		    	
		    		
		    			</br>"Information Processing by Homo-Oligomeric Proteins: From First Principles to Cardiac Arrhythmias"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Reversible protein homo-oligomerisation, i.e. the formation of larger protein complexes out of identical subunits, is observed for 30-50% of all vertebrate proteins. Despite being a ubiquitous phenomenon, the specific function of protein homo-oligomerisation remains poorly understood. I previously demonstrated theoretically that homo-oligomerisation could be a versatile mechanism for a range of signal processing capabilities such as dynamic signal encoding, homeostasis and bistability via pseudo-multisite modification. In this talk I will present the first dynamical systems model of phospholamban (PLN), a crucial mediator protein of β-adrenergic signaling and regulator of calcium cycling in heart muscle cells. Importantly, PLN forms homo-pentamers whose function remained unclear for decades. Simulations and model analyses demonstrate that pentamers enable bistable phosphorylation and further constitute substrate competition based low-pass filters for phosphorylation of monomeric PLN. I confirmed both predictions of my model experimentally by demonstrating substrate competition in vitro and hysteresis of pentamer phoshorylation in cardiomyocytes. These non-linear phenomena could ensure consistent monomer phosphorylation and calcium cycling despite noisy signaling activity in the upstream network and are likely impaired by a genetic mutation that causes arrhythmogenic heart disease. These studies show that homo-oligomerisation can play unanticipated and potentially disease relevant roles in biochemical signaling networks.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author4"><strong class="card-body-subgroup">Adriana Zanca</strong></a>
		    	 
		    		<em>The University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"	 Cell proliferation and migration during wound healing"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Experiments have suggested that during skin wound healing, the wound front is comprised of a hyperproliferative region behind a non-proliferating migrating tongue at the wound edge. Mathematical and computational models allow us to efficiently explore the effects of changing the characteristics of these proposed regions in order to suggest plausible hypotheses about the mechanics of the regions, and wound healing in general. In this work we use cell-based computational modelling to investigate competing experimental findings regarding whether the regions spatially overlap. Furthermore, we examine the effect of changing the size, location and cell characteristics, such as proliferation and migration rate, of each region on the behaviour of the wound edge.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT06.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT06-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT06.html#author1"><strong class="card-body-subgroup">Tim Liebisch</strong></a>
		    	 
		    		<em>Goethe Universität Frankfurt</em>
		    	
		    	
		    		
		    			</br>"Mathematical Modelling Identifies Core Principles of Epithelial Organoid Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Epithelial organoids are three-dimensional cell culture systems mimicking aspects of organ development and disease. Pancreas organoids are found to be very heterogeneous in culture and exhibit diverse, dynamic behaviour. One example is frequent size oscillations, which are hypothesised to occur in response to an interplay of the elasticity and the production of an osmotic active substance by the cell monolayer, as increasing osmotic pressure can lead to rupture of the cellular contacts, allowing the pressure to relax.Mathematical modelling allowed us to extract core principles driving these size oscillations of the organoids. By deriving a scaling law from the organoid dynamics, we can identify a dependence of the observed size oscillations and the cell proliferation dynamics. Furthermore, size oscillations also depend on the surface-to-volume ratio, hence, on the organoid size. A biomechanical 3D agent-based model confirms these mathematical considerations.The implemented model allows investigation of the interplay of the elasticity of the cells and their production rate in more detail and further observed phenomena such as organoid rotation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT06.html#author2"><strong class="card-body-subgroup">David M. Versluis</strong></a>
		    	 
		    		<em>Leiden University, Leiden, the Netherlands</em>
		    	
		    	
		    		
		    			</br>"How Oxygen and Lactose Metabolism Shape the Infant Gut Microbiota"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Nearly immediately after birth, a complex and dynamic ecosystem forms in the human infant gut. The characteristics of this system influence the infant's health in both the short and long term. 2'-FL, the most prevalent prebiotic in most human milk, varies greatly in presence and concentration between individuals. We use a multi-scale spatiotemporal model of the infant colon from birth to three weeks of age to reproduce the effects of variations in  nutritional components on the composition and metabolic activity of the microbiota. Using flux balance analysis with molecular crowding on genome-scale metabolic models from the AGORA project, we calculate bacterial fluxes for different locations and time points at a high resolution. The resulting fluxes are integrated together into a model of the ecosystem that feeds back into the flux calculations. The model can give insight and produce predictions for bacterial and metabolic composition of the infant microbiota over time and under different conditions. Our aim is to reach a deeper understanding of the influence that nutrition can have on the development of the infant microbiota. This in turn is the first step towards a comprehensive understanding of the formation of a steady state adult microbial environment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT06.html#author3"><strong class="card-body-subgroup">Renske Vroomans</strong></a>
		    	 
		    		<em>University of Amsterdam, Origins Center</em>
		    	
		    	
		    		
		    			</br>"Evolution of selfish multicellularity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent studies have shown that many functions of multicellular organisms were already present in their unicellular ancestors. For instance, many gene families involved in animal development and cell adhesion can also be found in unicellular relatives. This indicates that the evolutionary transition to multicellularity predominantly required changes in regulation and coordination, more than gene content. We use an evolutionary cell-based model to show that the emergent collective behaviour of multicellular clusters can drive the evolution of adhesion. We then extend this model to investigate how regulation of cell behaviour evolves in concert with the evolution of multicellularity. Cells have an evolvable gene regulatory network that determines when they divide and migrate. We observe that evolution of adhesion changes cell competition dynamics: cells evolve adhesion to migrate collectively and to get closer to resources. Within such cohesive clusters, competition drives cells to divide first, and then migrate to resources. When cells cannot evolve adhesion, cells instead migrate to reach resources first and then divide, blocking their competitors. Thus, the model demonstrates how the transition to multicellularity may have driven a drastic switch in cell behaviour, leading to complex coordinated dynamics compared to the unicellular cousins, without changing the genetic toolkit.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT06.html#author4"><strong class="card-body-subgroup">John Fozard</strong></a>
		    	 
		    		<em>John Innes Centre</em>
		    	
		    	
		    		
		    			</br>"Diffusion-mediated coarsening can explain crossover pattening in meiosis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Meiosis, the special cell divisions occurring prior to sexual reproduction, is a crucial process for most organisms.Early during meiosis, the chromosomes encoding the same genes group together to form linear structures.At a small number of points along these structures the DNA strands form crossovers, splicing together chromosomes and exchanging genetic information.These crossovers tend to be much more regularly spaced than would be expected by chance, and the mechanism controlling this is a long-standing open question.We have developed a model for crossover patterning, in which a protein diffuses along the one-dimensional structure, and is able to aggregate at a number of discrete foci.Competition between the foci for the protein leads to coarsening, selecting those foci that will become crossovers.This will allow us to explain many of the elements of crossover positioning in the model plant Arabidopsis thaliana, and we will show that it agrees well with our experimental observations.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT07.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT07-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT07.html#author1"><strong class="card-body-subgroup">Mikahl Banwarth-Kuhn</strong></a>
		    	 
		    		<em>University of California Merced</em>
		    	
		    	
		    		
		    			</br>"Multiscale Modeling of Neurodegenerative Diseases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prion proteins are associated with fatal neurodegenerative diseases in mammals, but they are also responsible for a number of harmless heritable phenotypes in yeast. Under certain experimental conditions, changes in protein aggregation dynamics between neighboring cells result in sectored colonies corresponding to loss of the prion phenotype. The resulting phenotypic organization provides a rich data set that can be used to uncover the non-intuitive relationships between protein aggregation mechanisms across multiple scales. In this talk, we introduce a novel, two-dimensional agent-based model of a budding yeast colony with detailed representation of cell-type specific biological processes, including budding, variation in cell-cycle length, and asymmetric protein segregation. The model is used to study the impact of budding cell division, nutrient limitation and yeast colony organization on yeast colony phenotypes. In the model, prion dynamics are simulated within each individual cell using simplified intracellular dynamics, and spatial arrangement of cells is modeled using a center-based modeling approach. The multiscale model may have the potential to predict mechanisms underlying experimentally observed phenomena such as sectored prion phenotypes in yeast colonies and serve as a tool for future hypothesis generation and testing.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT07.html#author2"><strong class="card-body-subgroup">Benjamin Brindle</strong></a>
		    	 
		    		<em>Lehigh University</em>
		    	
		    	
		    		
		    			</br>"Bifurcation Analysis in a Mathematical Model for Red Blood Cell Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Red blood cells are one of the most important components of life in humans. Loss of red blood cells has consequences, such as anemia or even death. Such a loss could be the result of parasitemia, viral infection, or phlebotomy. Red blood cell dynamics within a human involve several stages of precursor cells before a red blood cell fully matures to an erythrocyte. After blood loss, a feedback mechanism contingent on loss and level of erythrocytes causes the production of more precursor cells to return the blood dynamics to equilibrium. We seek to understand these dynamics from a biological perspective by using mathematics to model specific loss scenarios. We model this process using a system of nonlinear, deterministic, ordinary differential equations. Functions describing this feedback, the stem cell recruitment, and the erythrocyte loss are chosen to examine the system dynamics. Some parameter choices cause a Hopf bifurcation, while others lead to death, stable steady states, or limit cycles. Numerical methods are used to display bifurcation diagrams and transient dynamics. By understanding red blood cell dynamics through the utilization of mathematical modeling and dynamical systems tools, diseases can be mitigated as we understand the scenarios in which they occur.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT07.html#author3"><strong class="card-body-subgroup">Marcos Gouveia</strong></a>
		    	 
		    		<em>CFisUC-Center of Physics of the University of Coimbra, University of Coimbra</em>
		    	
		    	
		    		
		    			</br>" *	 Tip cell migration in sprouting angiogenesis: The role of extracellular matrix mechanics'"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Sprouting angiogenesis is the process by which new blood vessels grow from preexisting ones. It is a fundamental process during embryo development, as well as in many physiological (vascular remodeling, wound healing) and pathological phenomena (solid tumor growth, diabetic retinopathy). Angiogenesis is heavily dependent on a number of factors such as: the concentration of oxygen in the surrounding tissues, chemical signaling and endothelial cell coordination and proliferation. Cell migration is key for sprouting angiogenesis to occur since the new sprouts need to reach the cells in hypoxia. Thus, studying the forces exerted by cells on the extracellular matrix (ECM) and how the matrix's mechanical properties affect cell migration can help us understand the role angiogenesis plays in different processes.We have developed a mathematical model using the phase field method taking into account the elastic interactions between endothelial tissue and ECM, as well as the adhesion and traction forces exerted by cells. At the same time, we have used in vitro, aortic ring assays using mouse aorta slicesto quantify the cell's migration potential in gels with different rigidities.Both studies show that there is a value of ECM rigidity that results in an optimal migration distance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT07.html#author4"><strong class="card-body-subgroup">David Hardman</strong></a>
		    	 
		    		<em>Usher Institute, University of Edinburgh</em>
		    	
		    	
		    		
		    			</br>"A combined in vitro/agent-based modelling approach for optimising muscle cell culture medium"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present a combined in silico / in vitro approach to optimising experimental muscle cell culturing protocols using an agent-based model (ABM). Functions of experimentally derived and inferred cell behaviours are applied as inputs and levels of myocyte-myotube fusion, an indicator of experimental success, as an output. We conducted sets of in vitro experiments to obtain fixed and time-lapse images of myocyte differentiation and myotube maturation under a range of media serum concentrations and concentrations of N2B27 neuron differentiation medium. Metrics of myoblast motion and proliferation were quantified from time-lapse imaging and used to inform a cell residence-time dependent ABM of myoblast-myotube fusion. An Approximate Bayesian Computation method was applied to infer the myocyte residence-time threshold by comparing ABM simulations with cell fusion data extrapolated from stained images.Once validated, the ABM was used to run in silico experiments over a range of media conditions to predict the conditions most likely to produce quality muscle tissue. A sweep of the behavioural parameters defining the ABM was conducted to assess the relative effect size of individual cell behaviours on cell fusion.We also discuss extending the protocol to analyse the spatial distribution of nuclei as a quality indicator.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT08.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT08-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT08.html#author1"><strong class="card-body-subgroup">Flavia Feliciangeli</strong></a>
		    	 
		    		<em>University of Leeds, Bayer AG</em>
		    	
		    	
		    		
		    			</br>"How is a population of final cells maintained? A compartmental branching approach for cell differentiation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell differentiation is a process through which a generic cell evolves into a given type of cell, usually into a more specialized type. Cells of the human body have nearly identical genome but exhibit very different phenotypes that allow them to carry out specific functions and react to changes in the surrounding environment. We can model cells sharing the same surface attributes (same phenotype) as belonging to the same mathematical state (or compartment). Cells can either die, divide or change phenotype (entering another compartment). We derive a cell-compartmental model for cell differentiation; by defining a family of random variables we can model the progeny of a founder cell as a stochastic process. We can describe the evolution of mean quantities by a set of ordinary differential equations and we analyse a number of summary statistics to bring insight to the understanding of cellular dynamics. We show, with two case studies from Cellular Immunology, how our mathematical techniques can shed light on the dynamics of cell differentiation in different systems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author2"><strong class="card-body-subgroup">Sakurako Tanida</strong></a>
		    	 
		    		<em>The University of Tokyo</em>
		    	
		    	
		    		
		    			</br>"Organoid morphogenesis at various luminal fluid pressure and proliferation time in a multicellular phase-field model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Organoids are self-organizing cells that are grown from stem cells in vitro and are widely used to model organ development and disease. In organoids, while cell growth and hence proliferation are mechanically constrained due to the geometrical requirements to keep maintaining the cell cluster, various morphologies of organoids are achieved. However, it remains elusive how such mechanical constraint can affect organoid growth and the final morphology. In this study, we investigate the influences of mechanical constraint on organoid morphogenesis by numerical simulations with a multicellular phase-field model. In this mathematical model, we can isolate out mechanical interaction from other biological processes. More specifically, we examine the pattern formations of organoids emerging when changing luminal fluid pressure and proliferation time. Even if most organoids seem to be the same in the initial phase, they have distinctive features in the later phase in this numerical model. The patterns in the later phase include spheroid-like shape, star-like shape, and so on. Although all cells have identical natures, in the star-like organoid, cells that can divide are spatially fixed and show behavior like spontaneous differentiation. Classifying the patterns of organoids by several indexes, we discuss the mechanisms which generate the different pattern.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author3"><strong class="card-body-subgroup">Kana Fuji</strong></a>
		    	 
		    		<em>The University of Tokyo</em>
		    	
		    	
		    		
		    			</br>"Lumenogenesis simulations of organoids using a multicellular phase-field model with molecules of apical components"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Organoids are three-dimensional cultured organ models grown from stem cells. Epithelial organoids which have apico-basal polarity in each cell form lumens on the apical side and the lumens grow during the cell self-organization process. In addition to osmotic pressure in luminal fluid and cortical tension, the presence of non-adhesive apical membranes is involved in luminal area expansion recently. However, it remains unclear how the lumenogenesis processes were affected localized adhesive property changes in the membrane. In this study, to investigate the luminal pattern formed by cells with localized non-adhesive membranes, we extended the multicellular phase-field model with luminal fluid to a multicellular model with polarity by introducing the molecules of the apical component. The results of simulations with this model showed that lumens were formed even at pressures lower than the pressure required for the lumen growth without the introduction of the cell polarity. This model reproduces not only the round lumen but also the squeezed lumen that was not available in the previous models. In this talk, we will discuss the quantitative analysis of the lumen structure and its comparison with experiments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT08.html#author4"><strong class="card-body-subgroup">Mete Demircigil</strong></a>
		    	 
		    		<em>Institut Camille Jordan, Lyon</em>
		    	
		    	
		    		
		    			</br>"Aerotactic Waves in Dictyostelium discoideum : When Self-Generated Gradients engage with Expansion by Cell Division."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Using a self-generated hypoxic assay, it is shown that Dictyostelium discoideum displays a remarkable collective aerotactic behavior: when a cell colony is covered, cells quickly consume the available oxygen and form a dense ring moving outwards at constant speed and density. We propose a simple, yet original PDE model, that enables an analytical qualitative and quantitative study of the phenomenon and reveals that the collective migration can be explained by the interplay between cell division and the modulation of aerotaxis. The modeling and its conclusions supplement and are confirmed by an experimental investigation of the cell population behavior. This approach also gives rise to an explicit and novel formula of the collective migration speed of cells that encapsulates a surprising combination of expansion by cell divison, such as described by the Fisher/KPP equation, and aerotaxis. The conclusions of this model appear to extend to more complex models.This is joint work with Christophe ANJARD Vincent CALVEZ, Jean-Paul RIEU, Olivier COCHET-ESCARTIN and and is a subpart of the work presented in the preprint bioRxiv 2020.08.17.246082; doi: https://doi.org/10.1101/2020.08.17.246082.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT09.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT09-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT09.html#author1"><strong class="card-body-subgroup">William Martinson</strong></a>
		    	 
		    		<em>University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Extracellular matrix remodelling by neural crest cells provides a robust signal for collective migration"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Neural crest cells (NCCs) exhibit highly invasive phenotypes in vertebrates; they migrate from the neural tube of an embryo throughout its developing tissues. Since many NCC progenitors contribute to homeostasis in mature organisms, it is unsurprising that disruptions to NCC migration can have severe consequences on individual health, ranging from developmental defects to embryonic lethality. However, the relative importance of the biological mechanisms that contribute to the emergence and maintenance of NCC migration patterns remains to be established. Here, we model discrete NCC migratory streams using experimental data in the chick embryo. In collaboration with developmental biologists, we create a new agent-based model (ABM) for NCC migration that examines how remodelling of the extracellular matrix (ECM) can provide a non-local signal that allows cells to maintain coherent streams. We perform a global sensitivity analysis to identify model mechanisms that most contribute to successful migration, and use the ABM to generate in silico predictions to test through in vivo experiments. We find that ECM remodelling, haptotaxis, and contact guidance provide sufficient signals for NCCs to establish robust in silico streams; however, additional mechanisms are required to steer cells towards appropriate target sites.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT09.html#author2"><strong class="card-body-subgroup">Sangita Swapnasrita</strong></a>
		    	 
		    		<em>Maastricht University</em>
		    	
		    	
		    		
		    			</br>"Kinetic modeling of toxin transport in a bio-artificial kidney"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The organic anion transporters (OATs) in the kidney are mainly responsible for transepithelial removal of uremic toxins out of the blood. To improve current (passive) dialysis treatments, researchers are trying to mimic this active removal by culturing kidney cells expressing the toxin transporters directly on outer surface of a hollow fibre membrane. Using a computational model with independent contributions of the activity and density of the toxin transporters, we have theoretically shown how the transporter density distribution can influence the local toxin clearance. More specifically, we tested twelve different patterns with varying total cell area, while keeping the total number of transporters constant. The computational findings showed that a more homogeneous transporter distribution resulted in a higher toxin clearance. We also demonstrated that short, serially connected cultures of cells would provide equivalent clearance compared to long fibers. In summary, this study contributes to an improved understanding of toxin transport in cellularized hollow fibers, which represent a promising strategy for renal replacement therapies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT09.html#author3"><strong class="card-body-subgroup">Chiara Villa</strong></a>
		    	 
		    		<em>University of St Andrews</em>
		    	
		    	
		    		
		    			</br>"A mathematical model of endothelial progenitor cell cluster formation during the early stages of vasculogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The formation of new vascular networks is essential for tissue development and regeneration, in addition to playing a key role in pathological settings such as ischemia and tumour development. Experimental findings in the past two decades have led to the identification of a new mechanism of neovascularisation - cluster-based vasculogenesis - occurring in a variety of hypoxic settings in vivo. The focus of this talk is on the early stages of cluster-based vasculogenesis, during which endothelial progenitor cell (EPC) cluster formation is mediated by the action of matrix degrading enzymes and EPC proliferation. We present a mathematical model which sheds light on the spatio-temporal mechanisms responsible for cluster formation and cluster size. The numerical results, which qualitatively compare with data from in vitro experiments, provide further insights on the underlying dynamics indicating promising, fruitful future modelling and experimental research perspectives.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT09.html#author4"><strong class="card-body-subgroup">Supriya Krishnamurthy</strong></a>
		    	 
		    		<em></em>
		    	
		    	
		    		
		    			</br>"Stochastic chemical reaction networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
	                <a id="Posters"><h1 class="mb-3" id="MS1">Sub-group poster presentations</h1></a>
									<div class="row subgroup-name">
										<div class="col-12">
											<div class="card">															
										  	<h5 class="card-header card-header-subgroup CDEV">CDEV Posters</h5>
											  <div class="card-body">
											    <p class="card-text">																
														<div>
															
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-1 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-1.html">
															    	</br><strong>Parkrati Dangarh</strong>
														    	 
														    		<em>Imperial College London</em>
														    	
														    	
														    		
														    			</br>"Systems-level modeling of meiotic entry, commitment, progression and exit"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Upon nitrogen starvation, Schizosaccharomyces pombe (fission yeast) exits the mitotic cell cycle and becomes irreversibly committed to the completion of the meiosis program. In meiosis, DNA replication (S-phase) is followed by two successive rounds of cell divisions (Meiosis I and Meiosis II) without intermediate interphase. In this work, we developed a comprehensive model of the entire meiotic cell cycle, which couples exit from mitosis to meiotic commitment and progression under nitrogen starvation. This network was assembled from several experimental observations in the literature for meiotic cell divisions and exit. The core of the regulatory network is the regulation of cyclin-dependent kinase (Cdk) 1 and anaphase-promoting complex or cyclosome (APC/C) by meiosis-specific factors. The network was translated into a set of ordinary differential equations to simulate the dynamics of meiotic progression. We also performed one and two-parameter bifurcations to study the role of different feedback loops in meiosis. The model accounts for about 60 experimental situations including single and multiple mutations and demonstrates the control strategy involving multiple feedback loops to yield two successive division cycles. The model serves as a key tool for experimentalists to perform in silico mutations and test the hypothesis.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-10 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-10.html">
															    	</br><strong>Atchuta Srinivas Duddu</strong>
														    	 
														    		<em>Indian Institute of Science</em>
														    	
														    	
														    		
														    			</br>"Modelling epigenetic feedback in gene regulatory network consisting of three mutually inhibiting transcription factors"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Regulatory biochemical networks demonstrate different levels of control - transcriptional, translational, protein activity and epigenetic. Toggle triad is a network motif consisting of three mutually repressing transcription factors each driving a distinct cell fate. We have previously demonstrated underlying multistability in a toggle triad: it can give rise to six states in total, three of which are 'single positive' (A high, B low, C low;  A low, B high, C low; and A low, B low, C high) and three 'double positive' (A high, B high, C low; A high, B low, C high; and A low, B high, C high), which can be mapped on to Th1, Th2, Th17 and hybrid Th1/Th2, Th1/Th17 and Th2/Th17 states in CD4+ T cell differentiation. Here, we incorporated epigenetic feedback mechanism into the previously developed gene regulatory model for a toggle triad to study the stability of the various phenotypes of the motif identified previously and investigate the dynamics of irreversibility of cell-fate decisions as a function of epigenetic feedback incorporated on different links in the network. Our results offer insights into cell-fate decision-making vs. cell-fate commitment for a multistable network and helps understand plasticity and stability of different T-helper states.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-2 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-2.html">
															    	</br><strong>Takayuki Ohara</strong>
														    	 
														    		<em>Leibniz Institute for Farm Animal Biology</em>
														    	
														    	
														    		
														    			</br>"Mathematical Modeling of Rhythmic Gene Expression: Impact of Negative Autoregulation on Amplitude Preservation"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The mammalian circadian clock is an endogenous biochemical oscillator that generates rhythmicity with a period of about 24 hours in the expression of numerous clock-controlled genes (CCGs) mainly by controlling their transcription or mRNA stability. There are hurdles for propagating high amplitude oscillations from the circadian clock to CCG expression. Long molecular half-lives decrease relative oscillation amplitudes, and half-lives of proteins are, indeed, often long enough to significantly reduce amplitude. A question, then, arises; how does the gene expression process overcome the amplitude-dampening effect to retain strong rhythmicity?Here we theoretically investigate negative autoregulation as a possible scenario for propagation of strong circadian rhythmicity. We considered a CCG coding for, e.g., a transcription factor that undergoes post-translational modifications and represses its own expression. We studied mathematical models with or without the negative autoregulation, which were formulated in terms of parameters directly observable in omics scale data. Our analyses show that amplitudes can be strongly propagated with negative autoregulation, overcoming limits due to long half-lives. Moreover, when modification steps were increased, reliable and precise rhythm propagation, defying random cell-to-cell variation in rates and lifetimes, was readily achievable. Our results are general enough to be applicable to a variety of oscillation phenomena.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-3 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-3.html">
															    	</br><strong>Masayuki Kashiwa</strong>
														    	 
														    		<em>Akita Prefectural University</em>
														    	
														    	
														    		
														    			</br>"Visualization of cell flow by cell vertex and bubbly cell shape tracking"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			For clinical applications, researchers have paid attention to biological tissues and their multicellular mechanical structures. However, the mechanical aspect of morphological mechanisms remain unclear. So far, mathematical models have been developed to elucidate the mechanism: the vertex model (VM) by polygon approximation of cells, and the bubbly vertex model (BVM) with the curvature of cell boundaries, and so on. None of them leads to the basic equations for the migration and deformation of cell populations yet. The main reasons are that the physical properties of tissues differ among the morphogenetic stages, the cell boundary tensions are non-uniform, and the stress-strain relationship has not been clarified.In order to determine the physical properties for such basic equations, precise quantification of cell flow is necessary. However, conventional methods, such as PIV and PTV, do not fit naively to various cellular events: deformation, division, apoptosis, rearrangement, etc. Live-imaging techniques also limit the quality of experimental data.In this paper, we extract cell boundaries from the live data, fit them to the tissue shape defined by BVM, and perform vertex and edge (cell boundary) tracking similar to PTV. We also attempt to visualize and quantitatively evaluate the cell flow from real data.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-4 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-4.html">
															    	</br><strong>Sathvik Sanjeev Buggana</strong>
														    	 
														    		<em>CCNSB, IIIT Hyderabad</em>
														    	
														    	
														    		
														    			</br>"Modeling the liver circadian clock control by nutrients"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Circadian rhythms are 24-hour cycles in physiological processes. On a molecular level, the circadian clock is regulated by transcriptional/translational feedback loops. Experiments in recent years on mammalian circadian clocks have shown that external factors such as diet, nutrients and even blood gas concentrations have effects on circadian period, amplitude, and phase. Molecular players linked to these external factors have been found to regulate and be regulated by the circadian clock. In this work, we have developed a mathematical model to study the effect of high fat diet and nutrients on the liver circadian clock. The model interlinks feeding and fasting cycle and circadian clock and provide insights into the regulation by external factors.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-5 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-5.html">
															    	</br><strong>David Versluis</strong>
														    	 
														    		<em>Leiden University, Leiden, the Netherlands</em>
														    	
														    	
														    		
														    			</br>"How Oxygen and Lactose Metabolism Shape the Infant Gut Microbiota"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Nearly immediately after birth, a complex and dynamic ecosystem forms in the human infant gut. The characteristics of this system influence the infant's health in both the short and long term. 2'-FL, the most prevalent prebiotic in most human milk, varies greatly in presence and concentration between individuals. We use a multi-scale spatiotemporal model of the infant colon from birth to three weeks of age to reproduce the effects of variations in  nutritional components on the composition and metabolic activity of the microbiota. Using flux balance analysis with molecular crowding on genome-scale metabolic models from the AGORA project, we calculate bacterial fluxes for different locations and time points at a high resolution. The resulting fluxes are integrated together into a model of the ecosystem that feeds back into the flux calculations. The model can give insight and produce predictions for bacterial and metabolic composition of the infant microbiota over time and under different conditions. Our aim is to reach a deeper understanding of the influence that nutrition can have on the development of the infant microbiota. This in turn is the first step towards a comprehensive understanding of the formation of a steady state adult microbial environment.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-6 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-6.html">
															    	</br><strong>Elena Pascual Garcia</strong>
														    	 
														    		<em>Leiden University</em>
														    	
														    	
														    		
														    			</br>"Introducing the nucleus into the Cellular Potts Model: a multiscale approach for cell migration studies"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Cell migration is fundamental in multicellular organisms. It is involved, e.g., in early development, in maintenance of tissue homeostasis, in the functioning in the immune system. Importantly, besides cytoplasmic deformability, the nucleus stiffness can limit cellular migration through channels in dense environments. In order to adequately model this process, recent work has proposed compartmental Cellular Potts Models, in which one CPM compartment represents the cytoplasm, and a second, stiffer compartment is embedded within the cytoplasmic compartment to model the nucleus. Here we show that the increased stiffness of the nucleus introduces an artificial friction with the lattice.  We provide quantitative data showing that  the increased  nuclear stiffness can visibly slow down the cytoplasmic movement, thus producing an artifact in speed quantification. We present an alternative approach to model the nucleus that addresses this issue: a node-based CPM-independent layer, which maintains the deformability characteristics, but does not directly influence the cytoplasmic movement.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-7 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-7.html">
															    	</br><strong>Emine Atici Endes</strong>
														    	 
														    		<em>Heriot-Watt University</em>
														    	
														    	
														    		
														    			</br>"Keratinocyte Growth Factor-Based Mathematical Models of Epidermal Wound Healing"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The mammalian skin is the largest organ of the body after the skeletal system and its primary function is to protect the body against the external environment. The maintenance of the skin's functionality, integrity, and strength are coordinated by specialised cells localised in three intricate layers of the skin: epidermis, dermis, and hypodermis. Understanding the fundamental cause of changes in the cells in these three skin compartments, thus provides a base for progress in interpreting injuries in the skin and its healing process.Wound healing is a normal biological and dynamic process in the human skin in which many types of cells, various cytokines, and growth factors, such as KGF, act in harmony. This process starts immediately after skin injury and is completed into three distinct but highly integrated phases: an inflammatory stage, a proliferative and migration  (re-epithelialization) phase, and a long reformation and remodeling phase. Therefore, in this talk, I will give a sequence of PDE models presenting the interaction between the dermis and the epidermis, in which KGF plays an active role during the re-epithelialization.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-8 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-8.html">
															    	</br><strong>Voorsluijs Valerie</strong>
														    	 
														    		<em>Luxembourg Centre for Systems Biomedicine, University of Luxembourg</em>
														    	
														    	
														    		
														    			</br>"Energetic cost of the cross-talk between calcium dynamics and mitochondrial metabolism"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Ion exchanges across mitochondrial membrane play a crucial role for energy metabolism [1–2]. In this work, we focus on the interplay between Ca2+ dynamics and mitochondrial metabolism. Ca2+ activates different enzymes of the TCA cycle and thereby enhances ATP production, while Ca2+ exchanges via SERCA and PMCA pumps are consuming ATP. The kinetics of this coupling has been studied with mathematical modelling [3–5], but its concrete energetic cost remains elusive. Here, we investigate this cost computationally by means of the entropy production rate (EPR) of Ca2+ exchanges and major mitochondrial metabolic processes. We show that the EPR as a measure for the throughput of the Gibbs free enthalpy of the TCA cycle exhibits a maximum in dependence on the glucose level, which could represent an optimal working regime of cells.  1. Wu, F. et al. J. Biol. Chem. 2007, 282, 24525–24537. 2. Wei, A.-C. et al. Biophys. J. 2011, 100, 2894–2903. 3. Magnus, G.; Keizer, J. Am. J. Physiol. 1998, 274, C1158-1173. 4. Bertram, R. et al. A. J. Theor. Biol. 2006, 243, 575–586.  5. Wacquier, B. et al. Sci. Rep. 2016, 6, 19316.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-9 (Session: PS01)</small>
															    <a href="/CDEV/CDEV-PS01-CDEV-9.html">
															    	</br><strong>Md Hamidul Islam</strong>
														    	 
														    		<em>Lecturer, Department of Applied Mathematics, University of Rajshahi, Rajshahi-6205, Bangladesh</em>
														    	
														    	
														    		
														    			</br>"Modelling the Host Immune Response to Primary Dengue Infection"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Dengue is a mosquito borne viral infection triggering a series of intracellular events in the host immune system, which sometimes leads to severe dengue infection resulting in serious illness and even in death if the patient is not treated properly. We present stochastic model describing the interplay between dengue virus and host immune response in primary dengue infection. The stochastic model is derived from the deterministic model describing the dynamics of the disease. We analyze the deterministic model to explore the factors influencing the virus persistence in the body for extended periods. The results are then compared with the stochastic model. The stochastic model is shown to provide better insights into the viral dynamics. The stochastic model provides a wide range of results including different size of viral loads and different time of maximum infection occurring in the body. In addition, the stochastic model exhibits positive probability of viral extinction, as opposed to deterministic model, when the virus reproduction number R_0>1. We calculate the extinction probability as a function of R_0 where extinction probability is found to decrease with an increasing R_0, suggesting that at high infection rate the effect of uncertainties in underlying dengue dynamics maybe negligible.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-11 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-11.html">
															    	</br><strong>Georgia Pope</strong>
														    	 
														    		<em>Harvey Mudd College</em>
														    	
														    	
														    		
														    			</br>"Stability Analysis of a Mathematical Model of Hormonal Contraception"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Combination oral contraceptives (COCs), containing a combination of synthetic progestin and estrogen, have become a leading form of contraception in the United States. The pill is taken cyclically, meaning it is taken regularly during a certain “on” period, followed by a shorter “off” period during which menstruation occurs. In order for the pill to be effective, it must be taken daily during the “on” period and it is recommended that it be taken around the same time every day. This requirement poses a challenge to many users and can result in unwanted pregnancies. We explore the stability of the contraceptive state achieved by hormonal birth control using a mechanistic mathematical model of the menstrual cycle. Specifically, we build off a model by Wright and colleagues, in which the authors model concentrations of exogenous progestin and estrogen as a constant. We include the dynamics of the on/off dosing of COC's by introducing a time-dependent function for exogenous estrogen and progestin and investigate the stability of the model in response to changes in exogenous hormone dosing. Accurately modeling COC dosing could provide insight into when a contraceptive state has been lost due to inconsistency or changes in hormonal birth control use.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-12 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-12.html">
															    	</br><strong>Frederick Laud Amoah-Darko</strong>
														    	 
														    		<em>Clarkson University</em>
														    	
														    	
														    		
														    			</br>"Continuous Model for Microtubule Dynamic instability with Pausing"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Microtubules (MTs) are protein polymers found in all eukaryotic cells. They are crucial for normal cell development, providing structural support for the cell and aiding in the transportation of proteins and organelles. In order to perform these functions, MTs go through random periods of relatively slow polymerization (growth) and very fast depolymerization (shrinkage), a unique type of dynamics called dynamic instability. The onset of a MT shortening event is called a catastrophe, while the event at which a MT starts to grow again is called a rescue. Although MT dynamic instability has traditionally been described solely in terms of growth and shortening, MTs have also been shown to pause for extended periods of time. Here, we present a novel mathematical model to describe the population dynamics of MTs. The goal is to use this model to determine MT catastrophe rates, in addition to time spent growing, shortening and pausing. These are quantities that can be used to compare our model results with experimental findings.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-13 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-13.html">
															    	</br><strong>Victor Matveev</strong>
														    	 
														    		<em>New Jersey Institute of Technology, Dept of Math Sciences</em>
														    	
														    	
														    		
														    			</br>"Approximations of stationary calcium nanodomains in the presence of buffers with two binding sites"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Calcium ion (Ca2+) elevations near open Ca2+ channels, termed Ca2+ nanodomains, trigger secretory vesicle fusion, myocyte contraction, and other fundamental physiological processes. Ca2+ nanodomains are shaped by the interplay between Ca2+ influx, diffusion, and binding to Ca2+ buffers and sensors, which absorb most of the Ca2+ entering the cell. The dependence of Ca2+ concentration on the distance from the Ca2+ channel can be modeled with reasonable accuracy using closed-form approximations of quasi-stationary solutions of the corresponding reaction-diffusion equations. Such stationary approximations help to reveal the qualitative characteristics of Ca2+ nanodomains without resorting to computationally expensive numerical simulations. Although a variety of nanodomain approximations are known when Ca2+ is diffusing in the presence of Ca2+ buffers with a single Ca2+ binding site, most biological buffers have more complex Ca2+-binding stoichiometry. We present several closed-form approximations of Ca2+ nanodomains in the presence of buffers with two binding sites, extending prior work on stationary Ca2+ nanodomains. Our approximants interpolate between the short-range and long-range distance-dependence of Ca2+ concentration using a combination of rational and exponential functions. We shows that this method achieves significant accuracy for a very wide range of Ca2+ buffering parameter values. Supported by NSF DMS-1517085 (V.M)
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-14 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-14.html">
															    	</br><strong>Jia Lu</strong>
														    	 
														    		<em>Duke University</em>
														    	
														    	
														    		
														    			</br>"Distributed information encoding and decoding using self-organized spatial patterns"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Biology can generate distinct self-organized patterns according to different initial conditions, and one could infer the corresponding condition given a pattern. Moreover, under the same or similar conditions, these patterns share global similarity but vary in detail due to random noise. Here, we leverage the above properties of bacterial colony patterns and combine with machine learning (ML) to achieve distributed information encoding and decoding with guaranteed security. Specifically, to encode, a message is converted into cell seeding configuration followed by colony growth, during which a colony pattern develops; to decode, we input the pattern into a trained CNN to convert it back to the original message. By modulating the patterning dynamic and encoding setup, we could tune the trade-off among encoding capacity, decoding accuracy and security, characterized by ML decoding performance. We also implemented ensemble techniques for enhancing decoding reliability and making full use of the expensive-to-obtain patterning data, and combined the framework with established cryptography techniques (e.g., encryption and hashing) to further enhance the security. Our method is applicable for a wide variety of pattern-formation systems and demonstrates a novel way of utilizing biological noise, as well as quantifying the extent of convergence for dynamical systems by using ML.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-15 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-15.html">
															    	</br><strong>J. Cody Herron</strong>
														    	 
														    		<em>UNC Chapel Hill, Bioinformatics and Computational Biology</em>
														    	
														    	
														    		
														    			</br>"Quantification and modeling of podosomes during frustrated phagocytosis"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Podosomes are complex, actin-rich cellular adhesion structures important for migration, motility, tumor invasion, and more. One system in which podosomes play a crucial role is in phagocytosis, the recognition and engulfment of small particles by cells. To observe podosome dynamics, we use the experimental system of “frustrated” phagocytosis, in which cells attempt to engulf fixed, micropatterned discs of antibody. This process is frustrated because cells can recognize the antibody and engage in signaling yet are unable to fully engulf and internalize the fixed discs of antibody. This system is advantageous for studying both the physical structure of podosomes and the highly dynamic spatiotemporal signaling that occurs during phagocytosis. Strikingly, we observe as podosomes form rosettes (puncta in a ring) around the discs. We use computational approaches, including persistent homology (a type of topological data analysis), to automatically identify podosomes and quantify their nanoarchitecture from 3D super-resolution microscopy data. Furthermore, we use reaction-diffusion models to investigate the molecular mechanisms that generate the rosette patterns formed during frustrated phagocytosis.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-16 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-16.html">
															    	</br><strong>Alireza Ramezani</strong>
														    	 
														    		<em>UCR, Physics department</em>
														    	
														    	
														    		
														    			</br>"Subcellular Mechanochemical Model to Study Growth Regulation"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Growth regulation is an important question in developmental biology and remains unclear for many living systems. Abnormal development and fatal diseases, such as cancer, can be result of uncontrolled tissue growth. The Drosophila wing disc, an epithelial primordial organ that later forms the adult fruit fly wing, is appropriate to study growth regulation because of its relatively simple geometry, limited number of cells, rapid growth, and a well understood molecular signaling network. Nevertheless, the mechanism of growth regulation in Drosophila wing disc tissue remains a subject of intense debate. Multiple mechanisms for growth regulation have been proposed, following the substantial evidence that suggests morphogens regulate growth. However, most existing models focus on either the biochemical signaling pathway or mechanical properties. Very few attempt to incorporate both factors in a mechanistic way. Here we developed a coupled mechanochemical model at sub-cellular level to study growth regulation controlled by the morphogen gradient with cell mechanics taken into account to achieve spatial homogeneous growth and the asymmetric shape of the tissue. The model suggested the shape of the morphogen gradient affected the tissue growth rate and final shape. Moreover, the feedback regulation on the morphogen facilitated the tissue growth through shaping the gradient.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-17 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-17.html">
															    	</br><strong>Daniel Cruz</strong>
														    	 
														    		<em>Georgia Institute of Technology</em>
														    	
														    	
														    		
														    			</br>"Agent-based Modeling of Emergent Patterns Within Stem Cell Colonies"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The differentiation of stem cell colonies into specified tissue types is possible through local and long-distance intercellular communication; however, it is unclear which mechanisms take priority in context-specific situations. Here we consider human induced pluripotent stem cells (hiPSCs) whose therapeutic potential arises from their ability to differentiate into all germ lineages. Prior work in the literature suggests that both cell-autonomous and non-autonomous  (e.g. positional) mechanisms determine cell fate during the differentiation of hiSPCs, producing patterns and other system-level features in the process. Informed by experimental data, we develop a collection of agent-based models (ABMs) whose agents (i.e. cells) are each equipped with local rules that govern how the agents interact with their environment and with each other; the purpose of these ABMs is to simulate the early differentiation of hiPSCs according to different sets of biological assumptions. We also extend an existing mathematical framework which formalizes ABMs to estimate long-term model behavior with respect to time. Our estimates aim to establish connections between local interactions and certain system-level observations. Thus, we study both the emergent behaviors of our ABMs and the dynamics of the local rules governing each agent to ascertain which modes of intercellular communication determine cell fate.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-18 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-18.html">
															    	</br><strong>Mary Ellen Rosen</strong>
														    	 
														    		<em>Brigham Young University</em>
														    	
														    	
														    		
														    			</br>"A Mathematical Analysis of Focal Adhesion Lifetimes and Their Effect on Cell Motility"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Active cell motion is a fundamental process for most living organisms.  It is crucial for embryogenesis, pathologies such as fighting infections or the spread of cancer, and single cell organisms in finding favorable environments. In this research we analyze data regarding a subprocess of amoeboid cell motion - the lifetime of focal adhesions (FAs).  Cells attach to and gain traction from the extracellular matrix via FAs. We collect and analyze existing FA lifetime data and find that it is gamma distributed.  We also find that cell speed decreases as the mean FA lifetime increases. Mathematical modeling suggests that the detach-rate is both force and time dependent.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-19 (Session: PS02)</small>
															    <a href="/CDEV/CDEV-PS02-CDEV-19.html">
															    	</br><strong>Josué Manik Nava-Sedeño</strong>
														    	 
														    		<em>National Autonomous University of Mexico</em>
														    	
														    	
														    		
														    			</br>"Collective migrantion and pattern formation with zero-range interactions"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			We investigate the collective migration and pattern formation potential of zero-range-interacting agents, that is, agents interacting exclusively with others at exactly the same position. To this end, we use a lattice-gas cellular automaton model with no exclusion principles and polar/nematic velocity alignment interactions. We find that, in the case of polar alignment, the model shows a transition towards a highly ordered and condensed state with moving point clusters. In the nematic case, we observe the formation of high density, nematically-ordered bands. This suggests that migration is enough to relay information among individuals and to generate collective effects at the population level, even in the absence of spatially extended sensing.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-21 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-21.html">
															    	</br><strong>Olivia N.J.M. Marasco</strong>
														    	 
														    		<em>University of Lethbridge</em>
														    	
														    	
														    		
														    			</br>"Cycles of self-limiting ATF4 regulation: a potential dynamical motif."</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The ATF4 transcription factor network plays a critical role in controlling the shift from pro-survival to pro-apoptosis regimes when mammalian cells experience starvation, viral infection, ER or oxidative stresses. Continued activation of pro-survival pathways with failure to initiate apoptosis under chronic stress is associated with dysfunction of the ATF4 network and is a feature of cells undergoing tumorigenesis. The shift between the pro-survival and pro-apoptosis regimes is observed to be switch-like over time but the mechanisms by which this shift occurs, or which components contribute to this emergent behaviour, are not fully understood. It may be possible to gain a better understanding of the factors that control this network by studying it in contextually isolated modules identifying unique dynamical motifs that may contribute to the ATF4 network's emergent dynamical behaviour. Self-limiting ATF4 regulation has been observed with respect to CARE (C/EBP-ATF Response Element)-containing targets under amino acid limitation and describes behaviour in which the expression of these targets is initially promoted and then later repressed by other targets in a timed program. A model of ATF4's regulation of Cat-1, an amino-acid transporter that is upregulated in response to amino-acid starvation, is evaluated as a potential example of a self-limiting motif.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-22 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-22.html">
															    	</br><strong>Akshay Paropkari</strong>
														    	 
														    		<em>University of California, Merced</em>
														    	
														    	
														    		
														    			</br>"Using Machine Learning to Predict Novel Gene Regulatory Interactions During Candida albicans Biofilm Development"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Candida albicans is a common fungal pathogen of humans, capable of forming biofilms which are surface-adhered fungal cells within an extracellular matrix. C. albicans biofilms are attributed for over 50% hospital acquired infections. Previously, our lab identified six core transcription factors (TFs) required for the formation of mature biofilms in C. albicans. In this study, we utilize previously published data sets to identify the transcriptional network controlling C. albicans biofilm formation. We implemented a support vector machine classifier to identify novel TF binding sites by utilizing binding site 3D DNA shape and motif features. For each of the six TFs, novel TF-gene interactions were observed. Finally, active and inactive TF-gene interactions were identified by integrating novel TF-gene interactions with time-series gene expression data. This work, using interdisciplinary approaches, provides insights into potential molecular targets for therapeutic applications.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-24 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-24.html">
															    	</br><strong>Youngmin Park</strong>
														    	 
														    		<em>Brandeis University</em>
														    	
														    	
														    		
														    			</br>"The Dynamics of Vesicles Driven Through Closed Constrictions by Molecular Motors"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			We study the dynamics of a model of membrane vesicle transport into dendritic spines, which are bulbous intracellular compartments in neurons driven by molecular motors. We explore the effects of noise on the reduced lubrication model proposed in [Fai et al, Active elastohydrodynamics of vesicles in narrow, blind constrictions. Phys. Rev. Fluids, 2 (2017), 113601]. The Fokker-Planck approximation fails to capture mean first passage times of velocity switching (tug-of-war effect), and the agent-based model is computationally expensive. For relatively efficient computations, we turn to the master equation and find that it requires an additional calculation to account for non-equilibrium dynamics in the underlying myosin motor population. We discuss remaining questions and future directions in this ongoing work.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-25 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-25.html">
															    	</br><strong>Michael Norman</strong>
														    	 
														    		<em>North Carolina State University</em>
														    	
														    	
														    		
														    			</br>"Contraction and Connectivity in Simulated Cytoskeletal Networks"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The structure and mechanics of cytoskeletal networks are fundamental to cell morphology, migration and division. In this work, we develop methods to quantify the connectivity of fiber-motor networks and identify geometrical conditions that ensure network contraction through a mechanism known as polarity sorting. We  then  derived,  for such conditions, a theory that quantitatively predicts the rate of network contraction as a function of its connectivity and biochemical and physical parameters such as motor speed, binding rates, filament lengths and medium viscosity. Predictions are tested using the physics simulator CytoSim. Lastly we discuss how those outcomes are affected by the introduction of crosslinking proteins, which can increase connectivity but frustrate the contraction mechanisms.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-26 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-26.html">
															    	</br><strong>Elizabeth Diaz-Torres</strong>
														    	 
														    		<em>Center for Research and Advanced Studies</em>
														    	
														    	
														    		
														    			</br>"Cell recruitment may work as a temporal controller of size in the Drosophila wing"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			A fundamental question in developmental biology is how organs robustly attain a final size despite perturbations in cell growth and proliferation rates. Since organ growth is an exponential process driven mainly by cell proliferation, even small variations in cell proliferation rates, when integrated over a relatively long time, will lead to large differences in size, unless an intrinsic control mechanism compensates for these variations. Here, we use a mathematical model to propose the hypothesis that in the developing wing of Drosophila, cell recruitment, a process in which undifferentiated neighboring cells are incorporated or recruited into the wing primordium, determines the time in which growth is arrested in this system. As a consequence, perturbations in proliferation rates of wing-committed cells are compensated by an inversely proportional growth time to ensure a robust size of the wing. Furthermore, we show that growth control is lost when fluctuations in cell proliferation affects both wing-committed and recruitable cells. Our model suggests that cell recruitment may act as a temporal controller of growth to buffer fluctuations in cell proliferation rates and offers a plausible solution to a long-standing problem in the field.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-27 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-27.html">
															    	</br><strong>Ryan Godin</strong>
														    	 
														    		<em>Cleveland State University</em>
														    	
														    	
														    		
														    			</br>"Stripe Heterogeneity Affects Global Coordination of Oscillations in Synthetic Microbial Consortia"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Researchers recently utilized a sixteen delay-differential equations model to investigate globally-linked oscillations in two-strain, synthetic microbial consortia. Naturally, their model's complexity makes it challenging to comprehend and analyze, both analytically and numerically. In this presentation, we will discuss the work we have done towards developing a much simpler, non-dimensionalized model consisting of two diffusion equations based on one of the strain's underlying network topology. We will show that our model captures the consortia's qualitative behavior and is more suitable for analysis.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-28 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-28.html">
															    	</br><strong>Ariana Chriss</strong>
														    	 
														    		<em>Department of Biology, Geology and Environmental Sciences and Department of Mathematics and Statistics, Cleveland State University</em>
														    	
														    	
														    		
														    			</br>"Modeling Chromosome Dynamics During Prophase I of Meiosis"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			This study describes a mathematical model for dynamics between chromosomes in the cell nucleus, with a primary aim to predict matching times for homologous chromosomes. The pairing of homologous chromosomes during prophase I of meiosis allows for the exchange of genetic material and proper chromosome segregation during cell splitting. Hence, in order to elucidate meiotic defects that can lead to miscarriages or birth defects, it is crucial to understand this significant process. While homolog pairing can be monitored in the laboratory, the same cell cannot be followed for the duration of pairing. Cell samples die upon analysis, and thus different cells are evaluated at each timepoint. By simulating chromosome dynamics based on experimental data, we can track chromosome movement within one cell for the duration of pairing. Our agent-based model of chromosome dynamics involves capturing chromosome self-propulsion, collision dynamics, and thermal noise within the nucleus. The results are compared to the experimental data, and we observe the same pairing pattern. Our model validates the experimental method and strengthens the results. This model may then provide insight into the effects of mutations on pairing.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
													    		
																	<p><small class="card-body-subgroup">CDEV-29 (Session: PS04)</small>
															    <a href="/CDEV/CDEV-PS04-CDEV-29.html">
															    	</br><strong>Lachlan Elam</strong>
														    	 
														    		<em>Brandeis University</em>
														    	
														    	
														    		
														    			</br>"Diffusion on Dynamical Networks with Applications to Cell Biology"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The topic of research is the membranous networks of endoplasmic reticulum that transport proteins. The goal of the research is to develop accurate computational models to make inferences on the intended destination of a protein based on changes within the network. The morphology of these networks is a mysterious topic that only now may be uncovered with advances in technology and computing. One of the sub-goals is to accurately model the types of change that occur in these networks, through the discoveries made with testing on artificial networks. This is an important topic as it sheds light on the interactions that occur inside a cell and thus will help to understand how natural networks make decisions.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	    	
											  		</div>
													</p>
											  </div>
											</div>
										</div>
									</div>
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
					Click to show abstracts
									</span>		            
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
