<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Mathematical Oncology Subgroup (ONCO) | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Mathematical Oncology Subgroup (ONCO) | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Mathematical Oncology Subgroup (ONCO)" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/ONCO/" />
<meta property="og:url" content="http://localhost:4000/ONCO/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/ONCO/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Mathematical Oncology Subgroup (ONCO)","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Mathematical Oncology Subgroup (ONCO)">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Mathematical Oncology Subgroup (ONCO)" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Mathematical Oncology Subgroup (ONCO)" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
	    <div class="row">
	        <div class="col-md-12  text-center">
<!-- 		        <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
		        <h1 class="mb-3">Mathematical Oncology Subgroup (ONCO)</h1>
		        
<!-- 		        <div class="cleanslate w24tz-current-time w24tz-small" style="display: inline-block !important; visibility: hidden !important; min-width:200px !important; min-height:100px !important;"><p><a href="//24timezones.com/Tampa/time" style="text-decoration: none" class="clock24" id="tz24-1596739689-c1410-eyJob3VydHlwZSI6MTIsInNob3dkYXRlIjoiMSIsInNob3dzZWNvbmRzIjoiMCIsImNvbnRhaW5lcl9pZCI6ImNsb2NrX2Jsb2NrX2NiNWYyYzUwNjllNmUzMSIsInR5cGUiOiJkYiIsImxhbmciOiJlbiJ9" title="Tampa local time" target="_blank">US Eastern Time Zone (EDT)</a></p><div id="clock_block_cb5f2c5069e6e31"></div></div>
<script type="text/javascript" src="//w.24timezones.com/l.js" async></script>
	        	</br> -->
		        
		        
	        </div>
	    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			<h6>Jump to:</h6>
<ul class="pagination text-left">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#"><span class="text-ONCO">Minisymposia</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#CT"><span class="text-ONCO">Contributed Talks</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#Posters"><span class="text-ONCO">Posters</span></a></li>
</ul>

			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS1">Sub-group minisymposia</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS01.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS01-ONCO: </small></br>Mathematical approaches to advance clinical studies in oncology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Heyrim Cho (University of California Riverside, USA), Russell Rockne (City of Hope Comprehensive Cancer Center, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS01.html#author1"><strong class="card-body-subgroup">Hitesh Mistry</strong></a>
		    	 
		    		<em> (University of Manchester, UK)</em>
		    	
		    	
		    		
		    			</br>"Complexity/Simplicity of Oncology Pharmacodynamic Markers/Mathematical Models in the Clinic versus Drug Development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Pharmacodynamic markers provide information on what the drug is doing to the body, in this talk its a measure of what the drug is doing to the disease, cancer. The number and types of biomarkers in Oncology has increased dramatically over the last 20-30 years.  Our focus here will be on biomarkers that are used for selecting a dose/schedule. Many of these biomarkers are not typically used in the clinic but they do play a role in Oncology drug development. In this talk we shall compare the biomarkers/mathematical models in two Phase 1 Oncology trials, Rectal Carcinoma and metastatic Castrate Resistant Prostate Cancer, to those that are typically used in the clinic in the same settings. We shall highlight how the breadth and richness of data in Oncology drug development exceeds that in the clinic but that more complex mathematical models are used in the clinic versus drug development even though the question is the same - what dose/schedule should we use.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS01.html#author2"><strong class="card-body-subgroup">Renee Brady</strong></a>
		    	 
		    		<em> (H. Lee Moffitt Cancer Center and Research Institute, USA)</em>
		    	
		    	
		    		
		    			</br>"Predicting Response to Adaptive Therapy in Metastatic Prostate Cancer Using Prostate-Specific Antigen Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prostate cancer (PCa) remains the most prevalent cancer in men in the US. Standard treatment with androgen deprivation therapy (ADT) for localized disease often results in the competitive release of resistant cell phenotypes, causing patients to develop castration resistant PCa. Intermittent ADT has been shown to be a promising alternative to continuous treatment that can delay progression and may potentially reduce treatment-related adverse events. Second-line hormone therapy options, such as abiraterone acetate (AA), have been proven effective for metastatic castration resistant prostate cancer (mCRPC) and it has been proposed that similar to intermittent ADT, treatment with adaptive AA may reduce toxicity and prolong time to progression in mCRPC. We simulated and analyzed a simple quantitative model of prostate-specific antigen (PSA) dynamics to evaluate PCa stem cell enrichment as a plausible driver of treatment resistance. A Type 1b bootstrap internal validation leave-one-out analysis was used to calibrate and validate the model against longitudinal PSA data from 16 mCRPC patients receiving adaptive AA in a pilot study. Early PSA treatment response dynamics were then used to predict patient response to subsequent treatment. We extended the model to incorporate metastatic burden to improve predictive ability and also investigated the survival benefit of adding concurrent chemotherapy for patients predicted to become resistant. Model simulations demonstrated PCa stem cell self-renewal as a plausible driver of resistance to hormone therapy. The model was able to accurately describe patient-specific PSA dynamics and predict response with 78% accuracy. When incorporating metastatic burden, the predictive ability of the model increased to 81% (specificity = 92%, sensitivity = 50%). This study developed the first patient-specific mathematical model to use early treatment response dynamics to predict subsequent responses to adaptive AA.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS01.html#author3"><strong class="card-body-subgroup">Aleksandra Karolak</strong></a>
		    	 
		    		<em> (City of Hope Comprehensive Cancer Center, USA)</em>
		    	
		    	
		    		
		    			</br>"A Quantitative Systems Pharmacology Model to Improve Graft Versus Host Disease Outcomes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Allogeneic hematopoietic cell transplant (HCT) cures patients of underlying disease by replacing their hematopoietic system with that of a healthy donor (non-malignant disease) or by the donor cells eradicating the patient’s malignancy (graft-versus-tumor effect). A post-transplant cyclophosphamide (PTCy) regimen was recently established as a standard of care for preventing graft versus host disease (GVHD), which is the most common cause of non-relapse mortality in HCT. The PTCy regimen consists of three drugs: cyclophosphamide (Cy), mycophenolate mofetil (MMF, active metabolite mycophenolic acid - MPA) and tacrolimus (TAC). All three drugs need to be optimized: PTCy has a narrow dose range based on preclinical data; clinical data in other GVHD regimens suggest that low plasma exposure to MPA and TAC are associated with GVHD. To address this need, we are constructing a Quantitative Systems Pharmacology (QSP) model to optimize the PTCy regimen. Guided by our preliminary preclinical and clinical data, our hypothesis is that QSP modeling can successfully predict immunologic reactions resulting from PTCy to subsequently: 1) simulate alternative doses and administration schedules for all three drugs; 2) identify the optimal PTCy dose and administration schedule; and 3) identify which model parameters introduce the greatest variability to design subsequent clinical trials and obtain more data to improve model reliability. In order to achieve these aims, we combine computational modeling with experimental data from HCT patients to develop and validate mathematical approach. The novelty of our approach comes from a joint application of population pharmacokinetic (popPK) model with the fully integrated immune response model (FIRM). The hybrid popPK-FIRM QSP model uses patient-specific metabolite data of PTCy drugs activity to guide dosing optimization. Simulations help predict pharmacokinetic characteristics of the PTCy regimen with correlation to drugs’ metabolites and evaluate the effects of implicit drug-drug interactions. The progress on implementations of the mathematical models, results of the simulations, and validation with the human samples collected at City of Hope will be presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS01.html#author4"><strong class="card-body-subgroup">Kit Curtius</strong></a>
		    	 
		    		<em> (University of California San Diego, USA)</em>
		    	
		    	
		    		
		    			</br>"Predicting Risk of Progression to Advanced Neoplasia in Patients with Ulcerative Colitis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Patients with ulcerative colitis (UC) have an increased risk of developing colorectal cancer and thus are advised to participate in regular surveillance to remove pre-cancers that may be detected during colonoscopy.  In order to translate a validated statistical model for predicting patient-specific risk of progression over time, we developed UC-CaRE (Ulcerative Colitis-Cancer Risk Estimator) as a tool that can be used to calculate and communicate individualized cancer risk estimates to UC patients with low-grade dysplasia based on their clinicopathological features. This visual aid facilitates the risk stratification of the lowest risk patients, who can be reassured to continue surveillance, versus those at the highest risk of cancer who may benefit from preventive surgery. Using shallow whole genome sequencing, we also found that the evolution of copy number alterations in UC predicts future neoplastic risk in patients. As molecular-based decision-making becomes more prominent in the clinical setting of early cancer detection, we propose that models of evolving genotypes can be integrated into and will enhance tools like UC-CaRE.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS02.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS02-ONCO: </small></br>Mathematical approaches to advance clinical studies in oncology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Heyrim Cho (University of California Riverside, USA), Russell Rockne (City of Hope Comprehensive Cancer Center, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS01-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS02.html#author1"><strong class="card-body-subgroup">Jacob Scott</strong></a>
		    	 
		    		<em> (Cleveland Clinic, USA)</em>
		    	
		    	
		    		
		    			</br>"Evolutionary Control on Game Landscapes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Control of evolving populations has recently been postulated using control methods inspired by quantum computing and stochastic thermodynamics. These methods, which are essentially extensions of classical population genetics, require genotype-phenotype maps in the form of fitness seascapes, which are mapping from changes in drug dose to fitness in a combinatorially complete genotype space. These models rarely consider the interaction between individual types in heterogeneous populations (clonal interference) and are therefore of limited practical applicability. In this talk we will present a simplified deterministic (ODE) model of evolution on a landscape that includes these interactions (game landscape), show how the interactions can themselves drastically change the evolutionary dynamics, and sketch a path forward to evolutionary control.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS02.html#author2"><strong class="card-body-subgroup">Kristin Swanson</strong></a>
		    	 
		    		<em> (Mayo Clinic, USA)</em>
		    	
		    	
		    		
		    			</br>"Sex, Drugs and Radiomics of Brain Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS02.html#author3"><strong class="card-body-subgroup">Sebastien Benzekry</strong></a>
		    	 
		    		<em> (INRIA, France)</em>
		    	
		    	
		    		
		    			</br>"Quantitative modeling of metastasis: cancer at the organism scale"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In the majority of solid cancers, secondary tumors (metastases) are the main cause of death. Determining the burden of invisible metastases at diagnosis is a crucial challenge in the clinic, as it would allow personalization of therapeutic intervention, e.g. in the perioperative setting. I will present research efforts towards the establishment of such a predictive computational tools of metastatic development, with emphasis on the quantitative calibration of models to empirical data (experimental and clinical). The general framework is based on a physiologically-structured partial differential equation for the time dynamics of a population of metastases. Results will be presented in two clinical settings: brain metastasis from non-small cell lung cancer and  early-stage breast cancer. In the first application, comparison of models relying on different biological hypotheses about dissemination and growth indicated periods of dormancy of the order of several months. In the second application, a combination of machine learning techniques and mixed-effects statistical modeling methods was used for individualized predictions of the model parameters from data available at diagnosis. In turn, this allowed patient-specific prediction of the time to metastatic relapse. Together, these results represent a step towards the integration of mathematical modeling as a predictive tool for personalized oncology.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS02.html#author4"><strong class="card-body-subgroup">Chengyue Wu</strong></a>
		    	 
		    		<em> (University of Texas at Austin, USA)</em>
		    	
		    	
		    		
		    			</br>"Towards patient-specific prediction of breast cancer response to neoadjuvant therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Neoadjuvant therapy (NAT) has become the standard-of-care treatment for breast cancers. However, more than 50% of patients undergoing the standard NAT regimen show residual tumors which are associated with metastasis and recurrence. Patient-tailored treatment has been proposed to improve individual response. But with multiple factors to consider, including dose, schedule, and drug combinations, personalization of therapeutic regimens is a complex task which cannot be solved by population-based clinical trials. To address this problem, we develop a clinical-computational framework to systematically evaluate the response of breast cancer patients to different therapeutic regimens. Specifically, we employ quantitative MRI to measure tissue geometries and properties such as vessel permeability and drug diffusivity. Constrained by the patient-specific data, we establish a model consisting of an advection-diffusion equation for flow and drug transport, and a phase-filed equation for tumor growth and response. For each patient, we simulate a group of practical therapeutic regimens by varying administration schedules and doses, and drug combinations. The outcome of each regimen is assessed by the computed tumor cellularity and off-target ratio (accumulative drug outside tumor to that within tumor) at the end of treatment. Preliminary results indicate that the approach has the potential to personally optimize breast cancer NAT.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS03.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS03-ONCO: </small></br>Systems Biology Models of Tumor Metabolism</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Shubham Tripathi (Rice University, USA), Abhinav Achreja (University of Michigan, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS03.html#author1"><strong class="card-body-subgroup">Dongya Jia</strong></a>
		    	 
		    		<em> (Laboratory of Integrative Cancer Immunology, National Cancer Institute, National Institutes of Health, USA)</em>
		    	
		    	
		    		
		    			</br>"Elucidating cancer catabolism and anabolism by coupling gene regulation with metabolic pathways"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer cells can adapt their metabolic phenotypes to meet various bioenergetic and biosynthetic needs, and to survive the therapeutic treatments. It remains largely unclear how cancer cells orchestrate different metabolic phenotypes (glycolysis, oxidative phosphorylation etc.) and various metabolic ingredients (glucose, fatty acids, glutamine, etc.). Since recent efforts in targeting individual cancer metabolic pathways have been largely ineffective, a better understanding of cancer metabolic network and its plasticity will progressively facilitate the development of more effective therapeutic strategies. The goal of this study is to elucidate the mechanisms underlying cancer metabolic plasticity within both catabolism and anabolism by integrated theoretical-experimental approaches. We constructed a metabolic modeling framework featuring regulation by the master gene regulators (AMPK, HIF-1, MYC etc.) and their cross-talk with metabolic pathways. The beauty of the framework is at least two-fold. First, it has considered all three most important metabolic ingredients (glucose, fatty acids, glutamine) for tumorigenesis and metastasis. Second, it has allowed us to investigate the interaction between catabolism (glucose/glutamine oxidation, etc.) and anabolism (reductive glucose/glutamine metabolism), therefore offering a higher-level view of cancer metabolism. Our work elucidates how cancer cells can mix and match different metabolic phenotypes. For example, we show that cancer cells can acquire a hybrid metabolic phenotype where both glycolysis and OXPHOS are actively used, and a metabolically “low-low” phenotype where cells exhibit low activity of glycolysis and OXPHOS. Importantly, the hybrid metabolic phenotype characterizes highly metastatic breast cancer cells and the low-low phenotype can characterize drug-tolerant melanoma cells. Consequently, an accurate characterization of cancer metabolism enabled us to present effective combination therapies targeting metabolism in breast cancer.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS03.html#author2"><strong class="card-body-subgroup">Prahlad Ram</strong></a>
		    	 
		    		<em> (The University of Texas MD Anderson Cancer Center, USA)</em>
		    	
		    	
		    		
		    			</br>"4D Ex-vivo CRISPR / CAS9 Whole-genome Screen to Identify Genes Regulating Early Lung Cancer Metastasis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Metastatic lung cancer has a 5-year survival rate of 5%. Lung cancers tend to be asymptomatic until late stages, and almost 90% are not diagnosed until they are advanced. The genomic events early in the metastatic process has not been completely deciphered. Utilizing CRISPR/Cas9 whole genome knockout screen in the A549 lung adenocarcinoma cell line and coupling it with a novel ex vivo 4D lung metastasis model has now allowed us to examine early genomic events in metastasis. Using this approach we recovered genes previously implicated in lung cancer and metastasis validating this approach. Additionally we identified a transcription factor network driven by SPI1 which was enriched in our screen. Experimental validation of SPI1 uncovered a novel role of this network in the metastatic process.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS03.html#author3"><strong class="card-body-subgroup">Andrew Raddatz</strong></a>
		    	 
		    		<em> (The Wallace H. Coulter Department of Biomedical Engineering, Georgia Institute of Technology and Emory University, USA)</em>
		    	
		    	
		    		
		    			</br>"Kinetic Modeling of Redox Metabolism in Head and Neck Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Reactive oxygen species (ROS) levels are frequently elevated in head and neck tumors because of downstream tumor-promoting outcomes. Moderate levels of ROS promote tumorigenesis because they increase proliferation, initiate angiogenesis, and trigger survival signaling pathways. Additionally, treatment options such as radiation, chemotherapy, and even immunotherapy have been shown to involve tumor redox biology. A greater mechanistic understanding of how redox-based expression profiles in cancer affect susceptibility to certain treatments is needed to improve clinical decisions. Here, we developed an intracellular ODE model to represent how a cancer cell’s redox state would respond to treatment with a ROS-generating drug. The following antioxidant systems were included in the model based on previous H2O2 clearance modeling: catalase, peroxiredoxin, glutathione, and the protein thiol pool. Initial parameterization of the model included taking values reported in the literature and scanning the BRENDA database for remaining rate constants where available. To validate our model, we experimentally silenced antioxidant enzymes represented in the model by siRNA and observed the effect on production of H2O2. We found that knocking down PRDX1 (peroxiredoxin 1), CAT (catalase), and TXNRD1 (thioredoxin reductase 1) via siRNA led to a relative increase in extracellular H2O2 upon drug application. Then, using scRNA-seq data, we generated single cell models to predict how transcriptome variability across patients and within tumors can influence ROS accumulation and redox potentials within the cell under drug treatment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS03.html#author4"><strong class="card-body-subgroup">Stacey Finley</strong></a>
		    	 
		    		<em> (University of Southern California, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling tumor-stromal metabolic crosstalk in colorectal cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Colorectal cancer (CRC) is a major cause of morbidity and mortality in the United States. Tumor-stromal metabolic crosstalk in the tumor microenvironment promotes CRC development and progression, but exactly how stromal cells, in particular cancer-associated fibroblasts (CAFs), affect the metabolism of tumor cells remains unknown. Here we take a data-driven approach to investigate the metabolic interactions between CRC cells and CAFs, integrating constraint-based modeling and metabolomic profiling. Using metabolomics data, we perform unsteady-state parsimonious flux balance analysis to infer flux distributions for central carbon metabolism in CRC cells treated with or without CAF-conditioned media. We find that CAFs reprogram CRC metabolism through stimulation of glycolysis, the oxidative arm of the pentose phosphate pathway (PPP), and glutaminolysis as well as inhibition of the tricarboxylic acid cycle. To identify potential therapeutic targets, we simulate enzyme knockouts and find that inhibiting the hexokinase and glucose-6-phosphate dehydrogenase reactions exploits the CAF- induced dependence of CRC cells on glycolysis and oxidative PPP. Our work gives mechanistic insights into the metabolic interactions between CRC cells and CAFs and provides a framework for testing hypotheses towards CRC-targeted therapies.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS06.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS06-ONCO: </small></br>Blackboard to Bedside: Showcase of Translational Modeling</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Renee Brady-Nicholls (Moffitt Cancer Center, USA), Mohammad Zahid (Moffitt Cancer Center, USA), Stefano Pasetto (Moffitt Cancer Center, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS06.html#author1"><strong class="card-body-subgroup">Rene Bruno</strong></a>
		    	 
		    		<em> (Genentech-Roche, France)</em>
		    	
		    	
		    		
		    			</br>"Tumor dynamic modeling and overall survival predictions to support decisions in oncology clinical trials"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The key endpoints to support treatment approval in oncology and particularly for the treatment of advanced diseases is overall survival (OS). However, decisions to move to pivotal trials have to be made using earlier endpoints like overall response rate (ORR) or progression free survival (PFS) that often poorly predict OS and probability of success of a pivotal Phase III trial particularly with immunotherapies. Longitudinal tumor dynamic models estimate treatment effect on tumor growth inhibition (TGI)) and are linked to OS (TGI-OS models) in treatment independent biomarker-outcome models to offer a quantitative model-based approach that fully leverage to data generated in early trials. The use of TGI-OS models to simulate Phase III studies outcome and support early decisions will be illustrated (Bruno et al, Clin Cancer Res 2020;26:1787–95).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS06.html#author2"><strong class="card-body-subgroup">Pamela Jackson</strong></a>
		    	 
		    		<em> (Mayo Clinic, USA)</em>
		    	
		    	
		    		
		    			</br>"Instantiating an Imaging Digital Twin for a Brain Tumor Patient"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In medicine, digital twins are computational representations of some aspect of an individual patient and their disease.  An effective digital twin can incorporate mathematical models to recapitulate the patient’s current disease state and predict the individual patient’s response to a therapeutic intervention, such that multiple interventions can be tested on the twin prior to selecting the most effective therapy. For brain tumors specifically, clinical imaging will be an important part of any digital twin given the eloquent nature of the brain and the integral part imaging plays in identifying suspected brain tumors and determining response to therapy. Thus, an imaging digital twin that can capture the dynamic visualization of the disease will be critical for comparison to actual patient images. Before the dynamics of the disease can be captured, we must first instantiate the simulated version of a patient’s imaging for the pre-treatment timepoint. Our objective is to demonstrate the identification of an imaging digital twin for an individual patient’s brain tumor at the pretreatment time-point using a brain tumor growth mathematical model coupled to an imaging simulation utilizing MRI physics. To instantiate the imaging digital twin, we generated multiple candidate brain tumors and their associated simulated images using the Proliferation-Invasion-Hypoxia-Necrosis-Angiogenesis-Edema (PIHNA-E) model coupled to an MRI signal model [1,2]. Using the PIHNA-E model [1] incorporating the patient’s imaging-based anatomy, we created twenty-five phantoms based on unique combinations of 5 different rates of migration (D [mm2/year]) and 5 different rates of proliferation (ρ [1/year])]. These patient-specific PIHNA-E simulations were then passed into an MRI signal model for simulating corresponding T2-weighted MRIs [2]. We then compared the acquired patient image to the candidate simulations with various combinations of D and ρ.  To identify a “close” matching image, we calculated the L2-norm of twelve statistical features for both the acquired patient image and the simulated candidate images. The D and ρ of the acquired image with the lowest L2-norm relative to the candidate image was selected as the predictive parameter set.  Additionally, we examined the effect of noise on the selection process. We were able to both create patient-specific simulated MRIs and select parameters for the PIHNA-E brain tumor growth model. [1] A. Hawkins-Daarud, R. C. Rockne, A. R. A. Anderson, and K. R. Swanson. 'Modeling tumor-associated edema in gliomas during anti-angiogenic therapy and its impact on imageable tumor.' Frontiers in oncology 3:66, 2013. [2] P.R. Jackson, A. Hawkins-Daarud, S. C. Partridge, P. E. Kinahan, and K. R. Swanson. 'Simulating magnetic resonance images based on a model of tumor growth incorporating microenvironment.' Medical Imaging 2018: Image Perception, Observer Performance, and Technology Assessment, International Society for Optics and Photonics 10577:105771D, 2018.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS06.html#author3"><strong class="card-body-subgroup">Elsa Hansen</strong></a>
		    	 
		    		<em> (Penn State Huck Institutes of the Life Sciences, USA)</em>
		    	
		    	
		    		
		    			</br>"Maintenance therapy: A case study in trial design"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Treatment efficacy is often measured in terms of progression free survival (PFS) or tumor response. Viewing cancer treatment from the perspective of resistance management changes how we interpret these measures. I will discuss these issues in the context of a recent clinical trial of maintenance therapy for multiple myeloma.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS06.html#author4"><strong class="card-body-subgroup">Sarah Brüningk</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Intermittent radiotherapy as alternative treatment for recurrent high grade glioma: A modeling study based on longitudinal tumor measurements"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Treatment options for recurrent high grade glioma are greatly limited and non-curative. Radiotherapy (RT) is an integral part of palliative patient care. A recent phase I clinical trial (NCT02313272) recently demonstrated the safety of a combination treatment of high dose hypofractionated stereotactic radiotherapy (HFSRT, ≥ 6 Gyx5 in daily fractions) with pembrolizumab (immuno therapy; anti PD1 antibody) and bevacizumab (aiming at vasculature normalization). In this presentation we show a simulation study of intermittent RT (iRT, delivering RT fractions in intervals of several weeks) suggested as a personalized treatment strategy to prolong tumor control rather than using debulking HFSRT. Simu- lations were performed using a mathematical model of tumor growth, radiation response and patient-specific evolution of resistance to additional treatments (pembrolizumab and bevacizumab). Four models comprising different levels of patient specific parameters were fitted from tumor growth curves of 16 patients enrolled in the NCT02313272 trial. The model ranking highest based on the Akaike information criterion was used for simulation of iRT and iRT plus boost (≥ 6 Gyx3 in daily fractions at time of progression) schedules for varying numbers of treatment fractions and time between fractions. Kapalan Meier curves scoring time to progression beyond the initial tumour volume were used to com- pare treatments. We show that iRT+boost(-boost) treatment was equal or superior to HFSRT in 15(11) out of 16 cases and that patients that remained responsive to pem- brolizumab and bevacizumab would benefit most from iRT. Time to progression could be prolonged through the application of additional, intermittently delivered fractions. iRT hence provides a promising treatment option for recurrent high grade glioma patients.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS07.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS07-ONCO: </small></br>Modeling translational oncology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Russell Rockne (Beckman Research Institute, City of Hope National Medical Center, USA), Andrea Bild (Beckman Research Institute, City of Hope National Medical Center, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS07.html#author1"><strong class="card-body-subgroup">Jessica Leete</strong></a>
		    	 
		    		<em> (Pfizer Inc, Cambridge MA, USA)</em>
		    	
		    	
		    		
		    			</br>"Towards virtual populations for human efficacy prediction in lung cancer: preclinical to clinical translation of anti-PD-(L)1 treatments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Objectives: Immune checkpoint inhibitors such as anti-PD-1 and anti-PD-L1 are promising new therapeutic options that cease tumor cells’ immunosuppressive properties; however, low response rates to these therapeutics indicate an unmet medical need in treating non-small cell lung cancer (NSCLC). NSCLC is a highly heterogenous disease both spatially and genetically, and provides unique challenges to predicting treatment outcome.
We propose a model of anti-PD-(L)1 in both preclinical mouse models and human NSCLC. We use this model to explore the effect of parameters on model outcome in preparation for implementation of virtual population methods to explore inter-patient variability.
Methods: We present a drug and target focused model of anti-PD-(L)1 treatment in CT26 BALB/c mice and NSCLC in humans. The model includes the mechanisms of action of anti-PD-1 and anti-PD-L1. We translate the model structure to that of a 'typical' NSCLC patient using published popPK and binding affinities of approved anti-PD-(L)1 treatments. Parameter sensitivity is explored through both local and global sensitivity analyses.
Results: Pre-clinical simulations show the model's ability to match a wide variety of responses to treatment by allowing tumor growth parameters and tumor PD-1+ CD8+ T cell concentration to vary between individuals. Model simulations are sensitive to parameters that determine PD-(L)1 abundance and the speed and magnitude of T cell proliferation after treatment.
Conclusions: Focus on modeling a 'typical' human patient may lead to an incomplete characterization of treatment efficacy in situations where there is high inter-patient variability in responses. Immuno-oncology treatments in particular may benefit from methods that can quantify the effect of patient variability on treatment outcome.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS07.html#author2"><strong class="card-body-subgroup">Nataly Kravchenko-Balasha</strong></a>
		    	 
		    		<em> (The Institute of Biomedical and Oral Research, The Hebrew University of Jerusalem, Israel, Israel)</em>
		    	
		    	
		    		
		    			</br>"Computational quantification and characterization of independently evolving cellular subpopulations within tumors is critical to inhibit anti-cancer therapy resistance"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Drug resistance continues to be a principle limiting factor across diverse anti-cancer therapies. Contributing to the complexity of this challenge is cancer plasticity where one cancer subtype switches to another in response to treatment (e.g. Triple Negative Breast Cancer (TNBC) to Her2-positive breast cancer). For optimal treatment outcomes, accurate tumor diagnosis and subsequent therapeutic decisions are vital. In this study an information-theoretic single-cell quantification strategy was developed to provide a high resolution and individualized assessment of tumor composition for a customized treatment approach. Briefly, this single-cell quantification strategy computes a barcode based on at least 100, 000 tumor cells and reveals a set of ongoing processes in each cell. Using these cell-specific barcodes, distinct subpopulations evolving within the tumor in response to an outside influence (e.g. anticancer treatments) are revealed and mapped. Barcodes, are further applied to assign targeted drug combinations to each individual tumor to optimize tumor response to therapy. This unique strategy was validated using TNBC models and patient-derived tumors known to switch phenotypes in response to radiotherapy (RT). We show that a barcode-guided targeted cocktail significantly enhances tumor response to RT and prevents regrowth of once resistant tumors. The strategy presented herein has the potential to significantly reduce the occurrence of cancer treatment resistance, with a broad applicability in clinical use.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS07.html#author3"><strong class="card-body-subgroup">Alexander R. A. Anderson</strong></a>
		    	 
		    		<em> (Integrated Mathematical Oncology Department H. Lee Moffitt Cancer Center & Research Institute, USA)</em>
		    	
		    	
		    		
		    			</br>"Exploiting evolution to design better cancer therapies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Our current approach to cancer treatment has been largely driven by finding molecular targets, those patients fortunate enough to have a targetable mutation will receive a fixed treatment schedule designed to deliver the maximum tolerated dose (MTD). These therapies generally achieve impressive short-term responses, that unfortunately give way to treatment resistance and tumor relapse. The importance of evolution during both tumor progression, metastasis and treatment response is becoming more widely accepted.  However, MTD treatment strategies continue to dominate the precision oncology landscape and ignore the fact that treatments drive the evolution of resistance.  Here we present an integrated theoretical/experimental/clinical approach to develop treatment strategies that specifically embrace cancer evolution. We will consider the importance of using treatment response as a critical driver of subsequent treatment decisions, rather than fixed strategies that ignore it. We will also consider using mathematical models to drive treatment decisions based on limited clinical data. Through the integrated application of mathematical and experimental models as well as clinical data we will illustrate that, evolutionary therapy can drive either tumor control or extinction using a combination of drug treatments and drug holidays. Our results strongly indicate that the future of precision medicine shouldn’t be in the development of new drugs but rather in the smarter evolutionary, and model informed, application of preexisting ones.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS07.html#author4"><strong class="card-body-subgroup">Andrew Gentles</strong></a>
		    	 
		    		<em> (Departments of Medicine and Biomedical Informatics, Stanford University, USA)</em>
		    	
		    	
		    		
		    			</br>"Building an atlas of cell states and cellular ecosystems across human solid tumors"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Determining how cells vary with their local signaling environment and organize into distinct cellular communities is critical for understanding processes as diverse as development, aging, and cancer. We have developed EcoTyper, a new machine learning framework for large-scale identification and validation of cell states and multicellular communities from bulk, single-cell, and spatially-resolved gene expression data. When applied to 12 major cell lineages across nearly 6,000 tumor specimens from 16 types of human carcinoma, EcoTyper identified 69 transcriptionally-defined cell states. Most cell states were specific to neoplastic tissue, ubiquitous across tumor types, and significantly prognostic. By analyzing cell state co-occurrence patterns, we discovered 10 clinically-distinct multicellular communities with unexpectedly strong conservation, including four with unique myeloid and stromal elements, one enriched in normal tissue, and two associated with early cancer development. This work elucidates fundamental units of cellular organization in human carcinoma and provides a framework for large-scale profiling of cellular ecosystems in any tissue.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS08.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS08-ONCO: </small></br>Tumor-Immune Dynamics and Oncolytic Virotherapy</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Lisette dePillis (Department of Mathematics, Harvey Mudd College, United States), Amina Eladdadi (Department of Mathematics, The College of St. Rose, United States)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS08.html#author1"><strong class="card-body-subgroup">Raluca Eftime</strong></a>
		    	 
		    		<em> (University of Franche-Comté, France)</em>
		    	
		    	
		    		
		    			</br>"Modelling oncolytic virotherapies for cancer: the complex roles of innate immune responses"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Oncolytic viruses are emerging as important approaches in cancer treatment. However, the effectiveness of these therapies depends significantly on the interactions between the oncolytic viruses and the host immune response. Macrophages are one of the most important cell types in the anti-viral immune responses, as well as in the anti-cancer immune responses. Nevertheless, the heterogeneity of macrophage population (with the two extreme phenotypes represented by the M1 and M2 cells) makes it difficult to understand the anti-cancer as well as anti-viral roles of these cells. We start by focusing on a single-scale model for oncolytic virus--cancer cell interactions in the presence of immune responses represented by macrophages. We show that cell polarization towards either an M1 or M2 phenotype can enhance oncolytic virus therapy through either (i) anti-tumour immune activation, or (ii) enhanced oncolysis. Then, we discuss the impact of the spatial spread of macrophages inside solid tumours on the heterogeneous spatial distributions of oncolytic viruses.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS08.html#author2"><strong class="card-body-subgroup">Justin Le Sauteur</strong></a>
		    	 
		    		<em> (University of Montreal, Canada)</em>
		    	
		    	
		    		
		    			</br>"Optimizing combined oncolytic vaccinia and PAC-1 treatment of ovarian cancer using in silico clinical trials"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Ovarian cancer poses a unique challenge due to its late diagnosis and high rate of relapse. In response, oncolytic vaccinia virus (VACV), which selectively kills tumour cells through infection and viral replication, and  procaspase-activating compound 1 (PAC-1), a small tumour cell apoptosis-inducing molecule, have been recently proposed as a combination therapy that may better control ovarian cancer growth. The combination of VACV and PAC-1 has already been shown to be a promising treatment, however a delicate therapeutic balance must be stuck, as PAC-1 induces apoptosis in cells that VACV needs for continued replication. To provide a quantitative basis behind the use of VACV with PAC-1 in ovarian cancer, we developed a mathematical and computational biology model that accounts for tumour growth and treatment-induced death. Our model was calibrated to experimental measurements of the individual and combined effects of each molecule. To determine the optimal dose size and therapeutic schedule for combined VACV and PAC-1, we expanded an in silico clinical trial of 200 patients to bolster the preclinical translation of this investigational therapy. Our results contribute to the evaluation of the validity of this proposed treatment, and establish maximal PAC-1 concentrations that maintain VACV efficacy. Overall, this work demonstrates the ability to use simple mathematical modelling techniques to inform treatment design in real time.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS08.html#author3"><strong class="card-body-subgroup">Pantea Pooladvand</strong></a>
		    	 
		    		<em> (The University of Sydney, Australia)</em>
		    	
		    	
		    		
		    			</br>"The dynamics of oncolytic virotherapy in dense tumours"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The growth of a tumour can be characterised by a complex network of cells, fibers and molecules. Images of tumour histology show that cell-stroma landscapes can vastly differ from one tumour to another. These variations in structure, density and cell placement inevitably change the outcome of treatment.
Mathematical studies often focus on modelling the degradation and reconstruction of extracellular matrix (ECM) by tumour cells to capture tumour progression. However, the extracellular matrix can also significantly hinder anti-cancer therapy.
In this project we explore the role of ECM differently. Here, we focus on how changes in stroma affect oncolytic virotherapy. We want to understand how different configurations of tumour-ECM landscape change the spread and efficacy of viral treatment. By building a system of partial differential equations that includes a novel diffusion term for virus spread in ECM, we look for patterns in tumour-cell ratios, collagen density and collagen configurations to predict treatment outcome. We find that collagen density, cell-collagen ratio and gaps in the collagen surface can significantly affect tumour treatment. Therefore, to accurately describe treatment outcome in oncolytic virotherapy, models need to consider the influence of cell-collagen interactions on therapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS08.html#author4"><strong class="card-body-subgroup">Khaphetsi J. Mahasa</strong></a>
		    	 
		    		<em> (National University of Lesotho, Lesotho)</em>
		    	
		    	
		    		
		    			</br>"Natural killer cells recruitment in oncolytic virotherapy: a mathematical model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk, we investigate how natural killer (NK) cell recruitment to the tumor microenvironment (TME) affects oncolytic virotherapy. NK cells play a major role against viral infections. They are, however, known to induce early viral clearance of oncolytic viruses, which hinders the overall efficacy of oncolytic virotherapy. Here, we formulate and analyze a simple mathematical model of the dynamics of the tumor, OV and NK cells using currently available preclinical information. The aim of this study is to characterize conditions under which the synergistic balance between OV-induced NK responses and required viral cytopathicity may or may not result in a successful treatment. In this study, we found that NK cell recruitment to the TME must take place neither too early nor too late in the course of OV infection so that treatment will be successful. NK cell responses are most influential at either early (partly because of rapid response of NK cells to viral infections or antigens) or later (partly because of antitumoral ability of NK cells) stages of oncolytic virotherapy. The model also predicts that: (a) an NK cell response augments oncolytic virotherapy only if viral cytopathicity is weak; (b) the recruitment of NK cells modulates tumor growth; and (c) the depletion of activated NK cells within the TME enhances the probability of tumor escape in oncolytic virotherapy. Taken together, our model results demonstrate that OV infection is crucial, not just to cytoreduce tumor burden, but also to induce the stronger NK cell response necessary to achieve complete or at least partial tumor remission. Furthermore, our modeling framework supports combination therapies involving NK cells and OV which are currently used in oncolytic immunovirotherapy to treat several cancer types.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS09.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS09-ONCO: </small></br> Recent development in mathematical oncology in Asia and Australia</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yangjin Kim (Konkuk University, Korea, Republic of), Eunjung Kim (Korea Institute of Science and Technology, Korea)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS15-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS09.html#author1"><strong class="card-body-subgroup">Shinji Nakaoka</strong></a>
		    	 
		    		<em> (Faculty of Advanced Life Science, Hokkaido University, Japan)</em>
		    	
		    	
		    		
		    			</br>"A computational pseudo-tracking method for cancer progression by microbiome data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this presentation, we would like to present recent research progress on applying a pseudotime reconstruction method to microbiome data. Pseudotime reconstruction methods have been originally developed in the field of single-cell RNA-seq analysis. Pseudotime reconstruction is also known as trajectory inference, which utilizes many samples to infer a developmental path such as cell differentiation, from a non-time series dataset. Although the validity of applying pseudotime reconstruction methods to microbiome data is not confirmed, the potential of its usefulness has been demonstrated on some datasets. In our ongoing work, we have been trying to apply a pseudotime reconstruction method to microbiome data obtained from patients who are diagnosed with some cancer. In this presentation, we will report a summary of computational results for the comparison of different pseudotime reconstruction methods to infer a possible trajectory of cancer progression.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS09.html#author2"><strong class="card-body-subgroup">Aurelio A. de Los Reyes V</strong></a>
		    	 
		    		<em> (University of the Philippines Diliman, Philippines)</em>
		    	
		    	
		    		
		    			</br>"Polytherapeutic strategies in cancer treatment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This study aims to identify strategic infusion protocols of bortezomib, OV and natural killer (NK) cells to minimize cancer cells by utilizing optimal control theory. Three different therapeutic protocols will be presented: (i) periodic bortezomib and single administrations of both OV and NK cells therapy; (ii) alternating sequential combination therapy; and (iii) NK cell depletion and infusion therapy. The first treatment strategy shows that early OV administration followed by well-timed adjuvant NK cell infusion maximizes antitumour efficacy and the second scheme supports timely OV infusion. The last treatment protocol  indicates that transient NK cell depletion followed by appropriate NK cell adjuvant therapy yields the maximal benefits. This study could provide potential combination therapies in cancer treatment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS09.html#author3"><strong class="card-body-subgroup">Eunjung Kim</strong></a>
		    	 
		    		<em> (Korea Institute of Science and Technology, Korea)</em>
		    	
		    	
		    		
		    			</br>"Understanding the potential benefits of adaptive therapy for metastatic melanoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the potential benefits of adaptive therapy for metastatic melanoma Adaptive therapy is an evolution-based treatment approach that aims to maintain tumor volume by employing minimum effective drug doses or timed drug holidays. For successful adaptive therapy outcomes, it is critical to find the optimal timing of treatment switch points. Mathematical models are ideal tools to facilitate adaptive therapy dosing and switch time points. We developed two different mathematical models to examine interactions between drug-sensitive and resistant cells in a tumor. The first model assumes genetically fixed drug-sensitive and resistant populations that compete for limited resources. Resistant cell growth is inhibited by sensitive cells. The second model considers phenotypic switching between drug-sensitive and resistant cells. We calibrated each model to fit melanoma patient biomarker changes over time and predicted patient-specific adaptive therapy schedules. Overall, the models predict that adaptive therapy would have delayed time to progression by 6-25 months compared to continuous therapy with dose rates of 6%-74% relative to continuous therapy. We identified predictive factors driving the clinical time gained by adaptive therapy. The first model predicts 6-20 months gained from continuous therapy when the initial population of sensitive cells is large enough, and when the sensitive cells have a large competitive effect on resistant cells. The second model predicts 20-25 months gained from continuous therapy when the switching rate from resistant to sensitive cells is high and the growth rate of sensitive cells is low. This study highlights that there is a range of potential patient specific benefits of adaptive therapy, depending on the underlying mechanism of resistance, and identifies tumor specific parameters that modulate this benefit.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS09.html#author4"><strong class="card-body-subgroup">Masud MA</strong></a>
		    	 
		    		<em> (Korea Institute of Science and Technology, Korea)</em>
		    	
		    	
		    		
		    			</br>"The impact of spatial heterogeneity on treatment response"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A long-standing practice in cancer treatment is hit hard with maximum tolerated dose to eradicate the tumor. Such continuous therapy, however, selects for resistance cells leading to treatment failure.  A different type of treatment strategy, adaptive therapy, has recently shown a degree of success in both preclinical xenograft experiments and clinical trials. Adaptive therapy aims to maintain tumor volume by exploiting the competition between drug-sensitive and resistance cells with minimum effective drug doses or timed drug holidays.  To further understand the role of spatial competition between cancer cells, we develop a 2D on-lattice agent-based model. Specifically, we address the role of resistant cell distribution on the treatment outcomes. Our simulations show that the superiority of adaptive strategy over continuous therapy depends on the local competition shaped by the spatial distribution of resistant cells. Cancer cell migration and increased carrying capacity drive a faster tumor progression time under both types of treatment by reducing local competition. The intratumor competition can be modulated by fibroblasts, which produce microenvironmental factors that promote cancer cell growth. Our simulations show that the spatial architecture of fibroblasts modulates treatment outcomes. As proof of concept, we simulate adaptive therapy outcomes on multiple metastatic sites composed of different spatial distributions of fibroblasts and drug resistance cell populations. We predict that spatial distribution of resistance cells and fibroblasts metastatic lesions modulate the benefit of adaptive therapy.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS11.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS11-ONCO: </small></br>Mathematical Oncology: From methodological studies to clinical applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Saskia Haupt (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Vincent Heuveline (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Matthias Kloor (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Germany)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS11.html#author1"><strong class="card-body-subgroup">Calum Gabbutt</strong></a>
		    	 
		    		<em> (Centre for Genomics and Computational Biology, Barts Cancer Institute, Queen Mary University of London, London, United Kingdom (UK))</em>
		    	
		    	
		    		
		    			</br>"Reconstructing Contemporary Human Stem Cell Dynamics with Oscillatory Molecular Clocks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell histories can be reconstructed from their genomes by analysing ‘molecular clocks’ that accumulate heritable changes through time. Commonly used clocks, such as the accumulation of single nucleotide variants, change slowly over decades, recording cell dynamics that occur at the longer timescale of the change accumulation rate. Studies within mouse have revealed that normal colon epithelium is maintained by a pool of multipotent stem-cells which undergo neutral competition and inevitably drift towards monoclonality. Here we develop a new method that can measure contemporary human adult cell dynamics with rapidly oscillating CpG DNA methylation.
Ongoing (de)methylation causes switching between 0, 50 and 100% methylation at each CpG locus in a diploid cell – the clock ‘tick-tocks’ back-and-forth like a pendulum. In polyclonal cell populations, these oscillator states are unsynchronized between cells, hence the average oscillator methylation is randomly distributed about 50%. However, any clonal expansion will synchronize the oscillator clocks resulting in clonal populations that have characteristic “W-shaped” distributions (methylation peaks at 0, 50 and 100%), approximating the methylation of the progenitor cell. The precise shape of the W-distribution is determined by the underlying dynamics of cell growth and replacement. We show how to identify appropriate oscillators from standard methylation array data (Illumina EPIC) and develop a mathematical modelling framework to quantitatively measure stem cell dynamics from these data. We apply our method to measure stem cell dynamics in individual human intestinal crypt and endometrial gland populations, and test whether these tissues have different stem cell dynamics using a hierarchical Bayesian model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS11.html#author2"><strong class="card-body-subgroup">Toni Seppälä</strong></a>
		    	 
		    		<em> (Helsinki University Hospital and University of Helsinki, Finland)</em>
		    	
		    	
		    		
		    			</br>"Organoids and cell-free DNA in cancer precision medicine"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is generally believed that earlier diagnosis of a cancer recurrence might improve the outcome. Postoperative minimal residual disease (MRD) very deterministically predicts future recurrence after curative surgery, but the preliminary evidence suggests that the prognosis of a recurring cancer may be improved by timely chemotherapy. Patients are always followed up for years to detect cancer recurrences using clinical examinations and blood tests that are not optimal by sensitivity or specificity. In case of a recurrence seen in imaging, chemotherapy is usually initiated. Selection of chemotherapy regimen between multiple options is usually based on expected tolerability of toxicity and failure of earlier choices.
Tools to aid decision-making in these challenging clinical situations are required, and precision medicine holds great promise in delivering for the unmet need. Applications detecting bloodstream cell-free DNA have been developed to support diagnostics of MRD. Patient-derived organoid technology enables individualized cell culture from each tumor. Organoids may serve as a clinical tool to guide traditional primary tumor NGS, and facilitate in vitro response prediction to therapy. Data-intensive models for tumor microenvironment co-culture and combination pharmacotyping are needed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS11.html#author3"><strong class="card-body-subgroup">Vincent Jonchere</strong></a>
		    	 
		    		<em> (INSERM Sorbonne Université, UMRS 938, Équipe Instabilité des Microsatellites et Cancer, Équipe Labellisée par la Ligue Nationale Contre le Cancer et SIRIC, France)</em>
		    	
		    	
		    		
		    			</br>"Identification of Positively and Negatively Selected Driver Gene Mutations Associated With Colorectal Cancer With Microsatellite Instability"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Background & Aims Recent studies have shown that cancers arise as a result of the positive selection of driver somatic events in tumor DNA, with negative selection playing only a minor role, if any. However, these investigations were concerned with alterations at nonrepetitive sequences and did not take into account mutations in repetitive sequences that have very high pathophysiological relevance in the tumors showing microsatellite instability (MSI) resulting from mismatch repair deficiency investigated in the present study.
Methods We performed whole-exome sequencing of 47 MSI colorectal cancers (CRCs) and confirmed results in an independent cohort of 53 MSI CRCs. We used a probabilistic model of mutational events within microsatellites, while adapting pre-existing models to analyze nonrepetitive DNA sequences. Negatively selected coding alterations in MSI CRCs were investigated for their functional and clinical impact in CRC cell lines and in a third cohort of 164 MSI CRC patients.
Results Both positive and negative selection of somatic mutations in DNA repeats was observed, leading us to identify the expected true driver genes associated with the MSI-driven tumorigenic process. Several coding negatively selected MSI-related mutational events (n = 5) were shown to have deleterious effects on tumor cells. In the tumors in which deleterious MSI mutations were observed despite the negative selection, they were associated with worse survival in MSI CRC patients (hazard ratio, 3; 95% CI, 1.1–7.9; P = .03), suggesting their anticancer impact should be offset by other as yet unknown oncogenic processes that contribute to a poor prognosis.
Conclusions The present results identify the positive and negative driver somatic mutations acting in MSI-driven tumorigenesis, suggesting that genomic instability in MSI CRC plays a dual role in achieving tumor cell transformation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS11.html#author4"><strong class="card-body-subgroup">Johannes Witt</strong></a>
		    	 
		    		<em> (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Heidelberg, Germany)</em>
		    	
		    	
		    		
		    			</br>"Analyzing the influence of HLA class I genotype on cancer immunoediting"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Already in early stages of tumorigenesis, transformed cells are recognized and attacked by the immune system, leading to the elimination of precancerous cell clones. This process depends on the generation of neoantigens, which determine the immunogenicity of tumor cells. Highly immunogenic cancer cells are counterselected during tumor evolution, constituting a Darwinian selection process. In microsatellite-unstable (MSI) cancer, a high load of neoantigens accumulates due to frameshift mutations in coding microsatellites. The immunogenicity of frameshift peptides (FSP) depends on the presentation of cleaved peptides on the cell surface by human leucocyte antigen (HLA) molecules. Endogenously produced peptides are preferentially presented by HLA class I molecules. Among the HLA class I genes, HLA-A, -B and -C play the most prominent role in the immune response. Due to amino acid substitutions in the peptide-binding region, each HLA molecule is characterized by a specific repertoire of peptides that can be presented. Analyzing a single nucleotide polymorphism at the 5’-end of exon 2 of the HLA-A gene, we divided a set of 75 MSI colorectal cancer samples into two groups: samples possessing at least one HLA-A*02 allele and samples without any HLA-A*02 allele. For both constellations, we developed scores estimating the probability that at least one FSP-derived peptide is presented on the cell surface by a HLA-A molecule (OLLA,G2, OLLA,GN). We observed an inverse correlation between the predicted immunogenicity of 41 FSP and the mutation frequency, which may reflect a selection pressure exerted by the immune system. However, this correlation is not group-specific, indicating that the immunogenicity of FSP is potentially not only determined by the HLA type. With increasing length l of FSP, the number of possible FSP-derived peptides increases. Thus, the likelihood rises that at least one of them fits a random HLA molecule. For both the HLA-A*02 and the Non-HLA-A*02 group, we observed a negative correlation between l and the mutation frequency. Our predictions imply that HLA diversity may determine the likelihood of FSP recognition and therefore immune recognition of tumor cells.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS12.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS12-ONCO: </small></br>Mathematical Oncology: From methodological studies to clinical applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Saskia Haupt (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Vincent Heuveline (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Matthias Kloor (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Germany)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS12.html#author1"><strong class="card-body-subgroup">Natalia Komarova</strong></a>
		    	 
		    		<em> (Department of Mathematics, University of California Irvine, Irvine, California, USA)</em>
		    	
		    	
		    		
		    			</br>"CLL and the drug Ibrutinib: modeling and clinical applications"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chronic Lymphocytic leukemia is the most common leukemia, mostly arising in patients over the age of 50. The disease has been treated with chemo-immunotherapies with varying outcomes, depending on the genetic make-up of the tumor cells. Recently, a promising tyrosine kinase inhibitor, ibrutinib, has been developed, which resulted in successful responses in clinical trials, even for the most aggressive chronic lymphocytic leukemia types. The crucial questions include how long disease control can be maintained in individual patients, when drug resistance is expected to arise, and what can be done to counter it. Computational evolutionary models, based on measured kinetic parameters of patients, allow us to address these questions and to pave the way toward a personalized prognosis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS12.html#author2"><strong class="card-body-subgroup">Johannes G Reiter</strong></a>
		    	 
		    		<em> (Canary Center for Cancer Early Detection, Department of Radiology, Stanford University, California, USA)</em>
		    	
		    	
		    		
		    			</br>"Minimal intermetastatic heterogeneity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Genetic intratumoral heterogeneity is a natural consequence of imperfect DNA replication. Any two randomly selected cells, whether normal or cancerous, are therefore genetically different. I will discuss the extent of genetic heterogeneity among untreated cancers with particular regard to its clinical relevance and how it can be exploited to identify metastatic seeding patterns. While genomic heterogeneity within primary tumors is associated with relapse, heterogeneity among treatment‑naïve metastases has not been comprehensively assessed. Within individual patients a large majority of driver gene mutations are common to all metastases. Further analysis revealed that the driver gene mutations that were not shared by all metastases are unlikely to have functional consequences. A mathematical model of tumor evolution and metastasis formation provides an explanation for the observed driver gene homogeneity. Based on a statistical framework for quantifying metastatic phylogenetic diversity in dozens of inferred phylogenies of colorectal cancer patients, distant metastases were typically monophyletic and genetically similar to each other. Lymph node metastases, in contrast, exhibited high levels of inter-lesion diversity. These data indicate that the cells within the primary tumors that give rise to distant metastases evolved through a narrow bottleneck and are generally more homogeneous than the primary tumor and lymph node metastases.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS12.html#author3"><strong class="card-body-subgroup">Kamila Naxerova</strong></a>
		    	 
		    		<em> (Center for Systems Biology, Department of Radiology, Massachusetts General Hospital and Harvard Medical School, Boston, Massachusetts, USA)</em>
		    	
		    	
		    		
		    			</br>"On the evolutionary history of metastatic cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The evolutionary history of metastases is still largely unknown. Do metastases arise from distinct clones with special, genetically encoded properties or do they represent random samples of the primary tumor? Does metastatic spread happen early or late in tumor development? Do all metastases arise independently from the primary tumor, or do they give rise to each other? How heterogeneous are metastases? These fundamental questions have profound clinical implications but are difficult to study in human patients because relevant events predate diagnosis by many years. We are developing methods for high-efficiency lineage tracing in human tumor samples and apply these to study the roots of metastatic disease. Here, the joint insights from multiple published and unpublished studies will be presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS12.html#author4"><strong class="card-body-subgroup">Saskia Haupt</strong></a>
		    	 
		    		<em> (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany)</em>
		    	
		    	
		    		
		    			</br>"A computational model for investigating the evolution of colonic crypts during Lynch syndrome carcinogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Introduction Lynch syndrome (LS), the most common inherited colorectal cancer (CRC) syndrome, increases the cancer risk in affected individuals. LS is caused by pathogenic germline variants in one of the DNA mismatch repair (MMR) genes, complete inactivation of which causes numerous mutations in affected cells. As CRC is believed to originate in colonic crypts, understanding the intra-crypt dynamics caused by mutational processes is essential for a complete picture of LS CRC and may have significant implications for cancer prevention.
Methods We propose a computational model describing the evolution of colonic crypts during LS carcinogenesis. Extending existing modeling approaches for the non-Lynch scenario, we incorporated MMR deficiency and implemented recent experimental data demonstrating that somatic CTNNB1 mutations are common drivers of LS-associated CRCs if affecting both alleles of the gene. Further, we simulated the effect of different mutations on the entire crypt, distinguishing non-transforming and transforming mutations.
Results As an example, we analyzed the spread of mutations in the genes APC and CTNNB1, which are frequently mutated in LS tumors, as well as of MMR deficiency itself. We quantified each mutation's potential for monoclonal conversion and investigated the influence of the cell location and of stem cell dynamics on mutation spread.
Conclusion The in silico experiments underline the importance of stem cell dynamics for the overall crypt evolution. Further, simulating different mutational processes is essential in LS since mutations without survival advantages (the MMR deficiency-inducing second hit) play a key role. The effect of other mutations can be simulated with the proposed model. Our results provide first mathematical clues for effective surveillance protocols for LS carriers.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS13.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS13-ONCO: </small></br>Frontiers in Mathematical Oncology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Kasia Rejniak & Heiko Enderling (Moffitt Cancer Center, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS13.html#author1"><strong class="card-body-subgroup">Thomas E. Yankeelov</strong></a>
		    	 
		    		<em> (The University of Texas at Austin, USA)</em>
		    	
		    	
		    		
		    			</br>"Imaging-based mathematical modeling of brain cancer across scales"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Our lab is focused on developing tumor forecasting methods by integrating advanced imaging technologies with predictive, mathematical models to forecast tumor growth and treatment response.  In this presentation, we will provide an overview of three vignettes in mathematical oncology that span the in vitro (cells), in vivo pre-clinical (rats), and in vivo clinical (human) scales in brain cancer. Each project employs quantitative imaging to calibrate an appropriate mathematical model to predict how the tumors grow, how they respond to therapy, or how the therapy is delivered.
The first vignette employs time resolved microscopy data to calibrate a system of ordinary differential equations to predict the response of glioma cells to single- and multi-fraction radiation therapy in vitro. We then move to in vivo, pre-clinical studies where we make use of quantitative magnetic resonance imaging (MRI) data reporting on cellularity and perfusion to calibrate a system of reaction diffusion models to predict the response of glioma cells to single- and multi-fraction radiation therapy in a murine model of brain cancer. The final vignette is focused on employing MRI, x-ray computed tomography (CT), and single photon emission computed tomography (SPECT) to calibrate a reaction-diffusion-advection equation to predict and optimize the spatio-temporal distribution of radiolabeled liposomes for the treatment of recurrent glioblastoma multiforme in patients. The long-term goal of these studies is to provide a rigorous, but practical, experimental-computational approach describing tumor development, informed and validated by readily available imaging data.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS13.html#author2"><strong class="card-body-subgroup">Arne Traulsen</strong></a>
		    	 
		    		<em> (Max Planck Institute for Evolutionary Biology, Germany)</em>
		    	
		    	
		    		
		    			</br>"Measuring cancer heterogeneity and possibilities of exploiting it in treatment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Evolving populations naturally diversify. For populations of cancer cells, this has been extensively explored on the genotypic level and recognized as a potential problem in treatment. Phenotypic diversity, on the other hand, is typically harder to measure, but it may also be directly relevant for treatment, especially when different treatment options are available. Theoretical models show that cancer progression could be delayed substantially if the current phenotypic state can be taken into account in the choice of therapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS13.html#author3"><strong class="card-body-subgroup">Angélique Stéphanou</strong></a>
		    	 
		    		<em> (University of Grenoble, France)</em>
		    	
		    	
		    		
		    			</br>"Cell metabolism and intracellular acidity regulation in cancer cells, from experimental characterization to computational models with therapeutic perspectives"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The metabolism of cancer cells is characterized by increased glycolysis due to local hypoxic conditions. Glycolysis in turn induces an increase in acidity which is detrimental to cells. Cancer cells, however, exhibit a higher resistance to acidity than normal cells due to a better ability to regulate their intracellular pH. We have experimentally characterized the regulatory capacity of two glioma cell lines using fluorescence microscopy. We observed that the regulation of acidity is not the same for the two cell lines. This has consequences for cellular aggressiveness, metastatic potential and treatment planning since the main drug used against glioblastoma is highly pH dependent. Theoretically, we revised a model of cellular metabolism to specifically take into account the influence of pH on cellular metabolic adaptation. The model suggests that the Warburg effect, often described as a hallmark of cancer, can actually be viewed as a transient adaptation mechanism to a disturbed environment rather than an inherent characteristic of the cancer cell. As such, targeting the acidic environment rather than targeting the cancer cell could offer a good alternative therapeutic strategy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS13.html#author4"><strong class="card-body-subgroup">Elizabeth Wayne</strong></a>
		    	 
		    		<em> (Carnegie Mellon University, USA)</em>
		    	
		    	
		    		
		    			</br>"Developing experimental and mathematical models to measure changes in tumor associate macrophage polarization in response to immunotherapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumor associated macrophages (TAMs) are a significant player in cancer microenvironment. They can comprise 50%-80% of a solid tumor mass and M2, anti-inflammatory polarized TAMs are correlated with poorer clinical outcomes. Numerous therapeutic strategies attempt to modulate TAM polarization to decrease tumor growth. However, macrophage polarization is dependent on a number of intrinsic and extrinsic factors. Understanding the factors government TAM polarization can help us understand therapeutic response heterogeneity. Here the talk will discuss experimental models for deciphering the interplay of TAM polarization, drug accumulation, and tumor growth. Moreover, this talk will discuss ideas for developing models that work in tandem with experimental data. Being able to experimentally and mathematically model the effect of immunomodulatory drugs on TAM polarization could enhance decision making in personalized cancer treatment.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS13" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS14.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS14-ONCO: </small></br>Integrating quantitative imaging and mechanistic modeling to characterize tumor growth and therapeutic response</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Guillermo Lorenzo (University of Pavia, Italy), David Hormuth (The University of Texas at Austin, US), Angela Jarrett (The University of Texas at Austin, US), Thomas Yankeelov (The University of Texas at Austin, US)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS14.html#author1"><strong class="card-body-subgroup">Andrea Gardner</strong></a>
		    	 
		    		<em> (The University of Texas at Austin, US)</em>
		    	
		    	
		    		
		    			</br>"Quantification of interactions between epithelial-like and mesenchymal-like subpopulations in a triple-negative breast cancer cell line ecosystem"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many cancer cell lines once thought to be relatively homogeneous are composed of distinct subpopulations. Informed by single-cell RNA sequencing of the triple-negative breast cancer cell line MDA-MB-231, we discovered a surface marker which effectively separates epithelial-like (EL) cells from mesenchymal-like (ML) cells from this population. Growth characteristics of EL and ML subpopulations were determined in monoculture and in co-culture and under varying environmental conditions. We find that while the ML cells are neutral to the presence of EL cells, the growth rate of the already faster EL cells is further boosted in the presence of ML cells. One would expect this phenomenon to lead to the extinction of the ML cells, however, these cells co-exist over many generations in vitro. To investigate this paradox, experimental data from live-cell imaging was integrated with an extended Lotka-Volterra competition model to quantify the intrinsic properties and interactions of these two-subpopulations and we present our findings here.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS14.html#author2"><strong class="card-body-subgroup">Haley Bowers</strong></a>
		    	 
		    		<em> (Wake Forest School of Medicine, US)</em>
		    	
		    	
		    		
		    			</br>"Image Data-Driven Biophysical Mathematical Model Based Characterization of Multicellular Tumor Spheroids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Multicellular tumor spheroid (MCTS) systems provide an in vitro cell culture model system which replicates many of the complexities of an in vivo solid tumor and its tumor microenvironment. MCTS systems are often used to study cancer cell growth and drug efficacy. In this work, we present a coupled experimental-computational framework to estimate phenotypic growth and biophysical tumor microenvironment properties. This novel framework utilizes standard microscopy imaging of MCTS systems to drive a biophysical mathematical model of MCTS growth and mechanical interactions. This work is an extension of our previous in vivo mechanically-coupled reaction-diffusion modeling framework we developed a microscopy image processing framework capable of mechanistic characterization of MCTS systems. Using fluorescently labeled MDA-MB-231 breast cancer MCTS, we estimated biophysical parameters of cellular diffusion, rate of cellular proliferation, and cellular tractions forces. We found significant differences in between untreated and treated MCTS systems using these model-based biophysical parameters throughout the treatment time course, whereas traditional morphometric parameters were inconclusive. This experimental-computational framework estimates mechanistic MCTS growth and invasion parameters with significant potential to assist in better and more precise assessment of in vitro drug efficacy through the development of computational analysis methodologies for three-dimensional cell culture systems to improve the development and evaluation of antineoplastic drugs.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS14.html#author3"><strong class="card-body-subgroup">Anum Kazerouni</strong></a>
		    	 
		    		<em> (University of Washington, US)</em>
		    	
		    	
		    		
		    			</br>"Characterizing tumor heterogeneity using quantitative MRI habitats in breast cancer in vivo"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Within a tumor exists a dynamic interplay of spatially-varying cell populations and tissue microenvironments that both contributes to tumor progression and influences therapeutic response. For example, the uneven distribution of vasculature across a tumor can yield nonuniform drug delivery. Additionally, phenotypic diversity across cancer cells can result in variable response to treatment. These aspects of tumor heterogeneity provide a major challenge in the clinical treatment of breast cancer and result in significant diversity of outcomes across a patient population, emphasizing the need for personalized approaches to cancer treatment. Methods to characterize the spatiotemporal evolution of an individual tumor and its resulting heterogeneity can lend improved understanding of a patient’s tumor pathology and their potential response to therapy.
Quantitative magnetic resonance imaging (MRI) is noninvasive and can, therefore, longitudinally detect changes in physiological characteristics across a tumor volume. In particular, diffusion-weighted (DW-) MRI and dynamic contrast-enhanced (DCE-) MRI provide quantitative assessment of tissue cellularity and vascularity, respectively—key tumor attributes that are affected by therapy. Accordingly, quantitative MRI measures have demonstrated promise as biomarkers of breast cancer treatment response in both preclinical and clinical settings. Recent work has investigated methods to spatially resolve intratumoral heterogeneity using quantitative MRI through an approach known as habitat imaging6. With this technique, physiologically distinct tumors subregions (i.e., habitats) are identified by clustering multiparametric image data, thus facilitating quantitative characterization of microenvironmental heterogeneity for individual tumors.
In this presentation, we will describe how DW- and DCE-MRI data can be leveraged to spatially resolve physiologically distinct tumor habitats in vivo with biological validation ex vivo. Using preclinical models of breast cancer, we will demonstrate how this approach can be used to measure longitudinal alterations in the tumor microenvironment in response to treatment and identify tumor imaging phenotypes with differing therapeutic sensitivities. Additionally, we will describe the translation of this approach to the clinical setting and its promise in identifying breast cancer patients with increased likelihood of neoadjuvant therapy response based on tumor habitat composition. These tumor-specific characterizations of microenvironmental heterogeneity could provide a means to more accurately guide individualized patient treatment strategies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS14.html#author4"><strong class="card-body-subgroup">David Hormuth</strong></a>
		    	 
		    		<em> (The University of Texas at Austin, US)</em>
		    	
		    	
		    		
		    			</br>"Image-driven modeling of radiation therapy response in gliomas"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Radiotherapy is a fundamental component of the treatment and management of high-grade gliomas. The efficacy of radiotherapy can vary from tumor to tumor due to spatial and temporal heterogeneity in (for example) cellularity, blood volume, and perfusion. A rigorous understanding of the dynamics of tumor heterogeneity could enable the personalization of radiotherapy for individual tumors. Quantitative imaging techniques such as diffusion weighted (DW-) magnetic resonance imaging (MRI) and dynamic contrast-enhanced (DCE-) MRI provide an opportunity to longitudinally, and non-invasively, observe the dynamics of tumor heterogeneity in 3D. We have developed an experimental and computational framework to integrate these longitudinal measurements of tumor heterogeneity into mathematical models of tumor growth and response. Seven animals implanted intra-cranially with the U87 glioblastoma cell line were imaged before, during, and after the delivery of radiotherapy. We then initialized and calibrated a family of 18 models of response to radiotherapy for each animal using DW-MRI and DCE-MRI. Using the calibrated model parameters we assessed the error in response predictions at the local and global levels. At the global level, we observed less than 16.2% error in tumor volume predictions while at the local level we observed a Pearson correlation coefficient of greater than 0.87 for each animal. This effort demonstrates the strength of using longitudinal MRI data for personalization of models predicting the response of brain tumors to radiotherapy.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS14" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS15.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS15-ONCO: </small></br> Recent development in mathematical oncology in Asia and Australia</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yangjin Kim (Konkuk University, Korea, Republic of), Eunjung Kim (Korea Institute of Science and Technology, Korea)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS09-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS15.html#author1"><strong class="card-body-subgroup">Dumitru Trucu</strong></a>
		    	 
		    		<em> (Division of Mathematics, University of Dundee, DD1 4HN, Dundee, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Multiscale 3D Glioblastoma Modelling: Bulk and Leading Edge Dynamics within the Fibrous Brain Tissue"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma multiform is one of the most aggressive types of brain cancer, and the understanding of its progression remains one of the greatest challenges. In this talk we propose a multi-scale moving boundary approach for the glioblastoma cell population invasion within the brain fibrous environment. This will account on both the proteolytic dynamics at the tumour interface and on the interaction with brain fibres and the emerging collagen fibres at the site of the tumour. These interactions will be explored in their natural 3D setting by accounting on their genuinely multiscale character both in terms of the peritumoural proteolytic activity of the matrix degrading enzymes, and the cell-brain fibres interactions. Our 3D computational exportation suggests that although current imaging technologies provide valuable details of the brain’s underlying structure, in order to provide meaningful predictions for tumour growth and to test new hypotheses, we may need to use this information in a different, novel ways when we model glioblastoma mathematically.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS15.html#author2"><strong class="card-body-subgroup">Peter Kim</strong></a>
		    	 
		    		<em> (University of Sydney, Australia)</em>
		    	
		    	
		    		
		    			</br>"How do viruses move? Modelling diffusion of oncolytic virus in collagen-dense tumours"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Solid tumours develop much like a fortress, acquiring characteristics that protect them against invasion. A common trait observed in solid tumours is the synthesis of excess collagen which traps therapeutic agents, resulting in a lack of dispersion of treatment within the tumour mass. In most tumours this results in only a localised treatment. Often the tumour quickly recovers and continues to invade surrounding regions.
Anti-tumour viral therapy, although consisting of nano-sized particles, is no exception to this rule. Experimental results show collagen density affects viral diffusion. More specifically, when injected, viruses will move to regions of low collagen concentration; therefore, accurately modelling viral diffusion is an important aspect of modelling virotherapy. To understand the underlying dynamics that impede viral diffusion in collagen, we derive, from first principles, a novel non-Fickian diffusion term and show that this diffusion term can accurately capture experimental observations. Then, using a system of partial differential equations we explore how treatment under this diffusion term differs from the standard Fickian diffusion, commonly used in virotherapy models. The disparity between results highlights a significant gap in our understanding of virotherapy modelling and could mean estimates based on Fickian diffusion need to be reassessed for their biological impact.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS15.html#author3"><strong class="card-body-subgroup">Da Zhou</strong></a>
		    	 
		    		<em> (School of Mathematical Sciences, Xiamen University, China)</em>
		    	
		    	
		    		
		    			</br>"Cancer suppression: ingredients utilized by cellular hierarchy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many fast renewing tissues are characterized by a hierarchical cellular architecture, with tissue specific stem cells at the root of the cellular hierarchy, differentiating into a whole range of specialized cells. Growing evidence shows that the hierarchical cellular architecture has a profound effect on cancer suppression. In this talk, we will show some cancer-suppression mechanisms possibly utilized by cellular hierarchy using mathematical models. Specifically, we are concerned about cell competition, different modes of cell division and their effects on cancer suppression.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS15.html#author4"><strong class="card-body-subgroup">Junho Lee</strong></a>
		    	 
		    		<em> (Department of Mathematics, Konkuk University, Korea)</em>
		    	
		    	
		    		
		    			</br>"Role of neutrophil extracellular traps in regulation of lung cancer invasion : a computational model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Lung cancer is one of the leading causes of cancer-related deaths worldwide and is characterized by hijacking immune system for active growth and aggressive metastasis. Neutrophils, which require establishing immune activity against tumors as the first line of defense, are damaged by tumor cells, which in many ways promote tumor invasion. The mutual interaction between a tumor and neutrophils from bone marrow or in blood induces the critical transition of the naive form, called the N1 type, to the more aggressive phenotype, called the N2 tumor-associated neutrophils (TANs), which then promotes tumor invasion. In this study, we investigate the mutual interactions between the tumor cells and the neutrophils that facilitate tumor invasion by developing a mathematical model that involves taxis-reaction-diffusion equations for the critical components in the interaction. These include the densities of tumor and neutrophils, and the concentrations of signaling molecules (TGFbeta-CXCL8-MMP) and structure such as neutrophil extracellular traps (NETs). We apply the mathematical model to a Boyden invasion assay used in the experiments to demonstrate that the N2 TANs can enhance tumor cell invasion by secreting the neutrophil elastase. We show (i) that the model can reproduce the major experimental observation on NET-mediated cancer invasion, (ii) how stimulated neutrophils with different N1 and N2 landscapes shape the metastatic potential of the lung cancers and (iii) that the efficacy of anti-tumor and anti-invasion drugs depend on N1  N2 landscapes of stimulated neutrophils. The mathematical model tests several hypotheses to guide future experiments with the goal of the development of new anti-tumor strategies.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS15" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS18.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS18-ONCO: </small></br>Measuring and modeling the cell-state transitions in cancer progression and treatment</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Mohit Kumar Jolly ( Assistant Professor, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India), Kishore Hari (PhD Student, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS18.html#author1"><strong class="card-body-subgroup">Caterina AM La Porta</strong></a>
		    	 
		    		<em> (Professor of General Pathology Department of Environmental Science and Policy, University of Milan; CEO ComplexData SRL, Italy)</em>
		    	
		    	
		    		
		    			</br>"Explaining the dynamics of melanoma aggressiveness: at the crossroads between biology and artificial intelligence"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Melanoma is one of the most aggressive and highly resistant tumor. Cell plasticity in
melanoma is one the main reason behind its metastatic capacity. I will discuss the recent
results obtained by our group on cellular plasticity and CSCs in melanoma. The detailed
molecular mechanisms controlling melanoma plasticity are still not completely understood.
We combine mathematical models of phenotypic switching with experiments on IgR39
human melanoma cell line to identify possible key targets to impair phenotypic switching.
Our results shed new light on melanoma plasticity providing a potential target and guidance
for therapeutic studies
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS18.html#author2"><strong class="card-body-subgroup">Shensi Shen</strong></a>
		    	 
		    		<em> (West China Hospital, Sichuan University, Chengdu, China;  Gustave Roussy Cancer Campus, Villejuif, France, China)</em>
		    	
		    	
		    		
		    			</br>"Persistent cancer cells : blazing the trail with metastatic melanoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The probability to achieve an objective response with anti-BRAF+MEK therapy in patients with BRAFV600E/K mutant melanoma is around 70% . However, after one year, half of the patients who initially responded to this combined therapy develops secondary resistance. Among these patients, some also resist to anti-PD1 immunotherapy as single agent or in combination with anti-CTLA4. For these patients, the medical needs are huge since there is presently no effective alternative treatment. Resistance to targeted agents can be due to the presence of pre-existing rare resistant clones in heterogeneous tumor cell population or the stochastic acquisitions of drug resistance through genetic mutations under therapeutic selective pressures. The latter case is the most frequent, wherein some cells of an isogenic tumor cell-population survive in spite of the presence of anticancer drug(s). Such cells are defined as 'persistent cancer cells' and their survival capability is dependent on the presence of anticancer agents. I will discuss how persistent melanoma cells adaptively tolerate the treatment from the aspect of reversible mRNA translational reprogramming and accompanying metabolic rewiring, eventually how these aspects can be modelled in an agent-based stochastic modeling.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS18.html#author3"><strong class="card-body-subgroup">Michael P H Stumpf</strong></a>
		    	 
		    		<em> (Professor of Systems Biology, School of BioSciences, University of Melbourne, Australia)</em>
		    	
		    	
		    		
		    			</br>"  Stochastic Dynamics and Cell Fate Decision Dynamics in Development and Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The metaphor of the Waddington epigenetic landscape has become an iconic representation of the cellular differentiation process, in both health and disease. Recent accessibility of single-cell transcriptomic data has provided new opportunities for quantifying this originally conceptual tool that could offer insight into the gene regulatory networks underlying cellular development. Here, we highlight the complexities and limitations that arise when reconstructing the potential landscape in the presence of stochastic fluctuations. We consider how the landscape changes in accordance with different stochastic systems, and show that it is the subtle interplay between the deterministic and stochastic components of the system that ultimately shapes the landscape.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS18.html#author4"><strong class="card-body-subgroup">Kishore Hari</strong></a>
		    	 
		    		<em> (PhD Student, Center for Biosystems Sceince and Engineering, Indian Institute of Science Bengaluru, India)</em>
		    	
		    	
		    		
		    			</br>"Mechanisms of phenotypic plasticity in Metastasis, a network topology perspective"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Metastasis, the process of cancerous cells invading multiple organs of the body, causes more than 90% of cancer-related deaths. No unique mutations could be associated with metastasis, and no cancer treatment so far can target metastasis. Recent studies suggest that metastasis is driven mainly by multiple interdependent axes of phenotypic plasticity, such as metabolic plasticity, drug resistance, dormancy, stemness, and Epithelial-mesenchymal plasticity (EMP). In particular, EMP – a developmental axis of phenotypic plasticity – is believed to be crucial for metastasis as it imparts the adherence and migratory characteristics to cancerous cells. Despite extensive physicochemical investigations, the mechanisms of the emergence of such phenotypic plasticity are still not understood. To understand these mechanisms, we take a two-pronged approach. On the one hand, we study the regulatory network topologies underlying EMP to identify characteristics that can give rise to plasticity. On the other hand, we construct models based on population dynamics data to understand the dynamics of switching and infer phenotypic plasticity mechanisms other than network topology, such as stochasticity, ecological interactions between various EMP phenotypes, and epigenetics. Our results suggest that the EMP networks have a high fraction of positive feedback loops, which can give rise to phenotypic plasticity. Furthermore, small perturbations that reduce the number of positive feedback loops and increase the number of negative feedback loops can reduce phenotypic plasticity over a large parameter space.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS18" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS19.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS19-ONCO: </small></br>Measuring and modeling the cell-state transitions in cancer progression and treatment</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Mohit Kumar Jolly ( Assistant Professor, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India), Kishore Hari (PhD Student, Center for Biosystems Science and Engineering, Indian Institute of Sceince Bengaluru, India)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS18-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS19.html#author1"><strong class="card-body-subgroup">Sabrina L Spencer</strong></a>
		    	 
		    		<em> (Assistant Professor, Department of Biochemistry, University of Colerado-Boulder, United States of America)</em>
		    	
		    	
		    		
		    			</br>"  Real-time visualization of rapid escape from BRAF inhibition in single melanoma cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Despite the increasing number of effective anti-cancer therapies, successful treatment is limited by the development of drug resistance. While the contribution of genetic factors to drug resistance is undeniable, little is known about how drug-sensitive cells first evade drug action to proliferate in drug. Here we track the responses of thousands of single melanoma cells to BRAF inhibitors and show that a subset of cells escapes drug via non-genetic mechanisms within the first three days of treatment. Cells that escape drug rely on ATF4 stress signalling to cycle periodically in drug, experience DNA replication defects leading to DNA damage, and yet out-proliferate other cells over extended treatment. Together, our work reveals just how rapidly melanoma cells can adapt to drug treatment, generating a mutagenesis-prone subpopulation that expands over time.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS19.html#author2"><strong class="card-body-subgroup">Yogesh Goyal</strong></a>
		    	 
		    		<em> (Postdoctoral researcher, University of Pennsylvania, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Cellular plasticity and fate choices in single cancer cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			While cellular processes are often reproducible and precise, cells may also alter their molecular states and adopt new fates in response to stimuli, a phenomena referred to as “plasticity”. I am interested in understanding the control principles governing cellular plasticity and fate decisions in response to mutational and pharmacologic stresses in tissue development and single cancer cells. My postdoctoral work is motivated by recent studies revealing how rare and transient non-genetic fluctuations in individual cancer cells enable them to survive pharmacologic stress, such as molecularly targeted therapies. Unlike the binary nature of Darwinian selection whereby mutations are either present or not, non-genetic fluctuations can exist on one, or even multiple continuums of variation. How this non-genetic variability maps to the eventual resistant fates upon drug exposure is an emerging paradigm of cellular plasticity. Integrating novel theoretical and experimental frameworks, I will present my findings on 1. Identifying the origins and nature of the unique transcriptional molecular states underlying this plasticity; and 2. Connecting these molecular states to their eventual drug-resistant fates by tracking thousands of uniquely barcoded cell lineages. Moving forward, my own group will adapt these quantitative approaches and concepts to measure, model, and engineer plasticity and its roles in tissue development and disease.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS19.html#author3"><strong class="card-body-subgroup">Qing Nie</strong></a>
		    	 
		    		<em> ( Professor of Mathematics and Developmental & Cell Biology, University of California, Irvine ; Director of The NSF-Simons Center for Multiscale Cell Fate Research, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Inference and Multiscale Model of Epithelial-to-Mesenchymal Transition via Single-cell Transcriptomic Data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Epithelial to mesenchymal transition (EMT) plays an important role in many biological processes during development and cancer. The advent of single-cell transcriptome sequencing techniques allows the dissection of dynamical details underlying EMT with unprecedented resolution. We develop an integrative tool that combines unsupervised learning of single-cell transcriptomic data and multiscale mathematical modeling to analyze transitions during cell fate decision. Our approach allows identification of individual cells making transition between all cell states and inference of genes that drive transitions. Multiscale extractions of single-cell scale outputs naturally reveal intermediate cell states (ICS) and ICS-regulated transition trajectories, producing emergent population-scale models to be explored for design principles. Testing on the single-cell gene regulatory network model and applying to published single-cell EMT datasets in cancer and embryogenesis, we uncover the roles of ICS on adaptation, noise attenuation, and transition efficiency in EMT, and reveal their trade-off relations. Meanwhile, network topology analysis and multilayer gene-gene regulation networks suggest that the ICS during EMT serve as the signaling hub in the TGF-β signaling communication.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS19.html#author4"><strong class="card-body-subgroup">Einar Gunnarsson</strong></a>
		    	 
		    		<em> (Graduate student, University of Minnesota, Twin Cities, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Modeling the role of phenotypic switching in cancer drug resistance"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The emergence of acquired drug resistance in cancer represents a major barrier to treatment success. In this talk, we describe a simple mathematical model for studying how phenotypic switching at the single-cell level affects resistance evolution in cancer. We discuss how even short-term epigenetic modifications and stochastic fluctuations in gene expression can drive long-term drug resistance in the absence of any bona fide resistance mechanisms. We also show that an epigenetic drug that slightly perturbs the average retention of the resistant phenotype can turn guaranteed treatment failure into guaranteed success. We finally examine how the mode and time scale of resistance acquisition depends on the underlying switching dynamics and discuss potential implications for treatment.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-MS20.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS20-ONCO: </small></br>Integrating quantitative imaging and mechanistic modeling to characterize tumor growth and therapeutic response</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Guillermo Lorenzo (University of Pavia, Italy), David Hormuth (The University of Texas at Austin, US), Angela Jarrett (The University of Texas at Austin, US), Thomas Yankeelov (The University of Texas at Austin, US)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-MS20.html#author1"><strong class="card-body-subgroup">Darren Tyson</strong></a>
		    	 
		    		<em> (Vanderbilt University, US)</em>
		    	
		    	
		    		
		    			</br>"The many dimensions of anticancer drug response—quantifying cell population dynamics at single-cell resolution using automated live-cell microscopy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A tumor in a human patient is an evolving system of interacting components, including different cell types containing various genetic alterations, adjacent stromal cells, and many different types of cell–cell and cell–matrix interactions. In addition, many parameters affect how therapeutic drugs can (ideally) kill all the tumor cells while sparing normal adjacent cells, including pharmacodynamic/pharmacokinetic properties of the drugs, the specificity with which they target tumor cells, specific genetic alterations that affect how a cell responds to the drug, etc. I will demonstrate how human cancer cell can be analyzed for their responses to anticancer drugs in high throughput using automated fluorescence microscopy to enable the direct visualization of many features of individual cells over time and how we have modeled the dynamics of cell population-level changes by simultaneously estimating the rates of cell division, death and entry into a non-dividing state from the single-cell measurements. This model facilitates the interpretation of how single-cell fate decisions affect the overall cell population dynamics in a drug concentration- and time-dependent manner that removes biases inherent in more traditional end-point measurements. I will describe how we have used this basic model as a framework to develop more detailed models to interrogate different aspects of tumor cell biology, including: 1) transitions into more drug-tolerant cell states; 2) potential synergistic action of combinations of drugs at different concentration; and 3) effects of matrix stiffness on cellular responses to drugs.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS20.html#author2"><strong class="card-body-subgroup">Victor Perez-Garcia</strong></a>
		    	 
		    		<em> (University of Castilla-La Mancha, Spain)</em>
		    	
		    	
		    		
		    			</br>"From metabolic imaging to biomarkers through mathematical models in cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumor initiation and progression are evolutionary processes driven by the accumulation of genetic alterations, some of which confer selective fitness advantages to cancer cells. Selective pressures induced by microenvironmental conditions, treatments, the immune system and other effects have a role in the complex evolutionary dynamics in tumors. However, although the situation is changing fast, it is still very difficult to obtain longitudinal biological data of evolutionary dynamics of tumors in individual patients. Metabolic imaging provides a global perspective of the tumor metabolism and proliferation status and can be performed sequentially to assess tumor dynamics and response to treatments. In this talk I will present a extension of the Fisher-Kolmogorov classical model displaying evolutionary dynamics. The analysis of the model predicts a displacement of the location of  metabolic hotspots from the tumor core to its periphery during its natural history. This fact allows to define a novel metabolic imaging biomarker based on the distance from the metabolic hotspot to the tumor centroid, that is found to correlate with tumor aggressiveness and patient survival for different tumor histologies.
Moreover, further analysis of the model shows that the maximum metabolic activity (SUVmax) grows with tumor size following a scaling law with power 1/4. A fact that was confirmed in different metabolic imaging datasets. Deviations from this scaling law allow to define another biomarker related to the relation between observed peak activity and the value expected from the scaling law. That provides another biomarker with a strong prognostic factor in breast cancer, lung cancer, head and neck cancer and glioblastoma. The metric found outperformed classical metabolic prognostic variables used in nuclear medicine.
In conclusion, mathematical models with evolutionary dynamics suggests how to construct different metabolic imaging biomarkers with a strong prognostic value and thus clinical utility for different tumor histologies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS20.html#author3"><strong class="card-body-subgroup">Jana Lipkova</strong></a>
		    	 
		    		<em> (Brigham and Women’s Hospital, Harvard Medical School, US)</em>
		    	
		    	
		    		
		    			</br>"Personalized Radiotherapy Design for Glioblastoma:Integrating Mathematical Tumor Models,Multimodal Scans and Bayesian Inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma is a highly invasive brain tumor, whose cells infiltrate surrounding normal brain tissue beyond the lesion outlines visible in the current medical scans. These infiltrative cells are treated mainly by radiotherapy. Existing radiotherapy plans for brain tumors derive from population studies and scarcely account for patient-specific conditions. Here we provide a Bayesian machine learning framework for the rational design of improved, personalized radiotherapy plans using mathematical modeling and patient multimodal medical scans. Our method, for the first time, integrates complementary information from high resolution MRI scans and highly specific FET-PET metabolic maps to infer patient-specific tumor cell density, which in turn allow design of personalized radiotherapy plans. Initial clinical study showed that the proposed treatment plans spare more healthy tissue, this reducing radiation toxicity while yielding comparable accuracy with standard radiotherapy protocols. Moreover, the inferred regions of high tumor cell densities coincide with the tumor radioresistant areas, providing guidance for personalized dose-escalation. The proposed integration of multimodal scans and mathematical modeling provides a robust, non-invasive tool to assist personalized radiotherapy design.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-MS20.html#author4"><strong class="card-body-subgroup">Guillermo Lorenzo</strong></a>
		    	 
		    		<em> (University of Pavia, Italy)</em>
		    	
		    	
		    		
		    			</br>"Personalized image-based modeling of organ-confined prostate cancer: exploring the mechanical interactions between tumor growth and coexisting benign prostatic hyperplasia"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prostate cancer (PCa) is a public health burden and a major concern among ageing men worldwide, with high rates of incidence and mortality. Thanks to regular screening and risk-group triaging most patients are currently diagnosed and successfully treated when the tumor is in early stage and confined within the prostate. Benign prostatic hyperplasia (BPH) is another common pathology in ageing men that causes the prostate to gradually enlarge over time, which may produce bothersome lower urinary tract symptoms. PCa originating in men with larger prostates tend to present more favorable pathological features, but the fundamental mechanisms that explain this interaction between BPH and prostate cancer are largely unknown. Here, we propose a mechanical explanation for this phenomenon: the mechanical stress fields that originate as tumors grow are known to slow down their dynamics, and BPH contributes to these mechanical stress fields, hence further restraining PCa growth. To explore this hypothesis, we run a qualitative simulation study using a mechanically-coupled mathematical model of PCa growth. We run our study leveraging a patient-specific geometric model of the prostate and tumor extracted from magnetic resonance imaging data. Our simulations show that the mechanical stress fields accumulated in the prostate by BPH over time impede prostatic tumor growth and limit its invasiveness. We further explore the effect on tumor growth of a type of BPH drugs that are being investigated for the chemoprevention of PCa: 5-alpha reductase inhibitors (e.g., finasteride, dutasteride), which reduce the size of the prostate (thereby treating BPH symptoms) and might promote apoptosis in the tumor. Depending on the intensity of these two mechanisms, our simulations show different tumor growth dynamics ranging from long-term inhibition of PCa growth to rapidly growing large tumors, which may evolve towards advanced disease. The latter case may provide a mechanistic explanation for the controversial advanced PCa cases found in chemoprevention trials of these drugs. In the future, we think that our computational technology can contribute to further investigate the biophysical mechanisms underlying PCa and BPH, and ultimately assist physicians in the clinical management of these diseases by forecasting pathological and therapeutic outcomes on an organ-scale, patient-specific basis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <a id="CT"><h1 class="mb-3" id="MS1">Sub-group contributed talks</h1></a>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT01.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT01-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT01.html#author1"><strong class="card-body-subgroup">Andrei S Rodin</strong></a>
		    	 
		    		<em>City of Hope</em>
		    	
		    	
		    		
		    			</br>"Dissecting Response to Cancer Immunotherapy by Applying Bayesian Network Analysis to Flow Cytometry Data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer immunotherapy, specifically immune checkpoint blockade, has been found to be effec-tive in the treatment of metastatic cancers. However, only a subset of patients achieve clinical responses. Elucidating pretreatment biomarkers predictive of sustained clinical response is a major research priority. Another research priority is evaluating changes in the immune system before and after treatment in responders vs. nonresponders. Our group has been studying im-mune networks as an accurate reflection of the global immune state. Flow cytometry (FACS, fluorescence-activated cell sorting) data characterizing immune cell panels in peripheral blood mononuclear cells (PBMC) from gastroesophageal adenocarcinoma (GEA) patients were used to analyze changes in immune networks in this setting. Here, we describe a novel computational pipeline to perform secondary analyses of FACS data using systems biology/machine learning techniques and concepts. The pipeline is centered around comparative Bayesian network anal-yses of immune networks and is capable of detecting strong signals that conventional methods (such as FlowJo manual gating) might miss. Future studies are planned to validate and follow up the immune biomarkers (and combinations/interactions thereof) associated with clinical re-sponses identified with this computational pipeline.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author2"><strong class="card-body-subgroup">Tatiana Miti</strong></a>
		    	 
		    		<em>Moffitt Cancer Center & Research Institute</em>
		    	
		    	
		    		
		    			</br>"Integrating Spatial Point Pattern Analysis and Agent Based Modeling for Studies of Stroma Sheltering Effects on Tumor"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Over the past two decades, the advancement in visualization and analysis of molecular and cellular data led to the development of highly efficient, targeted cancer drugs. However, cancer relapse occurs frequently, indicating that the tumor microenvironment plays a crucial role in treatment response potentially sheltering tumor cells during drug administration. We studied the eco-evolutionary dynamics of stroma-proliferating tumor cells interaction via point pattern analysis methods and spatial agent-based modeling (ABM). We characterized the spatial extent and amplitude of stroma shielding in the presence and absence of treatment using various pairwise distance methods adapted from ecology and geology. The spatial distributions of tumor cells, their proliferation rates, and the identified stroma protective effects were used to parametrize the ABM and simulate the spatio-temporal dynamics of tumor growth. Our preliminary results show that stroma-proliferating tumor cell clustering is considerably higher under treatment than in control samples. Moreover, stroma's protective effect during treatment is limited to cells that are either in direct contact with stromal cells or in their immediate proximity, suggesting a paracrine mediate effect. We expect our results to lead to novel therapeutic interventions that aim to shift eco-evolutionary dynamics rather than maximize short-term tumor cell killing efficiency.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author3"><strong class="card-body-subgroup">Robert Noble</strong></a>
		    	 
		    		<em>City, University of London</em>
		    	
		    	
		    		
		    			</br>"Explaining modes of tumour evolution"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the mode of tumour evolution is important for accurate prognosis and designing effective treatment strategies. Whereas selective sweeps are prevalent during early tumour growth, later stages exhibit either sparse branching or effectively neutral evolution. The causes of these different patterns remain poorly understood. I will present a new model for determining the probability of selective sweeps versus clonal interference in one-, two- and three-dimensional expanding populations. The solutions of this model are surprisingly simple mathematical expressions that are independent of mutation rate. Given parameter values obtained with human cancer data, the model offers to explain why selective sweeps are rare except when tumours are relatively very small. I will discuss these results in the context of additional computational modelling and new indices for classifying modes of tumour evolution that I and my coauthors have developed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author4"><strong class="card-body-subgroup">Erdi Kara</strong></a>
		    	 
		    		<em>Texas Tech University Mathematics and Statistics</em>
		    	
		    	
		    		
		    			</br>"Diffusion Tensor Imaging (DTI) Based Drug Diffusion - Population Model for Solid Tumors"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this work, we study the effect of drug distribution on tumor cell death when the drug is internally injected in the tumorous tissue. We derive a full 3-dimensional inhomogeneous – anisotropic diffusion model. To capture the anisotropic nature of the diffusion process in the model, we use an MRI data of a 35-year old patient diagnosedwith Glioblastoma multiform(GBM) which is the most common and most aggressive primary brain tumor. Afterpreprocessing the data with a medical image processing software, we employ finite element method in MPI-basedparallel setting to numerically simulate the full model and produce dose-response curves. We then illustrate theapoptosis (cell death) fractions in the tumor region over the course of simulation and proposed several ways toimprove the drug efficacy. Our model also allows us to visually examine the toxicity. Since the model is builtdirectly on the top of a patient-specific data, we hope that this study will contribute to the individualized cancertreatment efforts from a computational bio-mechanics viewpoint.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT02.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT02-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT02.html#author1"><strong class="card-body-subgroup">Sabrina Neumaier</strong></a>
		    	 
		    		<em>Technical University of Munich</em>
		    	
		    	
		    		
		    			</br>"Introduction of an environmental stress level to model tumor cell growth and survival"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Survival of living cells underlies many influences such as nutrient saturation, oxygen level, drug concentrations or mechanical forces. Data-supported mathematical modeling can be a powerful tool to get a better understanding of cell behavior in different settings. However, under consideration of numerous environmental factors mathematical modeling can get challenging. We present an approach to model the separate influences of each environmental quantity on the cells in a collective manner by introducing the 'environmental stress level'. It is an artificial, immeasurable variable, which quantifies to what extent viable cells would get in a stressed state, if exposed to certain conditions. A high stress level can inhibit cell growth, promote cell death and influence cell movement. As a proof of concept, we compare two systems of ordinary differential equations, which model tumor cell dynamics under various nutrient saturations respectively with and without considering an environmental stress level. Particle-based Bayesian inversion methods are used to calibrate unknown model parameters with time resolved measurements of in vitro populations of liver cancer cells. While predictions of both calibrated models show good agreement with the data, the model considering the stress level yields a better fitting.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT02.html#author2"><strong class="card-body-subgroup">Mohit Kumar Jolly</strong></a>
		    	 
		    		<em>Indian Institute of Science</em>
		    	
		    	
		    		
		    			</br>"Topological signatures in regulatory network enable phenotypic heterogeneity in small cell lung cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Phenotypic (non-genetic) heterogeneity has significant implications for development and evolution of organs, organisms, and populations. Recent observations in multiple cancers have unravelled the role of phenotypic heterogeneity in driving metastasis and therapy recalcitrance. However, the origins of such phenotypic heterogeneity are poorly understood in most cancers. Here, we investigate a regulatory network underlying phenotypic heterogeneity in small cell lung cancer, a devastating disease with no molecular targeted therapy. Discrete and continuous dynamical simulations of this network reveal its multistable behavior that can explain co-existence of four experimentally observed phenotypes. Analysis of the network topology uncovers that multistability emerges from two teams of players that mutually inhibit each other but members of a team activate one another, forming a 'toggle switch' between the two teams. Deciphering these topological signatures in cancer-related regulatory networks can unravel their 'latent' design principles and offer a rational approach to characterize phenotypic heterogeneity in a tumor.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT02.html#author3"><strong class="card-body-subgroup">Meghan Rhodes</strong></a>
		    	 
		    		<em>University of Alberta</em>
		    	
		    	
		    		
		    			</br>"Comparing the effects of linear and one-term Ogden elasticity in a model of glioblastoma invasion."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present a model of glioblastoma (GBM) invasion which includes mass effects and tissue mechanics. Furthermore, we show how different brain tissue elasticity models affect the dynamics and invasion wave speed. Inspired by Budday et al. (2017) who mechanically tested brain tissue to determine an appropriate constitutive model of brain tissue mechanics, we explore two models: The linear elasticity model, and the one-term Ogden model. In a simplified 1D version of the model, we show the existence of travelling wave solutions. The traveling waves can be viewed as the invasion of GBM tumor cells into the surrounding healthy brain tissue. Thus, identifying the speed of the wave and how it is affected by model components and parameters is useful in determining what drives invasion. We show that although the wave speed is independent of the chosen mechanical model, the dynamics of GBM spread and the effects on surrounding brain tissue differ significantly between the linear and one-term Ogden elasticity models. Simulations predict opposite modes of GBM invasion depending on the mechanical model, with the linear and one-term Ogden models showing that GBM invades via either “pushing” or “pulling” on the surrounding tissue, respectively.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT02.html#author4"><strong class="card-body-subgroup">Matthias M. Fischer</strong></a>
		    	 
		    		<em>Charite Universitaetsmedizin Berlin, Institut fuer Pathologie; IRI Life Sciences, Humboldt University, Berlin, Germany</em>
		    	
		    	
		    		
		    			</br>"Mathematical modelling of colon epithelium population dynamics reveals conditions for maintaining tissue homoeostasis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The intestinal epithelium is one of the fastest renewing tissues in mammals and shows a remarkable degree of stability towards external perturbations such as physical injuries or radiation damage. This process is driven by intestinal stem cells as well as by differentiated cells being able to revert back to a stem cell state in situations of tissue regeneration. Self-renewal and regeneration, however, require a tightly regulated balance to uphold tissue homoeostasis, as failures in maintaining this balance may lead to tissue extinction or to unbounded growth, thereby giving rise to cancerous lesions.Here, we present and analyze a mathematical model of intestinal epithelium population dynamics. The model allows to derive conditions for stability and thereby helps to identify mechanisms that lead to loss of homoeostasis, causing either regenerative failure or unbounded, malignant growth. One of the key results is the existence of specific thresholds in feedbacks after which unbounded growth occur, and a subsequent convergence of the system to a stable ratio of stem to non-stem cells. Additionally, we demonstrate how allowing for dedifferentiation enables the system to recover more gracefully after certain external perturbations from equilibrium, however opens up another way to malignant growth.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT03.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT03-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT03.html#author1"><strong class="card-body-subgroup">Rafael Bravo</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Investigating the impact of tissue density on tumor growth and evolution in a 3D whole-organ model of lung cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A spatial mathematical model investigates how well-known tumor traits: increased resource consumption and angiogenesis, may alter tumor growth in different densities of normal tissue, using CT scan data to initialize the model.A novel modeling paradigm was developed specifically to model tissue at the resolution of CT imaging: the population-based model (PBM). The PBM uses discrete agents, however agents are compartmentalized into homogenous populations, which simplifies computation and allows modeling much larger populations than with conventional agent-based modeling. This PBM method allows us to seed a model with individual cells that operates at the CT scale of cubic millimeters. A virtual tumor can then be grown in this environment.Extensive parameter sweeping was done on different tumor phenotypes and normal tissue densities to assess how these affect marginal tumor growth rate and cell count.We find an optimal balance between angiogenesis and resource consumption by the tumor is needed to maximize invasiveness and tumor bulk, and that this balance changes depending on surrounding tissue density. These results suggest that such a balance may evolve in patient tumors and change depending on the density of the tissue on the tumor margin.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT03.html#author2"><strong class="card-body-subgroup">Stefano Pasetto</strong></a>
		    	 
		    		<em>H. Lee Moffitt Cancer Center & Research Institute</em>
		    	
		    	
		    		
		    			</br>"Intermittent hormone therapy models analysis and Bayesian-model-comparison for prostate cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The prostate is an exocrine gland of the male reproductive system dependent on androgens (testosterone and dihydrotestosterone) for development and maintenance. Since prostate cells and their malignant counterparts require androgen stimulation to grow, prostate cancer can be treated by androgen deprivation therapy (ADT). A significant problem in a continuous PCa ADT treatment at the maximum tolerable dose is the insurgence of cancer cell resistance; thus, intermittent adaptive therapy (IAT) is invoked to delay time to progression (TTP).Several mathematical models with different biological resistance mechanisms have been considered to simulate intermittent ADT treatment response dynamics. We present a comparison between 12 of these intermittent prostate-specific antigens (PSA) dynamical models over the Canadian Prospective Phase II Trial of IADT for locally advanced prostate cancer.We identified a few models with critical abilities to disentangle between relapsing and not relapsing patients, which can be exploited for clinical purposes. Finally, within the Bayesian framework, we detected the most compelling models in the trial description.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT03.html#author3"><strong class="card-body-subgroup">Kevin Murgas</strong></a>
		    	 
		    		<em>Stony Brook University Dept. of Biomedical Informatics</em>
		    	
		    	
		    		
		    			</br>"	 Hierarchical Modeling of DNA Methylation Conservation in Colon Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Conservation is broadly used to identify biologically important genomic regions. Indeed, preferential DNA methylation conservation during tumor growth can indicate areas of particular functional importance to the tumor. In a cohort of 21 colorectal cancer (CRC) patients with multiple tissue samples per patient, we measured methylation at over 850,000 CpG sites using the Infinium Methylation EPIC microarray. Next, we developed a Bayesian hierarchical model that allows for variance decomposition of methylation on 3 hierarchical levels built around the multiple tissue sampling. We fit the model to the CRC data using a Monte Carlo Markov Chain algorithm (Stan). Based on the posterior parameter distributions of the fits, we defined a conservation score to indicate reduced within-tumor variation of methylation relative to between-patient normal variation, thereby quantifying preferential methylation conservation at single CpG sites, individual genes, and molecular pathways. Across gene regulatory regions, preferential conservation was highest in the vicinity of gene transcription start sites and lowest at exon boundaries. Genes belonging to CRC gene sets exhibited increased preferential conservation, suggesting the model's ability to identify functionally relevant regions based on methylation conservation. A pathway analysis of significantly preferentially conserved genes implicated several CRC relevant pathways and pathways related to immune evasion.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT03.html#author4"><strong class="card-body-subgroup">Kathleen Wilkie</strong></a>
		    	 
		    		<em>Ryerson University</em>
		    	
		    	
		    		
		    			</br>"Chemotherapy Induced Cachexia: Insights from a Mathematical Model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cachexia is the loss of muscle and adipose tissues that directly correlates with patient energy levels, strength, and general quality of life.  Chemotherapy is a standard cancer treatment with notorious side effects including nausea, diarrhea, anorexia, and fatigue. Unfortunately, chemotherapy can also induce severe muscle loss. Cancer presence itself can induce cachexia, leading to a double-barrelled attack on healthy lean mass, and thus patient life quality.In this work, we develop a novel mathematical framework to investigate the response of muscle tissue to 5-FU chemotherapy. We model the role of stem cells in tissue maintenance and use the model to examine potential mechanisms of chemotherapy induced muscle loss, including disruption of the differentiation pathway. We confront our model to various treatment doses and dose schedules in an attempt to understand several qualitative features of chemotherapy-induced cachexia. In this talk I will review the model mechanisms we used to capture the qualitative features of the experimental data and discuss some of the computational challenges including parameterization of this dynamic process.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT04.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT04-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT04.html#author1"><strong class="card-body-subgroup">Jeffrey West</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Antifragile Therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We develop a novel paradigm of cancer therapy based on the 'anti-fragility' of cancer cell lines. Anti-fragility is a situation where the dose response function is convex. Treatment schedules with high variance of dose delivered result in maximum cell kill. For example, if the curvature is convex near a dose of 'x', continuous administration of x may have a less efficacious response compared to a regimen that switches equally between 120% and 80% of x, even though both regimens use the same total drug. We advocate for the need to disentangle first- and second-order treatment effects.Recent advances in personalized treatment scheduling known as 'adaptive' therapies typically result in a high level of variance in dose delivered in patients, similar to the theory behind anti-fragility. In this work we develop mathematical models of tumor pharmacodynamics (PKPD) and treatment resistance (Lotka-Volterra) to improve personalized dose protocols using principles from anti-fragile theory. PKPD dynamics are parameterized using in vitro dose response of H3122 non-small cell lung cancer cell lines confronted to ALK inhibitors. Competition between subpopulations (sensitive and resistant subclones) is the key determinant of optimal dose variance for individual patients. This work has implications for cancer therapy, antibiotics, and beyond.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT04.html#author2"><strong class="card-body-subgroup">Heiko Enderling</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Simulating tumor-immune ecosystem evolution during cancer radiotherapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Radiotherapy efficacy is the result of radiation-mediated cytotoxicity coupled with stimulation of anti-tumor immune responses. We developed an in silico three-dimensional agent-based model of diverse tumor-immune ecosystems (TIES) represented as anti- or pro-tumor immune phenotypes. We validate the model in 10,469 patients by demonstrating clinically-detected tumors have pro-tumor TIES. We then quantify the likelihood radiation induces anti-tumor TIES shifts by developing the individual Radiation Immune Sensitivity (iRIS), a novel biomarker. We show iRIS distribution across 31 tumor types is consistent with the clinical effectiveness of radiotherapy and predicts for local control and survival in a separate cohort of 59 lung cancer patients. This is the first clinically and biologically-validated model to represent the perturbation of the TIES by radiotherapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT04.html#author3"><strong class="card-body-subgroup">Emanuelle Arantes Paixão</strong></a>
		    	 
		    		<em>Laboratório Nacional de Computação Científica</em>
		    	
		    	
		    		
		    			</br>"CARTmath: an in silico laboratory to simulate CAR-T immunotherapy in preclinical models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			CAR-T cell immunotherapy has been obtaining expressive results in therapies against hematological cancers. Different antineoplastic targets are under investigation as well as therapy combinations with immune checkpoint blockade drugs, minimum effective CAR-T cell dose, memory pool formation, patient specificity, among others.  Many of these studies  require a preclinical proof-of-concept experiment using immunodeficient mouse models. Aiming at minimizing and optimizing in vivo experiments, we developed an open-source software in a Shiny R-based platform, named CARTmath. It allows simulating a three population mathematical model that represents the dynamics of tumor cells and effector and memory CAR-T cells in immunodeficient mouse models. Designed to be a friendly platform, even researchers unfamiliar with mathematical modeling can investigate the effects of different CAR-T cell immunotherapy protocols, types of tumors, immunosuppressive mechanisms, to mention a few, hopefully reducing in vivo experiments. CARTmath is available at github.com/tmglncc/CARTmath or directly on the webpage cartmath.lncc.br.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT04.html#author4"><strong class="card-body-subgroup">António Sergio Dias Morais</strong></a>
		    	 
		    		<em>Universidade de Coimbra</em>
		    	
		    	
		    		
		    			</br>"Role of prostate gland network structure in early stage prostate cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prostate cancer (PCa) is the second most frequent cancer in men. The limited individualization of the clinical management beyond risk-group definition leads to significant overtreatment/undertreatment rate. PCa is a paradigmatic condition in which an individualized predictive technology could make a difference in treatment.Mathematical modeling and simulation highlight the mechanisms behind disease progression. The prostate is a small organ with a structure composed by a network of glands within smooth muscle connectivity tissue. To explore the prostate structure in PCa growth we developed 2 mathematical models. The first, a 2D cellular Potts model (CPM), simulates the interactions between the different types of cells and the deformation of the glands in time. The second is a 3D phase-field model with tumor growth, prostate gland dynamics and nutrient consumption.The CPM gives important clues: how the cells and the glands rearrange locally in tumor growth. We import these insights to the 3D phase-field model to study how the adaptation of the grand morphology influences the lesion morphology.We conclude that the ramified structure of the prostate has a determinant impact in the tumor growth. The model parameter range that creates a ramified tumor phenotype is dramatically extended when prostate glands are considered.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT04" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT05.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT05-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT05.html#author1"><strong class="card-body-subgroup">Maalavika Pillai</strong></a>
		    	 
		    		<em>Indian Institute of Science, Bangalore</em>
		    	
		    	
		    		
		    			</br>"Systems-level analysis of phenotypic plasticity and heterogeneity in melanoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Phenotypic (i.e. non-genetic) heterogeneity in melanoma drives dedifferentiation, recalcitrance to targeted therapy and immunotherapy, and consequent tumor relapse and metastasis. Various markers or regulators associated with distinct phenotypes in melanoma have been identified, but, how does a network of interactions among these regulators give rise to multiple “attractor” states and phenotypic switching remains elusive. Here, we inferred a network of transcription factors (TFs) that act as master regulators for gene signatures of diverse cell-states in melanoma. Dynamical simulations of this network predicted how this network can settle into different “attractors” (TF expression patterns), suggesting that TF network dynamics drives the emergence of phenotypic heterogeneity. These simulations can recapitulate major phenotypes observed in melanoma and explain de-differentiation trajectory observed upon BRAF inhibition. Our systems-level modeling framework offers a platform to understand trajectories of phenotypic transitions in the landscape of a regulatory TF network and identify novel therapeutic strategies targeting melanoma plasticity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT05.html#author2"><strong class="card-body-subgroup">Jill Gallaher</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Using adaptive therapy to characterize collective and individual characteristics of metastases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Evolutionary-designed therapies, such as adaptive therapy, have been shown to be useful for late stage cancer to delay treatment resistance by exploiting competition between sensitive and resistant cells by alternating between drug applications and drug-free vacations. In addition, a single cycle of adaptive therapy could be used as a tool to probe tumor dynamics. In this work, we propose a framework for estimating individual and collective components of a metastatic system using tumor dynamics during adaptive therapy. We use a system of off-lattice agent-based models to represent individual metastatic lesions within independent domains, but subject to the same systemic therapy. We find the first cycle of adaptive therapy delineates several features of the metastatic system. Larger metastases have longer cycles, more drug resistance slows the cycles, and a faster cell turnover speeds up drug response time and slows the regrowth time. The number of metastases does not affect cycle times, as dynamics are dominated by the largest tumors rather than the aggregate. Changes in individual metastases gives insight on heterogeneity amongst metastases and can guide treatment. Generally, systems with more intertumor heterogeneity had better success with continuous therapy, while systems with more intratumor heterogeneity responded better to adaptive therapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT05.html#author3"><strong class="card-body-subgroup">Joshua Bull</strong></a>
		    	 
		    		<em>Wolfson Centre for Mathematical Biology, University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Novel spatial statistics describe phenotype transitions in an agent-based model of tumour associated macrophages"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumour associated macrophages adopt a range of phenotypes based on microenvironmental cues, with opposite ends of a spectrum of behaviours often summarised as “M1” (anti-tumour) and “M2” (pro-tumour). Transitions in macrophage phenotype play a role in cancer progression: for example, chemotactic gradients generated by macrophages of different phenotypes may be responsible for movement of tumour cells towards external vasculature and subsequent metastasis [1].We present an agent-based model based on [1] in which individual blood vessels, macrophages, tumour cells and stromal cells are resolved. Diffusible species in the tumour microenvironment determine macrophage phenotype, which we describe as a continuous variable. This variable governs phenotype specific macrophage behaviours such as phagocytosis and production of tumour cell chemoattractants. Our model reproduces patterns of macrophage localisation described in [1].Considering simulated macrophage locations as a point pattern, we develop novel spatial statistical techniques which account for points labelled with a continuous variable and hence identify how macrophage phenotype determines spatial localisation. This work suggests that spatial statistics accounting for real-valued labels could be used to better describe multiplexed imaging data, in which high or low expression of multiple markers can be identified from variations in stain intensity.[1] Arwert et al, 2018. doi:10.1016/j.celrep.2018.04.007.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT06.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT06-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT06.html#author1"><strong class="card-body-subgroup">Linnea C Franssen</strong></a>
		    	 
		    		<em>Roche, pRED, Basel</em>
		    	
		    	
		    		
		    			</br>"3D atomistic-continuum cancer invasion model: In silico simulations of an in vitro organotypic invasion assay"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We develop a three-dimensional hybrid atomistic-continuum model that describes the invasive growth dynamics of individual cancer cells in tissue. The framework explicitly accounts for phenotypic variation by distinguishing between cancer cells of an epithelial-like and a mesenchymal-like phenotype. It also describes mutations between these cell phenotypes in the form of epithelial-mesenchymal transition (EMT) and its reverse process mesenchymal-epithelial transition (MET). The model consists of a hybrid system of partial and stochastic differential equations that describe the evolution of epithelial-like and mesenchymal-like cancer cells, respectively, under the consideration of matrix-degrading enzyme concentrations and the extracellular matrix density. With the help of inverse parameter estimation and a sensitivity analysis, this three-dimensional model is then calibrated to an in vitro organotypic invasion assay experiment of oral squamous cell carcinoma cells.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT06.html#author2"><strong class="card-body-subgroup">Jakob Rosenbauer</strong></a>
		    	 
		    		<em>Forschungszentrum Jülich</em>
		    	
		    	
		    		
		    			</br>"In silico model of evolution in heterogeneous tumors and the influence of the microenvironment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In heterogeneous tumors, cell types of different properties compete over the available resources, that are nutrients and space. Rapid expansion leads to solid stress in in-vivo tumors that can collapse blood vessels, which together with angiogenesis leads to fluctuations in nutrient availability. Here, we observe the influence of such fluctuations on tumor evolution.We developed a 3D computational model that simulates the evolutionary trajectories of an evolving tumor. Cell motility and cell-cell adhesion are observed as free evolving parameters in tumor cells that grow in a medium of surrounding cells. A nutrient dependent cell cycle is introduced and constant and dynamic nutrient surroundings are compared.We find an evolutionary advantage of low adhesion cells independent of the surrounding. Furthermore we find a dependency between the evolution speed and the frequency of the nutrient fluctuations, with a significant increase of evolutionary speed for a frequency domain.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT06.html#author3"><strong class="card-body-subgroup">Alvaro Köhn-Luque</strong></a>
		    	 
		    		<em>Oslo Centre for Biostatistics and Epidemiology, Faculty of Medicine, University of Oslo</em>
		    	
		    	
		    		
		    			</br>"Deconvolution of drug-response heterogeneity in cancer cell populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In ex vivo drug-sensitivity assays, cells are treated with varying drug concentrations and viable cells are measured at one or more time points. Viability curves, and their characteristics (e.g. IC50), allow comparing drug sensitivity across multiple drugs and cell samples. However, the interpretation of those curves is confounded by the presence of cellular heterogeneity in each sample. The presence of several subclones with different drug sensitivities results in an aggregated drug-sensitivity profile that does not represent the cell population complexity, and thus hinders the design of precise treatment strategies.Here we show how to infer on the presence of cellular subclones with differential drug response, using standard cell viability data at total population level. We build cell population dynamic models of the evolution of individual subclones over time and dose. We estimate the number of subclones, their mixture frequencies and drug-response profiles. We validate our methodology on data from admixtures of synthetic and actual cancer cells at known frequencies. Finally, we explore the clinical usefulness of the method for multiple myeloma patients.This is joint work with J. Foo, K. Leder, A. Frigessi, E.M. Myklebust, J. Noory, S. Mumenthaler, D.S. Tadele, M. Giliberto, F. Schjesvold,  J. Enserink and K. Tasken.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT06.html#author4"><strong class="card-body-subgroup">Michael Raatz</strong></a>
		    	 
		    		<em>Max Planck Institute for Evolutionary Biology, Germany</em>
		    	
		    	
		    		
		    			</br>"Of slow cells and slower decline – Phenotypic heterogeneity and treatment type in cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is largely recognized that tumours consist of a diverse population of cancer cells. Treatment exerts selection on the phenotype and may shift the distribution of characteristic functional traits within the population. Taking the underlying phenotypic trait distribution into account, given for example by the growth rate of individual cells, allows to predict and compare the performance of different treatment options. Here, we investigate how treatment that is either growth rate selective or unselective affects a population of cancer cells with diverse growth rates. We find that different treatment types result in different cancer cell population dynamics and trait distributions. Further, we find that accounting for phenotypic diversity allows to select optimal treatment patterns for specific targets. To increase the likelihood of tumour eradication, the maximum mortality should be exerted on the cancer cell population. If tumour eradication is not achievable, maximizing the time until relapse may be achieved by a very different treatment strategy that aims not for maximum cancer cell mortality but rather for a specific, desirable trait distribution. It thus becomes evident that combining a trait-based approach with considering the phenotypic diversity of cancer allows for mechanistic understanding of cancer dynamics and optimization of personalized treatment.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT07.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT07-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT07.html#author1"><strong class="card-body-subgroup">Pirmin Schlicke</strong></a>
		    	 
		    		<em>Technical University of Munich, Germany</em>
		    	
		    	
		    		
		    			</br>"Bringing math into medical clinics: a model framework quantifying treatment outcomes in metastatic cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Since roughly 90% of lung cancer deaths occur due to the presence of metastases and their resulting symptoms. Therefore the identification and evaluation of metastases is of utter importance for optimal treatment. Modern imaging technology leaves most metastases undiscovered as their size is too small to be recognised. Nonetheless, they play an important role in therapy success. We quantified the metastatic size distribution in cancer patients and estimated the effects of possible different treatment applications. The framework presented is a coupled ODE/PDE model based on a McKendrick-von-Foerster equation introduced by Iwata et al. and modified along its characteristics to account for therapeutic effects in treatment applications. The continuous definition also allows to model the metastatic cascade, thus the transition of a single primary tumor to a metastatic disease. These simulations could help clinicians to compare outcomes and to choose among treatment possibilities based on different therapy goals. Retrospective analysis with clinical data allows for follow-up prognostic possibilities that will be shown in this presentation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT07.html#author2"><strong class="card-body-subgroup">Daniel Glazar</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Predicting Advanced Head and Neck Cancer Patients with High Risk of Early Treatment Failure"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			IntroductionThere is a need to discover better treatment strategies for patients with advanced head and neck squamous cell carcinoma (HNSCC). 45 patients with advanced HNSCC were treated with combination cetuximab (anti-EGFR) and nivolumab (anti-PD-1) every 2 weeks with a 2-week lead-in of cetuximab alone in a phase I/II clinical trial. However, not every patient responded to the protocol therapy. Therefore, there is a clinical need to identify high-risk patients. MethodsAvailable patient-specific information includes CT-derived sum of longest diameters every 8 weeks. We train a tumor growth inhibition (TGI) ODE model describing a uniform growth rate and initial treatment sensitivity and patient-specific rate of evolution of resistance. We forecast tumor burden and predict risk level at the second and third observations.ResultsThe TGI model is able to accurately represent tumor burden dynamics (R2=0.98). However, forecasts for tumor burden are rather poor. However, since our main concern is predicting risk level, we continue with the study and achieve decent predictions at second and third observations (n=25,14 patients, accuracy=0.64,0.71, respectively).ConclusionGiven enough on-treatment information, a clinician can use the TGI model to predict high-risk patients on the trial protocol.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT07.html#author3"><strong class="card-body-subgroup">Maximilian Strobl</strong></a>
		    	 
		    		<em>University of Oxford & Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Using eco-evolutionary modelling to improve the management of PARPi resistance in ovarian cancer maintenance therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			PARP inhibitors (PARPis) represent a great advancement in the treatment of ovarian cancer, yet these drugs still often fail after a few months due to emerging drug resistance. A recent clinical trial in prostate cancer showed that evolutionary-inspired, adaptive drug scheduling significantly delayed time to progression. This approach modulated treatment to maintain a pool of drug-sensitive cells that suppress resistant cells through competition. Here, we present results from a combined modelling and experimental study in which we investigated whether adaptive therapy can delay resistance to the PARPi Olaparib. We performed a series of in vitro experiments in which we used time-lapse microscopy to characterise the cell population dynamics under different PARPi schedules. Our work reveals a delay in drug response, and that cells recover quickly upon drug withdrawal. Thus, treatment interruptions or modulations need to be carefully timed. To explain this behaviour we develop an ODE model which attributes the dynamics to the fact that PARPis induce cell cycle arrest from which cells may still recover. This model can not only fit the in vitro data, but it also accurately predicts the response to unseen drug schedules. We conclude with in silico trials of a plausible adaptive PARPi strategy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT07.html#author4"><strong class="card-body-subgroup">Ryan Murphy</strong></a>
		    	 
		    		<em>Queensland University of Technology</em>
		    	
		    	
		    		
		    			</br>"Looking beneath the surface of tumour spheroids: insights from mathematical models parameterised to experimental data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In 1972 H. P. Greenspan proposed one of the first mathematical models to describe avascular tumour spheroid growth. He suggested that his work be experimentally validated when improved technology was available. Remarkably, even though his paper has been highly influential and well-cited it has not yet been experimentally validated. In this presentation we will directly connect the Greenspan model to experimental data for the first time. Using live-dead cell staining and fluorescent ubiquitination-based cell cycle indicator (FUCCI) technology, we reveal and measure necrotic, quiescent, and proliferative regions inside growing tumour spheroids. These novel data, that we collect across a number of initial tumour spheroid sizes, cell lines, and experimental designs, allows us to test the Greenspan model and form confidence intervals for its parameters.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT09.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT09-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT09.html#author1"><strong class="card-body-subgroup">Mohammad Zahid</strong></a>
		    	 
		    		<em>H. Lee Moffitt Cancer Center & Research Institute</em>
		    	
		    	
		    		
		    			</br>"In Silico Trial to Estimate Personalized Radiotherapy Dose in Head and Neck Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Current radiotherapy (RT) treatment schedules are not personalized for individual patients, with the prescribed dose being uniform for particular subtypes and stages of cancer, despite variable responses between patients.  Our objective is to determine optimal personalized RT dose in order to minimize excess RT dose without sacrificing tumor control.  Weekly tumor volume data were collected for 39 head and neck cancer patients from Moffitt and M.D. Anderson Cancer Centers that received RT over 6-8 weeks.  Tumor growth was modeled as logistic growth, and the effect of RT was modeled as an instantaneous reduction in carrying capacity. Tumor volume reduction was connected to locoregional control (LRC) by a volume reduction threshold associated with LRC.The in silico trial was performed in a leave-one-out fashion where model parameters calibrated to tumor volume data from N-1 patients and then the calibrated model parameters were combined with the Nth patient's tumor volume data from weeks 1-4 of RT to simulate tumor volumes forward in order to estimate minimum dose required for LRC. We found that 87% of the patients received a higher total dose than estimated as necessary by our model, while the remaining patients were estimated to have received too little dose.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT09.html#author2"><strong class="card-body-subgroup">Linus Schumacher</strong></a>
		    	 
		    		<em>University of Edinburgh</em>
		    	
		    	
		    		
		    			</br>"Mutational fitness in age-related clonal haematopoiesis quantified from longitudinal data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The production of blood can be disturbed by mutations in haematopoietic stem cells (HSCs). Though mostly inconsequential, some mutations confer fitness advantages resulting in growth of fitter clones (all progeny of a HSCs carrying the same mutation) that represent disproportionately large fractions of all blood cells. Clonal Haematopoiesis of Indeterminate Potential (CHIP) affects more than 10% of the population aged over 65 years and is currently diagnosed when 4% of blood cells carry the same mutation. CHIP is linked with a ten-fold increase in later onset of haematological cancers, highlighting the importance of detecting and predicting clonal growth early.We investigate CHIP in the Lothian Birth Cohort through targeted error-corrected sequencing of blood samples taken from participants every 3 years.Modelling the population dynamics of clones shows the commonly used threshold to diagnose CHIP can be reached due to neutral drift in synonymous mutations. This clinical detection method therefore leads to a ~50% false discovery rate of fit mutations. Using longitudinal data, we instead detect clones whose growth exceeds the distribution of fluctuations of neutral mutations. This allows us to uncover fitness-inducing mutations with high sensitivity and detect highly fit mutations before they achieve the threshold-based definition of CHIP.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT09.html#author3"><strong class="card-body-subgroup">Adam Malik</strong></a>
		    	 
		    		<em>Uppsala University</em>
		    	
		    	
		    		
		    			</br>"Using modelling to quantify the diversity of glioblastoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma grade IV is a highly aggressive form of brain cancer, with a short duration of survival after diagnosis even in the presence of treatment. A challenge with surgical removal is the diffuse nature of tumors, and the difficulty of removing the whole tumor when cancer cells have migrated away from the primary tumor site. During migration, cells are influenced by their microenvironment, and it has been observed that cells tend to migrate along white matter tracts or towards blood vessels. A great variation in growth patterns is found when glioblastoma cells from different patients are grown ex vivo in the brain of mice. In order to quantify these differences we construct an agent based model of tumor growth in the brain of mice. We make use of diffusion tensor imaging data to obtain information about the white matter tracts, as well as a dataset of the whole brain vasculature. The migration direction is biased by either the white matter, the blood vessels or both. The model is fitted to experimental data using Approximate Bayesian Computation to provide insights into the differences between both proliferation as well as migratory preference for white matter or blood vessels.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT09.html#author4"><strong class="card-body-subgroup">Marek Bodnar</strong></a>
		    	 
		    		<em>Institute of Applied Mathematics and Mechanics, University of Warsaw, Banacha 2, 02-097 Warsaw, Poland</em>
		    	
		    	
		    		
		    			</br>"On the optimal use of bevacizumab in unresected glioblastoma: An evidence-based mathematical approach"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma (GBM) is the most common and aggressive type of brain tumor in adults, with a median patient survival slightly above one year, despite aggressive combination therapy with the alkilating agent temozolomide (TMZ) and radiation therapy. Phase III clinical trials of the combination of bevacizumab (BEV, anti-angiogenic drug) with the standard chemoradiation protocol were negative in terms of providing survival improvements. In a very interesting study, Balaña et al. (2016) sought to determine the impact of BEC on reduction of tumor size prior tochemoradiotherapy treatment in unresected GBM patients. They found that the combination of BEV and TMZ was more active than TMZ alone and may confer benefit in terms of tumor shrinkage in unresected patients. We propose a simple mathematical model of tumor growth taking into account hypoxic cells and treatments (radiotheraphy, chemotherapy and anti-angiogenic treatment). We study mathematical properties of the model. Moreover, we show that solution of the model mimic well results of clinical trials.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
	                <a id="Posters"><h1 class="mb-3" id="MS1">Sub-group poster presentations</h1></a>
									<div class="row subgroup-name">
										<div class="col-12">
											<div class="card">															
										  	<h5 class="card-header card-header-subgroup ONCO">ONCO Posters</h5>
											  <div class="card-body">
											    <p class="card-text">																
														<div>
															
													    	
																	<p><small class="card-body-subgroup">ONCO-1 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-1.html">
																    	</br><strong>Elias Siguenza</strong>
														    	 
														    		<em>University of Birmingham</em>
														    	
														    	
														    		
														    			</br>"Feeding the Habit: The metabolic relationship between bone marrow mesenchymal stems cells and multiple myeloma"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Multiple myeloma (MM) is an incurable malignant disease of plasma cells with the poorest 5-year survival of any haematological malignancy. Bone marrow (BM) residency of malignant plasma cells is an absolute requirement for their survival and proliferation, suggesting that the microenvironment within this niche is a critical driver of disease. We previously showed that the metabolism of the BM is significantly altered in patients with MM, and that the BM mesenchymal stem cell (BMMSC), the major supportive cell type for malignant plasma cells, was significantly and irreversibly transformed. We hypothesise that these two cell types form a co-operative metabolic network within the BM that is critical for the survival and proliferation of malignant plasma cells. If true, then targeting this metabolic communication will directly impact on disease progression and response to therapy, improving patient outcomes. We created a testable model of the metabolic network formed by malignant plasma cells and BMMSCs. Our model will identify enzymes or transporters that represent hubs, the inhibition of which would result in a breakdown of the community and sensitisation to standard therapeutic approaches to treating this incurable cancer.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-10 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-10.html">
																    	</br><strong>Arran Hodgkinson</strong>
														    	 
														    		<em>University of Exeter</em>
														    	
														    	
														    		
														    			</br>"Population Scale Spatio-Structural Modelling of Directed Cancer Invasion"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			As treatments for cancer continue to elude the biomedical community, and although it is well documented that cancer cells exert significant forces to dynamically rearrange the extra-cellular matrix (ECM), there remains a need for tumour, or population, scale mathematical models fit for spatial comparison to in vivo data. Employing a spatio-structural partial differential equation (PDE) framework, we are able to model the tissue scale dynamics resulting from tensile forces exerted by the cell population upon the ECM and the subsequent invasion of cells into their local environments. We also develop qualitative methods to theoretically explore the effects of alignment between cell polarisation and ECM fibre orientation on the invasive displacement of cancer sub-clusters. Numerical results show the multi-dimensional capacity of cells to reorient the fibrous ECM environment and invade the local tissue, where initial conflict between the cellular polarisation and fibre alignment impedes this process. The model also demonstrates the emergent phenomenon of structural heterogeneity from near-homogeneous initial conditions. This modelling framework provides a novel opportunity for the quantitative exploration of the biochemical and spatial processes of cancer invasion, whilst the resulting images provide an interesting candidate for comparison with robust experimental results.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-11 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-11.html">
																    	</br><strong>Ghanendra Singh</strong>
														    	 
														    		<em>Indraprastha Institute of Information Technology Delhi</em>
														    	
														    	
														    		
														    			</br>"Accelerate Replication Fork velocity to kill cancer cells"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Replication fork plays an important role during DNA replication. During DNA replication Fork movement rate increases during S phase in mammalian cells and also reduces during replication stress. There exists a molecular mechanism through which the replication fork adjusts their speeds during the S phase. In cancer cells, DNA replication is slower compared to normal cells and replication forks move slowly. So, this work proposes that if the replication fork speed can cross a particular threshold, the cancer cells won't be able to cope with the DNA replication and die in the process. A mechanistic mathematical model has been developed based on recent experimental findings. Mec1, Rad53 and Mrc1 are needed to create a positive feedback loop to stabilize replisome during stalled forks.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-2 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-2.html">
																    	</br><strong>Subbalakshmi A R</strong>
														    	 
														    		<em>Indian Institute of Science</em>
														    	
														    	
														    		
														    			</br>"A computational systems biology approach identifies SLUG as a mediator of partial Epithelial-Mesenchymal Transition (EMT)"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Epithelial-mesenchymal plasticity comprises of reversible transitions among epithelial, hybrid epithelial/mesenchymal (E/M) and mesenchymal phenotypes, and underlies various aspects of aggressive tumor progression such as metastasis, therapy resistance and immune evasion. The process of cells attaining one or more hybrid E/M phenotypes is termed as partial EMT. Cells in hybrid E/M phenotype(s) can be more aggressive than those in either fully epithelial or mesenchymal state. Thus, identifying regulators of hybrid E/M phenotypes is essential to decipher the rheostats of phenotypic plasticity and consequent accelerators of metastasis. Here, using a computational systems biology approach, we demonstrate that SLUG (SNAIL2) – an EMT-inducing transcription factor – can inhibit cells from undergoing a complete EMT and thus stabilizing them in hybrid E/M phenotype(s). It expands the parametric range enabling the existence of a hybrid E/M phenotype, thereby behaving as a phenotypic stability factor (PSF). Our simulations suggest that this specific property of SLUG emerges from the topology of the regulatory network it forms with other key regulators of epithelial-mesenchymal plasticity. Clinical data suggests that SLUG associates with worse patient prognosis across multiple carcinomas. Together, our results indicate that SLUG can stabilize hybrid E/M phenotype(s).
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-3 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-3.html">
																    	</br><strong>Ielyaas Cloete</strong>
														    	 
														    		<em>Brighton & Sussex Medical School</em>
														    	
														    	
														    		
														    			</br>"Tackling mutational heterogeneity in DLBCL through mathematical modelling"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Diffuse large B cell lymphoma (DLBCL) is the most common subtype of non-Hodgkin lymphoma, accounting for nearly 40% of diagnosed non-Hodgkin lymphomas. DLBCL is clinically is further classified, based on putative cell-of-origin, into activated B cell-like (ABC) and germinal centre B cell-like (GCB) subtypes, each with distinct gene expression profiles and clinical outcomes. However, a recent genetic and molecular study of patient samples reveals substantial heterogeneity beyond these classifications. Despite an increasing understanding of the signalling dysregulation leading to DLBCL, standard combination therapy (R-CHOP) has remained unchanged for more than a decade. However, roughly 40% of patients treated with R-CHOP develop a recurring or progressive disease that is usually fatal. The substantial heterogeneity likely leads to our standard treatment failing for many. Thus we leverage this mutational characterisation to recreate, using a mathematical model, genetic events seen in distinct patient samples to determine B cell fates given each genetic alteration. In particular, we are interested in modelling the mutational heterogeneity to create `virtual DLBCL patients' and use this model as a tool to identify molecular targets and biomarkers that cluster patients based on their optimal target for therapy.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-4 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-4.html">
																    	</br><strong>Marc Vaisband</strong>
														    	 
														    		<em>University of Bonn</em>
														    	
														    	
														    		
														    			</br>"Validation of genetic variants from NGS data using Deep Convolutional Neural Networks"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			A crucial aspect of analysing next-generation sequencing (NGS) data from cancer patients lies in identifying mutations in the genetic code of tumor cells. This is done by considering the tumor DNA together with a reference germline sample, and inferring candidate somatic mutations by way of comparison. A multitude of tools exist for this purpose. In practice, however, sequencing artifacts or alignment errors are often mistakenly flagged as variants, necessitating extremely time-consuming manual validation by researchers.We demonstrate that this process can be largely automated using Deep Convolutional Neural Networks, whose utility has been a driving force behind many recent advances in applied machine learning. Using previously performed manual annotation as input data, we train a Deep Convolutional Neural Network of straightforward topology that recognises sequencing artifacts in called variants with high accuracy, achieving a score of 97.5% on a validation dataset. Moreover, its direct outputs are class probabilities instead of binary labels, and the remaining misclassified points lie in the region of low certainty, suggesting an effective modelling of the decision behaviour in manual annotation. This allows for a significant reduction in the workload for researchers, and can in the future be integrated into bioinformatics workflows for NGS data processing.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-5 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-5.html">
																    	</br><strong>Arran Pack</strong>
														    	 
														    		<em>Brighton and Sussex Medical School</em>
														    	
														    	
														    		
														    			</br>"Overcoming receptor proximal mutations in DLBCL through systems modelling"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			B Cell Receptors react to antigen stimulus, transducing signal to NFkB, and triggering an immune response. Inoue et al[1] modelled this pathway in health. B Cell Lymphoma frequently displays chronically active NFkB caused by mutations in this receptor-proximal signalling pathway. We sought to mechanistically investigate the impact of mutation on BCR signalling and predict therapeutic interventions.A common mutation in CARD11 was modelled by modifying parameters corresponding with experimentally-identified effects of this mutation. In response to stimulation, the mutant CARD11 model is significantly more active; and switches to chronically active when the basal BCR signal exceeds a very low level (<1%). This low activation threshold suggests that the CARD11 mutation alone may be sufficient for chronic NFkB activation.By performing parameter sensitivity analysis on the mutant model; the parameters with the greatest influence on NFkB were identified. This approach identified several IKK reactions which have been the target of much therapeutic development. Unfortunately, due to the ubiquitous expression of NFkB and lack of specificity, development of these inhibitors has generally stalled. This work is motivating ongoing expansion of the model to include the Cheng et al 2015 model of Toll Like Receptor signalling, to increase the number of mutable/druggable targets.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-6 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-6.html">
																    	</br><strong>Sara Hamis</strong>
														    	 
														    		<em>School of Mathematics and Statistics, University of St Andrews, St Andrews, Scotland, UK.</em>
														    	
														    	
														    		
														    			</br>"Mathematical modelling quantifies ERK-activity in response to vertical inhibition of the BRAFV600E-MEK-ERK cascade in melanoma"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Vertical inhibition of the BRAF-MEK-ERK cascade has become a standard of care for treating BRAF-mutant melanoma. However, the molecular mechanisms of how vertical inhibition synergistically suppresses intracellular ERK activity, and by extension cell proliferation, are yet to be fully elucidated. In this study, we develop a mechanistic mathematical model that describes how the BRAFV600E-inhibitor dabrafenib, and the MEK-inhibitor trametinib target the BRAFV600E-MEK-ERK cascade. We formulate a system of chemical reactions that describes cascade signalling dynamics. Using mass action kinetics, the system of chemical reactions is re-expressed as a system of ordinary differential equations, which we solve numerically to obtain the temporal evolution of cascade component concentrations.Our mathematical model provides a quantitative method to compute how dabrafenib and trametinib can be used in combination to synergistically inhibit ERK activity in BRAFV600E mutant cancer cells. Our work elucidates molecular mechanisms of vertical inhibition of the BRAFV600E-MEK-ERK cascade, and delineates how elevated cellular BRAF concentrations generate drug resistance to dabrafenib and trametinib. In addition, our model results suggest that elevated ATP levels lead to reduced sensitivity to dabrafenib.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-7 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-7.html">
																    	</br><strong>Ryan Schenck</strong>
														    	 
														    		<em>Integrated Mathematical Oncology, Moffitt Cancer Center; Intestinal Stem Cell Biology Lab, Wellcome Centre for Human Genetics, University of Oxford</em>
														    	
														    	
														    		
														    			</br>"Reconstructing Contemporary Human Stem Cell Dynamics with Oscillatory Molecular Clocks"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Cell histories can be reconstructed from their genomes by analysing 'molecular clocks' that accumulate heritable changes through time. Commonly used clocks, such as the accumulation of single nucleotide variants or DNA methylation, slowly change over decades, recording cell dynamics that occur over long timescales corresponding to the change accumulation (tick) rate. Faster clocks saturate and stop recording early in life, precluding the study of short-timescale cell dynamics such as renewal in adult tissues. Here we develop a new method that can measure contemporary adult cell dynamics with rapidly oscillating CpG DNA methylation, where like a pendulum, ongoing 'tick-tock' (de)methylation causes switching between 0, 50 and 100% methylation at each CpG locus in a diploid cell. In polyclonal cell populations, average oscillator methylation is ~50%, but “W-shaped” distributions with modal peaks at 0, 50 and 100% methylation are evident in clonal populations. The precise shape of the W-distribution is determined by the underlying dynamics of cell growth and replacement. Through our work, we've illustrated oscillator DNA methylation can be measured in many human tissues cheaply and routinely and enables the inference of otherwise elusive contemporary dynamics of normal and abnormal somatic cells.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-8 (Session: PS01)</small>
															    	<a href="/ONCO/ONCO-PS01-ONCO-8.html">
																    	</br><strong>Veselin Manojlović</strong>
														    	 
														    		<em>School of Mathematics, Computer Science and Engineering, City, University of London</em>
														    	
														    	
														    		
														    			</br>"Evolutionary Indeces for Classifying Modes of Tumour Evolution"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Trees are a useful mathematical tool in evolutionary theory, especially when describing the structure of an evolving population. In mathematical oncology, clonal trees can be used to construct the evolutionary history of a tumour, and tree balance and diversity indices further classify it within an oncoevotype.While obtaining diversity and balance indices for a specific tree is straightforward, going in reverse is not a trivial matter. One of our main goals is classifying clonal trees based on a minimal set of indices. To this end, we explore properties of new and existing balance and diversity indices, along with their mutual dependencies. This should lay the groundwork for further investigation of the possibility of defining an index metric, which would mark a step towards an analytic formulation of mathematical oncology.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-12 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-12.html">
																    	</br><strong>Maria Eliza Antunes</strong>
														    	 
														    		<em>Graduate Program in Biometrics - São Paulo State University</em>
														    	
														    	
														    		
														    			</br>"Numerical simulations for a metastatic papillary thyroid cancer model using RAI 131I treatment"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			In this work, we numerically simulated a mathematical model to study A metastatic papillary thyroid cancer (PTC) response to different periodic radioiodine 131I (RAI) treatment.  Within the simulated scenarios we consider different values for the RAI efficiency ratio.  Besides that, periodic treatment protocols with the same dose were considered and also with decreasing amount of doses, with a higher dose first, followed by smaller ones. Some protocols failed to decrease the number of tumor cells, where can understand as a resistance towards RAI treatment conditions by the lack of response. These failures may mean a poorly structured treatment protocol regarding the type of therapy, doses, application intervals, or any of their combinations. Notably, RAI treatment scenarios with alternating dosages presented a successful treatment response, i.e., tumor elimination. Therefore, mathematical models are essential tools in the study of cancer biology and could assist in determining the most suitable treatment protocols, including for the metastatic PTC.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-13 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-13.html">
																    	</br><strong>Hannah Anderson</strong>
														    	 
														    		<em>University of Florida</em>
														    	
														    	
														    		
														    			</br>"Team Approach to Integrating Mathematical and Biological Models to Target Myeloid-Derived Immune Cells in Glioblastoma"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Objective: Integrate mathematical models of immunosuppressive glioblastoma (GBM) infiltrating myeloid cells with experimental data to predict therapeutic responses to combined chemokine receptor and immune checkpoint blockade.Methods: Orthotopic murine KR158-luc gliomas were established in mice. Subsequently, an anti-CD31 injection was administered to label the vasculature. Fluorescent imaging and quantification of anti-CD3 stained sections were performed on a range of tumor sizes to acquire vasculature, tumor, T cell, and myeloid cell densities. In parallel, a system of ordinary differential equations was formulated based on biological assumptions to evaluate the change over time of tumor cells, T cells, and infiltrating myeloid cells. The model was then refined and validated by experimental results.Results: Fluorescent imaging and quantification revealed a correlation between tumor size and abundance of myeloid cell populations in the tumor microenvironment. The density of these cell populations and vasculature remained constant as the tumors increased in size. Computer simulations of the mathematical model will predict tumor, myeloid, and T cell dynamics. These simulations will be particularly useful to understand myeloid cell dynamics, such as cell entry time into the tumor microenvironment. Parameter sensitivity analysis of the model will inform us of the biological processes driving these tumor-immune cell dynamics.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-14 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-14.html">
																    	</br><strong>Jessica Kingsley</strong>
														    	 
														    		<em>University of South Florida</em>
														    	
														    	
														    		
														    			</br>"Bridging cell-scale simulations and radiologic images to explain short-time intratumoral oxygen fluctuations"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Radiologic images provide a longitudinal way to monitor tumor responses, but operate on a macroscopic scale and are not able to capture microscopic scale phenomena. We provide a link between the average data recorded in radiological image voxels and the tissue architecture that fills these voxels. Our in silico model includes individual tumor and stromal cells, tumor vasculature, and tumor metabolic landscape. This architecture was based on tissue characteristics acquired from electron paramagnetic resonance (EPR) images.  We used this model to optimize vascular influx and cellular uptake schedules to reproduce oxygen fluctuations recorded experimentally. By comparing simulation results within the schedules, we showed that sole alterations in vascular influx were able to reproduce experimental data well. On the other hand, in order to fit experimental data with metabolic changes in tumor cells, the cells would need to increase their oxygen absorption by 50-fold over a period of 3 minutes, which may not be biologically feasible. Additionally, we developed a procedure to identify plausible tissue morphologies for a given temporal series of average data from radiology images. In the future our approach can be used to simulate hypoxia-sensitive anti-cancer treatments on a cell-scale based on clinically-collected images.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-15 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-15.html">
																    	</br><strong>Rebecca Bekker</strong>
														    	 
														    		<em>H. Lee Moffitt Cancer Center & Research Institute</em>
														    	
														    	
														    		
														    			</br>"Investigating inter-replicate differences in cancer wound healing assays using an agent-based model"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Cellular migration, and thus motility, are important factors in a tumors ability to metastasize. Wound healing assays are a way of quantifying these properties in vitro, while mathematical modeling can be used to do so in silico. In silico models can be calibrated and validated on the collected migration data, and used to predict the effects of therapeutics on the migration of cancer cells. We focus on the murine cell lines TC-1 and mEER, both transformed using the oncogenes HPV16 E6, HPV16 E7 and hRAS. Wound healing assays were performed on these cell lines after irradiation with 0Gy, 2Gy, 8Gy and 10Gy to quantify the dose dependent effects of radiation on the motility of the cell lines. Herein we report on a calibrated agent-based model used to investigate how assumptions about the underlying distributions of migration speed data affects in silico experiments. Additionally, we discuss whether combining the replicates per experiment lends itself to more accurate predictions than using each data set individually.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-16 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-16.html">
																    	</br><strong>Javier Urcuyo</strong>
														    	 
														    		<em>Mayo Clinic</em>
														    	
														    	
														    		
														    			</br>"Exploring the Glioblastoma-Immune Dynamics with Mathematical Modeling and Transcriptome Sequencing"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Glioblastoma (GBM) is a deadly, heterogeneous disease with poor overall survival. Adding to the complexity of the disease, the tumor-immune environment is also heterogeneous. Glioma-associated macrophages and microglia (GAMMs) can exhibit either a tumor-suppressive or tumor-permissive response, resulting in a variety of different GBM growth patterns. However, the mechanism by which GAMMs affect GBM growth remains unclear. To explore the potential dynamics of these tumor-GAMM interactions, we created four candidate mathematical models, each capturing a different biological mechanism for the conversion between GAMM phenotypes. To better understand the parameters influential on tumor growth, we performed a sensitivity analysis. Initial analyses indicate that, beyond the growth kinetics of the tumor, the initial population of tumor-suppressive GAMMs is influential on tumor velocity. This preliminary finding is somewhat surprising, as it suggests that changes to the relative abundance of immune populations over time would not significantly impact the tumor growth. In future work, we plan to utilize deconvolution techniques on RNAseq from image-localized biopsies to identify relative cellular-subtype compositions and investigate if tumor growth kinetics are dependent on current GAMM composition. By developing a better understanding of the tumor-immune interface, we can aid in identifying potential immunotherapy strategies and in assessing their effectiveness.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-17 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-17.html">
																    	</br><strong>Chandler Gatenbee</strong>
														    	 
														    		<em>Moffitt Cancer Center</em>
														    	
														    	
														    		
														    			</br>"Immune escape at the onset of human colorectal cancer"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The evolutionary dynamics of tumor initiation remain undetermined, and the interplay between neoplastic cells and the immune system is hypothesized to be critical in transformation. Colorectal cancer (CRC) presents a unique opportunity to study the transition to malignancy as pre-cancers (adenomas) and early stage cancers are frequently detected and surgically removed. Here, we examine the role of the immune response in tumor initiation by studying tumor-immune eco-evolutionary dynamics from pre-cancer to carcinoma using a computational model, ecological analysis of digital pathology, and multi-region neoantigen prediction. Observed changes in antigenic intra-tumor heterogeneity (aITH), the tumor ecology, and spatial patterns of both cell associations and gene expression are consistent with simulations where immunogenic adenomas do not progress to CRC because they are under immune control. Conversely, adenomas that progress initially avoid detection through low immunogenicity, but gradually construct an immunosuppressive niche isolated from CD8+ cytotoxic T cells, thereby evading immune elimination and allowing for an increase in neoantigen burden. Both modeling and data indicate that immune blockade (e.g. PD-L1 expression) plays a secondary role to immune suppression in tumor initiation or progression. These results suggest that re-engineering the immunosuppressive niche may prove to be a most effective immunotherapy in CRC.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-18 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-18.html">
																    	</br><strong>Lee Curtin</strong>
														    	 
														    		<em>Mayo Clinic</em>
														    	
														    	
														    		
														    			</br>"Discerning Glioblastoma Subpopulation Interactions through In Vitro Experiments and Mathematical Modeling"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Glioblastoma is the most aggressive primary brain tumor with dismal median survival. Glioblastoma is known to be heterogeneous with multiple molecularly-distinct subpopulations, however, both the baseline change in growth with a given mutation and the compounded change due to dynamic interactions between subpopulations remain unknown. It is important to characterize these interactions for their potential impact on treatment resistance. Using both in vitro data and mathematical modeling, we aim to determine the interaction between two more-common glioblastoma subpopulations: amplification of epidermal growth factor receptor (EGFR) and platelet-derived growth factor receptor alpha (PDGFRA). As a preliminary study, two glioblastoma cell lines, LN229 and GBM22, were identified with similar genetic profiles. Then, two variants of each of these lines were developed to over-express EGFR and PDGFRA, resulting in 6 molecularly-distinct cell lines in total. Variants of each cell line were allowed to grow both independently, and separately in co-culture with their sister variant. Preliminary calibration of an exponential model with the independently-grown cell culture data demonstrated that the variant influence on the growth rate was different between the cell lines. Future analysis will involve calibrating a system of reaction-diffusion equations to identify the effects of co-culturing these subpopulations on the growth kinetics.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-19 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-19.html">
																    	</br><strong>Brydon Eastman</strong>
														    	 
														    		<em>University of Waterloo</em>
														    	
														    	
														    		
														    			</br>"Reinforcement learning derived chemotherapeutic schedules for robust patient-specific therapy given unknown patient response parameters"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			When developing a chemotherapeutic dosing schedule for treating cancer in-silico one relies upon a parameterization of a particular tumour growth model to describe the dynamics of the cancer in response to the dose of the drug. It is often prohibitively difficult, in practice, to ensure the validity of patient-specific parameterizations of these models for any particular patient. As a result, sensitivities to these particular parameters can result in therapeutic dosing schedules that are optimal in principle not performing well on particular patients. In this study, we demonstrate that chemotherapeutic dosing strategies learned via reinforcement learning methods are more robust to perturbations in patient-specific parameter values than those learned via classical optimal control methods. By training a reinforcement learning agent on mean-value parameters and allowing the agent access to a more easily measurable metric, relative bone marrow density, we are able to develop drug dosing schedules that outperform schedules learned via classical optimal control methods, even when such methods are allowed to leverage the same bone marrow measurements.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-20 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-20.html">
																    	</br><strong>Janielly Matos Vieira</strong>
														    	 
														    		<em>Graduate Program in Biometrics -São Paulo State University</em>
														    	
														    	
														    		
														    			</br>" Mathematical model of mestastasis involving immunotherapy with CAR T cells"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The essence of cancer is characterized by the disordered growth of cells having the ability to invade tissues and injured adjacent organs. Classified as a world health problem, cancer lies among the four leading death causes worldwide, being metastasis responsible for over 90% of all cancer-related deaths. In order to eradicate the disease, several therapies are under development, being the immunotherapy on prominence for not causing severe damages to the normal cells, reinforcing the patient's immune system so it can better fight the cancer cells.  Using differential equations, in this work, we proposed a mathematical model of mestastasis involving immunotherapy with CAR T cells. In the model, metastasis is modeled as a migratory phenomenon, where two populations of cancer cells coexist and develop in two different locations.Through numerical simulations, we studied the tumor dynamics in different scenarios, where these were obtained by varying the initial condition of the tumor cells and in the amount of CAR T cells used.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-21 (Session: PS02)</small>
															    	<a href="/ONCO/ONCO-PS02-ONCO-21.html">
																    	</br><strong>Thomas Veith</strong>
														    	 
														    		<em>Moffitt Cancer Center</em>
														    	
														    	
														    		
														    			</br>"Spatial Constraints On In-Vitro Cancer Cell Line Growth"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Population density puts a constraint on cell growth through the mechanism of contact inhibition. Dysregulation of this process is one of the hallmarks of cancer. Contact inhibition is correlated with expression of pathways associated with E-cadherin signaling such as Hippo and Wnt, or inhibition of mTOR signaling. We observe heterogeneous expression of these pathways across and within nine metastatic gastric cancer cell lines via single cell RNA sequencing data. Our working hypothesis is that high or low population densities will benefit certain cells over others, selecting for subclonal populations in an evolutionary process known as density dependent selection. Here, we present a method which integrates cell culturing experiments with single cell sequencing data to investigate the effects spatial constraints have on subclonal growth of cancer cells in-vitro. Our results evaluate the usefulness of population density as an informer of subclonal growth dynamics in a data driven, mechanistic model.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-22 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-22.html">
																    	</br><strong>Ellen Swanson</strong>
														    	 
														    		<em>Centre College</em>
														    	
														    	
														    		
														    			</br>"Enhancing CAR-T Immunotherapy to Attack Both Tumor and Cancer Stem Cells"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The stem cells hypothesis suggests that there is a small group of malignant cells, the cancer stem cells, that initiate the development of tumors, encourage its growth, and may even be the cause of metastases.   Traditional treatments, such as chemotherapy and radiation, primarily target the tumor cells leaving the stem cells to potentially cause a recurrence.  Chimeric antigen receptor (CAR) T-cell therapy is a form of immunotherapy where the immune cells are genetically modified to fight the tumor cells.Traditionally, the CAR T-cell therapy has been used to treat blood cancers and only recently has shown promising results against solid tumors. We create an ODE model which allows for the infusion of trained CAR-T cells to specifically attack the cancer stem cells that are present in the solid tumor microenvironment.  Additionally, we incorporate the influence of TGF-Beta which has a both a regulatory and promotion effect on the growth of the tumor.  We verify the model by comparing it to available data and then examine different immunotherapy treatments that attack the tumor cells, stem cells, and both.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-23 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-23.html">
																    	</br><strong>Stefano Casarin</strong>
														    	 
														    		<em>Houston Methodist Research Institute</em>
														    	
														    	
														    		
														    			</br>"Depicting Radium223 therapy efficacy for prostate cancer bone metastasis: a mathematical modeling approach"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Radium223 (Rad223) has lately improved survival in prostate cancer bone metastatic patients. However, clinical response is often followed by relapse and disease progression, and associated mechanisms of efficacy and resistance are poorly understood. Research efforts to overcome this gap require substantial time and resource investment. Computational models, integrated with experimental data, can overcome this limitation and drive research in a more effective fashion. Accordingly, we developed an agent-based model of prostate cancer bone metastasis progression and response to Rad223 as an agile platform to maximize its efficacy. The driving coefficients were calibrated on ad hoc experimental observations retrieved from intravital microscopy and the outcome further validated, in vivo. The in silico tumor growth matched in vivo trend with 98.3% confidence. Tumor size determined efficacy of Rad223, with larger lesions insensitive to therapy, while medium- and micro-sized tumors displayed up to 5.02 and 152.28-fold size decrease compared to control-treated tumors, respectively. Eradication events occurred in 65 ± 2% of cases in micro-tumors only. In addition, Rad223 lost any therapeutic effect, also on micro-tumors, for distances bigger than 400 μm from the bone interface. This model has the potential to be further developed to test additional bone targeting agents such as other radiopharmaceuticals or bisphosphonates.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-24 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-24.html">
																    	</br><strong>Emily Yang</strong>
														    	 
														    		<em>The University of Texas at Austin</em>
														    	
														    	
														    		
														    			</br>"Characterizing Phenotypic Dynamics of Chemoresistance in Breast Cancer Cells"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Despite the continuing advancements in chemotherapy for breast cancer, the development of chemoresistance remains a significant cause of treatment failure. Intratumoral and inter-patient heterogeneity pose critical challenges to designing patient-specific optimized treatment plans. We posit that a mathematical understanding of chemoresistance dynamics could be key to developing clinically-feasible, successful treatment regimens. In this study, we develop a model that describes the effects of a standard neoadjuvant drug (doxorubicin) on a common breast cancer cell line (MCF7). We assume that the tumor is composed of two subpopulations: drug-resistant cells, which continue proliferating after treatment, and drug-sensitive cells, which gradually transition from proliferating to treatment-induced death. The model is calibrated with time-resolved microscopy data including variations in drug concentration, inter-treatment interval, and number of doses. Our results show that the model can recapitulate tumor growth dynamics in all of these scenarios (CCC>0.95). We observe that higher dosages resulted in significantly lower drug-resistant fraction and proliferation rates (p<0.05). Our results also show superior tumor control with a higher number of doses and shorter inter-treatment intervals (p<0.05). Thus, we believe that our model may contribute to our understanding of doxorubicin action and may guide the selection of therapeutic regimens that achieve optimal tumor control.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-25 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-25.html">
																    	</br><strong>Joshua Scurll</strong>
														    	 
														    		<em>University of British Columbia</em>
														    	
														    	
														    		
														    			</br>"A new inter-cluster similarity measure for high-dimensional data can facilitate analysis of heterogeneous mass-cytometry data"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Mass cytometry (CyTOF) is a high-dimensional, high-throughput technology to analyze quantities of 30–40 proteins simultaneously in single cells and is widely used to investigate heterogeneity in tumour tissue samples. Also, by detecting phospho-proteins, (phospho-)CyTOF can be used to investigate intracellular signalling-pathway activity. Analysis of CyTOF data usually involves clustering and visualization of the high-dimensional data, which are typically performed using independent methods and are strongly influenced by user-controlled input parameters. To reduce user influence on CyTOF analysis results and harmonize the clustering and visualization processes, I developed ASTRICS, a new measure of similarity between clusters of high-dimensional data points based on local dimensionality reduction and triangulation of alpha shapes. I propose a multi-stage clustering strategy called CytoClue in which ASTRICS is used to automatically generate a weighted graph from an initial set of fine-grained clusters, which are obtained by an existing algorithm (e.g. FlowSOM) and are represented by graph nodes. The graph is visualized by force-directed layout and used for further clustering by a graph-based algorithm. This poster introduces ASTRICS and CytoClue and presents results of applying them to phospho-CyTOF experiments that were conducted to investigate heterogeneity between and within diffuse large B-cell lymphoma (DLBCL) cell lines.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-26 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-26.html">
																    	</br><strong>Alexander Brummer</strong>
														    	 
														    		<em>Department of Computational and Quantitative Medicine, Division of Mathematical Oncology, Beckman Research Institute, City of Hope National Medical Center</em>
														    	
														    	
														    		
														    			</br>"Destabilization of CAR T-cell treatment efficacy in the presence of dexamethasone"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			CAR T-cell therapy has proven to be a highly effective targeted immunotherapy for glioblastoma multiforme.  Yet, there is presently little known about the efficacy of CAR T-cell treatment when combined with the widely used anti-inflammatory and immunosuppressant glucocorticoid Dexamethasone.  We present an analysis of glioblastoma organoid cell lines treated along gradients of CAR T-cell therapy and Dexamethasone.  We demonstrate that Dexamethasone antagonizes CAR T-cell efficacy, and promotes tumor growth and recurrence. Advanced experimental technologies allow for highly resolved temporal dynamics of in vitro studies of combination therapies of CAR T-cell and Dexamethone, making this a valuable model system for studying the rich dynamics of nonlinear biological processes with translational applications.   We model the system as a non-autonomous, two-species, type I, predatory-prey interaction of tumor cells and CAR T-cells, with explicit time-dependence in the clearance rate of Dexamethasone.  Using time as a bifurcation parameter, we show that (1) the presence of Dexamethasone destabilizes coexistence equilibria between CAR T-cells and tumor cells and (2) as Dexamethasone is cleared from the system, a stable coexistence equilibrium returns in the form of a Hopf bifurcation.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-27 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-27.html">
																    	</br><strong>John Metzcar</strong>
														    	 
														    		<em>Indiana University</em>
														    	
														    	
														    		
														    			</br>"Impacts of communication length-scale on cellular invasion of tumor stroma using a novel ECM model"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Understanding long-range cellular migration is key to an understanding of cancer metastasis as well as other biological processes such as embryogenesis and formation of tissue morphology. Due to constraints on cellular and tissue level imaging, gaining that understanding as it emerges from cell-cell and cell-environmental interactions is challenging. To aid in gaining that understanding with respect to the physical environment, we developed a novel mathematical extracellular matrix (ECM) model capturing three components of ECM: fiber orientation, alignment, and density. This model is notable in its stripping down the ECM to its most essential components. We implemented this model in PhysiCell, a cell-based modeling framework, thus allowing coupling between it and cell-agents. We then explored rules of interactions between cells, their physical environment (the ECM), and the diffusive chemical environment. We observed that locally sensed, long range (on the order of a few cell diameters) physical signals communicated via ECM remodeling alter the dynamics of cellular invasion, producing new results. Comparing these results to simulations lacking long-range permanently written signals, we see more cellular trafficking and movement. This advance brings us one step closer to understanding processes of collective cellular migration and closer to understanding the initial steps of cancer metastasis.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-28 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-28.html">
																    	</br><strong>Nathan Lee</strong>
														    	 
														    		<em>University of Washington, Department of Applied Mathematics</em>
														    	
														    	
														    		
														    			</br>"Inferring parameters of cancer evolution from sequencing and clinical data"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia (CLL) patients. Fitting our model to longitudinal patient data reveals that the first driver mutation typically occurs very early in life in patients that go on to develop CLL, and that the appearance of the first driver mutation and the diagnosis of CLL are typically separated by 30-50 years.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-29 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-29.html">
																    	</br><strong>Jaewook Joo</strong>
														    	 
														    		<em>Cleveland Clinic Foundation</em>
														    	
														    	
														    		
														    			</br>"Optimal path to fluctuation-driven extinction of tumors with phenotypic switching in temporally varying environment"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Phenotypic plasticity/switching, in contrast to irreversible genetic intratumor variation, allows cancer cells to engage adaptive responses to the changed tumor microenvironment in a reversible fashion and begins to emerge as the explanatory mechanisms for therapy resistance. We consider a simple stochastic model of reversible phenotypic switching between two phenotypic states, a growing yet sensitive state and a quiescent yet tolerant state, in response to a time-varying tumor microenvironmental cue. An environmental stress induces the switching to a quiescent state and the absence of stress reverses its state back to a growing state. We utilized the WKB theory of large deviations, i.e., the eikonal approximation to the master equation, to find the optimal path to fluctuation-induced tumor extinction. This path to tumor extinction is forbidden in a deterministic model, but a rare event with large fluctuations brings the system from its long-lived quasi-stationary state to extinction. We also presented the necessary conditions for the temporal modulation of the environmental stress that could reduce the mean time to extinction exponentially.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-30 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-30.html">
																    	</br><strong>Harsh Jain</strong>
														    	 
														    		<em>University of Minnesota Duluth</em>
														    	
														    	
														    		
														    			</br>"Simulating Heterogeneity in Deterministic Models of Prostate Cancer Response to Immunotherapy with Standing Variations Modeling"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Advanced, hormonally refractive prostate cancer is typically treated with docetaxel, a chemotherapeutic compound that inhibits cell division. However, this treatment eventually fails due to onset of resistance. Multiple mechanisms driving docetaxel resistance have been identified and several drugs targeting these mechanisms are in various stages of clinical trial, including immunotherapy in the form of a vaccine. However, optimizing strategies to overcome such resistance remains a critical challenge because the problem is inherently multiscale due to characteristic variability at the subcellular, cellular and individual levels. In this talk, I present a simple dynamical systems model of prostate cancer response to immunotherapy. I then introduce our novel modeling approach, Standing Variations Modeling, which exploits uncertainty and variability in data to inform the probability distributions - rather than specific values - from which model parameters most likely arise. This differs from traditional modeling approaches that only use static or mean expression levels and cellular responses, thereby ignoring the significant variance that exists across cell populations as well as individuals being treated. Sampling from these posterior distributions allows us to generate a virtual cohort of individuals on which in silico clinical trials are conducted to predict optimal dosing combinations and subpopulations that would benefit most from such an intervention.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">ONCO-31 (Session: PS04)</small>
															    	<a href="/ONCO/ONCO-PS04-ONCO-31.html">
																    	</br><strong>Heber Rocha</strong>
														    	 
														    		<em>Indiana University</em>
														    	
														    	
														    		
														    			</br>"A multiscale model for tumoral vascular growth: blood flow and cell dynamics"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Vascularization is a fundamental factor in the progression of tumor growth. The vascular phase is characterized by angiogenesis, responsible for the growth of new blood vessels in tumor direction from an original vascular network.  Through an additional supply of nutrients, the tumor acquires unlimited resources for its uncontrolled progress. Besides, the risk of metastasis increases with the eventual invasion of blood vessels by cancer cells. In this work, we developed a multiscale model to represent the growth of vascular tumors, integrating angiogenesis and blood flow dynamics. Dynamics on the cell scale are represented discretely using agent-based modeling, while oxygen dispersions and pro-angiogenic factors are modeled using partial diffusion-reaction equations. Blood flow is obtained by solving Kirchhoff's circuit equations for flow in a connected network. The new blood vessels are formed through a set of rules based on stimuli of factors secreted by cancer cells submitted to the oxygen structure. The coupling between the models can capture phenomena that occur at the cellular and tissue scales, qualitatively representing the growth of vascular tumors. The computational model was developed using PhysiCell, an open-source C ++ framework that allows the construction of multicellular models at various scales in 2D and 3D.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	    	
											  		</div>
													</p>
											  </div>
											</div>
										</div>
									</div>
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
					Click to show abstracts
									</span>		            
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
