---
layout: group
author: esmb
day: "Tuesday (Wednesday)"
subgroup: "CBBS"
title: "Modeling and data analysis of dynamics from molecules, cells to populations"
organizers: "Lei Zhang (Peking University, China)"
description: "The systems biology approach integrating heterogeneous biological data in quantitative mathematical models has promised to facilitate the comprehensive understanding of complex biological systems. This A3 (China-Japan-Korea) minisymposium is to bring together asian mathematicians working in the field of mathematical modeling and data analysis of dynamic phenomena at all kinds of levels on molecules, cells to populations."
code: "MS09"
code2: "MS10"
author1: "Hao Ge"
inst1: " (Peking University, China)"
title1: "The Nonequilibrium Mechanism of Noise-Enhanced Drug Synergy in HIV Latency Reactivation"
abstract1: "The “shock and kill” strategy has become a promising way to cure HIV by eliminating latent HIV reservoirs, the main barrier to a clinical cure. Recently, single-cell screening experiments have shown the Noise-enhanced drug synergy on reactivating latent HIV. However, the underlying biomolecular mechanism is still a mystery. We propose here a generic model for HIV regulation and Tat transcription/translation. Using this model, we find out that the drug synergy is mainly determined by the magnitude and direction of energy input into the genetic regulatory kinetics of HIV promotor. We further show that the Noise-enhanced drug synergy requires the timescale of HIV promoter entering into a transcriptionally non-permissive state without drugs presented to be slower than the timescale of Tat transactivation. Our model reveals a generic nonequilibrium mechanism underpinning the Noise-enhanced drug synergy, which is useful for improving the drug effect and identifying other drug synergies on lentivirus latency reactivation."
author2: "Yusuke Imoto"
inst2: " (Kyoto University, Japan)"
title2: "Topological Trajectory Inference for Single-cell RNA Sequencing Data"
abstract2: "This study develops a framework that extracts single-cell differentiation structures from single-cell RNA sequencing data (scRNA-seq data) by using a topological data analysis method, Mapper [G. Singh et al., SPBG 91 (2007)]. Because the scRNA-seq data is quite high-dimensional and contains technical noise, the scRNA-seq data analysis encounters the inconsistency of computational values between true and observed data due to the accumulation of noise; this problem is known as the curse of dimensionality. Since requiring a clustering in the high-dimensional space, Mapper is also affected by the curse of dimensionality. To overcome the problem, this study proposes the procedure using a statistical noise reduction method for scRNA-seq data, as the preprocessing of the Mapper. In this talk, we will verify the effect of the noise reduction method in Mapper and show some applications to biological data. Moreover, we will introduce a visualization method to help with biological inference by using biological metadata."
author3: "Suoqin Jin"
inst3: " (University of California Irvine, U.S.)"
title3: "Understanding the role of cell-cell communication in cell fate decisions from single-cell data"
abstract3: "Cell-cell communication via soluble and membrane-bound factors is critical for informing diverse cell fate decisions, including decisions to activate programmed cell death, undergo migration or differentiate along the lineage. Single-cell RNA-sequencing (scRNA-seq) technologies have led to discovery of cellular heterogeneity and differentiation trajectories at unprecedented resolution level. scRNA-seq data inherently contains gene expression information on signaling crosstalk between cells. This offers an unprecedented opportunity for comprehensively understanding how cell-cell communication drives diverse cellular decisions in tissues.  In this talk, I will take about our recent efforts in how by applying systems biology and machine learning approaches, we can quantitatively build and analyze cell-cell communication networks in an easily interpretable way. Applying our framework to scRNA-seq datasets of embryonic mouse skin, we identify previously unrecognized signaling mechanisms regulating melanocyte migration during early hair follicle formation. Our framework can be potentially incorporated into cell lineage-based mechanistic models to further deepen our understanding of the signaling dynamics in cell fate decisions."
author4: "Dae Wook Kim"
inst4: " (Korea Advanced Institute of Science and Technology, Korea)"
title4: "Moment-based inference of cell-to-cell variability in signal transduction time"
abstract4: "As experimentally measuring biochemical reaction rates in single cells is costly and time-consuming, they are often estimated by fitting a mathematical model to time-lapse live-cell imaging data, which are relatively easy to measure. However, this is often limited because only the final output of a series of reactions (e.g. matured protein) can be observed. In this case, a series of hidden intermediate reactions can be replaced with a distributed time delay. However, the estimation of the delay distribution has remained challenging as models with the delay are non-Markovian. Here, we develop a moment-based Bayesian inference method for accurate and efficient estimation of the delay distribution in single-cell signal transduction by using queuing theory and mixed effects modeling. By applying our method to single-cell fluorescence trajectories that are the final output of cellular response to antibiotic stress, we find considerable magnitude of cell-to-cell heterogeneity in signal amplification rate and transduction delay of the stress. Surprisingly, we also find that the magnitude of cell-to-cell heterogeneity in signal amplification rate is positively correlated with the number of rate-limiting molecular steps underlying the stress response. To allow systematic estimation of the signal transduction time, we provide a user-friendly computational package, namely CMBI."
---
