---
layout: group
author: esmb
day: "Thursday (Friday)"
subgroup: "MFBM"
title: "Algebra, Combinatorics, and Topology in Modern Biology"
organizers: "Daniel Cruz (Georgia Institute of Technology, U.S.), Margherita Maria Ferrari (University of South Florida, U.S.)"
description: "Over the last few years, research at the interface of mathematics and biology has proven to be a powerful catalyst for advancing each of the individual fields by yielding new tools, discoveries, and open questions. In particular, techniques from algebra, combinatorics, topology, and related areas have complemented more mainstream approaches in mathematical biology while becoming natural tools for understanding biological structures and interactions. This mini-symposium will focus on recent developments and open problems involving the application of algebra, combinatorics, and/or topology to topics in biology. These topics include RNA branching, signaling networks, COVID-19 detection, and phylogenetics, among others. Our intention is not only to facilitate discussion and collaboration but also to promote inclusivity within the associated research communities. As such, we have invited a diverse group of junior and senior speakers with complementary expertise which includes a significant number of women mathematicians."
code: "MS19"
code2: "MS20"
author1: "Margherita Maria Ferrari"
inst1: " (University of South Florida, U.S.)"
title1: "Formal grammar modeling three-stranded DNA:RNA braids"
abstract1: "A formal grammar is a system to generate words; it consists of a set of symbols, partitioned into terminals and non-terminals, and a set of production rules. The production rules specify how to rewrite non-terminal symbols, so that successive applications of those rules yield words formed by only terminals. Adding probabilities to the production rules defines stochastic grammars, which can be used for biological sequence analysis. In this talk, we focus on a 'braid grammar' to model R-loops, that are three-stranded structures formed by a DNA:RNA hybrid plus a single strand of DNA, often appearing during transcription. R-loops are described as strings of terminal symbols representing the braiding of the strands in the structure, where each symbol corresponds to a different state of the braided structure. We discuss approaches to develop a stochastic grammar and a probabilistic model for R-loop prediction, as well as refinements of the model by incorporating the effect of DNA topology on R-loop formation"
author2: "Svetlana Poznanovic"
inst2: " (Clemson University, U.S.)"
title2: "Using Polytopes to Improve RNA Branching Predictions"
abstract2: "Minimum free energy prediction of RNA secondary structures is based on the Nearest Neighbor Thermodynamics Model. While such predictions are typically good, the accuracy can vary widely even for short sequences, and the branching thermodynamics are an important factor in this variance. Recently, the simplest model for multiloop energetics - a linear function of the number of branches and unpaired nucleotides - was found to be the best. We develop a branch-and-bound algorithm that finds the set of optimal parameters with the highest average accuracy for a given set of sequences. The search uses the  branching polytopes for RNA sequences. Our analysis shows that previous ad hoc parameters are nearly optimal for tRNA and 5S rRNA sequences on both training and testing sets. Moreover, cross-family improvement is possible but more difficult because competing parameter regions favor different families. The results also indicate that restricting the unpaired nucleotide penalty to small values is warranted. This reduction makes analyzing longer sequences using the present techniques more feasible."
author3: "Chad Giusti"
inst3: " (University of Delaware, U.S.)"
title3: "Comparing Topological Feature Coding Across Neural Populations"
abstract3: "A common feature of the types of information neural populations in the brain encode is cyclicity, meaning that the data is well-represented by one or more independent circular coordinate systems. Persistent homology, a common tool from topological data analysis can be applied to detect and study representations of cyclic features in neural population activity, even without reference to a behavioral correlate. Recent advances in experimental techniques have led to simultaneous recording of activity from populations of neurons across several brain regions, providing an opportunity to study how these representations propagate and change as they move through the brain. Classical topological wisdom tells us that we should apply functoriality to compare topological features across locations. However, in this setting  the goal is, in effect, to impute the map we would need in order to do so. Here, we present a novel method for comparing topological features detected in different brain regions leveraging dissimilarity matrices obtained from observations of activity. No background in topology and neuroscience on behalf of the audience will be assumed."
author4: "Abdulmelik Mohammed"
inst4: " (University of South Florida, U.S.)"
title4: "Topological Eulerian Circuits for the Design of DNA Nanostructures"
abstract4: "Graph theory has recently emerged as a powerful framework for the automated design of biomolecular nanostructures. A prime example of this is in the design of wireframe DNA origami nanostructures, where the routing of a circular viral DNA, called a scaffold strand, is modeled as an Eulerian circuit of a reconditioned triangulated mesh. In this setting, the knot type of the scaffold strand dictates the feasibility of an Eulerian circuit to be used as the scaffold route in the design. We investigate the knottedness of Eulerian circuits on surface-embedded graphs to characterize the class of such graphs that are constructible from unknotted and knotted scaffold strands. We show that certain graph embeddings, called checkerboard colorable, always admit unknotted Eulerian circuits. On the other hand, we prove that if a graph admits an embedding in a torus such that the embedding is not checkerboard colorable, then the graph can be re-embedded so that all its non-intersecting Eulerian circuits are knotted. For surfaces of genus greater than one, we present an infinite family of checkerboard-colorable graph embeddings where there exist knotted Eulerian circuits."
---
