---
layout: group
author: esmb
day: "Tuesday (Wednesday)"
subgroup: "MFBM"
title: "Stochastic Systems Biology: Theory and Simulation"
organizers: "Jae Kyoung Kim (Department of Mathematical Sciences, KAIST, Republic of Korea), Ramon Grima (University of Edinburgh, United Kingdom)"
description: "Biological systems consist of a large number of species with reactions which occur in multiple spatio-temporal scales. Because performing stochastic simulations of such systems are computationally expensive and prohibitive, various strategies to reduce the computational cost have been investigated, e.g. quasi-state-state approximation or hybrid methods. Also it is of interest how the interaction between different scales, e.g. between cellular and tissue scales, affects noise at the single cell level. In this mini-symposium, the focus will be on recent research reporting on advances in this area. Furthermore, the application of these methods to investigate embryonic development, cell size homeostasis and cell movement will be presented."
code: "MS11"
code2: "MS12"
author1: "Zhixing Gao"
inst1: " (Key Laboratory of Advanced Control and Optimization for Chemical Processes, Ministry of Education, East China University of Science and Technology, China)"
title1: "Neural network aided approximation and parameter inference of stochastic models of gene expression"
abstract1: "Non-Markov models of stochastic biochemical kinetics often incorporate explicit time delays to effectively model large numbers of intermediate biochemical processes. Analysis and simulation of these models, as well as the inference of their parameters from data, are fraught with difficulties because the dynamics depends on the system’s history. Here we use an artificial neural network to approximate the time-dependent distributions of non-Markov models by the solutions of much simpler time-inhomogeneous Markov models; the approximation does not increase the dimensionality of the model and simultaneously leads to inference of the kinetic parameters. The training of the neural network uses a relatively small set of noisy measurements generated by experimental data or stochastic simulations of the non-Markov model. We show using a variety of models, where the delays stem from transcriptional processes and feedback control, that the Markov models learnt by the neural network accurately reflect the stochastic dynamics across parameter space."
author2: "Abhyudai Singh"
inst2: " (University of Delaware, USA)"
title2: "Modeling stochasticity in timing of intracellular events: A first-passage time approach"
abstract2: " How the noisy expression of regulatory proteins affects timing of intracellular events is an intriguing fundamental problem that influences diverse cellular processes. Here we use the bacteriophage lambda to study event timing in individual cells where cell lysis is the result of expression and accumulation of a single protein (holin) in the Escherichia coli cell membrane up to a critical threshold level. Site-directed mutagenesis of the holin gene generated phage variants that vary in their lysis times from 30 to 190 min. Observation of the lysis times of single cells reveals an intriguing finding—the noise in lysis timing first decreases with increasing lysis time to reach a minimum and then sharply increases at longer lysis times. A mathematical model with stochastic expression of holin together with dilution from cell growth was sufficient to explain the non-monotonic noise profile and identify holin accumulation thresholds that generate precision in lysis timing."
author3: "Thomas Prescott"
inst3: " (Alan Turing Institute, United Kingdom)"
title3: "Learning a multifidelity simulation strategy for likelihood-free Bayesian inference."
abstract3: "Likelihood-free Bayesian inference is a popular approach to calibrating complex mathematical models typical of biological systems, where likelihoods are often intractable. However, being reliant on repeated model simulation, the complexity that prohibits the likelihood calculation can also cause these methods to suffer from a significant computational burden. Multifidelity inference methods have been shown to reduce this burden by exploiting approximate simulations, such as coarser numerics or lower-dimensional models. By incorporating both high- and low-fidelity simulations, computational savings can be achieved without introducing any further bias in the resulting likelihood-free posterior. Instead, these approaches are forced to trade between reducing computational burden and increasing estimator variance. This trade-off is balanced by optimally assigning a simulation budget between the models at different fidelities. We will discuss how the optimal multifidelity simulation strategy can be learned in parallel with the posterior, and the multifidelity algorithm thus adaptively tuned as the posterior is uncovered."
author4: "Ruben Perez-Carrasco"
inst4: " ( Imperial College London, United Kingdom)"
title4: "Should we care about cell cycle variability when studying stochastic gene expression?"
abstract4: "Many models of stochastic gene expression do not incorporate a cell cycle description. I will show how this can be tackled analytically studying how mRNA fluctuations are influenced by DNA replication for a prescribed cell cycle duration stochasticity. Results show that omitting cell cycle details can introduce significant errors in the predicted mean and variance of gene expression for prokaryotic and eukaryotic organisms, reaching 25% error in the variance for mouse fibroblasts. Furthermore, we can derive a negative binomial approximation to the mRNA distribution, indicating that cell cycle stochasticity introduces similar fluctuations to bursty transcription. Finally, I will show how disregarding cell cycle stochasticity can introduce inference errors in transcription rates bigger than 10%."
---
