---
layout: group
author: esmb
day: "Thursday (Friday)"
subgroup: "CDEV"
title: "Dynamics and networks in single-cell biology"
organizers: "Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)"
description: "This minisymposium will discuss current mathematical and theoretical approaches to address open questions in single-cell biology. As single-cell genomics technologies advance, computational data analysis becomes one of the greatest hurdles to biological discovery. As the field begins to mature, and standards slowly emerge, the most pressing mathematical challenges shift from core tasks -- such as normalization and clustering -- to higher-level tasks. New advances in several areas will be presented in this minisymposium, including: network inference, dynamical systems approaches, modeling across scales, spatial transcriptomics, and multi-modal data integration."
code: "MS20"
code2: "MS19"
author1: "Amy Brock"
inst1: " (University of Texas at Austin, USA)"
title1: "Clonally-resolved mapping of cancer cell trajectories under therapeutic pressure"
abstract1: "Heterogeneity across individual cancer cells and clonal populations impacts growth rate, tumor composition, and response to therapy. To improve treatment, new tools are required to measure and control the contributions of diverse cell subpopulations. Our lab has developed a high-complexity expressed barcode system, ClonMapper, that integrates expressed cell barcoding with single-cell RNA-sequencing and clonal isolation to characterize and track subpopulation trajectories. Using this approach, we uncovered subsets of cells from breast cancer cell models with distinct transcriptional signatures and chemotherapy survivorship trajectories. To gain a deeper understanding of the process of clonal diversification, we profiled clones and retrieved sub-clones over the course of expansion and treatment. Supervised learning indicated that clonal subpopulations have characteristic transcriptomic signatures that are well-conserved under a variety of therapeutic perturbations. By providing the capability for systematic dissection of complex clonal dynamics, ClonMapper enables the delineation of an underlying engine of clonal diversification in cancer cell populations and refines our understanding of clonal identity."
author2: "Meghan Ferrall-Fairbanks"
inst2: " (University of Florida, USA)"
title2: "Single-cell eco-evolutionary dynamics of intratumor heterogeneity"
abstract2: "Researchers have recognized that a one-size-fits-all approach is not effective at treating cancer and that tumor heterogeneity plays an important role in response. Current dogma stipulates that this heterogeneity results from compounding genetic and epigenetic changes and instability, ultimately driving unfavorable outcomes for these patients. Nonetheless, some cancers, including many pediatric cancers and some leukemias, have limited genomic diversity. As a result, we have a limited ability to stratify patients into high versus low-risk groups. To address this, we explore intratumor heterogeneity at the single-cell transcriptomic level to quantify and identify driving phenotypes in tumor evolution. We leverage the generalized diversity index (GDI) from ecology, which allows us to tailor the scale of cellular diversity in a given context. We show that the order of diversity parameter in GDI allows us to either emphasize clonal richness at low values while high values shift the analysis toward the abundance of potential drivers of the tumor evolution. We have explored GDI changes in both in vitro sequential single-cell RNA sequencing samples across many cancer tissue types including breast, lung, and ovarian cell lines as well as treatment naïve and treated patient samples in chronic myelomonocytic leukemia. Our analyses show how quantifying intratumor heterogeneity with GDI is a powerful tool to understand eco-evolutionary dynamics of a patient’s tumor."
author3: "Stephen Williams"
inst3: " (10X Genomics, USA)"
title3: "Analyzing spatial and high-resolution single cell multi-omic data"
abstract3: "Chromium Single Cell and Visium Spatial Solutions from 10x Genomics provide the ability to build a more comprehensive multidimensional understanding of complex biological systems. Analyzing datasets that measure different molecular modes in thousands of single cells or in a spatial context can be challenging. To enable interpretation of these complex datasets, 10x Genomics provides analysis and visualization software for evaluating genomic, epigenomic, transcriptomic, and proteomic data. In this seminar, we will discuss software, methods, and guidance that can be used to analyze your 10x Genomics datasets."
author4: "Sihem Cheloufi"
inst4: " (University of California, Riverside, USA)"
title4: "Mathematical modeling of chromatin accessibility to predict stem cell plasticity"
abstract4: "Stem and progenitor cells become progressively more restricted in their differentiation potential. This process of cell fate determination is driven by lineage-specific transcription factors and is accompanied by dynamic changes in chromatin structure. The chromatin assembly factor complex CAF-1 plays a central role in assembling nucleosomes during DNA replication and has been implicated in regulating cellular plasticity in various lineages in different organisms. However, whether CAF-1 sustains lineage identity during normal homeostasis by influencing chromatin accessibility is unknown. To address this question, we investigated the role of CAF-1 in a myeloid lineage differentiation paradigm. CAF-1 suppression in myeloid progenitors triggered their rapid commitment but incomplete differentiation toward granulocyte, megakaryocyte, and erythrocyte lineages, resulting in a mixed cellular state. Through comparison with a canonical program of directed terminal myeloid differentiation, we define changes in chromatin accessibility that underlie a unique single cell transcriptome of the matured CAF-1 deficient cells. Using mathematical modeling we further predict the cell fate trajectories of the mixed cellular state caused by CAF-1 inhibition. We use nonlinear dimensionality reduction algorithms to produce RNA velocity maps, pseudotime alignment and corresponding differentiation trajectories. We infer an equation of motion from the RNA velocity vector field, which allows a probability density function to be derived and, ultimately, to compute state transition probabilities. Importantly, the resulting mathematical model of cell fate will be used to identify, interrogate, and quantify states of genome organization and predict corresponding changes to cell identities. Furthermore, we can use the model to predict how interfering with lineage specific transcription factors or factors involved in chromatin organization will modify the differentiation trajectories. Future work with this system will identify new targets to restore normal hematopoiesis in disease and generate clinically relevant cell types."
---
