---
layout: group
author: esmb
day: "Wednesday (Thursday)"
subgroup: "CDEV"
title: "Combining modeling and inference in cell biology"
organizers: "Maria-Veronica Ciocanel (Duke University, United States), John Nardini (North Carolina State University, United States)"
description: "In many cell and developmental processes, both modeling and data analytic approaches are necessary in order to generate useful modeling predictions to guide the design of further experiments for both validating and improving biological insight. There is an increased understanding that the application of machine learning methods can also be used to enhance common data-driven modeling techniques, including parameter and equation inference, classification, and sensitivity analysis. The speakers in this session will discuss how continuous differential equation models, individual-based stochastic models, and methods from machine learning can be used to address questions related to mitosis, intracellular transport, cell migration, and tissue development. The speakers will highlight current research progress and challenges associated with combining modeling and inference approaches in cell and developmental biology."
code: "MS13"
code2: "MS14"
author1: "Alexandria Volkening"
inst1: " (Northwestern University, United States)"
title1: "Topological methods for quantitatively describing cell-based patterns"
abstract1: "Self-organization is present at many scales in biology, and here I will focus specifically on elucidating how brightly colored cells interact to form skin patterns in zebrafish. Wild-type zebrafish are named for their dark and light stripes, but mutant zebrafish feature variable skin patterns, including spots and labyrinth curves. All of these patterns form as the fish grow due to the interactions of tens of thousands of pigment cells, making agent-based modeling a natural approach for describing pattern formation. By identifying cell interactions that may change to create mutant patterns, the longterm motivation for my work is to help link genes, cell behavior, and visible animal characteristics in fish. However, agent-based models are stochastic and have many parameters, so comparing simulated patterns and fish images is often a qualitative process. Developing analytically tractable continuum models from agent-based systems is one means of addressing these challenges and better understanding the roles of different parameters in pattern formation. Alternatively, methods from topological data analysis can be applied to cell-based systems directly. In this talk, I will overview our models and present quantitative comparisons of in silico and in vivo cell-based patterns using our topological methods."
author2: "Fiona Macfarlane"
inst2: " (University of Saint Andrews, United Kingdom)"
title2: "A hybrid discrete-continuum approach to model Turing pattern formation"
abstract2: "We have developed a hybrid discrete-continuum modelling framework to investigate the formation of cellular patterns through the Turing mechanism. In this framework, a stochastic individual-based model of cell migration and proliferation is combined with a reaction-diffusion system for the concentrations of some interacting chemical species. As an illustrative example, we consider a model in which the dynamics of the morphogens are governed by an activator-inhibitor system that gives rise to Turing pre-patterns. The cells then interact with the morphogens in their local area through either of two forms of chemically-dependent cell action: Chemotaxis or chemically-controlled proliferation. We consider both the case of static spatial domains and additionally investigate the case of growing domains. In all cases we are able to derive the corresponding deterministic continuum limits, inferring an appropriate system of PDEs to model the dynamics of the hybrid model. We investigate parameter situations in which the numerical simulations of the PDE models give an accurate description of the hybrid models, and cases where they do not qualitatively match the hybrid models. This framework is intended to present a proof of concept for the ideas underlying the models, with the aim to then apply the related methods to the study of specific patterning and morphogenetic processes in the future."
author3: "Suzanne Sindi"
inst3: " (University of California Merced, United States)"
title3: "Multi-Scale Modeling and Parameter Inference in Yeast Protein Aggregation"
abstract3: "Unlike a disease caused by a virus or a bacteria, in prion diseases the infectious agent is created by the host organism itself. Prion proteins are responsible for a variety of neurodegenerative diseases in mammals such as Creutzfeldt-Jakob disease in humans and “mad-cow disease” (Bovine Spongiform Encephalopathy or BSE) in cattle. While these diseases are fatal to mammals, prions are harmful to yeast, making yeast an ideal model organism for prion diseases. 
Most mathematical approaches to modeling prion dynamics have focused on either the protein dynamics in isolation, absent from a changing cellular environment, or modeling prion dynamics in a population of cells by considering the “average” behavior. However, such models have been unable to recapitulate in vivo properties of yeast prion strains. My group develops physiologically relevant mathematical models by considering both the prion aggregates (which evolve inside individual yeast cells) and the yeast cells (which grow and divide).
In this talk, I will present a stochastic biochemical reaction system for protein aggregation and demonstrate that the standard computational assumption - fixed protein monomer mass - leads to incorrect biological conclusions. We relax the mass conservation restriction through the use of an additional “slack” species and discover new regimes of biologically relevant behavior. These regimes necessarily correspond to the biologically feasible regions of parameter space for prion aggregation."
author4: "Adam MacLean"
inst4: " (University of Southern California, United States)"
title4: "Bayesian inference of Calcium signaling dynamic provides a map from single-cell gene expression to cellular phenotypes"
abstract4: "Since single-cell RNA sequencing technologies have become widespread, great efforts have been made to develop appropriate computational methods to learn biological features from high dimensional datasets. Much less effort has gone into the important yet challenging task of learning about dynamic processes from genomic data. Here we employ spatial transcriptomic data (MERFISH) linked to dynamic Ca2+ responses in single cells for parameter inference. We quantify cell-cell similarity -- learnt via nonnegative matrix factorization of transcriptomic signatures -- and use it to define informative cell-specific priors. We show that these informative priors dramatically speed up Bayesian parameter inference for an ODE model of Ca2+ dynamics. Analysis of posterior parameter distributions across hundreds of single cells allows us to identify genes driving phenotypic changes and link these genes to specific Calcium pathway parameters that are sensitive to outputs. Finally, we test our ability to predict Ca2+ responses using only the cell-cell similarity. This allows us to quantify the amount of information on a dynamic cell phenotype that is contained in the gene expression data alone."
---
