<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-12 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-12 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-12" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS12/" />
<meta property="og:url" content="http://localhost:4000/MS12/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS12/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-12","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-12">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-12" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-12" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-12</h1>
        <h4>Wednesday, June 16 at 04:15am (PDT)</br>Wednesday, June 16 at 12:15pm (BST)</br>Wednesday, June 16 08:15pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS12">Minisymposia-12</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/CBBS-MS12.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS12-CBBS: </small></br>Lattice Models and Agent-Based Models in Biology: Linking Individual Properties to Population Properties</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Bhargav Karamched (Florida State University, United States of America)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-CBBS</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/CBBS-MS12.html#author1"><strong class="card-body-subgroup">Bhargav Karamched</strong></a>
		    	 
		    		<em> (Florida State University, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Spatial Model of Oncolytic Virotherapy:  Targeting Drug-Resistant Mutants"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Oncolytic virotherapy has emerged as a viable treatment for cancers.  Although successful cancer treatments with viruses have been observed, they are few and far between.  Here, we explore whether combining virotherapy with other methods of cancer treatment may lead to more robust, reliable cancer treatment.  For example, cancer cells sometimes undergo mutations that allow them to develop resistance to treatment drugs.  To address such a mutation, we explore the possibility of targeting drug-resistant mutants with viruses so that standard drug treatments of cancerous tumors may be used to target cancerous cells.  We develop a lattice model that describes cancer tumor growth dynamics and mutant cell dynamics.  We find that when mutant cells have a disadvantageous mutation, virus infection amplifies the nature of the disadvantage, with a slight caveat.  When infectivity is too high, the population-level death rate is increased so that for a tumor to reach a given size, more cell divisions are necessary.  This leads to the presence of more mutants than when no virus is present.  We explore this nuanced system with a mean field equation and discuss how viruses used in such a way can progress cancer treatments in the future.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CBBS-MS12.html#author2"><strong class="card-body-subgroup">Cicely Macnamara</strong></a>
		    	 
		    		<em> (University of St Andrews, Scotland)</em>
		    	
		    	
		    		
		    			</br>" Computational modelling and simulation of tumour growth and development within a 3D heterogeneous tissue"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The term cancer covers a multitude of bodily diseases, broadly categorised by having cells which do not behave normally. Cancer cells can arise from any type of cell in the body; cancers can grow in or around any tissue or organ making the disease highly complex. My research is focused on understanding the specific mechanisms that occur in the tumour microenvironment via mathematical and computational modelling. In this talk I shall present a 3D individual-based force-based model for tumour growth and development in which we simulate the behaviour of, and spatio-temporal interactions between, cells, extracellular matrix fibres and blood vessels. Each agent is fully realised, for example, cells are described as viscoelastic sphere with radius and centre given within the off-lattice model. Interactions are primarily governed by mechanical forces between elements. However, as well as he mechanical interactions we also consider chemical interactions, by coupling the code to a finite element solver to model the diffusion of oxygen from blood vessels to cells, as well as intercellular aspects such as cell phenotypes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CBBS-MS12.html#author3"><strong class="card-body-subgroup">Hamid Teimouri</strong></a>
		    	 
		    		<em> (Rice University, United States of America)</em>
		    	
		    	
		    		
		    			</br>"The impact of the temporal order of mutations on cancer initiation dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer is a set of genetic diseases that are driven by mutations. It was recently discovered that the temporal order of genetic mutations affects the cancer evolution and even the nature of the disease itself.  The mechanistic origin of these observations, however, remain not well understood.  We present a theoretical model for cancer initiation dynamics that allows us to quantify the impact of the temporal order of mutations. In our approach, the cancer initiation process is viewed as a set of stochastic transitions between discrete states defined by the different numbers of mutated cells. Using a first-passage analysis, probabilities and times before the cancer initiation are explicitly evaluated for two alternative sequences of two mutations. It is found that the probability of cancer initiation is determined only by the first mutation, while the dynamics depends on both mutations. In addition, it is shown that the acquisition of a mutation with  higher fitness before mutation with lower fitness increases the probability of the tumor formation but delays the cancer initiation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CBBS-MS12.html#author4"><strong class="card-body-subgroup">Namiko Mitarai</strong></a>
		    	 
		    		<em> (University of Copenhagen, Denmark)</em>
		    	
		    	
		    		
		    			</br>"Emergence of diversity in a model ecosystem of sessile species with mutually exclusive interactions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The biological requirements for an ecosystem to develop and maintain species diversity are in general unknown. Here we consider a lattice model ecosystem of sessile (immobile) and mutually excluding organisms competing for space. The competition is controlled by an interaction network with fixed links chosen randomly. New species are introduced in the system at a predefined rate. In the limit of small introduction rates, the system becomes bistable and can undergo a phase transition from a state of low diversity to high diversity. We suggest that patches of isolated meta-population spontaneously formed by the collapse of cyclic relations are essential for the transition to the state of high diversity. The high-diversity state is robust against small disturbance or spontaneous death. When new species evolve by mutating the species interaction network from ancestry species, the high-diversity state appears as long as there is a cost associated with the ability to invade another species.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/CDEV-MS12.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS12-CDEV: </small></br>Synergy between experiments and modelling in understanding morphogenetic processes</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Alessandra Bonfanti (Sainsbury Laboratory University of Cambridge, United Kingdom), Alexandre Kabla (University of Cambridge, United Kingdom)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/CDEV-MS12.html#author1"><strong class="card-body-subgroup">Shiladitya Banerjee</strong></a>
		    	 
		    		<em> (Carnegie Mellon University, USA)</em>
		    	
		    	
		    		
		    			</br>"Cell-scale modeling of epithelial morphogenesis using quantitative theory and optogenetics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During development, epithelial tissues form complex structures like organs through precise spatiotemporal coordination of cell shape changes. In vivo, many morphogenetic events are driven by pulsatile cellular contractions, which are rectified to produce irreversible tissue deformations. The functional significance of these pulsed contractions and their underlying mechanochemical circuits remain unknown. Here we develop quantitative cell-resolution models of epithelial tissues using live-cell imaging and optogenetic control of cytoskeletal force generation. We demonstrate that pulsed contraction acts as a mechanical ratchet to guide directed morphogenesis in epithelia and uncover the underlying feedback designs between cellular force generation and cell-cell adhesion. Our data and mathematical modeling provide new insights into how the localized production of cytoskeletal forces encode a fine-tuned instruction for cellular deformations that mediate epithelial morphogenesis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CDEV-MS12.html#author2"><strong class="card-body-subgroup">Jean-François Rupprecht</strong></a>
		    	 
		    		<em> (CNRS & Turing Centre for Living Systems Group Leader, Aix-Marseille University., France)</em>
		    	
		    	
		    		
		    			</br>"Epithelial tissues flows over hills, valleys and around potholes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    				Epithelial tissues constantly flow and renew while acting as a barrier against environmental stress and abrasion. Flows within epithelial tissues are known to be associated with cell shape changes - e.g. with shear flows contributing to cell stretching – yet, by exerting forces on their neighbours, elongated cells could in turn contribute to flows. Hydrodynamic theories incorporating such cell shape/tissue flow mechanical feedback have been proposed to explain the specific flow patterns observed within in vitro confluent epithelial tissues [1,2].
In this talk, I will present our recent results on the role of flows and cell-shape driven stresses in processes related to: (i) the loss of epithelial integrity [3]. Motivated by recent experiments revealing the spontaneous formation of holes within MDCK cell monolayers cultured on soft hydrogels, we implemented a cell-based computational framework (called vertex model) whereby cell-cell junctions can rupture. We also introduced cell-based nematic stresses which we show triggers global spontaneous flows. In both experiments and simulations, we observe the onset of specific patterns in cell shapes called topological defects. While cells at the tip of comet-like +1/2 defects were shown to be compressed and highly prone to extrusion [1], here, our simulations explain the experimental observation that holes are created in high tension regions located either at the tail of comet-like +1/2 defects or near trefoil-like -1/2 defects.  In addition, our work indicate that the progressive deformation of cells at the border of the hole further drives the hole opening process itself, hence suggesting an unexpected role of active stresses in regulating tissue integrity [3]. (ii) tissue flows and renewal within curved environment [4]. Several recent experimental work have shown that epithelial cells spread over curved substrates with a preferential orientation along specific curvature directions. In a recent preprint [4], we work out a set of hydrodynamic equations governing the cell shape and long-time flows of confluent tissues on non-deformable curved substrate. We derive analytical expressions for the threshold value of the local curvature and active stress strength above which a spontaneous global tissue flow arise at steady state. In particular, we predict the stability of a double-shear flow pattern which I will argue shares some similarities with the one observed during the Drosophila embryogenesis process of germ band extension.
1. Saw, T. B. et al. Topological defects in epithelia govern cell death and extrusion, Nature, (2017). 2. Duclos, G. et al. Spontaneous shear flow in confined cellular nematics, Nature Physics (2018). 3. S. Sonam, L. Balasubramaniam, S-Z. Lin, Y. M. Yow Ivan, C. Jebane, Y. Toyama, Philippe Marcq, J. Prost, R.-M. Mège,  J-F. R., B. Ladoux, Mechanical stress driven by rigidity sensing governs epithelial stability (2021). 4. Shear transitions of an active nematic in curved geometries, S. Bell, S.-Z. Lin, J-F. R., and Jacques Prost (2021).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CDEV-MS12.html#author3"><strong class="card-body-subgroup">Alan Lowe</strong></a>
		    	 
		    		<em> (University College London, UK)</em>
		    	
		    	
		    		
		    			</br>"Learning the rules of cell competition"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell competition is a quality control mechanism through which tissues eliminate unfit cells. In biochemical and mechanical competition, individual cell fate is determined by the local cellular neighbourhood. Despite this, cell competition remains poorly understood -- we do not know the interaction 'rules' that determine each cell's fate. This is largely because most studies only quantify whole population shifts for very few time points and for few cells. One major obstacle to understanding how population shifts occur as a result of single cell behaviours is that it requires thousands of cells to be tracked over long periods of time. To address this challenge, we recently built the first deep learning and automated single-cell microscopy system to analyse cell competition. We used this to analyse the cell cycle state of millions of single cells in mechanical competition, including cell division and death. These data suggest that tissue-scale population shifts are strongly affected by cellular-scale tissue organization. We find that local density has a dramatic effect on the rate of division and apoptosis under competitive conditions. Strikingly, our analysis reveals that proliferation of the winner cells is up-regulated in neighbourhoods mostly populated by loser cells. Finally, I present our current progress on developing a machine learning approach to learn interpretable “rules” of cell competition, by predicting the fate of cells in an evolving tissue.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/CDEV-MS12.html#author4"><strong class="card-body-subgroup">Pasquale Ciarletta</strong></a>
		    	 
		    		<em> (Politecnico di Milano, Italy)</em>
		    	
		    	
		    		
		    			</br>"Pattern formation and self-organization during cancer cell budding in-vitro"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tissue self-organization into defined and well-controlled three-dimensional structures is essential during development for the generation of organs. A similar, but highly deranged process might also occur during the aberrant growth of cancers, which frequently display a loss of the orderly structures of the tissue of origin, but retain a multicellular organization in the form of spheroids, strands, and buds. The latter structures are often seen when tumor masses switch to an invasive behavior into surrounding tissues. However, the general physical principles governing the self-organized architectures of tumor cell populations remain by and large unclear. In this work, we perform in-vitro experiments to characterize the growth properties of glioblastoma budding emerging from monolayers. We further propose a theoretical model and its finite element implementation to characterize such a topological transition, that is modelled as a self-organised, non-equilibrium phenomenon driven by the trade–off of mechanical forces and physical interactions exerted at cell-cell and cell–substrate adhesions. Notably, the unstable disorder states of uncontrolled cellular proliferation macroscopically emerge as complex spatio–temporal patterns that evolve statistically correlated by a universal law.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/DDMB-MS12.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS12-DDMB: </small></br>Mathematical Modeling of Protein Dynamics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Suzanne S. SINDI (University of California, Merced, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/DDMB-MS12.html#author1"><strong class="card-body-subgroup">Human REZAEI</strong></a>
		    	 
		    		<em> (INRAE, Jouy-en-Josas, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"to be announced"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			to be announced
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/DDMB-MS12.html#author2"><strong class="card-body-subgroup">Maria Carla TESI</strong></a>
		    	 
		    		<em> (Universitá di Bologna, ITALY)</em>
		    	
		    	
		    		
		    			</br>"The synergistic interplay between two proteins: a mathematical model for  Alzheimer's disease"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			There is currently a great deal of interest in the scientific community in investigating the effects of the synergistic interplay of Amyloid beta and tau on the dynamics of Alzheimer’s disease. I will present a mathematical model for the onset and progression of Alzheimer’s disease based on transport and diffusion equations for the two proteins.  In the model neurons are treated as a continuous medium and structured by their degree of mal- functioning. Three different mechanisms are assumed to be relevant for the temporal evolution of the disease: i) diffusion and agglomeration of soluble Amyloid beta, ii) effects of misfolded tau protein and iii) neuron-to-neuron prion-like transmission of the disease. These processes are modelled by a system of Smoluchowski equations for the Amyloid beta concentration, an evolution equation for the dynamics of tau protein and a kinetic-type transport equation for the distribution function of the degree of malfunctioning of neurons. The latter equation contains an integral term describing the random onset of the disease as a jump process localized in particularly sensitive areas of the brain. I will explain the structure of the model and give a hint of the main results obtained. Eventually I will also show the output of some numerical simulations, of some significance even if performed in an over-simplified 2D geometry.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/DDMB-MS12.html#author3"><strong class="card-body-subgroup">Léon Matar TINE</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Analysis and numerical simulations of a reaction-diffusion model with fixed active bodies: Application to Alzheimer's disease."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk focuses on a spatial interaction model of two substances (or molecules), one of which, concentration f, is produced by bodies located in- side the considered domain and is acting as an activator (positive effect) or a growth factor for the second substance which concentration is denoted by g. The substance or molecules of concentration g on the contrary acts as an inhibitor or a shrinkage for the substance f because of its cytotoxic effect on the bodies activity. The main goal is to analyze the dynamics and propose an adapted numerical approach for the simulation of such kind of model described above where existing bodies (sources for one of the substance) have polygonal shape and their activity can be altered by the presence of the second substance or molecule. For convenience and in accordance with [1] the bodies are taken as fix in the domain. In [1] authors introduced a model based on a discrete growth-fragmentation system with spatial diffusion in order to analyze the early stages of Alzheimer disease. Their model, containing at least five equations and fourteen parameters, aims at representing the process of repli- cation and spatial diffusion of Aβ-oligomers molecules in the neighborhood of neurons. They describe the whole process from Aβ-monomers molecules assembling first into proto-oligomers (unstable polymers) and then into Aβ- oligomers (stable polymers). In [1] the authors carried out a modeling work for the description and simulation of the model where oligomers neurotoxic effect is taken into account. Numerical difficulties are linked to this modeling. A first difficulty is to take into account the geometrical form of active bodies which can be arbitrary. Another difficulty is to manage the sent cytotoxic signals from the substance (or molecule) of concentration g to bodies. In fact, the efficacity of the signal depends on the distance from where it is sent. [1] M. Andrade-Restrepo, P. Lemarre, L. Pujo-Menjouet, L. M. Tine, and S. I. Ciuperca. Modeling the spatial propagation of Aβ oligomers in alzheimer’s disease. In CEMRACS 2018 - Numerical and mathematical modeling for biological and medical applications: deterministic, proba- bilistic and statistical descriptions, pages 1–10, Marseille, France, Jul. 2018
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/DDMB-MS12.html#author4"><strong class="card-body-subgroup">Laurent PUJO-MENJOUET</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Alzheimer and Prion: a dangerous liaison"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Alzheimer’s disease (AD) is a fatal incurable disease leading to progressive neuron destruction. AD is caused in part by the accumulation in the brain of Aβ monomers aggregating into oligomers and fibrils. Oligomers are amongst the most toxic structures as they can interact with neurons via membrane receptors, including PrPc proteins. This interaction leads to the misconformation of PrPc into pathogenic oligomeric prions, PrPol. We develop here a model describing in vitro Aβ polymerization process. We include interactions between oligomers and PrPc, causing the misconformation of PrPc into PrPol.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/IMMU-MS12.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>MS12-IMMU: </small></br>Mathematical modelling of the coronavirus disease</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Alexey Tokarev (Рeoples’ Friendship University of Russia, Russia)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/IMMU-MS12.html#author1"><strong class="card-body-subgroup">Vitaly Volpert</strong></a>
		    	 
		    		<em> (CNRS, University Lyon, France)</em>
		    	
		    	
		    		
		    			</br>"Introduction to the pathophysiology of the coronavirus disease"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A short overview of the current knowledge on the disease progression and its possible complications will be presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/IMMU-MS12.html#author2"><strong class="card-body-subgroup">Anass Bouchnita</strong></a>
		    	 
		    		<em> (Department of Integrative Biology, University of Texas at Austin, USA)</em>
		    	
		    	
		    		
		    			</br>"Multiscale modelling of SARS-CoV-2 infection to study the role of innate and adaptive immune responses in healthy and immunocompromised individuals"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection causes mild to severe outcomes depending on the balance of host immune response. The interaction between SARS-CoV-2 and the immune response is complex because it involves processes that span across several scales of biological hierarchy such as cells, tissues, organs, and the host. In this talk, we present a multiscale model that describes the interaction between SARS-CoV-2 and the immune response. In this model, dendritic cells are considered as individual objects that move within a section of the epithelial tissue and can be used by the virus to replicate and spread. They also secrete type I IFN which downregulates the production of the virus. At the same time, the model simulates the production of antigen-specific by lymph nodes as well as their interaction with infected cells and virions in the infection site. After model validation, we show that a moderately weak type I IFN could elicit a solid adaptive response that accelerates the virus's clearance. Numerical simulations suggest that the deficiency of naïve lymphocytes in immunocompromised individuals increases the persistence of the virus and exacerbates the disease's outcome.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/IMMU-MS12.html#author3"><strong class="card-body-subgroup">Bogdan Kazmierczak</strong></a>
		    	 
		    		<em> (Institute of Fundamental Technological Research, Polish Academy of Sciences, Poland)</em>
		    	
		    	
		    		
		    			</br>"Infection spreading in cell culture as a reaction-diffusion wave"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We formulate a reaction-diffusion system of equations modeling the progression of viral infection, e.g. of SARS-Cov viruses. Analytical and numerical results obtained in the framework of the model are in agreement with the 'in vitro' experimental findings.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/IMMU-MS12.html#author4"><strong class="card-body-subgroup">Alexey Tokarev</strong></a>
		    	 
		    		<em> (S.M. Nikolskii Mathematical Institute, Рeoples’ Friendship University of Russia (RUDN University), Russia)</em>
		    	
		    	
		    		
		    			</br>"Nonlinear dynamics in the homogeneous model of immune responses to SARS-CoV-2 virus"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Antiviral immune response is a highly nonlinear process governed by the cooperative behavior of variegated constituents of immune system. Depending on nature of virus, initial viral load, and patient peculiarities, infection can pass diversely and result from recovery to death. In the current pandemic of COVID-19 infection, in the part of patients the disease is complicated by abnormal inflammation response (hypercytokinemia, cytokine storm). We study the immune response to the SARS-CoV-2 virus by constructing the series of ODE-based mathematical models of different phases of this infection: (1) innate immune response, (2) innate plus adaptive immune response, (3) inflammation response. The innate immune response model shows the bistability and threshold properties, as well as possible oscillatory regime. The higher the initial viral load, the shorter is the incubation period and the higher is the maximal transient virus concentration. Depending on the effectiveness of antibodies production, the adaptive immune response can either fully eliminate the virus, or substantially postpone virus concentration burst with following higher virus concentration comparing to the case of innate response only. Inflammation response model also shows bistability and oscillatory behavior. We compare prediction of these models with clinical and epidemiological data. Finally, we study the duration of vaccine protection against the SARS-CoV-2 virus. This work was supported by the Ministry of Science and Higher Education of Russian Federation: agreement no. 075-03-2020-223/3 (FSSF-2020-0018).
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/MEPI-MS12.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS12-MEPI: </small></br>Modeling containment and mitigation of COVID-19: experiences from different countries worldwide</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Andrei Akhmetzhanov (National Taiwan University College of Public Health, Taiwan), Natalie Linton (Hokkaido University, Japan)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS15-MEPI</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/MEPI-MS12.html#author1"><strong class="card-body-subgroup">Michael Hochberg</strong></a>
		    	 
		    		<em> (Institute for Evolutionary Sciences, University of Montpellier, France)</em>
		    	
		    	
		    		
		    			</br>"Modeling COVID-19: Seeing the forest for the trees"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Given the pace of SARS-CoV-2 outbreaks, epidemiologists and mathematical biologists have had to apply their expertise in real time to understand COVID-19 epidemiology, sometimes modify traditional SEIR models, and evaluate how mitigation and suppression measures might impact outbreaks. This has produced a wide variety of mathematical and statistical models, from the overly simple to the extremely complex. I discuss the pros and cons of employing simple SEIR models for COVID-19 dynamics, and how simple but important constructs are often missed by more complex models. I then turn to how an SEIR model of intermediate complexity produces a rich range of outcomes when coupled with the optimization of stay-in-place decisions. These studies show how intuition and models combine to increase simulation and forecasting accuracy, and are integral in developing more effective control strategies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MEPI-MS12.html#author2"><strong class="card-body-subgroup">Natalie Linton</strong></a>
		    	 
		    		<em> (Hokkaido University, Kyoto University, Japan)</em>
		    	
		    	
		    		
		    			</br>"Variation in serial interval distribution among reported cases in Japan"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This study looks how the serial interval of coronavirus disease 2019 (COVID-19) cases in Japan changed over time during 2020 and assesses whether differences in the length of the serial interval exists based on age, sex, and transmission context. We collected publicly reported data on cases in Japan and determined 758 likely transmission pairs based on the types and timings of exposures. The serial interval for pairs detected during the second wave and third waves of COVID-19 transmission in Japan was shorter than the serial interval of cases detected in the first wave. Serial interval length did not vary greatly by sex or transmission context, however serial intervals tended to be a bit shorter when infectors were under 30 years of age and when secondary transmission occurred in a household setting. Accounting for shortening of the serial interval compared to estimates from early in the pandemic may improve inference of transmission dynamics.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MEPI-MS12.html#author3"><strong class="card-body-subgroup">Robin Thompson</strong></a>
		    	 
		    		<em> (University of Warwick, U.K.)</em>
		    	
		    	
		    		
		    			</br>"Inferring the effectiveness of interventions during infectious disease outbreaks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MEPI-MS12.html#author4"><strong class="card-body-subgroup">Sumire Sorano</strong></a>
		    	 
		    		<em> (London School of Hygiene and Tropical Medicine, U.K.)</em>
		    	
		    	
		    		
		    			</br>"The impact of COVID-19 from social and gender perspectives in Japan"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			COVID-19 pandemic disproportionately affected vulnerable populations, revealing the weakness of society in the world. According to the Ministry of Health, Labor and Welfare in Japan, the number of suicides nationwide in 2020 exceeded 21,000, marking the first increase since 2009. While the number of male suicides decreased, female suicides showed a marked increase (6091 in 2019 to 7026 in 2020; an increase by 15%), especially among young age (44% increase among girls below 20 years and 32% increase among women in their 20s). Unemployment and economic hardship during COVID-19 pandemic affected women harder. and there was an increased concerns over unintended pregnancy as domestic violence and sexual violence increased. This presentation overviews the impact of COVID-19 from social and gender perspectives in Japan.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/MFBM-MS12.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS12-MFBM: </small></br>Stochastic Systems Biology: Theory and Simulation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jae Kyoung Kim (Department of Mathematical Sciences, KAIST, Republic of Korea), Ramon Grima (University of Edinburgh, United Kingdom)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/MFBM-MS12.html#author1"><strong class="card-body-subgroup">Hyukpyo Hong</strong></a>
		    	 
		    		<em> (Department of Mathematical Sciences, KAIST, Republic of Korea)</em>
		    	
		    	
		    		
		    			</br>"Inference of stochastic dynamics in biochemical reaction networks by exploiting deterministic dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Biochemical reaction networks (BRNs) have a stochastic nature, so every reaction in BRNs display randomness. Inherent stochasticity can be captured only by stochastic models, but it is more challenging to analyze their dynamics while their deterministic counterparts are easier to be analyzed, in general. Thus, various methods exploiting deterministic dynamics to infer the stochastic one have been proposed. In particular, stochastic model reduction using deterministic quasi-steady-state approximations (QSSAs) of fast variables is widely used to efficiently simulate a stochastic model. For instance, Michaelis-Menten or Hill-type functions have been used for Gillespie stochastic simulation. In this talk, we provide a complete validity condition for stochastic model reduction using the deterministic QSSA to eliminate stochastic reversible binding, which is fundamental and ubiquitous in BRNs. Furthermore, we present a framework to analytically derive stationary distribution for a large class of BRNs using their deterministic steady states based on chemical reaction network theory.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MFBM-MS12.html#author2"><strong class="card-body-subgroup">Zhou Fang</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>" Stochastic filtering for multiscale stochastic reaction networks based on hybrid approximations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The advance in fluorescent technologies and microscopy has greatly improved scientists' ability to observe real-time single-cell activities. In this paper, we consider the associate filtering problem, i.e., how to estimate latent dynamic states of an intracellular reaction system from time-course measurements of fluorescent reporters. A straightforward approach to this filtering problem is to use a particle filter where samples are generated by simulation of the full model and weighted according to observations. However, the exact simulation of the full model usually takes an impractical amount of computational time and prevents this type of filters from being used for real-time applications. Inspired by the recent development of hybrid approximations to multiscale chemical reaction networks, we approach the filtering problem in an alternative way. We first prove that accurate solutions to the filtering problem can be constructed by solving the filtering problem for a reduced model that represents the dynamics as a hybrid process. The model reduction is based on exploiting the time-scale separations in the original network and, therefore, can greatly reduce the computational effort required to simulate the dynamics. Consequently, we are able to develop efficient particle filters to solve the filtering problem for the original model by applying particle filters to the reduced model. We illustrate the efficacy and efficiency of our approach using several numerical examples.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MFBM-MS12.html#author3"><strong class="card-body-subgroup">Samuel Isaacson</strong></a>
		    	 
		    		<em> (Boston University, Department of Mathematics and Statistics, USA)</em>
		    	
		    	
		    		
		    			</br>"Stochastic Reaction-Drift-Diffusion Methods for Studying Cell Signaling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Particle-based stochastic reaction-diffusion (PBSRD) models are one approach to study biological systems in which both the noisy diffusion of individual molecules, and stochastic reactions between pairs of molecules, may influence system behavior. They provide a more microscopic model than deterministic reaction-diffusion PDEs or stochastic reaction-diffusion SPDEs, which treat molecular populations as continuous fields. The reaction-diffusion master equation (RDME) and convergent RDME (CRDME) are lattice PBSRD models, with the latter providing a convergent approximation to the spatially-continuous volume-reactivity PBSRD model as the lattice spacing is taken to zero. In this talk I will present several generalizations of the RDME and CRDME to support spatial transport mechanisms needed for resolving membrane-bound signaling processes, including drift due to background potentials, interaction potentials between molecules, and continuous-time random walks to approximate molecular transport on surfaces.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MFBM-MS12.html#author4"><strong class="card-body-subgroup">Brian Munsky</strong></a>
		    	 
		    		<em> (Colorado State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Designing Optimal Microscopy Experiments to Harvest Single-Cell Fluctuation Information while Rejecting Image Distortion Effects"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Modern fluorescence labeling techniques and optical microscopy approaches have made it possible to experimentally visualize every stage of basic gene regulatory processes, even at the level of single cells and individual DNA, RNA, and protein molecules, in living cells, and within fluctuating environments. To complement these observations, the mechanisms and parameters of discrete stochastic models can be rigorously inferred to reproduce and quantitatively predict every step of the central dogma of molecular biology. As single-cell experiments and stochastic models become increasingly more complex and more powerful, the number of possibilities for their integrated application increases combinatorially, requiring efficient approaches for optimized experiment design. In this presentation, we will introduce two model-driven experimental design approaches: one based on detailed mechanistic simulations of optical experiments, and the other on a new formulation of Fisher Information for discrete stochastic gene regulation models. Using different combinations of biological experiments and simulated data for single-gene transcription and single-RNA translation, we will demonstrate how these experiment design approaches can be extended to account for non-gaussian intrinsic and extrinsic process noise within individual cells as well as for non-trivial measurement noise effects due to optical distortions and image processing errors.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/MMPB-MS12.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>MS12-MMPB: </small></br>Dynamics of hematopoiesis in health and disease - from governing principles to clinical implications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Peter Ashcroft (ETH Zurich, Switzerland), Tony Humphries (McGill University, Canada), Morten Andersen (Roskilde University, Denmark)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-MMPB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/MMPB-MS12.html#author1"><strong class="card-body-subgroup">Nathaniel Mon Père</strong></a>
		    	 
		    		<em> (Queen Mary University of London and Barts Cancer Institute, UK)</em>
		    	
		    	
		    		
		    			</br>"Somatic evolution in healthy hematopoietic stem cells"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The production of blood cells is known to be driven by a relatively small group of hematopoietic stem cells (HSCs) which both self-renew and provide lineage progenitors throughout the entirety of an individual’s lifetime. However, many properties of these dynamics are still debated or unknown, in part due to the difficulty of studying HSC behaviour in vivo. Because the stem cell pool self-renews it acquires somatic mutations which are subject to evolutionary pressures and stochastic drift. We show that information on the underlying dynamics is encoded in observations of this mutational landscape, which in turn can be obtained by modern sequencing methods. In particular we use observations of the distribution of mutational burdens and the variant allele frequency spectrum to estimate fundamental quantities such as the per division mutation rate, the size of the HSC pool, and the proportion of asymmetric divisions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MMPB-MS12.html#author2"><strong class="card-body-subgroup">Gladys Poon</strong></a>
		    	 
		    		<em> (University of Cambridge, UK)</em>
		    	
		    	
		    		
		    			</br>"Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Genetic alterations under positive selection in ostensibly healthy tissues have implications for cancer risk. However, total levels of positive selection across the genome remain unknown. How much positive selection elsewhere in the genome is missed by gene-focused sequencing panels? Synonymous passenger mutations that hitchhike to high variant allele frequency are influenced by any driver mutation, regardless of type or location in the genome, and can therefore be used to estimate total levels of positive selection in healthy tissues. By comparing observed numbers of synonymous passengers to the numbers expected due to driver mutations in canonical cancer genes, we show that it is possible to quantify missing selection left to be explained by unobserved drivers elsewhere in the genome. Here we analyse the variant allele frequency spectrum of synonymous mutations from physiologically healthy blood and oesophagus to quantify levels of missing positive selection. In blood we find that only 20% of synonymous passengers can be explained by SNVs in canonical driver genes, suggesting high levels of positive selection for other mutations elsewhere in the genome. In contrast, approximately half of all synonymous passengers in the oesophagus can be explained by just the two driver genes NOTCH1 and TP53, suggesting little positive selection elsewhere. In tissues with high levels of ‘missing’ selection, we show that our framework can be used to guide targeted driver mutation discovery.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MMPB-MS12.html#author3"><strong class="card-body-subgroup">Thomas Stiehl</strong></a>
		    	 
		    		<em> (RWTH Aachen University, Germany)</em>
		    	
		    	
		    		
		    			</br>"Relating competition in the stem cell niche to biomarkers of acute myeloid leukemia progression - Insights from mathematical modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Acute myeloid leukemia (AML) is one of the most aggressive cancers of the blood forming (hematopoietic) system. The disease is driven by a small population of leukemic stem cells (LSC). LSC give rise to the malignant cell bulk and out-compete hematopoietic stem cells (HSC) which are required to maintain healthy blood cell formation. HSC depend on a specific supportive micro-environment, the so-called stem cell niche, to fulfil their function. Based on recent experimental evidence we propose a mathematical model to quantitatively describe the competition of HSC and LSC for spaces in the stem cell niche. We calibrate the model to patient data and provide insights in the following questions: • Why and how can we use HSC counts as a prognostic biomarker in AML? • What can HSC counts at the time of diagnosis tell us about disease dynamics of individual patients? • Can measurements of HSC counts reveal information about LSC properties? • How can we use the mathematical model as a tool for risk-stratification and which additional information does it provide compared to clinical approaches? • For which subsets of AML patients is the model-based risk-stratification superior to the clinically established approach? • How can we simplify the model-based approach to render it more accessible to practitioners?
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/MMPB-MS12.html#author4"><strong class="card-body-subgroup">Johnny T. Ottesen</strong></a>
		    	 
		    		<em> (Roskilde University, Denmark)</em>
		    	
		    	
		    		
		    			</br>"Dynamics of Hematological Cancer-Infection Comorbidities – an in silico study"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Background: The immune system attacks threats like an emerging cancer or infections like COVID-19. Malignant cells may be in a dormant state or escape the immune system resulting in uncontrolled growth and cancer progression. If the immune system is busy fighting a cancer, a sudden severe infection may compromise the immunoediting and the comorbidity may be too taxing to control. Method: A novel mechanism based computational model coupling a cancer-infection development to the adaptive immune system is presented and analyzed. We used the model to investigate outcomes of two immunotherapies, interferon-alpha and CAR T-cell therapy as mono therapies as well as in combination with antibiotics. The model maps the outcome to the underlying physiological mechanisms and agree with numerous evidence based medical observations. Results and Conclusions: Progression of a cancer and the effect of treatments depend on the cancer size, the level of infection, and on the efficiency of the adaptive immune system. The model exhibits bi-stability, i.e. virtual patient trajectories gravitate towards one of the two stable steady states: a dormant state or a full-blown cancer-infection disease state. An infectious threshold curve exists and if infection exceed this separatrix for sufficiently long time cancer escapes and progresses. Thus, early treatment is vital for remission and severe infections may instigate cancer escape. Immunotherapy may sufficiently control cancer progression back into a dormant state but the therapy gains efficiency in combination with antibiotics.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/NEUR-MS12.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS12-NEUR: </small></br>The Control of the Cardiovascular System in Health and Disease</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Mette Olufsen (North Carolina State University, USA), Brian Carlson (University of Michigan, USA), Justen Geddes (North Carolina State University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-NEUR</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/NEUR-MS12.html#author1"><strong class="card-body-subgroup">Brian E. Carlson</strong></a>
		    	 
		    		<em> (University of Michigan, Ann Arbor, MI, USA)</em>
		    	
		    	
		    		
		    			</br>"Using Modeling to Understand Pathophysiology in the Cardiovascular Control System"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The state of the cardiovascular system can be assessed from time-series signals including heart rate and blood pressure. Characteristics of these signals are used to determine pathophysiology. Experienced clinicians can visually inspect signals and with high level of certainty determine key observed dynamics however analysis with computational models can uncover the underlying mechanisms driving these dynamics. This talk will address how mathematical modeling can be utilized to predict patient specific dynamics for patients with altered cardiovascular control system, including patients with heart failure and pulmonary hypertension. Focus will be on studying dynamics observed in response to an orthostatic challenge including the Valsalva maneuver and active standing. It is believed that through the analysis of these cardiovascular challenges a deeper level phenotyping of changes within the cardiovascular control system can be revealed. To understand how the system is impacted we use models to analyze patient specific changes observed during active standing, a Valsalva maneuver (breath holding), and deep breathing.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/NEUR-MS12.html#author2"><strong class="card-body-subgroup">John S. Clemmer</strong></a>
		    	 
		    		<em> (Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA)</em>
		    	
		    	
		    		
		    			</br>"Physiological Modeling of Hypertensive Kidney Disease in African Americans"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chronic kidney disease (CKD) is characterized by the progressive functional loss of nephrons and hypertension (HTN). Successful antihypertensive regimens attenuate the progression of CKD. While studies suggest that calcium channel blocker (CCB) therapy mitigates the decline in renal function in humans with essential HTN, there are few long-term clinical studies that determine the impact of CCBs in patients with hypertensive CKD. Dihydropyridine (DHP) or L-type CCBs preferentially vasodilate the afferent arteriole and have been shown to accelerate CKD in African Americans with low renal function, but the mechanisms are unknown. We used an established integrative mathematical model of human physiology, HumMod, to create a virtual population of African Americans using clinical data (ALLHAT trial). We tested the hypothesis that DHP CCB therapy exacerbates pressure-induced glomerular injury in hypertensive CKD. After two years of simulating angiotensin converting enzyme (ACE) inhibition or CCB, there were statistically similar blood pressure and glomerular filtration rate (GFR) before and after treatment as compared to African American patients. ACE inhibition decreased blood pressure in the virtual population and was not associated with significant changes in glomerular pressure or injury. However, despite having significant falls in blood pressure, chronic CCB therapy was associated with increases in glomerular pressure and significantly increased glomerular damage. High glomerular injury or pressure and single nephron GFR predicted glomerulosclerosis in these models. The results from these simulations suggest that DHP (L-type) CCBs may potentiate glomerular HTN in at risk African Americans (low renal function) and that efferent arteriolar vasodilation with blockers of the renin-angiotensin system may ameliorate CKD progression. While these simulations and results are clinically relevant, the predictions presented in these simulations are to be considered hypotheses until confirmed with experimental and clinical investigation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/NEUR-MS12.html#author3"><strong class="card-body-subgroup">Peng Li</strong></a>
		    	 
		    		<em> (Brigham and Women’s Hospital, Harvard Medical School, Boston, MA, USA)</em>
		    	
		    	
		    		
		    			</br>"Resting Heart Rate Complexity and All-Cause and Cardiorespiratory Mortality in a Middle-to-Older Aged, Population Cohort"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Spontaneous heart rate fluctuations contain rich information related to health and illness in terms of physiological complexity, an accepted indicator of plasticity and adaptability. However, it is challenging to make inferences on complexity from shorter, more practical epochs of data. Distribution entropy (DistEn) is a recently introduced complexity measure that is designed specifically for shorter duration heartbeat recordings. We hypothesized that reduced DistEn predicts increased mortality in a large population cohort. The prognostic value of DistEn was examined in 7,631 middle-older aged UK Biobank participants who had 2-minute resting electrocardiograms (ECG) conducted (mean age 59.5 years; 60.4% female). During a median follow-up period of 7.8 years, 451 (5.9%) participants died. In Cox proportional hazards models with adjustment for demographics, lifestyle factors, physical activity, cardiovascular risks, and comorbidities, for each 1 standard deviation decrease in DistEn, the risk increased by 36%, 56% and 73%, for all-cause, cardiovascular and respiratory disease related mortality, respectively. These effect sizes were equivalent to the risk of death from being over 5 years older, having been a former smoker or suffering from diabetes mellitus. Lower DistEn was most predictive of death in those under 55 years with a prior myocardial infarction, representing an additional 56% risk for mortality compared to older subjects without. These observations remained after controlling for traditional mortality predictors, resting heart rate and HRV. Resting heart rate complexity from short, resting ECGs was independently associated with mortality in middle to older aged adults. These risks appear most pronounced in middle-aged subjects with prior MI, and may uniquely contribute to mortality risk screening.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/NEUR-MS12.html#author4"><strong class="card-body-subgroup">Ashwin Belle</strong></a>
		    	 
		    		<em> (Fifth Eye Inc., Ann Arbor, MI, USA)</em>
		    	
		    	
		    		
		    			</br>"Hemodynamic Monitoring: Seeing the Unseen"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk will discuss the various challenges in cardiac monitoring particularly from a hemodynamic perspective and also discuss some of the current methods and research efforts to predict future cardiovascular events from real-time data. This discussion will be from a commercial prospective of how to plumb the interface between mathematics and diagnostics for better treatment and outcome.  A project using real time clinical data to predict future cardiovascular events was developed from a concept at the University of Michigan where sophisticated machine learning was applied to an information-dense ECG signal to predict patient deterioration in the context of hypovolemia. Bringing this research concept to the commercial arena involved developing a data collection framework along with software tools and computational infrastructure. This was the birth of of the company Fifth Eye which brings this technology to the clinic.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS12/ONCO-MS12.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS12-ONCO: </small></br>Mathematical Oncology: From methodological studies to clinical applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Saskia Haupt (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Vincent Heuveline (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Matthias Kloor (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Germany)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS12/ONCO-MS12.html#author1"><strong class="card-body-subgroup">Natalia Komarova</strong></a>
		    	 
		    		<em> (Department of Mathematics, University of California Irvine, Irvine, California, USA)</em>
		    	
		    	
		    		
		    			</br>"CLL and the drug Ibrutinib: modeling and clinical applications"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chronic Lymphocytic leukemia is the most common leukemia, mostly arising in patients over the age of 50. The disease has been treated with chemo-immunotherapies with varying outcomes, depending on the genetic make-up of the tumor cells. Recently, a promising tyrosine kinase inhibitor, ibrutinib, has been developed, which resulted in successful responses in clinical trials, even for the most aggressive chronic lymphocytic leukemia types. The crucial questions include how long disease control can be maintained in individual patients, when drug resistance is expected to arise, and what can be done to counter it. Computational evolutionary models, based on measured kinetic parameters of patients, allow us to address these questions and to pave the way toward a personalized prognosis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/ONCO-MS12.html#author2"><strong class="card-body-subgroup">Johannes G Reiter</strong></a>
		    	 
		    		<em> (Canary Center for Cancer Early Detection, Department of Radiology, Stanford University, California, USA)</em>
		    	
		    	
		    		
		    			</br>"Minimal intermetastatic heterogeneity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Genetic intratumoral heterogeneity is a natural consequence of imperfect DNA replication. Any two randomly selected cells, whether normal or cancerous, are therefore genetically different. I will discuss the extent of genetic heterogeneity among untreated cancers with particular regard to its clinical relevance and how it can be exploited to identify metastatic seeding patterns. While genomic heterogeneity within primary tumors is associated with relapse, heterogeneity among treatment‑naïve metastases has not been comprehensively assessed. Within individual patients a large majority of driver gene mutations are common to all metastases. Further analysis revealed that the driver gene mutations that were not shared by all metastases are unlikely to have functional consequences. A mathematical model of tumor evolution and metastasis formation provides an explanation for the observed driver gene homogeneity. Based on a statistical framework for quantifying metastatic phylogenetic diversity in dozens of inferred phylogenies of colorectal cancer patients, distant metastases were typically monophyletic and genetically similar to each other. Lymph node metastases, in contrast, exhibited high levels of inter-lesion diversity. These data indicate that the cells within the primary tumors that give rise to distant metastases evolved through a narrow bottleneck and are generally more homogeneous than the primary tumor and lymph node metastases.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/ONCO-MS12.html#author3"><strong class="card-body-subgroup">Kamila Naxerova</strong></a>
		    	 
		    		<em> (Center for Systems Biology, Department of Radiology, Massachusetts General Hospital and Harvard Medical School, Boston, Massachusetts, USA)</em>
		    	
		    	
		    		
		    			</br>"On the evolutionary history of metastatic cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The evolutionary history of metastases is still largely unknown. Do metastases arise from distinct clones with special, genetically encoded properties or do they represent random samples of the primary tumor? Does metastatic spread happen early or late in tumor development? Do all metastases arise independently from the primary tumor, or do they give rise to each other? How heterogeneous are metastases? These fundamental questions have profound clinical implications but are difficult to study in human patients because relevant events predate diagnosis by many years. We are developing methods for high-efficiency lineage tracing in human tumor samples and apply these to study the roots of metastatic disease. Here, the joint insights from multiple published and unpublished studies will be presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS12/ONCO-MS12.html#author4"><strong class="card-body-subgroup">Saskia Haupt</strong></a>
		    	 
		    		<em> (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany)</em>
		    	
		    	
		    		
		    			</br>"A computational model for investigating the evolution of colonic crypts during Lynch syndrome carcinogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Introduction Lynch syndrome (LS), the most common inherited colorectal cancer (CRC) syndrome, increases the cancer risk in affected individuals. LS is caused by pathogenic germline variants in one of the DNA mismatch repair (MMR) genes, complete inactivation of which causes numerous mutations in affected cells. As CRC is believed to originate in colonic crypts, understanding the intra-crypt dynamics caused by mutational processes is essential for a complete picture of LS CRC and may have significant implications for cancer prevention.
Methods We propose a computational model describing the evolution of colonic crypts during LS carcinogenesis. Extending existing modeling approaches for the non-Lynch scenario, we incorporated MMR deficiency and implemented recent experimental data demonstrating that somatic CTNNB1 mutations are common drivers of LS-associated CRCs if affecting both alleles of the gene. Further, we simulated the effect of different mutations on the entire crypt, distinguishing non-transforming and transforming mutations.
Results As an example, we analyzed the spread of mutations in the genes APC and CTNNB1, which are frequently mutated in LS tumors, as well as of MMR deficiency itself. We quantified each mutation's potential for monoclonal conversion and investigated the influence of the cell location and of stem cell dynamics on mutation spread.
Conclusion The in silico experiments underline the importance of stem cell dynamics for the overall crypt evolution. Further, simulating different mutational processes is essential in LS since mutations without survival advantages (the MMR deficiency-inducing second hit) play a key role. The effect of other mutations can be simulated with the proposed model. Our results provide first mathematical clues for effective surveillance protocols for LS carriers.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-12
										</br>
										Wednesday, June 16 at 04:15am (PDT)</br>Wednesday, June 16 at 12:15pm (BST)</br>Wednesday, June 16 08:15pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>













    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
