---
layout: group
author: esmb
day: "Monday (Tuesday)"
subgroup: "ONCO"
title: "Systems Biology Models of Tumor Metabolism"
organizers: "Shubham Tripathi (Rice University, USA), Abhinav Achreja (University of Michigan, USA)"
description: "Postulated as an emerging hallmark of tumor cells nearly a decade ago, metabolic reprogramming is now recognized as a key feature of tumor progression across cancer types, with widespread therapeutic implications. Experimental advances have played a key role in revealing the different metabolic behaviors exhibited by tumor cells that contribute towards uncontrolled proliferation, metastasis, immune evasion, and drug resistance. Simultaneously, mathematical modeling approaches have been crucial to the development of frameworks that provide insights unattainable with empirical data alone. This mini-symposium will bring together thought leaders who have been developing a diverse set of mathematical modeling approaches to understand tumor metabolism. These include coarse-grained metabolic models that couple to gene networks, detailed mechanistic models, and genome-scale metabolic models, which collectively reflect the different systems biology approaches to studying tumor metabolism. The speakers will discuss application of these models to predicting the response of tumor cells to therapeutic interventions and the current challenges in this domain. While introducing the audience to the latest developments at this frontier of investigating tumor vulnerabilities, the mini-symposium will facilitate interaction between researchers working on distinct approaches towards the same goal of discovering the metabolic vulnerabilities of tumor cells."
code: "MS03"
author1: "Dongya Jia"
inst1: " (Laboratory of Integrative Cancer Immunology, National Cancer Institute, National Institutes of Health, USA)"
title1: "Elucidating cancer catabolism and anabolism by coupling gene regulation with metabolic pathways"
abstract1: "Cancer cells can adapt their metabolic phenotypes to meet various bioenergetic and biosynthetic needs, and to survive the therapeutic treatments. It remains largely unclear how cancer cells orchestrate different metabolic phenotypes (glycolysis, oxidative phosphorylation etc.) and various metabolic ingredients (glucose, fatty acids, glutamine, etc.). Since recent efforts in targeting individual cancer metabolic pathways have been largely ineffective, a better understanding of cancer metabolic network and its plasticity will progressively facilitate the development of more effective therapeutic strategies. 
The goal of this study is to elucidate the mechanisms underlying cancer metabolic plasticity within both catabolism and anabolism by integrated theoretical-experimental approaches. We constructed a metabolic modeling framework featuring regulation by the master gene regulators (AMPK, HIF-1, MYC etc.) and their cross-talk with metabolic pathways. The beauty of the framework is at least two-fold. First, it has considered all three most important metabolic ingredients (glucose, fatty acids, glutamine) for tumorigenesis and metastasis. Second, it has allowed us to investigate the interaction between catabolism (glucose/glutamine oxidation, etc.) and anabolism (reductive glucose/glutamine metabolism), therefore offering a higher-level view of cancer metabolism. 
Our work elucidates how cancer cells can mix and match different metabolic phenotypes. For example, we show that cancer cells can acquire a hybrid metabolic phenotype where both glycolysis and OXPHOS are actively used, and a metabolically “low-low” phenotype where cells exhibit low activity of glycolysis and OXPHOS. Importantly, the hybrid metabolic phenotype characterizes highly metastatic breast cancer cells and the low-low phenotype can characterize drug-tolerant melanoma cells. Consequently, an accurate characterization of cancer metabolism enabled us to present effective combination therapies targeting metabolism in breast cancer."
author2: "Prahlad Ram"
inst2: " (The University of Texas MD Anderson Cancer Center, USA)"
title2: "4D Ex-vivo CRISPR / CAS9 Whole-genome Screen to Identify Genes Regulating Early Lung Cancer Metastasis"
abstract2: "Metastatic lung cancer has a 5-year survival rate of 5%. Lung cancers tend to be
asymptomatic until late stages, and almost 90% are not diagnosed until they are
advanced. The genomic events early in the metastatic process has not been completely
deciphered. Utilizing CRISPR/Cas9 whole genome knockout screen in the A549 lung
adenocarcinoma cell line and coupling it with a novel ex vivo 4D lung metastasis model has
now allowed us to examine early genomic events in metastasis. Using this approach we
recovered genes previously implicated in lung cancer and metastasis validating this
approach. Additionally we identified a transcription factor network driven by SPI1 which
was enriched in our screen. Experimental validation of SPI1 uncovered a novel role of this
network in the metastatic process."
author3: "Andrew Raddatz"
inst3: " (The Wallace H. Coulter Department of Biomedical Engineering, Georgia Institute of Technology and Emory University, USA)"
title3: "Kinetic Modeling of Redox Metabolism in Head and Neck Cancer"
abstract3: "Reactive oxygen species (ROS) levels are frequently elevated in head and neck
tumors because of downstream tumor-promoting outcomes. Moderate levels of ROS promote tumorigenesis because they increase proliferation, initiate angiogenesis, and trigger survival signaling pathways. Additionally, treatment options such as radiation, chemotherapy, and even immunotherapy have been shown to involve tumor redox biology. A greater mechanistic understanding of how redox-based expression profiles in cancer affect susceptibility to certain treatments is needed to improve clinical decisions. Here, we developed an intracellular ODE model to represent how a cancer cell’s redox state would respond to treatment with a ROS-generating drug. The following antioxidant systems were included in the model based on previous H2O2 clearance modeling: catalase, peroxiredoxin, glutathione, and the protein thiol pool. Initial parameterization of the model included taking values reported in the literature and scanning the BRENDA database for remaining rate constants where available. To validate our model, we experimentally silenced antioxidant enzymes represented in the model by siRNA and observed the effect on production of H2O2. We found that knocking down PRDX1 (peroxiredoxin 1), CAT (catalase), and TXNRD1 (thioredoxin reductase 1) via siRNA led to a relative increase in extracellular H2O2 upon drug application. Then, using scRNA-seq data, we generated single cell models to predict how transcriptome variability across patients and within tumors can influence ROS accumulation and redox potentials within the cell under drug treatment."
author4: "Stacey Finley"
inst4: " (University of Southern California, USA)"
title4: "Modeling tumor-stromal metabolic crosstalk in colorectal cancer"
abstract4: "Colorectal cancer (CRC) is a major cause of morbidity and mortality in the United States. Tumor-stromal metabolic crosstalk in the tumor microenvironment promotes CRC development and progression, but exactly how stromal cells, in particular cancer-associated fibroblasts (CAFs), affect the metabolism of tumor cells remains unknown. Here we take a data-driven approach to investigate the metabolic interactions between CRC cells and CAFs, integrating constraint-based modeling and metabolomic profiling. Using metabolomics data, we perform unsteady-state parsimonious flux balance analysis to infer flux distributions for central carbon metabolism in CRC cells treated with or without CAF-conditioned media. We find that CAFs reprogram CRC metabolism through stimulation of glycolysis, the oxidative arm of the pentose phosphate pathway (PPP), and glutaminolysis as well as inhibition of the tricarboxylic acid cycle. To identify potential therapeutic targets, we simulate enzyme knockouts and find that inhibiting the hexokinase and glucose-6-phosphate dehydrogenase reactions exploits the CAF- induced dependence of CRC cells on glycolysis and oxidative PPP. Our work gives mechanistic insights into the metabolic interactions between CRC cells and CAFs and provides a framework for testing hypotheses towards CRC-targeted therapies."
---
