---
layout: group
author: esmb
day: "Tuesday (Wednesday)"
subgroup: "ONCO"
title: "Modeling translational oncology"
organizers: "Russell Rockne (Beckman Research Institute, City of Hope National Medical Center, USA), Andrea Bild (Beckman Research Institute, City of Hope National Medical Center, USA)"
description: "This session focuses on application of mathematical models in cancer treatment and tumor biology.  Integrated clinical, biological, and mathematical approaches will be discussed as they apply to tumor models.  Results from these models will include optimization of treatment strategies directed at individual tumor features, quantification of evolving subclonal populations linked to therapy resistance, and prediction of treatment response to immunotherapy.  Speakers from both academia and pharmaceutical research centers will together discuss emergent opportunities for growth in this field."
code: "MS07"
author1: "Jessica Leete"
inst1: " (Pfizer Inc, Cambridge MA, USA)"
title1: "Towards virtual populations for human efficacy prediction in lung cancer: preclinical to clinical translation of anti-PD-(L)1 treatments"
abstract1: "Objectives: Immune checkpoint inhibitors such as anti-PD-1 and anti-PD-L1 are promising new therapeutic options that cease tumor cells’ immunosuppressive properties; however, low response rates to these therapeutics indicate an unmet medical need in treating non-small cell lung cancer (NSCLC). NSCLC is a highly heterogenous disease both spatially and genetically, and provides unique challenges to predicting treatment outcome.

We propose a model of anti-PD-(L)1 in both preclinical mouse models and human NSCLC. We use this model to explore the effect of parameters on model outcome in preparation for implementation of virtual population methods to explore inter-patient variability.
 
Methods: We present a drug and target focused model of anti-PD-(L)1 treatment in CT26 BALB/c mice and NSCLC in humans. The model includes the mechanisms of action of anti-PD-1 and anti-PD-L1. We translate the model structure to that of a 'typical' NSCLC patient using published popPK and binding affinities of approved anti-PD-(L)1 treatments. Parameter sensitivity is explored through both local and global sensitivity analyses. 
 
Results: Pre-clinical simulations show the model's ability to match a wide variety of responses to treatment by allowing tumor growth parameters and tumor PD-1+ CD8+ T cell concentration to vary between individuals. Model simulations are sensitive to parameters that determine PD-(L)1 abundance and the speed and magnitude of T cell proliferation after treatment.
 
Conclusions: Focus on modeling a 'typical' human patient may lead to an incomplete characterization of treatment efficacy in situations where there is high inter-patient variability in responses. Immuno-oncology treatments in particular may benefit from methods that can quantify the effect of patient variability on treatment outcome."
author2: "Nataly Kravchenko-Balasha"
inst2: " (The Institute of Biomedical and Oral Research, The Hebrew University of Jerusalem, Israel, Israel)"
title2: "Computational quantification and characterization of independently evolving cellular subpopulations within tumors is critical to inhibit anti-cancer therapy resistance"
abstract2: "Drug resistance continues to be a principle limiting factor across diverse anti-cancer therapies. Contributing to the complexity of this challenge is cancer plasticity where one cancer subtype switches to another in response to treatment (e.g. Triple Negative Breast Cancer (TNBC) to Her2-positive breast cancer). For optimal treatment outcomes, accurate tumor diagnosis and subsequent therapeutic decisions are vital. 
In this study an information-theoretic single-cell quantification strategy was developed to provide a high resolution and individualized assessment of tumor composition for a customized treatment approach.
Briefly, this single-cell quantification strategy computes a barcode based on at least 100, 000 tumor cells and reveals a set of ongoing processes in each cell.
Using these cell-specific barcodes, distinct subpopulations evolving within the tumor in response to an outside influence (e.g. anticancer treatments) are revealed and mapped. Barcodes, are further applied to assign targeted drug combinations to each individual tumor to optimize tumor response to therapy.
This unique strategy was validated using TNBC models and patient-derived tumors known to switch phenotypes in response to radiotherapy (RT).
We show that a barcode-guided targeted cocktail significantly enhances tumor response to RT and prevents regrowth of once resistant tumors. The strategy presented herein has the potential to significantly reduce the occurrence of cancer treatment resistance, with a broad applicability in clinical use."
author3: "Alexander R. A. Anderson"
inst3: " (Integrated Mathematical Oncology Department H. Lee Moffitt Cancer Center & Research Institute, USA)"
title3: "Exploiting evolution to design better cancer therapies"
abstract3: "Our current approach to cancer treatment has been largely driven by finding molecular targets, those patients fortunate enough to have a targetable mutation will receive a fixed treatment schedule designed to deliver the maximum tolerated dose (MTD). These therapies generally achieve impressive short-term responses, that unfortunately give way to treatment resistance and tumor relapse. The importance of evolution during both tumor progression, metastasis and treatment response is becoming more widely accepted.  However, MTD treatment strategies continue to dominate the precision oncology landscape and ignore the fact that treatments drive the evolution of resistance.  Here we present an integrated theoretical/experimental/clinical approach to develop treatment strategies that specifically embrace cancer evolution. We will consider the importance of using treatment response as a critical driver of subsequent treatment decisions, rather than fixed strategies that ignore it. We will also consider using mathematical models to drive treatment decisions based on limited clinical data. Through the integrated application of mathematical and experimental models as well as clinical data we will illustrate that, evolutionary therapy can drive either tumor control or extinction using a combination of drug treatments and drug holidays. Our results strongly indicate that the future of precision medicine shouldn’t be in the development of new drugs but rather in the smarter evolutionary, and model informed, application of preexisting ones."
author4: "Andrew Gentles"
inst4: " (Departments of Medicine and Biomedical Informatics, Stanford University, USA)"
title4: "Building an atlas of cell states and cellular ecosystems across human solid tumors"
abstract4: "Determining how cells vary with their local signaling environment and organize into distinct cellular communities is critical for understanding processes as diverse as development, aging, and cancer. We have developed EcoTyper, a new machine learning framework for large-scale identification and validation of cell states and multicellular communities from bulk, single-cell, and spatially-resolved gene expression data. When applied to 12 major cell lineages across nearly 6,000 tumor specimens from 16 types of human carcinoma, EcoTyper identified 69 transcriptionally-defined cell states. Most cell states were specific to neoplastic tissue, ubiquitous across tumor types, and significantly prognostic. By analyzing cell state co-occurrence patterns, we discovered 10 clinically-distinct multicellular communities with unexpectedly strong conservation, including four with unique myeloid and stromal elements, one enriched in normal tissue, and two associated with early cancer development. This work elucidates fundamental units of cellular organization in human carcinoma and provides a framework for large-scale profiling of cellular ecosystems in any tissue."
---
