<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Contributed Talk Session - CT05 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Contributed Talk Session - CT05 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Contributed Talk Session - CT05" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/CT05/" />
<meta property="og:url" content="http://localhost:4000/CT05/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/CT05/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Contributed Talk Session - CT05","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Contributed Talk Session - CT05">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Contributed Talk Session - CT05" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Contributed Talk Session - CT05" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Contributed Talk Session - CT05</h1>
        <h4>Tuesday, June 15 at 10:30pm (PDT)</br>Wednesday, June 16 at 06:30am (BST)</br>Wednesday, June 16 02:30pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="CT05">Contributed Talk Session - CT05</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CBBS/CBBS-CT05.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>CT05-CBBS: </small></br>CBBS Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CBBS/CBBS-CT05.html#author1"><strong class="card-body-subgroup">Donggu Lee</strong></a>
		    	 
		    		<em>Konkuk University</em>
		    	
		    	
		    		
		    			</br>"Role of OCT1 in regulation of miR-451-LKB1-AMPK-OCT1-mTOR core signaling network and cell invasion in glioblastoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma multiforme (GBM) is the most aggressive form of brain cancer with a short central survival time. GBM is characterized by aggressive proliferation and critical cellular infiltration. miR-451 and its downstream molecules (LKB1, AMPK, OCT1, mTOR) are known to play a pivotal role in balancing proliferation and aggressive invasion in response to metabolic stress in a tumor microenvironment (TME). Recent studies have shown that OCT1 and LKB1 play an important role in regulation of the mutual inhibition between cell proliferation and migration. In this work, we develop a mathematical model of signaling pathway dynamics in GBM evolution which focuses specifically on the relative balance of proliferative capacity and invasion potential. In the work, we represent the miR-451/LKB1/AMPK/OCT1/mTOR pathway by a mathematical model and show how the effect of fluctuating glucose on tumor cells needs to be reprogrammed by taking into account the recent history of glucose variations and an LKB1-OCT1 mutual feedback loop. The simulations show how changes in glucose have a significant effect on the level of key signaling molecules, determining in promotion or inhibition of glioma cell migration (Kim, Lee, & Lawler, Phil Trans Roy Soc-B, 2020).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT05.html#author2"><strong class="card-body-subgroup">Yukitaka Ishimoto</strong></a>
		    	 
		    		<em>Akita Prefectural University</em>
		    	
		    	
		    		
		    			</br>"In-vivo cell flow visualisation using deep learning and other means"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In recent years, measurements of cellular movements and forces in living body have been paid much attention, chiefly for regenerative therapy and medical applications. It is because they are thought to give deeper insight on tissue mechanics and engineering. There are various ways of invasive and non-invasive measurements. Among them, cell deformations and flow play an pivotal role for elucidation of tissue/organ morphogenesis. However, conventional flow visualization techniques, such as PIV and PTV, often fail to capture the cell flow due to cellular morphological events. To adequately develop such measurements, it is critical to establish precise detection of positions/shapes and correspondence between individual cell shapes at different timepoints. In this work, we show our two distinct attempts of flow visualization of deforming epithelial sheet. One is for particle tracking velocimetry (PTV) of four-dimensional cell flow by using deep neural network model (DNN) onto deforming nucleus images. The other is to track cell shape changes of the sheet by extracting cell boundaries from live-imaging data and further fitting them to a vertex-edge configuration of the bubbly vertex model. Further extensions of both attempts will also be discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT05.html#author3"><strong class="card-body-subgroup">Tara Hameed</strong></a>
		    	 
		    		<em>Imperial College London</em>
		    	
		    	
		    		
		    			</br>"A systematic workflow to assess the useability of data in model development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Data sparsity is one of the bottlenecks we often encounter in model development, especially for disease modelling or in fields where interdisciplinary cross-collaboration is still being developed. When a model is fit to sparse data, it is hard to discern whether potential model misfit is caused by inherent model misspecification, which requires reformulation of the model, or by data sparsity, which requires further data collection. We proposed a systematic workflow to assess the degree to which the available data can inform mathematical models theoretically, by upcycling a known statistical workflow that uses simulation studies. The proposed workflow quantifies the useability of the experimental data in terms of expected quality of parameter estimation and model prediction. Application of the workflow to our mathematical model of early-stage invasive aspergillosis (pulmonary fungal infection), adapted from a previous model, allowed us to suggest future experiments that could provide more “useable” data to infer the model's nonlinear interaction parameters and to make better predictions. The presented workflow could be useful when models are developed with data sparsity as a limiting factor for model-based inference.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT05.html#author4"><strong class="card-body-subgroup">Anibal Thiago Bezerra</strong></a>
		    	 
		    		<em>Instituto de Ciências Exatas, Universidade Federal de Alfenas</em>
		    	
		    	
		    		
		    			</br>"		 Gastric Emptying and Orocaecal Transit Analyses of Diabetic and Control Individuals Through Deep Neural Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Classical analysis of experimental data generally relies on statistical methods. These methods, however, can be contradictory depending on the methodology and the adopted metrics. In the quantification of gastric emptying (GE) and orocaecal transit (OCT), this is the case in the discrimination between rats who have dysfunctions or diseases like diabetes and the ones that do not. Metrics involved in this context are mean gastric emptying time (MGET), orocaecal transit time (OCTT), and mean caecum arrival time (MCAT). To overcome their limitations, here we present an artificial neural network (ANN) capable of discriminating between control and diabetic individuals rats through GE and OCT data analysis of alternate current biosusceptometry (ACB). For GE, the ANN classification reached an accuracy above 90% after a few epochs. The respective sensitivity was 88%, and the specificity was 83%. For OCT, the accuracy also achieved 90%, with a specificity of 75% and unitary sensitivity. These achieved results support that the proposed ANN can be an alternative methodology to the classical method employed over the years in the gastrointestinal transit area. This work is supported by funding from grant #2020/05556-0, São Paulo Research Foundation (FAPESP).
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT05.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT05-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT05.html#author1"><strong class="card-body-subgroup">David Holloway</strong></a>
		    	 
		    		<em>British Columbia Institute of Technology</em>
		    	
		    	
		    		
		    			</br>"Controlling the number of cotyledons in conifer embryos"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Conifers, unlike flowering plants, generate variable numbers of cotyledons (embryonic 'seed leaves'). Conifer cotyledons do not form all over the dome-shaped embryo, but form in a single ring at a particular distance from the tip. A 3-fold increase in this ring radius from 55 to 180 µm corresponds to the experimentally observed range of 2 to 10 regularly-spaced cotyledons. In the flowering plant Arabidopsis, leaves are also initiated at a particular distance from the growing tip. Molecularly, this is at a 'trough' between two expression domains, of REV (an HD-ZIP III protein) above the leaves and KAN below the leaves. REV and KAN are mutually inhibitory via miRNAs (tasiARF from REV, miR166 from KAN). This is at least partly shared by conifers: overexpression of miR166 in larch decreases HD-ZIP III expression and affects cotyledon formation. We have developed a model for HD-ZIP III (H), KAN (K) regulation to investigate how their interface position is controlled - in particular, what allows for the 3-fold natural variability in conifer cotyledon ring radius. Simulating Arabidopsis H/K experimental perturbations contributes to a general mechanism for radial positioning, as well as quantitatively predicting radial shifts in new conifer experiments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author2"><strong class="card-body-subgroup">Tamsin Spelman</strong></a>
		    	 
		    		<em>Sainsbury Laboratory, University of Cambridge, UK</em>
		    	
		    	
		    		
		    			</br>"Links between microtubules and nucleus shape in a plant root hair cell"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Root hair cells develop out of Trichoblast cells in the plant root epidermis and are characterised by a long thin protrusion which in Arabidopsis is ≈10μm wide and can grow to ≈1mm in length [1]. For growth of this protrusion, nucleus migration up the root hair is necessary, with the nucleus positioned ≈80μm back from the growing tip [2]. Our aim is to understand nucleus shape and position in the root hair, particularly focusing on how it is affected by the cytoskeleton (microtubules and actin). By segmenting experimental data, we analyse 3D nucleus shape and motion before and after treatment with drugs effecting osmolarity and microtubule organisation: Mannitol and Oryzalin. To further understand the relationship between the nucleus and microtubules, we then analyse the microtubule distribution in the nucleus-tip region of the root hair. Using 3D microtubule simulations and simple nuclear dynamics models, we analyse the microtubule density distributions and organisation for a range of conditions in the root hair cell and compare these to the distributions obtained experimentally. [1] Grierson C et. al. (2014) Root hairs. Arabidopsis Book. 12:e0172. [2]T. Ketelaar et. al. (2002) Positioning of Nuclei in Arabidopsis Root Hairs. The Plant Cell, 14(11):2941-2955;
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author3"><strong class="card-body-subgroup">Daniel Koch</strong></a>
		    	 
		    		<em>King's College London</em>
		    	
		    	
		    		
		    			</br>"Information Processing by Homo-Oligomeric Proteins: From First Principles to Cardiac Arrhythmias"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Reversible protein homo-oligomerisation, i.e. the formation of larger protein complexes out of identical subunits, is observed for 30-50% of all vertebrate proteins. Despite being a ubiquitous phenomenon, the specific function of protein homo-oligomerisation remains poorly understood. I previously demonstrated theoretically that homo-oligomerisation could be a versatile mechanism for a range of signal processing capabilities such as dynamic signal encoding, homeostasis and bistability via pseudo-multisite modification. In this talk I will present the first dynamical systems model of phospholamban (PLN), a crucial mediator protein of β-adrenergic signaling and regulator of calcium cycling in heart muscle cells. Importantly, PLN forms homo-pentamers whose function remained unclear for decades. Simulations and model analyses demonstrate that pentamers enable bistable phosphorylation and further constitute substrate competition based low-pass filters for phosphorylation of monomeric PLN. I confirmed both predictions of my model experimentally by demonstrating substrate competition in vitro and hysteresis of pentamer phoshorylation in cardiomyocytes. These non-linear phenomena could ensure consistent monomer phosphorylation and calcium cycling despite noisy signaling activity in the upstream network and are likely impaired by a genetic mutation that causes arrhythmogenic heart disease. These studies show that homo-oligomerisation can play unanticipated and potentially disease relevant roles in biochemical signaling networks.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT05.html#author4"><strong class="card-body-subgroup">Adriana Zanca</strong></a>
		    	 
		    		<em>The University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"	 Cell proliferation and migration during wound healing"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Experiments have suggested that during skin wound healing, the wound front is comprised of a hyperproliferative region behind a non-proliferating migrating tongue at the wound edge. Mathematical and computational models allow us to efficiently explore the effects of changing the characteristics of these proposed regions in order to suggest plausible hypotheses about the mechanics of the regions, and wound healing in general. In this work we use cell-based computational modelling to investigate competing experimental findings regarding whether the regions spatially overlap. Furthermore, we examine the effect of changing the size, location and cell characteristics, such as proliferation and migration rate, of each region on the behaviour of the wound edge.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/EVOP/EVOP-CT05.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>CT05-EVOP: </small></br>EVOP Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/EVOP/EVOP-CT05.html#author1"><strong class="card-body-subgroup">Barbara Boldin</strong></a>
		    	 
		    		<em>Faculty of Mathematics, Natural Sciences and Information Technologies, University of Primorska, Slovenia</em>
		    	
		    	
		    		
		    			</br>"The evolution of respiratory disease virulence and diversity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Theoretical studies of virulence evolution typically assume a positive trade-off between infectivity and harmfulness. This is a valid assumption for diseases where both quantities are determined solely by within-host infection load. However, epidemiological parameters in highly structured host organisms, such as mammals, are largely determined by how the disease agents distribute themselves over body compartments. In respiratory diseases there is even a negative trade-off, with diseases of the lower respiratory tract being both less infective and more harmful. In this talk, we discuss the evolutionary consequences of the interplay between virulence that decreases with an increase in transmission and cross-immunities between pathogen strains. The most salient outcomes of our study are that (i) the upper respiratory tract will support a higher disease diversity, (ii) that emerging respiratory diseases will tend to be more harmful and less infective and (iii) that disease diversity increases with host population density.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT05.html#author2"><strong class="card-body-subgroup">Yuanxiao Gao</strong></a>
		    	 
		    		<em>Max Planck Institute for Evolutionary Biology</em>
		    	
		    	
		    		
		    			</br>"Evolution of irreversible somatic differentiation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A key innovation emerging in complex animals is irreversible somatic differentiation: daughters of a vegetative cell perform a vegetative function as well, thus, forming a somatic lineage that can no longer be directly involved in reproduction. Primitive species use a different strategy: vegetative and reproductive tasks are separated in time rather than in space. Starting from such a strategy, how is it possible to evolve life forms which use some of their cells exclusively for vegetative functions? Here, we developed an evolutionary model of development of a simple multicellular organism and found that three components are necessary for the evolution of irreversible somatic differentiation: (i) costly cell differentiation, (ii) vegetative cells that significantly improve the organism's performance even if present in small numbers, and (iii) large enough organism size. Our findings demonstrate how an egalitarian development typical for loose cell colonies can evolve into germ-soma differentiation dominating metazoans.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT05.html#author3"><strong class="card-body-subgroup">Yoav Ram</strong></a>
		    	 
		    		<em>Tel Aviv University</em>
		    	
		    	
		    		
		    			</br>"Non-Vertical Cultural Transmission , Assortment , and the Evolution of Cooperation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present a model for the evolution of cooperation under vertical, horizontal, and oblique cultural transmission. We find that the evolution of cooperation is facilitated by its horizontal transmission and by an association between social interactions and horizontal transmission. The effect of oblique transmission depends on the horizontal transmission bias. Stable polymorphism of cooperation and defection can occur, and when it does, reduced association between social interactions and horizontal transmission evolves, which leads to a decreased frequency of cooperation and lower population mean fitness. We compare our results to outcomes of stochastic simulations of structured populations. Parallels are drawn with Hamilton's rule incorporating assortment and relatedness.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT05.html#author4"><strong class="card-body-subgroup">Christin Nyhoegen</strong></a>
		    	 
		    		<em>Max Planck Institute for Evolutionary Biology, Plön, Germany</em>
		    	
		    	
		    		
		    			</br>"Within-host evolution of antibiotic resistance under sequential therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The rapid evolution of antibiotic resistance and the resulting loss in treatment options call for the development of sustainable treatment strategies. Supported by laboratory experiments, alternating antibiotics during treatment has been proposed as a promising approach. Evolutionary trade-offs, especially collateral sensitivity, could potentially further improve the outcome.A limitation of in-vitro evolution experiments is that they do not account for the complex environment of the patient's body. Drugs persist in the body for some time at continuously decreasing concentrations, leading to a temporal overlap of the drugs in a cycling schedule. It is a priori not clear how drug-drug-interactions during these periods of drug overlap influence the outcome of sequential therapy. To close this gap, we set up a pharmacokinetic-pharmacodynamic model that incorporates drug-drug-interactions. We aim to reveal the treatment settings that optimize the outcome of sequential therapy, given the risk of resistance evolution. Our results suggest that drug-drug-interactions strongly influence the optimal protocol. For synergistic drugs pairs, rapid switching of drugs minimizes the time to eradication of the pathogen population. For antagonistic drugs, the decision is not as straightforward, and switching the drugs less often is sometimes preferable. Collateral sensitivity only improves the efficiency if cycling is slow.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/IMMU/IMMU-CT05.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>CT05-IMMU: </small></br>IMMU Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/IMMU/IMMU-CT05.html#author1"><strong class="card-body-subgroup">Ke Li</strong></a>
		    	 
		    		<em>The University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"Modelling the effect of MUC1 on influenza virus infection kinetics and macrophage dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The host immune response is important to defend against influenza viral infection. However, overstimulation of the host immune response can lead to pathology, indicating a subtle balance between a protective and a destruct response. Dysregulated immune responses are often associated with an excessive recruitment of macrophages. MUC1 belongs to the family of cell surface (cs-) mucins and has been shown to be an important and dynamic component of the host innate immune response, associated with recruitment of macrophages. Experimental evidence indicates that its presence reduces influenza infection severity. However, the detailed effects of MUC1 in vivo remain elusive, limiting our ability to predict the efficacy of potential treatments that target MUC1. To address this limitation, we fit two mathematical models to available in vivo kinetic data for both virus and macrophage populations in wild-type and MUC1 knockout mice. Both models provide evidence that MUC1 reduces the susceptibility of epithelial cells and show that the MUC1 regulates the recruitment of macrophages and thus the host immune response. This study improves our understanding of the dynamic role of MUC1 against influenza infection and may support the development of novel antiviral treatments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/IMMU/IMMU-CT05.html#author2"><strong class="card-body-subgroup">Juan Antonio Magalang</strong></a>
		    	 
		    		<em>Theoretical Physics Group, National Institute of Physics, University of the Philippines</em>
		    	
		    	
		    		
		    			</br>"Stochastic resetting antiviral therapies prevent drug resistance development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We study minimal mean-field models of viral drug resistance development in which the efficacy of a therapy is described by a one-dimensional stochastic resetting process with mixed reflecting-absorbing boundary conditions. We derive analytical expressions for the mean survival time for the virus to develop complete resistance to the drug. We show that the optimal therapy resetting rates that achieve a minimum and maximum mean survival times undergo a second- and first-order phase transition-like behaviour as a function of the therapy efficacy drift. We illustrate our results with simulations of a population dynamics model of HIV-1 infection.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MEPI/MEPI-CT05.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>CT05-MEPI: </small></br>MEPI Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MEPI/MEPI-CT05.html#author1"><strong class="card-body-subgroup">Kamuela Yong</strong></a>
		    	 
		    		<em>University of Hawaii - West Oahu</em>
		    	
		    	
		    		
		    			</br>"A Mathematical Model of the Transmission of Rat Lungworm Disease"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The parasite Angiostrongylus cantonensis (AC), known as the rat lungworm hasa complex life cycle that begins when adult worms found in rats reproduce. Larvae exitthe rats through their feces where terrestrial gastropods such as snails and slugsbecome infected after consuming the rat feces. The life cycle is complete when ratsconsume infected snails and slugs. In this paper, we develop a mathematical model torepresent the transmission of AC through its life cycle. Numerical simulations areconducted to determine the factors that have the most impact on the transmission ofAC.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT05.html#author2"><strong class="card-body-subgroup">Artem Novozhilov</strong></a>
		    	 
		    		<em>North Dakota State University</em>
		    	
		    	
		    		
		    			</br>"Parametric heterogeneity in epidemiological models and modeling of COVID-19"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The theory of heterogeneous populations with parametric heterogeneity is a well developed area of mathematical modeling in biology. We say that our mathematical models describe parametric heterogeneity if we assume that the individuals in populations are heterogeneous with respect to some unchangeable with time parameter, such as birth of death rates or individual's susceptibility to an infectious disease for instance. In particular, the applications of this theory to epidemiological modeling yield very tractable analytically but also quite profound general results  (e.g., that the epidemics is always less severe in heterogeneous populations compare to a homogeneous one). Recently, the observed avalanche of data related to the spread of COVID-19 around the world prompted the revival of interest in such heterogeneous models, a number of old results were rediscovered in different contexts, and also new results were obtained. In my talk I aim to present most of the known analytical results about heterogeneous populations with parametric heterogeneity from the general point of theory of heterogeneous populations and also discuss the dangers and pitfalls of applying this theory to the observed data. References: [1] Novozhilov, Math Biosc, 2008; [2] Novozhilov, Math Mod Nat Phen, 2012, Karev and Novozhilov, Math Biosc, 2019
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT05.html#author3"><strong class="card-body-subgroup">Taeyong Lee</strong></a>
		    	 
		    		<em>School of Mathematics and Computing (Mathematics), Yonsei University, Seoul, Republic of Korea</em>
		    	
		    	
		    		
		    			</br>"The impact of control strategies for COVID-19 in South Korea"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The COVID-19 (Coronavirus disease) has spread since the first occurrence on 20 Jan 2020 in South Korea. To mitigate the transmission, KDCA (Korea Disease Control and Prevention Agency) has taken various types of control measures including school-closure and social-distancing. We developed an age-stratified compartmental model considering quarantine and isolation to describe the disease dynamics, which has been calibrated to the newly confirmed data from 20 Jan to 2 Apr 2020. The effectiveness of intervention measures was investigated under several scenarios through the simulation of the proposed model.The results predict that the epidemic threshold for increase of contacts is 1.6 times, which brings the net reproduction number to 1. A second outbreak is expected if the contacts between teenage increase more than 3.3 times when school opens. The reduction of average time until isolation and quarantine from three days to two would decrease cumulative cases by 60% and 47%, respectively. We also study the impact of control strategies considering transmission from asymptomatic or mild symptomatic people, because the infectiousness of those has been controversial.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT05.html#author4"><strong class="card-body-subgroup">Anjana Pokharel</strong></a>
		    	 
		    		<em>Tribhuvan University Kathmandu, Nepal</em>
		    	
		    	
		    		
		    			</br>"Modeling the Impact of Vaccination on the Transmission Dynamics of Measles in Nepal"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Measles is one of the highly contagious human viral diseases caused by the virus of paramyxovirus family.  Despite availability of successful vaccine, measles outbreaks occur presumably due to the lack of compliance of vaccination. In this work, we will develop a deterministic mathematical model that explains the transmission dynamics of measles in Nepal.  We will perform the numerical simulation to explore the impact of the vaccinations. We will also explain the qualitative behavior of the model.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT05.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT05-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT05.html#author1"><strong class="card-body-subgroup">Fatemeh Sadat Fatemi Nasrollahi</strong></a>
		    	 
		    		<em>Pennsylvania State University</em>
		    	
		    	
		    		
		    			</br>"Attractor identification method based on generalized positive feedback loops and their functional relationships"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Boolean modeling has been shown to successfully capture the attractors (emergent behaviors) in complex systems. Here we propose an efficient attractor finding method that relies on the identification of stable motifs [1] in Boolean models of plant-pollinator community assembly [2]. Stable motifs are the smallest positive cycles in the network that can sustain a specific state regardless of the state of the nodes outside the stable motif. We find that stable motifs can have three types of functional relationships with each other: Mutual exclusivity: Two stable motifs stabilizing the same node(s) but in different states; Conditionality: Stabilization of a stable motif only when a set of conditions are met via the stabilization of a different stable motif; Logical determination: Automatic stabilization of a stable motif as a result of stabilization of another stable motif. Based on these relationships, we developed an algorithm to identify all self-consistent mutually exclusive groups of stable motifs, and showed that stabilization of any of these groups leads to a distinct attractor. We applied this algorithm to 4000 networks of 40-100 species, compared its performance with three other attractor identification methods, and showed that it can speed and simplify the attractor identification task considerably.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author2"><strong class="card-body-subgroup">Megan Coomer</strong></a>
		    	 
		    		<em>Melbourne University</em>
		    	
		    	
		    		
		    			</br>"Shaping the Epigenetic Landscape: Complexities and Consequences"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The metaphor of Waddington's epigenetic landscape has become an iconic representation of cellular differentiation.  Single-cell transcriptomic data allows us to probe this landscape and gain insights into the regulatory dynamics underlying developmental processes. Reconstructing such landscapes from data has typically been based on strong assumptions about the dynamics of cells through gene expression state. Often, concepts from equilibrium thermodynamics have been used. Since biological processes are inherently noisy it is important to consider the presence of stochastic fluctuations in this context.  We use a simple model to highlight complexities and limitations that arise when reconstructing the potential landscape in the presence of stochasticity. Specifically, we contrast ways in which additive and multiplicative noise shape the landscape on top of the deterministic dynamics.  We show that the subtle interplay between the deterministic and stochastic components of the system's dynamics can have very unsubtle consequences: depending on the dynamics and noise, even qualitative features of the system dynamics — number and nature of stationary points — can change. Casual or ad hoc modelling of noise in the underlying regulatory networks can mask these effects. We end with a discussion of how this can be accounted for when considering single cell transcriptomic data.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author3"><strong class="card-body-subgroup">Kumar Saurabh</strong></a>
		    	 
		    		<em>National Taiwan University</em>
		    	
		    	
		    		
		    			</br>"MATHEMATICAL MODELING OF ION CHANNEL FLOW WITHIN CONTINUUM FRAMEWORK"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Within the continuum framework, ion transport can be described using Poisson-Nernst-Planck (PNP) equations. Although accurate for dilute flows, PNP equations are not appropriate for modeling flows with high ion concentration or flows where non-ionic interactions are important. For ion channel flow, several extensions to continuum theory has been proposed. Effect of  finite size of ion can be modeled by including either Lennard-Jones potential [1, 2, 3] in the energetic formulation or Bikerman model [4, 5]. For effect of ion solvation, Born energy model can be included in the system [3, 4]. Additionally, to account for spatial variation of dielectric behavior of the aqueous medium on can resort to nonlocal electrostatics [5]. Numerically, the system is modeled using lattice Boltzmann method (LBM) in conjunction with immersed boundary method (IBM) to address the boundary conditions. Further, to reduce computational cost, the code has been parallelized on multiple GPUs using CUDA platform. These mathematical models have been successfully implemented for ion flow through SARS-CoV-1 and SARS-CoV-2 E protein ion channel, TRPV channel etc. In this study, we intend to explore the role and effect of these mathematical models on ion transport through a potassium channel.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author4"><strong class="card-body-subgroup">Nayana Wanasingha</strong></a>
		    	 
		    		<em>Department of Mathematical Sciences, University of Cincinnati</em>
		    	
		    	
		    		
		    			</br>"Molecular mechanisms regulating frequecy demultiplication of circadian rhythms in Neurospora Crassa"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Subharmonic entrainment or frequency demultiplication is a characteristic of circadian systems, which is the ability to entrain to cycles that are submultiples of external cycles. In this study, we used mathematical modeling and experiments to investigate potential mechanisms regulating frequency demultiplication under different temperature cycles in a model filamentous fugus, Neurospora crassa. Our results indicate that frequency demultiplication is a manifestation of the entrainment of circadian clock to external cycles and depends on the endogenous period and the strength and type of external cycles. Theoretical analysis reveals two necessary conditions to reproduce experimentally observed frequency demultiplication and frequency driven phenotypes: 1) temperature-modulated frq transcription and translation, and 2) a low level of cooperativity of transcriptional regulation of frq. In summary, we used mathematical modeling and experiments to uncover the architecture of circadian systems regulating frequency demultiplication, which broadens our fundamental understanding of entrainment of circadian rhythms.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT05.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT05-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT05.html#author1"><strong class="card-body-subgroup">Maalavika Pillai</strong></a>
		    	 
		    		<em>Indian Institute of Science, Bangalore</em>
		    	
		    	
		    		
		    			</br>"Systems-level analysis of phenotypic plasticity and heterogeneity in melanoma"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Phenotypic (i.e. non-genetic) heterogeneity in melanoma drives dedifferentiation, recalcitrance to targeted therapy and immunotherapy, and consequent tumor relapse and metastasis. Various markers or regulators associated with distinct phenotypes in melanoma have been identified, but, how does a network of interactions among these regulators give rise to multiple “attractor” states and phenotypic switching remains elusive. Here, we inferred a network of transcription factors (TFs) that act as master regulators for gene signatures of diverse cell-states in melanoma. Dynamical simulations of this network predicted how this network can settle into different “attractors” (TF expression patterns), suggesting that TF network dynamics drives the emergence of phenotypic heterogeneity. These simulations can recapitulate major phenotypes observed in melanoma and explain de-differentiation trajectory observed upon BRAF inhibition. Our systems-level modeling framework offers a platform to understand trajectories of phenotypic transitions in the landscape of a regulatory TF network and identify novel therapeutic strategies targeting melanoma plasticity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT05.html#author2"><strong class="card-body-subgroup">Jill Gallaher</strong></a>
		    	 
		    		<em>Moffitt Cancer Center</em>
		    	
		    	
		    		
		    			</br>"Using adaptive therapy to characterize collective and individual characteristics of metastases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Evolutionary-designed therapies, such as adaptive therapy, have been shown to be useful for late stage cancer to delay treatment resistance by exploiting competition between sensitive and resistant cells by alternating between drug applications and drug-free vacations. In addition, a single cycle of adaptive therapy could be used as a tool to probe tumor dynamics. In this work, we propose a framework for estimating individual and collective components of a metastatic system using tumor dynamics during adaptive therapy. We use a system of off-lattice agent-based models to represent individual metastatic lesions within independent domains, but subject to the same systemic therapy. We find the first cycle of adaptive therapy delineates several features of the metastatic system. Larger metastases have longer cycles, more drug resistance slows the cycles, and a faster cell turnover speeds up drug response time and slows the regrowth time. The number of metastases does not affect cycle times, as dynamics are dominated by the largest tumors rather than the aggregate. Changes in individual metastases gives insight on heterogeneity amongst metastases and can guide treatment. Generally, systems with more intertumor heterogeneity had better success with continuous therapy, while systems with more intratumor heterogeneity responded better to adaptive therapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT05.html#author3"><strong class="card-body-subgroup">Joshua Bull</strong></a>
		    	 
		    		<em>Wolfson Centre for Mathematical Biology, University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Novel spatial statistics describe phenotype transitions in an agent-based model of tumour associated macrophages"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumour associated macrophages adopt a range of phenotypes based on microenvironmental cues, with opposite ends of a spectrum of behaviours often summarised as “M1” (anti-tumour) and “M2” (pro-tumour). Transitions in macrophage phenotype play a role in cancer progression: for example, chemotactic gradients generated by macrophages of different phenotypes may be responsible for movement of tumour cells towards external vasculature and subsequent metastasis [1].We present an agent-based model based on [1] in which individual blood vessels, macrophages, tumour cells and stromal cells are resolved. Diffusible species in the tumour microenvironment determine macrophage phenotype, which we describe as a continuous variable. This variable governs phenotype specific macrophage behaviours such as phagocytosis and production of tumour cell chemoattractants. Our model reproduces patterns of macrophage localisation described in [1].Considering simulated macrophage locations as a point pattern, we develop novel spatial statistical techniques which account for points labelled with a continuous variable and hence identify how macrophage phenotype determines spatial localisation. This work suggests that spatial statistics accounting for real-valued labels could be used to better describe multiplexed imaging data, in which high or low expression of multiple markers can be identified from variations in stain intensity.[1] Arwert et al, 2018. doi:10.1016/j.celrep.2018.04.007.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Contributed Talk Session - CT05
										</br>
										Tuesday, June 15 at 10:30pm (PDT)</br>Wednesday, June 16 at 06:30am (BST)</br>Wednesday, June 16 02:30pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>
















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
