---
layout: group
author: esmb
day: "Tuesday (Wednesday)"
subgroup: "EVOP"
title: "Recent developments in phylogenetic network reconstruction and beyond"
organizers: "Guillaume Scholz (University of Leipzig, Germany), Katharina Huber (University of East Anglia, United Kingdom)"
description: "Phylogenetic is a burgeoning area at the interface between Mathematics (incl Computer Science and Probability Theory) and Molecular Biology concerned with developing mathematical methodology and algorithms to help understand molecular evolution. Although it has been around for some time resulting in numerous deep and beautiful mathematical results the vast amounts of data generated by current sequencing methods have given rise to some exciting new questions. These concern in particular the area of phylogenetic network reconstruction. Such a structure naturally generalises the notion of a phylogenetic tree by allowing for cycles to help accommodate reticulate evolutionary processes such as recombination which is of relevance for understanding virus evolution (e.g. Covid-19). The minisymposium will bring together researcher at various levels of the academic career spectrum to discuss recent developments in phylogenetic network reconstruction and beyond."
code: "MS11"
code2: "MS17"
author1: "Magnus Bordewich"
inst1: " (Durham University, United Kingdom)"
title1: "Diversity in phylogenetic networks"
abstract1: "Dating back to 1992, phylogenetic diversity (PD) is a prominent quantitative tool for measuring the biodiversity of a collection of species. This measure is based on the evolutionary distance among the species in the collection. Loosely speaking, if T is a phylogenetic tree whose leaf set X represents a set of species and whose edges have real-valued lengths (weights), then the PD score of a subset S of X is the sum of the weights of the edges of the minimal subtree of T connecting the species in S. In this talk we will discuss recent work on extending this concept from phylogenetic trees to phylogenetic networks and consider the computational complexity of the associated optimisation problems."
author2: "Simone Linz"
inst2: " (University of Auckland, New Zealand)"
title2: "Superfluous arcs in phylogenetic networks"
abstract2: "The last 15 years have seen a shift from the reconstruction of phylogenetic trees towards phylogenetic networks. The latter not only capture speciation events but also evolutionary processes such as hybridization and lateral gene transfer that cannot be explained by a single phylogenetic tree. Nevertheless, since the evolutionary history of a single gene or short DNA fragment is, in most cases, correctly described by a tree, the set of phylogenetic trees that are embedded in a network continue to be of recurring interest. For example, to score a phylogenetic network in a maximum parsimony or likelihood framework, one often scores each embedded tree instead of the network directly. In practice this often comes down to scoring a multiset of embedded trees whose size is exponential in the number of reticulations in the network. In this talk, we introduce the notion of a non-essential arc of a phylogenetic network N which is an arc whose deletion from N results in a phylogenetic network Nâ€™ whose set of embedded trees is equal to that of N.  We investigate the class of tree-child networks and characterize which arcs are non-essential. This characterization is based on a  family of directed graphs. Moreover, we show that identifying non-essential arcs in a tree-child network takes time that is polynomial in the number of leaves of the network."
author3: "Kristina Wicke"
inst3: " (The Ohio State University, United States of America)"
title3: "Linking phylogenetics and classical graph theory: edge-based phylogenetic networks and their relation to GSP graphs"
abstract3: "Recently, tree-based phylogenetic networks have attracted considerable attention in the literature. Roughly speaking, these networks can be constructed from a phylogenetic tree by inserting additional edges. However, in general, it is an NP-completeproblem to decide whether an unrooted phylogenetic network is tree-based or not. In this talk, I will introduce a class of unrooted networks, namely edge-based networks, that are necessarily tree-based and can be recognized in linear time. Surprisingly, the class of edge-based networks is closely related to a well-known family of graphs in classical graph theory, the class of generalized series-parallel (GSP) graphs, and I will explore this relationship in full detail."
author4: "Vincent Moulton"
inst4: " (University of East Anglia, United Kingdom)"
title4: "Reconstructibility of unrooted level-k phylogenetic networks from distances"
abstract4: "A phylogenetic network is a graph-theoretical tool that is used by biologists to represent the evolutionary history of a collection of species. One potential way of constructing such networks is via a distance-based approach, where one is asked to find a phylogenetic network that in some way represents a given distance matrix, which gives information on the evolutionary distances between present-day taxa. In this talk, we consider the following question. For which k are unrooted, edge-weighted level-k networks uniquely determined by their distance matrices? We consider this question for shortest distances as well as for the case that the multisets of all distances is given."
---
