---
layout: group
author: esmb
day: "Thursday (Friday)"
subgroup: "EVOP"
title: "Evolutionary Theory of Disease"
organizers: "Jesse Kreger (University of California, Irvine, United States), Natalia Komarova (University of California, Irvine, United States)"
description: "The use of fundamental principles of evolutionary biology can provide important insight into disease dynamics, both within and between hosts. For example, evolutionary mathematical models have been used to better understand disease progression, immune escape mechanisms/drug resistance, and optimal treatment regimens in many prominent diseases (such as virus infection and cancer).

Our mini-symposium focuses on the “Evolutionary Theory of Disease”, and in particular, on how evolutionary ideas in tandem with mathematical techniques and models can be used to understand disease dynamics as well as how to best combat the disease. In our mini-symposium, researchers will present their exciting and impactful recent research, including new mathematical and computational models, with a focus on understanding the evolutionary dynamics of the disease. Our list of speakers includes both accomplished/senior researchers as well as junior mathematicians at the postdoctoral and graduate student levels. Their research spans different diseases (from infectious diseases to cancer) and mathematical approaches (including differential equations, agent-based models, network models, etc), and should stimulate broad interest in the mathematical biology community. We look forward to an exciting exchange of ideas, as well as great networking opportunities for researchers at all career stages."
code: "MS19"
code2: "MS20"
author1: "Joceline Lega"
inst1: " (University of Arizona, United States)"
title1: "A novel take on outbreak dynamics"
abstract1: "During an outbreak, public health data typically consist of time series for the daily or weekly incidence (reported new cases) of the disease. This information is location-specific (e.g. at the level of a county, a state, or a country) and noisy. For each such time series, plotting incidence as a function of cumulative cases instead of time leads to a remarkable simplification: the data appear to fluctuate about a mean curve of universal shape. In this talk, I will illustrate the previous statement through examples of Influenza A and COVID-19 outbreaks, and describe recent work aimed at elucidating this behavior [1, 2]. In particular, exact results will be provided for the deterministic and stochastic SIR models. In addition, I will explain how this property can be combined with data assimilation to provide short-term forecasts of COVID-19 cases and deaths in the US [3]. This is joint work with Hannh Biegel, Bill Fries, Faryad Sahneh, and Joe Watkins. 

[1] J. Lega, Parameter estimation from ICC curves, Journal of Biological Dynamics 15, 195-212 (2021).
[2] F.D. Sahneh, W. Fries, J.C. Watkins, J. Lega, The COVID-19 Pandemic from the Eye of the Virus (2021); arxiv.org/abs/2103.12848
[3] H. Biegel & J. Lega, EpiCovDA: a mechanistic COVID-19 forecasting model with data assimilation (2021)."
author2: "Dylan H. Morris"
inst2: " (University of California, Los Angeles, United States)"
title2: "Evolving fast and slow: how asynchrony between virus diversity and antibody selection limits influenza virus evolution"
abstract2: "Seasonal influenza viruses create a persistent global disease burden by evolving to escape immunity induced by prior infections and vaccinations. New antigenic variants have a substantial selective advantage at the population level, but these variants are rarely selected within-host, even in previously immune individuals. Using a mathematical model, we show that the temporal asynchrony between within-host virus exponential growth and antibody-mediated selection could limit within-host antigenic evolution. If selection for new antigenic variants acts principally at the point of initial virus inoculation, where small virus populations encounter well-matched mucosal antibodies in previously-infected individuals, there can exist protection against reinfection that does not regularly produce observable new antigenic variants within individual infected hosts. Our results provide a theoretical explanation for how virus antigenic evolution can be highly selective at the global level but nearly neutral within-host, and providing a clear example of how evolution and ecology only make sense in light of one another.

Relevant reading: https://elifesciences.org/articles/62105"
author3: "Jesse Kreger"
inst3: " (University of California, Irvine, United States)"
title3: "The role of migration in mutant evolution in fragmented populations"
abstract3: "Complex population structures are an important determinant of the evolutionary dynamics of mutants. In fragmented populations, this has been studied using metapopulation models, which have been of great interest for questions related to ecology and population conservation. However, such models also have high relevance in a biomedical context – such as deme population structures that apply to evolution in hematopoietic systems. In this talk, we investigate the effects of population fragmentation on mutant cell dynamics using stochastic metapopulation modeling in conjunction with in vitro laboratory experiments. In the case of neutral mutations, we find that migration makes the demes look homogeneous to each other, resulting in a one-humped (unimodal) distribution, which matches well with experimental simulations. For disadvantageous mutations, we find that migration not only similarly impacts the distribution of mutant cells, but it can also change the expected frequency of mutants at stationary state compared to the selection-mutation balance. This could play an important role in disease progression."
author4: "Ali Mahdipour-Shirayeh"
inst4: " (University of Toronto, Canada)"
title4: "Clonal evolution and Intra-tumoral heterogeneity in cancer: A single-cell viewpoint"
abstract4: "Despite intense therapeutic advances, therapy failures in diverse cancers may suggest the existence of intra-tumoral diversity and presumably rare subclones of minimal residual disease that are persistent to current therapies. Although there is no comprehensive technique to determine such subclones and to identify evolution of the disease, single-cell data can shed light on intra-cellular heterogeneity and clonal evolution of individual cells in alternative cell contexts. Utilizing single-cell data, the potential genetic pathways can be detected across diverse pheno/geno-types within a heterogeneous population of cells. Moreover, in many cancers, particularly in Multiple Myeloma, the most reliable clonal features are copy number variations (CNVs) which can be best inferred from single-cell DNA/RNA study. To address all these challenges, we developed an extensive pipeline, referred to as sciCNV, which covers a range of analysis from a novel normalization to inferring CNV from single-cell data. During this talk, we first introduce some fundamental tools which are commonly used in studying single cells and then will introduce the sciCNV pipeline to be implemented to segregate tumor cells from normal individuals and to understand the genetic background of the disease. This technique may offer an efficient way to clone distinct CNV-compartments and to construct a phylogeny of subclonal structure and pathogenesis of the disease. Such analysis can reflect evolutionary dynamics and clonal dependencies of cancer in time/space frame. Our approach is general and can be applied to any transcription data and may tend to a better understanding of histological/pathogenesis of diverse cancers and their associated therapeutic strategies."
---
