<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Data Driven Modeling in Biology Subgroup (DDMB) | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Data Driven Modeling in Biology Subgroup (DDMB) | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Data Driven Modeling in Biology Subgroup (DDMB)" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/DDMB/" />
<meta property="og:url" content="http://localhost:4000/DDMB/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/DDMB/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Data Driven Modeling in Biology Subgroup (DDMB)","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Data Driven Modeling in Biology Subgroup (DDMB)">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Data Driven Modeling in Biology Subgroup (DDMB)" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Data Driven Modeling in Biology Subgroup (DDMB)" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
	    <div class="row">
	        <div class="col-md-12  text-center">
<!-- 		        <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
		        <h1 class="mb-3">Data Driven Modeling in Biology Subgroup (DDMB)</h1>
		        
<!-- 		        <div class="cleanslate w24tz-current-time w24tz-small" style="display: inline-block !important; visibility: hidden !important; min-width:200px !important; min-height:100px !important;"><p><a href="//24timezones.com/Tampa/time" style="text-decoration: none" class="clock24" id="tz24-1596739689-c1410-eyJob3VydHlwZSI6MTIsInNob3dkYXRlIjoiMSIsInNob3dzZWNvbmRzIjoiMCIsImNvbnRhaW5lcl9pZCI6ImNsb2NrX2Jsb2NrX2NiNWYyYzUwNjllNmUzMSIsInR5cGUiOiJkYiIsImxhbmciOiJlbiJ9" title="Tampa local time" target="_blank">US Eastern Time Zone (EDT)</a></p><div id="clock_block_cb5f2c5069e6e31"></div></div>
<script type="text/javascript" src="//w.24timezones.com/l.js" async></script>
	        	</br> -->
		        
		        
	        </div>
	    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			<h6>Jump to:</h6>
<ul class="pagination text-left">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#"><span class="text-DDMB">Minisymposia</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#CT"><span class="text-DDMB">Contributed Talks</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#Posters"><span class="text-DDMB">Posters</span></a></li>
</ul>

			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS1">Sub-group minisymposia</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS01.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS01-DDMB: </small></br>Stochastic models of cancer: An update of theory and data</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Marek Kimmel (Rice University, United States), Simon Tavare (Columbia University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS01.html#author1"><strong class="card-body-subgroup">Tibor Antal</strong></a>
		    	 
		    		<em> (School of Mathematics, Edinburgh University, Scotland, UK, UK)</em>
		    	
		    	
		    		
		    			</br>"Models of Tumor Progression"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			I'll review recent results on timing in evolving cell populations. The stochasticity is taken into account by the use of branching processes, and the general question is the time it takes to produce cells with some specific properties. As an application the (possible) relapse time after cancer treatment will be discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS01.html#author2"><strong class="card-body-subgroup">Robert Beckman</strong></a>
		    	 
		    		<em> (Lombardi Comprehensive Cancer Center and Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC, USA, USA)</em>
		    	
		    	
		    		
		    			</br>"Recent Advances in Genetic Instability and Dynamic Precision Medicine of Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer evolution has similarities with species evolution but also important differences. This talk will outline some of these differences, including results from the deepest, most accurate sequencing of a solid tumor done to date. By probing the evolution of extremely rare mutations, we learn that the amount of intratumoral heterogeneity is far greater than previously assumed, and that subclones harboring pre-existing resistance to any single therapy will be universally present in any clinically diagnosable tumor. Moreover, rare mutations evolve neutrally, and surprisingly the popular “infinite sites assumption” suggesting that a new mutation will appear in only one cell at once, does not apply at this stage of tumor growth.
Cancer treatment has also evolved, from an empirical science of killing dividing cells, to the current era of “precision medicine”, targeted to molecular features of individual cancers. However, current precision medicine views a single individual’s cancer as largely uniform and static. Moreover, from a strategic perspective, it thinks primarily of the current therapeutic maneuver. In contrast, dynamic precision medicine (DPM) plans ahead, considers intratumoral heterogeneity and evolutionary dynamics, and probabilistically weighs the future benefits of preventing resistance against the benefits of immediate cytoreduction. Simulations indicate it has the potential to double median survival broadly across cancers. This talk will provide background about DPM, highlighting differences from other approaches to evolutionarily directed therapy. It will describe recent advances, including the role of genetically unstable “hypermutator subclones”, the possibility of clinical translation in presurgical “neoadjuvant “ settings, and considerations in clinical trial design.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS01.html#author3"><strong class="card-body-subgroup">Alexandre Bouchard-Côté</strong></a>
		    	 
		    		<em> (Statistics, University of British Columbia, Vancouver, BC, Canada, Canada)</em>
		    	
		    	
		    		
		    			</br>"Inferring fitness of cancer subpopulations from time series --- Bayesian methods for the Wright-Fisher diffusion with selection"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			From timeseries of patient-derived xenograft data, we are interested in inferring fitness parameters for sub-populations of cancer cells measured using single cell sequencing. I will describe a statistical model for Bayesian inference of these fitness parameters. The model is based on a stochastic differential equation, the Wright-Fisher diffusion with fitness, the parameters of which are treated as random. We extend advanced MCMC methodologies such as PMCMC to perform Bayesian inference at scale, the posterior distribution being defined over a high-dimensional space and informed by a large dataset.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS01.html#author4"><strong class="card-body-subgroup">Ivana Bozic</strong></a>
		    	 
		    		<em> (Applied Mathematics, University of Washington, Seattle, WA, USA, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical model of colorectal cancer initiation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer evolution cannot be observed directly in patients, and new methodologies are needed for obtaining a quantitative understanding of this obscure process. We developed and analyzed a stochastic model of malignant transformation in the colon that precisely quantifies the process of colorectal carcinogenesis in patients through loss of tumor suppressors APC and TP53 and gain of the KRAS oncogene. Our study employs experimentally measured mutation rates in the colon and growth advantages provided by driver mutations. We calculate the probability of a colorectal malignancy, the sizes of premalignant lesions, and the order of acquisition of driver mutations during colorectal tumor evolution. We demonstrate that the order of driver events in colorectal cancer is determined primarily by the fitness effects that they provide, rather than their mutation rates.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS02.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS02-DDMB: </small></br>Stochastic models of cancer: An update of theory and data</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Marek Kimmel (Rice University, United States), Simon Tavare (Columbia University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS01-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS02.html#author1"><strong class="card-body-subgroup">Katharina Jahn</strong></a>
		    	 
		    		<em> (Computational Biology Group, ETH Zurich, Zurich, Switzerland, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Dissecting Clonal Diversity Through High-Throughput Single-Cell Genomics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Clonal heterogeneity allows tumours to adapt and survive under the selective pressure of treatment, leading to clinical resistance and relapse. An accurate dissection of the clonal architecture and the underlying mutational history is therefore of clinical importance and may help to design more effective treatment plans. Present studies on clonal diversity are primarily based on sequencing data obtained from bulk tumour tissue which systematically underestimate a tumour's mutational heterogeneity. However, through recent technological advances, high-throughput single-cell genomics has become a feasible alternative that allows to study clonal diversity at an unprecedented resolution.
In this talk, I will present a Bayesian inference scheme for tumour mutation histories based on single-cell sequencing data and the insights we obtained from analysing longitudinal bone marrow samples of 123 AML patients. Using a microfluidics-based single-cell DNA sequencing platform, we genotyped over 700,000 cells for a panel of genes recurrently mutated in AML. We observed patterns of mutual exclusivity, mutational co-occurrence, as well as instances of convergent evolution. Moreover, the longitudinal nature of the data revealed patterns of clonal dynamics in response to targeted AML therapy which correlated with clinical resistance and relapse.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS02.html#author2"><strong class="card-body-subgroup">Ximo Pechuan Jorge</strong></a>
		    	 
		    		<em> (Institute of Cancer Research, London, UK, UK)</em>
		    	
		    	
		    		
		    			</br>"A Simple Computational Model to Infer Selective Coefficients in Barcode Evolution Experiments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The advent of single cell sequencing technologies has propelled the usage of lineage barcoding to characterize the dynamics of heterogeneous cell populations. Following the population dynamics of tumor cells is of paramount importance to determine the details of their evolutionary process which, in turn, can influence therapeutic outcome. To characterize the evolutionary dynamics of barcoded organoids during the course of two years of serial passage 	extit{in vitro} after a genetic perturbation, we constructed a simple stochastic model accounting for drift and competition between lineages. We used sequential Monte Carlo to fit the model to the experimental data obtaining initial growth rate estimates for each lineage. Some of the samples exhibited evidence of mutation acquisition and thus required a model accounting for mutation accumulation. Our model explains the patterns observe in the data and shows the value of constructing simple interpretable models in the initial stages of data analysis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS02.html#author3"><strong class="card-body-subgroup">Luis Zapata Ortiz</strong></a>
		    	 
		    		<em> (Institute of Cancer Research, London, UK, UK)</em>
		    	
		    	
		    		
		    			</br>"Evolutionary dynamics of cancer immunoediting predicts response to immunotherapy."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer Immunoediting is an evolutionary force that shapes the genome of healthy and malignant cells in the human body. However, quantifying immunogenicity in the cancer genome and how the tumour-immune coevolutionary dynamics impact patient outcomes remain unexplored. Here, we developed a stochastic branching process coupled with an agent-based model to simulate the accumulation of mutations during immunoediting. We show how a metric of selection, the ratio of nonsynonymous to synonymous mutations in the immunopeptidome (immune dN/dS) quantifies tumor immunogenicity and differentiates between outcomes of immunoediting. We provide a theoretical explanation for the lack of signals of immune selection reported previously and analysed 8,543 primary tumors from TCGA and 376 metastatic tumors from immunotherapy trials. We validated immune dN/dS as a measure of CD8-T cell mediated selection in tumours that have not undergone immune escape. Moreover, In a cohort of 368 metastatic patients treated with checkpoint inhibitors, we observed that lesions of non-responders had strong immune selection (dN/dS < 1, negative), whereas responders did not show immune selection (dN/dS ~ 1, neutral), and instead harboured a higher proportion of genetic escape mechanisms. Our findings highlight the challenges of using dN/dS to estimate selection, suggest that the extent of immunogenicity can be read from the tumor genome, and that the evolutionary consequences of immunoediting determine immunotherapy efficacy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS02.html#author4"><strong class="card-body-subgroup">Jan Poleszczuk</strong></a>
		    	 
		    		<em> (Nalecz Institute of Biocybernetics and Biomedical Engineering, Polish Academy of Sciences, Warsaw, Poland, Poland)</em>
		    	
		    	
		    		
		    			</br>"Microsimulation-based optimization of colorectal cancer screening strategies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Colorectal cancer (CRC) is a substantial public hearth burden and is in the top three cancers with respect to incidence and mortality in US and many other industrialized countries. CRC screening tests based on the endoscopic visualization of the colon have proven effective in reducing mortality, both by allowing CRC at earlier stages and by CRC prevention since adenomatous precursors of CRC can be removed during endoscopy. However, the starting age and time intervals of screening colonoscopies for optimal protection against CRC are unknown. We used microsimulation to systematically optimize screening colonoscopy schedules.
We advanced our established open-source microsimulation model CMOST to simulate the effects of colonoscopy screening on the natural history and medical costs of CRC. In CMOST, carcinoma develops via early and advanced adenoma precursors. CMOST accounts for the gender- and age-dependent risks for adenoma development, the presence of multiple adenomas, as well as their locations within the colon. CMOST microsimulation tracks the history of a general population from birth until death for a maximum age of 100 years. Adenoma initiation, progression to advanced adenoma and cancer, cancer progression, screening and surveillance are all modeled in time increments of 3 months and are stochastically driven. We used CMOST to optimize colonoscopy schedules with one, two, three and four screening colonoscopies between 20 and 90 years of age. For each scenario, we calculated life years gained, incidence and mortality reduction, and cost-effectiveness.
A single screening colonoscopy is most effective in reducing life years lost from CRC when performed at 55 years of age. Two, three and four screening colonoscopy schedules are optimal at earlier ages. For maximum reduction of incidence and mortality, screening colonoscopies need to be scheduled later in life compared to optimal age for life years lost. The optima are influenced by adenoma detection rates, individual CRC risk, and adherence to screening, with lower values for these parameters favoring a later starting age of screening. Incremental cost-effectiveness remained below 100’000 discounted US dollars per discounted life year gained except for an optimal four-colonoscopy schedule, which was not cost-effective. In a personalized approach, optimal screening would start earlier for high-risk patients and later for low-risk individuals.
Our results support screening recommendations involving an early starting age of 45 years. Our optimized screening strategies are cost-effective and save more life years than currently
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS03.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS03-DDMB: </small></br>Mathematics of Cryopreservation: from tissue preparation to freezing and ice formation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Robyn Shuttleworth (University of Saskatchewan, Canada), James Benson (University of Saskatchewan, Canada)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS03.html#author1"><strong class="card-body-subgroup">Adam Higgins</strong></a>
		    	 
		    		<em> (Oregon State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Rational design of less toxic cryoprotectant solutions for cryopreservation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cryoprotectants (CPAs) are essential components of vitrification mixtures because they promote formation of a non-crystalline glassy state. However, CPAs can be toxic, and it remains a challenge to identify minimally toxic CPA mixtures for vitrification. This difficulty stems from two main issues. First, there are many different CPA types that can be combined in an infinite number of ways to create vitrification mixtures. It is therefore impractical to empirically determine the best vitrification mixture from among this infinite set. Second, the mechanisms of CPA toxicity are not well understood, making it difficult to identify promising mixtures using conceptual reasoning. To address these issues, we have developed a mathematical model of CPA toxicity that accounts for specific and nonspecific toxicity mechanisms, as well as formation of complexes between CPA pairs. We fit this model to experimental data for cultured endothelial cells exposed to five common CPAs [i.e., glycerol (Gly), dimethyl sulfoxide (DMSO), ethylene glycol (EG), propylene glycol (PG) and formamide (FA)], as well as their binary mixtures. The resulting best-fit model parameters were examined using Sloppy Model analysis to provide clues about the toxicity mechanisms. The results suggest that FA and Gly have the highest specific toxicity, PG exerts the most nonspecific toxicity, and that complexes between Gly-FA, DMSO-FA and Gly-EG affect the toxicity of mixtures containing these CPAs. To examine the predictive ability of the model, we predicted the toxicity of ternary CPA solutions, which resulted in reasonable agreement with experimental data. We then combined the toxicity model with a previously published model of glass formation in CPA mixtures to predict promising compositions for vitrification. The combined model predicts that the least toxic CPA cocktail that will result in formation of a glass is a mixture of Gly, FA and DMSO at concentrations of 7.5, 2.1 and 1.4 molal, respectively.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS03.html#author2"><strong class="card-body-subgroup">Ross Warner</strong></a>
		    	 
		    		<em> (Oregon State University, United States)</em>
		    	
		    	
		    		
		    			</br>"A general strategy for modeling the distribution of cryoprotectants in tissues"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The ability to successfully cryopreserve any biological specimen would undoubtedly change the face of modern medicine and scientific research. Single cell cryopreservation is a difficult problem by itself, but cryopreservation of complex specimens—mainly tissues and organs—is arguably an order of magnitude more difficult and is an active area of research. Vitrification is a promising avenue for successful complex specimen cryopreservation, but toxicity remains a major hurdle to overcome, as vitrification requires a high concentration of cryoprotectants (CPAs) to completely suppress ice formation. In the past, our group has leveraged mathematical modeling to minimize CPA toxicity for single cells. To do so, we developed a toxicity cost function and used mathematical optimization to minimize its value, which resulted in the prediction of a novel vitrification protocol that was experimentally verified to be less toxic than conventional methods. To extend this promising approach to tissues, an appropriate mass transfer model is needed. Fick’s law is commonly used, but it is limited due to its dilute assumption, as well as not accounting for tissue-specific phenomena such as fixed electrical charges, tissue size changes, and the coupling between cell membrane and extracellular mass transfer. In this work, we propose a general modeling paradigm for mass transfer in tissues. To accomplish this, we augmented an acellular mixture theory model in the literature for articular cartilage by incorporating cellular effects. With this augmentation, we show that the model can not only predict changes in CPA concentration and tissue size for the low cell density, rigid tissue of articular cartilage but also for the high cell density, elastic tissue of pancreatic islets. As such, this modeling paradigm is a promising general tissue model that can be used to further our mathematical optimization approach to cryopreservation and to better understand observations during tissue cryopreservation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS03.html#author3"><strong class="card-body-subgroup">Fatemeh Amiri</strong></a>
		    	 
		    		<em> (University of Saskatchewan, Canada)</em>
		    	
		    	
		    		
		    			</br>"Agent based tissue modeling of ice propagation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We model intracellular ice formation (IIF) in large multicellular tissues using Monte Carlo and agent-based modeling techniques. The previous implementa- tions have not allowed for within-tissue cell phenotype (i.e. parameter) het- erogeneity, nor have they coupled the models with key substrate diffusion and reaction equations. Therefore, to account for these critical differences and to understand IIF in large tissues, we have developed and validated a Monte Carlo method. In this model the tissue is described by a regular lattice in which each lattice site represents a cell, and intercellular ice propagation is allowed only between nearest neighbors. In our approach, each cell in the tissue is considered as an agent using the open source software PhysiCell, a multicellular system simulator which is designed to model tissues involving many interacting cells in multi-substrate 3D-microenvironments. We have validated the Monte Carlo method against theoretical Markov chain model for linear two-cell, four-cell and 2 × 2-cell constructs. Unlike the Markov model that involves exponential computational complexity associated with the tissue size, the Monte Carlo model has been successfully applied for large tissues with high numbers of cells. We also investigate the effects of tissue size on IIF in large tissues constructs and model IIF in mouse embryos.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS03.html#author4"><strong class="card-body-subgroup">Janet A. W. Elliott</strong></a>
		    	 
		    		<em> (University of Alberta, Canada)</em>
		    	
		    	
		    		
		    			</br>"Thermodynamics of Cell and Tissue Cryopreservation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cryobiology is the study of the effects of low temperature on biological systems with a major application being cryopreservation—the use of extremely low temperatures for the effectively indefinite storage of cells and tissues for later use. Cryoprotectants are used that modify the amount and location of ice formation during cryopreservation procedures. The ice–cryoprotectant-solution phase diagram and the osmotic and cryoprotectant transport across cell membranes and across tissues during cryoprotectant addition/removal and cooling/warming play crucial roles in whether or not cells survive. Thermodynamics is a broadly applicable subject whereby equations describing relationships among properties are derived from a few core postulates using multivariable calculus. Over more than 20 years our group has been developing equilibrium thermodynamic and nonequilibrium thermodynamic (transport) equations to describe cryobiological processes and gain insight to optimize cryopreservation protocols for a variety of cells and tissues. This talk will briefly introduce various areas of our prior and current work in cryobiological thermodynamics.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS06.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS06-DDMB: </small></br>Advances in deterministic models of biochemical interaction networks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Elisenda Feliu (University of Copenhagen, Denmark), Casian Pantea (West Virginia University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS06.html#author1"><strong class="card-body-subgroup">Balazs Boros</strong></a>
		    	 
		    		<em> (University of Vienna, Austria)</em>
		    	
		    	
		    		
		    			</br>"Oscillations in deficiency-one mass-action systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Whereas the positive equilibrium of a mass-action system with deficiency zero is always globally stable, for deficiency-one networks there are many different scenarios, mainly involving oscillatory behaviour. We present examples with centers or multiple limit cycles.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS06.html#author2"><strong class="card-body-subgroup">Beatriz Pascual Escudero</strong></a>
		    	 
		    		<em> (University of Copenhagen, Denmark)</em>
		    	
		    	
		    		
		    			</br>"Detecting concentration robustness in Reaction Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A biological system has absolute concentration robustness (ACR) for some species if the concentration of this species is identical at any possible equilibrium that the network admits. In particular, this concentration must be independent of the initial conditions. While some classes of networks with ACR have been described, as well as some techniques to check ACR for a given network, finding networks with this property is a difficult task in general.
The connection of this global version of robustness with other local notions leads to a practical test on necessary conditions for ACR, by means of algebraic-geometric techniques. This test allows to analyze networks in the search for the possibility of ACR or local ACR for some values of the reaction rates, or discard it for all values. This is based on joint work with E. Feliu.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS06.html#author3"><strong class="card-body-subgroup">Alan Rendall</strong></a>
		    	 
		    		<em> (Johannes Gutenberg University, Mainz, Germany)</em>
		    	
		    	
		    		
		    			</br>"Global convergence to steady states in a model for the in-host dynamics of hepatitis C"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider a model for the concentration of hepatitis C virus particles in a host which includes a simple description of the virus replication. This model has two virus-free steady states and two corresponding basic reproduction numbers. It has at most three positive steady states. Although it is not known whether there can be more than one steady state we prove that for certain ranges of the parameters every solution converges to a steady state. This is accomplished by applying a method of Li and Muldowney which uses the Lozinskii measure corresponding to a certain norm. An estimate for this Lozinskii measure of the second additive compound of the Jacobian matrix is the key condition which is required. The central idea of the method is to exclude all other kinds of asymptotic behaviour, such as convergence to a periodic solution.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS06.html#author4"><strong class="card-body-subgroup">Murad Banaji</strong></a>
		    	 
		    		<em> (Middlesex University London, UK)</em>
		    	
		    	
		    		
		    			</br>"Building Reaction Networks with Prescribed Properties"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In general, the problem of identifying reaction networks with some prescribed dynamical property is challenging. As an example of a dynamical property, let's consider stable oscillation. The question then becomes: does a given network allow stable oscillation for some choice of parameters (e.g., rate constants if the reaction network has mass action kinetics)? As networks grow in size, this question becomes harder and harder to check numerically. One way of making progress is via theorems which tell us how, given an oscillatory network, we can build a larger oscillatory network with more species or reactions. I'll give an overview of such theorems, focussing mainly on oscillation.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS07.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS07-DDMB: </small></br>Advances in deterministic models of biochemical interaction networks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Elisenda Feliu (University of Copenhagen, Denmark), Casian Pantea (West Virginia University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS06-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS07.html#author1"><strong class="card-body-subgroup">Anne Shiu</strong></a>
		    	 
		    		<em> (Texas A&M University, USA)</em>
		    	
		    	
		    		
		    			</br>"Absolute concentration robustness in networks with many conservation laws"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The concept of absolute concentration robustness (ACR) was introduced by Shinar and Feinberg in their investigations into how biochemical systems maintain their function despite changes in the environment.  A reaction system exhibits ACR in some species if the positive steady-state value of that species does not depend on initial conditions. Mathematically, this means that the positive part of the variety of the steady-state ideal lies entirely in a hyperplane of the form x_i=c, for some c>0. Deciding whether a given reaction system -- or those arising from some reaction network -- exhibits ACR is difficult in general, but this talk describes how for many simple networks, assessing ACR is straightforward. Indeed, our criteria for ACR can be performed by simply inspecting a network or its standard embedding into Euclidean space. Our main results pertain to networks with many conservation laws, so that all reactions are parallel to one other. Such 'one-dimensional' networks include those networks having only one species. We also consider networks with only two reactions, and show that ACR is characterized by a well-known criterion of Shinar and Feinberg. Finally, up to some natural ACR-preserving operations -- relabeling species, lengthening a reaction, and so on -- only three families of networks with two reactions and two species have ACR.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS07.html#author2"><strong class="card-body-subgroup">Stefan Mueller</strong></a>
		    	 
		    		<em> (University of Vienna, Austria)</em>
		    	
		    	
		    		
		    			</br>"Monomial parametrizations of positive equilibria"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider positive steady states of chemical reaction networks with (generalized) mass-action kinetics that allow a monomial parametrization. The latter is often a prerequisite in applications where one studies phenomena such as multistationarity and absolute concentration robustness.
In particular, we review results on complex-balanced equilibria (special equilibria given by binomial equations) and toric steady states (where all steady states are binomial). For example, a recent result states that a network with mass-action kinetics has toric steady states if it is dynamically equivalent to a network with generalized mass-action kinetics that has zero effective and kinetic-order deficiencies and hence complex-balanced (and no other) equilibria.
Finally, we discuss steps towards a characterization of networks with monomial parametrizations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS07.html#author3"><strong class="card-body-subgroup">Badal Joshi</strong></a>
		    	 
		    		<em> (California State University San Marcos, USA)</em>
		    	
		    	
		    		
		    			</br>"Preserving Robust Output despite highly variable reactant supplies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A cell/biochemical network must produce a consistently robust, easily readable output when interacting with its environment. However, the internal conditions of the cell and the available supplies of reactants are highly variable. To overcome this, the biochemical network must have architecture which is capable of producing the same output despite variations in reactant supplies, a property we will refer to as output robustness. As a possible means of achieving a robust system output, Shinar and Feinberg suggested the property of ACR (absolute concentration robustness), which requires that all steady states be in a hyperplane parallel to a coordinate hyperplane. However, ACR is neither necessary nor sufficient for output robustness, a fact that can be noticed in simple biochemical systems. To develop a stronger connection with output robustness, we define dynamic ACR, a property related to dynamics, rather than only to equilibrium behavior, and one that ensures convergence to a robust value. We illustrate the definition, and certain natural sub-types of dynamic ACR, with a rich body of examples of reaction networks. Towards the end, we will give a brief description of certain minimal motifs of dynamic ACR networks.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS07.html#author4"><strong class="card-body-subgroup">Jiaxin Jin</strong></a>
		    	 
		    		<em> (University of Wisconsin, Madison, USA)</em>
		    	
		    	
		    		
		    			</br>"Uniqueness of weakly reversible and deficiency zero realization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Weakly reversible, deficiency zero mass-action systems, being complex-balanced for any choice of rate constants, are remarkably stable. Here we show that if a dynamical system is generated by a weakly reversible network that has deficiency equal to zero, then this network must be unique. Moreover, we show that both weak reversibility and deficiency zero are necessary for uniqueness. We also describe an algorithm that can determine whether or not a system of differential equations can admit a weakly reversible, deficiency zero realization.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS08.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS08-DDMB: </small></br>Machine Learning and Data Science Approaches in Mathematical Biology: Recent Advances and Emerging Topics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Paul Atzberger (University of California Santa Barbara, USA), Smita Krishnaswamy (Yale University, USA), Kevin Lin (University of Arizona, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS09-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS08.html#author1"><strong class="card-body-subgroup">Smita Krishnaswamy</strong></a>
		    	 
		    		<em> (Yale University, USA)</em>
		    	
		    	
		    		
		    			</br>"Geometric and Topological Approaches to Representation Learning in Biomedical Data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			High-throughput, high-dimensional data has become ubiquitous in the biomedical, health and social sciences as a result of breakthroughs in measurement technologies and data collection. While these large datasets containing millions of observations of cells, peoples, or brain voxels  hold great potential for understanding generative state space of the data, as well as drivers of differentiation, disease and progression, they also pose new challenges in terms of noise, missing data, measurement artifacts, and the so-called “curse of dimensionality.” In this talk, I will cover data geometric and topological approaches to understanding the shape and structure of the data.  First, we show how diffusion geometry and deep learning can be  used to obtain useful representations of the data that enable denoising (MAGIC), dimensionality reduction (PHATE), and factor analysis (Archetypal Analysis Network) of the data.  Next we will show how to learn dynamics from static snapshot data by using a manifold-regularized neural ODE-based optimal transport (TrajectoryNet). Finally, we cover a novel approach to combine diffusion geometry with topology to extract multi-granular features from the data (Diffusion Condensation and Multiscale PHATE) to assist in differential and predictive analysis. On the flip side, we also create a manifold geometry from topological descriptors, and show its applications to neuroscience. Together, we will show a complete framework for exploratory and unsupervised analysis of big biomedical data.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS08.html#author2"><strong class="card-body-subgroup">Sui Tang</strong></a>
		    	 
		    		<em> (UCSB, United States)</em>
		    	
		    	
		    		
		    			</br>"Data-driven discovery of interacting particle system using Gaussian processes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Interacting particle or agent systems are widely used to model complicated collective motions of animal groups in biological science, such as flocking of birds, milling of fish, and swarming of prey. A fundamental goal is to understand the link between individual interaction rules and collective behaviors.  We consider  second-order interacting agent systems and study an inverse problem: given observed data,  can we discover the interaction rule?
For the interactions that only depends on pairwise distance, we propose a learning approach based on Gaussian processes that can simultaneously infer the interaction kernel without assuming a parametric form and learn other unknown parameters in the governing equations. The numerical results on prototype systems, including Cuker-Smale dynamics and fish milling dynamics, show that our approach produced faithful estimators from scarce and noisy trajectory data and made accurate predictions of collective behaviors. This talk is based on the joint work with Jinchao Feng.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS08.html#author3"><strong class="card-body-subgroup">Rose Yu</strong></a>
		    	 
		    		<em> (University of California San Diego, USA)</em>
		    	
		    	
		    		
		    			</br>"Physics-Guided Deep Learning for Forecasting COVID-19"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS08.html#author4"><strong class="card-body-subgroup">Alan Aspuru-Guzik</strong></a>
		    	 
		    		<em> (University of Toronto, USA)</em>
		    	
		    	
		    		
		    			</br>"Artificial Intelligence and Self-Driving Laboratories for Molecular Discovery"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS09.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS09-DDMB: </small></br>Machine Learning and Data Science Approaches in Mathematical Biology: Recent Advances and Emerging Topics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Paul Atzberger (University of California Santa Barbara, USA), Smita Krishnaswamy (Yale University, USA), Kevin Lin (University of Arizona, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS08-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS09.html#author1"><strong class="card-body-subgroup">Zhuo-Cheng Xiao</strong></a>
		    	 
		    		<em> (Courant Institute, NYU)</em>
		    	
		    	
		    		
		    			</br>"A data-informed mean-field approach to mapping cortical landscapes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cortical circuits are characterized by a high degree of structural and dynamical complexity, and this biological reality is reflected in the large number of parameters in even highly idealized cortical models.  A fundamental task of computational neuroscience is to understand how these parameters govern neuronal network dynamics.  While some neuronal parameters can be measured in vivo, many remain poorly constrained due to limitations of available experimental techniques.  Computational models can address this problem by relating difficult-to-measure parameters to observable quantities, but to do so one must overcome two challenges: (1) the computational expense of mapping a high dimensional parameter space, and (2) extracting biological insights from such a map.  In this study, we address these challenges in the following ways: First, we propose a data-informed, parsimonious mean-field algorithm that efficiently predicts spontaneous cortical activity, thereby speeding up the mapping of parameter landscapes.  Second, we show that lateral inhibition provides a basis for conceptualizing cortical parameter space, enabling us to begin to make sense of its geometric structure.  We illustrate our approach on a biologically realistic model of the Macaque primary visual cortex.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS09.html#author2"><strong class="card-body-subgroup">Andrea Arnold</strong></a>
		    	 
		    		<em> (Worcester Polytechnic Institute, USA)</em>
		    	
		    	
		    		
		    			</br>"Data Assimilation for Time-Varying Parameter Estimation in Biological Systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Estimating and quantifying uncertainty in system parameters remains a big challenge in many biological applications.  In particular, such problems may involve parameters that are known to vary with time but have unknown dynamics and/or cannot be measured.  This talk will address the use of data assimilation in novel approaches to time-varying parameter estimation, with emphasis on how uncertainty in the parameter estimates affects the corresponding model predictions.  Results will be demonstrated on several biological examples, including systems from computational neuroscience.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS09.html#author3"><strong class="card-body-subgroup">John Fricks</strong></a>
		    	 
		    		<em> (Arizona State University, USA)</em>
		    	
		    	
		    		
		    			</br>"A Bayesian Analysis of 2-D Motor-Cargo Complex Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Molecular motors, such as kinesin and dynein, move along microtubules in cells while the tails of the motors are connected to cargos.  The cargos can be tracked in fluorescence or dark field experiments yielding a stack of images.  Processing allows for the localization of the cargos yielding a two-dimensional time series; typically, further processing projects the data on to one-dimension along the direction of the microtubule.  However, curvature or misidentification of the microtubule may be relevant, but is generally not considered. In this talk, we will propose an analysis of the original two-dimensional time series, which can also extract additional information on the dynamics of these motor-cargo complexes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS09.html#author4"><strong class="card-body-subgroup">Mengyang Gu</strong></a>
		    	 
		    		<em> (University of California, Santa Barbara, USA)</em>
		    	
		    	
		    		
		    			</br>"Uncertainty quantification and estimation in differential dynamic microscopy for biomaterials characterization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Differential dynamic microscopy (DDM) is a form of video image analysis that combines the sensitivity of scattering and the direct visualization benefits of microscopy. DDM is broadly useful in determining dynamical properties including the intermediate scattering function for many spatiotemporally correlated systems. Despite its straightforward analysis, DDM has not been fully adopted as a routine characterization tool, largely due to computational cost and lack of algorithmic robustness. We present a comprehensive statistical framework that aims at quantifying error, reducing the computational order and enhancing the robustness of DDM analysis. We quantify the error, and propagate an independent noise term to derive a closed-form expression of the expected value and variance of the observed image structure function. Significantly, we propose an unbiased estimator of the mean of the noise in the observed image structure function, which can be determined experimentally and significantly improves the accuracy of applications of DDM. Furthermore, through use of Gaussian Process Regression (GPR), we find that predictive samples of the image structure function require only around 1% of the Fourier Transforms of the observed quantities. This vastly reduces computational cost, while preserving information of the quantities of interest, such as quantiles of the image scattering function, for subsequent analysis. The approach, which we call DDM with Uncertainty Quantification (DDM-UQ), is validated using both simulations and experiments with respect to accuracy and computational efficiency, as compared with conventional DDM and multiple particle tracking. Overall, we propose that DDM-UQ lays the foundation for important new applications of DDM, as well as to high-throughput characterization.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS11.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS11-DDMB: </small></br>Mathematical Modeling of Protein Dynamics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Suzanne S. SINDI (University of California, Merced, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS11.html#author1"><strong class="card-body-subgroup">Erwan HINGANT</strong></a>
		    	 
		    		<em> (Universidad del Bío-Bío, Concepción, CHILE)</em>
		    	
		    	
		    		
		    			</br>"Stochastic nucleation for amyloid diseases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			to be announced
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS11.html#author2"><strong class="card-body-subgroup">Florence HUBERT</strong></a>
		    	 
		    		<em> (Aix-Marseille Université, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Growth fragmention models to understand the dynamical instabilities of microtubules"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microtubules (MTs) are dynamic protein polymers that are found in all eukaryotic cells. They are crucial for normal cell development, aiding in many cellular processes, including cell division, cell polarisation, and cell motility .  Because of their role in cell movement and cell division, these polymers are often used as targets for a variety of cancer chemotherapy drugs. Many experimental studies have been completed to understand MT dynamics , and how these dynamics are altered by the addition of MT targeting drugs. However, a complete understanding of such dynamics is lacking, and so the development of new theoretical models to describe MT dynamics is important. We propose in this talk a mathematical model based on growth-fragmentation PDE and investigate the asymptotic behaviour of the solutions
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS11.html#author3"><strong class="card-body-subgroup">Paul LEMARRE</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"OvPrP oligomers - a short story of structural diversity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this presentation we explore the structural diversity of small OvPrP oligomers. These objects formed in vitro exhibit a surprisingly wide variety of structures and organisations. Using various experimental methods, we are able to devise hypotheses regarding the origin of this diversity and the interactions between the different species. In particular, we study a specific mutant of OvPrP, which selectively creates one type of object. We build a kinetic model for the dynamics of these objects, with the goal to reproduce two crucial qualitative features of the experiments: 1) a non-linear and non-monotonous effect of concentration 2) the interaction between multiple timescales.  Novel processes are included in order to obtain this qualitative behaviour, and the importance of structural diversity in the replication of oligomers is evidenced.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS11.html#author4"><strong class="card-body-subgroup">Stéphanie PORTET</strong></a>
		    	 
		    		<em> (University of Manitoba, CANADA)</em>
		    	
		    	
		    		
		    			</br>"Activation of OAS2 by dsRNA"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The activation of 2'-5'-oligoadenylate synthetase (OAS) enzymes by direct interaction with viral double-stranded RNA (dsRNA) is a key part of the innate immune response to viral infection. A downstream effect of the OAS-dsRNA interaction is to degrade the single-stranded RNA to prevent the spread of the virus. The activation of OAS2, one of the members of the OAS family, depends on dsRNA length. Combining in vitro experiments and mathematical modelling, we test different hypotheses for the OAS2 activation mechanisms by its cofactor dsRNA. After model calibration and selection, the cooperative binding of multiple OAS2 to a single dsRNA is shown to best represent the effect of its cofactor length on enzyme activity. Work from Lee et al. AIMS Mathematics 6: 5924-5941 (2021)
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS12.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS12-DDMB: </small></br>Mathematical Modeling of Protein Dynamics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Suzanne S. SINDI (University of California, Merced, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS12.html#author1"><strong class="card-body-subgroup">Human REZAEI</strong></a>
		    	 
		    		<em> (INRAE, Jouy-en-Josas, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"to be announced"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			to be announced
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS12.html#author2"><strong class="card-body-subgroup">Maria Carla TESI</strong></a>
		    	 
		    		<em> (Universitá di Bologna, ITALY)</em>
		    	
		    	
		    		
		    			</br>"The synergistic interplay between two proteins: a mathematical model for  Alzheimer's disease"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			There is currently a great deal of interest in the scientific community in investigating the effects of the synergistic interplay of Amyloid beta and tau on the dynamics of Alzheimer’s disease. I will present a mathematical model for the onset and progression of Alzheimer’s disease based on transport and diffusion equations for the two proteins.  In the model neurons are treated as a continuous medium and structured by their degree of mal- functioning. Three different mechanisms are assumed to be relevant for the temporal evolution of the disease: i) diffusion and agglomeration of soluble Amyloid beta, ii) effects of misfolded tau protein and iii) neuron-to-neuron prion-like transmission of the disease. These processes are modelled by a system of Smoluchowski equations for the Amyloid beta concentration, an evolution equation for the dynamics of tau protein and a kinetic-type transport equation for the distribution function of the degree of malfunctioning of neurons. The latter equation contains an integral term describing the random onset of the disease as a jump process localized in particularly sensitive areas of the brain. I will explain the structure of the model and give a hint of the main results obtained. Eventually I will also show the output of some numerical simulations, of some significance even if performed in an over-simplified 2D geometry.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS12.html#author3"><strong class="card-body-subgroup">Léon Matar TINE</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Analysis and numerical simulations of a reaction-diffusion model with fixed active bodies: Application to Alzheimer's disease."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk focuses on a spatial interaction model of two substances (or molecules), one of which, concentration f, is produced by bodies located in- side the considered domain and is acting as an activator (positive effect) or a growth factor for the second substance which concentration is denoted by g. The substance or molecules of concentration g on the contrary acts as an inhibitor or a shrinkage for the substance f because of its cytotoxic effect on the bodies activity. The main goal is to analyze the dynamics and propose an adapted numerical approach for the simulation of such kind of model described above where existing bodies (sources for one of the substance) have polygonal shape and their activity can be altered by the presence of the second substance or molecule. For convenience and in accordance with [1] the bodies are taken as fix in the domain. In [1] authors introduced a model based on a discrete growth-fragmentation system with spatial diffusion in order to analyze the early stages of Alzheimer disease. Their model, containing at least five equations and fourteen parameters, aims at representing the process of repli- cation and spatial diffusion of Aβ-oligomers molecules in the neighborhood of neurons. They describe the whole process from Aβ-monomers molecules assembling first into proto-oligomers (unstable polymers) and then into Aβ- oligomers (stable polymers). In [1] the authors carried out a modeling work for the description and simulation of the model where oligomers neurotoxic effect is taken into account. Numerical difficulties are linked to this modeling. A first difficulty is to take into account the geometrical form of active bodies which can be arbitrary. Another difficulty is to manage the sent cytotoxic signals from the substance (or molecule) of concentration g to bodies. In fact, the efficacity of the signal depends on the distance from where it is sent. [1] M. Andrade-Restrepo, P. Lemarre, L. Pujo-Menjouet, L. M. Tine, and S. I. Ciuperca. Modeling the spatial propagation of Aβ oligomers in alzheimer’s disease. In CEMRACS 2018 - Numerical and mathematical modeling for biological and medical applications: deterministic, proba- bilistic and statistical descriptions, pages 1–10, Marseille, France, Jul. 2018
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS12.html#author4"><strong class="card-body-subgroup">Laurent PUJO-MENJOUET</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Alzheimer and Prion: a dangerous liaison"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Alzheimer’s disease (AD) is a fatal incurable disease leading to progressive neuron destruction. AD is caused in part by the accumulation in the brain of Aβ monomers aggregating into oligomers and fibrils. Oligomers are amongst the most toxic structures as they can interact with neurons via membrane receptors, including PrPc proteins. This interaction leads to the misconformation of PrPc into pathogenic oligomeric prions, PrPol. We develop here a model describing in vitro Aβ polymerization process. We include interactions between oligomers and PrPc, causing the misconformation of PrPc into PrPol.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS13.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS13-DDMB: </small></br>Numerical methods in biomedical sciences</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yifan Wang (University of California, Irvine, USA), Pejman Sanaei (New York Institute of Technology, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS13.html#author1"><strong class="card-body-subgroup">Feng Fu</strong></a>
		    	 
		    		<em> (Dartmouth University, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical Modeling of Combination Cancer Immunotherapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is of fundamental importance to understand the key mechanisms that govern the progression of cancer and elucidate the often-unknown factors that account for treatment failures. Immunotherapies have had a significant impact, but only in a minority of late-stage lung cancer and melanoma patients. While potentially curative immunotherapies are being rapidly developed and tested, a major barrier is the lack of quantitative models to describe and evaluate their efficacy. We investigate clinically relevant mathematical and in-silico models of cancer cell dynamics for personalized immunotherapy that boost anti-tumor activities of effector immune cells using single-agent checkpoint blockade and their potential combinations. Our work can be used to interpret lab and clinical results and to guide the design of future lab experiments and clinical trials, all with an eye toward model-informed personalized immunotherapy.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS13.html#author2"><strong class="card-body-subgroup">Pejman Sanaei</strong></a>
		    	 
		    		<em> (Mathematical modeling in tissue engineering, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical modeling in tissue engineering"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell proliferation within a fluid-filled porous tissue-engineering scaffold depends on a sensitive choice of pore geometry and flow rates: regions of high curvature encourage cell proliferation, while a critical flow rate is required to promote growth for certain cell types. When the flow rate is too slow, the nutrient supply is limited; when it is too fast, cells may be damaged by the high fluid shear stress. As a result, determining appropriate tissue-engineering-construct geometries and operating regimes poses a significant challenge that cannot be addressed by experimentation alone. In this work, we present a mathematical theory for the fluid flow within a pore of a tissue-engineering scaffold, which is coupled to the nutrient concentration as well as the growth of cells on the pore walls. We exploit the slenderness of a pore that is typical in such a scenario, to derive a reduced model that enables a comprehensive analysis of the system to be performed. We derive analytical solutions in a particular case of a nearly piecewise constant growth law and compare these with numerical solutions of the reduced model. Qualitative comparisons of tissue morphologies predicted by our model, with those observed experimentally, are also made. We demonstrate how the simplified system may be used to make predictions on the design of a tissue-engineering scaffold and the appropriate operating regime that ensures a desired level of tissue growth.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS13.html#author3"><strong class="card-body-subgroup">Yifan Wang</strong></a>
		    	 
		    		<em> (University of California Irvine, USA)</em>
		    	
		    	
		    		
		    			</br>"Lattice Boltzmann approach to study the evolutionary dynamics of stem-cell driven cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We propose a new approach based on the Lattice Boltzmann Method to simulate tumor cell growth dynamics in the crowded intracellular system. The main advantage of this approach is that it resolves the cell-growth process at the mesoscopic level and thereby provides a more accurate and detailed description than the standard continuous approaches. It is also more computationally efficient than agent-based approaches. Moreover, our method can treat non-regular boundary surfaces efficiently and can capture the heterogeneous property of the intercellular micro-environment and the stochasticity in the tumor growth and other phenomena such as cell confinement from the tissue/extracellular matrix structure.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS13.html#author4"><strong class="card-body-subgroup">Sergey Shevkoplyas</strong></a>
		    	 
		    		<em> (University of Houston, USA)</em>
		    	
		    	
		    		
		    			</br>"Challenges in designing microfluidic devices with a specific functionality"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS13" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS14.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS14-DDMB: </small></br>Numerical methods in biomedical sciences</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Yifan Wang (University of California, Irvine, USA), Pejman Sanaei (New York Institute of Technology, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS14.html#author1"><strong class="card-body-subgroup">Sudhir Pathak</strong></a>
		    	 
		    		<em> (skpathak@pitt.edu, USA)</em>
		    	
		    	
		    		
		    			</br>"Computational Modeling of the human brain tissue, Estimation and Quantifying tissue type"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			MR imaging is a versatile technique that is used to image the anatomical micro-architecture of biological tissue, clinically affected regions such as traumatic injury, blood clot, tumor lesion, and tissue degeneration. In particular, diffusion MR imaging of the human brain can provide the connectivity pattern of the brain regions.
In this presentation, I am going to talk about the characterization of the human brain tissue using diffusion MRI. Diffusion MRI is a novel technique that can be used to characterize the diffusion pattern of the micro-environment of the tissue. From these diffusion patterns, one can characterize geometrical and micro-compartmental information of both healthy and pathological tissues. A volume element of diffusion MR images of human brain tissue contains diffusion signals from free, hindered, and restricted water pools. Using mathematical models and proper MR sequences, these water pools can be used to infer diseases and brain connectivity. In the talk, I will present four such mathematical models, DTI, CHARMED, NODDI, and SMT. I will present the assumptions, (dis)advantage, and feasibility of these mathematical models in a clinical setting. These models can be key to providing important information in clinical diagnosis, presurgical planning and possibly used in deciding treatment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS14.html#author2"><strong class="card-body-subgroup">Yuchi Qiu</strong></a>
		    	 
		    		<em> (Michigan State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Learning biomolecules in mutagenesis via topological and geometric modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mutagenesis is widely used to understand the structure and function of biomolecules. Relying on emerging large mutation datasets in recent years, machine learning methods provide economic approaches to examine function of new mutant biomolecules in silico. The high geometric dimensionality, which usually contains thousands of atoms for one protein, is the main challenge for machine learning models to learn the three-dimensional biomolecules data. Topological and geometric modeling provide informative geometric simplification and scalable representation of the 3D data. In this talk, we develop a multi-scale method utilizing Poincare-Hopf theorem and Morse theory to analyze protein structure. We apply this method to predict mutation induced protein stability changes and it outperforms other existing methods.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS14.html#author3"><strong class="card-body-subgroup">Yu (Andy) Huang</strong></a>
		    	 
		    		<em> (Memorial Sloan Kettering Cancer Center, USA)</em>
		    	
		    	
		    		
		    			</br>"Computational Models of Transcranial Electrical Stimulation: Methodology, Optimization and Validations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Transcranial electrical stimulation (TES) has been shown as a promising neurological therapy for a number of diseases. Nowadays, design of electrode montages and interpretation of experimental results for TES heavily rely on computational models, which predict the current-flow distribution inside the head. In this talk I will show you methodological details in building individualized TES models from structural magnetic resonance images of human heads, including image segmentation, electrode placement, finite element modeling, and numerical optimization for targeted stimulation. Model validations using intracranial in vivo recordings will also be discussed. I will also briefly talk about translational efforts that convert TES models into neuromodulation software, either open-source or proprietary, that are used for clinical research on stroke recovery
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS14.html#author4"><strong class="card-body-subgroup">Mac Hyman</strong></a>
		    	 
		    		<em> (Tulane University, USA)</em>
		    	
		    	
		    		
		    			</br>"A Bipartite Network Sexual Transmission Model to Inform Public Health Efforts for Controlling the Spread of Chlamydia Trachomatis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chlamydia trachomatis (Ct) is the most commonly reported sexually transmitted infection in the USA and causes important reproductive morbidity in women. We created an individual-based heterosexual network model to simulate a realistic chlamydia epidemic on sexual contact networks for a synthetic population.  The model is calibrated to the ongoing routine screening among sexually active men and women in New Orleans.  The Centers for Disease Control and Prevention recommend routine screening of sexually active women under age 25 but not among men. Despite three decades of screening women, chlamydia prevalence in women remains high. Untested and untreated men can serve as a reservoir of infection in women, and increased screening of both men and women can be an effective strategy to reduce infection in women. We assessed the impact of screening men on the Ct prevalence in women.  We used sensitivity analysis to quantify the relative importance of each intervention component.  The model suggested the importance of intervention components ranked from high to low as venue-based screening, expedited index treatment, expedited partner treatment, and rescreening. The findings indicated that male screening can substantially reduce the prevalence among women in high-prevalence communities. Joint research with Zhuolin Qu, Asma Azizi, and Patty Kissinger.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS14" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS15.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS15-DDMB: </small></br>Modelling and Methods in Mathematical Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Anthony Kearsley (National Institute of Standards and Technology, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS15.html#author1"><strong class="card-body-subgroup">Julia Seilert</strong></a>
		    	 
		    		<em> (Department of Food Process Engineering, Technische Universität Berlin, Germany)</em>
		    	
		    	
		    		
		    			</br>"Revisiting a model to predict pure triglyceride thermodynamic properties: parameter optimization and performance"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the thermodynamic properties of triglycerides and their mixtures is of major importance for food applications. Extensive experimental studies and mathematical modeling are needed to predict thermodynamic properties, namely melting temperature and enthalpy of fusion. To date, the most comprehensive work towards modeling triglyceride pure component properties was conducted by Wesdorp in “Liquid-multiple solid phase equilibria in fats: theory and experiments” (1990) building a semi-empirical model with a large set of parameters. The model generally performs well but is known to make thermodynamically inconsistent predictions for certain test cases. In this study, the underlying parameter set is improved in order to deliver more physically consistent predictions without deterioration of the primary model quality to describe the available experimental data. Thermodynamic constraints as well as bound constraints on variables are discussed regarding an interrelation of the model setup conditions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS15.html#author2"><strong class="card-body-subgroup">Adarsh Kumbhari</strong></a>
		    	 
		    		<em> (School of Mathematics and Statistics, University of Sydney, Australia)</em>
		    	
		    	
		    		
		    			</br>" Modeling PD-L1 inside the tumor microenvironment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The protein PD-1 and its ligand PD-L1 are upregulated on cancerous and immune cells within tumors, and blocking this pathway may induce anti-tumor immunity. The extent to which PD-L1 expression reflects immune activity, however, is poorly understood. Using mathematical modeling, we show that high PD-L1 expression can reflect both tumor escape and clearance. We also identify several T-cell populations that may better reflect dynamic changes to the tumor microenvironment. These findings suggest that moving beyond measuring PD-L1 expression could lead to better ways to predict patient responses to PD-L1 blockade.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS15.html#author3"><strong class="card-body-subgroup">Danielle Brager</strong></a>
		    	 
		    		<em> (National Institute of Standards and Technology, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematically Investigating Retinitis Pigmentosa"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Retinitis Pigmentosa (RP) is a collection of clinically and genetically heterogeneous degenerative retinal diseases.  Patients with RP experience a loss of night vision that progresses to day-light blindness due to the sequential degeneration of rod and cone photoreceptors.  While known genetic mutations associated with RP affect the rods, the degeneration of cones inevitably follows in a manner independent of those genetic mutations.  Investigation of this secondary death of cone photoreceptors led to the discovery of the rod-derived cone viability factor (RdCVF), a protein secreted by the rods that preserves the cones by accelerating the flow of glucose into cone cells stimulating aerobic glycolysis.  In this work, we formulate a predator-prey style system of nonlinear ordinary differential equations to mathematically model photoreceptor interactions in the presence of RP while accounting for the new understanding of RdCVF's role in enhancing cone survival.  We utilize the mathematical model and subsequent analysis to examine the underlying processes and mechanisms (defined by the model parameters) that affect cone photoreceptor vitality as RP progresses.  The physiologically relevant equilibrium points are interpreted as different stages of retinal degeneration.  We determine conditions necessary for the local asymptotic stability of these equilibrium points and use the results as criteria needed to remain in a stage in the progression of retinal degeneration.  Experimental data is used for parameter estimation.  Pathways to blindness are uncovered via bifurcations and narrows our focus to four of the model equilibria.  We perform a sensitivity analysis to determine mechanisms that have a significant effect on the cones at four stages of RP.  We derive a non-dimensional form of the mathematical model and perform a numerical bifurcation analysis using MATCONT to explore the existence of stable limit cycles because a stable limit cycle is a stable mode, other than an equilibrium point, where the rods and cones coexist.  In our analyses, a set of key parameters involved in photoreceptor outer segment shedding, renewal, and nutrient supply were shown to govern the dynamics of the system.  Our findings illustrate the benefit of using mathematical models to uncover mechanisms driving the progression of RP and opens the possibility to use in silico experiments to test treatment options in the absence of rods.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS15.html#author4"><strong class="card-body-subgroup">Anca Radulescu</strong></a>
		    	 
		    		<em> (State University of New York at New Paltz, USA)</em>
		    	
		    	
		    		
		    			</br>"Estimating glutamate transporter surface density in mouse hippocampal astrocytes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			One of the main functions of astrocytes is to remove glutamate from the extracellular space, a task that is accomplished through the activity of glutamate transporters expressed in abundance in the plasma membrane. This property allows astrocytes to limit glutamate diffusion out of the synaptic cleft, to limit extrasynaptic receptor activation and preserve the spatial specificity of synaptic transmission. The distribution of glutamate transporters on is known to be heterogeneous, as these molecules are enriched in astrocyte tip processes as opposed to the rest of the membrane. We investigate in depth the effect of this non-uniform distribution, while also evaluating how local crowding effects can limit the transporter expression in small astrocytic processes.
We first obtain an experimental estimate of the glutamate transporter surface expression in different sub-cellular compartments of mouse hippocampal astrocytes. We then generate a geometric model of astrocytes that capture statistically the main structural features of real astrocytes, to determine the proportion of the astrocyte cell membrane in different cellular compartments. We found stark differences in the density of expression of transporter molecules in different compartments, indicating that the extent to which astrocytes limit extrasynaptic glutamate diffusion depends not only on the level of astrocytic coverage, but also on the identity of the compartment in contact with the synapse. Together, these findings provide information on the spatial distribution of glutamate transporters in the mouse hippocampus, with potentially long-range implications for the fields of synaptic plasticity and astrocyte physiology.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS15" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS16.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS16-DDMB: </small></br>Image Analysis and Machine Learning for Bio-Medical Applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Amit Roy-Chowdhury (University of California, Riverside), G. Venugopala Reddy (University of California, Riverside)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS16-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS16.html#author1"><strong class="card-body-subgroup">B. S. Manjunath</strong></a>
		    	 
		    		<em>(University of California, Santa Barbara)</em>
		    	
		    	
		    		
		    			</br>"3D cell/nuclei segmentation and tracking using deep networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Accurate cell/nuclei segmentation and tracking play an important role in time-lapse 3D microscopy image analysis. Features of interest often depend on precise localization of 3D points on the boundary. Towards this, we present a deep network coupled with a conditional random field model for cell segmentation of 3D confocal membrane tagged image stacks, and a supervoxel based segmentation of 3D nuclei tagged images with few annotations. To track these segmented cells/nuclei, a computationally efficient algorithm is proposed that utilizes the relative cell/nuclei location while maintaining tracking accuracy. Detailed experimental results demonstrate the feasibility of the proposed methods on large 3d time-lapse imagery.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS16.html#author2"><strong class="card-body-subgroup">Michelle Digman</strong></a>
		    	 
		    		<em>(University of California, Irvine)</em>
		    	
		    	
		    		
		    			</br>"Quantifying Spatio-temporal dynamics and Metabolic Alterations of protein upon DNA Damage"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			DNA damage signaling is critical for the maintenance of genome integrity and cell fate decision. Our genome is constantly under assault by various endogenous and environmental agents, exposure of UV rays and even routine DNA replication can cause obstruction of replication or transcription. The DNA damage response is a highly integrated signaling network has a set of mechanism that can detect the type of severity of DNA damage to initiate repair or apoptosis. This talk will describe methodologies used to investigate p53 protein activity and alteration of the metabolic pathway upon DNA damage.  Here we present 2-Photon excitation laser microirradiation to induce different types of DNA damage, the Number and Molecular Brightness (N&B) method to map aggregation, and the phasor approach to FLIM to map metabolic changes upon DNA damage. Overall, our findings demonstrate that by multiplexing these techniques we have the ability to spatially and temporally quantify p53 activation and map p53‚Äôs influence in the metabolic pathway.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS16.html#author3"><strong class="card-body-subgroup">Cory Braker	Scott</strong></a>
		    	 
		    		<em>(University of California, Irvine)</em>
		    	
		    	
		    		
		    			</br>"Morphological Analysis of Biological Images Using Spectral Graph Theory and Graph Neural Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present a method for learning ``spectrally descriptive'' edge weights for graphs. We generalize a previously known distance measure between graphs (Graph Diffusion Distance), thereby allowing it to be tuned to minimize an arbitrary loss function. Because all steps involved in calculating this modified GDD are differentiable, we demonstrate that it is possible for a small neural network model to learn edge weights to minimize loss. We demonstrate this by applying this metric to two groups of graphs derived from samples from two genotypes of Arabidopsis. GDD by itself cannot distinguish between these two categories of graphs. However, training edge weights and kernel parameters with contrastive loss produces a learned distance metric with large margins between graph categories. We demonstrate this by showing improved performance of a simple k-nearest-neighbors classifier on the learned distance matrix. We also demonstrate further applications of this technique.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS16.html#author4"><strong class="card-body-subgroup">Kevin	Rodriguez</strong></a>
		    	 
		    		<em>(University of California, Riverside)</em>
		    	
		    	
		    		
		    			</br>"Interplay between layer specific chemical signals and mechanical properties maintain the structure and shape of the shoot apical meristem in Arabidopsis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The shoot apical meristem (SAM) is continually derived from a population of stem cells located at the growing tip of the plants. These stem cells shed off populations of daughter cells both radially and basally. As the daughter cells are displaced, they undergo increased cell division rates and changes in gene expression critical to determine the SAM structure and shape.  The cell division rates and changes in gene expression occur at a certain distance along the transcription factor-WUSCHEL and plant hormone cytokinin signaling domains. In addition, the cell division and gene expression are compromised in wuschel and cytokinin signaling mutants, suggesting these chemical signals regulate cell division rates. The changes in cell division rates and displacement of daughter cells affects the structure and shape of local cells and ultimately the SAM as a whole. Through a combination of transient gene expression, quantitative image analysis and biologically-calibrated computational model simulations we test the possible mechanisms regulating cell division to determine the SAM structure.  Our analysis suggests that WUSCHEL, cytokinin, and mechanical stress regulate patterns of cell expansion and cell division plane orientation in a layer specific fashion to maintain the layered structure and shape of SAMs which are critical for stem cell homeostasis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS16" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS17.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS17-DDMB: </small></br>Image Analysis and Machine Learning for Bio-Medical Applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Amit Roy-Chowdhury (University of California, Riverside), G. Venugopala Reddy (University of California, Riverside)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS16-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS17.html#author1"><strong class="card-body-subgroup">Henrik Jonsson</strong></a>
		    	 
		    		<em>(Cambridge Sainsbury Laboratories, UK)</em>
		    	
		    	
		    		
		    			</br>"Integration of live imaging and spatial modelling in plant development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The shoot apical meristem is a stem cell niche providing cells to the continuous development of new flower organs. By using live imaging we can track individual cells over several days of the meristem and flower development and by combining molecular markers into a single organ, we can address questions of regulatory aspects of patterning and morphogenesis. I will present how we use this to evaluate existing hypotheses for the regulation of gene expression and growth and how we can evaluate novel hypotheses in parallel at a large scale.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS17.html#author2"><strong class="card-body-subgroup">Anuradha Kar</strong></a>
		    	 
		    		<em>(ENS-Lyon, CNRS, France)</em>
		    	
		    	
		    		
		    			</br>"Deep learning for cellular segmentation in 3D confocal images"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Confocal microscopy is a prominent mode of imaging plant tissue surfaces and deeper cellular layers. Confocal images of plant organs are used to create three-dimensional digital models of the tissues with cellular resolution using image analysis algorithms. These digital models are the foundations for quantitative analysis of plant morphogenesis, lineage construction and understanding gene functions and expression patterns. The first step towards creating a 3D representation of a tissue from confocal images is the task of cellular segmentation in which each cell within an image is to be identified as an independent 3D object . Several computational methods for 3D cell segmentation have been developed over the years, a prominent one being the watershed technique. However, this method requires manual tuning of its parameters and its accuracy is frequently affected by poor signal and noise levels in the image.  In recent times, cell segmentation pipelines using advanced computational algorithms known as deep learning have emerged which have demonstrated high accuracy and automatic segmentation capabilities even in poor quality images. In this presentation , we will look into the concept of several such deep learning based segmentation pipelines and see how they can be trained to perform 3D segmentation of confocal images of floral meristems. We will discuss their pros and cons and present our tools and libraries which may be used for quantitative and visual comparison of the performances of such emerging deep learning based segmentation techniques.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS17.html#author3"><strong class="card-body-subgroup">Richard Smith</strong></a>
		    	 
		    		<em>(Univ of Koln, Germany and John Innes Center, Norwich, UK)</em>
		    	
		    	
		    		
		    			</br>"Quantifying life on surfaces with MorphoGraphX"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			How an organism achieves its shape is a fundamental question in developmental biology. Form emerges from the interaction of genetic and mechanical processes that drive changes in the geometry of cells and tissues. Ideally it would be great to quantify the evolution of cell shape, proliferation and gene expression in full 3D, however this is often technically challenging. 2D planar projections are sometimes an option, however they do not work on highly curved organs. Our lab has developed MorphoGraphX (www.MorphoGraphX.org) a software that bridges this gap by enabling image processing directly on curved surfaces, what we informally refer to as 2.5D images. Many developmental processes happen on surfaces, such as in the epidermal layer of cells in plants or on epithelial layers in animals. Once cells are segmented, they require annotation, it is not just enough to know the positions and shapes of 100s or 1000s of cells, we need to also know where they are in the organism or organ, in order to decipher how they are responding to developmental signals. Organs are thought to be patterned genetically by gradients of morphogens and that determine growth rates and cell and tissue polarity. Not unlike genomic sequence data, which is of little use without annotation, knowledge of the cells' position and polarity within the organ they are developing is key to make sense of the data. Here I will present an array of tools we have developed in MorphoGraphX to annotate cells with positional information, both for 2.5 and full 3D images.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS17.html#author4"><strong class="card-body-subgroup">Albert Do</strong></a>
		    	 
		    		<em>(University of California, Riverside)</em>
		    	
		    	
		    		
		    			</br>"Multiscale modeling of the Arabidopsis shoot meristem signaling network"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Growth in plants is coordinated by collections of undifferentiated cell clusters known as meristems. These meristems in turn are coordinated by highly complex regulatory networks. The WUSCHEL (WUS) transcription factor is a key regulator in the shoot apical meristem governing above ground growth. One of WUS‚Äôs most important targets is the CLAVATA3 (CLV3) signaling peptide. WUS and CLV3 have a complex bidirectional relationship both upregulating and repressing each other that does not easily fit within standard regulatory paradigms. To model this relationship, a hybrid system of meristem signaling consisting of deterministic ODE based and stochastic based dynamics was constructed. The ODE portion models protein/RNA dynamics while the stochastic portion models the binding of WUS to the CLV3 gene regulatory region/cis regulatory module (CRM). This deterministic/stochastic model is able to accurately replicate expression patterns seen in experimental data, generate data that fits what is known about the biology in scenarios that have not yet undergone rigorous wetlab analysis, and provides a way to directly observe the dynamics of WUS binding patterns on the CRM.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS17" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS18.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS18-DDMB: </small></br> Mathematical Modeling of Exposure and Target Interaction in Pharmaceutical Development of Therapeutic Proteins</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jeroen Elassaiss-Schaap (PD-value BV, Netherlands), Johannes Schropp (University of Konstanz, Germany)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS18.html#author1"><strong class="card-body-subgroup">Leonid Gibiansky</strong></a>
		    	 
		    		<em> (QuantPharm LLC, North Potomac, MD, USA)</em>
		    	
		    	
		    		
		    			</br>"Target-mediated drug disposition in the pharmacokinetics of monoclonal antibodies and its quasi-steady state solutions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The term TMDD refers to the biological processes and models where drug-target binding significantly influences both pharmacodynamics (PD) and pharmacokinetics (PK). These are typical for the biologic drugs with high specificity to the intended target. The TMDD model describes the processes on the widely different time scales: fast drug-target binding and relatively slow drug distribution and elimination. Given the typical clinically relevant sampling, this model is rarely identifiable thus requiring use of approximations. Various TMDD approximations have been developed. Investigation of the TMDD equations identified distinct phases in the concentration time profiles. The initial fast phase reflects drug-target binding processes. This phase is followed by a slow phase where the drug, target, and drug-target complex are in a slowly changing equilibrium. Several approximations that differ by the underlying assumptions have been developed. The quasi-steady state (QSS) approximation describes the TMDD system where the elimination of the drug-target complex is much slower than the elimination of the free target. In this case, the drug-target complex contributes significantly to the drug kinetics. The QSS approximation was successful in describing PK and PD of monoclonal antibodies that target soluble receptors. When the drug-target complex is eliminated faster than the free target, the QSS equations can be simplified to result in the Michaelis-Menten (MM) approximation. The MM approximation was shown to describe PK of many monoclonal antibodies that target membrane receptors. For drugs that bind to both soluble and membrane receptors, the QSS approximation of the two-target TMDD equations can be used. TMDD modeling framework can be adapted to many different systems, e.g. drugs that bind to targets with two binding sites, drugs with two identical binding sites, antibody-drug conjugates, etc.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS18.html#author2"><strong class="card-body-subgroup">Wojciech Krzyzanski</strong></a>
		    	 
		    		<em> (University at Buffalo, USA)</em>
		    	
		    	
		    		
		    			</br>"Application of Quasi-equilibrium Approximation to Reduction of Complex Physiologically Based Pharmacokinetic Models of Monoclonal Antibodies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Physiologically based pharmacokinetic (PBPK) models are commonly used to describe the time courses of plasma concentrations of monoclonal antibodies. These models use ordinary differential equations to quantify monoclonal antibodies disposition in compartments such as the plasma, lymph node, as well as major peripheral organs. In addition, each organ consists of vascular, interstitial, and endosomal sub-compartments. The drug molecules are distributed to all compartments, taken up by the organs, internalized to the cell cytoplasm, and degraded in the endosomes or recycled to the interstitial space. In result, a PBPK model becomes a high dimensional nonlinear system with many parameters. We present a model reduction technique that is based on the quasi-equilibrium assumptions about the rates of processes of the dynamical system that simplifies the PBPK model to a three-compartment linear model. The technique is applied to a previously published PBPK model for monoclonal antibodies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS18.html#author3"><strong class="card-body-subgroup">Johannes Schropp</strong></a>
		    	 
		    		<em> (University of Konstanz, Germany)</em>
		    	
		    	
		    		
		    			</br>"Bispecific-Antibodies: Properties, Approximation and Optimal Dosing Strategy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bispecific antibodies (BsAbs) bind to two different targets, and create two binary and one ternary complex (TC). These molecules have shown, i.e., promise as immune-oncology drugs. We present a general target-mediated drug disposition model for these BsAbs, which bind to two different targets on different cell membranes. In addition, a quasi-equilibrium approximation with less binding parameters and, if necessary, reduced internalization parameters is presented. The model is used to investigate the kinetics of BsAb and TC. The analysis shows that larger doses of BsAbs may delay the build-up of the TC. Consequently, an optimal dosing strategy of BsAbs, which immediately create and maintain maximal possible TC concentration, is presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS18.html#author4"><strong class="card-body-subgroup">Weirong Wang</strong></a>
		    	 
		    		<em> (Janssen, USA)</em>
		    	
		    	
		    		
		    			</br>"Target-mediated drug disposition of immuno-oncology drugs: mathematical models for exposure and pharmacodynamics, and its translation between animal and man"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The therapeutic effect of all biotherapeutics is driven by its interaction with the therapeutic target. These interactions are often called target engagement (TE) when the target is a soluble protein and receptor occupancy (RO) when the target is a cell surface receptor. TE and RO assessments play a central role in translational pharmacology. Although TE or RO itself does not guarantee efficacy, it reflects the dynamics of drug engagement and corresponding target modulation and provides a mechanism to extrapolate drug effects between preclinical species and humans, as well as between healthy and disease populations. Together with mechanism and physiologically based PK/PD modeling, TE and RO are commonly used to facilitate rational dose selection and clinical study design.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS18" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS19.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS19-DDMB: </small></br>Data-Driven Modeling and Analysis in Mathematical Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Tomas Carino-Bazan (University of California, Santa Barbara, United States), Daniel Wilson (Boston University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS19.html#author1"><strong class="card-body-subgroup">Julie Hussin</strong></a>
		    	 
		    		<em> (Université de Montréal, Montreal Heart Institute, Canada)</em>
		    	
		    	
		    		
		    			</br>"Evolutionary approaches to detect epistasis in large-scale genomic data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Whether gene-gene interactions, or epistasis, plays a major or minor role for any given human trait in any population remains an open question, and analytical methods to detect epistasis have become very popular in the last decade. However, there are important computational and statistical challenges for identifying novel epistatic interactions in human genetics. To help solve the paucity of uncovered epistasis in humans, we propose new approaches to characterize gene-by-gene interactions, in studying signatures of co-evolution. The underlying model is that interacting genes will undergo compensatory genetic mutations to maintain their interaction, which will result in correlation of allelic frequencies between physically unlinked loci. In this talk, I will present data-driven projects on two distinct systems, interactions among Cytochrome P450 genes and co-evolution involving the cholesterol metabolism gene CETP, and their implications for precision medicine. Our studies also demonstrate how data from diverse human populations in genetic studies can be leveraged to uncover biological mechanisms of importance for world-wide population health.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS19.html#author2"><strong class="card-body-subgroup">Elana Fertig</strong></a>
		    	 
		    		<em> (Johns Hopkins University, United States)</em>
		    	
		    	
		    		
		    			</br>"Identifying therapeutic resistance mechanisms in cancer with single-cell data and transfer learning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Tumors employ complex, multi-scale cellular and molecular interactions that evolve over the course of therapeutic response. The changes in these pathways enables tumors to overcome therapeutic regimens, and ultimately acquire resistance. New molecular profiling technologies, including notably single cell technologies, provide an unprecedented opportunity to characterize these molecular relationships. However, interpreting the specific cellular and molecular pathways in therapeutic response requires complementary computational analysis methods. We developed an unsupervised learning method, CoGAPS, that employs Bayesian non-negative matrix factorization to disentangle distinct biological processes from high-throughput molecular data. Notably, this algorithm discovers dynamic compensatory signaling in acquired therapeutic resistance from time course bulk RNA-seq data and novel NK cell activation in anti-CTLA4 response from post-treatment scRNA-seq data. To further demonstrate that the inferred pathways are biological rather than computational artifacts, we developed a complementary transfer learning method to relate learned patterns between datasets. We demonstrate that this approach identifies robust molecular processes between model systems and human tumors and enables multi-platform data integration to delineate the drivers of therapeutic response and resistance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS19.html#author3"><strong class="card-body-subgroup">Tomas Carino-Bazan</strong></a>
		    	 
		    		<em> (University of California, Santa Barbara, United States)</em>
		    	
		    	
		    		
		    			</br>"Machine learning methods for fluid mechanics for learning low dimensional representations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many empirical studies and experiments in applications ranging from biophysics to engineering design yield partial information of the flow fields and related hydrodynamic responses. We develop data-driven methods for learning representations of hydrodynamic responses for inference tasks.  From an analytic perspective the field of fluid mechanics traditionally has used transformations such as vorticity to represent localized flow structures and for numerical simulations.  For example for inviscid flows this often yields a sparse representation.  We seek to develop related machine learning methods that learn more general non-linear transformations that can featurize hydrodynamic flow data for making inferences about flow structure and dynamics.  We discuss our progress toward studying hydrodynamic flows using auto-encoders with associated regularizations to learn smooth low dimensional representations of flow structures.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS19.html#author4"><strong class="card-body-subgroup">Lorin Crawford</strong></a>
		    	 
		    		<em> (Microsoft Research New England, United States)</em>
		    	
		    	
		    		
		    			</br>"Statistical Frameworks for Discovering Biophysical Signatures in 3D Shapes and Images"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The recent curation of large-scale databases with 3D surface scans of shapes has motivated the development of tools that better detect global patterns in morphological variation. Studies which focus on identifying differences between shapes have been limited to simple pairwise comparisons and rely on pre-specified landmarks (that are often known). In this talk, we present SINATRA: a statistical pipeline for analyzing collections of shapes without requiring any correspondences. Our method takes in two classes of shapes and highlights the physical features that best describe the variation between them. We develop a rigorous simulation framework to assess our approach, which themselves are a novel contribution to 3D image and shape analyses. Lastly, as case studies, we use SINATRA to (1) analyze mandibular molars from four different suborders of primates and (2) facilitate the visual identification of structural signatures differentiating between two protein ensembles.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/DDMB/DDMB-MS20.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS20-DDMB: </small></br>Data-Driven Modeling and Analysis in Mathematical Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Tomas Carino-Bazan (University of California, Santa Barbara, United States), Daniel Wilson (Boston University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/DDMB/DDMB-MS20.html#author1"><strong class="card-body-subgroup">Daniel Wilson</strong></a>
		    	 
		    		<em> (Boston University, United States)</em>
		    	
		    	
		    		
		    			</br>"Inferring the molecular reach of antibodies from antigen binding data using an agent-based spatial model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Surface Plasmon Resonance (SPR) is a widely-used biophysical technique used to produce high-resolution temporal signals of molecular binding interactions. In SPR, one molecule is immobilised on a 3D matrix whilst another, known as the analyte, is injected over the surface. The instrument provides a highly sensitive measure of binding in the matrix. When the analyte is monovalent, the binding data can be fit by a well-mixed 1:1 binding model to determine the kinetic rate constants. However, there are many situations where the analyte is bivalent. A prominent example is the study of antibodies that have two binding sites for their immobilised antigen. This produces complex SPR binding data that is not well fit by the 1:1 binding model. In this talk, we present a computational method to infer the binding parameters from bivalent analytes. Using a stochastic spatial model of bivalent binding we train a surrogate model that allows for highly efficient parametrisation of antibody SPR data. In addition to inferring binding parameters, our new method allows us to estimate the ‘molecular reach’ of antibodies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS20.html#author2"><strong class="card-body-subgroup">Paul Atzberger</strong></a>
		    	 
		    		<em> (University of California, Santa Barbara, United States)</em>
		    	
		    	
		    		
		    			</br>"Variational Autoencoders with Manifold Latent Spaces for Learning Nonlinear Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We develop data-driven methods for learning parsimonious representations of nonlinear dynamical systems by incorporating physical information and other priors.  Our approach is based on Variational Autoencoders (VAEs) for learning nonlinear state space models from observation data.  VAE use noise-based regularizations and priors to help ensure continuity in latent encoding and in disentangling latent features.  To obtain low dimensional parsimonious representations, we introduce ways to incorporate geometric and topological priors through general manifold latent spaces.  We demonstrate our methods for learning non-linear dynamics in non-linear fluid mechanics and reaction-diffusion systems. Co-authors: Ryan Lopez, Paul J. Atzberger, University of California Santa Barbara.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS20.html#author3"><strong class="card-body-subgroup">Guy Wolf</strong></a>
		    	 
		    		<em>(Université de Montréal; Mila - Quebec AI Institute, Canada)</em>
		    	
		    	
		    		
		    			</br>"Multiscale exploration of single cell data with geometric harmonic analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			High-throughput data collection technologies are becoming increasingly common in many fields, especially in biomedical applications involving single cell data genomics and transcriptomics. These introduce a rising need for exploratory analysis to reveal and understand hidden structure in the collected (high-dimensional) Big Data. A crucial aspect in such analysis is the separation of intrinsic data geometry from data distribution, as (a) the latter is typically biased by collection artifacts and data availability, and (b) rare subpopulations and sparse transitions between meta-stable states are often of great interest in biomedical data analysis. In this talk, I will show several tools that leverage manifold learning, graph signal processing, and harmonic analysis for biomedical (in particular, genomic/proteomic) data exploration, with emphasis on visualization, and nonlinear feature extraction, and multiresolution analysis. A common thread in the presented tools is the construction of a data-driven diffusion geometry that both captures intrinsic structure in data and provides a generalization of Fourier harmonics on it. These, in turn, are used to process data features along the data geometry for multiple purposes, including preprocessing of single cell data and enabling batch-level geometric exploration, e.g., over and between medical conditions, health states, and drug reactions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/DDMB/DDMB-MS20.html#author4"><strong class="card-body-subgroup">John Lagergren</strong></a>
		    	 
		    		<em> (Oak Ridge National Laboratory, United States)</em>
		    	
		    	
		    		
		    			</br>"Data-driven network analysis detects longitudinal environmental changes with impacts on food, energy, and pandemics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			To address the needs of a growing human population, which includes the significant expansion of sustainable food and bio-energy production capacities in the context of a changing climate, we develop novel climatype identification methods to predict longitudinal processes relevant to these challenges. In this work, we leverage the DUO algorithm to compute two-way and three-way environmental comparisons at unprecedented scale and accuracy to find high-order relationships between geospatial coordinates with high resolution at global scale. Novel network analysis methods are applied to the series of emergent climatype networks to identify climate zones that share similar environmental relationships and to track how these relationships are changing over time. The methods discussed herein are also applicable to correlation analyses in other diverse fields such as systems biology, ecology, materials science, carbon cycles, biogeochemistry, additive manufacturing, and zoonosis research.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <a id="CT"><h1 class="mb-3" id="MS1">Sub-group contributed talks</h1></a>
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
	                <a id="Posters"><h1 class="mb-3" id="MS1">Sub-group poster presentations</h1></a>
									<div class="row subgroup-name">
										<div class="col-12">
											<div class="card">															
										  	<h5 class="card-header card-header-subgroup DDMB">DDMB Posters</h5>
											  <div class="card-body">
											    <p class="card-text">																
														<div>
															
													    	    	
											  		</div>
													</p>
											  </div>
											</div>
										</div>
									</div>
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
					Click to show abstracts
									</span>		            
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
