---
layout: group
author: esmb
day: "Thursday (Friday)"
subgroup: "DDMB"
title: "Data-Driven Modeling and Analysis in Mathematical Biology"
organizers: "Tomas Carino-Bazan (University of California, Santa Barbara, United States), Daniel Wilson (Boston University, United States)"
description: "Recent advances in data science and machine learning are providing
novel ways to learn models and perform analysis of biological systems.
This session brings together researchers to discuss recent developments
in the field, advances in methodology and computational methods, and
emerging application domains in the biological sciences. Topics include
data-driven development of mechanistic and mechanical models in cell
biology, analysis of genomic data with applications to disease progression
and precision medicine, and statistical methods for investigating protein
structure. The session aims to discuss both general topics concerning
methodology as well as specific motivating application domains."
code: "MS20"
code2: "MS19"
author1: "Daniel Wilson"
inst1: " (Boston University, United States)"
title1: "Inferring the molecular reach of antibodies from antigen binding data using an agent-based spatial model"
abstract1: "Surface Plasmon Resonance (SPR) is a widely-used biophysical technique used to produce high-resolution temporal signals of molecular binding interactions. In SPR, one molecule is immobilised on a 3D matrix whilst another, known as the analyte, is injected over the surface. The instrument provides a highly sensitive measure of binding in the matrix. When the analyte is monovalent, the binding data can be fit by a well-mixed 1:1 binding model to determine the kinetic rate constants. However, there are many situations where the analyte is bivalent. A prominent example is the study of antibodies that have two binding sites for their immobilised antigen. This produces complex SPR binding data that is not well fit by the 1:1 binding model. In this talk, we present a computational method to infer the binding parameters from bivalent analytes. Using a stochastic spatial model of bivalent binding we train a surrogate model that allows for highly efficient parametrisation of antibody SPR data. In addition to inferring binding parameters, our new method allows us to estimate the ‘molecular reach’ of antibodies."
author2: "Paul Atzberger"
inst2: " (University of California, Santa Barbara, United States)"
title2: "Variational Autoencoders with Manifold Latent Spaces for Learning Nonlinear Dynamics"
abstract2: "We develop data-driven methods for learning parsimonious representations of nonlinear dynamical systems by incorporating physical information and other priors.  Our approach is based on Variational Autoencoders (VAEs) for learning nonlinear state space models from observation data.  VAE use noise-based regularizations and priors to help ensure continuity in latent encoding and in disentangling latent features.  To obtain low dimensional parsimonious representations, we introduce ways to incorporate geometric and topological priors through general manifold latent spaces.  We demonstrate our methods for learning non-linear dynamics in non-linear fluid mechanics and reaction-diffusion systems. Co-authors: Ryan Lopez, Paul J. Atzberger, University of California Santa Barbara."
author3: "Guy Wolf"
inst3: "(Université de Montréal; Mila - Quebec AI Institute, Canada)"
title3: "Multiscale exploration of single cell data with geometric harmonic analysis"
abstract3: "High-throughput data collection technologies are becoming increasingly common in many fields, especially in biomedical applications involving single cell data genomics and transcriptomics. These introduce a rising need for exploratory analysis to reveal and understand hidden structure in the collected (high-dimensional) Big Data. A crucial aspect in such analysis is the separation of intrinsic data geometry from data distribution, as (a) the latter is typically biased by collection artifacts and data availability, and (b) rare subpopulations and sparse transitions between meta-stable states are often of great interest in biomedical data analysis. In this talk, I will show several tools that leverage manifold learning, graph signal processing, and harmonic analysis for biomedical (in particular, genomic/proteomic) data exploration, with emphasis on visualization, and nonlinear feature extraction, and multiresolution analysis. A common thread in the presented tools is the construction of a data-driven diffusion geometry that both captures intrinsic structure in data and provides a generalization of Fourier harmonics on it. These, in turn, are used to process data features along the data geometry for multiple purposes, including preprocessing of single cell data and enabling batch-level geometric exploration, e.g., over and between medical conditions, health states, and drug reactions."
author4: "John Lagergren"
inst4: " (Oak Ridge National Laboratory, United States)"
title4: "Data-driven network analysis detects longitudinal environmental changes with impacts on food, energy, and pandemics"
abstract4: "To address the needs of a growing human population, which includes the significant expansion of sustainable food and bio-energy production capacities in the context of a changing climate, we develop novel climatype identification methods to predict longitudinal processes relevant to these challenges. In this work, we leverage the DUO algorithm to compute two-way and three-way environmental comparisons at unprecedented scale and accuracy to find high-order relationships between geospatial coordinates with high resolution at global scale. Novel network analysis methods are applied to the series of emergent climatype networks to identify climate zones that share similar environmental relationships and to track how these relationships are changing over time. The methods discussed herein are also applicable to correlation analyses in other diverse fields such as systems biology, ecology, materials science, carbon cycles, biogeochemistry, additive manufacturing, and zoonosis research."
---
