---
layout: group
author: esmb
day: "Wednesday (Thursday)"
subgroup: "DDMB"
title: "Image Analysis and Machine Learning for Bio-Medical Applications"
organizers: "Amit Roy-Chowdhury (University of California, Riverside), G. Venugopala Reddy (University of California, Riverside)"
description: "Image analysis is common in many bio-medical research applications. In spite of the prevalence of a number of relevant tools, a large part of the analysis is still manual and extremely tedious. This limits the amount of data that can be analyzed and prevents drawing statistically relevant conclusions. The reasons for the existing computational tools to not perform satisfactorily in many applications are often driven by poor image quality, and this calls for developing advanced methods that would be able to overcome these limitations. There has been tremendous progress in image analysis in the last decade, building upon machine learning methods. A pressing question is how do we translate these methods from the computer science fields into application in biomedical domains? This workshop will focus on such issues. Biomedical applications bring their own challenges that these advance computing approaches will need to address. In this workshop, we will focus on a few of these problems, e.g., how to use machine learning when there is limited labeled data, how do we combine physics-driven approaches with data-driven methods, and how do we combine human feedback effortlessly into the learning paradigm."
code: "MS17"
code2: "MS16"
author1: "Henrik Jonsson"
inst1: "(Cambridge Sainsbury Laboratories, UK)"
title1: "Integration of live imaging and spatial modelling in plant development"
abstract1: "The shoot apical meristem is a stem cell niche providing cells to the continuous development of new flower organs. By using live imaging we can track individual cells over several days of the meristem and flower development and by combining molecular markers into a single organ, we can address questions of regulatory aspects of patterning and morphogenesis. I will present how we use this to evaluate existing hypotheses for the regulation of gene expression and growth and how we can evaluate novel hypotheses in parallel at a large scale."
author2: "Anuradha Kar"
inst2: "(ENS-Lyon, CNRS, France)"
title2: "Deep learning for cellular segmentation in 3D confocal images"
abstract2: "Confocal microscopy is a prominent mode of imaging plant tissue surfaces and deeper cellular layers. Confocal images of plant organs are used to create three-dimensional digital models of the tissues with cellular resolution using image analysis algorithms. These digital models are the foundations for quantitative analysis of plant morphogenesis, lineage construction and understanding gene functions and expression patterns. The first step towards creating a 3D representation of a tissue from confocal images is the task of cellular segmentation in which each cell within an image is to be identified as an independent 3D object . Several computational methods for 3D cell segmentation have been developed over the years, a prominent one being the watershed technique. However, this method requires manual tuning of its parameters and its accuracy is frequently affected by poor signal and noise levels in the image.  In recent times, cell segmentation pipelines using advanced computational algorithms known as deep learning have emerged which have demonstrated high accuracy and automatic segmentation capabilities even in poor quality images. In this presentation , we will look into the concept of several such deep learning based segmentation pipelines and see how they can be trained to perform 3D segmentation of confocal images of floral meristems. We will discuss their pros and cons and present our tools and libraries which may be used for quantitative and visual comparison of the performances of such emerging deep learning based segmentation techniques."
author3: "Richard Smith"
inst3: "(Univ of Koln, Germany and John Innes Center, Norwich, UK)"
title3: "Quantifying life on surfaces with MorphoGraphX"
abstract3: "How an organism achieves its shape is a fundamental question in developmental biology. Form emerges from the interaction of genetic and mechanical processes that drive changes in the geometry of cells and tissues. Ideally it would be great to quantify the evolution of cell shape, proliferation and gene expression in full 3D, however this is often technically challenging. 2D planar projections are sometimes an option, however they do not work on highly curved organs. Our lab has developed MorphoGraphX (www.MorphoGraphX.org) a software that bridges this gap by enabling image processing directly on curved surfaces, what we informally refer to as 2.5D images. Many developmental processes happen on surfaces, such as in the epidermal layer of cells in plants or on epithelial layers in animals. Once cells are segmented, they require annotation, it is not just enough to know the positions and shapes of 100s or 1000s of cells, we need to also know where they are in the organism or organ, in order to decipher how they are responding to developmental signals. Organs are thought to be patterned genetically by gradients of morphogens and that determine growth rates and cell and tissue polarity. Not unlike genomic sequence data, which is of little use without annotation, knowledge of the cells' position and polarity within the organ they are developing is key to make sense of the data. Here I will present an array of tools we have developed in MorphoGraphX to annotate cells with positional information, both for 2.5 and full 3D images."
author4: "Albert Do"
inst4: "(University of California, Riverside)"
title4: "Multiscale modeling of the Arabidopsis shoot meristem signaling network"
abstract4: "Growth in plants is coordinated by collections of undifferentiated cell clusters known as meristems. These meristems in turn are coordinated by highly complex regulatory networks. The WUSCHEL (WUS) transcription factor is a key regulator in the shoot apical meristem governing above ground growth. One of WUS‚Äôs most important targets is the CLAVATA3 (CLV3) signaling peptide. WUS and CLV3 have a complex bidirectional relationship both upregulating and repressing each other that does not easily fit within standard regulatory paradigms. To model this relationship, a hybrid system of meristem signaling consisting of deterministic ODE based and stochastic based dynamics was constructed. The ODE portion models protein/RNA dynamics while the stochastic portion models the binding of WUS to the CLV3 gene regulatory region/cis regulatory module (CRM). This deterministic/stochastic model is able to accurately replicate expression patterns seen in experimental data, generate data that fits what is known about the biology in scenarios that have not yet undergone rigorous wetlab analysis, and provides a way to directly observe the dynamics of WUS binding patterns on the CRM."
---
