<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-11 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-11 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-11" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS11/" />
<meta property="og:url" content="http://localhost:4000/MS11/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS11/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-11","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-11">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-11" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-11" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-11</h1>
        <h4>Wednesday, June 16 at 02:15am (PDT)</br>Wednesday, June 16 at 10:15am (BST)</br>Wednesday, June 16 06:15pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS11">Minisymposia-11</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/CBBS-MS11.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS11-CBBS: </small></br>Recent advances in random and deterministic modeling in biology/health sciences</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Maria C.A. Leite, (University of South Florida St.Petersburg), Juan Carlos Cortés López (Instituto Universitario de Matemática Multidisciplinar. Universitat Politècnica de València, Spain), Rafael J. Villanueva Micó (Instituto Universitario de Matemática Multidisciplinar. Universitat Politècnica de València, Spain)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS17-CBBS</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/CBBS-MS11.html#author1"><strong class="card-body-subgroup">Óscar Angulo</strong></a>
		    	 
		    		<em> (Universidad de Valladolid, Spain)</em>
		    	
		    	
		    		
		    			</br>"Numerical  integration of an age-structured model with unbounded age-domain"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The analysis of an unbounded life-span age-structured population model is motivated because, not only new models continue to appear in this framework, but also it is required by the study of the asymptotic behaviour of its dynamics. The numerical integration of the corresponding model is usually performed in bounded domains  through the truncation of the age life-span. Here, we present a new numerical method that avoids the truncation of the unbounded age domain. We completely analyze it and we establish its second order of convergence. We finish with some experiments to exhibit numerically the theoretical results and the behaviour of the problem in the simulation of the evolution of the Nicholson's blowflies model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/CBBS-MS11.html#author2"><strong class="card-body-subgroup">Carla Pinto</strong></a>
		    	 
		    		<em> (School of Engineering, Polytechnic of Porto, Portugal)</em>
		    	
		    	
		    		
		    			</br>"Modified SIQR model for the COVID-19 outbreak"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk we consider a generalization of the Susceptible-Infected-Quarantine-Recovered model, the mSIQR, for the COVID-19 pandemic. The main goal is to study the importance of the value of the contact rate, proportion of unkown infectious, and hospital care in the disease propagation. We test the model and fit the results for COVID-19 pandemic data from some countries, including France, US, and Portugal. We discuss the epidemiological relevance of the results and provide insights on future patterns, subjected to health policies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/CBBS-MS11.html#author3"><strong class="card-body-subgroup">Clara Burgos</strong></a>
		    	 
		    		<em> (Instituto Universitario de Matemática Multidisciplinar. Universitat Politècnica de València, Spain)</em>
		    	
		    	
		    		
		    			</br>"A computational procedure describe breast tumor growth capturing the uncertainty in the volume data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The aim of this talk is to describe a theoretical-computational approach to model the breast tumor growth taking into account the uncertainty of the retrieved data. To do it, we will seek suitable random inputs of a discretized version of a logistic model. The random model parameters will be described via its probability density function.  The theoretical-computational approach seems to be flexible enough to be adapted to describe different biological dynamics problems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/CBBS-MS11.html#author4"><strong class="card-body-subgroup">Gilberto Gonzalez-Parra</strong></a>
		    	 
		    		<em> (Department of Mathematics, New Mexico Tech, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical modeling of COVID-19 pandemic under social behavior uncertainty (Pre-recorded)"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mathematical modeling of COVID-19 pandemic has been challenging due to the complexity of the phenomena including the variability of the social behavior. The uncertainty in some of the mechanisms involved in the transmission of the SARS-CoV-2 and its fatality rate make forecasting an extremely difficult problem as the outcomes have shown. In this talk, we present a mathematical model approach to study the effect of uncertainty in social behavior on the COVID-19 pandemic. Specifically, we rely on stochastic differential equations to give some insights regarding this topic. We illustrate with some scenarios the consequences of social behavior uncertainty on the COVID-19 pandemic. Finally, we will show an application of computational tools such as bootstrapping and Markov chain Monte Carlo that allow us to investigate some uncertainties related to the mathematical modeling of COVID-19 pandemic.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/DDMB-MS11.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS11-DDMB: </small></br>Mathematical Modeling of Protein Dynamics</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Suzanne S. SINDI (University of California, Merced, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/DDMB-MS11.html#author1"><strong class="card-body-subgroup">Erwan HINGANT</strong></a>
		    	 
		    		<em> (Universidad del Bío-Bío, Concepción, CHILE)</em>
		    	
		    	
		    		
		    			</br>"Stochastic nucleation for amyloid diseases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			to be announced
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/DDMB-MS11.html#author2"><strong class="card-body-subgroup">Florence HUBERT</strong></a>
		    	 
		    		<em> (Aix-Marseille Université, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"Growth fragmention models to understand the dynamical instabilities of microtubules"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microtubules (MTs) are dynamic protein polymers that are found in all eukaryotic cells. They are crucial for normal cell development, aiding in many cellular processes, including cell division, cell polarisation, and cell motility .  Because of their role in cell movement and cell division, these polymers are often used as targets for a variety of cancer chemotherapy drugs. Many experimental studies have been completed to understand MT dynamics , and how these dynamics are altered by the addition of MT targeting drugs. However, a complete understanding of such dynamics is lacking, and so the development of new theoretical models to describe MT dynamics is important. We propose in this talk a mathematical model based on growth-fragmentation PDE and investigate the asymptotic behaviour of the solutions
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/DDMB-MS11.html#author3"><strong class="card-body-subgroup">Paul LEMARRE</strong></a>
		    	 
		    		<em> (Université de Lyon, FRANCE)</em>
		    	
		    	
		    		
		    			</br>"OvPrP oligomers - a short story of structural diversity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this presentation we explore the structural diversity of small OvPrP oligomers. These objects formed in vitro exhibit a surprisingly wide variety of structures and organisations. Using various experimental methods, we are able to devise hypotheses regarding the origin of this diversity and the interactions between the different species. In particular, we study a specific mutant of OvPrP, which selectively creates one type of object. We build a kinetic model for the dynamics of these objects, with the goal to reproduce two crucial qualitative features of the experiments: 1) a non-linear and non-monotonous effect of concentration 2) the interaction between multiple timescales.  Novel processes are included in order to obtain this qualitative behaviour, and the importance of structural diversity in the replication of oligomers is evidenced.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/DDMB-MS11.html#author4"><strong class="card-body-subgroup">Stéphanie PORTET</strong></a>
		    	 
		    		<em> (University of Manitoba, CANADA)</em>
		    	
		    	
		    		
		    			</br>"Activation of OAS2 by dsRNA"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The activation of 2'-5'-oligoadenylate synthetase (OAS) enzymes by direct interaction with viral double-stranded RNA (dsRNA) is a key part of the innate immune response to viral infection. A downstream effect of the OAS-dsRNA interaction is to degrade the single-stranded RNA to prevent the spread of the virus. The activation of OAS2, one of the members of the OAS family, depends on dsRNA length. Combining in vitro experiments and mathematical modelling, we test different hypotheses for the OAS2 activation mechanisms by its cofactor dsRNA. After model calibration and selection, the cooperative binding of multiple OAS2 to a single dsRNA is shown to best represent the effect of its cofactor length on enzyme activity. Work from Lee et al. AIMS Mathematics 6: 5924-5941 (2021)
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/EVOP-MS11.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>MS11-EVOP: </small></br>Recent developments in phylogenetic network reconstruction and beyond</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Guillaume Scholz (University of Leipzig, Germany), Katharina Huber (University of East Anglia, United Kingdom)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS17-EVOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/EVOP-MS11.html#author1"><strong class="card-body-subgroup">Magnus Bordewich</strong></a>
		    	 
		    		<em> (Durham University, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Diversity in phylogenetic networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Dating back to 1992, phylogenetic diversity (PD) is a prominent quantitative tool for measuring the biodiversity of a collection of species. This measure is based on the evolutionary distance among the species in the collection. Loosely speaking, if T is a phylogenetic tree whose leaf set X represents a set of species and whose edges have real-valued lengths (weights), then the PD score of a subset S of X is the sum of the weights of the edges of the minimal subtree of T connecting the species in S. In this talk we will discuss recent work on extending this concept from phylogenetic trees to phylogenetic networks and consider the computational complexity of the associated optimisation problems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/EVOP-MS11.html#author2"><strong class="card-body-subgroup">Simone Linz</strong></a>
		    	 
		    		<em> (University of Auckland, New Zealand)</em>
		    	
		    	
		    		
		    			</br>"Superfluous arcs in phylogenetic networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The last 15 years have seen a shift from the reconstruction of phylogenetic trees towards phylogenetic networks. The latter not only capture speciation events but also evolutionary processes such as hybridization and lateral gene transfer that cannot be explained by a single phylogenetic tree. Nevertheless, since the evolutionary history of a single gene or short DNA fragment is, in most cases, correctly described by a tree, the set of phylogenetic trees that are embedded in a network continue to be of recurring interest. For example, to score a phylogenetic network in a maximum parsimony or likelihood framework, one often scores each embedded tree instead of the network directly. In practice this often comes down to scoring a multiset of embedded trees whose size is exponential in the number of reticulations in the network. In this talk, we introduce the notion of a non-essential arc of a phylogenetic network N which is an arc whose deletion from N results in a phylogenetic network N’ whose set of embedded trees is equal to that of N.  We investigate the class of tree-child networks and characterize which arcs are non-essential. This characterization is based on a  family of directed graphs. Moreover, we show that identifying non-essential arcs in a tree-child network takes time that is polynomial in the number of leaves of the network.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/EVOP-MS11.html#author3"><strong class="card-body-subgroup">Kristina Wicke</strong></a>
		    	 
		    		<em> (The Ohio State University, United States of America)</em>
		    	
		    	
		    		
		    			</br>"Linking phylogenetics and classical graph theory: edge-based phylogenetic networks and their relation to GSP graphs"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recently, tree-based phylogenetic networks have attracted considerable attention in the literature. Roughly speaking, these networks can be constructed from a phylogenetic tree by inserting additional edges. However, in general, it is an NP-completeproblem to decide whether an unrooted phylogenetic network is tree-based or not. In this talk, I will introduce a class of unrooted networks, namely edge-based networks, that are necessarily tree-based and can be recognized in linear time. Surprisingly, the class of edge-based networks is closely related to a well-known family of graphs in classical graph theory, the class of generalized series-parallel (GSP) graphs, and I will explore this relationship in full detail.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/EVOP-MS11.html#author4"><strong class="card-body-subgroup">Vincent Moulton</strong></a>
		    	 
		    		<em> (University of East Anglia, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Reconstructibility of unrooted level-k phylogenetic networks from distances"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A phylogenetic network is a graph-theoretical tool that is used by biologists to represent the evolutionary history of a collection of species. One potential way of constructing such networks is via a distance-based approach, where one is asked to find a phylogenetic network that in some way represents a given distance matrix, which gives information on the evolutionary distances between present-day taxa. In this talk, we consider the following question. For which k are unrooted, edge-weighted level-k networks uniquely determined by their distance matrices? We consider this question for shortest distances as well as for the case that the multisets of all distances is given.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/MEPI-MS11.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS11-MEPI: </small></br>Models of COVID-19 Vaccination, Non-Pharmaceutical Interventions, and Relaxation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jane Heffernan (York University, Canada), Miranda Teboh Ewungkem (Lehigh University, USA), Zhilan Feng (Purdue University, USA), John Glasser (Centres for Disease Control, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS16-MEPI</strong>.
					
					
						The third session is <strong>MS20-MEPI</strong>.
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/MEPI-MS11.html#author1"><strong class="card-body-subgroup">Bruce	Mellado</strong></a>
		    	 
		    		<em>(University of Witwatersand, South Africa)</em>
		    	
		    	
		    		
		    			</br>"Modelling the COVID-19 pandemic in South Africa: the role of AI"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			 In this presentation work performed by the Gauteng Province Premier COVID-19 Advisory Committee in data analysis, modelling, predictions and vaccine roll-out straggles. The use of Artificial Intelligence through Machine Learning in devising smart algorithms will be highlighted. The challenges of interfacing advanced analytics with advising policy-makers will also be discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MEPI-MS11.html#author2"><strong class="card-body-subgroup">Ellen Brooks Pollock</strong></a>
		    	 
		    		<em> (University of Bristol, England)</em>
		    	
		    	
		    		
		    			</br>"Mapping social distancing measures to the reproduction number during vaccine rollout"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Background: In the absence of a vaccine, SARS-CoV-2 transmission has been controlled by preventing person-to-person interactions via social distancing measures. As vaccination is rolled out and social distancing restrictions are lifted, policy-makers need to consider how combinations of measures will affect transmission and understand the trade-offs between them. Methods: We use age-specific social contact data collect in the UK in 2010, together with epidemiological data, to quantify the components of the COVID-19 reproduction number. We estimate the impact of social distancing policies and vaccination rollout on the reproduction number in the UK. Results: We demonstrate that pre-collected social contact data can be used to provide a time-varying estimate of the reproduction number (R). Transmission associated with primary schools is predicted to have a smaller impact on community prevalence than transmission in secondary schools. Prioritising older age groups for vaccination leads to modest initial indirect benefits of vaccination. Some levels of contact tracing and COVID security are required until the majority of the adult population are vaccinated. The results can be explored at https://ellenbp.shinyapps.io/reckoners/.  Conclusions: Our approach has been widely used by policy-makers to project the impact of social distancing measures and assess the trade-offs between them. Effective social distancing, contact tracing and COVID-security are required while vaccination is rolled out.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MEPI-MS11.html#author3"><strong class="card-body-subgroup">Nick Golding</strong></a>
		    	 
		    		<em> (Curtin University, Australia)</em>
		    	
		    	
		    		
		    			</br>"Real-time tracking and forecasting of COVID-19 transmission potential in Australia"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Australian COVID-19 pandemic experience has been characterised by long periods of no transmission interspersed with localised, and mostly small outbreaks linked to spillover from quarantine facilities for international arrivals. COVID-19 preparedness and response decision making in Australia has therefore been focused on the potential for outbreaks to take-off and the likely impact of interventions on preventing that. However in the absence of cases, standard models for estimating the reproduction rate of the virus cannot be used. We will detail a novel semi-mechanistic Bayesian statistical model developed to track COVID-19 transmission potential in Australia over time. This quantity can be tracked even in the absence of cases by drawing on mobility data streams, behavioural surveys, and data on health surveillance systems. The effects of lockdowns, varying adherence to hygiene measures, age-structured vaccination roll-out, variants with different transmissibility, and the effectiveness of health surveillance systems are all explicitly considered. This model is able to track the potential rate of transmission in the absence of cases, the realised rate of transmission in the presence of cases, and to move smoothly between these metrics. A particular advantage of this approach for the Australian context is the ability to derive estimates of transmission rates in the very early stages of an outbreak, when numbers of cases are still in the single figures. This model has informed the Australian response to COVID-19 throughout the pandemic - as discussed in a separate talk by Dr Freya Shearer at this conference.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MEPI-MS11.html#author4"><strong class="card-body-subgroup">Wilfred Ndifon</strong></a>
		    	 
		    		<em> (African Institute for the Mathematical Sciences, South Africa)</em>
		    	
		    	
		    		
		    			</br>"Vaccinating to Minimize COVID-19 Morbidity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The morbidity caused by an acute infectious disease like COVID-19 is frequently measured only with respect to the short-term effects of infection. We consider an important longer-term effect, namely the deterioration of immune functioning due to accelerated senescence of pathogen-responsive T cells. Using both mathematics and data, we argue that this type of immune deterioration is negligible in young adults but substantial in older adults. A consideration that emerges in the current context of a limited COVID-19 vaccine supply is how to optimize vaccination in order to minimize such immune deterioration. We show that, compared to alternative vaccination strategies, prioritizing older adults as well as individuals who have an already significantly deteriorated immune functioning is optimal. Because, as we argue using data, the severity of SARS-CoV-2 infections increases with immune deterioration, this vaccination strategy would also save the most lives. Our mathematical framework offers a natural explanation for the higher risk of SARS-CoV-2 infection-induced death that has been observed in men compared to women.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/MFBM-MS11.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS11-MFBM: </small></br>Stochastic Systems Biology: Theory and Simulation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jae Kyoung Kim (Department of Mathematical Sciences, KAIST, Republic of Korea), Ramon Grima (University of Edinburgh, United Kingdom)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/MFBM-MS11.html#author1"><strong class="card-body-subgroup">Zhixing Gao</strong></a>
		    	 
		    		<em> (Key Laboratory of Advanced Control and Optimization for Chemical Processes, Ministry of Education, East China University of Science and Technology, China)</em>
		    	
		    	
		    		
		    			</br>"Neural network aided approximation and parameter inference of stochastic models of gene expression"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Non-Markov models of stochastic biochemical kinetics often incorporate explicit time delays to effectively model large numbers of intermediate biochemical processes. Analysis and simulation of these models, as well as the inference of their parameters from data, are fraught with difficulties because the dynamics depends on the system’s history. Here we use an artificial neural network to approximate the time-dependent distributions of non-Markov models by the solutions of much simpler time-inhomogeneous Markov models; the approximation does not increase the dimensionality of the model and simultaneously leads to inference of the kinetic parameters. The training of the neural network uses a relatively small set of noisy measurements generated by experimental data or stochastic simulations of the non-Markov model. We show using a variety of models, where the delays stem from transcriptional processes and feedback control, that the Markov models learnt by the neural network accurately reflect the stochastic dynamics across parameter space.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MFBM-MS11.html#author2"><strong class="card-body-subgroup">Abhyudai Singh</strong></a>
		    	 
		    		<em> (University of Delaware, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling stochasticity in timing of intracellular events: A first-passage time approach"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			 How the noisy expression of regulatory proteins affects timing of intracellular events is an intriguing fundamental problem that influences diverse cellular processes. Here we use the bacteriophage lambda to study event timing in individual cells where cell lysis is the result of expression and accumulation of a single protein (holin) in the Escherichia coli cell membrane up to a critical threshold level. Site-directed mutagenesis of the holin gene generated phage variants that vary in their lysis times from 30 to 190 min. Observation of the lysis times of single cells reveals an intriguing finding—the noise in lysis timing first decreases with increasing lysis time to reach a minimum and then sharply increases at longer lysis times. A mathematical model with stochastic expression of holin together with dilution from cell growth was sufficient to explain the non-monotonic noise profile and identify holin accumulation thresholds that generate precision in lysis timing.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MFBM-MS11.html#author3"><strong class="card-body-subgroup">Thomas Prescott</strong></a>
		    	 
		    		<em> (Alan Turing Institute, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Learning a multifidelity simulation strategy for likelihood-free Bayesian inference."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Likelihood-free Bayesian inference is a popular approach to calibrating complex mathematical models typical of biological systems, where likelihoods are often intractable. However, being reliant on repeated model simulation, the complexity that prohibits the likelihood calculation can also cause these methods to suffer from a significant computational burden. Multifidelity inference methods have been shown to reduce this burden by exploiting approximate simulations, such as coarser numerics or lower-dimensional models. By incorporating both high- and low-fidelity simulations, computational savings can be achieved without introducing any further bias in the resulting likelihood-free posterior. Instead, these approaches are forced to trade between reducing computational burden and increasing estimator variance. This trade-off is balanced by optimally assigning a simulation budget between the models at different fidelities. We will discuss how the optimal multifidelity simulation strategy can be learned in parallel with the posterior, and the multifidelity algorithm thus adaptively tuned as the posterior is uncovered.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/MFBM-MS11.html#author4"><strong class="card-body-subgroup">Ruben Perez-Carrasco</strong></a>
		    	 
		    		<em> ( Imperial College London, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Should we care about cell cycle variability when studying stochastic gene expression?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many models of stochastic gene expression do not incorporate a cell cycle description. I will show how this can be tackled analytically studying how mRNA fluctuations are influenced by DNA replication for a prescribed cell cycle duration stochasticity. Results show that omitting cell cycle details can introduce significant errors in the predicted mean and variance of gene expression for prokaryotic and eukaryotic organisms, reaching 25% error in the variance for mouse fibroblasts. Furthermore, we can derive a negative binomial approximation to the mRNA distribution, indicating that cell cycle stochasticity introduces similar fluctuations to bursty transcription. Finally, I will show how disregarding cell cycle stochasticity can introduce inference errors in transcription rates bigger than 10%.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/NEUR-MS11.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS11-NEUR: </small></br>The Control of the Cardiovascular System in Health and Disease</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Mette Olufsen (North Carolina State University, USA), Brian Carlson (University of Michigan, USA), Justen Geddes (North Carolina State University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-NEUR</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/NEUR-MS11.html#author1"><strong class="card-body-subgroup">Leszek Pstras</strong></a>
		    	 
		    		<em> (Nalecz Institute of Biocybernetics and Biomedical Engineering, Polish Academy of Sciences, Warsaw, Poland)</em>
		    	
		    	
		    		
		    			</br>"Blood Volume Regulation During Hemodialysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Hemodialysis is the most common renal replacement therapy allowing for the removal of excess body water and waste products of metabolism in patients with chronic or acute kidney failure. Unfortunately, this life-sustaining treatment often puts a lot of strain on the cardiovascular system. During a typical dialysis session, a few liters of fluid are gradually removed from the blood flowing through the dialyzer, which usually leads to a progressive decrease in blood volume. The proper function of the cardiovascular system relies then on the microvascular absorption of fluid from tissues (the vascular refilling mechanism) and the activity of blood pressure regulatory mechanisms. In many dialysis patients, however, these mechanisms work insufficiently, leading to a drop in blood pressure and accompanying symptoms. Intradialytic hypotension is a multi-factorial and still not fully understood phenomenon and remains the major issue in dialysis units for both patients and staff. In this talk, a lumped-parameter mathematical model of the cardiovascular response to hemodialysis will be presented to discuss the cardiovascular regulatory mechanisms focusing on the mechanical aspects of blood volume regulation at the level of microcirculation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/NEUR-MS11.html#author2"><strong class="card-body-subgroup">Nikolai L. Bjørdalsbakke</strong></a>
		    	 
		    		<em> (Norwegian University of Science and Technology, Trondheim, Norway)</em>
		    	
		    	
		    		
		    			</br>"Is Non-Invasive Finger Pressure Informative About Cardiovascular Adaptation to Physical Activity Interventions?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Non-invasive finger pressure is a simple and flexible method to continuously record blood pressure waves in various clinical settings; however, the waveform is substantially different from the central aortic blood pressure waveform, and further the systolic and diastolic values may differ from hypertension research's gold standard of 24 hour ambulatory brachial-cuff measured systolic and diastolic blood pressure. We present preliminary analysis of non-invasive finger pressure before and after physical activity intervention in pre-hypertensive and hypertensive adults. Additionally, we present the results of interpreting these blood pressure data through both mechanistic and data driven models as a means to quantify cardiovascular adaptations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/NEUR-MS11.html#author3"><strong class="card-body-subgroup">Justen Geddes</strong></a>
		    	 
		    		<em> (North Carolina State University, Raleigh, NC, USA)</em>
		    	
		    	
		    		
		    			</br>"Cardiovascular Regulation in POTS Patients"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Postural Orthostatic Tachycardia Syndrome (POTS) is characterized by an excessive increase in heart rate upon an upright postural change along with the presence of orthostatic symptoms possibly caused by an overactive baroreflex control system. The main contributor to heart-rate control is the baroreflex control system, a negative feedback system operating at a resonance frequency of ~0.1 Hz. Another sign that the control system is overactive is that POTS patients exhibit larger low-frequency (~0.1 Hz) heart rate and blood pressure oscillations than controls. In this talk we use signal processing to demonstrate the presence of 0.1 Hz oscillations and we build a cardiovascular systems model predicting baroreflex control of heart rate and blood pressure for controls and POTS patients. The cardiovascular model uses an RC electrical circuit analogy while the baroreflex control is modeled using first-order equations controlling heart rate, vascular resistance, and cardiac contractility ensuring saturation at high and low pressure. Our model is able to predict amplified low-frequency oscillations observed in POTS patient heart rate and blood pressure data both at rest and during head-up tilt, and we show how control changes for the various hypothesized causes of POTS known as endophenotypes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/NEUR-MS11.html#author4"><strong class="card-body-subgroup">Feng Gu</strong></a>
		    	 
		    		<em> (University of Michigan, Ann Arbor, MI and Xiangya Hospital, Central South University, Changsha, Hunan, China, USA)</em>
		    	
		    	
		    		
		    			</br>"Probing the Potential Role of Intermittent Functioning of Baroreflexes in the Etiology of Hypertension Using an Integrated Computational and Experimental Approach"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The potential role of the baroreflexes in long‐term arterial blood pressure (BP) regulation and in the etiology of primary hypertension have been long debated. To elucidate the potential mechanisms underlying the pathophysiology of primary hypertension, we analyzed dynamic baroreflex responses to spontaneous fluctuations in arterial pressure in the conscious spontaneously hypertensive rat (SHR), the most widely used genetic rat model of primary hypertension, as well as in the Wistar-Kyoto (WKY), the Dahl salt-sensitive, the Dahl salt-resistant, and the Sprague-Dawley rat. Observations revealed the existence of long intermittent periods (lasting up to several minutes) of continuous engagement and disengagement of baroreflex-mediated control of heart rate. Analysis of these intermittent periods reveals a predictive relationship between increased mean arterial pressure and progressive baroreflex disengagement that is present in the SHR and WKY strains but absent in others. To further investigate the mechanism underlying the intermittent baroreflexes engagement/disengagement, video was recorded during measurements of spontaneous dynamic baroreflex function in animals. Preliminary results indicate a positive correlation between baroreflex disengagement periods and activity levels. Measurements on human subjects reveal that, rather than the relationship between BP and progressive baroreflex disengagement observed in the rat models, baroreflex disengagement is more closely associated with arterial blood pH, suggesting a competitive interaction between baroreflex- and chemoreflex-mediated regulation of autonomic function. Based on these observations we hypothesize that chemoreflex-mediated sympathetic outflow can override baroreflex-derived parasympathetic outflow causing an apparent intermittent functioning of the baroreflex and potentially leading to hyper-sympathetic activity. We further hypothesize that in the SHR rat this intermittent chemoreflex-mediated override of the baroreflex contributes chronically elevated BP.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS11/ONCO-MS11.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS11-ONCO: </small></br>Mathematical Oncology: From methodological studies to clinical applications</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Saskia Haupt (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Vincent Heuveline (Engineering Mathematics and Computing Lab (EMCL), Interdisciplinary Center for Scientific Computing (IWR), Heidelberg University, Germany), Matthias Kloor (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Germany)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS11/ONCO-MS11.html#author1"><strong class="card-body-subgroup">Calum Gabbutt</strong></a>
		    	 
		    		<em> (Centre for Genomics and Computational Biology, Barts Cancer Institute, Queen Mary University of London, London, United Kingdom (UK))</em>
		    	
		    	
		    		
		    			</br>"Reconstructing Contemporary Human Stem Cell Dynamics with Oscillatory Molecular Clocks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell histories can be reconstructed from their genomes by analysing ‘molecular clocks’ that accumulate heritable changes through time. Commonly used clocks, such as the accumulation of single nucleotide variants, change slowly over decades, recording cell dynamics that occur at the longer timescale of the change accumulation rate. Studies within mouse have revealed that normal colon epithelium is maintained by a pool of multipotent stem-cells which undergo neutral competition and inevitably drift towards monoclonality. Here we develop a new method that can measure contemporary human adult cell dynamics with rapidly oscillating CpG DNA methylation.
Ongoing (de)methylation causes switching between 0, 50 and 100% methylation at each CpG locus in a diploid cell – the clock ‘tick-tocks’ back-and-forth like a pendulum. In polyclonal cell populations, these oscillator states are unsynchronized between cells, hence the average oscillator methylation is randomly distributed about 50%. However, any clonal expansion will synchronize the oscillator clocks resulting in clonal populations that have characteristic “W-shaped” distributions (methylation peaks at 0, 50 and 100%), approximating the methylation of the progenitor cell. The precise shape of the W-distribution is determined by the underlying dynamics of cell growth and replacement. We show how to identify appropriate oscillators from standard methylation array data (Illumina EPIC) and develop a mathematical modelling framework to quantitatively measure stem cell dynamics from these data. We apply our method to measure stem cell dynamics in individual human intestinal crypt and endometrial gland populations, and test whether these tissues have different stem cell dynamics using a hierarchical Bayesian model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/ONCO-MS11.html#author2"><strong class="card-body-subgroup">Toni Seppälä</strong></a>
		    	 
		    		<em> (Helsinki University Hospital and University of Helsinki, Finland)</em>
		    	
		    	
		    		
		    			</br>"Organoids and cell-free DNA in cancer precision medicine"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is generally believed that earlier diagnosis of a cancer recurrence might improve the outcome. Postoperative minimal residual disease (MRD) very deterministically predicts future recurrence after curative surgery, but the preliminary evidence suggests that the prognosis of a recurring cancer may be improved by timely chemotherapy. Patients are always followed up for years to detect cancer recurrences using clinical examinations and blood tests that are not optimal by sensitivity or specificity. In case of a recurrence seen in imaging, chemotherapy is usually initiated. Selection of chemotherapy regimen between multiple options is usually based on expected tolerability of toxicity and failure of earlier choices.
Tools to aid decision-making in these challenging clinical situations are required, and precision medicine holds great promise in delivering for the unmet need. Applications detecting bloodstream cell-free DNA have been developed to support diagnostics of MRD. Patient-derived organoid technology enables individualized cell culture from each tumor. Organoids may serve as a clinical tool to guide traditional primary tumor NGS, and facilitate in vitro response prediction to therapy. Data-intensive models for tumor microenvironment co-culture and combination pharmacotyping are needed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/ONCO-MS11.html#author3"><strong class="card-body-subgroup">Vincent Jonchere</strong></a>
		    	 
		    		<em> (INSERM Sorbonne Université, UMRS 938, Équipe Instabilité des Microsatellites et Cancer, Équipe Labellisée par la Ligue Nationale Contre le Cancer et SIRIC, France)</em>
		    	
		    	
		    		
		    			</br>"Identification of Positively and Negatively Selected Driver Gene Mutations Associated With Colorectal Cancer With Microsatellite Instability"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Background & Aims Recent studies have shown that cancers arise as a result of the positive selection of driver somatic events in tumor DNA, with negative selection playing only a minor role, if any. However, these investigations were concerned with alterations at nonrepetitive sequences and did not take into account mutations in repetitive sequences that have very high pathophysiological relevance in the tumors showing microsatellite instability (MSI) resulting from mismatch repair deficiency investigated in the present study.
Methods We performed whole-exome sequencing of 47 MSI colorectal cancers (CRCs) and confirmed results in an independent cohort of 53 MSI CRCs. We used a probabilistic model of mutational events within microsatellites, while adapting pre-existing models to analyze nonrepetitive DNA sequences. Negatively selected coding alterations in MSI CRCs were investigated for their functional and clinical impact in CRC cell lines and in a third cohort of 164 MSI CRC patients.
Results Both positive and negative selection of somatic mutations in DNA repeats was observed, leading us to identify the expected true driver genes associated with the MSI-driven tumorigenic process. Several coding negatively selected MSI-related mutational events (n = 5) were shown to have deleterious effects on tumor cells. In the tumors in which deleterious MSI mutations were observed despite the negative selection, they were associated with worse survival in MSI CRC patients (hazard ratio, 3; 95% CI, 1.1–7.9; P = .03), suggesting their anticancer impact should be offset by other as yet unknown oncogenic processes that contribute to a poor prognosis.
Conclusions The present results identify the positive and negative driver somatic mutations acting in MSI-driven tumorigenesis, suggesting that genomic instability in MSI CRC plays a dual role in achieving tumor cell transformation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS11/ONCO-MS11.html#author4"><strong class="card-body-subgroup">Johannes Witt</strong></a>
		    	 
		    		<em> (Department of Applied Tumor Biology (ATB), Institute of Pathology, University Hospital Heidelberg, Heidelberg, Germany)</em>
		    	
		    	
		    		
		    			</br>"Analyzing the influence of HLA class I genotype on cancer immunoediting"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Already in early stages of tumorigenesis, transformed cells are recognized and attacked by the immune system, leading to the elimination of precancerous cell clones. This process depends on the generation of neoantigens, which determine the immunogenicity of tumor cells. Highly immunogenic cancer cells are counterselected during tumor evolution, constituting a Darwinian selection process. In microsatellite-unstable (MSI) cancer, a high load of neoantigens accumulates due to frameshift mutations in coding microsatellites. The immunogenicity of frameshift peptides (FSP) depends on the presentation of cleaved peptides on the cell surface by human leucocyte antigen (HLA) molecules. Endogenously produced peptides are preferentially presented by HLA class I molecules. Among the HLA class I genes, HLA-A, -B and -C play the most prominent role in the immune response. Due to amino acid substitutions in the peptide-binding region, each HLA molecule is characterized by a specific repertoire of peptides that can be presented. Analyzing a single nucleotide polymorphism at the 5’-end of exon 2 of the HLA-A gene, we divided a set of 75 MSI colorectal cancer samples into two groups: samples possessing at least one HLA-A*02 allele and samples without any HLA-A*02 allele. For both constellations, we developed scores estimating the probability that at least one FSP-derived peptide is presented on the cell surface by a HLA-A molecule (OLLA,G2, OLLA,GN). We observed an inverse correlation between the predicted immunogenicity of 41 FSP and the mutation frequency, which may reflect a selection pressure exerted by the immune system. However, this correlation is not group-specific, indicating that the immunogenicity of FSP is potentially not only determined by the HLA type. With increasing length l of FSP, the number of possible FSP-derived peptides increases. Thus, the likelihood rises that at least one of them fits a random HLA molecule. For both the HLA-A*02 and the Non-HLA-A*02 group, we observed a negative correlation between l and the mutation frequency. Our predictions imply that HLA diversity may determine the likelihood of FSP recognition and therefore immune recognition of tumor cells.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-11
										</br>
										Wednesday, June 16 at 02:15am (PDT)</br>Wednesday, June 16 at 10:15am (BST)</br>Wednesday, June 16 06:15pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>













    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
