---
layout: group
author: esmb
day: "Thursday (Friday)"
subgroup: "CDEV"
title: "Dynamics and networks in single-cell biology"
organizers: "Adam Maclean (Univeristy of Southern California) & Russell Rockne (City of Hope, USA)"
description: "This minisymposium will discuss current mathematical and theoretical approaches to address open questions in single-cell biology. As single-cell genomics technologies advance, computational data analysis becomes one of the greatest hurdles to biological discovery. As the field begins to mature, and standards slowly emerge, the most pressing mathematical challenges shift from core tasks -- such as normalization and clustering -- to higher-level tasks. New advances in several areas will be presented in this minisymposium, including: network inference, dynamical systems approaches, modeling across scales, spatial transcriptomics, and multi-modal data integration."
code: "MS19"
code2: "MS20"
author1: "Stephanie Hicks"
inst1: " (Johns Hopkins University, USA)"
title1: "Scalable statistical methods and software for single-cell data science"
abstract1: "Single-cell RNA-Seq (scRNA-seq) is the most widely used high-throughput technology to measure genome-wide gene expression at the single-cell level. However, single-cell data present unique challenges that have required the development of specialized methods and software infrastructure to successfully derive biological insights. Compared to bulk RNA-seq, there is an increased scale of the number of observations (or cells) that are measured and there is increased sparsity of the data, or fraction of observed zeros. Furthermore, as single-cell technologies mature, the increasing complexity and volume of data require fundamental changes in data access, management, and infrastructure alongside specialized statistical methods to facilitate scalable analyses. I will discuss some challenges in the analysis of scRNA-seq data and present some solutions that we have made towards addressing these challenges."
author2: "Geoffrey Schiebinger"
inst2: " (University of British Colombia, Canada)"
title2: "Towards a mathematical theory of trajectory inference"
abstract2: "This talk develops a rigorous mathematical framework for trajectory inference. We examine the problem of recovering temporal couplings of stochastic processes, motivated by applications in developmental biology and cellular reprogramming. We develop methodology based on optimal transport and test it on data from stem cell reprogramming, sea urchin embryonic development, arabidopsis root growth, and hematopoeisis. We then perform a theoretical analysis and establish rigorous guarantees. Our approach provides a rigorous, general framework for investigating cellular differentiation, and poses some interesting questions in probability, statistics and optimization."
author3: "Gioele La Manno"
inst3: " (Swiss Federal Institute of Technology in Lausanne, Switzerland)"
title3: "Revealing the brain’s molecular anatomy with single-cell and tomography-based spatial transcriptomics"
abstract3: "I will present our comprehensive single-cell transcriptome atlas of mouse brain development spanning from gastrulation to birth. In this atlasing effort, we identified almost a thousand distinct cellular states, including the initial emergence of the neuroepithelium, different glioblasts, and a rich set of region-specific secondary organizers that we localize spatially. In this context, I will provide an example of how the  spatially-resolved transcriptomic data can be particularly useful to interpret the complexity of such complex atlases.

Continuing in this direction, I will show the approach that we recently proposed as a general way to spatially resolve different types of next-generation sequencing data.  We designed an imaging-free framework to localize high throughput readouts within a tissue by combining compressive sampling and image reconstruction. Our first implementation of this framework transformed a low-input RNA sequencing protocol into an imaging-free spatial transcriptomics technique (STRP-seq).

Finally, I will showcase the technique with the profiling of the brain of the Australian bearded dragon Pogona vitticeps. With this analysis, we revealed the molecular anatomy of the telencephalon of this lizard and provided evidence for a marked regionalization of the reptilian pallium and subpallium."
author4: "Kenji Kamimoto"
inst4: " (Washington University in St. Louis, USA)"
title4: "CellOracle: Dissecting cell identity via network inference and in silico gene perturbation"
abstract4: "Recent technological advances in single-cell sequencing enable the acquisition of multi-dimensional data in a high-throughput manner. These technologies reveal the existence of heterogeneity and the diversity of cell states and identities. To reveal the regulatory mechanism underlying such phenomena, many computational Gene regulatory Network (GRN) inference methods have been proposed. However, understanding biological events from a GRN perspective remains difficult. Even if a computational algorithm can infer GRN, the biological network is so complex that it is challenging to understand how it systematically dictates cell identities. There is significant demand for new methodologies that bridge the gap between cellular phenotypes and the underlying GRN. Thus, we have developed a new method, CellOracle, a new computational approach for the inference and analysis of GRN. By utilizing machine learning algorithms and genetic information, CellOracle infers sample-specific GRN configurations from single-cell RNA-seq and ATAC-seq data. Our GRN models are designed to be used for the simulation of cell identity changes in response to gene perturbation. This simulation enables network configurations to be interrogated in relation to cell-fate regulation, facilitating their interpretation. To validate CellOracle’s GRN inference method, we present benchmarking on various tissues and cell-types. We also validate the efficacy of CellOracle to recapitulate known outcomes of well-characterized gene perturbations in developmental processes, including mouse hematopoiesis and zebrafish embryogenesis. Our benchmarking and validation results demonstrate the efficacy of CellOracle to infer and interpret the dynamics of GRN configurations, promoting new mechanistic insights into the regulation of cell identity."
---
