<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-06 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-06 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-06" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS06/" />
<meta property="og:url" content="http://localhost:4000/MS06/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS06/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-06","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-06">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-06" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-06" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-06</h1>
        <h4>Tuesday, June 15 at 04:15am (PDT)</br>Tuesday, June 15 at 12:15pm (BST)</br>Tuesday, June 15 08:15pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS06">Minisymposia-06</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/CBBS-MS06.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS06-CBBS: </small></br>Exploring the processes of bacteria self-organization using mathematical modelling and experimental studies</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Diane Peurichard (Inria Paris, France), Marie Doumic (Inria Paris, France)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/CBBS-MS06.html#author1"><strong class="card-body-subgroup">Nicolas Desprat</strong></a>
		    	 
		    		<em> (Laboratoire de Physique de l’École normale supérieure, ENS, Université PSL, CNRS, Sorbonne Université, Université de Paris, 75005 Paris, France)</em>
		    	
		    	
		    		
		    			</br>"Mutliscale morphogenesis of bacterial microcolonies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Unicellular microorganisms are unicellular in the sense, that each individual is able to establish a new population. However, populations of microorganisms are not limited to a collection of individuals, but are highly organized so that the group can perform better than the sum of its individuals. In this presentation, we'll explore how the  asymmetric distribution of adhesins on single rod-shaped bacteria shapes the organization of the group and how this affects higher level functions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CBBS-MS06.html#author2"><strong class="card-body-subgroup">Sophie Hecht</strong></a>
		    	 
		    		<em> (Inria Paris, France)</em>
		    	
		    	
		    		
		    			</br>"On the modelling of the morphogenesis of rod-shaped bacteria micro-colony."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bacteria are abundant organisms whose roles are included in many processes such as medicine, agriculture, ecology, industry... From a single organism, they quickly develop into organised micro-colonies and biofilm structures. The formation of these microcolonies, while broadly studied in the past decade, is still poorly understood.  We consider an individual-based model where each bacterium is modelled by a spherocylinder and bacteria interact only through non-overlapping constraints.  Introducing asymmetric friction and mass for the bacterium, which are taking into account the asymmetry of the pole of the bacteria, we retrieve mechanical behaviours of micro-colony growth, this without implementing attraction or adhesion. We compare our model to various sets of experiments, discuss our results, and propose several quantifiers to compare model to data in a systematic way.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CBBS-MS06.html#author3"><strong class="card-body-subgroup">Laura Kanzler</strong></a>
		    	 
		    		<em> (Laboratoire Jacques-Louis Lions, Sorbonne Université, France)</em>
		    	
		    	
		    		
		    			</br>"Kinetic Modelling of Myxobacteria"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Myxobacteria are rod-shaped, social bacteria that are able to move on flat surfaces by ’gliding’ and form a fascinating example of how simple cell-cell interaction rules can lead to emergent, collective behavior. Observed movement patterns of individual bacteria in such a colony include straight runs with approximately constant velocity, alignment interactions and velocity reversals.  Experimental evidence shows that above mentioned behavior is a consequence of direct cell-contact interaction rather than diffusion of chemical signals, which indicates the suitability of kinetic modeling. In this talk a new kinetic model of Boltzmann-type for such colonies of myxobacteria will be introduced and investigated. For the spatially homogeneous case an existence and uniqueness result will be shown, as well as exponential decay to an equilibrium for the Maxwellian collision operator. The methods used for the analysis combine several tools from kinetic theory, entropy methods as well as optimal transport. The talk will be concluded with numerical simulations confirming the analytical results.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CBBS-MS06.html#author4"><strong class="card-body-subgroup">Marc Hoffmann</strong></a>
		    	 
		    		<em> (INRIA, Mamba team & University Paris-Dauphine, France)</em>
		    	
		    	
		    		
		    			</br>"Statistical estimation of the interaction kernel in McKean-Vlasov model in a mean-field limit"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider the problem of detecting or estimating the interaction in a large system of particles over a fixed time horizon. The particles are subject to a common external force and diffusion, and they interact via a smooth interaction kernel in a mean-field sense, and possibly via a common noise term. We identify some properties of the model that enables one to identify the presence of interactions, in a large population limit, from a statistical perspective.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/CDEV-MS06.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS06-CDEV: </small></br>Computational models of extracellular matrix effects on cell migration and tissue formation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Magdalena Stolarska (University of St. Thomas, United States), Lisanne Rens (Delft University of Technology, Netherlands)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-CDEV</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/CDEV-MS06.html#author1"><strong class="card-body-subgroup">Leonie van Steijn</strong></a>
		    	 
		    		<em> (Leiden University, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Obstacle-induced contact-inihibition of locomotion explains topotactic cell navigation in dense microenvironments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During biological development, cancer metastasis and in the immune system, cells navigate through dense environments filled with obstacles such as other cells and the extracellular matrix. Recently, the term `topotaxis' has been introduced for the navigation of cells along topographic cues such as density gradients of obstacles. As a model of amoeboid cell motility through pores in the ECM, we study the motility of Dictyostelium discoideum cells on a substrate covered with microscopic pillars. The pillars are spaced widely enough to let the cells through and there is a gradient from densely packed pillars to more widely spaced pillars. The D. discoideum cells perform a random walk with a bias towards the more widely spaced area. A previous model based on active Brownian particles (ABP) has shown that ABPs perform topotaxis in a persistence-driven manner. However, the predicted drift is lower than measured experimentally. Here, we use a Cellular Potts model to how cell persistence mode affects topotaxis using the actin-derived persistence of the Act-model [1] and an active Brownian particle-based persistence [2]. Both persistence modes predict topotaxis, but the actin-based persistent cells show a more efficient drift.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CDEV-MS06.html#author2"><strong class="card-body-subgroup">Lisanne Rens</strong></a>
		    	 
		    		<em> (Delft University of Technology, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Computational models for feedback between cell shape, cell signaling and extracellular matrix"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell shape changes and cell migration in mammalian cells are regulated by many sig- naling proteins within the cell. Cells also interact with a meshwork of protein fibers, called the extracellular matrix (ECM), that affects signaling proteins that regulate cell motility, Rac and Rho. The feedback between Rac-Rho-ECM affects the invasiveness of melanoma cancer cells. In our models, we expand on a previous 2-compartment model (coupled ODEs in [3] and [1]) that describes Rac-Rho mutual inhibition, self-activation, the effect of each protein on the amount of contact with the ECM, and ECM activation of Rho [4]. We explore the effects of slip and catch-bond dynamics [2] for the assembly of cell-ECM adhesion. We study the full spatial dynamics in 1D and in static 2D domains, demon- strating oscillations and static/dynamic waves. These results give insight into how distinct types of cell migration emerge. By simulating the set of PDEs in a fully deformable 2D cell using a Cellular Potts model, we predict how spatially distributed signaling is coupled to cell motility. Predicted cell shapes and behavior resemble experimental observations. This full 2D model reveals how ECM anisotropy, cell stiffness, and other cell parameters affect cell migration, leading to experimentally testable predictions. Our computational models suggests insights into how the invasiveness of melanoma cells is regulated.
References [1] William R Holmes, JinSeok Park, Andre Levchenko, and Leah Edelstein-Keshet. A mathematical model coupling polarity signaling to cell adhesion explains diverse cell migration patterns. PLoS computational biology, 13(5):e1005524, 2017.
[2] Elizaveta A Novikova and Cornelis Storm. Contractile fibers and catch-bond clusters: A biological force sensor? Biophys. J., 105(6):1336–1345, 2013.
[3] JinSeok Park, William R Holmes, Sung Hoon Lee, Hong-Nam Kim, Deok-Ho Kim, Moon Kyu Kwak, Chiaochun Joanne Wang, Leah Edelstein-Keshet, and Andre Levchenko. Mechanochemical feedback underlies coexistence of qualitatively distinct cell polarity patterns within diverse cell populations. Proceedings of the National Academy of Sciences, 114(28):E5750–E5759, 2017.
[4] Elisabeth G. Rens and Leah Edelstein-Keshet. Cellular tango: How extracellular matrix adhesion choreographs rac-rho signaling and cell movement, 2021.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CDEV-MS06.html#author3"><strong class="card-body-subgroup">Magda Stolarska</strong></a>
		    	 
		    		<em> (University of St. Thomas, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling the effects of membrane mechanics on cell-substrate interaction during spreading"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It has been well established that the mechanical stiffness of the substrate with which cells interact affects various intracellular processes, including cell spread areas, speeds at which motile cells translocate, and the number and strength of cell-substrate adhesions.   This mechanosensitivity is modulated through conformational changes in cell-substrate adhesion proteins that in turn regulate downstream processes, including those associated with the cell membrane (Kalappurakkal et al., Cell, 2019).  Membrane dynamics, including unfolding and exocytosis from intracellular reservoirs to the lipid bilayer, is necessary for large changes in cell shape, which occur during cell spreading and motility (Figard & Sokac, BioArchitecture, 2014) and for the release of membrane tension that occurs during these shape changes (Pontes et al., J Cell Bio, 2017).  The aim of this work is to understand how membrane dynamics affects the mechanics of cell spreading. To do this, we model the cell as viscous material surrounded by a viscoelastic, actively deforming membrane.  The model also incorporates stress-dependent focal adhesion dynamics and their effect on actin polymerization and myosin contractility.  By using the finite element method to simulate cell spreading in an axisymmetric geometry, we show that the membrane plays a critical role in controlling focal adhesions and in balancing protrusive activity and actin retrograde flow. This balance of protrusive activity not only recapitulate the three phases of cell spreading dynamics described in Gianonne et al. (Cell, 2004), but also plays a critical role in modulating the dependence of total amounts of adhesion proteins and cell spread areas on substrate stiffness.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/CDEV-MS06.html#author4"><strong class="card-body-subgroup">Wanda Strychalski</strong></a>
		    	 
		    		<em> (Case Western Reserve University, United States)</em>
		    	
		    	
		    		
		    			</br>"Computational estimates of mechanical constraints on cell migration through the extracellular matrix"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cell migration through a three-dimensional (3D) extracellular matrix (ECM) underlies important physiological phenomena and is based on a variety of mechanical strategies depending on the cell type and the properties of the ECM. Using computational simulations, we investigate two such migration mechanisms: 'push-pull' (forming a finger-like protrusion, adhering to an ECM node, and pulling the cell body forward) and 'rear-squeezing' (pushing the cell body through the ECM by contracting the cell cortex and ECM at the cell rear). We present a computational model that accounts for both elastic deformation and forces of the ECM, an active cell cortex and nucleus, and for hydrodynamic forces and flow of the extracellular fluid, cytoplasm, and nucleoplasm. The model is formulated using the method of regularized Stokeslets to simulate fluid-structure interactions. We find that relations between three mechanical parameters, the cortex's contractile force, nuclear elasticity, and ECM rigidity, determine the effectiveness of cell migration through the dense ECM. The cell can migrate persistently even if its cortical contraction cannot deform a near-rigid ECM, but then the contraction of the cortex has to be able to sufficiently deform the nucleus. The cell can also migrate even if it fails to deform a stiff nucleus, but then it has to be able to sufficiently deform the ECM. Simulations show the rear-squeezing mechanism of motility results in more robust migration with larger cell displacements than those with the push-pull mechanism over a range of parameter values. Additionally, results show that the rear-squeezing mechanism is aided by hydrodynamics through a pressure gradient.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/DDMB-MS06.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS06-DDMB: </small></br>Advances in deterministic models of biochemical interaction networks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Elisenda Feliu (University of Copenhagen, Denmark), Casian Pantea (West Virginia University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/DDMB-MS06.html#author1"><strong class="card-body-subgroup">Balazs Boros</strong></a>
		    	 
		    		<em> (University of Vienna, Austria)</em>
		    	
		    	
		    		
		    			</br>"Oscillations in deficiency-one mass-action systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Whereas the positive equilibrium of a mass-action system with deficiency zero is always globally stable, for deficiency-one networks there are many different scenarios, mainly involving oscillatory behaviour. We present examples with centers or multiple limit cycles.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/DDMB-MS06.html#author2"><strong class="card-body-subgroup">Beatriz Pascual Escudero</strong></a>
		    	 
		    		<em> (University of Copenhagen, Denmark)</em>
		    	
		    	
		    		
		    			</br>"Detecting concentration robustness in Reaction Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A biological system has absolute concentration robustness (ACR) for some species if the concentration of this species is identical at any possible equilibrium that the network admits. In particular, this concentration must be independent of the initial conditions. While some classes of networks with ACR have been described, as well as some techniques to check ACR for a given network, finding networks with this property is a difficult task in general.
The connection of this global version of robustness with other local notions leads to a practical test on necessary conditions for ACR, by means of algebraic-geometric techniques. This test allows to analyze networks in the search for the possibility of ACR or local ACR for some values of the reaction rates, or discard it for all values. This is based on joint work with E. Feliu.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/DDMB-MS06.html#author3"><strong class="card-body-subgroup">Alan Rendall</strong></a>
		    	 
		    		<em> (Johannes Gutenberg University, Mainz, Germany)</em>
		    	
		    	
		    		
		    			</br>"Global convergence to steady states in a model for the in-host dynamics of hepatitis C"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider a model for the concentration of hepatitis C virus particles in a host which includes a simple description of the virus replication. This model has two virus-free steady states and two corresponding basic reproduction numbers. It has at most three positive steady states. Although it is not known whether there can be more than one steady state we prove that for certain ranges of the parameters every solution converges to a steady state. This is accomplished by applying a method of Li and Muldowney which uses the Lozinskii measure corresponding to a certain norm. An estimate for this Lozinskii measure of the second additive compound of the Jacobian matrix is the key condition which is required. The central idea of the method is to exclude all other kinds of asymptotic behaviour, such as convergence to a periodic solution.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/DDMB-MS06.html#author4"><strong class="card-body-subgroup">Murad Banaji</strong></a>
		    	 
		    		<em> (Middlesex University London, UK)</em>
		    	
		    	
		    		
		    			</br>"Building Reaction Networks with Prescribed Properties"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In general, the problem of identifying reaction networks with some prescribed dynamical property is challenging. As an example of a dynamical property, let's consider stable oscillation. The question then becomes: does a given network allow stable oscillation for some choice of parameters (e.g., rate constants if the reaction network has mass action kinetics)? As networks grow in size, this question becomes harder and harder to check numerically. One way of making progress is via theorems which tell us how, given an oscillatory network, we can build a larger oscillatory network with more species or reactions. I'll give an overview of such theorems, focussing mainly on oscillation.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/ECOP-MS06.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>MS06-ECOP: </small></br>Mathematical modeling of gene drives</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Gili Greenbaum (The Hebrew University of Jerusalem, Israel), Jaehee Kim (Cornell University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS01-ECOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/ECOP-MS06.html#author1"><strong class="card-body-subgroup">Keith Harris</strong></a>
		    	 
		    		<em> (The Hebrew University of Jerusalem, Israel)</em>
		    	
		    	
		    		
		    			</br>"Rescue by gene swamping as a fail-safe strategy in gene drive deployment"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Gene drives are genetic constructs that can spread deleterious alleles in wild populations by generating non-Mendelian inheritance patterns. Lab experiments of CRISPR-Cas9-based gene drives have been shown to drive populations to extinction within a few generations, paving the way for deployment of gene drives to control disease vectors and invasive species. However, given that a gene drive can potentially spill over to and modify other populations or even other species, they must be designed in a way that allows this process to be controlled. Due to the ecological risks involved in deployment, studying behaviors of gene drive spread in wild populations currently relies on mathematical and computational models. We developed a model of gene drive spillover that combines evolutionary and demographic dynamics, in a two-population setting. The model demonstrates how feedback between these dynamics produces additional outcomes to those demonstrated by the evolutionary dynamics alone. We identify an outcome where the short-term suppression of the target population is followed by gene swamping and loss of the gene drive. Using our model, we demonstrate the robustness of this outcome to spillover and the evolution of resistance, and suggest it as a fail-safe strategy for gene drive deployment.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ECOP-MS06.html#author2"><strong class="card-body-subgroup">Leo Girardin</strong></a>
		    	 
		    		<em> (Université Claude Bernard Lyon-1, France)</em>
		    	
		    	
		    		
		    			</br>"Demographic feedbacks can hamper the spatial spread of a gene drive"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk is concerned with a reaction--diffusion system modeling the fixation and the invasion in a population of a gene drive (an allele biasing inheritance, increasing its own transmission to offspring). In our model, the gene drive has a negative effect on the fitness of individuals carrying it, and is therefore susceptible of decreasing the total carrying capacity of the population locally in space. This tends to generate an opposing demographic advection that the gene drive has to overcome in order to invade. While previous reaction--diffusion models neglected this aspect, here we focus on it and try to predict the sign of the traveling wave speed. It turns out to be an analytical challenge, only partial results being within reach, and we complete our theoretical analysis by numerical simulations. Our results indicate that taking into account the interplay between population dynamics and population genetics might actually be crucial, as it can effectively reverse the direction of the invasion and lead to failure.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ECOP-MS06.html#author3"><strong class="card-body-subgroup">Lena Klay</strong></a>
		    	 
		    		<em> (Sorbonne Université, France)</em>
		    	
		    	
		    		
		    			</br>"Spatial spread of suppression and eradication drives"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the spatial and temporal spread of gene drive (mechanism that disrupts the laws of heredity by biasing transmission) through modeling is an essential step before any field experiments. In this talk, I will present a work based on a deterministic reaction-diffusion system proposed by L. Girardin and F. Débarre (presented in L. Girardin’s talk). I will focus on the case of eradication, when the population goes extinct after the drive has spread. Firstly, I will extend the original model to various timings of gene conversion (considering conversion can happen in the zygote or in the germline) and different probabilities of gene conversion (instead of assuming 100% conversion). In contrast with the initial model assuming systematic gene conversion in the zygote, heterozygous individuals must be accounted for. As the model is then quite complex, numerical studies will provide us with information regarding the emergence conditions of eradication waves. If time allows on a second part, I will simplify the system through linearization, to better understand the theoretical behavior (shape, speed…) of those waves.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ECOP-MS06.html#author4"><strong class="card-body-subgroup">Richard Gomulkiewicz</strong></a>
		    	 
		    		<em> (Washington State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Resistance-proofing Gene Drives for Population Suppression"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The advent of CRISPR technology has brought us to the cusp of engineering gene drives capable of eradicating plant and animal species and with it urgent concerns about the evolution of resistances that could undermine the drives. This talk will present results from a mathematical modeling study that reveal the fundamental mechanics of how non-allelic resistance evolves and especially how one may design a gene drive to evade resistance. The findings are used to suggest design principles to guide the engineering of resistance-proof suppression drives.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/EVOP-MS06.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>MS06-EVOP: </small></br>Predicting ecological dynamics in fluctuating environments</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Anna Miller (Department of Integrated Mathematical Oncology, Moffitt Cancer Center, United States), Nancy Huntly (Ecology Center and Department of Biology, Utah State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-EVOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/EVOP-MS06.html#author1"><strong class="card-body-subgroup">Ivana Gudelj</strong></a>
		    	 
		    		<em> (Biosciences, University of Exeter, UK)</em>
		    	
		    	
		    		
		    			</br>"Predicting community dynamics of antibiotic sensitive and resistant species in fluctuating environments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microbes occupy almost every niche within and on their human hosts. Whether colonising the gut, mouth or bloodstream, microorganisms face temporal fluctuations in resources and stressors within their niche. Yet we still know little of how environmental fluctuations mediate certain microbial phenotypes, notably antimicrobial resistant ones. For instance, do rapid or slow fluctuations in nutrient and antimicrobial concentrations select for, or against, resistance?  We tackle this question using an ecological approach by studying the dynamics of a synthetic and pathogenic microbial community containing two species, one sensitive and one resistant to an antibiotic drug where the community is exposed to different rates of environmental fluctuation. We provide mathematical models, supported by experimental data, to demonstrate that simple community outcomes, like competitive exclusion, can shift to coexistence and ecosystem bi-stability as fluctuation rates vary. Theory gives mechanistic insight into how these dynamical regimes are related.  Our approach highlights a fundamental difference between resistance in single species populations and in communities. While fast environmental changes are known to select against resistance in single-species populations, here we show that they can promote the resistant species in mixed-species communities. Our theoretical observations are verified empirically using a two-species Candida community.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/EVOP-MS06.html#author2"><strong class="card-body-subgroup">Shota Shibasaki</strong></a>
		    	 
		    		<em> (Department of Fundamental Microbiology, University of Lausanne, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Environmental and demographic stochasticity together changes microbial interactions and diversity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microorganisms live in environments that fluctuate between mild and harsh conditions. As harsh conditions may cause extinctions, the rate at which fluctuations occur can shape microbial communities and their diversity, but we still lack an intuition on how. Here, we build a mathematical model describing two microbial species living in an environment where substrate supplies randomly switch between abundant and scarce. We then vary the rate of switching as well as different properties of the interacting species, and measure the probability of the weaker species driving the stronger one extinct. We find that this probability increases with the strength of demographic noise, and peaks at either low, high, or intermediate switching rates depending on both species' ability to withstand the harsh environment. This complex relationship shows why finding patterns between environmental fluctuations and diversity has historically been difficult. In parameter ranges where the fittest species was most likely to be excluded, however, the beta diversity in larger communities also peaked. In sum, while we find no simple rules on how the frequency of fluctuations shapes species diversity, we show that their effect on interactions between two representative species predicts how diversity in the whole community will change.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/EVOP-MS06.html#author3"><strong class="card-body-subgroup">Audrey Freischel</strong></a>
		    	 
		    		<em> (Department of Integrated Mathematical Oncology, Moffitt Cancer Center, USA)</em>
		    	
		    	
		    		
		    			</br>"Utilizing a Consumer-Resource model to hypothesize foraging trade-offs in “cream skimmers” and “crumb pickers”"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Solid tumors consist of heterogeneous clones presenting unique metabolism and function. Metabolic variation allows cancer cells to be characterized as either “cream-skimmers,” which consume resources quickly at the cost of efficiency (glycolysis), or “crumb-pickers,” which consume resources slowly but have a higher metabolic payoff (oxidative phosphorylation). As observed in nature, fluctuating resources allow for coexistence of different species. To better understand the coexistence of “cream-skimmers” and “crumb-pickers” in the tumor, we utilized a classic consumer-resource model with fluctuating resource to evaluate tradeoffs in encounter probability, handling time, and fixed and variable costs. These models elucidate novel hypotheses in tumor cell competition as well as provide new insights to consumer-resource dynamics.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/EVOP-MS06.html#author4"><strong class="card-body-subgroup">David Demory</strong></a>
		    	 
		    		<em> (School of Biological Sciences, Georgia Institute of Technology, USA)</em>
		    	
		    	
		    		
		    			</br>"Temperature drives virus-host coexistence in the ocean"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Diverse marine viruses coexist with microbial hosts across a range of fluctuating marine environments. Here, we used population dynamic models to explore the role of temperature variation in modulating virus-phytoplankton coexistence. Dynamic models suggest that variation in sea surface temperature influences the range of viral life-history traits underlying coexistence amongst virus-microbe pairs, including the prediction that warmer temperatures can suppress viral persistence. Using in situ ocean datasets, we find evidence of a latitudinal trend in viral diversity, decreasing in warmer regions.  Yet, we also find that temperature fluctuations can be a driver of coexistence, allowing for a succession of (in)favorable conditions, potentially promoting the coexistence of different virus types infecting the same host via the storage effect. These findings highlight the importance of integrating environmental feedback into the study of host-virus coexistence in the global oceans.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/MEPI-MS06.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS06-MEPI: </small></br>Vector-borne Diseases: Data, Modeling, and Analysis</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jing Chen (Nova Southeastern University, United States), Shigui Ruan (University of Miami, United States), Xi Huo (University of Miami, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-MEPI</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/MEPI-MS06.html#author1"><strong class="card-body-subgroup">Yijun Lou</strong></a>
		    	 
		    		<em> (The Hong Kong Polytechnic University, China)</em>
		    	
		    	
		    		
		    			</br>"Dynamics of a periodic tick-borne disease model with co-feeding and multiple patches"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk presents a mechanistic model for describing the spatial dynamics of tick-borne diseases by considering systemic transmission, seasonal climate impacts, host movement as well as the co-feeding transmission route.  The net reproduction number for tick growth and basic reproduction number for disease transmission are derived, which predict the global dynamics of tick population growth and disease transmission.  Further numerical simulations will also be presented in the talk to not only verify the analytical results, but also characterize the contribution of co-feeding transmission route on disease prevalence in a habitat and the effect of host movement on the spatial spreading of the pathogen.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MEPI-MS06.html#author2"><strong class="card-body-subgroup">Kaniz Fatema Nipa</strong></a>
		    	 
		    		<em> (University of Georgia, United States)</em>
		    	
		    	
		    		
		    			</br>"The Effect of Demographic and Environmental Variability on Disease Outbreak for a Dengue Model with a Seasonally Varying Vector Population"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Seasonal changes in temperature, humidity, and rainfall affect vector survival and emergence of mosquitoes and thus impact the dynamics of vector-borne disease outbreaks. Recent studies of deterministic and stochastic epidemic models with periodic environments have shown that the average basic reproduction number is not sufficient to predict an outbreak. We extend these studies to time-nonhomogeneous stochastic dengue models with demographic variability wherein the adult vectors emerge from the larval stage vary periodically. The combined effects of variability and periodicity provide a better understanding of the risk of dengue outbreaks. A multitype branching process approximation of the stochastic dengue model near the disease-free periodic solution is used to calculate the probability of a disease outbreak. The approximation follows from the solution of a system of differential equations derived from the backward Kolmogorov differential equation. This approximation shows that the risk of a disease outbreak is also periodic and depends on the particular time and the number of the initial infected individuals. Numerical examples are explored to demonstrate that the estimates of the probability of an outbreak from that of branching process approximations agree well with that of the continuous-time Markov chain. In addition, we propose a simple stochastic model to account for the effects of environmental variability on the emergence of adult vectors from the larval stage.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MEPI-MS06.html#author3"><strong class="card-body-subgroup">Necibe Tuncer</strong></a>
		    	 
		    		<em> (Florida Atlantic University, United States)</em>
		    	
		    	
		    		
		    			</br>"Determining Reliable Parameter Estimates for Within-host and Within-vector models of Zika Virus of Zika Epidemiological Models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this study, we introduce three within-host and one within-vector models of Zika virus. The within-host models are (i) the target cell limited model, (ii) the target cell limited model with natural killer cells class and (iii) a within-host-within-fetus model of a pregnant individual. The within-vector model includes the Zika virus dynamics in the midgut and the salivary glands. The within-host models are not structurally identifiable with respect to data on viral load and natural killer cell counts. After rescaling, the scaled within-host models are locally structurally identifiable. The within-vector model is structurally identifiable with respect to viremia data in the midgut and salivary glands. Using Monte Carlo Simulations we find that target cell limited model is practically identifiable from data on viremia; the target cell limited model with natural killer cell class is practically identifiable, except for the rescaled half saturation constant. The  within-host-within-fetus model has all fetus related parameters not practically identifiable without data on the fetus, as well as the rescaled half saturation constant is also not practically identifiable. The remaining parameters are practically identifiable. Finally we find that none of the parameters of the within-vector model is practically identifiable.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MEPI-MS06.html#author4"><strong class="card-body-subgroup">Jianhong Wu</strong></a>
		    	 
		    		<em> (York University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Multi-scale dynamic models for vector-borne disease transmission dynamics: infestation, co-feeding and systemic infection"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We present a series of collaborative studies on using structured models to understand the intertwined infestation (vector feeding on host), systemic and co-feeding transmission processes. We show how this multi-scale approach leads to a new class of nonlinearity, novel classes of dynamical systems, and interesting threshold dynamics including bistability and oscillation.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/MFBM-MS06.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS06-MFBM: </small></br>Mathematical and computational methods to augment the reliability of biological models for better decision-making</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Vincent Lemaire (Genentech, CA, USA, United States), Khamir Mehta (Amgen, Inc, United States), Malidi Ahamadi (Amgen, CA, United States)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/MFBM-MS06.html#author1"><strong class="card-body-subgroup">Chris Rackauckas</strong></a>
		    	 
		    		<em> (MIT and Pumas AI, MA, USA, United States)</em>
		    	
		    	
		    		
		    			</br>"Accelerating Quantitative Systems Pharmacology with Machine Learning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Scientific machine learning (SciML) is the burgeoning field combining scientific knowledge with machine learning for data-efficient predictive modeling. We will introduce the Julia SciML ecosystem by describing some of its recent advances, showing how the GPU-accelerated differential equation solvers gave 175x acceleration on Pfizer's internal C-based QSP models and the 15,000x acceleration seen by the NASA Launch Services upon switching from Simulink to ModelingToolkit.jl. After describing the advances in differential equation solvers and automated model discovery, we will describe the JuliaSim simulation ecosystem and its ability to use continuous-time echo state networks (CTESNs) for automatically generating surrogates of highly stiff QSP models. This technique is shown to be validated on a wide variety of models by using CellML and SBML imports to automate the surrogate training process on ~1000 models. Using the Robertson chemical reaction network as an example case, we will see how multi-layer perceptrons (MLPs),  recurrent neural networks (RNNs), Long short term memory networks (LSTMs), and physics-informed neural networks (PINNs) all fail to adequately train while only the CTESN succeeds in building a stable surrogate. Examples of accelerating simulations by over 560x over the Dymola Modelica compiler will showcase the scalability of the technique. The will showcase how JuliaSim composes with tools like Pumas to bridge QSP into clinical pharmacology. We will end by describing new adjoint techniques which are required to build neural ODE surrogates on stiff ODE models. Together this showcases the practical changes users of the JuliaSim ecosystem are seeing through scientific simulation
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MFBM-MS06.html#author2"><strong class="card-body-subgroup">Oleg Demin Jr</strong></a>
		    	 
		    		<em> (InSysBio, Russia)</em>
		    	
		    	
		    		
		    			</br>"Implementation of variability or uncertainty in parameter values to validate QSP models."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Validation is an important step to test the reliability of the mathematical models including quantitative systems pharmacology (QSP) models. Clinical endpoints for the population of patients are usually used to validate QSP models. For example, percent of responders or mean +/- SD of the particular biomarker. Variability or uncertainty in parameter values should be implemented to describe these endpoints. There are various approaches to extract and implement variability or uncertainty in parameters in model predictions. These methods and cases of their implementation in mechanistic and QSP models will be discussed in the framework of this presentation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MFBM-MS06.html#author3"><strong class="card-body-subgroup">Gianluca Selvaggio</strong></a>
		    	 
		    		<em> (Fondazione The Microsoft Research - University of Trento Centre for Computational and Systems Biology (COSBI), Italy)</em>
		    	
		    	
		    		
		    			</br>"Parameter free approaches in QSP: modelling the cytokine release following bispecific T-cell engager therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bispecific T-cell Engaging therapy is a promising treatment that leverages patient’s own immune system to eliminate cancerous cells. To realize the full potential of  therapy, it is necessary to mitigate the adverse effects of cytokine release from the immune activation, which eventually lead to adverse effect of cytokine release syndrome (CRS). Computational approaches can be instrumental to explore, systematically, the effects of combined therapies on the tumor killing efficacy and CRS. However, to be fully characterized and validated, quantitative models (such as ODEs) require a priori information, that may be poorly available. An alternative parameter free approach is to use the logical formalism to provide a qualitative representation of the processes. This modelling approach can overcome the data/knowledge gap and the sparsity of clinical data by leveraging on several types of information and integrating both qualitative and quantitative information into computable networks. The presentation will demonstrate a logical QSP model that was used to investigate, through systematic sensitivity analysis, the system behavior and then applied to understand strategies to hamper the inflammatory response without impairing the tumor killing capacity. Our analysis suggests that IFN-γ may be a good mechanism to control CRS risk in patients. Furthermore, it entails the existence of a time window to administrate anti-PDL1 therapy and mitigate inflammation without compromising tumor clearance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MFBM-MS06.html#author4"><strong class="card-body-subgroup">Sietse Braakman</strong></a>
		    	 
		    		<em> (AbbVie Inc., Quantitative Translational Modeling Group, United States)</em>
		    	
		    	
		    		
		    			</br>"A framework for the evaluation of QSP models, with a focus on verification, validation and uncertainty quantification (VVUQ) methods"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Quantitative systems pharmacology (QSP) and other mechanistic mathematical models are increasingly used to support decisions in drug research and development, as well as regulatory decisions (Nijsen et al., 2018; Zineh, 2019). However, despite their demonstrated value, QSP models are not as widely used as they could be (Leil and Bertz, 2014). Reasons for this include the complexity of these models, a lack of consensus on standards for the evaluation of systems models, and short project timelines that are incompatible with the development of complex models. To work towards a consensus on evaluation standards, we introduce a framework for the evaluation of QSP models (Braakman et al., 2021). The framework is designed to accommodate the wide variety of risk and application settings common for QSP models, by applying certain quantitative and qualitative methods to a model. We include verification, validation, and uncertainty quantification (VVUQ) methods such as global sensitivity analysis, identifiability analysis, confidence and profile likelihood intervals, and model validation with hold-out or external data.
Nijsen MJMA, et al., Preclinical QSP Modeling in the Pharmaceutical Industry: An IQ Consortium Survey Examining the Current Landscape. Clinical Pharmacology and Therapeutics: Pharmacometrics and Systems Pharmacology, 2018 7(3): 135-146.  https://doi.org/10.1002/psp4.12282 Zineh I, Quantitative Systems Pharmacology: A Regulatory Perspective on Translation. Clinical Pharmacology and Therapeutics: Pharmacometrics and Systems Pharmacology, 2019 8(6): 336-339. https://doi.org/10.1002/psp4.12403 Leil TA and Bertz R, Quantitative Systems Pharmacology can reduce attrition and improve productivity in pharmaceutical research and development. Frontiers in Pharmacology 2014 5:247. https://doi.org/10.3389/fphar.2014.00247 Braakman S, Pathmanathan P, Moore H, Evaluation Framework for Systems Models. Under review 2021.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/MMPB-MS06.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>MS06-MMPB: </small></br>How can mathematical modelling aid medical decision making?</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jasmina Panovska-Griffiths (University of Oxford), Eduard Campillo-Funollet (University of Sussex)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/MMPB-MS06.html#author1"><strong class="card-body-subgroup">Elizabeth Ford</strong></a>
		    	 
		    		<em>(Brighton and Sussex Medical School)</em>
		    	
		    	
		    		
		    			</br>"Can modelling of primary care patient records enable detection of dementia earlier than the treating physician?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Timely diagnosis of dementia is a policy priority in the United Kingdom (UK). However, recent research shows that a third to a half of patients with dementia do not have a diagnosis recorded in their primary care patient record, and for those that get a diagnosis, it takes over three years for the diagnosis to be made. We explored using modelling to automate early detection of dementia using data from electronic health records (EHRs). We investigated: a) how early a machine-learning model could accurately identify dementia before the physician; b) if models could be tuned for dementia subtype; and c) what the best clinical features were for achieving detection. Using EHRs from Clinical Practice Research Datalink in a case-control design, we selected patients aged >65y with a diagnosis of dementia recorded 2000-2012 (cases) and matched them 1:1 to controls, giving a total of 95k patients. We trained random forest classifiers, and evaluated models using Area Under the Receiver Operating Characteristic Curve (AUC). We examined models by year prior to diagnosis, dementia subtype, and the most important features contributing to classification. Classification of dementia cases and controls was poor 2-5 years prior to physician-recorded diagnosis but good in the year before. Features indicating increasing cognitive and physical frailty dominated models 2-5 years before diagnosis; in the final year, initiation of the dementia diagnostic pathway (memory loss symptoms, screening and referral) explained the sudden increase in accuracy. This leads us to think that automated detection of dementia earlier than the treating physician may be problematic using only primary care data, and that linking multiple sources of healthcare data could improve model performance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MMPB-MS06.html#author2"><strong class="card-body-subgroup">Robin Thompson</strong></a>
		    	 
		    		<em>(University of Warwick)</em>
		    	
		    	
		    		
		    			</br>"Can modelling be used to predict whether or not the novel coronavirus will spread in the UK?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The most devastating infectious disease epidemics are those that have a wide geographical range, as opposed to being confined to a small region. Early in the COVID-19 epidemic, an important question was whether or not SARS-CoV-2 would spread elsewhere and cause local outbreaks outside of China. A vital factor was the probability of establishment whenever a pathogen arrives in a new location, since this is a key component of any pathogens pandemic potential. We assessed this in real-time during the COVID- 19 epidemic. In this talk, we show how the probability of sustained transmission in other locations can be estimated from data that are available during infectious disease outbreaks. We show how estimates can be extended to include features such as transmission from paucisymptomatic infectors (infectious individuals with few symptoms). If time allows, we will also show how estimates can be generated for other pathogens and other epidemiological settings.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MMPB-MS06.html#author3"><strong class="card-body-subgroup">Fred Vermolen</strong></a>
		    	 
		    		<em>(Delft University of Technology)</em>
		    	
		    	
		    		
		    			</br>"Can modelling aid the process of deep tissue healing without scarring?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Deep tissue injury is often followed by contraction of the scar. This contraction is caused by the pulling forces exerted by myofibroblasts and fibroblasts, which are cells that are responsible for the regeneration of collagen. In this talk, we will review several mechanical frameworks, such as viscoelasticity and morpho- elasticity, in which the latter framework can be used to simulate plastic deformations. Furthermore, we will consider cell-based as well as continuum simulation frameworks and some remarks about our upscaling efforts will be given. These upscaling strategies currently incorporate the relation between the use of the immerse boundary method and smoothed particle approach. Since many input parameters are patient-dependent, we will also present some results from the quantification of uncertainty that we have carried out.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/MMPB-MS06.html#author4"><strong class="card-body-subgroup">Jasmina Panovska-Griffiths</strong></a>
		    	 
		    		<em>(University of Oxford)</em>
		    	
		    	
		    		
		    			</br>"Can combining modelling and brain radiomics non- invasively stratify brain gliomas?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Combining MRI techniques with modelling is rapidly gaining attention as a promising method for staging of brain gliomas. This study assesses the diagnostic value of such a framework applied to stratify- ing treatment-nave gliomas from a multi-center patients into WHO grades II-IV and across their isocitrate dehydrogenase (IDH) mutation status. 333 patients from 6 tertiary centres, diagnosed histologically and molecularly with primary gliomas were retrospectively identified. Shape, intensity distribution and tex- ture features over the tumour mask were extracted. A random-forest algorithm was employed (2-fold cross-validation, 250 repeats) to predict grades or mutation status using the extracted features. Shape, distribution and texture features showed significant differences across mutation status. WHO grade II-III differentiation was mostly driven by shape features while texture and intensity feature were more relevant for the III-IV separation. Increased number of features became significant when differentiating grades further apart from one another. Gliomas were correctly stratified by mutation status in 71% and by grade in 53% of the cases (87% of the gliomas grades predicted with distance less than 1). Combining brain radiomics with modelling presents a promising approach for non-invasive glioma molecular subtyping and grading.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/NEUR-MS06.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS06-NEUR: </small></br>Effects of stochasticity and heterogeneity on networks' synchronization properties</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Zahra Aminzare (University of Iowa, United States), Vaibhav Srivastava (Michigan State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-NEUR</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/NEUR-MS06.html#author1"><strong class="card-body-subgroup">James Roberts</strong></a>
		    	 
		    		<em> (QIMR Berghofer Medical Research Institute, Australia)</em>
		    	
		    	
		    		
		    			</br>"Noise-enhanced synchronization of dynamics on the human connectome"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Synchronization is a collective mechanism by which oscillatory networks achieve their functions. However, it is not well understood how potentially disruptive external inputs like stochastic perturbations affect synchronization. This is particularly so for real-world systems with relatively complex network topologies and dynamical properties, such as the human brain. Here, we aim to address this problem using a large-scale model of the human brain network (i.e., the human connectome). Using the Kuramoto model, we show that when nodes in the network are coupled at some critical strength, a counterintuitive phenomenon emerges where the addition of noise increases the synchronization of global and local dynamics, with structural hub nodes benefiting the most. We link this stochastic synchronization effect to the intrinsic hierarchy of neural timescales of the brain and the heterogeneous complex topology of the connectome. We find that the human connectome supports the formation of frustrated clusters, which, in the presence of moderate levels of noise, reconfigure via phase shifts and frequency shifts to increase the overall synchronization. Overall, the work provides theoretical insights into the emergence and mechanisms of stochastic synchronization, highlighting its putative contribution in achieving network integration underpinning brain function.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/NEUR-MS06.html#author2"><strong class="card-body-subgroup">Giovanni Russo</strong></a>
		    	 
		    		<em> (University of Salerno, Italy)</em>
		    	
		    	
		    		
		    			</br>"On noise-induced phenomena in complex networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk is focused on the study of noise-induced emerging behaviors in complex networks. We will explore how the interplay between the dynamics at the nodes, the network topology and noise diffusion processes play a key role in determining stability of certain manifolds in the network state-space. After introducing the mathematical framework, we present a perhaps counter-intuitive result for network synchronization. Indeed we show how certain noise diffusion processes (also termed as relative-state- dependent noise) force stability of the synchronization/consensus manifold that, without noise, would be unstable. Applications of the results to biochemical systems are also discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/NEUR-MS06.html#author3"><strong class="card-body-subgroup">Matin Jafarian</strong></a>
		    	 
		    		<em> (Delft University of Technology, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Stochastic stability of discrete-time phase-coupled oscillators"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk, we study the conditions of stochastic stability for a class of discrete-time phase-coupled oscillators. We introduce the notion of stochastic phase-cohesiveness using the concept of Harris recurrency of Markov chains. We study the stochastic phase-cohesiveness of oscillators in a network with an underlying connected topology subject to both multiplicative and additive stochastic uncertainties. We derive sufficient conditions for achieving the phase-cohesiveness considering stochastic uncertainties realized according to probability distributions with both positive and negative mean values. We further discuss the phase-cohesiveness of oscillators in a random network as a special case of the aforementioned problem.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/NEUR-MS06.html#author4"><strong class="card-body-subgroup">Supravat Dey</strong></a>
		    	 
		    		<em> (University of Delaware, United States)</em>
		    	
		    	
		    		
		    			</br>"Role of intercellular coupling and delay on the synchronization of biomolecular clocks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Living cells encode diverse biological clocks for circadian timekeeping and formation of rhythmic structures during embryonic development. These biomolecular clocks are subject to unavoidable fluctuations due to the inherent stochasticity of biochemical reactions. How do these clocks synchronize across cells through intercellular coupling mechanisms? To address this question, we leverage the classical motif for genetic clocks, the Goodwin oscillator, where a gene product inhibits its own synthesis via time-delayed negative feedback. More specifically, we study an interconnected system of two identical Goodwin oscillators (each operating in a single cell), where state information is conveyed between cells via a signaling pathway whose dynamics is modeled as a first-order system. Our results show intercellular coupling strength and intercellular time delay play a vital role in the synchronization of the coupled oscillators.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS06/ONCO-MS06.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS06-ONCO: </small></br>Blackboard to Bedside: Showcase of Translational Modeling</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Renee Brady-Nicholls (Moffitt Cancer Center, USA), Mohammad Zahid (Moffitt Cancer Center, USA), Stefano Pasetto (Moffitt Cancer Center, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS06/ONCO-MS06.html#author1"><strong class="card-body-subgroup">Rene Bruno</strong></a>
		    	 
		    		<em> (Genentech-Roche, France)</em>
		    	
		    	
		    		
		    			</br>"Tumor dynamic modeling and overall survival predictions to support decisions in oncology clinical trials"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The key endpoints to support treatment approval in oncology and particularly for the treatment of advanced diseases is overall survival (OS). However, decisions to move to pivotal trials have to be made using earlier endpoints like overall response rate (ORR) or progression free survival (PFS) that often poorly predict OS and probability of success of a pivotal Phase III trial particularly with immunotherapies. Longitudinal tumor dynamic models estimate treatment effect on tumor growth inhibition (TGI)) and are linked to OS (TGI-OS models) in treatment independent biomarker-outcome models to offer a quantitative model-based approach that fully leverage to data generated in early trials. The use of TGI-OS models to simulate Phase III studies outcome and support early decisions will be illustrated (Bruno et al, Clin Cancer Res 2020;26:1787–95).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ONCO-MS06.html#author2"><strong class="card-body-subgroup">Pamela Jackson</strong></a>
		    	 
		    		<em> (Mayo Clinic, USA)</em>
		    	
		    	
		    		
		    			</br>"Instantiating an Imaging Digital Twin for a Brain Tumor Patient"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In medicine, digital twins are computational representations of some aspect of an individual patient and their disease.  An effective digital twin can incorporate mathematical models to recapitulate the patient’s current disease state and predict the individual patient’s response to a therapeutic intervention, such that multiple interventions can be tested on the twin prior to selecting the most effective therapy. For brain tumors specifically, clinical imaging will be an important part of any digital twin given the eloquent nature of the brain and the integral part imaging plays in identifying suspected brain tumors and determining response to therapy. Thus, an imaging digital twin that can capture the dynamic visualization of the disease will be critical for comparison to actual patient images. Before the dynamics of the disease can be captured, we must first instantiate the simulated version of a patient’s imaging for the pre-treatment timepoint. Our objective is to demonstrate the identification of an imaging digital twin for an individual patient’s brain tumor at the pretreatment time-point using a brain tumor growth mathematical model coupled to an imaging simulation utilizing MRI physics. To instantiate the imaging digital twin, we generated multiple candidate brain tumors and their associated simulated images using the Proliferation-Invasion-Hypoxia-Necrosis-Angiogenesis-Edema (PIHNA-E) model coupled to an MRI signal model [1,2]. Using the PIHNA-E model [1] incorporating the patient’s imaging-based anatomy, we created twenty-five phantoms based on unique combinations of 5 different rates of migration (D [mm2/year]) and 5 different rates of proliferation (ρ [1/year])]. These patient-specific PIHNA-E simulations were then passed into an MRI signal model for simulating corresponding T2-weighted MRIs [2]. We then compared the acquired patient image to the candidate simulations with various combinations of D and ρ.  To identify a “close” matching image, we calculated the L2-norm of twelve statistical features for both the acquired patient image and the simulated candidate images. The D and ρ of the acquired image with the lowest L2-norm relative to the candidate image was selected as the predictive parameter set.  Additionally, we examined the effect of noise on the selection process. We were able to both create patient-specific simulated MRIs and select parameters for the PIHNA-E brain tumor growth model. [1] A. Hawkins-Daarud, R. C. Rockne, A. R. A. Anderson, and K. R. Swanson. 'Modeling tumor-associated edema in gliomas during anti-angiogenic therapy and its impact on imageable tumor.' Frontiers in oncology 3:66, 2013. [2] P.R. Jackson, A. Hawkins-Daarud, S. C. Partridge, P. E. Kinahan, and K. R. Swanson. 'Simulating magnetic resonance images based on a model of tumor growth incorporating microenvironment.' Medical Imaging 2018: Image Perception, Observer Performance, and Technology Assessment, International Society for Optics and Photonics 10577:105771D, 2018.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ONCO-MS06.html#author3"><strong class="card-body-subgroup">Elsa Hansen</strong></a>
		    	 
		    		<em> (Penn State Huck Institutes of the Life Sciences, USA)</em>
		    	
		    	
		    		
		    			</br>"Maintenance therapy: A case study in trial design"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Treatment efficacy is often measured in terms of progression free survival (PFS) or tumor response. Viewing cancer treatment from the perspective of resistance management changes how we interpret these measures. I will discuss these issues in the context of a recent clinical trial of maintenance therapy for multiple myeloma.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS06/ONCO-MS06.html#author4"><strong class="card-body-subgroup">Sarah Brüningk</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>"Intermittent radiotherapy as alternative treatment for recurrent high grade glioma: A modeling study based on longitudinal tumor measurements"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Treatment options for recurrent high grade glioma are greatly limited and non-curative. Radiotherapy (RT) is an integral part of palliative patient care. A recent phase I clinical trial (NCT02313272) recently demonstrated the safety of a combination treatment of high dose hypofractionated stereotactic radiotherapy (HFSRT, ≥ 6 Gyx5 in daily fractions) with pembrolizumab (immuno therapy; anti PD1 antibody) and bevacizumab (aiming at vasculature normalization). In this presentation we show a simulation study of intermittent RT (iRT, delivering RT fractions in intervals of several weeks) suggested as a personalized treatment strategy to prolong tumor control rather than using debulking HFSRT. Simu- lations were performed using a mathematical model of tumor growth, radiation response and patient-specific evolution of resistance to additional treatments (pembrolizumab and bevacizumab). Four models comprising different levels of patient specific parameters were fitted from tumor growth curves of 16 patients enrolled in the NCT02313272 trial. The model ranking highest based on the Akaike information criterion was used for simulation of iRT and iRT plus boost (≥ 6 Gyx3 in daily fractions at time of progression) schedules for varying numbers of treatment fractions and time between fractions. Kapalan Meier curves scoring time to progression beyond the initial tumour volume were used to com- pare treatments. We show that iRT+boost(-boost) treatment was equal or superior to HFSRT in 15(11) out of 16 cases and that patients that remained responsive to pem- brolizumab and bevacizumab would benefit most from iRT. Time to progression could be prolonged through the application of additional, intermittently delivered fractions. iRT hence provides a promising treatment option for recurrent high grade glioma patients.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-06
										</br>
										Tuesday, June 15 at 04:15am (PDT)</br>Tuesday, June 15 at 12:15pm (BST)</br>Tuesday, June 15 08:15pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>














    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
