---
layout: group
author: esmb
day: "Monday (Tuesday)"
subgroup: "DDMB"
title: "Stochastic models of cancer: An update of theory and data"
organizers: "Marek Kimmel (Rice University, United States), Simon Tavare (Columbia University, United States)"
description: "Much has been learned recently about mechanisms of cancer progression, as well as about cancer stochasticity at the molecular and population level, and about interaction of tumors with normal cells of the organism. These developments prompted progress in mathematical, computational and statistical models and tools. This mini-symposium brings together a diverse group of representatives of several leading institutions who will discuss their recent work. Topics range from branching processes and cellular automata, to mathematical models of mutation, genetic drift and selection, immune infiltration of tumors, and evolutionary dynamics of specific types of cancer. They also include statistical methods, such as cancer phylodynamics. The organizers hope these will provide inspiration for further work in the area."
code: "MS02"
code2: "MS01"
author1: "Katharina Jahn"
inst1: " (Computational Biology Group, ETH Zurich, Zurich, Switzerland, Switzerland)"
title1: "Dissecting Clonal Diversity Through High-Throughput Single-Cell Genomics"
abstract1: "Clonal heterogeneity allows tumours to adapt and survive under the selective pressure of treatment, leading to clinical resistance and relapse. An accurate dissection of the clonal architecture and the underlying mutational history is therefore of clinical importance and may help to design more effective treatment plans. Present studies on clonal diversity are primarily based on sequencing data obtained from bulk tumour tissue which systematically underestimate a tumour's mutational heterogeneity. However, through recent technological advances, high-throughput single-cell genomics has become a feasible alternative that allows to study clonal diversity at an unprecedented resolution.

In this talk, I will present a Bayesian inference scheme for tumour mutation histories based on single-cell sequencing data and the insights we obtained from analysing longitudinal bone marrow samples of 123 AML patients. Using a microfluidics-based single-cell DNA sequencing platform, we genotyped over 700,000 cells for a panel of genes recurrently mutated in AML. We observed patterns of mutual exclusivity, mutational co-occurrence, as well as instances of convergent evolution. Moreover, the longitudinal nature of the data revealed patterns of clonal dynamics in response to targeted AML therapy which correlated with clinical resistance and relapse."
author2: "Ximo Pechuan Jorge"
inst2: " (Institute of Cancer Research, London, UK, UK)"
title2: "A Simple Computational Model to Infer Selective Coefficients in Barcode Evolution Experiments"
abstract2: "The advent of single cell sequencing technologies has propelled the usage of lineage barcoding to characterize the dynamics of heterogeneous cell populations. Following the population dynamics of tumor cells is of paramount importance to determine the details of their evolutionary process which, in turn, can influence therapeutic outcome. To characterize the evolutionary dynamics of barcoded organoids during the course of two years of serial passage \textit{in vitro} after a genetic perturbation, we constructed a simple stochastic model accounting for drift and competition between lineages. We used sequential Monte Carlo to fit the model to the experimental data obtaining initial growth rate estimates for each lineage. Some of the samples exhibited evidence of mutation acquisition and thus required a model accounting for mutation accumulation. Our model explains the patterns observe in the data and shows the value of constructing simple interpretable models in the initial stages of data analysis."
author3: "Luis Zapata Ortiz"
inst3: " (Institute of Cancer Research, London, UK, UK)"
title3: "Evolutionary dynamics of cancer immunoediting predicts response to immunotherapy."
abstract3: "Cancer Immunoediting is an evolutionary force that shapes the genome of healthy and malignant cells in the human body. However, quantifying immunogenicity in the cancer genome and how the tumour-immune coevolutionary dynamics impact patient outcomes remain unexplored. Here, we developed a stochastic branching process coupled with an agent-based model to simulate the accumulation of mutations during immunoediting. We show how a metric of selection, the ratio of nonsynonymous to synonymous mutations in the immunopeptidome (immune dN/dS) quantifies tumor immunogenicity and differentiates between outcomes of immunoediting. We provide a theoretical explanation for the lack of signals of immune selection reported previously and analysed 8,543 primary tumors from TCGA and 376 metastatic tumors from immunotherapy trials. We validated immune dN/dS as a measure of CD8-T cell mediated selection in tumours that have not undergone immune escape. Moreover, In a cohort of 368 metastatic patients treated with checkpoint inhibitors, we observed that lesions of non-responders had strong immune selection (dN/dS < 1, negative), whereas responders did not show immune selection (dN/dS ~ 1, neutral), and instead harboured a higher proportion of genetic escape mechanisms. Our findings highlight the challenges of using dN/dS to estimate selection, suggest that the extent of immunogenicity can be read from the tumor genome, and that the evolutionary consequences of immunoediting determine immunotherapy efficacy."
author4: "Jan Poleszczuk"
inst4: " (Nalecz Institute of Biocybernetics and Biomedical Engineering, Polish Academy of Sciences, Warsaw, Poland, Poland)"
title4: "Microsimulation-based optimization of colorectal cancer screening strategies"
abstract4: "Colorectal cancer (CRC) is a substantial public hearth burden and is in the top three cancers with respect to incidence and mortality in US and many other industrialized countries. CRC screening tests based on the endoscopic visualization of the colon have proven effective in reducing mortality, both by allowing CRC at earlier stages and by CRC prevention since adenomatous precursors of CRC can be removed during endoscopy. However, the starting age and time intervals of screening colonoscopies for optimal protection against CRC are unknown. We used microsimulation to systematically optimize screening colonoscopy schedules.

We advanced our established open-source microsimulation model CMOST to simulate the effects of colonoscopy screening on the natural history and medical costs of CRC. In CMOST, carcinoma develops via early and advanced adenoma precursors. CMOST accounts for the gender- and age-dependent risks for adenoma development, the presence of multiple adenomas, as well as their locations within the colon. CMOST microsimulation tracks the history of a general population from birth until death for a maximum age of 100 years. Adenoma initiation, progression to advanced adenoma and cancer, cancer progression, screening and surveillance are all modeled in time increments of 3 months and are stochastically driven. We used CMOST to optimize colonoscopy schedules with one, two, three and four screening colonoscopies between 20 and 90 years of age. For each scenario, we calculated life years gained, incidence and mortality reduction, and cost-effectiveness.

A single screening colonoscopy is most effective in reducing life years lost from CRC when performed at 55 years of age. Two, three and four screening colonoscopy schedules are optimal at earlier ages. For maximum reduction of incidence and mortality, screening colonoscopies need to be scheduled later in life compared to optimal age for life years lost. The optima are influenced by adenoma detection rates, individual CRC risk, and adherence to screening, with lower values for these parameters favoring a later starting age of screening. Incremental cost-effectiveness remained below 100â€™000 discounted US dollars per discounted life year gained except for an optimal four-colonoscopy schedule, which was not cost-effective. In a personalized approach, optimal screening would start earlier for high-risk patients and later for low-risk individuals.

Our results support screening recommendations involving an early starting age of 45 years. Our optimized screening strategies are cost-effective and save more life years than currently"
---
