<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Contributed Talk Session - CT01 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Contributed Talk Session - CT01 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Contributed Talk Session - CT01" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/CT01/" />
<meta property="og:url" content="http://localhost:4000/CT01/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/CT01/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Contributed Talk Session - CT01","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Contributed Talk Session - CT01">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Contributed Talk Session - CT01" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Contributed Talk Session - CT01" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Contributed Talk Session - CT01</h1>
        <h4>Monday, June 14 at 03:15pm (PDT)</br>Monday, June 14 at 11:15pm (BST)</br>Tuesday, June 15 07:15am (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="CT01">Contributed Talk Session - CT01</h1>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CBBS/CBBS-CT01.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>CT01-CBBS: </small></br>CBBS Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CBBS/CBBS-CT01.html#author1"><strong class="card-body-subgroup">Jolene Britton</strong></a>
		    	 
		    		<em>University of California, Riverside</em>
		    	
		    	
		    		
		    			</br>"A Metabolism-Based Multiscale Model of Fungal Development and Growth"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bacterial-fungal interactions play a fundamental role in many processes including crop biofuel development and biosystem design. In this work, we focus on the interactions between the fungi Laccaria bicolor and the bacterium Psuedomonas fluorescens and their integral role in the fitness of the roots of the Populus tree. L. bicolor synthesizes trehalose which stimulates growth and chemotaxis of P. flourescens. Furthermore, P. flourescens provides L. bicolor with thiamine thereby increasing fungal mass. We developed a multiscale computational model to investigate these interdependent interactions. The growth and branching of the fungal mycelia are modeled using an off-lattice spatial discrete submodel which is dependent on both diffusive and active translocation of internal nutrients and uptake of external nutrients. The fungal growth model is coupled with a thermodynamic-kinetic maximum entropy ODE model for metabolism, containing over 200 reactions including protein and nucleic acid synthesis, from which the costs of growth and maintenance can be calculated. Trehalose secretion, especially at the tips of the hyphae, acts as a source of diffusive chemoattractant for P. fluorescens colony. Numerical simulations of these coupled models under various conditions aid in characterizing the energetic costs of growth and maintenance of L. bicolor in the presence of P. fluorescens.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT01.html#author2"><strong class="card-body-subgroup">Brenda Lyn A. Gavina</strong></a>
		    	 
		    		<em>University of the Philippines</em>
		    	
		    	
		    		
		    			</br>"Optimization of dosing strategy for anovulation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A female's reproductive life from the average age of 12.5 until 51 is governed by the menstrual cycle. During this cycle, pituitary and ovarian hormones fluctuate. Abnormal concentrations of these hormones often cause cycle irregularities. However, there are cases where an abnormal cycle, in particular anovulation, is desired. For instance, in contraception and in managing premenstrual symptoms. Exogenous hormones such as synthetic progesterone and synthetic estrogen have been used to attain anovulatory state by controlling hormone levels in the body. Nonetheless, large doses are associated with adverse effects such as increased risk for thrombosis and myocardial infarction. This talk focuses on the application of optimal control to a simple modification of the model in (Margolskee et al., 2011) in order to determine the minimum dosages of exogenous estrogen and progesterone that result to anovulation. Exogenous hormone profile and timing of administration are obtained.  These results may give clinicians insights to improve dosing strategies in ovulation suppression.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT01.html#author3"><strong class="card-body-subgroup">Daniel Plaugher</strong></a>
		    	 
		    		<em>University of Kentucky</em>
		    	
		    	
		    		
		    			</br>"Modeling the Pancreatic Cancer Microenvironment in Search of Control Targets"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Pancreatic Ductal Adenocarcinoma is among the leading causes of cancer related deaths globally due to its extreme difficulty to detect and treat. Recently, research focus has shifted to analyzing the microenvironment of pancreatic cancer to better understand its key molecular mechanisms. This microenvironment can be represented with a multi-scale model consisting of pancreatic cancer cells, pancreatic stellate cells, as well as cytokines and growth factors which are responsible for intercellular communication between the PCCs and PSCs. We have built a stochastic Boolean model, validated by literature and clinical data, in which we probed for intervention strategies that force this gene regulatory network from a diseased state to a healthy state. We implemented methods from phenotype control theory to determine a procedure for regulating specific genes within the microenvironment. After applying well studied control methods such as stable motifs, feedback vertex set and computational algebra, we discovered that each produces a different set of control targets that are not necessarily minimal nor unique. Each control set contains cytokines, KRas, and HER2/neu which suggests they are key players in the system's dynamics. Many of these model predictions are supported by literature and have potential to be new targets.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CBBS/CBBS-CT01.html#author4"><strong class="card-body-subgroup">Furkan Kurtoglu</strong></a>
		    	 
		    		<em>Indiana University</em>
		    	
		    	
		    		
		    			</br>"Integration of Intracellular Kinetic Models to Multiscale Agent-Based Models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Multiscale Agent-Based Models (ABM) provide a framework to model across different scales while intracellular kinetic models capture dynamic trends at the molecular level. In this work, we integrated intracellular kinetic models to a 3-D physics-based multiscale ABM tool (PhysiCell). Cells are represented as intelligent agents which behave according to rules and parameters. As a result, dynamic molecular models which are represented as ordinary differential equations (ODEs) in Systems Biology Markup Language (SBML) format were solved in PhysiCell using libRoadrunner, a fast, SBML solver package. To achieve this goal, cells uptake/secrete chemicals to/from the microenvironment. Custom data associated with PhysiCell agents is used as an interface between ABM and ODEs, updating the SBML in pre-defined time intervals. Kinetic ODEs are simulated at this point and results are updated to the custom data which can be used to control phenotypic parameters. Therefore, phenotypic changes in intelligent agents are determined by molecular level events. Moreover, having cell-specific custom data provides the heterogeneity through tissue or domain. This advancement makes PhysiCell models easier to produce since modelers can use SBML to write their dynamic phenotypes without writing complex functions in C++.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/CDEV/CDEV-CT01.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>CT01-CDEV: </small></br>CDEV Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/CDEV/CDEV-CT01.html#author1"><strong class="card-body-subgroup">Denis Patterson</strong></a>
		    	 
		    		<em>Princeton University</em>
		    	
		    	
		    		
		    			</br>"A Mathematical Model of Neuronal Identity with Ectopic Domains"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Recent experiments studying the development of cortical structures in mice have identified COUP-TF1 as a crucial determinant of both the position and sharpness of the boundary between the neo and entorhinal cortices. When COUP-TF1 is under expressed, neocortex invades into territory occupied by the entorhinal cortex in wild-type mice, but the sharp boundary between cortical regions is maintained. However, if COUP-TF1 is over-expressed, the boundary fractures and entorhinal cortex invades the neocortical domain, resulting in mice with ectopic regions of misplaced cortex.We introduce a novel PDE model based on a Keller-Segel-type chemotaxis mechanism to account for both the sharp cortical boundaries of wild-type mice and the ectopic regions observed in mutant mice. Competition between entorhinal and neocortical progenitor cells is mediated by a gradient of COUP-TF1 across the spatial domain and chemotaxis operators model each cell's affinity for cells of their own type. We verify the well-posedness of the system and establish necessary conditions for pattern forming Turing bifurcations; we also numerically study the structure of the Turing space and its dependence on model parameters. Numerical simulations show excellent agreement with experimental observations and we present experimental data verifying the differential adhesion hypothesis underpinning the model's phenomenology.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author2"><strong class="card-body-subgroup">Yoshito Hirata</strong></a>
		    	 
		    		<em>University of Tsukuba</em>
		    	
		    	
		    		
		    			</br>"Reconstructing 3D chromosome structures from single diploid cell Hi-C data via  recurrence plots"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Previously, we have proposed a method for reconstructing 3D chromosome structures from single haploid cell Hi-C data by regarding a contact map as a recurrence plot and applying a method for converting a recurrence plot back to its original time series (Hirata, Oda, Ohta, and Aihara, Sci. Rep. 2016). Here, we extend our previous method to single diploid cell Hi-C data. We discuss that the reconstructed 3D chromosome structures are consistent mathematically as well as biologically. We will start our presentation with a small intuitive quiz for understanding what kind of question we have to solve. The research of Y.H. was partially supported by AMED under Grant Number JP21gm1310004.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author3"><strong class="card-body-subgroup">Dan Tudor</strong></a>
		    	 
		    		<em>University of Edinburgh</em>
		    	
		    	
		    		
		    			</br>"Inferring chemoattractant properties from cell tracking data using mathematical modelling and Bayesian inference"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The rapid recruitment of immune cells during the inflammatory response is vital to dealing with injury or infection. Immune cells are guided by chemoattractants produced at the wound site. Visualising the underlying chemoattractant gradient can be experimentally complex. In comparison, the cells response to the chemoattractant gradient can be captured more easily via their trajectories. Thus, we are faced with the inverse problem of inferring the chemoattractant gradient from the observed cell movements, which are also subject to noise.  We use an established mathematical framework to model cell migration as a biased persistent random walk, and chemoattractant production and diffusion using a reaction-diffusion equation. By applying Bayesian inference, we can infer the underlying chemoattractant properties. We apply this framework to analyse different wound conditions, to answer if immune cell recruitment can be explained by a single chemoattractant model. We also use Bayesian model comparison to compare different chemoattractant production and release dynamics. Furthermore, we extend the model to infer subpopulations of immune cells with different migratory behaviour without labelling.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/CDEV/CDEV-CT01.html#author4"><strong class="card-body-subgroup">Philipp Thomas</strong></a>
		    	 
		    		<em>Imperial College London</em>
		    	
		    	
		    		
		    			</br>"Exact solutions for stochastic gene expression in growing cell populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The chemical master equation and the stochastic simulation algorithm are widely used to model reaction kinetics inside living cells. It is sometimes assumed that cell growth and division can be modelled through a  chemical master equation with effective dilution reactions and extrinsic noise sources. We here re-examine this paradigm by developing an analytical agent-based framework of growing and dividing cells. Apart from the common intrinsic noise contribution the theory predicts extrinsic noise without the need to introduce fluctuating rate constants. Instead, extrinsic fluctuations arise from the population structure of a growing cell population that includes cell cycle fluctuations, differences in cell age and cell size variability. We show that, surprisingly, the solution of the chemical master equation - including effective dilution reactions and static extrinsic noise - exactly agrees with the agent-based formulation when the network under study exhibits stochastic concentration homeostasis, a novel condition that generalises concentration homeostasis in deterministic systems to higher order moments and distributions. We illustrate that this result allows us to exactly solve agent-based models for a range of common gene expression networks inside growing cells.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ECOP/ECOP-CT01.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>CT01-ECOP: </small></br>ECOP Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ECOP/ECOP-CT01.html#author1"><strong class="card-body-subgroup">Abdennasser Chekroun</strong></a>
		    	 
		    		<em>University of Tlemcen</em>
		    	
		    	
		    		
		    			</br>"Traveling waves of a differential-difference diffusive Kermack-McKendrick epidemic model with age-structured protection phase"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider a general class of diffusive Kermack-McKendrick SIR epidemic models with an age-structured protection phase with limited duration, for example due to vaccination or drugs with temporary immunity. A saturated incidence rate is also considered which is more realistic than the bilinear rate. The characteristics method reduces the model to a coupled system of a reaction-diffusion equation and a continuous difference equation with a time-delay and a nonlocal spatial term caused by individuals moving during their protection phase. We study the existence and non-existence of non-trivial traveling wave solutions. We get almost complete information on the threshold and the minimal wave speed that describes the transition between the existence and non-existence of non-trivial traveling waves that indicate whether the epidemic can spread or not. We discuss how model parameters, such as protection rates, affect the minimal wave speed. The difficulty of our model is to combine a reaction-diffusion system with a continuous difference equation. We deal with our problem mainly by using Schauder's fixed point theorem. More precisely, we reduce the problem of the existence of non-trivial traveling wave solutions to the existence of an admissible pair of upper and lower solutions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ECOP/ECOP-CT01.html#author2"><strong class="card-body-subgroup">Laura Wadkin</strong></a>
		    	 
		    		<em>Newcastle University</em>
		    	
		    	
		    		
		    			</br>"Mathematical modelling of the spread of tree disease through forests"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The past decades have seen a dramatic rise in the number of emerging diseases of plants and trees across the world. These diseases threaten the survival of native trees and have huge social, economic, and environmental impacts. The Department for Environmental, Food and Rural Affairs have highlighted the importance of mathematical modelling in developing robust management policies to minimise the impacts of these threats. We are working to mathematically model the spread of tree diseases, using a combination of agent-based models, partial differential equations, and statistical inference techniques. The aim is to combine local lattice modelling approaches with global continuum models to perform systemic modelling and parameter inference of past and present tree epidemics in the mainland UK. The results can be used to deepen our understanding of the process of tree disease spread and crucially, explore intervention and management strategies to find the best methods of stopping the disease spread.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ECOP/ECOP-CT01.html#author3"><strong class="card-body-subgroup">Laura Jimenez</strong></a>
		    	 
		    		<em>University of Hawaii</em>
		    	
		    	
		    		
		    			</br>"Statistical models to estimate the fundamental niche of a species using occurrence data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The fundamental niche of a species is the set of environmental conditions that allow the species to survive in the absence of biotic interactions and dispersal limitations. Estimating the center (i.e., the optimal environmental conditions for the species) and extent of the fundamental niche is of great importance when the fitted models are used to predict the effects of climate change on the geographic distribution of the species. However, most of the existing approaches to estimate niches use occurrence samples that are biased, and often fit complex models that are not a biologically realistic representation of the fundamental niche' border. Occurrence samples come from the realized niche (a subset of the fundamental niche that includes biotic interactions and dispersal limitations) and may not represent the full environmental potentiality of a species; samples may be biased towards well-represented regions of niche space. I will present two new models to estimate the fundamental niche of a species that use occurrence data and assume a simple, biologically realistic shape for the fundamental niche. I will show how to incorporate known tolerance ranges for the species into the models and how to account for environmental biases in the samples.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ECOP/ECOP-CT01.html#author4"><strong class="card-body-subgroup">Lee Altenberg</strong></a>
		    	 
		    		<em>University of Hawai`i at Mānoa</em>
		    	
		    	
		    		
		    			</br>"Going Against the Flow: Selection for Counter-Current Dispersal in Gyres"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Much attention has been given to the 'drift paradox' for river organisms: how populations in streams can maintain themselves despite being constantly swept downstream.  Here we shall consider a different situation:  where circular currents produce time-irreversible dispersal dynamics.  We will see that when there is environmentally produced cyclical dispersal among habitats with spatial variability in quality, organisms that disperse against the cyclical flows will have an aggregate population growth advantage.  These results are obtained by applying some classical results from spectral theory, including theorems by Karlin and Levinger.  Temporal variation in habitat quality or dispersal is not addressed.  Open problems for further work include the degree to which these result extend to dispersal that is only partially or approximately counter-current.  The widespread occurrence of positive rheotaxis among ocean organisms may conceivably be a manifestation of these selection dynamics.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/EVOP/EVOP-CT01.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>CT01-EVOP: </small></br>EVOP Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/EVOP/EVOP-CT01.html#author1"><strong class="card-body-subgroup">Hannah Götsch</strong></a>
		    	 
		    		<em>Faculty of Mathematics - University of Vienna, Vienna Graduate School of Population Genetics</em>
		    	
		    	
		    		
		    			</br>"A mathematical model for the adaptation of a quantitative trait in a panmictic population"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The genetic architecture of a quantitative trait ranges from selective sweeps at few loci to subtle allele frequency shifts at many loci. By genetic architecture we mean the number of loci responding evolutionarily to adaptation, their mutation rates and the distribution of their mutational effects, their linkage relations, as well as their epistatic interactions. Höllinger et al. [1] showed for a panmictic population that the population-scaled background mutation rate determines crucially the shape of the mutant allele frequency distribution at the end of the adaptive phase. Remarkably, the strength of selection is not important as long as the locus effects are all the same.We report about results how variation among locus effects alter these findings. For the infinite sites model, we present analytical results for the locus-based distribution of the mutants as well as the phenotypic mean and variance, which are based on a combination of branching process theory (for the initial stochastic phase) and deterministic theory. They are compared with comprehensive computer simulations.[1] I. Höllinger, P.S. Pennings, and J. Hermisson, Polygenic adaptation: From sweeps to subtle frequency shifts, PLOS Genetics, 15: 1–26, 2019.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT01.html#author2"><strong class="card-body-subgroup">Alexander B. Brummer</strong></a>
		    	 
		    		<em>Department of Computational and Quantitative Medicine, Division of Mathematical Oncology, Beckman Research Institute, City of Hope National Medical Center</em>
		    	
		    	
		    		
		    			</br>"Cancer as a model system for testing metabolic scaling theory"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Biological allometries, such as the scaling of metabolism to mass, are hypothesized to result from natural selection to maximize how vascular networks fill space yet minimize internal transport distances and resistance to blood flow.  Metabolic scaling theory argues two guiding principles—conservation of fluid flow and space-filling fractal distributions—describe a diversity of biological networks and predict how the geometry of these networks influences organismal metabolism.  Yet, absent from these past efforts are studies that independently measure both metabolic function and vascular form.  We present simultaneous and consistent measurements of metabolic scaling exponents from clinical lung cancer imaging, and identify potential quantitative imaging biomarkers indicative of tumor growth.We analyze 65 clinical PET-CT scans of patients with non-small cell lung carcinoma.  Examination of the scaling of maximum standard uptake value with metabolic tumor volume, and metabolic tumor volume with gross tumor volume, yields metabolic scaling exponents of 0.64 (0.20) and 0.70 (0.17), respectively. We compare these to the value of 0.85 (0.06) derived from the geometric scaling of the tumor-supplying vasculature.  These results (1) identify imaging biomarkers in vascular geometry related to blood volume and flow and (2) inform energetic models of growth and development for tumor forecasting.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/EVOP/EVOP-CT01.html#author3"><strong class="card-body-subgroup">Bo Zhang</strong></a>
		    	 
		    		<em>Oklahoma State University</em>
		    	
		    	
		    		
		    			</br>"How to integrate mechanical treatment and biological control to improve field treatment efficiency on invasions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Projecting controlling outcomes of different management strategies on invasive populations has broad implications in field management. Different to herbicide usage that may cause environmental pollution and non-target effects on native plants, nonchemical methods, have shown great targeted effectiveness on invasion. However, an interesting and important question remains unclear is that how to decrease the repetition of nonchemical treatments. One possible approach is to integrate nonchemical treatments with biological control agents, which can attack and limit invasion spread after being established in the field. We hypothesize that applying nonchemical methods to remove occurring invasive plant while establishing biological control agents, then using the established biological control agents to limit future regrowth of invasive plant will decrease the use of nonchemical treatments. We developed a spatial modeling framework, including their dispersal processes, to capture population dynamics change under various strategies of control. We found that applying nonchemical treatment in a higher frequency with smaller treated areas per time is a more efficient approach than vice versa. More importantly, we emphasized that a high biological control efficiency can continuously decrease the requirement of repeated treatment of nonchemical methods and maintain the invasive population at a low level.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/IMMU/IMMU-CT01.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>CT01-IMMU: </small></br>IMMU Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/IMMU/IMMU-CT01.html#author1"><strong class="card-body-subgroup">Aaron Meyer</strong></a>
		    	 
		    		<em>University of California, Los Angeles</em>
		    	
		    	
		    		
		    			</br>"Developing a mechanistic view of mixed IgG antibody immune effector responses"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			IgG antibodies bind antigen targets and then interact with Fcγ receptors (FcγR) on effector cells to direct cellular responses. Effector responses involve multiple cell types and processes (e.g., cytokines, phagocytosis) making it a systems-level challenge to precisely engineer these responses. We previously showed a multivalent binding model could accurately predict in vitro binding to synthetic complexes and in vivo anti-tumor antibody response.Here, we extend this work to predict the binding and immune response to complexes with combinations of Fc domains using a multivalent, multi-receptor, and multi-ligand model. We first validated the accuracy of this model through binding experiments using synthetic IgG mixtures. Applying this model, we could predict the effector-elicited target depletion of individual IgG and their combinations in mouse models of both ITP and melanoma. Our model correctly inferred that Kupffer cells are essential for platelet depletion in ITP, and specifically identified a FcγRIIBhigh subpopulation with outsized importance. Exploring the predicted effects of IgG combinations, we develop a framework for drug additivity. IgG synergy cannot occur with antibodies of identical antigen binding but antagonism is widespread through Fc receptor competition. These results demonstrate a suite of capabilities to more precisely engineer antibodies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/IMMU/IMMU-CT01.html#author2"><strong class="card-body-subgroup">Yafei Wang</strong></a>
		    	 
		    		<em>Indiana University Bloomington</em>
		    	
		    	
		    		
		    			</br>"Multiscale modeling of SARS-CoV-2 infectious dynamics and antiviral drugs intervention"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The 2019 novel coronavirus, SARS-CoV-2, is a pathogen of critical significance to international public health. Antiviral drugs and vaccines are currently under development and test to address this pandemic. However, the knowledge of dynamics of viral endocytosis, replication, single-cell response and pharmacodynamics is still limited. Therefore, the establishment of such dynamics is very urgent for understanding how SARS-CoV-2 infectious spread and finding an optimal therapeutic strategy. More importantly, this pandemic will likely not be the last one as new pathogens emergency, so we may be able to reuse the dynamics for faster response to another pandemic in the future. In this talk, we will present our work on multiscale modeling of SARS-CoV-2 infectious spread and antiviral drugs intervention (through an agent-based modeling approach-PhysiCell). This work is an extension of a multi-institution, multi-disciplinary coalition of over 40 mathematical biologists, immunologists, virologists, pharmacologists, and others to build a comprehensive multiscale model of SARS-CoV-2 infection dynamics and immune response. The model of this work can be run on a cloud-hosted platform at: https://nanohub.org/resources/virion2pbpd.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/IMMU/IMMU-CT01.html#author3"><strong class="card-body-subgroup">Wenjing Zhang</strong></a>
		    	 
		    		<em>Texas Tech University</em>
		    	
		    	
		    		
		    			</br>"Deterministic and Stochastic in-host Tuberculosis Models for Bacterium-directed and Host-directed Therapy Combination"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The goal of this paper is to investigate in-host tuberculosis models to provide insights into therapy development. Focusing on therapy-targeting parameters, the parameter regions for different disease outcomes are identified from an established ODE model. Interestingly, the ODE model also demonstrates that the immune responses can both benefit and impede disease progression, which depend on the number of bacteria engulfed and released by macrophages. We then develop two Ito SDE models, which consider demographic variations at the cellular level only and  environmental variations during therapies along with demographic variations. The SDE model with demographic variation suggests that stochastic fluctuations at the cellular level have significant influences on (1) the T-cell population in all parameter regions, (2) the bacterial population  when parameters locate in the region with multiple disease outcomes, and (3) the uninfected macrophage population in parameter region representing active disease. Further, considering environmental variations from therapies, the second SDE model suggests that disease progression is more likely to be inhibited if therapies (1) can have fast return rates and (2) are able to bring parameter values into the disease clearance regions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/IMMU/IMMU-CT01.html#author4"><strong class="card-body-subgroup">Aniruddha Deka</strong></a>
		    	 
		    		<em>Pennsylvania State University</em>
		    	
		    	
		    		
		    			</br>"Pathogen competition and mutant invasion in face of human choice in vaccination:"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Competition between multiple strain for vaccine preventable diseases often leads to exclusion of some pathogens, while it may influence the invasion of an emerging mutant in the population. Previous studies have shown that basic reproductive numbers among multiple strains are sufficient to predict which strains will invade a population. But human vaccination decision plays crucial role in shaping the type of strain that will invade or persist or get eliminated. Humans adapt to changing behavior or virulence of strains and for highly transmissible strains, they vaccinate at a faster rate due to higher perceived severity from the diseases. This on the other-hand gives scope for mutant strains to invade new number of susceptible in the population. In our study, we have coupled game dynamic model of vaccination choice and compartmental disease transmission model of two-strains to explore invasion, extinction and persistence of a mutant in the population which have a lower reproduction rate than the resident one. We illustrate that higher perceived strain severity and lower perceived vaccine efficacy are necessary conditions for persistence of a mutant strain. Numerically we explore these invasion and persistence analyses under asymmetric cross-protective immunity of these strains.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MEPI/MEPI-CT01.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>CT01-MEPI: </small></br>MEPI Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MEPI/MEPI-CT01.html#author1"><strong class="card-body-subgroup">Michael Irvine</strong></a>
		    	 
		    		<em>Simon Fraser University</em>
		    	
		    	
		    		
		    			</br>"Quantifying transmissibility of COVID-19 and the impact of intervention in long-term care facilities"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Estimates of the basic reproduction number (R0) for Coronavirus disease 2019 (COVID-19) are particularly variable in the context of transmission within locations such as long-term health care (LTHC) facilities. We sought to characterise the heterogeneity of R0 across known outbreaks within these facilities. We used a unique comprehensive dataset of all outbreaks that have occurred within LTHC facilities in British Columbia, Canada. We estimated R0 with a Bayesian hierarchical dynamic model of susceptible, exposed, infected, and recovered individuals, that incorporates heterogeneity of R0 between facilities. We further compared these estimates to those obtained with standard methods that utilize the exponential growth rate and maximum likelihood. The total size of an outbreak varied dramatically, with a range of attack rates of 2%-86%. The Bayesian analysis provides more constrained overall estimates of R0 = 2.83 (90% CrI 0.25--7.19) than standard methods, with a range within facilities of 0.66 - 10.06. We further estimated that intervention led to 67% (56%-73%) of all cases being averted within the LTHC facilities. Understanding the risks and impact of intervention are essential in planning during the ongoing global pandemic, particularly in high-risk environments such as LTHC facilities.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT01.html#author2"><strong class="card-body-subgroup">Carlo Davila-Payan</strong></a>
		    	 
		    		<em>Centers for Disease Control and Prevention</em>
		    	
		    	
		    		
		    			</br>"Analysis of the yearly transition function in measles disease modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Globally, there were an estimated 9.8 million measles cases and 207,500 measles deaths in 2019. As the worldwide effort to eliminate measles continues, modeling remains a valuable tool for public health decision makers and program implementers. This study presents a novel approach to the use of a yearly transition function to account for the effects of the timing of vaccination (based on vaccination schedules for different age groups) and disease seasonality on the yearly number of measles cases in a given country.Our methodology adds to and expands on the existing modeling framework of Eilertson et al. (Stat. Med. 2019; 38: 4146-4158) by developing explicit functional expressions for each underlying component of the transition function in order to adjust for the temporal interaction between vaccination and exposure to disease.  Assumption of specific distributional forms provides multipliers that can be applied to estimated yearly counts of cases and vaccine doses to estimate impacts more precisely on population immunity. These new model features provide the ability to forecast and compare the effects of different vaccination timing scenarios and seasonality of transmission on the expected disease incidence. Although this application is to measles, the method has potential relevance to modeling other vaccine-preventable diseases.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT01.html#author3"><strong class="card-body-subgroup">Luis Manuel Munoz-Nava</strong></a>
		    	 
		    		<em>Center for Research and Advanced Studies</em>
		    	
		    	
		    		
		    			</br>"‘Learning Bubbles’ are an effective and safe alternative to schools reopening during the COVID-19 pandemic"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			According to estimates of the UNESCO, the COVID-19 pandemic has affected more than 1.4 billion (aprox. 84 %) students worldwide. In many countries, schools have remained closed for more than a year and this situation is likely to persist for several additional months before local vaccination programs start to slow down virus propagation. While policymakers debate on how and when children should go back to school buildings, closures are expecting to have a profound and long-term impact in children education, nutrition, social skills, and mental health, as well as in the economy and psychosocial behavior of students and their families. As an alternative to reopening of schools, 'Learning Bubbles' are groups of a few children that their parents voluntarily set-up for in-person instruction either from one of the parents or an external tutor. 'Learning bubbles' were very popular in the United States started remotely the academic year in the Fall of 2020, but to the best of our knowledge, a report on the effectiveness of 'learning bubbles' in mitigating the propagation of the COVID-19 disease has not been analyzed. We developed a mathematical model of 'learning bubbles' and discuss its effectiveness in mitigating the disease compared with schools reopening.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MEPI/MEPI-CT01.html#author4"><strong class="card-body-subgroup">Glenn Ledder</strong></a>
		    	 
		    		<em>University of Nebraska-Lincoln</em>
		    	
		    	
		    		
		    			</br>"A Model for COVID-19 with Limited Vaccination"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Now that vaccines for COVID-19 are available and distribution has begun, a critical question arises: To what extent do protective measures need to be maintained as more people are vaccinated?  Addressing this question requires careful attention to the way vaccination is incorporated into the model.  We augment our SEAIHRD (Susceptible, Exposed, Asymptomatic, (symptomatic) Infectious, Hospitalized, Recovered, Deceased) model by breaking up the susceptible class into a standard (S)usceptible class and a (P)re-vaccinated class, with proportions determined by a vaccine acceptance parameter.  Susceptible and pre-vaccinated individuals move to the Exposed through infection in the standard way.  In addition, a vaccination process moves individuals directly out of the pre-vaccinated class at a rate that follows a Michaelis-Menten mechanism; that is, the rate is linear when the pre-vaccinated class is small but quickly saturates due to limitations in the distribution speed.  The most recent update accounts for prioritization of high-risk people.  Most individuals who leave the pre-vaccinated class move into the recovered class, but a small fraction move back to the standard susceptible class, representing the probability of failing to mount a proper immune response.  We use the model to investigate the impact of reduced compliance with protective measures.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT01.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT01-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT01.html#author1"><strong class="card-body-subgroup">Wayne Hayes</strong></a>
		    	 
		    		<em>UC Irvine</em>
		    	
		    	
		    		
		    			</br>"The One True Way to use GO terms to evaluate Network Alignments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Sequence alignment has contributed immensely to our understanding of biology, evolution, and disease. While the genome encodes recipes for making proteins, the function of many proteins remains elusive. Since the function of a protein is intimately tied to its interaction partners, the topology of protein-protein interaction (PPI) networks holds promise as way to decode function. Topologically-driven network alignment attempts to find the best mapping between the PPI networks of two species by finding the greatest amount of common network topology. However, network alignment research is still in its infancy and there are dozens of proposed methods but no objective, mathematically rigorous methods to compare their results.  Here we propose a rigorous, formal method to compute the p-values of shared GO terms between pairs of proteins found by a network alignment, compared to random alignments. We compare our p-values to billions of actual random alignments to demonstrate that the p-values are correct within statistical uncertainty of the sample random alignments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author2"><strong class="card-body-subgroup">Giulia Palermo</strong></a>
		    	 
		    		<em>UCR</em>
		    	
		    	
		    		
		    			</br>"Harnessing graph theory to decrypt the allosteric mecahnism in CRISPR-Cas9"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			CRISPR-Cas9 is a bacterial adaptive immune system that emerged as the centerpiece of a transformative genome editing technology. In this system, an intriguing allosteric communication has been suggested to control the DNA cleavage activity through the flexibility of the catalytic HNH domain. Here, we report about the use of molecular dynamics and graph theory-based analysis methods to describe the structural and dynamic determinants of the allosteric signaling in the CRISPR-Cas9 complex. Network models derived from graph theory reveal the existence of a contiguous dynamic pathway that enables the information transfer across the HNH domain. This pathway spans HNH from the region interfacing the RuvC nuclease and propagates up to the DNA recognition lobe in the full-length CRISPR-Cas9, such transferring the signal of DNA binding at the nuclease domains for concerted cleavages of the two DNA strands. These findings reveal the mechanism of signal transduction within the CRISPR-Cas9 nuclease and pose the basis for the complete mapping of the allosteric pathway, and of its role in the DNA on-target specificity, helping engineering efforts aimed at improving the genome editing capability of CRISPR-Cas9.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author3"><strong class="card-body-subgroup">Dennis Manjaly Joshy</strong></a>
		    	 
		    		<em>UC Santa Barbara</em>
		    	
		    	
		    		
		    			</br>"A Koopman Operator Approach for Genetic Circuit Design"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider the problem of genetic circuit design to achieve an arbitrary data-driven or function-based performance specification.   We review the open nature of this nonlinear design problem and its relation to the optimal and robust controller synthesis problem.  We show how a class of biological networks, modeled with first order, zeroth order, and Hill function dynamics can be represented with a Koopman operator to yield a linear representation of system dynamics on a space of functions.  This formulation allows us to directly solve the controller synthesis problem to meet a given performance specification as an optimization problem on a particular physical basis of observable functions.  We demonstrate our approach on the optimization of a positive amplifier circuit in bacteria, showing how design recommendations from our controller synthesis algorithm can be translated to DNA sequence-level specification.  These results solve an outstanding problem in genetic circuit design - synthesis of closed-loop systems to meet a target performance specification.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author4"><strong class="card-body-subgroup">Ying Zhang</strong></a>
		    	 
		    		<em>Brandeis University</em>
		    	
		    	
		    		
		    			</br>"Immersed Boundary Simulations of Red Blood Cells Near Vessel Walls"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Platelets constitute an essential component of human blood due to their role in the formation of hemostatic plug and thrombus. The occurrence of these biological phenomena requires platelets stay within close proximity to the vessel walls, initiating platelet-wall interaction. It has been understood that the red blood cells (RBCs) play an important role in platelet near-wall excess. Healthy RBCs are highly deformable objects, and thus can acquire lift forces from vessel walls from their deformation to propel them away from the wall, a phenomenon known as wall-induced migration. Migration of RBCs away from the wall leads to the formation of a cell-depleted layer near the wall, which has a large effect on the motion of platelets. Here we use the immersed boundary method to investigate the influence of cell stiffness and shape on the wall-induced migration. In particular, we focus on analyzing how lift force and mobility change over time when a RBC is placed close to the wall. Our preliminary results suggest that deformation of a RBC leads to a larger lift force when the RBC is closer to the wall, increasing the likelihood of RBCs migrating away from the vessel wall.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MMPB/MMPB-CT01.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>CT01-MMPB: </small></br>MMPB Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MMPB/MMPB-CT01.html#author1"><strong class="card-body-subgroup">Brendan Fry</strong></a>
		    	 
		    		<em>Metropolitan State University of Denver</em>
		    	
		    	
		    		
		    			</br>"A hybrid model for metabolic signaling in the human retinal microcirculation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Impaired blood flow regulation and oxygenation have been implicated as contributors to glaucomatous damage in the retina. Here, a mathematical model is presented that combines an image-based heterogeneous representation of the retinal arteriolar vasculature with a compartmental description of the downstream capillaries and venules.  The arteriolar model of the human retina is extrapolated from a previous mouse model based on confocal microscopy images.  This hybrid model is used to predict blood flow and oxygenation throughout the entire retinal microcirculation; in addition, a metabolic wall signal is calculated in each vessel from blood and tissue oxygen levels, and is conducted upstream to communicate the metabolic status of the retina to the arterioles.  Model results predict a wide range of metabolic signals generated throughout the microvascular network, dependent both on oxygen levels and vascular path lengths.  Overall, the model predicts that a higher metabolic wall signal is generated in pathways with a lower oxygen level at the terminal arteriole.  This model framework will be used in the future to simulate blood flow regulation in a realistic, spatially non-uniform representation of the human retina, in order to assess the role of metabolic blood flow dysregulation in glaucoma.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MMPB/MMPB-CT01.html#author2"><strong class="card-body-subgroup">Thomas Bury</strong></a>
		    	 
		    		<em>McGill University</em>
		    	
		    	
		    		
		    			</br>"Long ECGs reveal rich and robust dynamical regimes in patients with frequent premature ventricular complexes."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Heart disease is one of the leading causes of disability and death. One manifestation of heart disease is abnormal heart rhythms, called arrhythmia. A very common arrhythmia consists of abnormal extra heart beats called premature ventricular complexes (PVCs). Though considered benign in most cases, recent studies have shown that frequent PVCs pose an increased risk for more serious arrhythmia that can lead to sudden cardiac death. Risk stratification for these patients remains a significant challenge in part since the mechanism generating the PVCs is usually unknown. In this talk, we will show how analysis of multi-day ECGs reveal robust dynamical regimes in PVC dynamics that vary as a function of heart rate and hour of the day. This analysis facilitates the development of basic mathematical models that can help reveal the underlying mechanism of PVCs. With the current advances in wearable technology and corresponding influx of ECG data, such approaches can bring about a dynamics-based personalised medicine.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MMPB/MMPB-CT01.html#author3"><strong class="card-body-subgroup">Jeungeun Park</strong></a>
		    	 
		    		<em>University of Cincinnati</em>
		    	
		    	
		    		
		    			</br>"A swimming strategy of polarly-flagellated bacteria"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Flagellar bacteria swim through fluid by rotating their flagella that are connected to rotary motors in their cell wall. The physical, geometrical, and material properties of flagella characterize bacterial swimming patterns. In this talk, we present a mathematical model of a lophotrichous bacterium swimming through fluid. We introduce a recently reported swimming mode in which a bacterium undergoes a slow swimming phase by wrapping its flagella around the cell body. By using our mathematical model, we investigate the mechanism of wrapping motion, and suggest benefits of the motion in bacterial native habitats. Furthermore, we compare our numerical examples with experimental observations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MMPB/MMPB-CT01.html#author4"><strong class="card-body-subgroup">Dongheon Lee</strong></a>
		    	 
		    		<em>Department of Biomedical Engineering, Duke University</em>
		    	
		    	
		    		
		    			</br>"Hybrid Data-driven Mechanistic Modeling Approach to Describe Uncertain Intracellular Signaling Pathways"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Developing an accurate mechanistic model is important in analyzing an intracellular signaling pathway. However, a model is difficult to be developed since it requires in-depth understandings. Since underlying mechanisms are not fully understood, significant discrepancy exists between predicted and actual signaling dynamics. Motivated by these considerations, this work proposes a hybrid modeling approach that combines a mechanistic model and an artificial neural network (ANN) model so that predictions of the hybrid model surpass those of the original model. First, the proposed approach determines an optimal subset of model states whose dynamics should be corrected by the ANN by examining the correlation between each state and outputs through relative order. Second, an L2-regularized least-squares problem is solved to infer values of the correction terms that are necessary to minimize the discrepancy between the model predictions and available measurements. Third, an ANN is developed to generalize relationships between the values of the correction terms and the system dynamics. Lastly, the original first-principle model is coupled with the developed ANN to finalize the hybrid model development so that the model will possess generalized prediction capabilities while retaining the model interpretability.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/ONCO/ONCO-CT01.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>CT01-ONCO: </small></br>ONCO Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/ONCO/ONCO-CT01.html#author1"><strong class="card-body-subgroup">Andrei S Rodin</strong></a>
		    	 
		    		<em>City of Hope</em>
		    	
		    	
		    		
		    			</br>"Dissecting Response to Cancer Immunotherapy by Applying Bayesian Network Analysis to Flow Cytometry Data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer immunotherapy, specifically immune checkpoint blockade, has been found to be effec-tive in the treatment of metastatic cancers. However, only a subset of patients achieve clinical responses. Elucidating pretreatment biomarkers predictive of sustained clinical response is a major research priority. Another research priority is evaluating changes in the immune system before and after treatment in responders vs. nonresponders. Our group has been studying im-mune networks as an accurate reflection of the global immune state. Flow cytometry (FACS, fluorescence-activated cell sorting) data characterizing immune cell panels in peripheral blood mononuclear cells (PBMC) from gastroesophageal adenocarcinoma (GEA) patients were used to analyze changes in immune networks in this setting. Here, we describe a novel computational pipeline to perform secondary analyses of FACS data using systems biology/machine learning techniques and concepts. The pipeline is centered around comparative Bayesian network anal-yses of immune networks and is capable of detecting strong signals that conventional methods (such as FlowJo manual gating) might miss. Future studies are planned to validate and follow up the immune biomarkers (and combinations/interactions thereof) associated with clinical re-sponses identified with this computational pipeline.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author2"><strong class="card-body-subgroup">Tatiana Miti</strong></a>
		    	 
		    		<em>Moffitt Cancer Center & Research Institute</em>
		    	
		    	
		    		
		    			</br>"Integrating Spatial Point Pattern Analysis and Agent Based Modeling for Studies of Stroma Sheltering Effects on Tumor"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Over the past two decades, the advancement in visualization and analysis of molecular and cellular data led to the development of highly efficient, targeted cancer drugs. However, cancer relapse occurs frequently, indicating that the tumor microenvironment plays a crucial role in treatment response potentially sheltering tumor cells during drug administration. We studied the eco-evolutionary dynamics of stroma-proliferating tumor cells interaction via point pattern analysis methods and spatial agent-based modeling (ABM). We characterized the spatial extent and amplitude of stroma shielding in the presence and absence of treatment using various pairwise distance methods adapted from ecology and geology. The spatial distributions of tumor cells, their proliferation rates, and the identified stroma protective effects were used to parametrize the ABM and simulate the spatio-temporal dynamics of tumor growth. Our preliminary results show that stroma-proliferating tumor cell clustering is considerably higher under treatment than in control samples. Moreover, stroma's protective effect during treatment is limited to cells that are either in direct contact with stromal cells or in their immediate proximity, suggesting a paracrine mediate effect. We expect our results to lead to novel therapeutic interventions that aim to shift eco-evolutionary dynamics rather than maximize short-term tumor cell killing efficiency.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author3"><strong class="card-body-subgroup">Robert Noble</strong></a>
		    	 
		    		<em>City, University of London</em>
		    	
		    	
		    		
		    			</br>"Explaining modes of tumour evolution"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the mode of tumour evolution is important for accurate prognosis and designing effective treatment strategies. Whereas selective sweeps are prevalent during early tumour growth, later stages exhibit either sparse branching or effectively neutral evolution. The causes of these different patterns remain poorly understood. I will present a new model for determining the probability of selective sweeps versus clonal interference in one-, two- and three-dimensional expanding populations. The solutions of this model are surprisingly simple mathematical expressions that are independent of mutation rate. Given parameter values obtained with human cancer data, the model offers to explain why selective sweeps are rare except when tumours are relatively very small. I will discuss these results in the context of additional computational modelling and new indices for classifying modes of tumour evolution that I and my coauthors have developed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/ONCO/ONCO-CT01.html#author4"><strong class="card-body-subgroup">Erdi Kara</strong></a>
		    	 
		    		<em>Texas Tech University Mathematics and Statistics</em>
		    	
		    	
		    		
		    			</br>"Diffusion Tensor Imaging (DTI) Based Drug Diffusion - Population Model for Solid Tumors"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this work, we study the effect of drug distribution on tumor cell death when the drug is internally injected in the tumorous tissue. We derive a full 3-dimensional inhomogeneous – anisotropic diffusion model. To capture the anisotropic nature of the diffusion process in the model, we use an MRI data of a 35-year old patient diagnosedwith Glioblastoma multiform(GBM) which is the most common and most aggressive primary brain tumor. Afterpreprocessing the data with a medical image processing software, we employ finite element method in MPI-basedparallel setting to numerically simulate the full model and produce dose-response curves. We then illustrate theapoptosis (cell death) fractions in the tumor region over the course of simulation and proposed several ways toimprove the drug efficacy. Our model also allows us to visually examine the toxicity. Since the model is builtdirectly on the top of a patient-specific data, we hope that this study will contribute to the individualized cancertreatment efforts from a computational bio-mechanics viewpoint.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Contributed Talk Session - CT01
										</br>
										Monday, June 14 at 03:15pm (PDT)</br>Monday, June 14 at 11:15pm (BST)</br>Tuesday, June 15 07:15am (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>

















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
