<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Methods for Biological Modeling Subgroup (MFBM) | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Methods for Biological Modeling Subgroup (MFBM) | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Methods for Biological Modeling Subgroup (MFBM)" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MFBM/" />
<meta property="og:url" content="http://localhost:4000/MFBM/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MFBM/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Methods for Biological Modeling Subgroup (MFBM)","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Methods for Biological Modeling Subgroup (MFBM)">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Methods for Biological Modeling Subgroup (MFBM)" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Methods for Biological Modeling Subgroup (MFBM)" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
	    <div class="row">
	        <div class="col-md-12  text-center">
<!-- 		        <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
		        <h1 class="mb-3">Methods for Biological Modeling Subgroup (MFBM)</h1>
		        
<!-- 		        <div class="cleanslate w24tz-current-time w24tz-small" style="display: inline-block !important; visibility: hidden !important; min-width:200px !important; min-height:100px !important;"><p><a href="//24timezones.com/Tampa/time" style="text-decoration: none" class="clock24" id="tz24-1596739689-c1410-eyJob3VydHlwZSI6MTIsInNob3dkYXRlIjoiMSIsInNob3dzZWNvbmRzIjoiMCIsImNvbnRhaW5lcl9pZCI6ImNsb2NrX2Jsb2NrX2NiNWYyYzUwNjllNmUzMSIsInR5cGUiOiJkYiIsImxhbmciOiJlbiJ9" title="Tampa local time" target="_blank">US Eastern Time Zone (EDT)</a></p><div id="clock_block_cb5f2c5069e6e31"></div></div>
<script type="text/javascript" src="//w.24timezones.com/l.js" async></script>
	        	</br> -->
		        
		        
	        </div>
	    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			<h6>Jump to:</h6>
<ul class="pagination text-left">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#"><span class="text-MFBM">Minisymposia</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#CT"><span class="text-MFBM">Contributed Talks</span></a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="#Posters"><span class="text-MFBM">Posters</span></a></li>
</ul>

			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="MS1">Sub-group minisymposia</h1>
		                
<!-- 		                	 -->	                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS01.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS01-MFBM: </small></br>Mathematical Modeling Applied to Pharmaceutical Sciences Problems</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Carl Panetta (St. Jude Children's Research Hospital, US), Helen Moore (Laboratory for Systems Medicine, University of Florida, US)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS01.html#author1"><strong class="card-body-subgroup">C.J. Musante</strong></a>
		    	 
		    		<em> (Pfizer, US)</em>
		    	
		    	
		    		
		    			</br>"A Few Open Mathematical Modeling Problems in Drug Discovery & Development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Now, more than ever, pharmaceutical companies are relying on mathematical modeling & simulation to inform drug discovery and development decisions. In many cases, particularly for novel compounds, targets, and/or combinations, modelers must rely on incomplete data and/or extrapolation beyond existing data to address key questions, such as dose and dose regimen selection for clinical trials. In this talk, I will present a few case studies and related mathematical challenges that my team has faced and discuss why developing a better understanding of these problems is important in the context of drug discovery & development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS01.html#author2"><strong class="card-body-subgroup">Jane Bai</strong></a>
		    	 
		    		<em> (FDA, US)</em>
		    	
		    	
		    		
		    			</br>"Conducting sensitivity analysis and uncertainty analysis for QSP needs more than mathematical computation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In quantitative systems pharmacology (QSP) modeling, sensitivity analysis is often conducted to identify a set of sensitive parameters to avoid overparameterization for robust calibration and validation. There are different global sensitivity analysis methods to choose from. Furthermore, for each model output, sensitivity analysis generates a rank-ordered list. Combining individual lists of rank-ordered sensitive parameters from all model outputs in a QSP model to obtain the final list may be subject to modeler’s judgement. Capturing variability in a trial population through uncertainty analysis and virtual patient trials can improve the predictive performance of a model and inform trial designs for a drug development program. However, multiple different algorithms can be used. This talk will discuss methodological considerations when applying sensitivity analysis and uncertainty analysis to QSP modeling for drug development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS01.html#author3"><strong class="card-body-subgroup">Freya Bachmann</strong></a>
		    	 
		    		<em> (Department of Mathematics and Statistics, University of Konstanz, Germany)</em>
		    	
		    	
		    		
		    			</br>"Computing the Individualized Optimal Drug Dosing Regimen Using Optimal Control"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Providing the optimal dosing strategy of a drug for an individual patient is an important task in pharmaceutical sciences and daily clinical application. By solving an optimal control problem (OCP) especially tailored to pharmacokinetic-pharmacodynamic (PKPD) models the optimal individualized dosing regimen can be computed for substantially different scenarios with various routes of administration.
The aim is to compute a control that brings the underlying system as closely as possible to a desired reference state by minimizing an objective function. In PKPD modeling the controls are the administered doses and the reference state can be the disease progression. Therefore, the objective function which shall be minimized is quantifying the difference between a desired disease state and the actual state generated by a particular treatment. Drug administration at certain time points gives a finite number of discrete controls, the drug doses, determining the drug concentration and its effect on the disease state. Hence, it is possible to construct a finite-dimensional OCP depending only on the doses and apply robust quasi-Newton algorithms from finite-dimensional optimization.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS01.html#author4"><strong class="card-body-subgroup">Tongli Zhang</strong></a>
		    	 
		    		<em> (Department of Pharmacology & Systems Physiology, College of Medicine, University of Cincinnati, US)</em>
		    	
		    	
		    		
		    			</br>"Coping with the Challenge of Heterogeneity with Integrated Modeling, Machine Learning, and Dynamical Analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Heterogeneity among individual patients presents a fundamental challenge to effective treatment, since a treatment protocol would only work for a portion of the population. We hypothesize that a computational pipeline integrating mathematical modelling and machine learning could be used to address this fundamental challenge and facilitate the optimization of individualized treatment protocols. We tested our hypothesis with the neuroendocrine systems controlled by the hypothalamic-pituitary-adrenal axis. With a synergistic combination of mathematical modelling and machine learning, this integrated computational pipeline could indeed efficiently reveal optimal treatment targets that could significantly improve the treatment efficacy of a heterogeneous individuals, despite of the challenge that the simultaneous changes of multiple parameters result in complex dynamical patterns. Dynamical Analysis of the computational results then revealed mechanistic insights that connect heterogeneous behavior to model structure. We believe that this integrated computational pipeline, properly applied in combination with other computational, experimental and clinical research tools, can be used to optimize treatment targets against a broad range of complex diseases.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS02.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS02-MFBM: </small></br>Generalized Boolean network models and the concept of canalization</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Claus Kadelka (Iowa State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS03-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS02.html#author1"><strong class="card-body-subgroup">Gleb Pogudin</strong></a>
		    	 
		    		<em> (LIX, CNRS, Ecole Polytechnique, Institute Polytechnique de Paris, France)</em>
		    	
		    	
		    		
		    			</br>"Attractor stucture of Boolean networks of small canalizing depth"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Canalization property often occurs in Boolean networks used in systems biology literature. I will describe our computational experiments and mathematical results that indicate that the attractor structure of a random Boolean network with this property differs significantly from the attractor structure of a completely random Boolean network. In particular, there are usually less attractors and they are smaller. These properties turn out to be relevant to many biological applications. I will also discuss how further increase of canalization of a network impacts the attractor structure.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS02.html#author2"><strong class="card-body-subgroup">S. S. Ravi</strong></a>
		    	 
		    		<em> (Biocomplexity Institute & Initiative, University of Virginia, and Department of Computer Science, University at Albany, United States)</em>
		    	
		    	
		    		
		    			</br>"Efficient Algorithms for Boolean Nested Canalyzing Functions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We study several computational problems for Boolean nested canalyzing functions (NCFs). We show that unlike general Boolean functions, there are simple algorithms for many computational problems for NCFs (e.g., equivalence & implication of NCFs, computing the probability of satisfying a given NCF, computing the sensitivity and expected sensitivity of a given NCF). The running times of these algorithms are O(n) or O(n log n), where n is the number of variables in the input function. We also present a linear time algorithm that converts any given NCF into an equivalent weighted threshold function, thus showing that weighted threshold functions generalize the class of NCFs.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS02.html#author3"><strong class="card-body-subgroup">Daniel Rosenkrantz</strong></a>
		    	 
		    		<em> (Biocomplexity Institute & Initiative, University of Virginia, and Department of Computer Science, University at Albany, United States)</em>
		    	
		    	
		    		
		    			</br>"Testing Phase Space Properties of Synchronous Dynamical Systems with Nested Canalyzing Local Functions: Complexity Results and Algorithms"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Discrete graphical dynamical systems serve as effective formal models in many contexts, including simulations of agent-based models, propagation of contagions in social networks and study of bio- logical phenomena. Motivated by the biological applications of nested canalyzing functions (NCFs), we study a variety of analysis problems for synchronous graphical dynamical systems (SyDSs) over the Boolean domain, where each local function is an NCF. Each analysis problem involves testing whether the phase space of a given SyDS satisfies a certain property. Problems considered include reachability, predecessor existence, fixed point existence and garden of Eden existence. We present intractability results for some properties as well as efficient algorithms for others. In several cases, our results clearly delineate intractable and efficiently solvable versions of problems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS02.html#author4"><strong class="card-body-subgroup">Matthew Wheeler</strong></a>
		    	 
		    		<em> (Department of Medicine, University of Florida, United States)</em>
		    	
		    	
		    		
		    			</br>"Reducibility of Boolean Networks: Toward a Theory of Modularity"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Modularity is believed to be a fundamental characteristic of biological systems.  As such, any model built to represent such a system should also exhibit some form of modularity.  One common and powerful way of modeling biological systems is through Boolean networks.  In this direction, we introduce the concept of Boolean network extensions. We will discuss what these extensions are and how these extensions relate to the concept of modularity.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS03.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS03-MFBM: </small></br>Generalized Boolean network models and the concept of canalization</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Claus Kadelka (Iowa State University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS03.html#author1"><strong class="card-body-subgroup">Claus Kadelka</strong></a>
		    	 
		    		<em> (Iowa State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Collective canalization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk, I introduce collectively canalizing Boolean functions, a class of functions that has arisen from applications in systems biology. Boolean networks are an increasingly popular modeling framework for regulatory networks, and the class of collectively canalizing functions captures a key feature of biological network dynamics, namely that a subset of one or more variables, under certain conditions, can dominate the value of a Boolean function, to the exclusion of all others. These functions have rich mathematical properties to be explored. We show how the number and type of such sets influence a function’s behavior and define a new measure for the canalizing strength of any Boolean function. We further connect the concept of collective canalization with the well-studied concept of the average sensitivity of a Boolean function. The relationship between Boolean functions and the dynamics of the networks they form is important in a wide range of applications beyond biology, such as computer science, and has been studied with statistical and simulation-based methods. However, the rich relationship between structure and dynamics remains largely unexplored, and we attempt a first step towards its mathematical foundation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS03.html#author2"><strong class="card-body-subgroup">Elena Dimitrova</strong></a>
		    	 
		    		<em> (California Polytechnic State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Revealing the canalizing structure of Boolean functions — algorithms and applications"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Nested canalization, a type of hierarchical clustering of the inputs of a Boolean function, has been studied in the context of network modeling where each layer of canalization adds a degree of stability in the dynamics of the network. Boolean functions, however, can be represented in many ways, including logical forms, truth tables, and polynomials, as well as different canonical representations such as minimal disjunctive normal form. These representations may obscure the canalizing structure of a Boolean function making its extraction a challenge. In this talk, we show that the problem of determining the specific layer structure of a Boolean function is NP-hard and present and compare algorithms for finding the canalizing layers. Further, we discuss applications of these algorithms for computing disjunctive normal forms and for reverse engineering of Boolean functions according to a prescribed layering format.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS03.html#author3"><strong class="card-body-subgroup">Matthew Macauley</strong></a>
		    	 
		    		<em> (Clemson University, United States)</em>
		    	
		    	
		    		
		    			</br>"Toggling independent sets as an asynchronous Boolean network"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The notion of 'generalized toggle groups' has been a recent popular topic in the field of dynamic algebraic combinatorics. In this talk, I will introduce what it means to toggle independent sets of a graph. Loosely speaking, toggling at a vertex adds it (if possible) when it is absent, removes it if it is present, and otherwise does nothing. I will frame this problem in terms of asynchronous Boolean networks, and summarize the mathematics that we have developed to analyze it. If your interest is piqued by covering spaces consisting of (co-)snakes on a plane that project down to a (co-)ouroborus on a torus, and how the (co-)snake and (co-)ouroborous groups act on the (co)-slithers, then you won't want to miss this talk. It will be widely accessible, and there will be no shortage of open problems, colorful pretty pictures, and puns.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS03.html#author4"><strong class="card-body-subgroup">Alan Veliz-Cuba</strong></a>
		    	 
		    		<em> (University of Dayton, Ohio, United States)</em>
		    	
		    	
		    		
		    			</br>"Identification of control targets in Boolean networks via computational algebra"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many problems in biology have the goal of finding strategies to change an undesirable state of a biological system into another state through an intervention. The identification of such strategies is typically based on a mathematical model such as Boolean networks. In this talk we will see how to find node and edge interventions using computational algebra.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS06.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS06-MFBM: </small></br>Mathematical and computational methods to augment the reliability of biological models for better decision-making</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Vincent Lemaire (Genentech, CA, USA, United States), Khamir Mehta (Amgen, Inc, United States), Malidi Ahamadi (Amgen, CA, United States)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS06.html#author1"><strong class="card-body-subgroup">Chris Rackauckas</strong></a>
		    	 
		    		<em> (MIT and Pumas AI, MA, USA, United States)</em>
		    	
		    	
		    		
		    			</br>"Accelerating Quantitative Systems Pharmacology with Machine Learning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Scientific machine learning (SciML) is the burgeoning field combining scientific knowledge with machine learning for data-efficient predictive modeling. We will introduce the Julia SciML ecosystem by describing some of its recent advances, showing how the GPU-accelerated differential equation solvers gave 175x acceleration on Pfizer's internal C-based QSP models and the 15,000x acceleration seen by the NASA Launch Services upon switching from Simulink to ModelingToolkit.jl. After describing the advances in differential equation solvers and automated model discovery, we will describe the JuliaSim simulation ecosystem and its ability to use continuous-time echo state networks (CTESNs) for automatically generating surrogates of highly stiff QSP models. This technique is shown to be validated on a wide variety of models by using CellML and SBML imports to automate the surrogate training process on ~1000 models. Using the Robertson chemical reaction network as an example case, we will see how multi-layer perceptrons (MLPs),  recurrent neural networks (RNNs), Long short term memory networks (LSTMs), and physics-informed neural networks (PINNs) all fail to adequately train while only the CTESN succeeds in building a stable surrogate. Examples of accelerating simulations by over 560x over the Dymola Modelica compiler will showcase the scalability of the technique. The will showcase how JuliaSim composes with tools like Pumas to bridge QSP into clinical pharmacology. We will end by describing new adjoint techniques which are required to build neural ODE surrogates on stiff ODE models. Together this showcases the practical changes users of the JuliaSim ecosystem are seeing through scientific simulation
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS06.html#author2"><strong class="card-body-subgroup">Oleg Demin Jr</strong></a>
		    	 
		    		<em> (InSysBio, Russia)</em>
		    	
		    	
		    		
		    			</br>"Implementation of variability or uncertainty in parameter values to validate QSP models."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Validation is an important step to test the reliability of the mathematical models including quantitative systems pharmacology (QSP) models. Clinical endpoints for the population of patients are usually used to validate QSP models. For example, percent of responders or mean +/- SD of the particular biomarker. Variability or uncertainty in parameter values should be implemented to describe these endpoints. There are various approaches to extract and implement variability or uncertainty in parameters in model predictions. These methods and cases of their implementation in mechanistic and QSP models will be discussed in the framework of this presentation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS06.html#author3"><strong class="card-body-subgroup">Gianluca Selvaggio</strong></a>
		    	 
		    		<em> (Fondazione The Microsoft Research - University of Trento Centre for Computational and Systems Biology (COSBI), Italy)</em>
		    	
		    	
		    		
		    			</br>"Parameter free approaches in QSP: modelling the cytokine release following bispecific T-cell engager therapy"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bispecific T-cell Engaging therapy is a promising treatment that leverages patient’s own immune system to eliminate cancerous cells. To realize the full potential of  therapy, it is necessary to mitigate the adverse effects of cytokine release from the immune activation, which eventually lead to adverse effect of cytokine release syndrome (CRS). Computational approaches can be instrumental to explore, systematically, the effects of combined therapies on the tumor killing efficacy and CRS. However, to be fully characterized and validated, quantitative models (such as ODEs) require a priori information, that may be poorly available. An alternative parameter free approach is to use the logical formalism to provide a qualitative representation of the processes. This modelling approach can overcome the data/knowledge gap and the sparsity of clinical data by leveraging on several types of information and integrating both qualitative and quantitative information into computable networks. The presentation will demonstrate a logical QSP model that was used to investigate, through systematic sensitivity analysis, the system behavior and then applied to understand strategies to hamper the inflammatory response without impairing the tumor killing capacity. Our analysis suggests that IFN-γ may be a good mechanism to control CRS risk in patients. Furthermore, it entails the existence of a time window to administrate anti-PDL1 therapy and mitigate inflammation without compromising tumor clearance.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS06.html#author4"><strong class="card-body-subgroup">Sietse Braakman</strong></a>
		    	 
		    		<em> (AbbVie Inc., Quantitative Translational Modeling Group, United States)</em>
		    	
		    	
		    		
		    			</br>"A framework for the evaluation of QSP models, with a focus on verification, validation and uncertainty quantification (VVUQ) methods"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Quantitative systems pharmacology (QSP) and other mechanistic mathematical models are increasingly used to support decisions in drug research and development, as well as regulatory decisions (Nijsen et al., 2018; Zineh, 2019). However, despite their demonstrated value, QSP models are not as widely used as they could be (Leil and Bertz, 2014). Reasons for this include the complexity of these models, a lack of consensus on standards for the evaluation of systems models, and short project timelines that are incompatible with the development of complex models. To work towards a consensus on evaluation standards, we introduce a framework for the evaluation of QSP models (Braakman et al., 2021). The framework is designed to accommodate the wide variety of risk and application settings common for QSP models, by applying certain quantitative and qualitative methods to a model. We include verification, validation, and uncertainty quantification (VVUQ) methods such as global sensitivity analysis, identifiability analysis, confidence and profile likelihood intervals, and model validation with hold-out or external data.
Nijsen MJMA, et al., Preclinical QSP Modeling in the Pharmaceutical Industry: An IQ Consortium Survey Examining the Current Landscape. Clinical Pharmacology and Therapeutics: Pharmacometrics and Systems Pharmacology, 2018 7(3): 135-146.  https://doi.org/10.1002/psp4.12282 Zineh I, Quantitative Systems Pharmacology: A Regulatory Perspective on Translation. Clinical Pharmacology and Therapeutics: Pharmacometrics and Systems Pharmacology, 2019 8(6): 336-339. https://doi.org/10.1002/psp4.12403 Leil TA and Bertz R, Quantitative Systems Pharmacology can reduce attrition and improve productivity in pharmaceutical research and development. Frontiers in Pharmacology 2014 5:247. https://doi.org/10.3389/fphar.2014.00247 Braakman S, Pathmanathan P, Moore H, Evaluation Framework for Systems Models. Under review 2021.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS07.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS07-MFBM: </small></br>From Machine Learning to Deep Learning Methods in Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Erica Rutter (University of California, Merced, United States), Suzanne Sindi (University of California, Merced, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS08-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS07.html#author1"><strong class="card-body-subgroup">Emilia Kozlowska</strong></a>
		    	 
		    		<em> (Departement of Systems Biology and Engineering, Silesian University of Technology, Poland)</em>
		    	
		    	
		    		
		    			</br>"Application of mechanistic and machine learning modeling to predict long-term response to treatment for cancer patients"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The most common subtype of lung cancer is non-small cell lung cancer (NSCLC) that constitutes 80% of all lung cancer cases. NSCLC is usually diagnosed at an advanced stage because of non-specific symptoms, leading to high mortality. The standard treatment for NSCLC patients is a combination of chemotherapy and radiotherapy and, as an emerging type of treatment, immunotherapy.
We collected clinical data from over 500 patients with non-small cell lung cancer. From the cohort, we extracted 50 patients who were treated only with platinum-based chemotherapy with palliative intent i.e., under the assumption of failed cure. The clinical data including, among others, short and long-term response to chemotherapy and amount of chemotherapy cycles, were applied to calibrate the mechanistic model using a machine learning approach.
We developed a computational platform to find the best protocol for the administration of platinum-doublet chemotherapy in the palliative setting. The core of the platform is the mathematical model, in the form of a system of ordinary differential equations, describing dynamics of platinum-sensitive and platinum-resistant cancer cells and interactions reflecting competition for space and resources. The model is simulated stochastically by sampling the parameter values from a joint probability distribution.
The model simulations faithfully reproduce the clinical cohort at three levels, long-term response (OS), initial response, and the relationship between the number of chemotherapy cycles and time between two consecutive chemotherapy cycles. In addition, we investigated the relationship between initial and long-term responses. We showed that these two variables do not correlate, which means that we cannot predict patient survival based solely on the initial response. We also tested several chemotherapy schedules to find the best one for patients treated with palliative intent. We found that optimal treatment schedule depends, among others, on the strength of competition among sensitive and resistant subclones in a tumor.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS07.html#author2"><strong class="card-body-subgroup">Sara Ranjbar</strong></a>
		    	 
		    		<em> (Mathematical NeuroOncology Lab, Precision Neurotherapeutics Program, Mayo Clinic, Arizona, United States)</em>
		    	
		    	
		    		
		    			</br>"MRI-based estimation of the abundance of immunohistochemistry markers in GBM brain using deep learning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Glioblastoma (GBM) is a devastating primary brain tumor known for its heterogeneity and invasion. Despite uniformly aggressive therapies including surgery, radiation, and chemotherapy, the median survival rate remains about 15 months. There are many targeted therapies in clinical trials; however, the eloquence of the location makes both the drug delivery and the local efficacy of any drug difficult to assess. Clinical imaging remains the primary modality to assess tumor response, but it is an obscured lens through which it is nearly impossible to distinguish between actual tumor growth and tumor cell death from a large immune response. Over the past decade, MRI has been suggested by many studies to reflect the underlying tumor biology. In this talk, we will discuss our groups’ approach to building a robust deep learning model to connect MRI patterns at GBM biopsied locations with cell proliferation abundance measured by immunohistochemistry staining. If successful, this model can provide a non-invasive readout of cell proliferation and reveal the effectiveness of a given cytotoxic therapy including standard-of-care radiotherapy that targets cell proliferation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS07.html#author3"><strong class="card-body-subgroup">Joan Ponce</strong></a>
		    	 
		    		<em> (UCLA, United States)</em>
		    	
		    	
		    		
		    			</br>"An integrated framework for building trustworthy data-driven epidemiological models: Application to the COVID-19 outbreak in New York City"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Epidemiological models can provide the dynamic evolution of a pandemic but they are based on many assumptions and parameters that have to be adjusted over the time when the pandemic lasts. However, often the available data are not sufficient to identify the model parameters and hence infer the unobserved dynamics. Here, we develop a general framework for building a trustworthy data-driven epidemiological model, consisting of a workflow that integrates data acquisition and event timeline, model development, identifiability analysis, sensitivity analysis, model calibration, model robustness analysis, and forecasting with uncertainties in different scenarios. In particular, we apply this framework to propose a modified susceptible–exposed–infectious–recovered (SEIR) model, including new compartments and model vaccination in order to forecast the transmission dynamics of COVID-19 in New York City (NYC). We find that we can uniquely estimate the model parameters and accurately predict the daily new infection cases, hospitalizations, and deaths, in agreement with the available data from NYC's government's website. In addition, we employ the calibrated data-driven model to study the effects of vaccination and timing of reopening indoor dining in NYC.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS07.html#author4"><strong class="card-body-subgroup">Emily Zhang</strong></a>
		    	 
		    		<em> (North Carolina State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Deep Learning and Regression Approaches to Forecasting Blood Glucose Levels for Type 1 Diabetes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Controlling blood glucose in the euglycemic range is the main goal of developing sensor-augmented pump therapy for type 1 diabetes patients. The pump therapy delivers the amount of insulin dose determined by glucose predictions through the use of computational algorithms. A computationally efficient and accurate model that can capture the physiological nonlinear dynamics is critical for developing an accurate therapy device. Four data-driven models are compared, including different neural network architectures, a reservoir computing model, and a novel linear regression approach. Model predictions are evaluated over continuous 30 and 60 minute time horizons using real-world data from wearable sensor measurements, a continuous glucose monitor, and self-reported events through mobile applications. The four data-driven models are trained on 12 data contributors for around 32 days, 8 days of data are used for validation, with an additional 10 days of data for out-of-sample testing. Model performance was evaluated by the root mean squared error and the mean absolute error. A neural network model using an encoder-decoder architecture has the most stable performance and is able to recover missing dynamics in short time intervals. Regression models performed better at long-time prediction horizons (i.e., 60 minutes) and with lower computational costs. The performance of several distinct models was tested for individual-level data from a type 1 diabetes data set. These results may enable a feasible solution with low computational costs for the time-dependent adjustment of pump therapy for diabetes patients.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS08.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS08-MFBM: </small></br>From Machine Learning to Deep Learning Methods in Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Erica Rutter (University of California, Merced, United States), Suzanne Sindi (University of California, Merced, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS08.html#author1"><strong class="card-body-subgroup">Ali Heydari</strong></a>
		    	 
		    		<em> ((i) UC Merced Department of Applied Mathematics and (ii) Health Sciences Research Institute at UC Merced, USA)</em>
		    	
		    	
		    		
		    			</br>"Deep Generative Models for Realistic Single-Cell RNA-Seq Data Augmentation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Single-cell RNA sequencing (scRNAseq) technologies allow for measurements of gene expression at a single-cell resolution. This provides researchers with a tremendous advantage for detecting heterogeneity, delineating cellular maps or identifying rare subpopulations. However, a critical challenge remains: the low number of single-cell observations due to limitations by cost  or rarity of subpopulation. This absence of sufficient data may cause inaccuracy or irreproducibility of downstream analysis. In this talk, we present ACTIVA (Automated Cell-Type-informed Introspective Variational Autoencoder): a novel deep learning framework for generating realistic synthetic data using a single-stream adversarial variational autoencoder conditioned with cell-type information. We train and evaluate ACTIVA, and competing models, on multiple public scRNAseq datasets. Under the same conditions, ACTIVA trains up to 17 times faster than the GAN-based state-of-the-art model while performing better or comparably in our quantitative and qualitative evaluations. We show that augmenting rare-populations with ACTIVA significantly increases the classification accuracy of the rare population (more than 45% improvement in our rarest test case). Data generation and augmentation with ACTIVA can enhance scRNAseq pipelines and analysis, such as benchmarking new algorithms, studying the accuracy of classifiers and detecting marker genes. ACTIVA will facilitate analysis of smaller datasets, potentially reducing the number of patients and animals necessary in initial studies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS08.html#author2"><strong class="card-body-subgroup">Mohammad Jafari</strong></a>
		    	 
		    		<em> (Postdoctoral Scholar,  Department of Applied Mathematics  Jack Baskin School of Engineering  University of California, Santa Cruz, USA)</em>
		    	
		    	
		    		
		    			</br>"Machine Learning-based Feedback Controller for Directing Stem Cell Membrane Potential"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Driving biological response with spatiotemporal precision can help advance biomedical applications for customized therapeutics where bioelectronic devices are suitable to directly interfacing with the biological systems using bioelectronic actuators and sensors. Implementation of feedback control by using these devices can help achieve this but has not been widely adopted, in part, due to a limited understanding of the complexities involved. Modeling, identification, prediction, and control, which are essential to this end, are challenging due to the presence of uncertainties, stochasticity, unmodeled dynamics, and complex nonlinearities. For example, in biological systems, cellular response can change in different environmental conditions such as changing flow characteristics and temperature. Thus, Machine Learning (ML)-based techniques, which can be applied to solve different modeling and control problems when system dynamics are fully or partially unknown, may prove suitable here. The best-known ML techniques rely on the availability
of large datasets a priori and have not been applied to control biological systems using bioelectronic devices. We proposed that ML-based techniques that are explored as control solutions outside of biology for cases involving complex nonlinear systems are also suitable for closing the loop for biological systems [1]. To do this, an adaptive external “sense and respond” learning algorithm is derived using adaptive Lyapunov-based methods [2]. The satisfactory performance of the proposed method is experimentally validated by maintaining the pH in a microfluidic system that houses pluripotent mammalian stem cells. This pH control affects the membrane voltage (Vmem) of the cells that is measured using genetically encoded fluorescent Vmem reporters [3]. To the best of our knowledge, this is the first learning control method demonstrated for biological applications of its kind.
[1]. Selberg, J., Jafari, M., Bradley, C., Gomez, M., & Rolandi, M. (2020). Expanding biological control to bioelectronics with machine learning. APL Materials, 8(12), 120904.
[2]. Jafari, M., Marquez, G., Selberg, J., Jia, M., Dechiraju, H., Pansodtee, P., ... & Gomez, M. (2020). Feedback Control of Bioelectronic Devices Using Machine Learning. IEEE Control Systems Letters, 5(4), 1133-1138.
[3]. Selberg, J., Jafari, M., Mathews, J., Jia, M., Pansodtee, P., Dechiraju, H., ... & Rolandi, M. (2020). Machine Learning‐Driven Bioelectronics for Closed‐Loop Control of Cells. Advanced Intelligent Systems, 2(12), 2000140.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS08.html#author3"><strong class="card-body-subgroup">Thomas de Mondesir</strong></a>
		    	 
		    		<em> (Université Claude Bernard Lyon 1 & UC Merced, France)</em>
		    	
		    	
		    		
		    			</br>"Generating biological images to train deep-learning-based segmentation models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Medical and biological imaging are areas where image segmentation plays a critical role when diagnosing pathologies or analysing experimental results study. While recent studies show that methods using deep learning achieve superior accuracy when neural networks are trained on pixel-level labeled data, sufficient amounts of annotated images are often difficult to gather. We introduce an image generation method to produce images containing biological objects and corresponding segmentation masks. Our approach creates realistic images by using B-splines to reproduce shapes of interest. Simulating objects is done by choosing control points and adjusting parameters that allow their geometries to be diverse. Adapted to cases with limited or no training data, our method offers the possibility to train any machine learning based segmentation method on generated images. Obtaining a good segmentation on real images relies on similarity with artificial images.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS08.html#author4"><strong class="card-body-subgroup">Jordan Collignon</strong></a>
		    	 
		    		<em> (University of California, Merced, USA)</em>
		    	
		    	
		    		
		    			</br>"A High-throughput Pipeline for Analyzing Experimental Images of Sectored Yeast Colonies"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			 Prion proteins are most commonly associated with fatal neurodegenerative diseases in mammals, but are also responsible for a number of harmless heritable phenotypes in Saccharomyces cerevisiae (yeast). In normal conditions yeast colonies grow in a circular shape with a uniform white or pink color related to the fraction of normal (non-prion) protein in a typical cell. However, under mild experimental manipulations, which introduce changes in protein aggregation dynamics, colonies exhibit red sectors corresponding to cells with no prion protein. Such phenotypic organization provides a rich data set that can be used to uncover relationships between colony-level phenotypic transitions, molecular processes, and individual cell behaviors.  In this project, we use deep learning tools to develop an automated image processing pipeline for extracting and quantifying the shape, size, and frequency of sectors in yeast colonies grown under experimental conditions.  Our approach will allow us to draw conclusions about the formation of sectors in the experimental data and will help uncover more information about the mechanisms driving colony-level phenotypic transitions.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS08" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS09.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS09-MFBM: </small></br> Emergent behavior across scales: locomotion, mixing, and collective motion in active swimmers</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Robert Guy (University of California Davis, United States), Arvind Gopinath (University of California Merced, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS15-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS09.html#author1"><strong class="card-body-subgroup">Henry Fu</strong></a>
		    	 
		    		<em> (University of Utah, United States)</em>
		    	
		    	
		    		
		    			</br>"Symmetry breaking propulsion of magnetically rotated spheres in nonlinearly viscoelastic fluids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Symmetries have long been used to understand when propulsion is possible in microscale systems.  Currently, artificially propelled magnetic micro- and nanoparticles are being utilized in a variety of techniques including hyperthermia, drug delivery, and magnetic resonance imaging. Rotation of rigid magnetic particles by an external magnetic field is a promising category of such artificial propulsion.  Propulsion would seem to be prohibited by geometries with fore-aft symmetry along their rotation axis, such as a rotating sphere.  We have shown that in nonlinearly viscoelastic fluids, a symmetry breaking propulsion is possible for rotating microspheres.  We show that this propulsion occurs in both mucin and polyacrylamide solutions, and propose that it results from rod-climbing-like effects which squeeze the sphere and reinforce its translation.  A perturbative analysis of the forces on a rotating sphere in a nonlinear polymeric fluid corroborates this mechanism.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS09.html#author2"><strong class="card-body-subgroup">Kathryn Link</strong></a>
		    	 
		    		<em> (University of California Davis, United States)</em>
		    	
		    	
		    		
		    			</br>"Emergent Properties of Flagellar Waveforms in Viscoelastic Fluids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Eukaryotic cells move in rheologically complex environments via deformations of their flagella, which are slender threadlike structures that are powered by internal molecular motors.  It is an ongoing scientific pursuit to determine how flagellar beat emerges from the coordination of the mechanics of the flagella, the interactions with the external fluid environment, and the mechano-chemical feedback of the molecular motors. Existing theories have shed light on the origins of this behavior in a viscous fluid, however, due to the inherent nonlinearity and mathematical complexity involved in modeling viscoelastic fluids, both analytical and numerical predictions require nonstandard approaches. In this work we propose an extension to the current models to make a prediction about how viscoelasticity changes the beat frequency of the emergent waveform.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS09.html#author3"><strong class="card-body-subgroup">Rudi Schuech</strong></a>
		    	 
		    		<em> (Tulane University, United States)</em>
		    	
		    	
		    		
		    			</br>"Viscoelastic Network Remodeling by Microswimmers"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Microorganisms often navigate a complex environment composed of a viscous fluid with suspended microstructures such as elastic polymers and filamentous networks.  These microstructures can have similar length scales to the microorganisms, leading to complex swimming dynamics.  Some microorganisms are known to remodel the viscoelastic networks they move through. In order to gain insight into the coupling between swimming dynamics and network remodeling, we use a regularized Stokeslet boundary element method to compute the motion of a microswimmer consisting of a spherical body and rotating helical flagellum. The viscoelastic network is represented by a cloud of points with virtual Maxwell element links. We consider two models of network remodeling in which (1) links break based on their distance to the microswimmer body, modeling enzymatic dissolution by bacteria or microrobots, or (2) links break based on a threshold tension force. We compare the swimming performance of the microbes in each remodeling paradigm as they penetrate and move through the network.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS09.html#author4"><strong class="card-body-subgroup">Sookkyung Lim</strong></a>
		    	 
		    		<em> (University of Cincinnati, United States)</em>
		    	
		    	
		    		
		    			</br>"Simulations of microswimmers propelled by multiple flagella"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Peritrichously flagellated bacteria swim in a fluid environment by rotating motors embedded in the cell membrane and consequently rotating multiple helical flagella. We present a novel mathematical model of a microswimmer that can freely run propelled by a flagellar bundle and tumble upon motor reversals. Our cell model is composed of a rod-shaped rigid cell body and multiple flagella randomly distributed over the cell body. These flagella can go through polymorphic transformations. We demonstrate that flagellar bundling is influenced by flagellar distribution and hence the number of flagella. Moreover, reorientation of cells is affected by the number of flagella, how many flagella change their polymorphisms within a cell, the tumble timing, different combinations of polymorphic sequences, and random motor reversals. Our mathematical method can be applied to numerous types of microorganisms and may help to understand their characteristic swimming mechanisms.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS11.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS11-MFBM: </small></br>Stochastic Systems Biology: Theory and Simulation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jae Kyoung Kim (Department of Mathematical Sciences, KAIST, Republic of Korea), Ramon Grima (University of Edinburgh, United Kingdom)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS12-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS11.html#author1"><strong class="card-body-subgroup">Zhixing Gao</strong></a>
		    	 
		    		<em> (Key Laboratory of Advanced Control and Optimization for Chemical Processes, Ministry of Education, East China University of Science and Technology, China)</em>
		    	
		    	
		    		
		    			</br>"Neural network aided approximation and parameter inference of stochastic models of gene expression"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Non-Markov models of stochastic biochemical kinetics often incorporate explicit time delays to effectively model large numbers of intermediate biochemical processes. Analysis and simulation of these models, as well as the inference of their parameters from data, are fraught with difficulties because the dynamics depends on the system’s history. Here we use an artificial neural network to approximate the time-dependent distributions of non-Markov models by the solutions of much simpler time-inhomogeneous Markov models; the approximation does not increase the dimensionality of the model and simultaneously leads to inference of the kinetic parameters. The training of the neural network uses a relatively small set of noisy measurements generated by experimental data or stochastic simulations of the non-Markov model. We show using a variety of models, where the delays stem from transcriptional processes and feedback control, that the Markov models learnt by the neural network accurately reflect the stochastic dynamics across parameter space.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS11.html#author2"><strong class="card-body-subgroup">Abhyudai Singh</strong></a>
		    	 
		    		<em> (University of Delaware, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling stochasticity in timing of intracellular events: A first-passage time approach"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			 How the noisy expression of regulatory proteins affects timing of intracellular events is an intriguing fundamental problem that influences diverse cellular processes. Here we use the bacteriophage lambda to study event timing in individual cells where cell lysis is the result of expression and accumulation of a single protein (holin) in the Escherichia coli cell membrane up to a critical threshold level. Site-directed mutagenesis of the holin gene generated phage variants that vary in their lysis times from 30 to 190 min. Observation of the lysis times of single cells reveals an intriguing finding—the noise in lysis timing first decreases with increasing lysis time to reach a minimum and then sharply increases at longer lysis times. A mathematical model with stochastic expression of holin together with dilution from cell growth was sufficient to explain the non-monotonic noise profile and identify holin accumulation thresholds that generate precision in lysis timing.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS11.html#author3"><strong class="card-body-subgroup">Thomas Prescott</strong></a>
		    	 
		    		<em> (Alan Turing Institute, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Learning a multifidelity simulation strategy for likelihood-free Bayesian inference."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Likelihood-free Bayesian inference is a popular approach to calibrating complex mathematical models typical of biological systems, where likelihoods are often intractable. However, being reliant on repeated model simulation, the complexity that prohibits the likelihood calculation can also cause these methods to suffer from a significant computational burden. Multifidelity inference methods have been shown to reduce this burden by exploiting approximate simulations, such as coarser numerics or lower-dimensional models. By incorporating both high- and low-fidelity simulations, computational savings can be achieved without introducing any further bias in the resulting likelihood-free posterior. Instead, these approaches are forced to trade between reducing computational burden and increasing estimator variance. This trade-off is balanced by optimally assigning a simulation budget between the models at different fidelities. We will discuss how the optimal multifidelity simulation strategy can be learned in parallel with the posterior, and the multifidelity algorithm thus adaptively tuned as the posterior is uncovered.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS11.html#author4"><strong class="card-body-subgroup">Ruben Perez-Carrasco</strong></a>
		    	 
		    		<em> ( Imperial College London, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Should we care about cell cycle variability when studying stochastic gene expression?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many models of stochastic gene expression do not incorporate a cell cycle description. I will show how this can be tackled analytically studying how mRNA fluctuations are influenced by DNA replication for a prescribed cell cycle duration stochasticity. Results show that omitting cell cycle details can introduce significant errors in the predicted mean and variance of gene expression for prokaryotic and eukaryotic organisms, reaching 25% error in the variance for mouse fibroblasts. Furthermore, we can derive a negative binomial approximation to the mRNA distribution, indicating that cell cycle stochasticity introduces similar fluctuations to bursty transcription. Finally, I will show how disregarding cell cycle stochasticity can introduce inference errors in transcription rates bigger than 10%.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS11" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS12.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS12-MFBM: </small></br>Stochastic Systems Biology: Theory and Simulation</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Jae Kyoung Kim (Department of Mathematical Sciences, KAIST, Republic of Korea), Ramon Grima (University of Edinburgh, United Kingdom)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS11-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS12.html#author1"><strong class="card-body-subgroup">Hyukpyo Hong</strong></a>
		    	 
		    		<em> (Department of Mathematical Sciences, KAIST, Republic of Korea)</em>
		    	
		    	
		    		
		    			</br>"Inference of stochastic dynamics in biochemical reaction networks by exploiting deterministic dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Biochemical reaction networks (BRNs) have a stochastic nature, so every reaction in BRNs display randomness. Inherent stochasticity can be captured only by stochastic models, but it is more challenging to analyze their dynamics while their deterministic counterparts are easier to be analyzed, in general. Thus, various methods exploiting deterministic dynamics to infer the stochastic one have been proposed. In particular, stochastic model reduction using deterministic quasi-steady-state approximations (QSSAs) of fast variables is widely used to efficiently simulate a stochastic model. For instance, Michaelis-Menten or Hill-type functions have been used for Gillespie stochastic simulation. In this talk, we provide a complete validity condition for stochastic model reduction using the deterministic QSSA to eliminate stochastic reversible binding, which is fundamental and ubiquitous in BRNs. Furthermore, we present a framework to analytically derive stationary distribution for a large class of BRNs using their deterministic steady states based on chemical reaction network theory.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS12.html#author2"><strong class="card-body-subgroup">Zhou Fang</strong></a>
		    	 
		    		<em> (ETH Zurich, Switzerland)</em>
		    	
		    	
		    		
		    			</br>" Stochastic filtering for multiscale stochastic reaction networks based on hybrid approximations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The advance in fluorescent technologies and microscopy has greatly improved scientists' ability to observe real-time single-cell activities. In this paper, we consider the associate filtering problem, i.e., how to estimate latent dynamic states of an intracellular reaction system from time-course measurements of fluorescent reporters. A straightforward approach to this filtering problem is to use a particle filter where samples are generated by simulation of the full model and weighted according to observations. However, the exact simulation of the full model usually takes an impractical amount of computational time and prevents this type of filters from being used for real-time applications. Inspired by the recent development of hybrid approximations to multiscale chemical reaction networks, we approach the filtering problem in an alternative way. We first prove that accurate solutions to the filtering problem can be constructed by solving the filtering problem for a reduced model that represents the dynamics as a hybrid process. The model reduction is based on exploiting the time-scale separations in the original network and, therefore, can greatly reduce the computational effort required to simulate the dynamics. Consequently, we are able to develop efficient particle filters to solve the filtering problem for the original model by applying particle filters to the reduced model. We illustrate the efficacy and efficiency of our approach using several numerical examples.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS12.html#author3"><strong class="card-body-subgroup">Samuel Isaacson</strong></a>
		    	 
		    		<em> (Boston University, Department of Mathematics and Statistics, USA)</em>
		    	
		    	
		    		
		    			</br>"Stochastic Reaction-Drift-Diffusion Methods for Studying Cell Signaling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Particle-based stochastic reaction-diffusion (PBSRD) models are one approach to study biological systems in which both the noisy diffusion of individual molecules, and stochastic reactions between pairs of molecules, may influence system behavior. They provide a more microscopic model than deterministic reaction-diffusion PDEs or stochastic reaction-diffusion SPDEs, which treat molecular populations as continuous fields. The reaction-diffusion master equation (RDME) and convergent RDME (CRDME) are lattice PBSRD models, with the latter providing a convergent approximation to the spatially-continuous volume-reactivity PBSRD model as the lattice spacing is taken to zero. In this talk I will present several generalizations of the RDME and CRDME to support spatial transport mechanisms needed for resolving membrane-bound signaling processes, including drift due to background potentials, interaction potentials between molecules, and continuous-time random walks to approximate molecular transport on surfaces.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS12.html#author4"><strong class="card-body-subgroup">Brian Munsky</strong></a>
		    	 
		    		<em> (Colorado State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Designing Optimal Microscopy Experiments to Harvest Single-Cell Fluctuation Information while Rejecting Image Distortion Effects"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Modern fluorescence labeling techniques and optical microscopy approaches have made it possible to experimentally visualize every stage of basic gene regulatory processes, even at the level of single cells and individual DNA, RNA, and protein molecules, in living cells, and within fluctuating environments. To complement these observations, the mechanisms and parameters of discrete stochastic models can be rigorously inferred to reproduce and quantitatively predict every step of the central dogma of molecular biology. As single-cell experiments and stochastic models become increasingly more complex and more powerful, the number of possibilities for their integrated application increases combinatorially, requiring efficient approaches for optimized experiment design. In this presentation, we will introduce two model-driven experimental design approaches: one based on detailed mechanistic simulations of optical experiments, and the other on a new formulation of Fisher Information for discrete stochastic gene regulation models. Using different combinations of biological experiments and simulated data for single-gene transcription and single-RNA translation, we will demonstrate how these experiment design approaches can be extended to account for non-gaussian intrinsic and extrinsic process noise within individual cells as well as for non-trivial measurement noise effects due to optical distortions and image processing errors.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS12" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS13.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS13-MFBM: </small></br> Data-driven methods for biological modeling in industry</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Kevin Flores (North Carolina State University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS14-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS13.html#author1"><strong class="card-body-subgroup">Anna Sher</strong></a>
		    	 
		    		<em> (Pfizer, USA)</em>
		    	
		    	
		    		
		    			</br>"Quantitative Systems Pharmacology (QSP) in cardiovascular disease: Preclinical case studies with real-world data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many pharmaceutical companies are starting to utilize mechanistic modeling of physiological systems, in particular Quantitative Systems Pharmacology (QSP) modeling, at all stages of drug discovery and development, including exploratory, preclinical, and clinical studies. At Pfizer, ongoing efforts in cardiovascular and metabolic programs involve investigating target rationale,  preclinical to clinical translation, drug efficacy and safety using systems modeling and simulations of various aspects of cardiometabolic abnormalities. I will discuss modeling and simulation techniques used in these efforts and highlight challenges related to the incorporation of real-world data preclinically. Examples will include Metabolic Flux Analysis as well as translation from cellular to whole heart mechanical function studies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS13.html#author2"><strong class="card-body-subgroup">Doris Fuertinger</strong></a>
		    	 
		    		<em> (Fresenius Medical Care, Germany)</em>
		    	
		    	
		    		
		    			</br>"1 year of precision therapy: Experiences in optimal drug administration based on an  individualized biomathematical anemia model"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The majority of patients suffering from end-stage kidney disease develop anemia at some point. Management of anemia with erythropoiesis stimulating agents (ESA) has been established more than three decades ago, however, it remains difficult to stabilize hemoglobin levels within the desired target range.
We developed a comprehensive mathematical model that describes the reproduction of red blood cells and the effect of ESAs on it. The resulting system of hyperbolic partial differential equations is adapted to individual patients using routine clinical data by estimating a set of key parameters on the individual level. A nonlinear model predictive controller was designed around the PDE model incorporating several techniques used to create robust and adaptive feedback control systems. The resulting software solution is currently used in a randomized clinical trial.
Challenges around adapting a complex PDE system to noisy and missing data will be addressed and interim results from the clinical study presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS13.html#author3"><strong class="card-body-subgroup">Alhaji Cherif</strong></a>
		    	 
		    		<em> (Renal Research Institute, USA)</em>
		    	
		    	
		    		
		    			</br>"Bone and mineral disturbances in uremic patients"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Reduced renal function has a significant impact on a myriad of interlinked secondary pathophysiological abnormalities, including metabolic acidemia, and mineral and bone disorder (CKD-MBD), which comprise secondary hyperparathyroidism (SHPT) and vascular calcification. These sequelae contribute to increased morbidity and mortality in patients with chronic kidney and end-stage renal diseases.  We developed a multi-scale comprehensive physiology-based mathematical model describing bone remodeling and mineral homeostasis that enables in silico exploration of the ramifications of disease- and therapy-induced disturbances  Using a multi-scale mechanistic physiology-based model quantitating the interrelations of osteoclasts, osteoblasts, and osteocytes on bone remodeling, we incorporate intercellular and intracellular signaling pathways, cytokines, parathyroid hormone (PTH), sclerostin, and endocrine and paracrine feedbacks (Cherif et al., ΝDΤ 2018, 33 (Suppl. 1): 165–166).  The predictions of the model are demonstrated by comparing model results of different pathologies (e.g., primary hyperparathyroidism (PHPT) and SHPT, chronic metabolic acidemia, uremia) to clinical observations. In addition, we explore the effect of altered PTH (teriparatide) administration regimen (e.g., dosing frequency and amplitude) on bone catabolism and anabolism, respectively.  Our model correctly predicts clinically observed responses to induced primary and secondary hyperparathyroidism, metabolic acidosis, and their impact on extracellular calcium (Ca) and phosphate (PO4) levels and bone mineral density (BMD). In particular, the model predicts the catabolic effect of metabolic acidosis on bone remodeling, including decreased bone mineral density, and increased efflux of Ca and PO4 from the bone. The model shows the differential responses of osteo-anabolic and catabolic effects of continuously and intermittently elevated levels of PTH (teriparatide), respectively. Furthermore, we observe that intermittent administration of PTH with a high frequency and amplitude induces bone catabolism similar to that seen in pathologies with continuously elevated PTH (i.e., PHT, or SHPT). Low PTH frequency with high dosing amplitude induces both osteoclastic and osteoblastic activities, but the net result is bone anabolism.  Our results suggest that both frequency and amplitude of PTH (teriparatide) cycling affect the balance of osteo-catabolic and -anabolic effects, and there exists optimal PTH (teriparatide) frequency-amplitude combinations that enhance anabolic gains.  The model provides an opportunity to investigate the effects of reduced renal function on the complex interlinked pathophysiological processes of CKD-MBD. The in-silico assessment can serve as a complementary tool for (1) gaining further insights into the features of bone and mineral metabolism, (2) exploring optimal therapeutic modalities for patients with metabolic bone diseases, and minimize unintended disease-specific outcomes, and (3) performing virtual clinical trials for newly emerging and off-label therapeutic options.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS13.html#author4"><strong class="card-body-subgroup">Malidi Ahamadi</strong></a>
		    	 
		    		<em> (Amgen, USA)</em>
		    	
		    	
		    		
		    			</br>"Disease progression platform for Leucine-Rich Repeat Kinase 2 in Parkinson's Disease to Inform Clinical Trial Designs"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Drug discovery and development of new therapeutics for Parkinson’s Disease (PD) has a high attrition rate which has been attributed to incomplete understanding of the complex pathophysiology of neurodegenerative disorders and difficulties in designing efficient clinical trials to develop new disease modifying agents among other several factors. Clinical assessments (e.g., disability or quality of life scales) are affected/confounded by symptomatic effects of therapy and are unable to differentiate this effect from disease-modification, at least in the short-term. A quantitative assessment of patient characteristics and patient enrichment is one of valuable tools to improve clinical trial efficiency. A disease progression model1,2, identifying relevant patient characteristics impacting the temporal change in disease status assessed using Movement Disorder Society-Unified Parkinson's disease rating scale, was developed to evaluate optimal study designs. Results showed that the progression rate in motor symptoms in individuals with PD who carry a leucine-rich repeat kinase 2 (LRRK2) mutation was slightly slower (~0.170 points/month) compared to idiopathic PD patients (~0.222 points/month). Trial simulations showed that for a non-enriched placebo-controlled clinical trial approximately 70 subjects/arm would be required to detect a drug effect of 50% reduction in the progression rate with 80% probability. Whereas 85, 93 and 100 subjects/arm would be required for an enriched clinical trial with 30%, 50% and 70% subjects with LRRK2 mutations, respectively, to detect a 50% drug effect with 80% power. These findings are expected to play an important role in designing long-term trials for PD programs. Reference 1. Malidi Ahamadi et al., Development of a Disease Progression Model for Leucine-Rich Repeat Kinase 2 in Parkinson's Disease to Inform Clinical Trial Designs, Clin Pharmacol Ther, Volume 107, Number 3, March 2020. 2. Malidi Ahamadi et al., A disease progression model to quantify the non‐motor symptoms of Parkinson’s disease in participants with leucine‐rich repeat kinase 2 mutation, Clin Pharmacol Ther., 2021 Apr 24. doi: 10.1002/cpt.2277.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS13" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS14.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS14-MFBM: </small></br> Data-driven methods for biological modeling in industry</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Kevin Flores (North Carolina State University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS13-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS14.html#author1"><strong class="card-body-subgroup">Richard Allen</strong></a>
		    	 
		    		<em> (Quantitative Systems Pharmacology, Early Clinical Development, Pfizer Worldwide Research Development and Medical, USA)</em>
		    	
		    	
		    		
		    			</br>"Analyzing and Predicting Clinical Trial Data with Systems Modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Systems modeling approaches have found increasing utility in supporting the discovery and development of novel therapeutics. By capturing key biological interactions and incorporating a wide range of data to inform the model, a systems model can be a powerful tool to design, predict, and analyze clinical trials. However, typical clinical trials show a highly variable response – such that fitting a model to the mean at best might be losing some information, and at worse fully mischaracterizing the response. Conversely, mathematical representations of complex biology lead to large models and associated uncertainty in parameter estimation. In this talk I will introduce how systems modeling is being used in drug discovery and development, and the challenges of such an approach. In particular, I will discuss how we generate virtual patients and populations to explore parameter uncertainty in a model while constraining the response using the observed clinical variability. Furthermore, I will show – by example - how analysis of a virtual population can lead to physiological insights.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS14.html#author2"><strong class="card-body-subgroup">Florencio Serrano Castillo</strong></a>
		    	 
		    		<em> (Clinical Pharmacology, Modelling and Simulations, Amgen Inc., USA)</em>
		    	
		    	
		    		
		    			</br>"Dosing guidance optimization, leveraging real world heterogeneity to forecast clinical biomarker response"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Understanding the dynamic and variability of a drug’s pharmacokinetic (its concentration in the body) and pharmacodynamic (its effect on the body) profiles is of critical importance for the design and success of any clinical study. However, the development, implementation and validation of system-level models that explicitly relate the complex biological mechanisms ruling the relationships between dose, exposure, clinical response and safety of a drug while simultaneously describing clinical variability is often unfeasible due to both technical and logistic limitations. Population pharmacokinetic and pharmacodynamic models (popPK/PD) are a powerful tool to circumvent this limitations. popPK/PD models leverage the inherent nosiness of clinical/biological data to inform statistical models embedded into their core dynamic structure to describe both intrinsic and extrinsic sources of variability representative of the clinical setting. This hierarchical structure can then be leveraged to perform clinical trial simulations that predict the impact of various design options and thus inform strategic decisions throughout the development and life cycle management of a therapeutic. This talk will provide a general example of how to leverage a pseudo-mechanistic popPKPD to identify complex patterns between highly heterogeneous clinical dose and biomarker data in order to provide guidance regarding the feasibility and uncertainty associated with various proposed clinical scenarios. Furthermore, it will also describe methodologies on how to address typical challenges with this process, such as the validation of a previously developed model for a different patient population, the generation and implementation of appropriate clinical trial simulation schemes to address possible population and strategy characteristics, and the collation and interpretation of model-derived outputs in order to inform development strategies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS14.html#author3"><strong class="card-body-subgroup">Zackary Kenz</strong></a>
		    	 
		    		<em> (DILIsym Services, a Simulations Plus Company, USA)</em>
		    	
		    	
		    		
		    			</br>"Quantitative Systems Pharmacology Modeling of Fibrotic Diseases"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Fibrotic diseases occur in multiple organs, characterized by escalating fibrosis affecting organ function. For example, in idiopathic pulmonary fibrosis (IPF), normal lung is progressively replaced by fibrotic architecture resulting in compromised movement and gas exchange. Similarly, in non-alcoholic steatohepatitis (NASH), normal liver cells are replaced with fibrotic matrix resulting in compromised liver clearance mechanisms. In both cases, there are no cures and few treatment options. These fibrotic diseases represent areas of unmet clinical need, where improved understanding of pathophysiology and treatment interventions could impact the drug development pipeline and patient care. To accelerate the clinical development of treatments in IPF and NASH, DILIsym Services has developed QSP models of each disease state. These models contain mechanistic representations of ongoing injury, inflammation, and accumulation of extracellular matrix, each of which represent potential targets for treatment intervention which can be quantitatively assessed within the model. Further, mechanisms are dynamically linked with clinical outcomes, providing insight across multiple scales from molecular intervention to cellular response to tissue response. Selected portions of the model development and validation will be discussed, along with example treatments. These QSP platforms are available and actively in use to support ongoing development of effective treatments for IPF and NASH patients.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS14.html#author4"><strong class="card-body-subgroup">Anna Neely</strong></a>
		    	 
		    		<em> (TigerRisk Partners, USA)</em>
		    	
		    	
		    		
		    			</br>"Estimating the growing risk of severe thunderstorms"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			'Severe weather is one of the biggest drivers of insured catastrophe losses in the US.  Catastrophe models are used to estimate risk of severe weather conditions - are they aiming at a moving target?  Losses to the insurance industry have increased at a rate of 9% annually since 2000.  Far outpacing expectations.  In this talk we'll dive into some of the drivers of this increase and de-mystify some insurance industry folklore.'
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS14" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS15.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS15-MFBM: </small></br> Emergent behavior across scales: locomotion, mixing, and collective motion in active swimmers</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Robert Guy (University of California Davis, United States), Arvind Gopinath (University of California Merced, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS09-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS15.html#author1"><strong class="card-body-subgroup">Maxime Theillard</strong></a>
		    	 
		    		<em> (University of California Merced, United States)</em>
		    	
		    	
		    		
		    			</br>"Multi-scale multi-species modeling of emergent flows and active mixing in confined bacterial swarms"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			 Autonomous collective motion of disparate agents in nonequilibrium is fundamental to many biological and engineering systems. An example from biology is bacterial swarms, that are prototypical dense multi-phase active fluids. Here we present a new method for modeling such fluids under confinement. We use a continuum multiscale mean-field approach to represent each specie by its first three orientational moments, and couple their evolution with those of the suspending fluid. The resulting coupled system is solved using parallel hybrid level-set based discretization on adaptive cartesian grids for high computational efficiency and maximal flexibility in the confinement geometry. Motivated by recent experimental work, we employ our method to study emergent flows in bacterial swarms. Our computational exploration demonstrate that we can reproduce the observed emergent collective patterns including active dissolution. This work lays the foundation for a systematic characterization of natural and synthetic systems such as bacterial colonies, bird flocks, fish schools, colloidal swimmers, or programmable active matter.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS15.html#author2"><strong class="card-body-subgroup">Paulo Arratia</strong></a>
		    	 
		    		<em> (University of Pennsylvania, United States)</em>
		    	
		    	
		    		
		    			</br>"Bacteria hinder stretching and large-scale transport in  time-periodic flows"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk, I will show recent experiments on the mixing of a passive scalar (dye) in dilute suspensions of swimming textit{Escherichia coli} in time-periodic flows. Results show that the presence of bacteria hinders large scale transport and reduce overall mixing rate. Stretching fields, calculated from experimentally measured velocity fields, show that bacterial activity attenuates fluid stretching and lowers flow chaoticity. Simulations suggest that this attenuation may be attributed to a transient accumulation of bacteria along regions of high stretching. Spatial power spectra and correlation functions of dye concentration fields show that the transport of scalar variance across scales is also hindered by bacterial activity, resulting in an increase in average size and lifetime of structures. On the other hand, at small scales, activity seems to enhance local mixing. One piece of evidence is that the probability distribution of the spatial concentration gradients is nearly symmetric with a vanishing skewness. Overall, our results show that the coupling between activity and flow can lead to nontrivial effects on mixing and transport.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS15.html#author3"><strong class="card-body-subgroup">Bin Liu</strong></a>
		    	 
		    		<em> (University of California Merced, United States)</em>
		    	
		    	
		    		
		    			</br>"Anomalous size-dependent active transport in structured environments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Variations of transport efficiency in structured environments between distinct individuals in actively self-propelled systems is both hard to study and poorly understood. Here, we study the transport of a non-tumbling Escherichia coli strain, an active-matter archetype with intrinsic size variation but fairly uniform speed, through a periodic pillar array. We show that long-term transport switches from a trapping dominated state for shorter cells to a much more dispersive state for longer cells above a critical bacterial size set by the pillar array geometry. Using a combination of experiments and modeling, we show that this anomalous size-dependence arises from an enhancement of the escape rate from trapping for longer cells caused by nearby pillars. Our results show that geometric effects can lead to size being a sensitive tuning knob for transport in structured environments, with implications in general for active matter systems and, in particular, for the morphological adaptation of bacteria to structured habitats, spatial structuring of communities and for anti-biofouling materials design.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS15.html#author4"><strong class="card-body-subgroup">Nick Cogan</strong></a>
		    	 
		    		<em> (Florida State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling the Origin of Life Reaction in Microfluidic Chambers"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The origins of life are rooted in the organization from small molecules to larger molecules into self-assemblies. This organization requires energetic input that appears to have been driven by temperature and pressure differentials near hydrothermic vents. It has been hypothesized that the building blocks of life originated at the crossroads of high temperature water exacting into the oceans via these vents. Many different chemical reactions have been proposed to study the dynamics of self assemblies across steep chemical gradients. In our study, we focus on the development of a solid membrane via a simplified chemical precipitate reaction. The aims are to understand the physical interaction between the precipitating solid and the fluid dynamics as the membrane barrier is formed. Mathematically, we use a multiphase framework that is highly customizable and addresses the transitions between solids and liquids in a variety of settings. We introduce a slight change in the standard formulation and show that this model is compatible with Darcys’ law and standard porous media equations in different limits. We also provide numerical and  linearized results indicating the affect of a developing solid within a flowing liquid.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS15" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS18.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS18-MFBM: </small></br>Mechanical Models of Complex Diseases</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Fabian Spill (University of Birmingham, USA)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS18.html#author1"><strong class="card-body-subgroup">Vijay Rajagopal</strong></a>
		    	 
		    		<em> (University of Melbourne, Australia)</em>
		    	
		    	
		    		
		    			</br>"Surface area-to-volume ratio, not cellular viscoelasticity is the major determinant of red blood cell traversal through small channels."</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The remarkable deformability of red blood cells (RBCs) depends on the viscoelasticity of the plasma membrane and cell contents and the surface area to volume (SA:V) ratio; however, it remains unclear which of these factors is the key determinant for passage through small capillaries. We used a microfluidic device to examine the traversal of normal, stiffened, swollen, parasitised and immature RBCs. We show that dramatic stiffening of RBCs had no measurable effect on their ability to traverse small channels. By contrast, a moderate decrease in the SA:V ratio had a marked effect on the equivalent cylinder diameter that is traversable by RBCs of similar cellular viscoelasticity. We developed a finite element model that provides a coherent rationale for the experimental observations, based on the nonlinear mechanical behaviour of the RBC membrane skeleton. We conclude that the SA:V ratio should be given more prominence in studies of RBC pathologies.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS18.html#author2"><strong class="card-body-subgroup">Bindi Brook</strong></a>
		    	 
		    		<em> (University of Nottingham, UK)</em>
		    	
		    	
		    		
		    			</br>"Inflammation driven mechanical model of asthmatic airway remodelling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Inflammation, airway hyper-responsiveness and airway remodelling are well-established hallmarks of asthma, but their inter-relationships remain elusive. In order to obtain a better understanding of their inter-dependence, we have developed a mechanochemical morphoelastic model of the airway wall accounting for local volume changes in airway smooth muscle (ASM) and extracellular matrix in response to transient inflammatory or contractile agonist challenges. We use constrained mixture theory, together with a multiplicative decomposition of growth from the elastic deformation, to model the airway wall as a nonlinear fibre-reinforced elastic cylinder. Local contractile agonist drives ASM cell contraction, generating mechanical stresses in the tissue that drive further release of mitogenic mediators and contractile agonists via underlying mechanotransductive signalling pathways. In this talk I will discuss our model predictions and in particular how they: (i)  reveal novel mechanotransductive feedback by which hyper-responsive airways exhibit increased remodelling, for example, via stress-induced release of pro-mitogenic and pro- contractile cytokines; (ii) emergence of a persistent contractile tone observed in asthmatics; (iii) enable identification of various parameter combinations that may contribute to the existence of different asthma phenotypes, and combination of factors which may predispose severe asthmatics to fatal bronchospasms. Finally I will discuss how we plan to use this model to investigate how perturbations from a homoeostatic state might drive asthma pathogenesis.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS18.html#author3"><strong class="card-body-subgroup">Herbert Levine</strong></a>
		    	 
		    		<em> (Northeastern University, USA)</em>
		    	
		    	
		    		
		    			</br>"The role of extracellular matrix in motility and metastasis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In order for cells to migrate from a primary tumor to the circulation as part of the metastatic cascade, it needs to traverse region of fibrous extracellular matrix (ECM). This material has interesting mechanical properties such as strain-stiffening and plasticity, and interesting effects on cells moving through it, such as contact guidance . And, cells themselves can secrete enzymes that modify the ECM, thereby engaging in 'reciprocal' communication with their microenvironment. Here we use simple computational models to try to better understand this set of phenomena.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS18.html#author4"><strong class="card-body-subgroup">Stephanie Fraley</strong></a>
		    	 
		    		<em> (University of California San Diego, USA)</em>
		    	
		    	
		    		
		    			</br>"A spatial model of YAP/TAZ  mechanotransduction reveals new insights into how cells sense ECM dimensionality"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			YAP/TAZ is a master regulator of mechanotransduction; cytoplasmic-to-nuclear translocation of YAP/TAZ responds to different physical cues, including substrate stiffness, substrate dimensionality, and cell shape, and is critical for cellular function and tissue homeostasis. However, the relative contributions and synergies of these biophysical signals to YAP/TAZ translocation remains unclear. For example, in 2D culture, YAP/TAZ nuclear localization correlates strongly with substrate stiffness while in 3D, YAP/TAZ translocation can increase with stiffness, decrease with stiffness, or remain unchanged. Here, we use spatial modeling of YAP/TAZ translocation in response to substrate stiffness to quantitatively analyze the relationships between substrate stiffness, cytosolic stiffness, nuclear mechanics, cell shape, and substrate dimensionality. Our model predicts that increasing substrate activation area through changes in culture dimensionality, while conserving cell volume, forces distinct shape changes that result in nonlinear effect on YAP/TAZ nuclear localization. Moreover, differences in substrate activation area versus total membrane area can account for counterintuitive trends in YAP/TAZ nuclear localization in 3D. Based on this multiscale investigation of the different system features of YAP/TAZ nuclear translocation, we predict that how a cell reads its environment is a complex information transfer function of multiple mechanical and biochemical factors. These predictions reveal design principles of cellular and tissue engineering for YAP/TAZ mechanotransduction.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS18" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS19.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS19-MFBM: </small></br>Algebra, Combinatorics, and Topology in Modern Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Daniel Cruz (Georgia Institute of Technology, U.S.), Margherita Maria Ferrari (University of South Florida, U.S.)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS20-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS19.html#author1"><strong class="card-body-subgroup">Margherita Maria Ferrari</strong></a>
		    	 
		    		<em> (University of South Florida, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Formal grammar modeling three-stranded DNA:RNA braids"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A formal grammar is a system to generate words; it consists of a set of symbols, partitioned into terminals and non-terminals, and a set of production rules. The production rules specify how to rewrite non-terminal symbols, so that successive applications of those rules yield words formed by only terminals. Adding probabilities to the production rules defines stochastic grammars, which can be used for biological sequence analysis. In this talk, we focus on a 'braid grammar' to model R-loops, that are three-stranded structures formed by a DNA:RNA hybrid plus a single strand of DNA, often appearing during transcription. R-loops are described as strings of terminal symbols representing the braiding of the strands in the structure, where each symbol corresponds to a different state of the braided structure. We discuss approaches to develop a stochastic grammar and a probabilistic model for R-loop prediction, as well as refinements of the model by incorporating the effect of DNA topology on R-loop formation
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS19.html#author2"><strong class="card-body-subgroup">Svetlana Poznanovic</strong></a>
		    	 
		    		<em> (Clemson University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Using Polytopes to Improve RNA Branching Predictions"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Minimum free energy prediction of RNA secondary structures is based on the Nearest Neighbor Thermodynamics Model. While such predictions are typically good, the accuracy can vary widely even for short sequences, and the branching thermodynamics are an important factor in this variance. Recently, the simplest model for multiloop energetics - a linear function of the number of branches and unpaired nucleotides - was found to be the best. We develop a branch-and-bound algorithm that finds the set of optimal parameters with the highest average accuracy for a given set of sequences. The search uses the  branching polytopes for RNA sequences. Our analysis shows that previous ad hoc parameters are nearly optimal for tRNA and 5S rRNA sequences on both training and testing sets. Moreover, cross-family improvement is possible but more difficult because competing parameter regions favor different families. The results also indicate that restricting the unpaired nucleotide penalty to small values is warranted. This reduction makes analyzing longer sequences using the present techniques more feasible.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS19.html#author3"><strong class="card-body-subgroup">Chad Giusti</strong></a>
		    	 
		    		<em> (University of Delaware, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Comparing Topological Feature Coding Across Neural Populations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A common feature of the types of information neural populations in the brain encode is cyclicity, meaning that the data is well-represented by one or more independent circular coordinate systems. Persistent homology, a common tool from topological data analysis can be applied to detect and study representations of cyclic features in neural population activity, even without reference to a behavioral correlate. Recent advances in experimental techniques have led to simultaneous recording of activity from populations of neurons across several brain regions, providing an opportunity to study how these representations propagate and change as they move through the brain. Classical topological wisdom tells us that we should apply functoriality to compare topological features across locations. However, in this setting  the goal is, in effect, to impute the map we would need in order to do so. Here, we present a novel method for comparing topological features detected in different brain regions leveraging dissimilarity matrices obtained from observations of activity. No background in topology and neuroscience on behalf of the audience will be assumed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS19.html#author4"><strong class="card-body-subgroup">Abdulmelik Mohammed</strong></a>
		    	 
		    		<em> (University of South Florida, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Topological Eulerian Circuits for the Design of DNA Nanostructures"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Graph theory has recently emerged as a powerful framework for the automated design of biomolecular nanostructures. A prime example of this is in the design of wireframe DNA origami nanostructures, where the routing of a circular viral DNA, called a scaffold strand, is modeled as an Eulerian circuit of a reconditioned triangulated mesh. In this setting, the knot type of the scaffold strand dictates the feasibility of an Eulerian circuit to be used as the scaffold route in the design. We investigate the knottedness of Eulerian circuits on surface-embedded graphs to characterize the class of such graphs that are constructible from unknotted and knotted scaffold strands. We show that certain graph embeddings, called checkerboard colorable, always admit unknotted Eulerian circuits. On the other hand, we prove that if a graph admits an embedding in a torus such that the embedding is not checkerboard colorable, then the graph can be re-embedded so that all its non-intersecting Eulerian circuits are knotted. For surfaces of genus greater than one, we present an infinite family of checkerboard-colorable graph embeddings where there exist knotted Eulerian circuits.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS19" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-MS20.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS20-MFBM: </small></br>Algebra, Combinatorics, and Topology in Modern Biology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Daniel Cruz (Georgia Institute of Technology, U.S.), Margherita Maria Ferrari (University of South Florida, U.S.)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS19-MFBM</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-MS20.html#author1"><strong class="card-body-subgroup">Mustafa Hajij</strong></a>
		    	 
		    		<em> (Santa Clara University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"TDA-Net: Fusion of Persistent Homology and Deep Learning Features for COVID-19 Detection in Chest X-Ray Images"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Topological Data Analysis (TDA) has emerged recently as a robust tool to extract and compare the structure of datasets. TDA identifies features in data such as connected components and holes and assigns a quantitative measure to these features. Several studies reported that topological features extracted by TDA tools provide unique information about the data, discover new insights, and determine which feature is more related to the outcome. On the other hand, the overwhelming success of deep neural networks in learning patterns and relationships has been proven on a vast array of data applications, images in particular. To capture the characteristics of both powerful tools, we propose TDA-Net, a novel ensemble network that fuses topological and deep features for the purpose of enhancing model generalizability and accuracy. We apply the proposed TDA-Net to a critical application, which is the automated detection of COVID-19 from CXR images. The experimental results showed that the proposed network achieved excellent performance and suggests the applicability of our method in practice.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS20.html#author2"><strong class="card-body-subgroup">Hector Banos</strong></a>
		    	 
		    		<em> (Dalhousie University, Canada)</em>
		    	
		    	
		    		
		    			</br>"Identifiability of Species Network Topologies from Genomic Sequences"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Hybridization plays an important role during the evolutionary process of some species. In such cases, phylogenetic trees are sometimes insufficient to describe species-level relationships. We show that most topological features of a level-1 species network (a network with no interlocking cycles) are identifiable under the network multi-species coalescent model (NMSC) using the log-det distance between aligned DNA sequences of concatenated genes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS20.html#author3"><strong class="card-body-subgroup">Nida Obatake</strong></a>
		    	 
		    		<em> (Texas A&M University, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Mixed Volume of Chemical Reaction Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chemical reaction networks model the interactions of chemical substances. An important invariant of a chemical reaction network is its maximum number of positive steady states. This number, however, is in general difficult to compute. We introduce an upper bound on this number - namely, a network's mixed volume - that is easy to compute. We show that, for certain biological signaling networks, the mixed volume does not greatly exceed the maximum number of positive steady states. Furthermore, we investigate this overcount and also compute the mixed volumes of small networks (those with only a few species or reactions).
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-MS20.html#author4"><strong class="card-body-subgroup">Raina Robeva</strong></a>
		    	 
		    		<em> (Randolph-Macon College, U.S.)</em>
		    	
		    	
		    		
		    			</br>"Algebraic Biology in the Curriculum"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Unlike difference and differential equations models, algebraic models in biology have remained largely invisible at the undergraduate level despite their increasing popularity in solving a wide range of biological problems. This discrepancy is puzzling as, in many cases, an introduction to algebraic modeling relies on mathematics covered in traditional mathematics courses. In addition, Boolean and finite dynamical systems, could be used as an alternative to modeling system dynamics by way of difference and differential equations. The talk will discuss the benefits of increasing the profile of algebraic models in the undergraduate curriculum and share insights from an honors class in systems biology for students with minimal mathematics and biology backgrounds.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS20" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <a id="CT"><h1 class="mb-3" id="MS1">Sub-group contributed talks</h1></a>
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT01.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT01-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT01.html#author1"><strong class="card-body-subgroup">Wayne Hayes</strong></a>
		    	 
		    		<em>UC Irvine</em>
		    	
		    	
		    		
		    			</br>"The One True Way to use GO terms to evaluate Network Alignments"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Sequence alignment has contributed immensely to our understanding of biology, evolution, and disease. While the genome encodes recipes for making proteins, the function of many proteins remains elusive. Since the function of a protein is intimately tied to its interaction partners, the topology of protein-protein interaction (PPI) networks holds promise as way to decode function. Topologically-driven network alignment attempts to find the best mapping between the PPI networks of two species by finding the greatest amount of common network topology. However, network alignment research is still in its infancy and there are dozens of proposed methods but no objective, mathematically rigorous methods to compare their results.  Here we propose a rigorous, formal method to compute the p-values of shared GO terms between pairs of proteins found by a network alignment, compared to random alignments. We compare our p-values to billions of actual random alignments to demonstrate that the p-values are correct within statistical uncertainty of the sample random alignments.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author2"><strong class="card-body-subgroup">Giulia Palermo</strong></a>
		    	 
		    		<em>UCR</em>
		    	
		    	
		    		
		    			</br>"Harnessing graph theory to decrypt the allosteric mecahnism in CRISPR-Cas9"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			CRISPR-Cas9 is a bacterial adaptive immune system that emerged as the centerpiece of a transformative genome editing technology. In this system, an intriguing allosteric communication has been suggested to control the DNA cleavage activity through the flexibility of the catalytic HNH domain. Here, we report about the use of molecular dynamics and graph theory-based analysis methods to describe the structural and dynamic determinants of the allosteric signaling in the CRISPR-Cas9 complex. Network models derived from graph theory reveal the existence of a contiguous dynamic pathway that enables the information transfer across the HNH domain. This pathway spans HNH from the region interfacing the RuvC nuclease and propagates up to the DNA recognition lobe in the full-length CRISPR-Cas9, such transferring the signal of DNA binding at the nuclease domains for concerted cleavages of the two DNA strands. These findings reveal the mechanism of signal transduction within the CRISPR-Cas9 nuclease and pose the basis for the complete mapping of the allosteric pathway, and of its role in the DNA on-target specificity, helping engineering efforts aimed at improving the genome editing capability of CRISPR-Cas9.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author3"><strong class="card-body-subgroup">Dennis Manjaly Joshy</strong></a>
		    	 
		    		<em>UC Santa Barbara</em>
		    	
		    	
		    		
		    			</br>"A Koopman Operator Approach for Genetic Circuit Design"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We consider the problem of genetic circuit design to achieve an arbitrary data-driven or function-based performance specification.   We review the open nature of this nonlinear design problem and its relation to the optimal and robust controller synthesis problem.  We show how a class of biological networks, modeled with first order, zeroth order, and Hill function dynamics can be represented with a Koopman operator to yield a linear representation of system dynamics on a space of functions.  This formulation allows us to directly solve the controller synthesis problem to meet a given performance specification as an optimization problem on a particular physical basis of observable functions.  We demonstrate our approach on the optimization of a positive amplifier circuit in bacteria, showing how design recommendations from our controller synthesis algorithm can be translated to DNA sequence-level specification.  These results solve an outstanding problem in genetic circuit design - synthesis of closed-loop systems to meet a target performance specification.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT01.html#author4"><strong class="card-body-subgroup">Ying Zhang</strong></a>
		    	 
		    		<em>Brandeis University</em>
		    	
		    	
		    		
		    			</br>"Immersed Boundary Simulations of Red Blood Cells Near Vessel Walls"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Platelets constitute an essential component of human blood due to their role in the formation of hemostatic plug and thrombus. The occurrence of these biological phenomena requires platelets stay within close proximity to the vessel walls, initiating platelet-wall interaction. It has been understood that the red blood cells (RBCs) play an important role in platelet near-wall excess. Healthy RBCs are highly deformable objects, and thus can acquire lift forces from vessel walls from their deformation to propel them away from the wall, a phenomenon known as wall-induced migration. Migration of RBCs away from the wall leads to the formation of a cell-depleted layer near the wall, which has a large effect on the motion of platelets. Here we use the immersed boundary method to investigate the influence of cell stiffness and shape on the wall-induced migration. In particular, we focus on analyzing how lift force and mobility change over time when a RBC is placed close to the wall. Our preliminary results suggest that deformation of a RBC leads to a larger lift force when the RBC is closer to the wall, increasing the likelihood of RBCs migrating away from the vessel wall.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT02.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT02-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT02.html#author1"><strong class="card-body-subgroup">Yifei Li</strong></a>
		    	 
		    		<em>Queensland University of Technology</em>
		    	
		    	
		    		
		    			</br>"Travelling waves in nonlinear reaction-diffusion equations"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Reaction-diffusion equations (RDEs) are often derived as continuum limits of lattice-based discrete models. Recently, a discrete model which allows the rates of movement, proliferation and death to depend upon whether the agents are isolated has been proposed, and this approach gives various RDEs where the diffusion term is convex and can become negative (Johnston et al., Sci. Rep. 7, 2017). Numerical simulations suggest these RDEs, under certain choices of the system parameters, support smooth and shock-fronted travelling waves. In this talk, I will formalise these preliminary numerical observations by analysing these two types of travelling wave solutions through a dynamical systems approach.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT02.html#author2"><strong class="card-body-subgroup">Juan Carlo Flores Mallari</strong></a>
		    	 
		    		<em>Ateneo de Manila University, Nara Institute of Science and Technology</em>
		    	
		    	
		    		
		    			</br>"Modeling Collective Behavior of Passengers Boarding and Disembarking from Public Transport in the Philippines"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this study, we develop a self-propelled particle model of pedestrian motion based on social interaction forces for passengers boarding and disembarking from buses in the Philippines. The forces—attractive, repulsive, and frictional—are derived from positional data acquired by tracking numerous drone videos of a section of a busy highway in Metro Manila. Model validity is tested by simulating passenger trajectories and comparing these simulations with actual data. The model will be used to explore the transmission of COVID-19 through the Philippine public transportation system and to build a framework with which policy interventions and organized crowd control systems (e.g., proper queuing systems and bus stop usage, physical distancing requirements to prevent disease transmission) can be conceptualized and justified. In the future, we will be analyzing the spread of COVID-19 by introducing a variable number of infected passengers to the system and use existing transmission onset data to give particular attention to pre-symptomatic transmission. While the study focuses on bus passengers, the model can be easily modified to be applied to other modes of transportation. In addition, while the study is motivated by COVID-19, the framework to be generated will be usable for analyzing outbreaks of other infectious diseases.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT02.html#author3"><strong class="card-body-subgroup">Leo Diaz</strong></a>
		    	 
		    		<em>University of Melbourne</em>
		    	
		    	
		    		
		    			</br>"Hypergraphs as a tool for automated and reproducible modelling of biochemical systems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			It is becoming obvious that we need better modelling tools as we strive to build larger and more interpretable models of biological systems. Specifically, we cannot keep relying on bespoke, hand-coded models when we want to be able to explore model spaces comprehensively.In this talk I demonstrate that using hypergraphs results in a computationally efficient scheme to represent and generate mathematical models of biological systems. I show that chemical hypergraphs can represent biochemical reaction networks exactly, and that hypergraphs have the potential to represent dynamical systems more generally. This framework allows us to easily manipulate and compose mathematical models, even of large systems, thereby enabling us to explore model spaces automatically. I illustrate the use of chemical hypergraphs using models of gene regulation processes.This framework allows us to construct models that other approaches may be unable to -- considering feedback loops, for instance, is trivial here. Further advantages of this approach are a more accessible, less error-prone and reproducible modelling process.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT02.html#author4"><strong class="card-body-subgroup">Shawn Means</strong></a>
		    	 
		    		<em>Massey University</em>
		    	
		    	
		    		
		    			</br>"A Permutation Method to Assemble Networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Networks can represent entities as disparate as genes, computers, infected people, predators and prey or neurons of the brain. Details of underlying structures for given systems amenable to network representations are typically limited to numbers of connections between entities or their node-degree. These degrees may be number of sexual partners, prey species, or synaptic connections in a brain. Realising a network with a given sequence of node-degrees presents a challenge especially if multiple connections or loop-backs for nodes are forbidden — otherwise known as a simple graph. Standard methods of network assembly for sampling a graph space, or all potential realisations of some degree sequence, typically require significant post-processing of initial assemblies to remove multiple connections and loop-backs. We devised an alternative method that not only permits outright exclusion of these edges, but also can target prescribed proportions of them for networks with weighted-edges. These weights may represent multiple interactions, say, between sexual partners, prey species or synaptic connections. We present our method that successfully builds networks with order 10^7 edges on scales of minutes running on a laptop, as well as links to our implementation on the GitHub repository.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT02" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT03.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT03-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT03.html#author1"><strong class="card-body-subgroup">Gregory Szep</strong></a>
		    	 
		    		<em>King's College London</em>
		    	
		    	
		    		
		    			</br>"Parameter Inference with Bifurcation Diagrams"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Estimation of parameters in differential equation models can be achieved by applying learning algorithms to quantitative time-series data. However, sometimes it is only possible to measure qualitative changes of a system in response to a controlled condition. In dynamical systems theory, such change points are known as bifurcations and lie on a function of the controlled condition called the bifurcation diagram. In this work, we propose a gradient-based semi-supervised approach for inferring the parameters of differential equations that produce a user-specified bifurcation diagram. The cost function contains a supervised error term that is minimal when the model bifurcations match the specified targets and an unsupervised bifurcation measure which has gradients that push optimisers towards bifurcating parameter regimes. The gradients can be computed without the need to differentiate through the operations of the solver that was used to compute the diagram. We demonstrate parameter inference with minimal models which explore the space of saddle-node and pitchfork diagrams and the genetic toggle switch from synthetic biology. Furthermore, the cost landscape allows us to organise models in terms of topological and geometric equivalence.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT03.html#author2"><strong class="card-body-subgroup">Sta Léa</strong></a>
		    	 
		    		<em>University of Leeds, United Kingdom</em>
		    	
		    	
		    		
		    			</br>" IL-7R mathematical modelling: algebraic expressions for amplitude and EC50"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Effector T cells rely on the cytokine IL-7 to receive receptor-mediated signalling for their survival. The IL-7 receptor (IL-7R), composed of the common gamma chain and the specific alpha chain, is also associated with the kinase JAK3, which triggers its signalling pathway. Recently, study of cell-to-cell variability and flow cytometry data yielded a seemingly paradoxical observation: increased expression of gamma chains reduces the IL-7 response. We introduce a mathematical model of  cytokine IL-7  and IL-7R signalling that provides an explanation for this empirical observation. Our results show the formation of dummy complexes  (those receptors that are bound to ligand but not to  the JAK3 kinase,  and are thus, unable to signal)  and indicate that the balance between the number of IL-7R subunits in one cell is crucial for optimal signalling. We make use of a method in algebraic geometry, the Groebner basis, to compute exact analytical expressions for the maximum IL-7 response (or amplitude) and for the half-maximal effective concentration of ligand (EC50) of our mathematical models of cytokine-receptor signalling. While predicted amplitudes agree with the experimental data, measurements of EC50 exhibit more complicated behaviour than we have managed to capture with  a variation of our IL-7R model.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT03.html#author3"><strong class="card-body-subgroup">Johannes Borgqvist</strong></a>
		    	 
		    		<em>Wolfson Centre for Mathematical Biology, Mathematical Institute, University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Symmetry methods for model-construction and analysis in the context of collective cell migration"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mathematical modelling is a vital tool in coping with complexity on numerous spatial and temporal scales, and a key goal of modelling is to be able to predict future outcomes using model analysis and simulation. The challenge for this dream scenario is the difficulty of validating a particular model, and it is often achieved by attempting to fit the model to observed data. However, often there are multiple candidate models available which renders the task of knowing which description is “correct” very difficult. In order to encode physical properties of the studied system in the construction phase of a model, a novel mathematical technique called symmetry methods can be used. The method of symmetries originates from mathematical physics, and they are transformations that encode physical properties, often formulated as conservation laws. Symmetries have been used with huge success in theoretical physics, but are relatively unexplored in a biological context. Here, the application of symmetries for finding analytical solutions to partial differential equation models of cell migration is showcased, as well as a methodology for model selection. Finally, the difficulties of finding symmetries of large biological models in an automated fashion are discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT03.html#author4"><strong class="card-body-subgroup">Remus Stana</strong></a>
		    	 
		    		<em>University of Leeds</em>
		    	
		    	
		    		
		    			</br>"Diffusion in a domain with inclusion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many cells of the immune system have molecules which are produced in the nucleus and these move under the influence of diffusion until they reach the outer membrane of the cell. Depending on the type of molecule, they might also diffuse on the surface of the cell until either a certain period of time has passed or the molecule forms a complex. After either of these events the molecules re-enter the cytoplasm and diffuse until they are absorbed by the nucleus. We are interested in the first passage properties of the molecules. For this purpose, we derive an analytic expression for the Green's function of the Laplace equation for a domain bounded by non-concentric surfaces in two dimensions and three dimensions subject to mixed boundary conditions. Utilizing the Green's function we derive an exact expression for the mean time for a Brownian molecule to return to the nuclear surface given that it hit the cellular surface and compare with previous results in the literature. Furthermore, using the Green's function we calculate an exact formula for the hitting density of molecules on the cellular surface and compare it with numerical results.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT03" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT04.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT04-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT04.html#author1"><strong class="card-body-subgroup">Elizabeth Trofimenkoff</strong></a>
		    	 
		    		<em>University of Lethbridge</em>
		    	
		    	
		    		
		    			</br>"An algorithm for obtaining parametric conditions for the validity of the steady-state approximation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mathematical models, particularly in biology and biochemistry, can increase in complexity very quickly. The steady-state approximation (SSA) is an extremely powerful and valuable tool for simplifying the behaviour of a dynamical system. This model reduction process results in fewer variables, which allows equations to integrate faster, and fewer parameters to find values for, thus making the modelling process much more efficient. Rigorously establishing conditions that validate the SSA is laborious, and requires detailed time scale estimates specific to each system of interest. We have developed a versatile algorithm that provides a stepwise prescription for scaling from which conditions for the validity of the SSA based on Tikhonov's theorem can be elucidated. The algorithm requires only elementary algebraic manipulations to determine sufficient conditions at which the SSA is valid, making this algorithm more accessible to non-specialists. The algorithm recovers the Segel and Slemrod scaling for the Michaelis-Menten mechanism. The algorithm is robust, and can also be applied relatively easily to much more complex mechanisms.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT04.html#author2"><strong class="card-body-subgroup">Alexander D. Kaiser</strong></a>
		    	 
		    		<em>Stanford University</em>
		    	
		    	
		    		
		    			</br>"Design-based models of heart valves and bicuspid aortic valve flows"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			This talk presents new methods for modeling and simulation of the aortic and mitral heart valves and use of these methods to study congenital heart disease. To construct model heart valves, we specify that the heart valve supports a pressure and derive an associated system of partial differential equations for its loaded state. Using the solution to this system, we then derive reference geometry and material properties. By tuning the parameters in this process, we design the model valves. This process produces material properties that are consistent with known values, yet also includes material heterogeneity. Results will be shown for both the aortic and mitral valves. When used in fluid-structure interaction simulations, these models are highly effective, producing realistic flow rates and robust closure under physiological driving pressures. Using these models, we study flows through the bicuspid aortic valve. Simulations show that a bicuspid valve, without alterations to the aorta anatomy, alters blood flow patterns dramatically. These flows suggest that hemodynamics play a strong role in aortic dilation and aneurysm formation.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT04.html#author3"><strong class="card-body-subgroup">Jay Stotsky</strong></a>
		    	 
		    		<em>University of Minnesota</em>
		    	
		    	
		    		
		    			</br>"The Impact of Cell-Level Details on Tissue-Scale Properties"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The transport of various chemical species through cellular tissues is a widespread and important phenomenon in biology. At a microscopic level, such processes are often extremely complicated, possibly involving binding, diffusive transport, chemical changes, among other steps - all of this occurring in domains with non-trivial geometry. Nonetheless, at a tissue-scale, these processes are often modeled, as advection-diffusion-reaction equations occurring in homogeneous media. Thus, an important question is how the parameters that appear in such macro-scale models relate to what occurs at the cellular level (and vice-versa). In this talk, I will discuss how multi-state continuous-time random walks and generalized master equations can be used to model transport processes involving spatial jumps, immobilization at particular sites, and stochastic internal state changes. The underlying spatial models, which are framed as graphs, may have different types of nodes and edges, and walkers may have internal states that are governed by a Markov process. I will then discuss the key question of how macro-scale coefficients may be obtained from such models. This work is motivated by problems arising in the transport of proteins in biological tissues, specifically the Drosophila wing-imaginal disc, but the results are applicable to a broad array of problems.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT04.html#author4"><strong class="card-body-subgroup">Fawaz K Alalhareth</strong></a>
		    	 
		    		<em>The University of Texas at Arlington</em>
		    	
		    	
		    		
		    			</br>"Higher-Order Modified Nonstandard FiniteDifference Methods for Dynamical Systems in Biology"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Nonstandard finite difference (NSFD) methods have been widely used to numerically solve various problems in Biology. NSFD methods also have several advantages over standard techniques, such as preserving many of the essential properties of the solutions of the differential equations with no restriction on the time-step size. However, most of the NSFD methods developed to date are only of first-order accuracy. In this talk, we discuss the construction and analysis of a new class of second-order modified NSFD methods for general classes of autonomous differential equations. The proposed new methods are easy to implement and represent higher-order generalizations of the positive and elementary stable nonstandard (PESN) methods. Numerical simulations are also presented to support the theoretical results.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT04" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT05.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT05-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT05.html#author1"><strong class="card-body-subgroup">Fatemeh Sadat Fatemi Nasrollahi</strong></a>
		    	 
		    		<em>Pennsylvania State University</em>
		    	
		    	
		    		
		    			</br>"Attractor identification method based on generalized positive feedback loops and their functional relationships"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Boolean modeling has been shown to successfully capture the attractors (emergent behaviors) in complex systems. Here we propose an efficient attractor finding method that relies on the identification of stable motifs [1] in Boolean models of plant-pollinator community assembly [2]. Stable motifs are the smallest positive cycles in the network that can sustain a specific state regardless of the state of the nodes outside the stable motif. We find that stable motifs can have three types of functional relationships with each other: Mutual exclusivity: Two stable motifs stabilizing the same node(s) but in different states; Conditionality: Stabilization of a stable motif only when a set of conditions are met via the stabilization of a different stable motif; Logical determination: Automatic stabilization of a stable motif as a result of stabilization of another stable motif. Based on these relationships, we developed an algorithm to identify all self-consistent mutually exclusive groups of stable motifs, and showed that stabilization of any of these groups leads to a distinct attractor. We applied this algorithm to 4000 networks of 40-100 species, compared its performance with three other attractor identification methods, and showed that it can speed and simplify the attractor identification task considerably.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author2"><strong class="card-body-subgroup">Megan Coomer</strong></a>
		    	 
		    		<em>Melbourne University</em>
		    	
		    	
		    		
		    			</br>"Shaping the Epigenetic Landscape: Complexities and Consequences"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The metaphor of Waddington's epigenetic landscape has become an iconic representation of cellular differentiation.  Single-cell transcriptomic data allows us to probe this landscape and gain insights into the regulatory dynamics underlying developmental processes. Reconstructing such landscapes from data has typically been based on strong assumptions about the dynamics of cells through gene expression state. Often, concepts from equilibrium thermodynamics have been used. Since biological processes are inherently noisy it is important to consider the presence of stochastic fluctuations in this context.  We use a simple model to highlight complexities and limitations that arise when reconstructing the potential landscape in the presence of stochasticity. Specifically, we contrast ways in which additive and multiplicative noise shape the landscape on top of the deterministic dynamics.  We show that the subtle interplay between the deterministic and stochastic components of the system's dynamics can have very unsubtle consequences: depending on the dynamics and noise, even qualitative features of the system dynamics — number and nature of stationary points — can change. Casual or ad hoc modelling of noise in the underlying regulatory networks can mask these effects. We end with a discussion of how this can be accounted for when considering single cell transcriptomic data.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author3"><strong class="card-body-subgroup">Kumar Saurabh</strong></a>
		    	 
		    		<em>National Taiwan University</em>
		    	
		    	
		    		
		    			</br>"MATHEMATICAL MODELING OF ION CHANNEL FLOW WITHIN CONTINUUM FRAMEWORK"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Within the continuum framework, ion transport can be described using Poisson-Nernst-Planck (PNP) equations. Although accurate for dilute flows, PNP equations are not appropriate for modeling flows with high ion concentration or flows where non-ionic interactions are important. For ion channel flow, several extensions to continuum theory has been proposed. Effect of  finite size of ion can be modeled by including either Lennard-Jones potential [1, 2, 3] in the energetic formulation or Bikerman model [4, 5]. For effect of ion solvation, Born energy model can be included in the system [3, 4]. Additionally, to account for spatial variation of dielectric behavior of the aqueous medium on can resort to nonlocal electrostatics [5]. Numerically, the system is modeled using lattice Boltzmann method (LBM) in conjunction with immersed boundary method (IBM) to address the boundary conditions. Further, to reduce computational cost, the code has been parallelized on multiple GPUs using CUDA platform. These mathematical models have been successfully implemented for ion flow through SARS-CoV-1 and SARS-CoV-2 E protein ion channel, TRPV channel etc. In this study, we intend to explore the role and effect of these mathematical models on ion transport through a potassium channel.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT05.html#author4"><strong class="card-body-subgroup">Nayana Wanasingha</strong></a>
		    	 
		    		<em>Department of Mathematical Sciences, University of Cincinnati</em>
		    	
		    	
		    		
		    			</br>"Molecular mechanisms regulating frequecy demultiplication of circadian rhythms in Neurospora Crassa"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Subharmonic entrainment or frequency demultiplication is a characteristic of circadian systems, which is the ability to entrain to cycles that are submultiples of external cycles. In this study, we used mathematical modeling and experiments to investigate potential mechanisms regulating frequency demultiplication under different temperature cycles in a model filamentous fugus, Neurospora crassa. Our results indicate that frequency demultiplication is a manifestation of the entrainment of circadian clock to external cycles and depends on the endogenous period and the strength and type of external cycles. Theoretical analysis reveals two necessary conditions to reproduce experimentally observed frequency demultiplication and frequency driven phenotypes: 1) temperature-modulated frq transcription and translation, and 2) a low level of cooperativity of transcriptional regulation of frq. In summary, we used mathematical modeling and experiments to uncover the architecture of circadian systems regulating frequency demultiplication, which broadens our fundamental understanding of entrainment of circadian rhythms.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT05" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT06.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT06-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT06.html#author1"><strong class="card-body-subgroup">Miroslav Phan</strong></a>
		    	 
		    		<em>ETH Zurich, Department of Biosystems Science and Engineering, Basel, Switzerland</em>
		    	
		    	
		    		
		    			</br>"A Rejection Based Gillespie Algorithm for non-Markovian Stochastic Processes with Individual Reactant Properties"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Gillespie algorithm is commonly applied for simulating memoryless processes that follow an exponential waiting-time. However, stochastic processes governing biological interactions, such as cell apoptosis and epidemic spreading, are empirically known to exhibit properties of memory, an inherently non-Markovian feature. The presence of such non-Markovian processes can significantly influence the outcome of a simulation. While several extensions to the Gillespie algorithm have been proposed, most of them suffer from either a high computational cost, or are only applicable to a narrow selection of probability distributions that do not match the experimentally observed biological data distributions. To tackle the aforementioned issues, we developed a Rejection Gillespie for non-Markovian Reactions (REGINR) that is capable of generating simulations with non-exponential waiting-times, while remaining an order of magnitude faster than alternative approaches. REGINR uses the Weibull distribution, which interpolates between the exponential, normal, and heavy-tailed distributions. We applied our algorithm to a mouse stem cell dataset with known non-Markovian dynamics and found it to faithfully recapitulate the underlying biological processes. We conclude that our algorithm is suitable for gaining insight into the role of molecular memory in stochastic models, as well as for accurately simulating real-world biological processes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT06.html#author2"><strong class="card-body-subgroup">Elba Raimundez</strong></a>
		    	 
		    		<em>University of Bonn</em>
		    	
		    	
		    		
		    			</br>"Efficient sampling by marginalization of scaling parameters for mechanistic models with relative data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Mathematical models are standard tools for understanding the underlying mechanisms of biological systems. Generally, the parameters of these models are unknown and they need to be inferred from experimental data using statistical methods. Most common measurement techniques only provide relative information about the absolute molecular state and often data is noise-corrupted. Therefore, introducing scaling and noise parameters in the model observables is necessary. Since frequently these parameters are also unknown, the dimensionality of the estimation problem is augmented. Sampling methods are widely used in systems biology to assess parameter and prediction uncertainties. However, the evaluation of sampling methods is usually demanding and often on the border of computational feasibility. Hence, efficient sampling algorithms are required.We propose a marginal sampling scheme for estimating the parameter uncertainties of mechanistic models with relative data. We integrate out the scaling and noise parameters from the original problem, leading to a dimension reduction of the parameter space. Herewith, only reaction rate constants have to be sampled. We find that the marginal sampling scheme retrieves the same parameter probability distributions and outperforms sampling on the full parameter space by substantially increasing the effective sample size and smoothing the transition probability between posterior modes.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT06.html#author3"><strong class="card-body-subgroup">Aden Forrow</strong></a>
		    	 
		    		<em>Mathematical Institute, University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Learning stochastic dynamics with measurement noise"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A core challenge of modern mathematical biology is fitting models to the ever increasing quantities of biological data. Frequently, the data is affected by both measurement noise and stochasticity intrinsic to the underlying system in ways that are difficult to disentangle. Building on prior work addressing the two kinds of stochasticity independently, we present a method for inferring dynamics without assuming either noise-free measurements or an underlying deterministic system. Our approach is motivated by measurement techniques available in single-cell sequencing and generically applicable for learning stochastic differential equations from noisy data.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT06.html#author4"><strong class="card-body-subgroup">Marco Berghoff</strong></a>
		    	 
		    		<em>Karlsruhe Institute of Technology</em>
		    	
		    	
		    		
		    			</br>"Cells In Sillico â€“ Parallel Tissue Development Simulation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Insights in cell dynamics and tissue development are constantly changing our understanding of fundamental biological processes including embryogenesis, wound healing, and tumorigenesis. The availability of high-quality microscopy data and an increasing understanding of single-cell effects are speeding up discoveries. However, many computational models still describe either a few cells in high detail or larger cell ensembles and tissues in rather coarse detail. We combine these two scales, therefore we developed a highly parallel version of the cellular Potts model and provides an agent-based model for controlling cellular events. The model can be modularly extended to a multimodel simulation at both scales. Based on the NAStJA framework, we implemented a scalable version that runs efficiently on high-performance computing systems.Our model scales beyond 10,000 cores in an approximately linear manner, enabling the simulation of large three-dimensional tissues. The strictly modular design allows flexible configuration of arbitrary models and enables applications in a wide range of research questions. Cells in Silico can be easily adapted to different modeling assumptions and helps computational scientists to extend their simulations to a new area of tissue simulations. As an example, we show a 1000^3 voxel cancer tissue simulation with sub-cellular resolution.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT06" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT07.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT07-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT07.html#author1"><strong class="card-body-subgroup">Eli Newby</strong></a>
		    	 
		    		<em>The Pennsylvania State University</em>
		    	
		    	
		    		
		    			</br>" 	 Identifying Driver Nodes in Biological Networks using Subsets of the Feedback Vertex Set"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In network control theory, driving all the nodes in the Feedback Vertex Set (FVS) forces the network into one of its attractors (long-term dynamic behaviors), but the FVS is often composed of more nodes than can be realistically manipulated in a system (e.g., 1-3 nodes for intracellular networks). Thus, we developed a method to identify subsets of the FVS on Boolean models of intracellular networks using topological, dynamics-independent measures. We identified seven topological measures sorted into three categories — centrality measures, propagation measures, and cycle-based measures. Each measure was ranked and then evaluated against two dynamics-based metrics that measure ability of interventions to drive the system towards or away from their attractors: To Control and Away Control. After examining various biological networks, we found that the FVS subsets that ranked highest according to the propagation metrics could most effectively control the network. This result was independently corroborated on an array of different Boolean models of biological networks. Consequently, overriding the entire FVS is not required to drive a biological network to one of its attractors, and this method provides a way to reliably identify these FVS subsets without knowledge of the network's dynamics.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT07.html#author2"><strong class="card-body-subgroup">Xiaoyu Duan</strong></a>
		    	 
		    		<em>University of Pittsburgh</em>
		    	
		    	
		    		
		    			</br>"Parameter identification of linear-in-parameters systems from a single trajectory"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			A fundamental problem in mathematical modeling is the determination of model parameter values from experimental data. In disease and immunological studies, repeated collection of data from a single subject is impossible, because the disease or the manipulations performed in the experiments are fatal to the subject, or permanently alter the subject's immune system. Therefore, parameter identification or estimation of nonlinear ODE systems from a single trajectory has been studied in my research. Many computational techniques for parameter estimation has been employed. However, it is also important to have rigorous, theoretical results, addressing the questions whether there exists solution, a unique solution, or no solution to the parameter estimation problem. If we view the models as forward mappings from parameter values to states of model variables, then the parameter identification/estimation from data can be formulated as the problem of inverting this mapping. Therefore this is an inverse problem and its solution is the set of parameters values. In particular, in my research, I focus on a certain type of nonlinear systems, which is linear-in-parameters system. I will use the two-dimensional Lotka-Volterra system as an example to show our results and explain difficulties in this research area.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT07.html#author3"><strong class="card-body-subgroup">Katherine Pearce</strong></a>
		    	 
		    		<em>Department of Mathematics, North Carolina State University</em>
		    	
		    	
		    		
		    			</br>"Modeling and parameter subset selection for fibrin polymerization kinetics with applications to wound healing"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			During hemostasis in wound healing, vascular injury leads to endothelial cell damage, initiation of a coagulation cascade involving platelets, and formation of a fibrin-rich clot. Activation of the protease thrombin occurs and soluble fibrinogen is converted into an insoluble polymerized fibrin network. Fibrin polymerization is critical for bleeding cessation and subsequent stages of wound healing. We present a cooperative enzyme kinetics model for in vitro fibrin matrix polymerization capturing dynamic interactions among fibrinogen, thrombin, fibrin and intermediate complexes. A tailored parameter subset selection technique is implemented to evaluate parameter identifiability for a representative dataset for fibrin accumulation. Our approach is based on systematic analysis of the eigenvalues and eigenvectors of the information matrix for the quantity of interest fibrin matrix via optimization, based on a least squares objective function. Capabilities of this approach to decrease the objective cost and integrate non-overlapping subsets of the data to enhance the evaluation of parameter identifiability and aid in model reduction are also demonstrated. These findings illustrate the high degree of information within a single fibrin accumulation curve using a tailored model and parameter subset selection approach that can improve optimization and reduce model complexity.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT07.html#author4"><strong class="card-body-subgroup">Yue Liu</strong></a>
		    	 
		    		<em>University of Oxford</em>
		    	
		    	
		    		
		    			</br>"Organisation of diffusion-driven stripe formation in expanding domains"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In certain biological systems, such as the plumage pattern of birds and stripes on certain species of fishes, pattern formation take place behind a wave of competency. For these systems, one needs to consider the patterns that form when a particular type of growth -- apical growth -- is included. In this study, we use a particular type of partial differential equation model, known as a Turing diffusion-driven instability model, to study pattern formation on apically growing domains, under a variety of rates of growth. Numerical simulations show that in one spatial dimension a slower growth rate drives a sequence of peak splittings in the pattern, whereas a higher growth rate leads to peak insertions. In two spatial dimensions, we observe stripes that are either perpendicular or parallel to the moving boundary under slow or fast growth rates, respectively. To understand this phenomenon, we use stability and bifurcation analysis to understand the process of selection of stripes or spots. Finally, we argue that there is a correspondence between the one- and two-dimensional phenomena, and that apical growth can account for the pattern organization observed in many biological systems.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT07" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MFBM/MFBM-CT09.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>CT09-MFBM: </small></br>MFBM Subgroup Contributed Talks</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
			
			<ul>
	    	
		    	<li><a href="/MFBM/MFBM-CT09.html#author1"><strong class="card-body-subgroup">Ivan Borisov</strong></a>
		    	 
		    		<em>INSYSBIO LLC</em>
		    	
		    	
		    		
		    			</br>"Constrained Optimization Approach to Predictability Analysis in Bio-Mathematical Modeling"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Background:  Identifiability analysis is a crucial step in improving reliability and predictability of biological models. Profile Likelihood (PL) is a reliable though computationally expensive approach to identifiability analysis. PL-based algorithm Confidence Intervals by Constraint Optimization (CICO), which was recently published (https://doi.org/10.1371/journal.pcbi.1008495/), reduces computational requirements and increases the accuracy of the estimated parameters' confidence intervals. The CICO algorithm is available in a free software package LikelihoodProfiler based on Julia (https://github.com/insysbio/LikelihoodProfiler.jl). CICO can be potentially extended to  predictability analysis and confidence bands estimation.Objectives: The goal of this study is to examine the application of CICO to estimation of confidence and prediction bands. The analysis was performed on a number of published biological models, including STAT5 Dimerization model, Cancer Taxol Treatment model, etc.Results: The original CICO algorithm can be extended to a broader use-case of confidence bands. The analysis demonstrates good performance characteristics for both identifiable and non-identifiable cases. The approach can be used with complex biological models where each likelihood estimation is computationally expensive and some output values are non-identifiable. Detailed analysis of each model can be found on the GitHub repository likelihoodprofiler-cases https://github.com/insysbio/likelihoodprofiler-cases.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT09.html#author2"><strong class="card-body-subgroup">Anna Molyavko</strong></a>
		    	 
		    		<em>ICM SB RAS, Krasnoyarsk Mathematical Center</em>
		    	
		    	
		    		
		    			</br>"Novel alignment-free highly parallel method to compare symbol sequences of an arbitrary length"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Comparison of long symbolic sequences corresponding to various biological macromolecules is the principal tool in bioinformatics, biophysics, and other life sciences. However, symbolic sequence alignment, currently widely used for computing comparison, suffers from multiple downsides. Some of them are subjective parameters' choice, divergence, and high computational complexity. The paper proposes an alignment-free non-parametric, highly efficient novel method to compare symbolic sequences based on a binary multi-channel encoding and their Fourier convolution. Due to the high O(n*log(n)) efficiency of the Fourier convolution, the method can process sequences up to 10^7 symbols long on consumer-level hardware under half an hour. Also, the method lends itself to a straightforward parallelization. The base version of the algorithm determines the number of exact matches in any overlapping configuration of two sequences and provides it in a single run of the convolution calculation. The advanced version determines the number of exactly matching k-mers in those configurations. The insertions and deletions, which present significant challenges for the alignment-based computations, do not affect the proposed method's efficiency.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT09.html#author3"><strong class="card-body-subgroup">Dimitrios Patsatzis</strong></a>
		    	 
		    		<em>National Technical University of Athens</em>
		    	
		    	
		    		
		    			</br>"Towards extending the arsenal of cancer immunology modeling with algorithmic asymptotic analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The recent advances in cancer immunotherapy paved the way for the development of mathematical models formulating the complex interactions between the tumor and the immune system, with the aim to indicate more efficient treatment regimes. However, the complexity of such models and their multi-scale character renders them inaccessible for wide utilization and hinders the acquisition of physical understanding. In order to tackle these obstacles, here the algorithmic tools of asymptotic analysis are utilized in a fundamental model formulating the interactions of tumor cells with natural killer cells, CD8+ T cells and circulating lymphocytes. It is firstly revealed that the long-term evolution of the system towards the high-tumor or the tumor-free equilibrium is determined by the dynamics of an initial explosive stage of tumor progression. Focusing on this stage, the algorithmic Computational Singular Perturbation methodology is employed to identify the underlying mechanisms confining the system to evolve towards the high-tumor equilibrium and the governing slow dynamics along them. The results demonstrate the potential of algorithmic asymptotic analysis to simplify the complex, overeparameterized and multi-scale cancer immunology models and to indicate the interactions and cell types to target for more effective treatment development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MFBM/MFBM-CT09.html#author4"><strong class="card-body-subgroup">Viktoriia Fedotovskaia</strong></a>
		    	 
		    		<em>Siberian federal university, Institute of fundamental biology and biotechnology</em>
		    	
		    	
		    		
		    			</br>"The prevalence of function over taxonomy for triplet composition of mitochondrial and chloroplast plant genes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We studied the relation between triplet composition of genes and taxonomy of the bearers in case of plant genes. There are two gene families that are common for mitochondria and chloroplast genomes of plants: atp genes family and nad family. In this study we compared mitochondrial and chloroplast atp genes of the same species. These genes encode subunits of ATP synthase. Totally, 170 (85 mitochondrial and 85 chloroplast) plant genomes were studied. Each gene sequence was transformed into a triplet frequency dictionary, where the reading frame shift was equal to t = 1. Then the points in 64-dimensional space of the triplet frequencies of the genes were clustered with ViDaExpert software. Three types of clustering have been analyzed: for mitochondria genes solely, for chloroplast genes solely, and for the merged set of the genes from both organelles' genomes. It was observed that clusters are formed on a functional basis. To be more precise, the genes encoded different subunits were split into separate clusters. Moreover, each cluster contained genes encoding only one subunit of ATP synthase. Thus, the prevalence of function over the taxonomy for atp genes family of organelles genomes of plants has been proven.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-CT09" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
	                <a id="Posters"><h1 class="mb-3" id="MS1">Sub-group poster presentations</h1></a>
									<div class="row subgroup-name">
										<div class="col-12">
											<div class="card">															
										  	<h5 class="card-header card-header-subgroup MFBM">MFBM Posters</h5>
											  <div class="card-body">
											    <p class="card-text">																
														<div>
															
													    	
																	<p><small class="card-body-subgroup">MFBM-1 (Session: PS02)</small>
															    	<a href="/MFBM/MFBM-PS02-MFBM-1.html">
																    	</br><strong>Paulina Wodarz</strong>
														    	 
														    		<em>University High School, Irvine</em>
														    	
														    	
														    		
														    			</br>"Analyzing Text Corpora to Determine the Emotional State of Humans"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			People's feelings are often reflected in the way they write. For this project, texts were used to characterize the emotional state of people both throughout the decades and in different parts of society in the present time. For temporal analysis, an online Corpus of Historical American English was used (400 million words, 1810-2000). For the present-day analysis, a collection of bloggers' posts from Kaggle (grouped by gender, age, and occupation) was put through a sentiment analysis tool. It was found that in the course of 200 years, energetic words decreased in frequency, and less energetic words increased. Negative and positive words decreased, and neutral words increased, indicating that there may have been a rise of apathy in society. Further, it was found that in present day's common usage, females, younger people, and those with a background in the arts exhibit more negative emotions than the other groups. These findings indicate that mathematical and computational analysis can be used to detect not only long-term societal trends, but also to study the emotional characteristics of different groups of people. In particular, methods of data science could be a valid tool to identify vulnerable populations that can be targeted for depression evaluation.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-2 (Session: PS02)</small>
															    	<a href="/MFBM/MFBM-PS02-MFBM-2.html">
																    	</br><strong>Samarth Kadaba</strong>
														    	 
														    		<em>University of California, Santa Barbara</em>
														    	
														    	
														    		
														    			</br>"Discovering Sequence-Activity Relationships using Machine Learning: Convolutional Neural Networks (CNNs) and Gaussian Process Regressions (GPRs)."</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			We show how recent machine learning methods can be utilized to learn representations for classes of proteins and other macromolecules that relate sequence information to predicted activity on specific substrates. In the synthetic evolution of enzymes, predicting activity is a crucial step towards finding functional unscreened variants. However, identifying sequence-activity relationships for organic polymers is complicated by latent features associated with the 3D structure of the folded molecule. Here we propose methods that use Convolutional Neural Networks (CNN's) to extract from 3D structural information, such as crystallographic data and Molecular Dynamics (MD) simulation data, relationships between sequence and activity. In particular, we develop CNN feature extractors for kernels within Gaussian Process Regressions (GPRs) to make predictions on sequence space. As a demonstration, we use data of amino acid rotamers to show how CNN's can predict amino acid torsion angles from a sequence of Chi angle conformation types. Applying these networks to develop kernel functions for GPRs we predict conformational state information and predicted activities. We perform studies also on toy models to show how using deep learning approximations of macromolecular structure can yield representations of sequence-activity relationships potentially useful for synthetic evolution.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-3 (Session: PS02)</small>
															    	<a href="/MFBM/MFBM-PS02-MFBM-3.html">
																    	</br><strong>Emilee Carson</strong>
														    	 
														    		<em>University of Waterloo</em>
														    	
														    	
														    		
														    			</br>"A machine learning approach for analyzing bistable systems in biology"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Bistable systems arise frequently in the modelling of biological systems, particularly in systems biology. A famous example is the Collins toggle switch, a gene regulatory network with two genes that repress the expression of each other. Typically, the qualitative behaviour of these systems is examined using traditional techniques such as phase portraits and stability analysis. These approaches rely on the accuracy of the proposed model equations. Recently, machine learning has been used increasingly in the analysis of models stemming from applications in physics, but these methods have not yet been used widely for biological models. We develop a machine learning approach to analyze the behaviour of bistable systems in biology, particularly in cases where there may be information missing in the model equations and demonstrate its effectiveness in the case of the Collins toggle switch.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-4 (Session: PS02)</small>
															    	<a href="/MFBM/MFBM-PS02-MFBM-4.html">
																    	</br><strong>Alex John Quijano</strong>
														    	 
														    		<em>University of California Merced</em>
														    	
														    	
														    		
														    			</br>"Evolving Contextual Semantics"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Similar to biological systems, natural languages are evolving systems with words as their measurable units. Words have certain functions within a body of text to convey ideas and thought. This poster presentation introduces the method of Latent Semantic Analysis (LSA) and the Skip-Gram with Negative Sampling (SGNS) approaches to extract contextual semantics within a body of text taken from a social media platform called Twitter. Contextual semantics refers to a semantic space that is expressed as a linear combination of words from a matrix subspace. Due to the natural volatility of some words and languages as a whole, the semantic space is evolving in time. The objective is to study the emergence of online social movements particularly the use of hashtags. We explore the evolving contextual semantics of the social movement hashtags “#blacklivesmatter” and “#metoo” as examples. From our results and observations, we hypothesize that these hashtags exhibit selective language transmission (i.e. horizontal transmission), the process of passing words and phrases between people that lead to changes in meanings due to selective cultural pressures.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-10 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-10.html">
																    	</br><strong>Yuki Takahashi</strong>
														    	 
														    		<em>Akita prefectural University</em>
														    	
														    	
														    		
														    			</br>"Chromatin dynamics by polymer models and Brownian dynamics with collisions"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The primary structure of chromosome is a complex of histone octamer wrapped by double stranded DNA, that is, nucleosome, and the linker DNA connecting them. The mechanism of the chromosome condensation and decondensation processes is to be fully understood, particularly in order to elucidate such dynamics in morphogenetic stages in human. So far, some tertiary structures of chromosome have been proposed, but recent experiments revealed that polymer-melt-like state was found without any specific tertiary structure. These suggest that the dynamics in a dense phase for a longer time span is to be theoretically investigated. However, in general, the Brownian dynamics method assumes constant external force, and is valid with a short time step (1ps ~ 1ns). It is insufficient to simulate chromatin dynamics ina morphogenetic stage in human. The purpose of this research is to investigate and develop an extension of the Brownian dynamics by implementing two types of collision algorithms. Our results confirmed the diffusive nature of nucleosomes in a HeLa cell.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-11 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-11.html">
																    	</br><strong>Eun Han Goo</strong>
														    	 
														    		<em>Korea Advanced Institute of Science and Technology(KAIST)</em>
														    	
														    	
														    		
														    			</br>"Development of a Mathematical Model for Predicting Accurate Hepatic Clearance of Drug"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Clearance (CL), the amounts of a drug metabolized by enzymes per unit time, is the major pharmacokinetic measurement used in the development and determination of the dosage of medications. In vivo hepatic CL of a drug has been predicted by extrapolating in vitro intrinsic CL whose estimation is based on the Michaelis-Menten (MM) equation, which is the result of the standard quasi-steady-state approximation (QSSA). However, in vivo drug CL predicted in that way becomes inaccurate if in vivo hepatic enzyme concentration is not sufficiently lower than the MM constant of the drug. Here, we develop an alternative approach based on the total QSSA, which accurately predicts in vivo CL of drugs regardless of their MM constant values.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-12 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-12.html">
																    	</br><strong>Peerdeman GY</strong>
														    	 
														    		<em>University of Applied Sciences Leiden</em>
														    	
														    	
														    		
														    			</br>"Standardizing the exchange of model states between cellular models using MultiCellDS"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Cell-based modeling has become a standard tool in biology for developing mechanistic hypotheses of tissue morphogenesis. A large number of model frameworks is available, each of which store represent and store cells as different types of mathematical objects, Storing and exchanging snapshots between different modeling frameworks is useful: it allows comparison of simulations, e.g., through measuring divergence of configurations, or usage of annotated experimental data as initial conditions. Recently, MultiCellDS was proposed as a standard for exchanging cellular data between modeling frameworks. Implementing MultiCellDS for modeling frameworks is challenging. Here we propose an extension to the MultiCellDS standard and introduced libCellShape, that acts as an interface to MultiCellDS  for modeling frameworks.  The library takes on much of the burden of storing and loading cellular snapshots to and from MultiCellDS files, and includes algorithms to convert lattice based snapshots to vertex based snapshots and vice versa. We demonstrate the use of libCellShape by exchanging snapshots between our Cellular Potts framework Tissue Simulation Toolkit and our vertex-based framework VirtualLeaf. We hope that libCellShape will allow different frameworks to exchange snapshots more easily and that more model types can be supported in the future.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-13 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-13.html">
																    	</br><strong>Yifei Li</strong>
														    	 
														    		<em>Queensland University of Technology</em>
														    	
														    	
														    		
														    			</br>"Dimensionality affects extinction of bistable populations"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			The question of whether biological populations survive or are eventually driven to extinction has long been examined using mathematical models. In this work we study population survival or extinction using a stochastic, discrete lattice-based random walk model where individuals undergo movement, birth and death events. The discrete model is defined on a two-dimensional hexagonal lattice with periodic boundary conditions. A key feature of the discrete model is that crowding effects are introduced by specifying two different crowding functions that govern how local agent density influences movement events and birth/death events. The continuum limit description of the discrete model is a nonlinear reaction-diffusion equation, and in this work we focus on crowding functions that lead to linear diffusion and a bistable reaction term that is often associated with the strong Allee effect. Using both the discrete and continuum modelling tools we explore the complicated relationship between the long-term survival or extinction of the population and the initial spatial arrangement of the population. Our results indicate that dimensionality of the initial population distributions affects extinction of bistable populations.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-5 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-5.html">
																    	</br><strong>Hyundong Kim</strong>
														    	 
														    		<em>Korea University</em>
														    	
														    	
														    		
														    			</br>"Numerical simulation of the pattern formation in reaction-diffusion equations on time-stepped moving curved surfaces"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			In this talk, we present an explicit time-dependent numerical scheme for the pattern formation in reaction-diffusion equations on time-stepped moving curved surfaces. The proposed method is consisting of discretization scheme of Laplace-Beltrami operator over triangulated surfaces. We conduct several numerical experiments to demonstrate a growing domain effect, and pattern formations by the proposed numerical method.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-6 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-6.html">
																    	</br><strong>Galina Kolesova</strong>
														    	 
														    		<em>InSysBio, Moscow, Russia</em>
														    	
														    	
														    		
														    			</br>"Application of different approaches to generate virtual patient populations for QSP model of Erythropoiesis"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Objectives: In the study we describe and compare four different techniques to generate virtual patient populations basing on experimentally measured statistics.Methods: QSP model of erythropoiesis was constructed to comprehensively describe cell dynamics from hematopoietic stem cell to circulating red cells. The model describes cell self-renewal, differentiation, proliferation, migration from bone marrow into circulation and cell death.Data describing time series of plasma reticulocyte count in response to single dose erythropoietin administered to 5 healthy subjects is used to find out final population of virtual patients (VP). Experimental data are given in the form of mean and standard deviation (SD). Four different approaches were applied to generate virtual patient populations (VPpop): (1) Monte-Carlo Markov Chain, (2) Model fitting to Monte-Carlo sample, (3) Population of clones, (4) Stochastically bounded selection. 39 parameters of the erythropoiesis model were chosen to be responsible for variability in observed clinical data.   Conclusions: The approaches proposed are capable for reproducing distribution characteristics of plasma reticulocyte count observed in clinical trials. The approach 4 is the most universal, as it allows to describe any number of patients from clinical trials and it can be applied in case of complex models with large number of variable parameters.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-7 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-7.html">
																    	</br><strong>Veronika Musatova</strong>
														    	 
														    		<em>INSYSBIO</em>
														    	
														    	
														    		
														    			</br>"Unified approach for in vitro data based parameters estimation"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Our aim was to develop a unified approach to parameters estimation from the in vitro data and its implementation in an upgrade of the Immune Response Template (IRT). IRT is QSP platform of the immune system and a tool for the development of QSP and mechanistic models related to immunity. The core of IRT is an ordinary differential equations-based model describing immune cells cycle processes, cytokines secretion and surface molecules interaction. Model parameters may be assessed from the in vitro experiments with typical design for cells survival, proliferation, migration, etc. We derived formulas for direct parameters calculation from the in vitro data for apoptosis, proliferation, migration, differentiation and cytokine synthesis.Typical in vitro experiments were modeled, processes were described with an approach from [1]. Derived formulas allow direct calculation of parameters from the in vitro data, resulting in a reduction of a model development time. Developed formulas were used in parameter identification for IRT version 3 and 3.1, allowing the calculation of nearly 950 parameters directly from the in vitro data.1. Mechanistic approach to describe multiple effects of regulatory molecules on cell dynamics process in immune response. Oleg Demin, Evgeny Metelkin, Galina Lebedeva, Sergey Smirnov, ACoP7, Bellevue, WA, US.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-8 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-8.html">
																    	</br><strong>Dmitry Shchelokov</strong>
														    	 
														    		<em>InSysBio, Moscow, Russia</em>
														    	
														    	
														    		
														    			</br>"Specific lysis modeling and parameter extraction from experimental data"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			Specific lysis is a complex multi-step process of target cell death mediated by cytotoxic granules containing enzymes (perforins, granzymes) that are released by effector cells after engagement with a target. It is also known that the probability of target lysis should depend on the number of surrounding effector cells (i.e., effector to target ratio). Since the cell-cell interactions during target killing are discrete processes and an exact number of certain cell aggregates is unknown, a detailed description of cell lysis is complicated and contains plenty of parameters. This work aims to provide simple biologically relevant equations describing cell lysis dependence on both time and effector-target ratio. The derived equations were validated against different types of experimental data on T and NK cell cytotoxic activity and exhibited saturation behavior at increasing concentration of both effector and target cells according to observations. Thereby, our approach allows calculating the parameters of cell lysis (rate constant and EC50 values) from experimental data without a curve fitting.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	
																	<p><small class="card-body-subgroup">MFBM-9 (Session: PS05)</small>
															    	<a href="/MFBM/MFBM-PS05-MFBM-9.html">
																    	</br><strong>Yun Min Song</strong>
														    	 
														    		<em>Korea Advanced Institute of Science and Technology</em>
														    	
														    	
														    		
														    			</br>"Universally valid reduction of multiscale stochastic biochemical systems with simple non-elementary propensities"</p>
														    		
														    	
														    	
														    	
															    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
															    		<div class="hide-abstracts">
															    			As experimentally characterizing all underlying kinetics of reactions in biochemical systems is almost impossible, their combined effects have frequently been described by simplified non-elementary reaction functions (e.g., Hill and Morrison functions) for over a century. Recently, deterministically driven non-elementary reaction functions have been heuristically used for stochastic simulations with the Gillespie algorithm. While this approach has been one of the most popular methods for efficient stochastic simulations, its validity condition has remained poorly understood. In this presentation, we derive a complete condition under which this approach can accurately capture the stochastic dynamics of reversible binding, the critical reaction to describe nearly all biochemical systems such as gene regulation and enzyme-catalysis. Furthermore, we develop alternative simplified reaction functions for stochastic reversible binding. This provides a universally valid framework for the simplification of stochastic biochemical systems with rapid reversible bindings.
															    		</div>
															    	</article>
														    	
														    	</a>
														    
														  
													    	    	
											  		</div>
													</p>
											  </div>
											</div>
										</div>
									</div>
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
					Click to show abstracts
									</span>		            
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
