<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Posters-05 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Posters-05 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Posters-05" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/PS05/" />
<meta property="og:url" content="http://localhost:4000/PS05/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/PS05/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Posters-05","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Posters-05">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Posters-05" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Posters-05" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Posters-05</h1>
        <h4>Wednesday, June 16 at 11:30pm (PDT)</br>Thursday, June 17 at 07:30am (BST)</br>Thursday, June 17 03:30pm (KST)</h4>
      </div>
    </div>
	</div>
	
	
	
   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
		                <h1 class="mb-3" id="PS05">Posters-05</h1>
		                
		                
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-IMMU">IMMU</h5> </strong>IMMU-6 (Session: PS05)</small>
										    	<a href="/IMMU/IMMU-PS05-IMMU-6.html">
											    	</br><strong>Neha Singaravelan</strong>
									    	 
									    		<em>Texas Christian University</em>
									    	
									    	
									    		
									    			</br>"Viral coinfection interaction through interferon"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Coinfection affects up to 60% of patients hospitalized influenza-like illnesses, however, the role of the innate immune response in coinfections is not understood. Interferons, part of the innate immune response, are a type of chemical released by infected cells that can help establish an antiviral state in cells by increasing resistance to infection and reducing production of viruses. Although the increased resistance to infection can help suppress both viruses, the reduction in the production of one virus may aid in increasing the growth of another virus during coinfection due to less competition. We will use a mathematical model to examine the interaction via interferons between respiratory syncytial virus (RSV) and influenza A virus (IAV) during coinfections. This model will measure viral titer, duration of the viral infection, and interferon production allowing us to understand how interferon production of one virus helps or hinders the secondary virus.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-59 (Session: PS05)</small>
										    	<a href="/MEPI/MEPI-PS05-MEPI-59.html">
											    	</br><strong>Khagendra Adhikari</strong>
									    	 
									    		<em>Tribhuvan University</em>
									    	
									    	
									    		
									    			</br>" Modelling COVID-19 in Nepal: Effectiveness of Control Measures"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			While most of the countries around the globe are combating with the pandemic of COVID-19, the level of its impact is quite variable among different countries. In particular, the data from Nepal, a developing country having open border provision with highly COVID-19 affected country India, have shown a biphasic pattern of epidemic, a controlled phase (until July 21, 2020) followed by an outgrown phase (after July 21, 2020). In this presentation, I will describe the biphasic epidemic pattern of Nepal via a mathematical model and analyze the effectiveness of the control strategies implemented in Nepal.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-60 (Session: PS05)</small>
										    	<a href="/MEPI/MEPI-PS05-MEPI-60.html">
											    	</br><strong>Ramesh Gautam</strong>
									    	 
									    		<em>Tribhuvan University</em>
									    	
									    	
									    		
									    			</br>"Modeling Imported malaria in Nepal with control strategies"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The cross-border mobility of seasonal migrant workers between Nepal and India is a major challenge for the malaria elimination program of Nepal. Having the open border provision with highly endemic country of malaria India, most of the recorded malaria cases of Nepal are imported. Here, we proposed a malaria model including control strategies to protect the migrant workers from mosquito biting during their stay in India. Moreover, we will analyze the backward bifurcation and level of control strategies for the elimination of malaria in Nepal by 2026
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-61 (Session: PS05)</small>
										    	<a href="/MEPI/MEPI-PS05-MEPI-61.html">
											    	</br><strong>Akhil Kumar Srivastav</strong>
									    	 
									    		<em>Vellore Institute of Technology</em>
									    	
									    	
									    		
									    			</br>"Modeling and optimal control analysis of COVID-19: Case studies from Italy and Spain"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Coronavirus disease 2019 (COVID-19) is a viral disease which is declared asa pandemic by WHO. This disease is posing a global threat, and almost everycountry in the world is now affected by this disease. Currently, there is no vaccine for this disease, and because of this, containing COVID-19 is not an easytask. It is noticed that elderly people got severely affected by this disease spe-cially in Europe. In the present paper, we propose and analyze a mathematicalmodel for COVID-19 virus transmission by dividing whole population in oldand young groups. We find disease-free equilibrium and the basic reproductionnumber (R0). We estimate the parameter corresponding to rate of transmissionand rate of detection of COVID-19 using real data from Italy and Spain by leastsquare method. We also perform sensitivity analysis to identify the key parameters which influence the basic reproduction number and hence regulate thetransmission dynamics of COVID-19. Finally, we extend our proposed model tooptimal control problem to explore the best cost-effective and time-dependentcontrol strategies that can reduce the number of infectives in a specified intervalof time.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-62 (Session: PS05)</small>
										    	<a href="/MEPI/MEPI-PS05-MEPI-62.html">
											    	</br><strong>Laura Marcela Guzman Rincon</strong>
									    	 
									    		<em>University of Warwick</em>
									    	
									    	
									    		
									    			</br>"Estimation of the Growth-Rate of the SARS-CoV-2 epidemic at a subpopulation level"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The rapid identification of potential SARS-CoV-2 outbreaks is key in the design of optimal intervention strategies and the control of its propagation. We propose a fast and accurate approach to estimate the growth rate of positive results of SARS-CoV-2 tests in a given subset of the population. This estimation has been used in the collected PCR tests in England, for different local authorities and age groups. We describe the mathematical structure of the model and how it also provides an estimation of the weekday effect in data collection.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-63 (Session: PS05)</small>
										    	<a href="/MEPI/MEPI-PS05-MEPI-63.html">
											    	</br><strong>Amit Sharma</strong>
									    	 
									    		<em>JC DAV COLLEGE DASUYA</em>
									    	
									    	
									    		
									    			</br>"Modelling the COVID-19 epidemic using delayed-impulsive differential equations"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Presently the large focus of the world community is on controlling the spread of COVID-19 infection. As of 30 March 2021, the COVID -19 infection has already accounted 121 million people and 2.9 million deaths worldwide. Many vaccines are now been approved for the prevention of COVID-19. Particularly in India as per government data available on 30 March 2021 more than 60 million people have been vaccinated. Forecasting is important for the alleviation of potentially fatal impacts of infectious diseases. In a pandemic, pronouncements are given in short supply of data in uncertain conditions. Also, this is not possible to know when the next pandemic will occur; however, mathematical modeling has the potential to increase the efficacy when a pandemic occurs. We analyze the Susceptible-Exposed-Infected-Vaccinated-Recovered (SEIVR) epidemic mathematical model of COVID-19. Our model includes two important aspects of COVID-19 infection: delayed start and effect of impulsive vaccination. The model has been analyzed theoretically and numerically both. We found that the COVID-19 infection-free periodic solution is globally asymptotically stable. Numerical simulations further show that impulsive vaccination with the vaccine of high efficacy will have the potential to reduce the spread of COVID-19 infection.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-10 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-10.html">
											    	</br><strong>Yuki Takahashi</strong>
									    	 
									    		<em>Akita prefectural University</em>
									    	
									    	
									    		
									    			</br>"Chromatin dynamics by polymer models and Brownian dynamics with collisions"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The primary structure of chromosome is a complex of histone octamer wrapped by double stranded DNA, that is, nucleosome, and the linker DNA connecting them. The mechanism of the chromosome condensation and decondensation processes is to be fully understood, particularly in order to elucidate such dynamics in morphogenetic stages in human. So far, some tertiary structures of chromosome have been proposed, but recent experiments revealed that polymer-melt-like state was found without any specific tertiary structure. These suggest that the dynamics in a dense phase for a longer time span is to be theoretically investigated. However, in general, the Brownian dynamics method assumes constant external force, and is valid with a short time step (1ps ~ 1ns). It is insufficient to simulate chromatin dynamics ina morphogenetic stage in human. The purpose of this research is to investigate and develop an extension of the Brownian dynamics by implementing two types of collision algorithms. Our results confirmed the diffusive nature of nucleosomes in a HeLa cell.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-11 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-11.html">
											    	</br><strong>Eun Han Goo</strong>
									    	 
									    		<em>Korea Advanced Institute of Science and Technology(KAIST)</em>
									    	
									    	
									    		
									    			</br>"Development of a Mathematical Model for Predicting Accurate Hepatic Clearance of Drug"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Clearance (CL), the amounts of a drug metabolized by enzymes per unit time, is the major pharmacokinetic measurement used in the development and determination of the dosage of medications. In vivo hepatic CL of a drug has been predicted by extrapolating in vitro intrinsic CL whose estimation is based on the Michaelis-Menten (MM) equation, which is the result of the standard quasi-steady-state approximation (QSSA). However, in vivo drug CL predicted in that way becomes inaccurate if in vivo hepatic enzyme concentration is not sufficiently lower than the MM constant of the drug. Here, we develop an alternative approach based on the total QSSA, which accurately predicts in vivo CL of drugs regardless of their MM constant values.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-12 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-12.html">
											    	</br><strong>Peerdeman GY</strong>
									    	 
									    		<em>University of Applied Sciences Leiden</em>
									    	
									    	
									    		
									    			</br>"Standardizing the exchange of model states between cellular models using MultiCellDS"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Cell-based modeling has become a standard tool in biology for developing mechanistic hypotheses of tissue morphogenesis. A large number of model frameworks is available, each of which store represent and store cells as different types of mathematical objects, Storing and exchanging snapshots between different modeling frameworks is useful: it allows comparison of simulations, e.g., through measuring divergence of configurations, or usage of annotated experimental data as initial conditions. Recently, MultiCellDS was proposed as a standard for exchanging cellular data between modeling frameworks. Implementing MultiCellDS for modeling frameworks is challenging. Here we propose an extension to the MultiCellDS standard and introduced libCellShape, that acts as an interface to MultiCellDS  for modeling frameworks.  The library takes on much of the burden of storing and loading cellular snapshots to and from MultiCellDS files, and includes algorithms to convert lattice based snapshots to vertex based snapshots and vice versa. We demonstrate the use of libCellShape by exchanging snapshots between our Cellular Potts framework Tissue Simulation Toolkit and our vertex-based framework VirtualLeaf. We hope that libCellShape will allow different frameworks to exchange snapshots more easily and that more model types can be supported in the future.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-13 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-13.html">
											    	</br><strong>Yifei Li</strong>
									    	 
									    		<em>Queensland University of Technology</em>
									    	
									    	
									    		
									    			</br>"Dimensionality affects extinction of bistable populations"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The question of whether biological populations survive or are eventually driven to extinction has long been examined using mathematical models. In this work we study population survival or extinction using a stochastic, discrete lattice-based random walk model where individuals undergo movement, birth and death events. The discrete model is defined on a two-dimensional hexagonal lattice with periodic boundary conditions. A key feature of the discrete model is that crowding effects are introduced by specifying two different crowding functions that govern how local agent density influences movement events and birth/death events. The continuum limit description of the discrete model is a nonlinear reaction-diffusion equation, and in this work we focus on crowding functions that lead to linear diffusion and a bistable reaction term that is often associated with the strong Allee effect. Using both the discrete and continuum modelling tools we explore the complicated relationship between the long-term survival or extinction of the population and the initial spatial arrangement of the population. Our results indicate that dimensionality of the initial population distributions affects extinction of bistable populations.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-5 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-5.html">
											    	</br><strong>Hyundong Kim</strong>
									    	 
									    		<em>Korea University</em>
									    	
									    	
									    		
									    			</br>"Numerical simulation of the pattern formation in reaction-diffusion equations on time-stepped moving curved surfaces"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			In this talk, we present an explicit time-dependent numerical scheme for the pattern formation in reaction-diffusion equations on time-stepped moving curved surfaces. The proposed method is consisting of discretization scheme of Laplace-Beltrami operator over triangulated surfaces. We conduct several numerical experiments to demonstrate a growing domain effect, and pattern formations by the proposed numerical method.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-6 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-6.html">
											    	</br><strong>Galina Kolesova</strong>
									    	 
									    		<em>InSysBio, Moscow, Russia</em>
									    	
									    	
									    		
									    			</br>"Application of different approaches to generate virtual patient populations for QSP model of Erythropoiesis"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Objectives: In the study we describe and compare four different techniques to generate virtual patient populations basing on experimentally measured statistics.Methods: QSP model of erythropoiesis was constructed to comprehensively describe cell dynamics from hematopoietic stem cell to circulating red cells. The model describes cell self-renewal, differentiation, proliferation, migration from bone marrow into circulation and cell death.Data describing time series of plasma reticulocyte count in response to single dose erythropoietin administered to 5 healthy subjects is used to find out final population of virtual patients (VP). Experimental data are given in the form of mean and standard deviation (SD). Four different approaches were applied to generate virtual patient populations (VPpop): (1) Monte-Carlo Markov Chain, (2) Model fitting to Monte-Carlo sample, (3) Population of clones, (4) Stochastically bounded selection. 39 parameters of the erythropoiesis model were chosen to be responsible for variability in observed clinical data.   Conclusions: The approaches proposed are capable for reproducing distribution characteristics of plasma reticulocyte count observed in clinical trials. The approach 4 is the most universal, as it allows to describe any number of patients from clinical trials and it can be applied in case of complex models with large number of variable parameters.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-7 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-7.html">
											    	</br><strong>Veronika Musatova</strong>
									    	 
									    		<em>INSYSBIO</em>
									    	
									    	
									    		
									    			</br>"Unified approach for in vitro data based parameters estimation"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Our aim was to develop a unified approach to parameters estimation from the in vitro data and its implementation in an upgrade of the Immune Response Template (IRT). IRT is QSP platform of the immune system and a tool for the development of QSP and mechanistic models related to immunity. The core of IRT is an ordinary differential equations-based model describing immune cells cycle processes, cytokines secretion and surface molecules interaction. Model parameters may be assessed from the in vitro experiments with typical design for cells survival, proliferation, migration, etc. We derived formulas for direct parameters calculation from the in vitro data for apoptosis, proliferation, migration, differentiation and cytokine synthesis.Typical in vitro experiments were modeled, processes were described with an approach from [1]. Derived formulas allow direct calculation of parameters from the in vitro data, resulting in a reduction of a model development time. Developed formulas were used in parameter identification for IRT version 3 and 3.1, allowing the calculation of nearly 950 parameters directly from the in vitro data.1. Mechanistic approach to describe multiple effects of regulatory molecules on cell dynamics process in immune response. Oleg Demin, Evgeny Metelkin, Galina Lebedeva, Sergey Smirnov, ACoP7, Bellevue, WA, US.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-8 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-8.html">
											    	</br><strong>Dmitry Shchelokov</strong>
									    	 
									    		<em>InSysBio, Moscow, Russia</em>
									    	
									    	
									    		
									    			</br>"Specific lysis modeling and parameter extraction from experimental data"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Specific lysis is a complex multi-step process of target cell death mediated by cytotoxic granules containing enzymes (perforins, granzymes) that are released by effector cells after engagement with a target. It is also known that the probability of target lysis should depend on the number of surrounding effector cells (i.e., effector to target ratio). Since the cell-cell interactions during target killing are discrete processes and an exact number of certain cell aggregates is unknown, a detailed description of cell lysis is complicated and contains plenty of parameters. This work aims to provide simple biologically relevant equations describing cell lysis dependence on both time and effector-target ratio. The derived equations were validated against different types of experimental data on T and NK cell cytotoxic activity and exhibited saturation behavior at increasing concentration of both effector and target cells according to observations. Thereby, our approach allows calculating the parameters of cell lysis (rate constant and EC50 values) from experimental data without a curve fitting.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MFBM">MFBM</h5> </strong>MFBM-9 (Session: PS05)</small>
										    	<a href="/MFBM/MFBM-PS05-MFBM-9.html">
											    	</br><strong>Yun Min Song</strong>
									    	 
									    		<em>Korea Advanced Institute of Science and Technology</em>
									    	
									    	
									    		
									    			</br>"Universally valid reduction of multiscale stochastic biochemical systems with simple non-elementary propensities"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As experimentally characterizing all underlying kinetics of reactions in biochemical systems is almost impossible, their combined effects have frequently been described by simplified non-elementary reaction functions (e.g., Hill and Morrison functions) for over a century. Recently, deterministically driven non-elementary reaction functions have been heuristically used for stochastic simulations with the Gillespie algorithm. While this approach has been one of the most popular methods for efficient stochastic simulations, its validity condition has remained poorly understood. In this presentation, we derive a complete condition under which this approach can accurately capture the stochastic dynamics of reversible binding, the critical reaction to describe nearly all biochemical systems such as gene regulation and enzyme-catalysis. Furthermore, we develop alternative simplified reaction functions for stochastic reversible binding. This provides a universally valid framework for the simplification of stochastic biochemical systems with rapid reversible bindings.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-1 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-1.html">
											    	</br><strong>Emmanuel Adabor</strong>
									    	 
									    		<em>Ghana Institute of Management and Public Administration</em>
									    	
									    	
									    		
									    			</br>"On the analysis of antigenic relatedness of influenza A (H3N2) viruses"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			An accurate assessment of antigenic relatedness between influenza viruses is important for vaccine strain recommendations and influenza surveillance. Due to the mechanisms that result in frequent changes in the antigenicities of strains, it is desirable to obtain an antigenic relatedness measure that account for specific changes in strains that are of epidemiological importance in influenza. A computational model was developed using distinguishing features of antigenic variants to analyze antigenic relatedness among influenza strains. The features comprised of cluster information, amino acid sequences located in known antigenic and receptor-binding sites of influenza A (H3N2). In order to assess validity of parameters, accuracy and relevance of model to vaccine effectiveness, the model was applied to influenza A (H3N2) viruses due to their abundant genetic data and epidemiological relevance to influenza surveillance. It was found that all model parameters were determinants of antigenic relatedness between strains and that the model accurately predicts the antigenic relatedness between influenza A (H3N2) viruses. The methods presented in this study will potentially complement the global efforts in influenza surveillance.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-10 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-10.html">
											    	</br><strong>Robert West</strong>
									    	 
									    		<em>Department of Physics at Bar-Ilan University</em>
									    	
									    	
									    		
									    			</br>"Evolution of a Fluctuating Population in a Switching Environment: Random versus Periodic"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Environmental changes greatly influence the evolution of populations. In this talk, we discuss the dynamics of a population of two strains, one growing slightly faster than the other, competing for resources in a time-varying binary environment modeled by a carrying capacity that switches either randomly or periodically between states of resources abundance and scarcity [1,2]. The population dynamics is characterized by demographic noise (birth and death events) coupled to the fluctuating population size [2,3]. By combining analytical and simulation methods, we elucidate the similarities and differences of evolving subject to  stochastic and periodic switching. The population size distribution is generally found to be broader under intermediate and fast random switching than under periodic variations. This results in markedly different asymptotic behaviors of the fixation probability under random and periodic switching environments [1]. We also determine the conditions under which the fixation probability of the slow strain is maximal [1].[1] A. Taitelbaum, R. West, M. Assaf, and M. Mobilia, Physical Review Letters 125, 048105:1-6 (2020).[2] K. Wienand, E. Frey, and M. Mobilia, Physical Review Letters 119, 158301:1-6 (2017) and J. Royal Society Interface 15, 20180343:1-12 (2018).[3] R. West and M. Mobilia, Journal of Theoretical Biology 491, 110135:1-14 (2020).
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-2 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-2.html">
											    	</br><strong>Matthew Edgington</strong>
									    	 
									    		<em>The Pirbright Institute</em>
									    	
									    	
									    		
									    			</br>"Split drive killer-rescue: A novel threshold-dependent gene drive"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			A wide range of gene drive mechanisms are predicted to increase in frequency within a population even when deleterious to individuals carrying them. This should also allow associated desirable genetic material to increase in frequency. Gene drives have garnered much attention for their potential use against a range of globally important problems including disease vectors, crop pests and invasive species. Here we propose a novel gene drive mechanism that could be engineered using a combination of toxin-antidote and CRISPR components, each of which are already being developed for other gene drive designs. Population genetics mathematical models are developed here and used to demonstrate the threshold-dependent nature of the proposed system alongside its robustness to a wide range of performance parameters, each of which are of practical significance given that real-world components are inevitably imperfect. We show that although a mechanism known to cause resistance may cause the system to break down, under certain conditions, it should persist over time scales relevant for genetic control programs. This work proposes a promising new class of gene drive (with several highly desirable characteristics) that may be engineered by combining components already widely in development.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-3 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-3.html">
											    	</br><strong>Lucy Lansch-Justen</strong>
									    	 
									    		<em>The University of Edinburgh</em>
									    	
									    	
									    		
									    			</br>"Quantifying Stress-induced Mutagenesis"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Exposure to low concentrations of antimicrobials selects for resistance mutations and can induce phenotypic stress responses in microbes. Some of these responses increase the mutation rate, called stress-induced mutagenesis (SIM). But because stress responses additionally influence the whole population dynamics it is unclear whether SIM actually results in more or fewer resistant mutants. Moreover, SIM affects mutation rate estimates via fluctuation assays (a standard lab approach for measuring microbial mutation rates) because underlying modelling assumptions are not met. We describe an appropriate model of a microbial population which is exposed to stress and expresses a stress response and propose a new method for inferring the mutation rate in this case. Using the bacterial SOS response as an example we demonstrate that our derived mutant count distribution fits simulated data. In contrast, current methods are able to estimate the mean mutation rate in the population but not distinct mutation rates of subpopulations with low/high stress response levels.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-4 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-4.html">
											    	</br><strong>Pierre Lafont</strong>
									    	 
									    		<em>University of Edinburgh</em>
									    	
									    	
									    		
									    			</br>"Capturing Bacterial Ecology in models of antibiotic treatments"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Understanding how bacteria react to antibiotic challenge is key in optimising treatments. Bacteria grow in an ever-changing environment, where growth is limited by competition for space and/or resources. But bacteria can also cross-protect or help each other, for instance by absorbing or degrading antibiotics. When faced with treatment, a denser population may thus be able to tolerate a higher dose. Upon lysis, bacteria cells can also release nutrients that will be recycled for others. These multiple potentially counteracting factors highlight the need for mathematical models to understand the effects of these ecological interactions. From simple to complex formulations, even in ODE systems, there are many modelling choices one can make depending on the processes of interest. Here we aim at a clear overview of the different modelling approaches available and what they mean biologically. We recognise a lack of extended mathematical analysis in the literature and aim to develop a more thorough understanding of model behaviours through equilibrium and stability analysis. Ultimately, we aim to understand how ecological forces of both competition and cooperation affect bacterial population response to antibiotics and probability of resistance emergence.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-5 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-5.html">
											    	</br><strong>Tahani Alkarkhi</strong>
									    	 
									    		<em>University of Essex</em>
									    	
									    	
									    		
									    			</br>"Population Dynamics and Pattern Formation in a Plankton Model"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We study a spatio–temporal prey–predator model of plankton. This model has spatial interaction terms, which has the DeAngelis-Beddington functional response, to describe the grazing pressure of  microzooplankton (M) on phytoplankton (P) is controlled through external info–chemical (C) mediated predation by copepods (Z). The Beddington DeAngelis functional response plays a critical role in modeling plankton. It is an advance on the prey-dependent Holling's type II functional response. It can be used to explain the predators' per capita feeding rates on prey. This functional response can also be used to provide better descriptions of predator prey abundances and how these affect predator feeding, discussed that in their predator prey system, Beddington DeAngelis was used to describe mutual interference by predators within the ecosystem. In relation to this, the concept was used to highlight the effect of changes in prey density on the predator density attached per unit time.The Beddington DeAngelis functional response can be used to perform a detailed mathematical analysis of the intra-specific competition among predators. We undertake a stability analysis of the two species model and compare the system dynamics. In relation to this, the critical conditions for Kinesis are derived; these are necessary and sufficient.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-6 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-6.html">
											    	</br><strong>Anni S. Halkola</strong>
									    	 
									    		<em>Department of Mathematics and Statistics, University of Turku, Finland</em>
									    	
									    	
									    		
									    			</br>"Strategy dynamics in a metapopulation model of cancer cells"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Tumors consist of cells with abnormal phenotypes. These cells might be or become cancerous, which can lead to increased cell growth and even metastases. In this work, we have considered cancer as a metapopulation, in which habitat patches correspond to possible sites for a cluster of cancer cells. Cancer cells may emigrate into dispersal pool ( e.g. circulation system) and spread to new areas (i.e. metastatic disease). In the patches, cells divide and new mutations may arise, possibly leading into an invasion if the mutation is favorable. We consider various relevant strategies (phenotypes), such as the emigration rate and their contribution to angiogenesis, which is an important part of early stages of tumor development. We use the metapopulation fitness of new mutations to investigate how these strategies evolve in cancer through natural selection and disease progression. We further add treatment effects and investigate how different therapy regimens affect the evolution of the strategies. These aspects are relevant, for example, when examining the process of a benign tumor becoming cancerous, and how to best treat the early stages of cancer development.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-7 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-7.html">
											    	</br><strong>Kyohei Suzuki</strong>
									    	 
									    		<em>Akita Prefectural University</em>
									    	
									    	
									    		
									    			</br>"Collective behavior and ambient flow in barnacle cypris larvae"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Barnacles are small crustaceans, having two types of larval periods. While both of them swim, cypris larva is specialized in searching for and attaching to a surface without feeding. They tend to live in groups. It is known that the grouping can be induced by the settlement-inducing protein complex (SIPC). However, the grouping may be induced by various other factors such as phototaxis, water flow, substrate state, and communication between individuals. Few studies have focused on the detailed behavior of cypris larva, and none has on its collective nature. The phenomenon of collective behavior can be confirmed in various organisms. It is natural to expect some collective behavior of cyprids while swimming, since they live in groups, but no definitive evidence has been found. In this work, we visualized the flow around cypris larva during swimming, quantified the state of collective behavior, and calculated various statistics such as the correlation coefficient, in order to elucidate the communication between allogeneic individuals. We found the surrounding viscous flow and the small yet nontrivial correlation between them.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-8 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-8.html">
											    	</br><strong>Román Zapién-Campos</strong>
									    	 
									    		<em>Max Planck Institute for Evolutionary Biology</em>
									    	
									    	
									    		
									    			</br>"The effect of fitness differences in death-birth models with immigration"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Mathematical models have been instrumental in understanding the dynamics of ecological systems. Notable examples are models where the events of death, birth, and migration of individuals within a community only depend on their abundance. In other words, rates are equal regardless of the specific population.The proven utility of such models, used from gut microbiomes to forests, lies in their capacity to contrast experimental data to a 'neutral' prediction.  Surprisingly, such predictions often agree with experimental data, indicating that population-specific rates might be absent or at most irrelevant.But what if, instead, rates are assumed to be population-specific in these models? What patterns emerge? How resilient are the neutral community patterns? Our work addresses these questions incrementally, going from simple to many-populations communities. We focus on changes in various community composition indicators, specifically, on the occurrence-abundance pattern and how to identify 'non-neutrality' in data.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-POPD">POPD</h5> </strong>POPD-9 (Session: PS05)</small>
										    	<a href="/POPD/POPD-PS05-POPD-9.html">
											    	</br><strong>Alan Scaramangas</strong>
									    	 
									    		<em>City, University of Lodnon</em>
									    	
									    	
									    		
									    			</br>"Evolutionarily stable aposematic signalling in prey-predator systems where the prey population consists of one species."</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Aposematism is the signalling of a defence for the deterrence of predators. Our research focuses on aposematic organisms that exhibit chemical defences, which are usually signalled by bright skin pigmentation; although our treatment is likely transferable to other forms of secondary defence. This setup is a natural one to consider and opens up the possibility for robust mathematical modelling: the strength of aposematic traits (signalling and defence) can be unambiguously realised using variables that are continuously quantifiable, independent from one another and which together define a two-dimensional strategy space. We develop a mathematical model and explore the joint co-evolution of aposematic traits within the context of evolutionary stability. Even though empirical and model-based studies are conflicting regarding how aposematic traits are related to one another in nature, most allude to a positive correlation. We suggest that both positively and negatively correlated combinations of traits can achieve evolutionarily stable outcomes and further, that for a given level of signal strength there can be more than one optimal level of defence. Our findings are novel and relevant to a sizeable body of physical evidence, much of which could, until presently, not be addressed in terms of a single, well-understood mechanism.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Posters-05
										</br>
										Wednesday, June 16 at 11:30pm (PDT)</br>Thursday, June 17 at 07:30am (BST)</br>Thursday, June 17 03:30pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
