<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Minisymposia-01 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Minisymposia-01 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Minisymposia-01" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2020." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2020." />
<link rel="canonical" href="http://localhost:4000/MS01/" />
<meta property="og:url" content="http://localhost:4000/MS01/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2020.","@type":"WebPage","url":"http://localhost:4000/MS01/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Minisymposia-01","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Minisymposia-01">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Minisymposia-01" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Minisymposia-01" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Minisymposia-01</h1>
        <h4>Monday, June 14 at 09:30am (PDT)</br>Monday, June 14 at 05:30pm (BST)</br>Tuesday, June 15 01:30am (KST)</h4>
      </div>
    </div>
	</div>
	
	

   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
			            
			            							
									  
									 	
			            
		                <h1 class="mb-3" id="MS01">Minisymposia-01</h1>
		                
		                
		                
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/CBBS-MS01.html">
		
  	<h5 class="card-header card-header-subgroup CBBS"><small>MS01-CBBS: </small></br>Mathematical modeling of emergent phenomena in cell colonies</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Shawn D. Ryan (Cleveland State University, United States), Mykhailo Potomkin (UC Riverside, United States), Jia Gou (UC Riverside, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-CBBS</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/CBBS-MS01.html#author1"><strong class="card-body-subgroup">Gil Ariel</strong></a>
		    	 
		    		<em> (Bar Ilan University, Israel)</em>
		    	
		    	
		    		
		    			</br>"A Phase Diagram for Bacterial Swarming"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Bacterial swarming is a mode of motion in which dense collectives of rod-shaped flagellated cells migrate rapidly on surfaces. The transition into swarming involves several cellular processes, including changes in cell aspect ratio, suggesting that bacteria manipulate these properties in order to promote physical conditions that are favorable for swarming. New results with monolayer swarms of Bacillus subtilis with different aspect ratios were analyzed at different cell-densities. A comprehensive analysis of the individual and collective dynamics of bacteria in a swarm brings forth a phase diagram, showing sharp transitions between phases corresponding to qualitatively different swarm statistics. From a biological perspective, we find that under standard conditions bacteria inhabit a region of phase space in which the swarm dynamics is highly robust and insensitive to fluctuations. In this regime, bacteria do not form very large clusters and lack global orientational order, properties which may reduce the colony's ability to expand rapidly in the absence of external directional cues.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CBBS-MS01.html#author2"><strong class="card-body-subgroup">Fernando Peruani</strong></a>
		    	 
		    		<em> (CY Cergy Paris University, France)</em>
		    	
		    	
		    		
		    			</br>"A mathematical approach to bacterial infections: models for bacterial exploration, aggregation, and infection"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Combining mathematical models and bacterial experiments, we have learned how pathogenic bacteria explore surfaces, form early aggregates, and infect cells. I will provide a brief summary of main challenges behind these bacterial phenomena. References: Perez Ipina et al., Nature Physics 15, 610-615 (2019); Otte et al., Nature Communications 12, 1-9 (2021)
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CBBS-MS01.html#author3"><strong class="card-body-subgroup">Silke Henkes</strong></a>
		    	 
		    		<em> (University of Bristol, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Flow, fluctuate, freeze: Cell sheets as soft active matter"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CBBS-MS01.html#author4"><strong class="card-body-subgroup">Kevin Painter</strong></a>
		    	 
		    		<em> (Politecnico di Torino, Italy)</em>
		    	
		    	
		    		
		    			</br>"Sticking together by going against the flow"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Forming colonies, swarms, schools, flocks, herds, etc is a classical example of self organization, with the benefits of forming a high density group ranging from efficient migration to higher  fecundity. Often, groups form through a mechanism of chemical signaling between population members, an evolutionary ancient communication used by both microscopic and macroscopic species. Populations in fluid environments, though, must contend with complex and turbulent flows, potentially detrimental (e.g. splitting up groups) or beneficial (e.g. coalescing individuals) to the formation and maintenance of a group. As a counter to flow, rheotaxis describes a behavior in which individuals orient their body axis with respect to the current and is observed in both unicellular and multicellular organisms. Here we investigate the extent to which rheotaxis and flow impact on chemically-mediated aggregation, revealing these can impact both negatively and positively according to the population state and flow conditions. A hypothesized density-dependent rheotaxis appears capable of optimizing group formation and maintenance, exploiting the positive benefits from each of flow and rheotaxis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CBBS-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/CDEV-MS01.html">
		
  	<h5 class="card-header card-header-subgroup CDEV"><small>MS01-CDEV: </small></br>Data-driven modeling across scales - from cytoskeleton to bacterial swarms to multicellular motility to angiogenesis</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Alex Mogilner (NYU, United States), Angelika Manhart (UCL, UK)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/CDEV-MS01.html#author1"><strong class="card-body-subgroup">Angelika Manhart</strong></a>
		    	 
		    		<em> (UCL, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Explaining the dynamic steady state of actin networks"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Many motile cells use dense, branched actin networks for movement. This requires “macroscopic treadmilling”, where assembly at the front balances disassembly at the rear. We combine the use of a biomimetic motility system with data-driven mathematical modeling to investigate how cofilin, a known disassembly agent, creates dynamic networks of fixed lengths. To capture the observed macroscopic fragmentation of the network, we combine PDE-based modeling of the cofilin binding dynamics with a discrete network disassembly model. This allows to derive a simple formula predicting the equilibrium network length across various control parameters.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CDEV-MS01.html#author2"><strong class="card-body-subgroup">Hannah Jeckel</strong></a>
		    	 
		    		<em> (University of Marburg, Germany)</em>
		    	
		    	
		    		
		    			</br>"Learning the space-time phase diagram of bacterial swarm expansion"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Coordinated dynamics of individual components in active matter are an essential aspect of life on all scales. Establishing a comprehensive, causal connection between intracellular, intercellular, and macroscopic behaviors has remained a major challenge due to limitations in data acquisition and analysis techniques suitable for multiscale dynamics. Here, we combine a high-through-put adaptive microscopy approach with machine learning, to identify key biological and physical mechanisms that determine distinct microscopic and macroscopic collective behavior phases which develop as Bacillus subtilis swarms expand over five orders of magnitude in space. Our experiments, continuum modeling, and particle-based simulations reveal that macroscopic swarm expansion is primarily driven by cellular growth kinetics, whereas the microscopic swarming motility phases are dominated by physical cell–cell interactions. These results provide a unified understanding of bacterial multiscale behavioral complexity in swarms.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CDEV-MS01.html#author3"><strong class="card-body-subgroup">Dhananjay Bhaskar</strong></a>
		    	 
		    		<em> (Yale and Brown Universities, USA)</em>
		    	
		    	
		    		
		    			</br>"Discrete Agent Modeling and Topological Data Analysis of Self-Organized Multicellular Architectures"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Animal tissues are spatially patterned into complex topologies via directed motility and cell-cell interactions during development, repair, and disease. Segregation and cell sorting, driven by differential adhesion and interfacial tension, generate complex yet stable configurations that underlie tissue morphogenesis. Moreover, alterations of cell-cell adhesion and polarity in epithelial-mesenchymal transition (EMT) are associated with tumor dissemination and metastasis. In this talk, I will describe the use of topological data analysis for the automated classification of multicellular structures associated with EMT and cell sorting.
First, individual and collective phases of epithelial migration are characterized by varying adhesion and random propulsion parameters in an agent-based model derived from experimental observations. Next, persistent homology is computed using cell positions as input, followed by unsupervised classification of the topological features (connected components and loops). Finally, classification results are mapped onto the adhesion-propulsion phase diagram for automatic delineation of phase boundaries. A similar methodology, applied to co-culture simulations with varying adhesion parameters, reveals phase transitions between various patterns of self-assembly in cell sorting. I envision this computational approach will enable new quantitative insights into the emergence of complex tissue topologies via spatiotemporal interactions between one or more cell types.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/CDEV-MS01.html#author4"><strong class="card-body-subgroup">John Nardini</strong></a>
		    	 
		    		<em> (NC State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Topology discriminates parameter regimes in a model of angiogenesis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Angiogenesis is the process by which blood vessels form from pre-existing vessels. It plays a key role in many biological processes, including embryonic development and wound healing, and contributes to many diseases including cancer and rheumatoid arthritis. The structure of the resulting vessel networks determines their ability to deliver nutrients and remove waste products from biological tissues. Here we simulate the Anderson-Chaplain model of angiogenesis at different parameter values and quantify the vessel architectures of the resulting synthetic data. Specifically, we propose a topological data analysis (TDA) pipeline for systematic analysis of the model. TDA is a vibrant and relatively new field of computational mathematics for studying the shape of data. We compute topological and standard descriptors of model simulations generated by different parameter values. We show that TDA of model simulation data stratifies parameter space into regions with similar vessel morphology. The methodologies proposed here are widely applicable to other synthetic and experimental data including wound healing, development, and plant biology.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#CDEV-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/DDMB-MS01.html">
		
  	<h5 class="card-header card-header-subgroup DDMB"><small>MS01-DDMB: </small></br>Stochastic models of cancer: An update of theory and data</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Marek Kimmel (Rice University, United States), Simon Tavare (Columbia University, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-DDMB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/DDMB-MS01.html#author1"><strong class="card-body-subgroup">Tibor Antal</strong></a>
		    	 
		    		<em> (School of Mathematics, Edinburgh University, Scotland, UK, UK)</em>
		    	
		    	
		    		
		    			</br>"Models of Tumor Progression"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			I'll review recent results on timing in evolving cell populations. The stochasticity is taken into account by the use of branching processes, and the general question is the time it takes to produce cells with some specific properties. As an application the (possible) relapse time after cancer treatment will be discussed.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/DDMB-MS01.html#author2"><strong class="card-body-subgroup">Robert Beckman</strong></a>
		    	 
		    		<em> (Lombardi Comprehensive Cancer Center and Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC, USA, USA)</em>
		    	
		    	
		    		
		    			</br>"Recent Advances in Genetic Instability and Dynamic Precision Medicine of Cancer"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer evolution has similarities with species evolution but also important differences. This talk will outline some of these differences, including results from the deepest, most accurate sequencing of a solid tumor done to date. By probing the evolution of extremely rare mutations, we learn that the amount of intratumoral heterogeneity is far greater than previously assumed, and that subclones harboring pre-existing resistance to any single therapy will be universally present in any clinically diagnosable tumor. Moreover, rare mutations evolve neutrally, and surprisingly the popular “infinite sites assumption” suggesting that a new mutation will appear in only one cell at once, does not apply at this stage of tumor growth.
Cancer treatment has also evolved, from an empirical science of killing dividing cells, to the current era of “precision medicine”, targeted to molecular features of individual cancers. However, current precision medicine views a single individual’s cancer as largely uniform and static. Moreover, from a strategic perspective, it thinks primarily of the current therapeutic maneuver. In contrast, dynamic precision medicine (DPM) plans ahead, considers intratumoral heterogeneity and evolutionary dynamics, and probabilistically weighs the future benefits of preventing resistance against the benefits of immediate cytoreduction. Simulations indicate it has the potential to double median survival broadly across cancers. This talk will provide background about DPM, highlighting differences from other approaches to evolutionarily directed therapy. It will describe recent advances, including the role of genetically unstable “hypermutator subclones”, the possibility of clinical translation in presurgical “neoadjuvant “ settings, and considerations in clinical trial design.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/DDMB-MS01.html#author3"><strong class="card-body-subgroup">Alexandre Bouchard-Côté</strong></a>
		    	 
		    		<em> (Statistics, University of British Columbia, Vancouver, BC, Canada, Canada)</em>
		    	
		    	
		    		
		    			</br>"Inferring fitness of cancer subpopulations from time series --- Bayesian methods for the Wright-Fisher diffusion with selection"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			From timeseries of patient-derived xenograft data, we are interested in inferring fitness parameters for sub-populations of cancer cells measured using single cell sequencing. I will describe a statistical model for Bayesian inference of these fitness parameters. The model is based on a stochastic differential equation, the Wright-Fisher diffusion with fitness, the parameters of which are treated as random. We extend advanced MCMC methodologies such as PMCMC to perform Bayesian inference at scale, the posterior distribution being defined over a high-dimensional space and informed by a large dataset.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/DDMB-MS01.html#author4"><strong class="card-body-subgroup">Ivana Bozic</strong></a>
		    	 
		    		<em> (Applied Mathematics, University of Washington, Seattle, WA, USA, USA)</em>
		    	
		    	
		    		
		    			</br>"Mathematical model of colorectal cancer initiation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Cancer evolution cannot be observed directly in patients, and new methodologies are needed for obtaining a quantitative understanding of this obscure process. We developed and analyzed a stochastic model of malignant transformation in the colon that precisely quantifies the process of colorectal carcinogenesis in patients through loss of tumor suppressors APC and TP53 and gain of the KRAS oncogene. Our study employs experimentally measured mutation rates in the colon and growth advantages provided by driver mutations. We calculate the probability of a colorectal malignancy, the sizes of premalignant lesions, and the order of acquisition of driver mutations during colorectal tumor evolution. We demonstrate that the order of driver events in colorectal cancer is determined primarily by the fitness effects that they provide, rather than their mutation rates.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#DDMB-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/ECOP-MS01.html">
		
  	<h5 class="card-header card-header-subgroup ECOP"><small>MS01-ECOP: </small></br>Mathematical modeling of gene drives</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Gili Greenbaum (The Hebrew University of Jerusalem, Israel), Jaehee Kim (Cornell University, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS06-ECOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/ECOP-MS01.html#author1"><strong class="card-body-subgroup">Marcus Feldman</strong></a>
		    	 
		    		<em> (Stanford University, USA)</em>
		    	
		    	
		    		
		    			</br>"The antecedents of modern gene-drive models: Some history of meiotic drive models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In the late 1950s, it was discovered that factor on the second chromosome of Drosophila melanogaster produced extreme departure from Mendelian segregation in males. This segregation distorter (SD) factor reached a frequency as high as 30 percent in some populations. In small isolated populations of Mus musculus a similar phenomenon, the t-factor, was found to be produce strong segregation distortion as well as infertility in males. Even earlier, segregation distortion has been observed in the XY sex determination of Drosophila pseudoobscura. In all cases models for the balance of pre- and post-zygotic types of selection were invoked to explain the existence of polymorphism for these “anti-Darwinian” meiotic drive phenomena. Dynamics of evolutionary genetic systems with Mendelian segregation are very special, and most well-known results in population genetic theory do not apply if there is meiotic drive. Modern approaches to gene-drive analysis have their antecedent in this early research.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ECOP-MS01.html#author2"><strong class="card-body-subgroup">Chaitanya Gokhale</strong></a>
		    	 
		    		<em> (Max Planck Institute for Evolutionary Biology, Germany)</em>
		    	
		    	
		    		
		    			</br>"Synthetic gene drives and the control problem"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Synthetic gene drives are a marvel at par with any other technologies with a capacity for massive global impact. However, the discussion about drive control or intervention dilemmas is not at the same level as with some other technologies that are further behind but overhyped, such as generalized AI. With mathematical models, we show the resultant dynamics of synthetic drive technologies given the forces ecological forces they may face in the wild- particularly mate choice, different mating systems and structured populations. We assess the risk of the drive succeeding, failing or going rogue. In closing, we discuss the control problem from AI while simultaneously acknowledging the differences between artificial and natural selection
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ECOP-MS01.html#author3"><strong class="card-body-subgroup">Philipp Messer</strong></a>
		    	 
		    		<em> (Cornell University, USA)</em>
		    	
		    	
		    		
		    			</br>"Suppression gene drive in continuous space can result in unstable persistence of both drive and wild‐type alleles"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Rapid evolutionary processes can produce drastically different outcomes when studied in panmictic population models vs. spatial models. One such process is gene drive, which describes the spread of “selfish” genetic elements through a population. Engineered gene drives are being considered for the suppression of disease vectors or invasive species. While laboratory experiments and modelling in panmictic populations have shown that such drives can rapidly eliminate a population, it remains unclear if these results translate to natural environments where individuals inhabit a continuous landscape. Using spatially explicit simulations, we show that the release of a suppression drive can result in what we term “chasing” dynamics, in which wild‐type individuals recolonize areas where the drive has locally eliminated the population. Despite the drive subsequently reconquering these areas, complete population suppression often fails to occur or is substantially delayed. This increases the likelihood that the drive is lost or that resistance evolves. We analyze how chasing dynamics are influenced by the type of drive, its efficiency, fitness costs, and ecological factors such as the maximal growth rate of the population and levels of dispersal and inbreeding. Our results demonstrate that the population dynamics of suppression gene drives are determined by a complex interplay of genetic and ecological factors, highlighting the need for realistic spatial modelling to predict the outcome of drive releases in natural populations.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ECOP-MS01.html#author4"><strong class="card-body-subgroup">John Marshall</strong></a>
		    	 
		    		<em> (University of California Berkeley, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling priorities as gene drive mosquito projects transition from lab to field"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Despite significant reductions in malaria incidence and prevalence over the last decade following the wide-spread distribution of long-lasting insecticide-treated nets, malaria is not expected to be eliminated with currently available tools. Consequently, there is interest in novel interventions that complement existing ones, including gene drive-modified mosquitoes. Mathematical modeling has a central role to play in determining the impact that gene drive systems could have, alongside other interventions, towards the goal of malaria elimination. In this talk, we survey modeling priorities as gene drive mosquito projects advance from the lab to the field. We begin by highlighting priorities in model building, including: i) capturing nuances in the inheritance-biasing impacts of gene drive systems, ii) incorporating data and insights on mosquito vector ecology, including life history, habitat distribution and movement patterns, and iii) aligning entomological models with detailed models of malaria transmission, including the impacts of currently available and novel interventions. We then highlight several priorities in model application as gene drive products advance from the lab to the field. These include informing target product profiles for gene drive products to assess when they satisfy safety and efficacy criteria, and informing the design of cage trials, field trials and eventually vector and disease control interventions. Other priorities include developing monitoring programs to assess the safety and efficacy of trials and interventions, developing surveillance programs to detect unintended spread, and addressing risk and regulatory questions requiring a quantitative analysis.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ECOP-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/EDUC-MS01.html">
		
  	<h5 class="card-header card-header-subgroup EDUC"><small>MS01-EDUC: </small></br>Highlights of the Special Issue of BMB on Mathematical Biology Education</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> John R Jungck (University of Delaware, USA), Raina Robeva (Randolph Macon College, USA), Louis Gross (University of Tennessee, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-EDUC</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/EDUC-MS01.html#author1"><strong class="card-body-subgroup">Midge Cozzens</strong></a>
		    	 
		    		<em> (Rutgers University, USA)</em>
		    	
		    	
		    		
		    			</br>"Introductory College Mathematics for the Life Sciences: Has Anything Changed?"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Our paper focused on issues concerning the introductory college mathematics sequence with an emphasis on students interested in the life sciences, and concentration on the time after the publication of BIO2010. It also explored the potential uses of books targeted at introductory mathematics courses for life science majors today. As relevant background, we looked at the evolution of the way that calculus has been taught over the past 50 years, including at the high school level. We also explored the implications of changes in technology and course delivery, such as online education. As we discussed different books and introductory course ideas, we focused on the needs of biology students, inclusion of real-world problems and models, the role of technology, and the impact of data science. Our paper dealt with 8 issues: Section 1 provided some personal background with calculus dating back to the 70’s, and changes in calculus prior to BIO2010.  Section 2 introducesd goals for an introductory mathematics sequence and evaluates the calculus sequence in light of those goals. Sections 3-7 discussed various issues that will help to understand issues and challenges for introductory mathematics for the life sciences: Calculus in high school (Section 3), equity issues relative to calculus and other math topics (Section 4), the impact of online education (Section 5), math as a stumbling block for college students (Section 6), and the increasing importance and value of teaching data science (Section 7). Section 8 reviewed the development of books in light of these issues and challenges. The last section (Section 9) summarizes conclusions.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EDUC-MS01.html#author2"><strong class="card-body-subgroup">Melissa Aikens</strong></a>
		    	 
		    		<em> (University of New Hampshire, USA)</em>
		    	
		    	
		    		
		    			</br>"Advances and Challenges in Undergraduate Biology Education"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Over the last 25 years, reforms in undergraduate biology education have transformed the way biology is taught at many institutions of higher education. This has been fueled in part by a burgeoning discipline-based education research community, which has advocated for evidence-based instructional practices based on findings from research. This perspective reviews some of the changes to undergraduate biology education that have gained or are currently gaining momentum, becoming increasingly common in undergraduate biology classrooms. However, there are still areas in need of improvement. Although more underrepresented minority students are enrolling in and graduating from biology programs than in the past, there is a need to understand the experiences and broaden participation of other underserved groups in biology and ensure biology classroom learning environments are inclusive. Additionally, although understanding biology relies on understanding concepts from the physical sciences and mathematics, students still rarely connect the concepts they learn from other STEM disciplines to biology. Integrating concepts and practices across the STEM disciplines will be critical for biology graduates as they tackle the biological problems of the 21st century.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EDUC-MS01.html#author3"><strong class="card-body-subgroup">Raina Robeva</strong></a>
		    	 
		    		<em> (Randolph Macon College, USA)</em>
		    	
		    	
		    		
		    			</br>"Changing the Nature of Quantitative Biology Education: Data Science as a Driver"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We live in a data-rich world with rapidly growing databases with zettabytes of data. Innovation, computation, and technological advances have now tremendously accelerated the pace of discovery, providing driverless cars, robotic devices, expert healthcare systems, precision medicine, and automated discovery to mention a few. Even though the definition of the term data science continues to evolve, the sweeping impact it has already produced on society is undeniable. We are at a point when new discoveries through data science have enormous potential to advance progress but also to be used maliciously, with harmful ethical and social consequences. Perhaps nowhere is this more clearly exemplified than in the biological and medical sciences. The confluence of (1) machine learning, (2) mathematical modeling, (3) computation/simulation, and (4) big data have moved us from the sequencing of genomes to gene editing and individualized medicine; yet, unsettled policies regarding data privacy and ethical norms could potentially open doors for serious negative repercussions. The data science revolution has amplified the urgent need for a paradigm shift in undergraduate biology education. It has reaffirmed that data science education interacts and enhances mathematical education in advancing quantitative conceptual and skill development for the new generation of biologists. These connections encourage us to strive to cultivate a broadly skilled workforce of technologically savvy problem-solvers, skilled at handling the unique challenges pertaining to biological data, and capable of collaborating across various disciplines in the sciences, the humanities, and the social sciences. To accomplish this, we suggest development of open curricula that extend beyond the job certification rhetoric and combine data acumen with modeling, experimental, and computational methods through engaging projects, while also providing awareness and deep exploration of their societal implications. This process would benefit from embracing the pedagogy of experiential learning and involve students in open-ended explorations derived from authentic inquiries and ongoing research. On this foundation, we encourage development of flexible data science initiatives for the education of life science undergraduates within and across existing models.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EDUC-MS01.html#author4"><strong class="card-body-subgroup">Padmanabhan Seshaiyer</strong></a>
		    	 
		    		<em> (George Mason University, USA)</em>
		    	
		    	
		    		
		    			</br>"Conneccting with teachers through Modeling Mathematical Biology"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We describe some effective teaching and research practices that can help to integrate mathematics and biology efficiently to enhance student learning at all levels. One of the successful approaches proposed is to employ mathematical modeling that can help transform pedagogical practices. In this regard, we introduce some modeling activities that have been shared with teachers through professional development programs and hav been incorporated in the classrooms. We also present how engaging teachers in research experiences in mathematical modeling can help to transform their pedagogical practices and provide opportunities for students to consider pursuing areas at the interface of mathematics and biology.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EDUC-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/EVOP-MS01.html">
		
  	<h5 class="card-header card-header-subgroup EVOP"><small>MS01-EVOP: </small></br>Non-equilibrium Thermodynamics in Biology: from Chemical Reaction Networks to Natural Selection</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> John Baez (University of California, Riverside, USA), William Cannon (Pacific Northwest National Laboratory, USA), Larry Li (University of California, Riverside, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-EVOP</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/EVOP-MS01.html#author1"><strong class="card-body-subgroup">John Harte</strong></a>
		    	 
		    		<em> (University of California, Berkeley, USA)</em>
		    	
		    	
		    		
		    			</br>"Nonequilibrium dynamics of disturbed ecosystems"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The Maximum Entropy Theory of Ecology (METE) predicts the shapes of macroecological metrics in relatively static ecosystems, across spatial scales, taxonomic categories, and habitats, using constraints imposed by static state variables. In disturbed ecosystems, however, with time-varying state variables, its predictions often fail. We extend macroecological theory from static to dynamic, by combining the MaxEnt inference procedure with explicit mechanisms governing disturbance. In the static limit, the resulting theory, DynaMETE, reduces to METE but also predicts a new scaling relationship among static state variables. Under disturbances, expressed as shifts in demographic, ontogenic growth, or migration rates, DynaMETE predicts the time trajectories of the state variables as well as the time-varying shapes of macroecological metrics such as the species abundance distribution and the distribution of metabolic rates over individuals. An iterative procedure for solving the dynamic theory is presented. Characteristic signatures of the deviation from static predictions of macroecological patterns are shown to result from different kinds of disturbance. By combining MaxEnt inference with explicit dynamical mechanisms of disturbance, DynaMETE is a candidate theory of macroecology for ecosystems responding to anthropogenic or natural disturbances.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EVOP-MS01.html#author2"><strong class="card-body-subgroup">Hong Qian</strong></a>
		    	 
		    		<em> (University of Washington, USA)</em>
		    	
		    	
		    		
		    			</br>"Large deviations theory and emergent landscapes in biological dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The mathematical theory of large deviations provides a nonequilibrium thermodynamic description of complex biological systems that consist of heterogeneous individuals. In terms of the notions of stochastic elementary reactions and pure kinetic species, the continuous-time, integer-valued Markov process dictates a thermodynamic structure that generalizes (i) Gibbs’ macroscopic chemical thermodynamics of equilibrium matters to nonequilibrium small systems such as living cells and tissues; and (ii) Gibbs’ potential function to the landscapes for biological dynamics, such as that of C. H. Waddington’s and S. Wright’s.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EVOP-MS01.html#author3"><strong class="card-body-subgroup">Pierre Gaspard</strong></a>
		    	 
		    		<em> (Université libre de Bruxelles, Belgium)</em>
		    	
		    	
		    		
		    			</br>"Nonequilibrium biomolecular information processes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Nearly 70 years have passed since the discovery of DNA structure and its role in coding genetic information. Yet, the kinetics and thermodynamics of genetic information processing in DNA replication, transcription, and translation remain poorly understood. These template-directed copolymerization processes are running away from equilibrium, being powered by extracellular energy sources. Recent advances show that their kinetic equations can be exactly solved in terms of so-called iterated function systems. Remarkably, iterated function systems can determine the effects of genome sequence on replication errors, up to a million times faster than kinetic Monte Carlo algorithms. With these new methods, fundamental links can be established between molecular information processing and the second law of thermodynamics, shedding a new light on genetic drift, mutations, and evolution.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/EVOP-MS01.html#author4"><strong class="card-body-subgroup">Carsten Wiuf</strong></a>
		    	 
		    		<em> (University of Copenhagen, Denmark)</em>
		    	
		    	
		    		
		    			</br>"Reduction and the Quasi-Steady State Approximation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Chemical reactions often occur at different time-scales. In applications of chemical reaction network theory it is often desirable to reduce a reaction network to a smaller reaction network by elimination of fast species or fast reactions. There exist various techniques for doing so, e.g. the Quasi-Steady-State Approximation or the Rapid Equilibrium Approximation. However, these methods are not always mathematically justifiable.  Here, a method is presented for which (so-called) non-interacting species are eliminated by means of QSSA. It is argued that this method is mathematically sound. Various examples are given (Michaelis-Menten mechanism, two-substrate mechanism, ...) and older related techniques from the 50-60ies are briefly discussed.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#EVOP-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/IMMU-MS01.html">
		
  	<h5 class="card-header card-header-subgroup IMMU"><small>MS01-IMMU: </small></br>Mathematical tools for understanding viral infections within-host and between-host</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Hana Dobrovolny (Texas Christian University, United States), Gilberto Gonzalez-Parra (New Mexico Tech, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-IMMU</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/IMMU-MS01.html#author1"><strong class="card-body-subgroup">Guang Lin</strong></a>
		    	 
		    		<em> (Purdue University, United States)</em>
		    	
		    	
		    		
		    			</br>"Predicting the COVID-19 pandemic with uncertainties using data-driven models"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We have developed an integer-order COVID-19 epidemic model and a fractional-order COVID-19 epidemic model to reconstruct and forecast the transmission dynamics of COVID-19 in New York City. To quantify the uncertainties in the proposed data-driven epidemic model, we have investigated model sensitivity analysis, structural and practical identifiability analysis, model calibration, and uncertainty quantification. We have employed Bayesian model calibration and physics-informed machine learning algorithms to calibrate the model parameters. In the early stage of the outbreak in New York City, the reproduction number was around 4.3, which indicates this outbreak has high transmissibility. We observed that multi-pronged interventions, such as the stay-at-home order and social distancing, had positive effects on controlling the outbreak and slowing the virus's spread. In addition, we employed the proposed data-driven models to evaluate the effects of various strategies to deploy the Covid-19 vaccine to control the pandemic. We have also applied the formulation to infer the dynamics of COVID-19 in other cities/states, where the spread dynamic is different from New York City.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/IMMU-MS01.html#author2"><strong class="card-body-subgroup">Ana Vivas-Barber</strong></a>
		    	 
		    		<em> (Norfolk State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Using Seasonality and Variable Incubation Periods to Study the Impact of Including Domestic Animals on the Dynamics of Malaria Transmission"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We investigate the impact of including variable mosquito population and added variable long and short incubation periods on the transmission dynamics of Malaria in Korea. The SEIS model is based on malaria infected mosquitoes which bite humans or animals. This model studies plasmodium vivax malaria and has variables for animal population and mosquito attraction to animals. The basic reproduction number of the ODE model with seasonal mosquito population (exponential) is presented and analyzed. The existing time-independent Malaria population ODE model was extended to time-dependent model with the difference explored. Also, using bi-modal Malaria incubation, changes to the infectious population when constant incubation period is extended to varied in the ODE model. Endemic equilibrium and stability analysis for the model was conducted with conditions on variables to insure solvability and DFE.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/IMMU-MS01.html#author3"><strong class="card-body-subgroup">Imelda Trejo Lorenzo</strong></a>
		    	 
		    		<em> (Los Alamos National Laboratory, United States)</em>
		    	
		    	
		    		
		    			</br>"A modified Susceptible-Infected-Recovered model for observed under-reported incidence data"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In this talk, I will present a mathematical model to quantify the fraction of unreported infected individuals during epidemic outbreaks. The model consists of three parts (1) a dynamical system base on the classical Susceptible-Infected-Recovered (SIR) epidemic model, (2) a stochastic model for the observed incidence and (3) a Bayesian approach to estimate the model parameters. We use the model to estimate the infection rate and fraction of under-reported individuals for the current Coronavirus 2019 outbreak in some American Countries. Our analysis reveals that consistently, about 50% of infected individuals were not observed in various South American outbreaks.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/IMMU-MS01.html#author4"><strong class="card-body-subgroup">Naveen Vaidya</strong></a>
		    	 
		    		<em> (San Diego State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Modeling the risk of SARS-CoV-2 transmission from fomites"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The novel coronavirus disease (COVID-19) constitutes one of the most devastating pandemics of the 21st century. While direct person-to-person transmission of SARS-CoV-2, the etiological agent of COVID-19, appears to be the primary route of transmission, the contraction of SARS-CoV-2 from fomites in the environment is also considered a potential contributor to the disease transmission. In this talk, I will present a mathematical model to predict the probability of detecting SARS-CoV-2 in the environmental reservoirs during the COVID-19 outbreak in a community. Furthermore, we extend our model to predict the potential contribution of fomite transmission to the generation of new COVID-19 cases. We validate our model using experimental data with a large number of swab samples collected from commonly touched surfaces across San Diego County.  Our model, which is capable of describing transmission dynamics of COVID-19 within San Diego county, allows us to compute the risk for an individual to encounter virus in the environment. The results indicate that the persistence of virus in some environmental surfaces can lead to a significant number of COVID-19 cases in the community.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#IMMU-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/MEPI-MS01.html">
		
  	<h5 class="card-header card-header-subgroup MEPI"><small>MS01-MEPI: </small></br>From Primate to Vectors to Humans: Understanding the underlying mechanisms of disease transmission and control</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Folashade Agusto (University of Kansas, United States), Majid Bani Yaghoub (University of Missouri Kansas City, United States)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-MEPI</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/MEPI-MS01.html#author1"><strong class="card-body-subgroup">Amy Goldberg</strong></a>
		    	 
		    		<em> (Duke University, United States)</em>
		    	
		    	
		    		
		    			</br>"Model-based estimates of zoonotic malaria spillover in Atlantic Forest, Brazil"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Malaria was thought to have been eradicated from the Atlantic Coast of Brazil by the late 1970s. Previously thought to only infect non-human primates, recent molecular studies have identified the malaria parasite Plasmodium simium in humans along the Atlantic Coast of Brazil. Clinical symptoms present similarly to the common human-associated malaria parasite Plasmodium vivax, and the two parasites are difficult to distinguish with standard PCR assays or microscopy. Together, these observations raise the possibility that local monkey populations, particularly howler monkeys, act as reservoirs for zoonotic malaria that has been infecting human populations long-term. Here, we use a mathematical-modeling approach to estimate the rate of cryptic P. simiam infection that has been misdiagnosed as P. vivax in the Rio de Janiero state. We use coupled differential equations based on the Ross-MacDonald model, with two host populations representing humans and monkeys to model the infection rate of humans and howler monkeys with P. simiam. Based on elasticity analyses, we find that for the same intensity, interventions in the monkey patch reduces the overall number of human malaria cases more than interventions in the human patch. We simulate the model across a spatial grid, with the two-population system in each patch and migration between patches. Under various spillover scenarios, we compare results to clinical incidence rates of P. vivax and consider the impact on malaria elimination probability. Based on the frequency and spatial distribution of the cases, under our model, we expect spill over to be recurrent, with minimal human-to-human transmission.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MEPI-MS01.html#author2"><strong class="card-body-subgroup">Ibrahim M. ELMojtaba</strong></a>
		    	 
		    		<em> (Sultan Qaboos University, Oman)</em>
		    	
		    	
		    		
		    			</br>"The role of primates and human movement on the dynamics of zika virus"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			We build a mathematical model to understand the role of human movement and primates in the dynamics of zika virus. The model considers the dynamics of the disease between four different populations, namely humans, primates, vectors in the rural areas, and vectors in urban areas. Our model possesses three different equilibrium, the disease-free equilibrium which is locally asymptotically stable when the basic reproduction number is less than unity, an axial equilibrium point (endemic with respect to human and vectors in urban areas, and disease-free with respect to primates and vectors in rural areas), and endemic equilibrium. The model exhibits very rich dynamics where there is a possibility of multiple bifurcations. Numerical simulations were carried out to study the effect of several parameters and to show the theoretical results.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MEPI-MS01.html#author3"><strong class="card-body-subgroup">Omar Saucedo</strong></a>
		    	 
		    		<em> (Virginia Polytechnic Institute and State University, United States)</em>
		    	
		    	
		    		
		    			</br>"Tick-borne Diseases in Virginia"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Ticks are known for being a source of disease infections and are cause of great concern within the public health community.  Throughout the world, there are a variety of tick species that are associated with different tick-borne pathogens.  Diseases such as Lyme Disease have surfaced in areas of the Commonwealth of Virginia where they previously have not been detected, and the incidence of these diseases have been steadily increasing.    A better understanding of tick-borne viral pathogens is needed as this poses a threat to agriculture and livestock.   In this talk, we will explore the relevant features of landscape and ecological influences on tick species and pathogen prevalence through mathematical modeling.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MEPI-MS01.html#author4"><strong class="card-body-subgroup">Sean Cavany</strong></a>
		    	 
		    		<em> (University of Notre Dame, United States)</em>
		    	
		    	
		    		
		    			</br>"The impacts of COVID-19 mitigation on dengue virus transmission: a modelling study"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The COVID-19 pandemic has induced unprecedented reductions in human mobility and social contacts throughout the world. Because dengue virus (DENV) transmission is strongly driven by human mobility, behavioral changes associated with the pandemic have been hypothesized to impact dengue incidence. By discouraging human contact, COVID-19 control measures have also disrupted dengue vector control interventions, the most effective of which require entry into homes. We used an agent-based model with a realistic treatment of human mobility and vector control to investigate how and why dengue incidence could differ under a lockdown scenario with a proportion of the population sheltered at home. We found that a lockdown in which 70% of the population sheltered at home led to a small average increase in cumulative DENV infections of up to 10%, depending on the time of year the lockdown occurred. Lockdown had a more pronounced effect on the spatial distribution of DENV infections, with higher incidence under lockdown in regions with high mosquito abundance. Transmission was also more focused in homes following lockdown. The proportion of people infected in their own home rose from 54% under normal conditions to 66% under lockdown, and the household secondary attack rate rose from 0.109 to 0.128, a 17% increase. When we considered that lockdown measures could disrupt regular, city-wide vector control campaigns, the increase in incidence was more pronounced than with lockdown alone, especially if lockdown occurred at the optimal time for vector control. Our results indicate that an unintended outcome of COVID-19 control measures may be to adversely alter the epidemiology of dengue. This observation has important implications for an improved understanding of dengue epidemiology and effective application of dengue vector control. When coordinating public health responses during a syndemic, it is important to monitor multiple infections and understand that an intervention against one disease may exacerbate another.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MEPI-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/MFBM-MS01.html">
		
  	<h5 class="card-header card-header-subgroup MFBM"><small>MS01-MFBM: </small></br>Mathematical Modeling Applied to Pharmaceutical Sciences Problems</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Carl Panetta (St. Jude Children's Research Hospital, US), Helen Moore (Laboratory for Systems Medicine, University of Florida, US)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/MFBM-MS01.html#author1"><strong class="card-body-subgroup">C.J. Musante</strong></a>
		    	 
		    		<em> (Pfizer, US)</em>
		    	
		    	
		    		
		    			</br>"A Few Open Mathematical Modeling Problems in Drug Discovery & Development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Now, more than ever, pharmaceutical companies are relying on mathematical modeling & simulation to inform drug discovery and development decisions. In many cases, particularly for novel compounds, targets, and/or combinations, modelers must rely on incomplete data and/or extrapolation beyond existing data to address key questions, such as dose and dose regimen selection for clinical trials. In this talk, I will present a few case studies and related mathematical challenges that my team has faced and discuss why developing a better understanding of these problems is important in the context of drug discovery & development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MFBM-MS01.html#author2"><strong class="card-body-subgroup">Jane Bai</strong></a>
		    	 
		    		<em> (FDA, US)</em>
		    	
		    	
		    		
		    			</br>"Conducting sensitivity analysis and uncertainty analysis for QSP needs more than mathematical computation"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			In quantitative systems pharmacology (QSP) modeling, sensitivity analysis is often conducted to identify a set of sensitive parameters to avoid overparameterization for robust calibration and validation. There are different global sensitivity analysis methods to choose from. Furthermore, for each model output, sensitivity analysis generates a rank-ordered list. Combining individual lists of rank-ordered sensitive parameters from all model outputs in a QSP model to obtain the final list may be subject to modeler’s judgement. Capturing variability in a trial population through uncertainty analysis and virtual patient trials can improve the predictive performance of a model and inform trial designs for a drug development program. However, multiple different algorithms can be used. This talk will discuss methodological considerations when applying sensitivity analysis and uncertainty analysis to QSP modeling for drug development.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MFBM-MS01.html#author3"><strong class="card-body-subgroup">Freya Bachmann</strong></a>
		    	 
		    		<em> (Department of Mathematics and Statistics, University of Konstanz, Germany)</em>
		    	
		    	
		    		
		    			</br>"Computing the Individualized Optimal Drug Dosing Regimen Using Optimal Control"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Providing the optimal dosing strategy of a drug for an individual patient is an important task in pharmaceutical sciences and daily clinical application. By solving an optimal control problem (OCP) especially tailored to pharmacokinetic-pharmacodynamic (PKPD) models the optimal individualized dosing regimen can be computed for substantially different scenarios with various routes of administration.
The aim is to compute a control that brings the underlying system as closely as possible to a desired reference state by minimizing an objective function. In PKPD modeling the controls are the administered doses and the reference state can be the disease progression. Therefore, the objective function which shall be minimized is quantifying the difference between a desired disease state and the actual state generated by a particular treatment. Drug administration at certain time points gives a finite number of discrete controls, the drug doses, determining the drug concentration and its effect on the disease state. Hence, it is possible to construct a finite-dimensional OCP depending only on the doses and apply robust quasi-Newton algorithms from finite-dimensional optimization.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MFBM-MS01.html#author4"><strong class="card-body-subgroup">Tongli Zhang</strong></a>
		    	 
		    		<em> (Department of Pharmacology & Systems Physiology, College of Medicine, University of Cincinnati, US)</em>
		    	
		    	
		    		
		    			</br>"Coping with the Challenge of Heterogeneity with Integrated Modeling, Machine Learning, and Dynamical Analysis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Heterogeneity among individual patients presents a fundamental challenge to effective treatment, since a treatment protocol would only work for a portion of the population. We hypothesize that a computational pipeline integrating mathematical modelling and machine learning could be used to address this fundamental challenge and facilitate the optimization of individualized treatment protocols. We tested our hypothesis with the neuroendocrine systems controlled by the hypothalamic-pituitary-adrenal axis. With a synergistic combination of mathematical modelling and machine learning, this integrated computational pipeline could indeed efficiently reveal optimal treatment targets that could significantly improve the treatment efficacy of a heterogeneous individuals, despite of the challenge that the simultaneous changes of multiple parameters result in complex dynamical patterns. Dynamical Analysis of the computational results then revealed mechanistic insights that connect heterogeneous behavior to model structure. We believe that this integrated computational pipeline, properly applied in combination with other computational, experimental and clinical research tools, can be used to optimize treatment targets against a broad range of complex diseases.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MFBM-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/MMPB-MS01.html">
		
  	<h5 class="card-header card-header-subgroup MMPB"><small>MS01-MMPB: </small></br>Mathematical Modeling of Blood Clotting: From Surface-Mediated Coagulation to Fibrin Polymerization</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Karin Leiderman (Colorado School of Mines, United States), Anna Nelson (University of Utah, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS07-MMPB</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/MMPB-MS01.html#author1"><strong class="card-body-subgroup">Anna Nelson</strong></a>
		    	 
		    		<em> (University of Utah, USA)</em>
		    	
		    	
		    		
		    			</br>"Understanding the effect of fibrinogen interactions on fibrin gel structure"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Fibrin polymerization, an important component of blood clotting, involves the conversion of soluble fibrinogen molecules in the blood plasma to fibrin monomers. These monomers can then polymerize to form a gel that is a major structural component of a blood clot. Oligomers composed of both fibrinogen and fibrin have been observed experimentally and are thought to impact the kinetics of the fibrin gelation process. Fibrinogen plays a dual role in fibrin polymerization; it can occupy available binding sites by binding to fibrin, inhibiting gelation, and monomeric fibrinogen and fibrinogen contained in oligomers can be converted to fibrin. To study the effects of fibrin-fibrinogen interactions on fibrin polymerization and fibrin gel structure, we developed a kinetic polymerization model with two monomers, where the reaction sites on the different species of monomers can participate in different binding reactions. With the chosen framework, gelation can occur, which is defined to be the finite time blow-up of a particular second moment of the oligomer distribution. We characterize the conditions under which a gel forms and examine the impact of fibrin-fibrinogen binding and fibrinogen conversion to fibrin on the branch point density in a gel, if one forms.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MMPB-MS01.html#author2"><strong class="card-body-subgroup">Michael Kelley</strong></a>
		    	 
		    		<em> (Colorado School of Mines, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling the effects of bivalently bound thrombin on fibrin polymerization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Thrombin is an enzyme generated during the blood coagulation process and is crucial to the formation of a stable blood clot. Thrombin cleaves fibrinogen into fibrin, which polymerizes to form a stabilizing gel matrix. Thrombin can also bind directly to fibrin and become sequestered for long periods of time. Experimental models support the dogma that this retention is due to the dynamic interplay of thrombin binding to both low- and high-affinity binding sites on fibrinogen and an alternative splice variant of fibrinogen, $gamma’$ that makes up about 15% of the total fibrinogen pool. Recent experimental studies have suggested that $gamma’$ decreases the rate of fibrin polymerization but there are conflicting results in the literature regarding its effects on other aspects of fibrin polymerization such as rates of fibrinopeptide release and clot morphology. The goal of this study was to use a mathematical modeling approach to help interpret some of the disparate results. We built on an existing model of fibrin polymerization and added our previous model of bivalent thrombin-fibrin binding to investigate how thrombin and fibrin interact dynamically during polymerization. Preliminary results show that during  dynamic fibrin polymerization, a large fraction of thrombin can become trapped within fibers as they form. Additionally, we show that the $gamma’$ binding of thrombin to fibrin acts to increase fiber thickness, modulating the formation of and polymerization of fibrin.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MMPB-MS01.html#author3"><strong class="card-body-subgroup">Francesco Pancaldi</strong></a>
		    	 
		    		<em> (University of California Riverside, USA)</em>
		    	
		    	
		    		
		    			</br>"Modeling study of clot contraction"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Blood clots are one major cause of death and disability worldwide. Blood clot formation has been relatively well studied, however, little is known about the contraction or retraction of clots. Clot contraction is driven by activated platelets that pull on fibrin fibers, causing a reduction in clot volume. In this talk, we present a model to quantify platelet and fibrin-mediated blood clot contraction mechanisms. The model combines a fibrin network mechanical model and a sub-model accounting for the forces generated by activated platelets. We used experimental measurements to calibrate model parameters and model simulations were used to reveal contraction mechanisms. The contraction was shown to depend on how pulling forces, generated by platelets, change based on local fiber stiffness and the number of filopodia per platelet. In particular, the number of filopodia per platelet contributed to the formation of distinct numbers and length of contraction phases, as defined by peaks in the change of fibrin density. Our simulations show that the number of filopodia per platelet is important to obtain the correct number and length of contraction phases. Finally, the model reproduced experimentally observed clustering of platelets within the contracting clot and predicted more rapid clustering at the initial stages of contraction.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/MMPB-MS01.html#author4"><strong class="card-body-subgroup">Sumith Yesudasan</strong></a>
		    	 
		    		<em> (Sam Houston State University, USA)</em>
		    	
		    	
		    		
		    			</br>"Coarse-grained Molecular Model for Fibrin Polymerization"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The study on the polymerization of fibrinogen molecules into fibrin monomers and eventually a stable, mechanically robust fibrin clot is a persistent and enduring topic in the field of thrombosis and hemostasis. Despite many research advances in fibrin polymerization, the change in the structure of fibrin clots and its influence on the formation of a fibrous protein network are still poorly understood. In this paper, we develop a new computational method to simulate fibrin clot polymerization using dissipative particle dynamics simulations. With an effective combination of reactive molecular dynamics formularies and many body dissipative particle dynamics principles, we constructed the reactive dissipative particle dynamics (RDPD) model to predict the complex network formation of fibrin clots and branching of the fibrin network. The 340 kDa fibrinogen molecule is converted into a spring-bead coarse-grain system with 11 beads using a topology representing network algorithm, and using RDPD, we simulated polymerization and formation of the fibrin clot. The final polymerized structure of the fibrin clot qualitatively agrees with experimental results from the literature, and to the best of our knowledge this is the first molecular-based study that simulates polymerization and structure of fibrin clots.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#MMPB-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/NEUR-MS01.html">
		
  	<h5 class="card-header card-header-subgroup NEUR"><small>MS01-NEUR: </small></br>Mathematical neuroscience</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Sunil Modhara (University of Nottingham, United Kingdom), Stephen Coombes (University of Nottingham, United Kingdom), Ruediger Thul (University of Nottingham, United Kingdom), Daniele Avitabile (Vrije Universiteit Amsterdam, Netherlands)
				
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/NEUR-MS01.html#author1"><strong class="card-body-subgroup">Sunil Modhara</strong></a>
		    	 
		    		<em> (University of Nottingham, United Kingdom)</em>
		    	
		    	
		    		
		    			</br>"Neural fields with rebound currents: novel routes to patterning"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The understanding of how spatio-temporal patterns of neural activity may arise in the cortex of the brain has advanced with the development and analysis of neural field models. To replicate this success for sub-cortical tissues, such as the thalamus, requires an extension to include relevant ionic currents that can further shape firing response. Here we advocate for one such approach that can accommodate slow currents. By way of illustration we focus on incorporating a T-type calcium current into the standard neural field framework. Direct numerical simulations are used to show that the resulting tissue model has many of the properties seen in more biophysically detailed model studies, and most importantly the generation of oscillations, waves, and patterns that arise from rebound firing. To explore the emergence of such solutions we focus on one- and two-dimensional spatial models and show that exact solutions describing homogeneous oscillations can be constructed in the limit that the firing rate nonlinearity is a Heaviside function. A linear stability analysis, using techniques from non-smooth dynamical systems, is used to determine the points at which bifurcations from synchrony can occur. Furthermore, we construct periodic travelling waves and investigate their stability with the use of an appropriate Evans function. The stable branches of the dispersion curve for periodic travelling waves are found to be in excellent agreement with simulations initiated from an unstable branch of the synchronous solution.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/NEUR-MS01.html#author2"><strong class="card-body-subgroup">Louisiane Lemaire</strong></a>
		    	 
		    		<em> (Inria Sophia Antipolis Méditerranée Research Centre, France)</em>
		    	
		    	
		    		
		    			</br>"Mathematical model of the mutations of a sodium channel (NaV1.1) capturing both migraine and epilepsy scenarios"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			NaV1.1 is a Na+ voltage-gated channel expressed in GABAergic neurons, crucial for their excitability. Gain-of-function mutations of this channel cause familial hemiplegic migraine type 3 (FHM3), while loss-of-function mutations lead to epilepsy. The pathological mechanisms are unclear. Cortical spreading depression (CSD) is a wave of intense firing followed by depolarization block, propagating in the cortex. In the case of FHM3, it is thought that CSD sensitizes meningeal nociceptors, inducing headache. However, the link between FHM3 mutations and the initiation of CSD remains to be understood. We develop a two-neuron (GABAergic and pyramidal) conductance-based model with dynamic ion concentrations, since the considered pathologies disrupt ionic gradients. Keeping the other parameter values unchanged, we implement FMH3 mutations using persistent Na+ current and epilepsy ones with reduced transient Na+ current. Our results suggest the importance of other mechanisms of action of NaV1.1 mutations than changes in the firing frequency of GABAergic neurons. In our model, FHM3 mutations modify ion fluxes at each action potential. The resulting accumulation of extracellular potassium facilitates CSD initiation and reduces its latency, in agreement with recent experimental findings. Epilepsy mutations make the GABAergic neuron more susceptible to depolarization block. The removal of their inhibitory restraint causes a simultaneous increase of the pyramidal cell's firing frequency.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/NEUR-MS01.html#author3"><strong class="card-body-subgroup">Manu Kalia</strong></a>
		    	 
		    		<em> (University of Twente, Netherlands)</em>
		    	
		    	
		    		
		    			</br>"Modeling ischemic vulnerability at the tripartite synapse"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			The work that I will present is about single-cell neuron-astrocyte interactions at the synapse. The model considered describes ion dynamics at such an interaction during ischemia (which means low oxygen due to reduced blood flow, say, during the onset of stroke). I will present some results about bistability in the model and the tipping from one state to another, and some predictions about possible pharmacological blockers to recover cells from ischemic injury.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/NEUR-MS01.html#author4"><strong class="card-body-subgroup">John Rinzel</strong></a>
		    	 
		    		<em> (New York University, USA)</em>
		    	
		    	
		    		
		    			</br>"A neuronal model for learning to keep a rhythmic beat"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			When listening to music, we typically lock onto and move to a beat (1-6 Hz). Behavioral studies on such synchronization (Repp 2005) abound, yet the neural mechanisms remain poorly understood.  Some models hypothesize an array of self-sustaining entrainable neural oscillators that resonate when forced with rhythmic stimuli (Large et al. 2010).   In contrast, our formulation focuses on event time estimation and plasticity: a  neuronal beat generator that adapts its intrinsic frequency and phase to match the extermal rhythm. The model quickly learns new rhythms, within a few cycles as found in human behavior. When the stimulus is removed the beat generator continues to produce the learned rhythm in accordance with a synchronization continuation task.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#NEUR-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
											<div class="row subgroup-name">
												<div class="col-12">
													
<div class="card">
	<a href="/MS01/ONCO-MS01.html">
		
  	<h5 class="card-header card-header-subgroup ONCO"><small>MS01-ONCO: </small></br>Mathematical approaches to advance clinical studies in oncology</h5>
  	
	</a>
  <div class="card-body">
    <p class="card-text">

	    
	    
	    
	    
				<strong>Organized by:</strong> Heyrim Cho (University of California Riverside, USA), Russell Rockne (City of Hope Comprehensive Cancer Center, USA)
				
				
					</br>
					<small>Note: this minisymposia has multiple sessions. The second session is <strong>MS02-ONCO</strong>.
					
					
					
					</small>
				
				</br>
			
			
			<ul>
	    	
		    	<li><a href="/MS01/ONCO-MS01.html#author1"><strong class="card-body-subgroup">Hitesh Mistry</strong></a>
		    	 
		    		<em> (University of Manchester, UK)</em>
		    	
		    	
		    		
		    			</br>"Complexity/Simplicity of Oncology Pharmacodynamic Markers/Mathematical Models in the Clinic versus Drug Development"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Pharmacodynamic markers provide information on what the drug is doing to the body, in this talk its a measure of what the drug is doing to the disease, cancer. The number and types of biomarkers in Oncology has increased dramatically over the last 20-30 years.  Our focus here will be on biomarkers that are used for selecting a dose/schedule. Many of these biomarkers are not typically used in the clinic but they do play a role in Oncology drug development. In this talk we shall compare the biomarkers/mathematical models in two Phase 1 Oncology trials, Rectal Carcinoma and metastatic Castrate Resistant Prostate Cancer, to those that are typically used in the clinic in the same settings. We shall highlight how the breadth and richness of data in Oncology drug development exceeds that in the clinic but that more complex mathematical models are used in the clinic versus drug development even though the question is the same - what dose/schedule should we use.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ONCO-MS01.html#author2"><strong class="card-body-subgroup">Renee Brady</strong></a>
		    	 
		    		<em> (H. Lee Moffitt Cancer Center and Research Institute, USA)</em>
		    	
		    	
		    		
		    			</br>"Predicting Response to Adaptive Therapy in Metastatic Prostate Cancer Using Prostate-Specific Antigen Dynamics"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Prostate cancer (PCa) remains the most prevalent cancer in men in the US. Standard treatment with androgen deprivation therapy (ADT) for localized disease often results in the competitive release of resistant cell phenotypes, causing patients to develop castration resistant PCa. Intermittent ADT has been shown to be a promising alternative to continuous treatment that can delay progression and may potentially reduce treatment-related adverse events. Second-line hormone therapy options, such as abiraterone acetate (AA), have been proven effective for metastatic castration resistant prostate cancer (mCRPC) and it has been proposed that similar to intermittent ADT, treatment with adaptive AA may reduce toxicity and prolong time to progression in mCRPC. We simulated and analyzed a simple quantitative model of prostate-specific antigen (PSA) dynamics to evaluate PCa stem cell enrichment as a plausible driver of treatment resistance. A Type 1b bootstrap internal validation leave-one-out analysis was used to calibrate and validate the model against longitudinal PSA data from 16 mCRPC patients receiving adaptive AA in a pilot study. Early PSA treatment response dynamics were then used to predict patient response to subsequent treatment. We extended the model to incorporate metastatic burden to improve predictive ability and also investigated the survival benefit of adding concurrent chemotherapy for patients predicted to become resistant. Model simulations demonstrated PCa stem cell self-renewal as a plausible driver of resistance to hormone therapy. The model was able to accurately describe patient-specific PSA dynamics and predict response with 78% accuracy. When incorporating metastatic burden, the predictive ability of the model increased to 81% (specificity = 92%, sensitivity = 50%). This study developed the first patient-specific mathematical model to use early treatment response dynamics to predict subsequent responses to adaptive AA.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ONCO-MS01.html#author3"><strong class="card-body-subgroup">Aleksandra Karolak</strong></a>
		    	 
		    		<em> (City of Hope Comprehensive Cancer Center, USA)</em>
		    	
		    	
		    		
		    			</br>"A Quantitative Systems Pharmacology Model to Improve Graft Versus Host Disease Outcomes"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Allogeneic hematopoietic cell transplant (HCT) cures patients of underlying disease by replacing their hematopoietic system with that of a healthy donor (non-malignant disease) or by the donor cells eradicating the patient’s malignancy (graft-versus-tumor effect). A post-transplant cyclophosphamide (PTCy) regimen was recently established as a standard of care for preventing graft versus host disease (GVHD), which is the most common cause of non-relapse mortality in HCT. The PTCy regimen consists of three drugs: cyclophosphamide (Cy), mycophenolate mofetil (MMF, active metabolite mycophenolic acid - MPA) and tacrolimus (TAC). All three drugs need to be optimized: PTCy has a narrow dose range based on preclinical data; clinical data in other GVHD regimens suggest that low plasma exposure to MPA and TAC are associated with GVHD. To address this need, we are constructing a Quantitative Systems Pharmacology (QSP) model to optimize the PTCy regimen. Guided by our preliminary preclinical and clinical data, our hypothesis is that QSP modeling can successfully predict immunologic reactions resulting from PTCy to subsequently: 1) simulate alternative doses and administration schedules for all three drugs; 2) identify the optimal PTCy dose and administration schedule; and 3) identify which model parameters introduce the greatest variability to design subsequent clinical trials and obtain more data to improve model reliability. In order to achieve these aims, we combine computational modeling with experimental data from HCT patients to develop and validate mathematical approach. The novelty of our approach comes from a joint application of population pharmacokinetic (popPK) model with the fully integrated immune response model (FIRM). The hybrid popPK-FIRM QSP model uses patient-specific metabolite data of PTCy drugs activity to guide dosing optimization. Simulations help predict pharmacokinetic characteristics of the PTCy regimen with correlation to drugs’ metabolites and evaluate the effects of implicit drug-drug interactions. The progress on implementations of the mathematical models, results of the simulations, and validation with the human samples collected at City of Hope will be presented.
			    		</div>
			    	</article>
		    	
		    	
		    
		    
		    
		    	<li><a href="/MS01/ONCO-MS01.html#author4"><strong class="card-body-subgroup">Kit Curtius</strong></a>
		    	 
		    		<em> (University of California San Diego, USA)</em>
		    	
		    	
		    		
		    			</br>"Predicting Risk of Progression to Advanced Neoplasia in Patients with Ulcerative Colitis"</li>
		    		
		    	
		    	
		    	
			    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
			    		<div class="hide-abstracts">
			    			Patients with ulcerative colitis (UC) have an increased risk of developing colorectal cancer and thus are advised to participate in regular surveillance to remove pre-cancers that may be detected during colonoscopy.  In order to translate a validated statistical model for predicting patient-specific risk of progression over time, we developed UC-CaRE (Ulcerative Colitis-Cancer Risk Estimator) as a tool that can be used to calculate and communicate individualized cancer risk estimates to UC patients with low-grade dysplasia based on their clinicopathological features. This visual aid facilitates the risk stratification of the lowest risk patients, who can be reassured to continue surveillance, versus those at the highest risk of cancer who may benefit from preventive surgery. Using shallow whole genome sequencing, we also found that the evolution of copy number alterations in UC predicts future neoplastic risk in patients. As molecular-based decision-making becomes more prominent in the clinical setting of early cancer detection, we propose that models of evolving genotypes can be integrated into and will enhance tools like UC-CaRE.
			    		</div>
			    	</article>
			    
		    	
	    	
	    	
	    	
	    	
	    	
	    	
  		</ul>
  		
  		
	    <button id="/session-recordings#ONCO-MS01" onclick="passWord(this.id)">Session video recording</button>

  		
		</p>
  </div>
</div>

												</div>
											</div>
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>


<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Minisymposia-01
										</br>
										Monday, June 14 at 09:30am (PDT)</br>Monday, June 14 at 05:30pm (BST)</br>Tuesday, June 15 01:30am (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>



















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
