<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Posters-01 | SMB 2021</title>

    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Posters-01 | SMB 2021</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Posters-01" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Society for Mathematical Biology - e-Conference 2021." />
<meta property="og:description" content="The Society for Mathematical Biology - e-Conference 2021." />
<link rel="canonical" href="http://localhost:4000/PS01/" />
<meta property="og:url" content="http://localhost:4000/PS01/" />
<meta property="og:site_name" content="SMB 2021" />
<script type="application/ld+json">
{"description":"The Society for Mathematical Biology - e-Conference 2021.","@type":"WebPage","url":"http://localhost:4000/PS01/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/images/logo.png"}},"headline":"Posters-01","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="shortcut icon" type="image/x-icon" href="/assets/images/favicon.ico">

    <!-- Font Awesome Icons -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">

    <!-- Google Fonts-->
    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700" rel="stylesheet">

		<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>


		<!-- Twitter card validator -->
		<meta name="twitter:card" content="summary_large_image">
	  <meta name="twitter:site" content="@smb2021">
		<meta name="twitter:creator" content="@smb2021">
		<meta name="twitter:title" content="Posters-01">
		<meta name="twitter:description" content="The Society for Mathematical Biology annual meeting - 2021.">
		<meta name="twitter:image" content="http://schedule.smb2021.org/assets/images/twitter.png">
		
		<meta name="title" content="Posters-01" />
		<meta name="description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:title" content="Posters-01" />
		<meta property="og:description" content="The Society for Mathematical Biology annual meeting - 2021." />
		<meta property="og:image" content="http://schedule.smb2021.org/assets/images/twitter.png"/>



    


    

		<!-- Bootstrap Modified -->
		
			<link rel="stylesheet" href="../assets/css/main.css">
		
		
		<!-- Theme Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/theme.css">
		
		
		<!-- Custom Themes Stylesheet -->
		
			<link rel="stylesheet" href="../assets/css/custom.css">
		



    <!-- Jquery on header to make sure everything works, the rest  of the scripts in footer for fast loading -->
    <script
    src="https://code.jquery.com/jquery-3.3.1.min.js"
    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
    crossorigin="anonymous"></script>
    
    <script src="/assets/js/custom.js"></script>


</head>

<body class="">
  
	<nav id="MagicMenu" class="topnav navbar navbar-expand-lg navbar-light bg-white fixed-top">
	  <div class="container">
		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		
		  <div class="collapse navbar-collapse" id="navbarColor02">
		    <ul class="navbar-nav mr-auto d-flex align-items-center">
			    <li class="nav-item"> <img src="/assets/images/logo.png" width="65" alt="" class="d-inline-block align-middle mr-2"> </li>
					<a class="navbar-brand" href="/"><strong>SMB 2021</strong></a>
		      <!--  Replace menu links here -->

<!-- <li class="nav-item"> <img src="/assets/images/logo.jpg" width="45" alt="" class="d-inline-block align-middle mr-2"> </li> -->
<!-- <a class="navbar-brand" href="/"><strong>SMB 2021</strong></a> -->

<li class="nav-item"><a class="nav-link" id="strong-nav" href="/">Home</a></li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Schedule
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/">Overview</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/your-smb2021">Plan Your SMB2021</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/pre-conference">Pre-conference</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/americas">Americas</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/europe-africa">Europe/Africa</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/asia-oceania">Asia/Oceania</a>
		
  </div>
</li>

<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Search by Name
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item" style="color:#000000 !important;" href="/contributed-speakers">Contributed Talks</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/minisymposia-speakers">Minisymposia</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/plenary">Plenary</a>
		<a class="dropdown-item" style="color:#000000 !important;" href="/posters">Posters</a>
  </div>
</li>


<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Subgroup / Theme
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		<a class="dropdown-item text-CBBS" style="color:#000000 !important;" href="/CBBS"><span class="text-CBBS">CBBS</span> (Complex Biological and Biomedical Systems)</a>
		<a class="dropdown-item text-CDEV" style="color:#000000 !important;" href="/CDEV"><span class="text-CDEV">CDEV</span> (Cell and Developmental Biology)</a>
		<a class="dropdown-item text-DDMB" style="color:#000000 !important;" href="/DDMB"><span class="text-DDMB">DDMB</span> (Data Driven Modeling in Biology)</a>
		<a class="dropdown-item text-DEI" style="color:#000000 !important;" href="/DEI"><span class="text-DEI">DEI</span> (Diversity, Equity, and Inclusion)</a>
		<a class="dropdown-item text-ECOP" style="color:#000000 !important;" href="/ECOP"><span class="text-ECOP">ECOP</span> (Population Dynamics & Ecology)</a>
		<a class="dropdown-item text-EDUC" style="color:#000000 !important;" href="/EDUC"><span class="text-EDUC">EDUC</span> (Education)</a>
		<a class="dropdown-item text-EVOP" style="color:#000000 !important;" href="/EVOP"><span class="text-EVOP">EVOP</span> (Population Dynamics & Evolution)</a>
		<a class="dropdown-item text-IMMU" style="color:#000000 !important;" href="/IMMU"><span class="text-IMMU">IMMU</span> (Immunobiology and Infection)</a>
		<a class="dropdown-item text-MEPI" style="color:#000000 !important;" href="/MEPI"><span class="text-MEPI">MEPI</span> (Mathematical Epidemiology)</a>
		<a class="dropdown-item text-MFBM" style="color:#000000 !important;" href="/MFBM"><span class="text-MFBM">MFBM</span> (Methods for Biological Modeling)</a>
		<a class="dropdown-item text-MMPB" style="color:#000000 !important;" href="/MMPB"><span class="text-MMPB">MMPB</span> (Multiscale Modeling in Physiology and Biophysics )</a>
		<a class="dropdown-item text-NEUR" style="color:#000000 !important;" href="/NEUR"><span class="text-NEUR">NEUR</span> (Mathematical Neuroscience)</a>
		<a class="dropdown-item text-ONCO" style="color:#000000 !important;" href="/ONCO"><span class="text-ONCO">ONCO</span> (Mathematical Oncology)</a>
  </div>
</li>
<li class="nav-item">
	<a class="nav-link" href="https://www.smb2021.org/">Return to SMB2021.org</a>
</li>

		    </ul>
		    
		  </div>
	  </div>
	</nav>
	


    

    <!-- Search Results -->
    <div id="lunrsearchresults">
        <ul class="mb-0"></ul>
    </div>

    <!-- Content -->
    <main role="main" class="site-content" style="margin-bottom: 300px;">
        <div class="container">
	
	<div class="col-md-12 main-loop">
    <div class="row">
      <div class="col-md-12  text-center">
<!--         <div class="row">
  <div class="col-md-12">
		<a href="https://www.smb2021.org/"><img class="w-100" src="/assets/images/logo-banner.jpeg" alt="SMB 2021 Logo"></a>        
  </div>
</div>
</br>
</br> -->
        <h1 class="mb-3">Posters-01</h1>
        <h4>Monday, June 14 at 11:30pm (PDT)</br>Tuesday, June 15 at 07:30am (BST)</br>Tuesday, June 15 03:30pm (KST)</h4>
      </div>
    </div>
	</div>
	
	

   	
		
	<div class="row mt-3">   
		<div class="col-md-12 main-loop">
			
			<div class="jumbotron jumbotron-fluid jumbotron-home pt-0 pb-0 mt-3 mb-2rem parallel position-relative">
		    <div class="pl-4 pr-0 h-100 tofront">
		        <div class="row justify-content-between">
		            <div class="col-md-12 pt-6 pb-6 pr-lg-4 align-self-center">
			            
			            							
									  
									 	
			            
		                <h1 class="mb-3" id="PS01">Posters-01</h1>
		                
		                										
										
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-1 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-1.html">
											    	</br><strong>Parkrati Dangarh</strong>
									    	 
									    		<em>Imperial College London</em>
									    	
									    	
									    		
									    			</br>"Systems-level modeling of meiotic entry, commitment, progression and exit"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Upon nitrogen starvation, Schizosaccharomyces pombe (fission yeast) exits the mitotic cell cycle and becomes irreversibly committed to the completion of the meiosis program. In meiosis, DNA replication (S-phase) is followed by two successive rounds of cell divisions (Meiosis I and Meiosis II) without intermediate interphase. In this work, we developed a comprehensive model of the entire meiotic cell cycle, which couples exit from mitosis to meiotic commitment and progression under nitrogen starvation. This network was assembled from several experimental observations in the literature for meiotic cell divisions and exit. The core of the regulatory network is the regulation of cyclin-dependent kinase (Cdk) 1 and anaphase-promoting complex or cyclosome (APC/C) by meiosis-specific factors. The network was translated into a set of ordinary differential equations to simulate the dynamics of meiotic progression. We also performed one and two-parameter bifurcations to study the role of different feedback loops in meiosis. The model accounts for about 60 experimental situations including single and multiple mutations and demonstrates the control strategy involving multiple feedback loops to yield two successive division cycles. The model serves as a key tool for experimentalists to perform in silico mutations and test the hypothesis.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-10 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-10.html">
											    	</br><strong>Atchuta Srinivas Duddu</strong>
									    	 
									    		<em>Indian Institute of Science</em>
									    	
									    	
									    		
									    			</br>"Modelling epigenetic feedback in gene regulatory network consisting of three mutually inhibiting transcription factors"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Regulatory biochemical networks demonstrate different levels of control - transcriptional, translational, protein activity and epigenetic. Toggle triad is a network motif consisting of three mutually repressing transcription factors each driving a distinct cell fate. We have previously demonstrated underlying multistability in a toggle triad: it can give rise to six states in total, three of which are 'single positive' (A high, B low, C low;  A low, B high, C low; and A low, B low, C high) and three 'double positive' (A high, B high, C low; A high, B low, C high; and A low, B high, C high), which can be mapped on to Th1, Th2, Th17 and hybrid Th1/Th2, Th1/Th17 and Th2/Th17 states in CD4+ T cell differentiation. Here, we incorporated epigenetic feedback mechanism into the previously developed gene regulatory model for a toggle triad to study the stability of the various phenotypes of the motif identified previously and investigate the dynamics of irreversibility of cell-fate decisions as a function of epigenetic feedback incorporated on different links in the network. Our results offer insights into cell-fate decision-making vs. cell-fate commitment for a multistable network and helps understand plasticity and stability of different T-helper states.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-2 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-2.html">
											    	</br><strong>Takayuki Ohara</strong>
									    	 
									    		<em>Leibniz Institute for Farm Animal Biology</em>
									    	
									    	
									    		
									    			</br>"Mathematical Modeling of Rhythmic Gene Expression: Impact of Negative Autoregulation on Amplitude Preservation"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The mammalian circadian clock is an endogenous biochemical oscillator that generates rhythmicity with a period of about 24 hours in the expression of numerous clock-controlled genes (CCGs) mainly by controlling their transcription or mRNA stability. There are hurdles for propagating high amplitude oscillations from the circadian clock to CCG expression. Long molecular half-lives decrease relative oscillation amplitudes, and half-lives of proteins are, indeed, often long enough to significantly reduce amplitude. A question, then, arises; how does the gene expression process overcome the amplitude-dampening effect to retain strong rhythmicity?Here we theoretically investigate negative autoregulation as a possible scenario for propagation of strong circadian rhythmicity. We considered a CCG coding for, e.g., a transcription factor that undergoes post-translational modifications and represses its own expression. We studied mathematical models with or without the negative autoregulation, which were formulated in terms of parameters directly observable in omics scale data. Our analyses show that amplitudes can be strongly propagated with negative autoregulation, overcoming limits due to long half-lives. Moreover, when modification steps were increased, reliable and precise rhythm propagation, defying random cell-to-cell variation in rates and lifetimes, was readily achievable. Our results are general enough to be applicable to a variety of oscillation phenomena.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-3 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-3.html">
											    	</br><strong>Masayuki Kashiwa</strong>
									    	 
									    		<em>Akita Prefectural University</em>
									    	
									    	
									    		
									    			</br>"Visualization of cell flow by cell vertex and bubbly cell shape tracking"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			For clinical applications, researchers have paid attention to biological tissues and their multicellular mechanical structures. However, the mechanical aspect of morphological mechanisms remain unclear. So far, mathematical models have been developed to elucidate the mechanism: the vertex model (VM) by polygon approximation of cells, and the bubbly vertex model (BVM) with the curvature of cell boundaries, and so on. None of them leads to the basic equations for the migration and deformation of cell populations yet. The main reasons are that the physical properties of tissues differ among the morphogenetic stages, the cell boundary tensions are non-uniform, and the stress-strain relationship has not been clarified.In order to determine the physical properties for such basic equations, precise quantification of cell flow is necessary. However, conventional methods, such as PIV and PTV, do not fit naively to various cellular events: deformation, division, apoptosis, rearrangement, etc. Live-imaging techniques also limit the quality of experimental data.In this paper, we extract cell boundaries from the live data, fit them to the tissue shape defined by BVM, and perform vertex and edge (cell boundary) tracking similar to PTV. We also attempt to visualize and quantitatively evaluate the cell flow from real data.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-4 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-4.html">
											    	</br><strong>Sathvik Sanjeev Buggana</strong>
									    	 
									    		<em>CCNSB, IIIT Hyderabad</em>
									    	
									    	
									    		
									    			</br>"Modeling the liver circadian clock control by nutrients"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Circadian rhythms are 24-hour cycles in physiological processes. On a molecular level, the circadian clock is regulated by transcriptional/translational feedback loops. Experiments in recent years on mammalian circadian clocks have shown that external factors such as diet, nutrients and even blood gas concentrations have effects on circadian period, amplitude, and phase. Molecular players linked to these external factors have been found to regulate and be regulated by the circadian clock. In this work, we have developed a mathematical model to study the effect of high fat diet and nutrients on the liver circadian clock. The model interlinks feeding and fasting cycle and circadian clock and provide insights into the regulation by external factors.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-5 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-5.html">
											    	</br><strong>David Versluis</strong>
									    	 
									    		<em>Leiden University, Leiden, the Netherlands</em>
									    	
									    	
									    		
									    			</br>"How Oxygen and Lactose Metabolism Shape the Infant Gut Microbiota"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Nearly immediately after birth, a complex and dynamic ecosystem forms in the human infant gut. The characteristics of this system influence the infant's health in both the short and long term. 2'-FL, the most prevalent prebiotic in most human milk, varies greatly in presence and concentration between individuals. We use a multi-scale spatiotemporal model of the infant colon from birth to three weeks of age to reproduce the effects of variations in  nutritional components on the composition and metabolic activity of the microbiota. Using flux balance analysis with molecular crowding on genome-scale metabolic models from the AGORA project, we calculate bacterial fluxes for different locations and time points at a high resolution. The resulting fluxes are integrated together into a model of the ecosystem that feeds back into the flux calculations. The model can give insight and produce predictions for bacterial and metabolic composition of the infant microbiota over time and under different conditions. Our aim is to reach a deeper understanding of the influence that nutrition can have on the development of the infant microbiota. This in turn is the first step towards a comprehensive understanding of the formation of a steady state adult microbial environment.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-6 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-6.html">
											    	</br><strong>Elena Pascual Garcia</strong>
									    	 
									    		<em>Leiden University</em>
									    	
									    	
									    		
									    			</br>"Introducing the nucleus into the Cellular Potts Model: a multiscale approach for cell migration studies"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Cell migration is fundamental in multicellular organisms. It is involved, e.g., in early development, in maintenance of tissue homeostasis, in the functioning in the immune system. Importantly, besides cytoplasmic deformability, the nucleus stiffness can limit cellular migration through channels in dense environments. In order to adequately model this process, recent work has proposed compartmental Cellular Potts Models, in which one CPM compartment represents the cytoplasm, and a second, stiffer compartment is embedded within the cytoplasmic compartment to model the nucleus. Here we show that the increased stiffness of the nucleus introduces an artificial friction with the lattice.  We provide quantitative data showing that  the increased  nuclear stiffness can visibly slow down the cytoplasmic movement, thus producing an artifact in speed quantification. We present an alternative approach to model the nucleus that addresses this issue: a node-based CPM-independent layer, which maintains the deformability characteristics, but does not directly influence the cytoplasmic movement.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-7 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-7.html">
											    	</br><strong>Emine Atici Endes</strong>
									    	 
									    		<em>Heriot-Watt University</em>
									    	
									    	
									    		
									    			</br>"Keratinocyte Growth Factor-Based Mathematical Models of Epidermal Wound Healing"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The mammalian skin is the largest organ of the body after the skeletal system and its primary function is to protect the body against the external environment. The maintenance of the skin's functionality, integrity, and strength are coordinated by specialised cells localised in three intricate layers of the skin: epidermis, dermis, and hypodermis. Understanding the fundamental cause of changes in the cells in these three skin compartments, thus provides a base for progress in interpreting injuries in the skin and its healing process.Wound healing is a normal biological and dynamic process in the human skin in which many types of cells, various cytokines, and growth factors, such as KGF, act in harmony. This process starts immediately after skin injury and is completed into three distinct but highly integrated phases: an inflammatory stage, a proliferative and migration  (re-epithelialization) phase, and a long reformation and remodeling phase. Therefore, in this talk, I will give a sequence of PDE models presenting the interaction between the dermis and the epidermis, in which KGF plays an active role during the re-epithelialization.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-8 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-8.html">
											    	</br><strong>Voorsluijs Valerie</strong>
									    	 
									    		<em>Luxembourg Centre for Systems Biomedicine, University of Luxembourg</em>
									    	
									    	
									    		
									    			</br>"Energetic cost of the cross-talk between calcium dynamics and mitochondrial metabolism"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Ion exchanges across mitochondrial membrane play a crucial role for energy metabolism [1–2]. In this work, we focus on the interplay between Ca2+ dynamics and mitochondrial metabolism. Ca2+ activates different enzymes of the TCA cycle and thereby enhances ATP production, while Ca2+ exchanges via SERCA and PMCA pumps are consuming ATP. The kinetics of this coupling has been studied with mathematical modelling [3–5], but its concrete energetic cost remains elusive. Here, we investigate this cost computationally by means of the entropy production rate (EPR) of Ca2+ exchanges and major mitochondrial metabolic processes. We show that the EPR as a measure for the throughput of the Gibbs free enthalpy of the TCA cycle exhibits a maximum in dependence on the glucose level, which could represent an optimal working regime of cells.  1. Wu, F. et al. J. Biol. Chem. 2007, 282, 24525–24537. 2. Wei, A.-C. et al. Biophys. J. 2011, 100, 2894–2903. 3. Magnus, G.; Keizer, J. Am. J. Physiol. 1998, 274, C1158-1173. 4. Bertram, R. et al. A. J. Theor. Biol. 2006, 243, 575–586.  5. Wacquier, B. et al. Sci. Rep. 2016, 6, 19316.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-CDEV">CDEV</h5> </strong>CDEV-9 (Session: PS01)</small>
										    	<a href="/CDEV/CDEV-PS01-CDEV-9.html">
											    	</br><strong>Md Hamidul Islam</strong>
									    	 
									    		<em>Lecturer, Department of Applied Mathematics, University of Rajshahi, Rajshahi-6205, Bangladesh</em>
									    	
									    	
									    		
									    			</br>"Modelling the Host Immune Response to Primary Dengue Infection"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Dengue is a mosquito borne viral infection triggering a series of intracellular events in the host immune system, which sometimes leads to severe dengue infection resulting in serious illness and even in death if the patient is not treated properly. We present stochastic model describing the interplay between dengue virus and host immune response in primary dengue infection. The stochastic model is derived from the deterministic model describing the dynamics of the disease. We analyze the deterministic model to explore the factors influencing the virus persistence in the body for extended periods. The results are then compared with the stochastic model. The stochastic model is shown to provide better insights into the viral dynamics. The stochastic model provides a wide range of results including different size of viral loads and different time of maximum infection occurring in the body. In addition, the stochastic model exhibits positive probability of viral extinction, as opposed to deterministic model, when the virus reproduction number R_0>1. We calculate the extinction probability as a function of R_0 where extinction probability is found to decrease with an increasing R_0, suggesting that at high infection rate the effect of uncertainties in underlying dengue dynamics maybe negligible.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-EDUC">EDUC</h5> </strong>EDUC-1 (Session: PS01)</small>
										    	<a href="/EDUC/EDUC-PS01-EDUC-1.html">
											    	</br><strong>Keith Harris</strong>
									    	 
									    		<em>Hebrew University of Jerusalem</em>
									    	
									    	
									    		
									    			</br>"Biomaton: a platform for the visualisation, analysis and computation of biological models"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Biological simulations with large parametric spaces can require immense computational power and complex visualizations to provide comprehensive model analyses. As a result, published simulation results often cover only a small subset of model parameters. While the release of the simulation code is becoming a publication requirement, there are no coding standards for simulations (in terms of programming language or framework). This can represent a significant hurdle to validating model results and further exploration of the parametric space of the model by other researchers, or the wider public. We developed a platform, called ‘biomaton’, for the distribution, computation and visualization of a wide range of biological models. Biomaton is cloud-based and allows users to explore and analyze, through a simple user interface, the parametric spaces of published models uploaded to the platform. Biomaton supports multiple programming languages, and can be extended to support virtually any visualization type. We foresee the platform as a means to make mathematical and computational biological models more accessible to researchers, as well as to be used as an educational tool providing accessible visualizations of models dynamics and results.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-1 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-1.html">
											    	</br><strong>Michael Eguia Subido</strong>
									    	 
									    		<em>University of the Philippines, Diliman</em>
									    	
									    	
									    		
									    			</br>"Assessing the Effect of Temperature on Multi-Strain Dengue Transmission Dynamics in the Philippines"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Dengue is one of the most important infectious diseases with more than 55% of the world population at risk of acquiring the infection. Recent climate changes related to global warming have increased the potential risk of dengue outbreaks in the world. In this paper, we propose an SEIR model for the human population and an SEI model for the vector population by incorporating temperature-dependent parameters to describe the transmission dynamics of a multi-strain dengue model. Sensitivity analysis of both the constant and temperature-dependent parameters are performed to explore the effects of the changes in temperature on the multi-strain dengue transmission dynamics. The adapted model will then be investigated to describe the dengue epidemics that occurred in the Philippines in the year 2015-2018 using Philippine epidemiological and climatological data. We then test the identifiability of the proposed multi-strain dengue model using the reported dengue cases by morbidity week in the Philippines for the same year.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-11 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-11.html">
											    	</br><strong>Peter Rashkov</strong>
									    	 
									    		<em>Institute of mathematics and informatics, Bulgarian Academy of Sciences</em>
									    	
									    	
									    		
									    			</br>"Viable controls in models for vector-borne diseases"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Analysis of transient dynamic behaviour of controlled trajectories is a novel problem in the context of vector-borne diseases. Epidemiological modelling focuses often on investigation of local or global asymptotic stability of equilibria or on trajectories corresponding to optimal resource allocation if control is introduced. The study is motivated by the application of mosquito repellents as protective measure in textiles, paints and other household items. The model for a vector-borne disease is SIR for the host and SI for the vector with time-dependent controls. We determine the viability kernel comprising those initial states for the dynamical system such that the proportion of infected individuals is kept below a certain maximum level for all future times, and the respective viable trajectories. Analysis of viable controls has been done earlier for a SIS model for the host (DeLara & Salcedo 2016), which has properties of a quasi-monotone system. Our results (Rashkov 2021) extend the analysis to a more complex model system. We compute numerical approximations of the viability kernels and the viable trajectories using a variational framework.This work is partially supported by the Bulgarian National Science Fund within the National Science Program 'Petar Beron i NIE' [contract number KP-06-DB-5].
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-12 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-12.html">
											    	</br><strong>Dimitris A. Goussis</strong>
									    	 
									    		<em>Khalifa University</em>
									    	
									    	
									    		
									    			</br>"Time-scale analysis of population dynamics models for the COVID-19 pandemic"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The identification of the various factors influencing the spread of the COVID-19 outbreak, especially during the early stages of the pandemics, is critical to determine appropriate interventions to control the outbreak and prevent its resurgence. In this regard, it is demonstrated here that the time scale characterizing various phases of the COVID-19 outbreak provides most useful information. The analysis is based on a number of popular population dynamics models and data from various countries. It is further demonstrated that this characteristic time scale is robust, when considering (i) different population models, (ii) fitting the parameters of a model to data spanning different periods of the growth phase and (iii) different parameters sets resulting from different fittings of the same data sets. This time scale characterizes the largest portion of the epidemic-growth period and is promoted by the infecting paths of the models and is opposed by the recovery ones. This approach provides a robust and systematic framework for the assessment of the control measures of the COVID-19 outbreak.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-13 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-13.html">
											    	</br><strong>María Gamboa Pérez</strong>
									    	 
									    		<em>Complutense University of Madrid</em>
									    	
									    	
									    		
									    			</br>"Measures to asses an optimal vaccination coverage in a stochastic SIV model with imperfect vaccine"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			This communication is framed within the area of epidemic modelling in a stochastic approach. An additional compartment of vaccinated individuals is considered in a stochastic SIS model within a not isolated, homogeneous, and uniformly mixed population. The vaccine is not 100% effective and individuals are partially protected against the disease.  The propagation of a contagious disease is modelled in terms of a continuous time Markov chain where individuals evolve among susceptible, S, vaccinated, V, and infected, I; compartments.A well-known measure of the initial transmission potential is the basic reproduction number R_0, which determines the herd immunity threshold or the critical proportion of immune individuals required to stop the spread of a disease when a vaccine offers a complete protection. Assuming that the vaccine is imperfect, alternative measures to R_0 are defined in order to study the influence of the initial coverage on the transmission of the epidemic. The talk is based on the paper: Gamboa, M., and Lopez-Herrero, M. J. (2020). Measuring infection transmission in a stochastic SIV model with infection reintroduction and imperfect vaccine. Acta biotheoretica, 1-26.https://doi.org/10.1007/s10441-019-09373-9.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-14 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-14.html">
											    	</br><strong>Salisu Garba</strong>
									    	 
									    		<em>Department fo Mathematics and Applied Mathematics, University of Pretoria</em>
									    	
									    	
									    		
									    			</br>"Modeling the transmission dynamics of yellow fever with optimal control"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			In this presentation, a model for yellow fever transmission dynamics in a human-mosquito setting is constructed and analyzed. The model incorporates vertical transmission within mosquito population. Threshold quantities (such as the basic offspring and the type reproduction numbers) and their interpretations for the models are presented. Analysis of the mosquito-only component shows that the reduced model has a mosquito-extinction equilibrium, which is globally-asymptotically stable whenever the basic offspring number is less than unity. Optimal control theory is applied to the model to characterize the controls parameters. Using Pontryagin's maximum principle and modified forward-backward sweep technique, the necessary conditions for existence of solutions to the optimal control problem is determined. The effect of various control strategies (bed nets, adulticides and vaccination) were assess via numerical simulations.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-15 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-15.html">
											    	</br><strong>Christopher Overton</strong>
									    	 
									    		<em>University of Manchester</em>
									    	
									    	
									    		
									    			</br>"Data driven compartmental modelling of the COVID-19 hospital burden in England"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			The COVID-19 pandemic in England has put considerable strain on the national healthcare system. To predict the effect of the pandemic on hospital capacity in England, we used a variety of data streams to inform the construction and parameterisation of a hospital progression model, which was coupled to a model of the generalised epidemic. Data from a partially complete patient-pathway line-list was used to provide initial estimates of the mean duration that individuals spend in the different hospital compartments. We then fitted the model using complete data on hospital occupancy and hospital deaths, enabling estimation of the proportion of individuals that follow different clinical pathways, and the reproduction number (Rt) of the generalised epidemic. The construction of the model makes it straightforward to adapt to different patient pathways and settings beyond England. As part of the UK response to the pandemic, this model has provided weekly forecasts to the NHS for hospital bed occupancy in England, Wales, Scotland and Northern Ireland, and formed part of the UK combined reproduction number estimates.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-3 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-3.html">
											    	</br><strong>Elba Raimúndez</strong>
									    	 
									    		<em>University of Bonn</em>
									    	
									    	
									    		
									    			</br>"COVID-19 outbreak in Wuhan demonstrates the limitations of publicly available case numbers for epidemiological modeling"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Mathematical models are standard tools for understanding the underlying mechanisms of biological systems. Generally, the parameters of these models are unknown and they need to be inferred from experimental data using statistical methods. Most common measurement techniques only provide relative information about the absolute molecular state and often data is noise-corrupted. Therefore, introducing scaling and noise parameters in the model observables is necessary. Since frequently these parameters are also unknown, the dimensionality of the estimation problem is augmented. Sampling methods are widely used in systems biology to assess parameter and prediction uncertainties. However, the evaluation of sampling methods is usually demanding and often on the border of computational feasibility. Hence, efficient sampling algorithms are required.We propose a marginal sampling scheme for estimating the parameter uncertainties of mechanistic models with relative data. We integrate out the scaling and noise parameters from the original problem, leading to a dimension reduction of the parameter space. Herewith, only reaction rate constants have to be sampled. We find that the marginal sampling scheme retrieves the same parameter probability distributions and outperforms sampling on the full parameter space by substantially increasing the effective sample size and smoothing the transition probability between posterior modes.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-4 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-4.html">
											    	</br><strong>Uljana Apel</strong>
									    	 
									    		<em>Technical University of Munich</em>
									    	
									    	
									    		
									    			</br>"How long is long enough? The impact of the contact tracing interval"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Contact tracing is one of the most important non-pharmaceutical control measures for infectious diseases. The corona epidemic revealed that many aspects of contact tracing are still not well understood, from the theoretical as well as from the practical point of view.In the present work, we focus on the tracing interval. The tracing interval is the time interval for which people are asked to give their contacts. The current practical guidelines mainly orient themselves at the infectious interval. In our research we vary this tracing interval and examine its effect on the spread of the disease in the onset. Thereto we setup a differential-difference equation model for the probability to be infectious at a certain age since infection when contact tracing is used. The probability to be infectious is then used to calculate the incidence depending on the tracing interval. Therewith we discuss the impact of contact tracing in dependence on the tracing interval.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-5 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-5.html">
											    	</br><strong>Sheryl Grace Buenaventura</strong>
									    	 
									    		<em>Center for Applied Modeling, Data Analytics, and Bioinformatics for Decision-Support Systems in Health (AMDABIDS) - University of the Philippines Mindanao</em>
									    	
									    	
									    		
									    			</br>"Understanding COVID-19 spread in the National Capital Region, Philippines using Genomic Sequences: A Phylodynamic Investigation"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			To understand the disease dynamics in a particular location, incidence reports are used to estimate key epidemiological parameters such as transmission rates and reproductive numbers. However, incidence data often suffer from underreporting due to logistical concerns in disease surveillance, insufficient testing, etc. One way to address this concern is to use information from viruses' genomes to infer the past ecological dynamics of the disease.  Here, we use publicly available SARS-CoV-2 genomic sequences data from the National Capital Region of the Philippines. We use the BEAST2 software to model its dynamics using a Birth-Death Susceptible-Infected-Recovered (BDSIR) model and infer its transmission history. Nineteen whole-genome sequences from NCR, sampled from 3 April to 18 July 2020, were used. We also model the spread of COVID-19 using incidence data through a deterministic ODE-based SIR model. The estimated transmission rate using the genomic sequences is 4.2x10-6 which is greater than the estimated transmission rate using the incidence reports at 2.0x10-8. The estimated basic reproduction number of the BDSIR (2.21) is also higher than that of the SIR (1.21). These results point out the need to cautiously use the reported incidences as basis in making policies in managing infectious diseases outbreak.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-6 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-6.html">
											    	</br><strong>Tatiana Filonets</strong>
									    	 
									    		<em>National Taiwan University, Taipei, Taiwan</em>
									    	
									    	
									    		
									    			</br>"Simulation of the first COVID-19 wave in Taiwan by using new epidemiological compartmental model"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As of March 10, 2021, there were 978 officially confirmed cases of COVID-19 in Taiwan, of which 862 were imported from outside of the country. Taiwan very quickly implemented non-pharmaceutical interventions, namely rapid case finding, hospital isolation, tracing and testing the contacts of infected individuals, central distribution of masks, and imposing home-quarantine on travelers from COVID-19 affected countries. In this work, we simulated the Taiwanese scenario of the first wave of COVID-19 January-March) in order to investigate the effectiveness of the public medical masks wearing and contact tracing to maintain the pandemic at a manageable level. For this purpose, we used a modified version of the SEIR model which takes into account asymptomatic cases, contact tracing, self-isolation, and masks wearing. In addition, we estimated the basic reproduction number and its dependence on the model parameters by using the next-generation matrix approach.Our results show that good realization of contact tracing program, the fast isolation, together with medical masks wearing by 90% of the population can help to control the local virus spreading. However, only high-quality implementation of these interventions can provide the basic reproduction number value below one.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-7 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-7.html">
											    	</br><strong>Vizda Anam</strong>
									    	 
									    		<em>Basque Center for Applied Mathematics</em>
									    	
									    	
									    		
									    			</br>"Modeling dengue immune responses mediated by antibodies: insight on the immunopathogenesis of severe disease"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Dengue fever is a viral mosquito-borne infection, a major international public health concern. With four antigenically related but distinct viruses (DENV-1 to DENV-4), the occurrence of the virus as four distinct serotypes results in many complications in disease response. Infection with one serotype results in lifelong protective immunity. Additionally, antibodies generated by exposure to any one type cross-react with other types, providing short duration cross protective immunity. Subsequent infection by a different dengue serotype increases the risk of developing severe disease with a high fatality rate. This disease augmentation phenomenon is called antibody-dependent enhancement (ADE).Here we present a minimalistic mathematical model developed to describe the dengue immunological response mediated by antibodies. Based on body cells and free virus interactions resulting infected cells activating antibody production, we explore the feature of ADE when pre-existing antibodies, analyzing: i) primary dengue infection, ii) secondary dengue infection with the same virus and iii) secondary dengue infection with a different dengue virus. Our mathematical results are qualitatively similar to the ones described in the empiric immunology literature and this framework will now be refined to be validated with the available laboratory data.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-8 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-8.html">
											    	</br><strong>Hannah Lepper</strong>
									    	 
									    		<em>University of Edinburgh</em>
									    	
									    	
									    		
									    			</br>"Integrating sewage and hospital-based surveillance data on antimicrobial resistance: resistance type affects community resistance patterns"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Using waste water to detect and quantify abundances of antibiotic resistance genes has the potential to improve our understanding of resistance in the community and study the relationship with resistance in hospitals. By investigating similarities and differences in patterns and drivers of resistance in hospital and sewage surveillance data, and how this differs between resistance types, we can gain insights in this relationship.Here we use a multivariate regression model to investigate correlations between sewage and hospital data, and the effects of antimicrobial usage on hospital and community resistance levels. A Poisson model for resistance gene abundance in waste water (Global Sewage Surveillance Project) and a binomial model for clinical isolate resistance testing (EARS-Net, ECDC) are combined through country-level covariance between the datasets.Our results show that fluoroquinolone resistance was positively associated with antimicrobial consumption (ESAC-Net, ECDC) in both the hospital and the community, whereas carbapenem resistance was not. After taking antimicrobial consumption into account, resistance to fluoroquinolones in hospitals and waste water was not correlated, but carbapenem resistance was. This indicates that emergence and transmission of different types of resistance have different drivers in hospitals and the community, and highlights the need for flexible approaches to surveillance and prevention.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-MEPI">MEPI</h5> </strong>MEPI-9 (Session: PS01)</small>
										    	<a href="/MEPI/MEPI-PS01-MEPI-9.html">
											    	</br><strong>Woldegebriel Assefa Woldegerima</strong>
									    	 
									    		<em>Postdoc Research fellow; University of Pretoria, South Africa</em>
									    	
									    	
									    		
									    			</br>"Mathematical assessment of the impact of human-antibodies during  the   within-mosquito dynamics of Plasmodium falciparum parasites"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			We develop and analyze  a model for the within-mosquito dynamics of the Plasmodium falciparum malaria parasite. Our model takes into account the action and effect of blood resident human-antibodies, ingested by the mosquito during a blood meal from humans, in inhibiting gamete fertilization. The model also captures subsequent developmental processes that lead to the different forms of the parasite within the mosquito. Continuous functions are used to model the switching transition from oocyst to sporozoites as well as human antibody density variations within the  mosquito gut are proposed and used.  We quantify the average sporozoite load produced at the end of the within-mosquito malaria parasite's developmental stages. Our analysis shows that an increase in the efficiency of the ingested human antibodies in inhibiting fertilization within the mosquito's gut results in lowering the density of oocysts and hence sporozoites that are eventually produced by each mosquito vector. So, it is possible to control and limit oocysts development and hence sporozoites development within a mosquito by boosting the efficiency of antibodies as a pathway to the development of transmission-blocking vaccines which could potentially reduce oocysts prevalence among mosquitoes and hence reduce the transmission potential from mosquitoes to human.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-1 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-1.html">
											    	</br><strong>Elias Siguenza</strong>
									    	 
									    		<em>University of Birmingham</em>
									    	
									    	
									    		
									    			</br>"Feeding the Habit: The metabolic relationship between bone marrow mesenchymal stems cells and multiple myeloma"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Multiple myeloma (MM) is an incurable malignant disease of plasma cells with the poorest 5-year survival of any haematological malignancy. Bone marrow (BM) residency of malignant plasma cells is an absolute requirement for their survival and proliferation, suggesting that the microenvironment within this niche is a critical driver of disease. We previously showed that the metabolism of the BM is significantly altered in patients with MM, and that the BM mesenchymal stem cell (BMMSC), the major supportive cell type for malignant plasma cells, was significantly and irreversibly transformed. We hypothesise that these two cell types form a co-operative metabolic network within the BM that is critical for the survival and proliferation of malignant plasma cells. If true, then targeting this metabolic communication will directly impact on disease progression and response to therapy, improving patient outcomes. We created a testable model of the metabolic network formed by malignant plasma cells and BMMSCs. Our model will identify enzymes or transporters that represent hubs, the inhibition of which would result in a breakdown of the community and sensitisation to standard therapeutic approaches to treating this incurable cancer.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-10 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-10.html">
											    	</br><strong>Arran Hodgkinson</strong>
									    	 
									    		<em>University of Exeter</em>
									    	
									    	
									    		
									    			</br>"Population Scale Spatio-Structural Modelling of Directed Cancer Invasion"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			As treatments for cancer continue to elude the biomedical community, and although it is well documented that cancer cells exert significant forces to dynamically rearrange the extra-cellular matrix (ECM), there remains a need for tumour, or population, scale mathematical models fit for spatial comparison to in vivo data. Employing a spatio-structural partial differential equation (PDE) framework, we are able to model the tissue scale dynamics resulting from tensile forces exerted by the cell population upon the ECM and the subsequent invasion of cells into their local environments. We also develop qualitative methods to theoretically explore the effects of alignment between cell polarisation and ECM fibre orientation on the invasive displacement of cancer sub-clusters. Numerical results show the multi-dimensional capacity of cells to reorient the fibrous ECM environment and invade the local tissue, where initial conflict between the cellular polarisation and fibre alignment impedes this process. The model also demonstrates the emergent phenomenon of structural heterogeneity from near-homogeneous initial conditions. This modelling framework provides a novel opportunity for the quantitative exploration of the biochemical and spatial processes of cancer invasion, whilst the resulting images provide an interesting candidate for comparison with robust experimental results.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-11 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-11.html">
											    	</br><strong>Ghanendra Singh</strong>
									    	 
									    		<em>Indraprastha Institute of Information Technology Delhi</em>
									    	
									    	
									    		
									    			</br>"Accelerate Replication Fork velocity to kill cancer cells"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Replication fork plays an important role during DNA replication. During DNA replication Fork movement rate increases during S phase in mammalian cells and also reduces during replication stress. There exists a molecular mechanism through which the replication fork adjusts their speeds during the S phase. In cancer cells, DNA replication is slower compared to normal cells and replication forks move slowly. So, this work proposes that if the replication fork speed can cross a particular threshold, the cancer cells won't be able to cope with the DNA replication and die in the process. A mechanistic mathematical model has been developed based on recent experimental findings. Mec1, Rad53 and Mrc1 are needed to create a positive feedback loop to stabilize replisome during stalled forks.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-2 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-2.html">
											    	</br><strong>Subbalakshmi A R</strong>
									    	 
									    		<em>Indian Institute of Science</em>
									    	
									    	
									    		
									    			</br>"A computational systems biology approach identifies SLUG as a mediator of partial Epithelial-Mesenchymal Transition (EMT)"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Epithelial-mesenchymal plasticity comprises of reversible transitions among epithelial, hybrid epithelial/mesenchymal (E/M) and mesenchymal phenotypes, and underlies various aspects of aggressive tumor progression such as metastasis, therapy resistance and immune evasion. The process of cells attaining one or more hybrid E/M phenotypes is termed as partial EMT. Cells in hybrid E/M phenotype(s) can be more aggressive than those in either fully epithelial or mesenchymal state. Thus, identifying regulators of hybrid E/M phenotypes is essential to decipher the rheostats of phenotypic plasticity and consequent accelerators of metastasis. Here, using a computational systems biology approach, we demonstrate that SLUG (SNAIL2) – an EMT-inducing transcription factor – can inhibit cells from undergoing a complete EMT and thus stabilizing them in hybrid E/M phenotype(s). It expands the parametric range enabling the existence of a hybrid E/M phenotype, thereby behaving as a phenotypic stability factor (PSF). Our simulations suggest that this specific property of SLUG emerges from the topology of the regulatory network it forms with other key regulators of epithelial-mesenchymal plasticity. Clinical data suggests that SLUG associates with worse patient prognosis across multiple carcinomas. Together, our results indicate that SLUG can stabilize hybrid E/M phenotype(s).
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-3 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-3.html">
											    	</br><strong>Ielyaas Cloete</strong>
									    	 
									    		<em>Brighton & Sussex Medical School</em>
									    	
									    	
									    		
									    			</br>"Tackling mutational heterogeneity in DLBCL through mathematical modelling"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Diffuse large B cell lymphoma (DLBCL) is the most common subtype of non-Hodgkin lymphoma, accounting for nearly 40% of diagnosed non-Hodgkin lymphomas. DLBCL is clinically is further classified, based on putative cell-of-origin, into activated B cell-like (ABC) and germinal centre B cell-like (GCB) subtypes, each with distinct gene expression profiles and clinical outcomes. However, a recent genetic and molecular study of patient samples reveals substantial heterogeneity beyond these classifications. Despite an increasing understanding of the signalling dysregulation leading to DLBCL, standard combination therapy (R-CHOP) has remained unchanged for more than a decade. However, roughly 40% of patients treated with R-CHOP develop a recurring or progressive disease that is usually fatal. The substantial heterogeneity likely leads to our standard treatment failing for many. Thus we leverage this mutational characterisation to recreate, using a mathematical model, genetic events seen in distinct patient samples to determine B cell fates given each genetic alteration. In particular, we are interested in modelling the mutational heterogeneity to create `virtual DLBCL patients' and use this model as a tool to identify molecular targets and biomarkers that cluster patients based on their optimal target for therapy.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-4 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-4.html">
											    	</br><strong>Marc Vaisband</strong>
									    	 
									    		<em>University of Bonn</em>
									    	
									    	
									    		
									    			</br>"Validation of genetic variants from NGS data using Deep Convolutional Neural Networks"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			A crucial aspect of analysing next-generation sequencing (NGS) data from cancer patients lies in identifying mutations in the genetic code of tumor cells. This is done by considering the tumor DNA together with a reference germline sample, and inferring candidate somatic mutations by way of comparison. A multitude of tools exist for this purpose. In practice, however, sequencing artifacts or alignment errors are often mistakenly flagged as variants, necessitating extremely time-consuming manual validation by researchers.We demonstrate that this process can be largely automated using Deep Convolutional Neural Networks, whose utility has been a driving force behind many recent advances in applied machine learning. Using previously performed manual annotation as input data, we train a Deep Convolutional Neural Network of straightforward topology that recognises sequencing artifacts in called variants with high accuracy, achieving a score of 97.5% on a validation dataset. Moreover, its direct outputs are class probabilities instead of binary labels, and the remaining misclassified points lie in the region of low certainty, suggesting an effective modelling of the decision behaviour in manual annotation. This allows for a significant reduction in the workload for researchers, and can in the future be integrated into bioinformatics workflows for NGS data processing.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-5 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-5.html">
											    	</br><strong>Arran Pack</strong>
									    	 
									    		<em>Brighton and Sussex Medical School</em>
									    	
									    	
									    		
									    			</br>"Overcoming receptor proximal mutations in DLBCL through systems modelling"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			B Cell Receptors react to antigen stimulus, transducing signal to NFkB, and triggering an immune response. Inoue et al[1] modelled this pathway in health. B Cell Lymphoma frequently displays chronically active NFkB caused by mutations in this receptor-proximal signalling pathway. We sought to mechanistically investigate the impact of mutation on BCR signalling and predict therapeutic interventions.A common mutation in CARD11 was modelled by modifying parameters corresponding with experimentally-identified effects of this mutation. In response to stimulation, the mutant CARD11 model is significantly more active; and switches to chronically active when the basal BCR signal exceeds a very low level (<1%). This low activation threshold suggests that the CARD11 mutation alone may be sufficient for chronic NFkB activation.By performing parameter sensitivity analysis on the mutant model; the parameters with the greatest influence on NFkB were identified. This approach identified several IKK reactions which have been the target of much therapeutic development. Unfortunately, due to the ubiquitous expression of NFkB and lack of specificity, development of these inhibitors has generally stalled. This work is motivating ongoing expansion of the model to include the Cheng et al 2015 model of Toll Like Receptor signalling, to increase the number of mutable/druggable targets.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-6 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-6.html">
											    	</br><strong>Sara Hamis</strong>
									    	 
									    		<em>School of Mathematics and Statistics, University of St Andrews, St Andrews, Scotland, UK.</em>
									    	
									    	
									    		
									    			</br>"Mathematical modelling quantifies ERK-activity in response to vertical inhibition of the BRAFV600E-MEK-ERK cascade in melanoma"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Vertical inhibition of the BRAF-MEK-ERK cascade has become a standard of care for treating BRAF-mutant melanoma. However, the molecular mechanisms of how vertical inhibition synergistically suppresses intracellular ERK activity, and by extension cell proliferation, are yet to be fully elucidated. In this study, we develop a mechanistic mathematical model that describes how the BRAFV600E-inhibitor dabrafenib, and the MEK-inhibitor trametinib target the BRAFV600E-MEK-ERK cascade. We formulate a system of chemical reactions that describes cascade signalling dynamics. Using mass action kinetics, the system of chemical reactions is re-expressed as a system of ordinary differential equations, which we solve numerically to obtain the temporal evolution of cascade component concentrations.Our mathematical model provides a quantitative method to compute how dabrafenib and trametinib can be used in combination to synergistically inhibit ERK activity in BRAFV600E mutant cancer cells. Our work elucidates molecular mechanisms of vertical inhibition of the BRAFV600E-MEK-ERK cascade, and delineates how elevated cellular BRAF concentrations generate drug resistance to dabrafenib and trametinib. In addition, our model results suggest that elevated ATP levels lead to reduced sensitivity to dabrafenib.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-7 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-7.html">
											    	</br><strong>Ryan Schenck</strong>
									    	 
									    		<em>Integrated Mathematical Oncology, Moffitt Cancer Center; Intestinal Stem Cell Biology Lab, Wellcome Centre for Human Genetics, University of Oxford</em>
									    	
									    	
									    		
									    			</br>"Reconstructing Contemporary Human Stem Cell Dynamics with Oscillatory Molecular Clocks"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Cell histories can be reconstructed from their genomes by analysing 'molecular clocks' that accumulate heritable changes through time. Commonly used clocks, such as the accumulation of single nucleotide variants or DNA methylation, slowly change over decades, recording cell dynamics that occur over long timescales corresponding to the change accumulation (tick) rate. Faster clocks saturate and stop recording early in life, precluding the study of short-timescale cell dynamics such as renewal in adult tissues. Here we develop a new method that can measure contemporary adult cell dynamics with rapidly oscillating CpG DNA methylation, where like a pendulum, ongoing 'tick-tock' (de)methylation causes switching between 0, 50 and 100% methylation at each CpG locus in a diploid cell. In polyclonal cell populations, average oscillator methylation is ~50%, but “W-shaped” distributions with modal peaks at 0, 50 and 100% methylation are evident in clonal populations. The precise shape of the W-distribution is determined by the underlying dynamics of cell growth and replacement. Through our work, we've illustrated oscillator DNA methylation can be measured in many human tissues cheaply and routinely and enables the inference of otherwise elusive contemporary dynamics of normal and abnormal somatic cells.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
								    	
									    	<p><small class="card-body-subgroup"><strong><h5  class="text-ONCO">ONCO</h5> </strong>ONCO-8 (Session: PS01)</small>
										    	<a href="/ONCO/ONCO-PS01-ONCO-8.html">
											    	</br><strong>Veselin Manojlović</strong>
									    	 
									    		<em>School of Mathematics, Computer Science and Engineering, City, University of London</em>
									    	
									    	
									    		
									    			</br>"Evolutionary Indeces for Classifying Modes of Tumour Evolution"</p>
									    		
									    	
									    	
									    	
										    	<article class="article-post text-justify" style="font-size:12px; margin-bottom: 10px; margin-top: 10px;">
										    		<div class="hide-abstracts">
										    			Trees are a useful mathematical tool in evolutionary theory, especially when describing the structure of an evolving population. In mathematical oncology, clonal trees can be used to construct the evolutionary history of a tumour, and tree balance and diversity indices further classify it within an oncoevotype.While obtaining diversity and balance indices for a specific tree is straightforward, going in reverse is not a trivial matter. One of our main goals is classifying clonal trees based on a minimal set of indices. To this end, we explore properties of new and existing balance and diversity indices, along with their mutual dependencies. This should lay the groundwork for further investigation of the possibility of defining an index metric, which would mark a step towards an analytic formulation of mathematical oncology.
										    		</div>
										    	</article>
									    	
									    	</a>
									    
									  
										
										
										
		                
		                
		                
		                
		            </div>             
		        </div>
		    </div>
			</div>
			
						
			</br>
			</br>
			
			<div class="row">
			  <div class="col-md-5 offset-md-3">
				  <!--
<ul class="pagination justify-content-center">
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/">Overview</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/sunday">Sunday</a></li>
  <li class="page-item" ><a class="page-link" style="color:#000000 !important;" href="/monday">Monday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/tuesday">Tuesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/wednesday">Wednesday</a></li>
  <li class="page-item"><a class="page-link" style="color:#000000 !important;" href="/thursday">Thursday</a></li>
</ul>
-->
			  </div>
			</div>
			
		</div>
	</div>
	
	
</div>

<div class="alertbar">
    <div class="container">
        <div class="row prevnextlinks small font-weight-bold">
	            <div class="col-12 text-center pl-0">		            
									<span class="click-legend" id="switch-text" onclick="abstractsHide()">
										Click to show abstracts
									</span>
			            </br>
									<hr>
									<small>
										Posters-01
										</br>
										Monday, June 14 at 11:30pm (PDT)</br>Tuesday, June 15 at 07:30am (BST)</br>Tuesday, June 15 03:30pm (KST)
									</small>        
	            </div>
        </div>
    </div>
</div>
















    </main>


    <!-- Scripts: popper, bootstrap, theme, lunr -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>

    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>


    
    
			<script src="../assets/js/theme.js"></script>
		
    
    <!-- All this area goes before </body> closing tag --> 


</body>

</html>
